<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Part 1 Introduction | HP1 Deficiency Results in Accelerated Neurodegeneration</title>
  <meta name="description" content="Part 1 Introduction | HP1 Deficiency Results in Accelerated Neurodegeneration">
  <meta name="generator" content="bookdown  and GitBook 2.6.7">

  <meta property="og:title" content="Part 1 Introduction | HP1 Deficiency Results in Accelerated Neurodegeneration" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Part 1 Introduction | HP1 Deficiency Results in Accelerated Neurodegeneration" />
  
  
  

<meta name="author" content="Andrew G. Newman">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="acknowledgements.html">
<link rel="next" href="2-results.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />







<script src="libs/kePrint-0.0.1/kePrint.js"></script>


<style type="text/css">
div.sourceCode { overflow-x: auto; }
table.sourceCode, tr.sourceCode, td.lineNumbers, td.sourceCode {
  margin: 0; padding: 0; vertical-align: baseline; border: none; }
table.sourceCode { width: 100%; line-height: 100%; }
td.lineNumbers { text-align: right; padding-right: 4px; padding-left: 4px; color: #aaaaaa; border-right: 1px solid #aaaaaa; }
td.sourceCode { padding-left: 5px; }
code > span.kw { color: #007020; font-weight: bold; } /* Keyword */
code > span.dt { color: #902000; } /* DataType */
code > span.dv { color: #40a070; } /* DecVal */
code > span.bn { color: #40a070; } /* BaseN */
code > span.fl { color: #40a070; } /* Float */
code > span.ch { color: #4070a0; } /* Char */
code > span.st { color: #4070a0; } /* String */
code > span.co { color: #60a0b0; font-style: italic; } /* Comment */
code > span.ot { color: #007020; } /* Other */
code > span.al { color: #ff0000; font-weight: bold; } /* Alert */
code > span.fu { color: #06287e; } /* Function */
code > span.er { color: #ff0000; font-weight: bold; } /* Error */
code > span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
code > span.cn { color: #880000; } /* Constant */
code > span.sc { color: #4070a0; } /* SpecialChar */
code > span.vs { color: #4070a0; } /* VerbatimString */
code > span.ss { color: #bb6688; } /* SpecialString */
code > span.im { } /* Import */
code > span.va { color: #19177c; } /* Variable */
code > span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code > span.op { color: #666666; } /* Operator */
code > span.bu { } /* BuiltIn */
code > span.ex { } /* Extension */
code > span.pp { color: #bc7a00; } /* Preprocessor */
code > span.at { color: #7d9029; } /* Attribute */
code > span.do { color: #ba2121; font-style: italic; } /* Documentation */
code > span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code > span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code > span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
</style>

</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="preface.html"><a href="preface.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="" data-path="contribution-of-authors.html"><a href="contribution-of-authors.html"><i class="fa fa-check"></i>Contribution of Authors</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="1" data-path="1-introduction.html"><a href="1-introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="1-introduction.html"><a href="1-introduction.html#epigenetics"><i class="fa fa-check"></i><b>1.1</b> Epigenetics</a><ul>
<li><a href="1-introduction.html#dna-modifications"><em>DNA Modifications</em></a></li>
<li><a href="1-introduction.html#chromatin"><em>Chromatin</em></a></li>
<li><a href="1-introduction.html#chromatin-remodelling"><em>Chromatin Remodelling</em></a></li>
<li><a href="1-introduction.html#krab-zfps-transposable-elements-and-exaptation"><em>KRAB-ZFPs, Transposable Elements and Exaptation</em></a></li>
<li><a href="1-introduction.html#heterochromatinization-and-hp1"><em>Heterochromatinization and HP1</em></a></li>
<li><a href="1-introduction.html#non-coding-rna"><em>Non-Coding RNA</em></a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="1-introduction.html"><a href="1-introduction.html#the-heterochromatin-loss-model-of-aging"><i class="fa fa-check"></i><b>1.2</b> The Heterochromatin Loss Model of Aging</a></li>
<li class="chapter" data-level="1.3" data-path="1-introduction.html"><a href="1-introduction.html#the-aging-neuronal-epigenome"><i class="fa fa-check"></i><b>1.3</b> The Aging Neuronal Epigenome</a><ul>
<li class="chapter" data-level="1.3.1" data-path="1-introduction.html"><a href="1-introduction.html#DNAging"><i class="fa fa-check"></i><b>1.3.1</b> <em>DNA Methylation and Hydroxymethylation in Aged Neurons</em></a></li>
<li class="chapter" data-level="1.3.2" data-path="1-introduction.html"><a href="1-introduction.html#chraging"><i class="fa fa-check"></i><b>1.3.2</b> <em>Chromatin in the Aging Brain</em></a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="1-introduction.html"><a href="1-introduction.html#objective"><i class="fa fa-check"></i><b>1.4</b> Objective</a></li>
<li class="chapter" data-level="1.5" data-path="1-introduction.html"><a href="1-introduction.html#hp1-proteins---a-background"><i class="fa fa-check"></i><b>1.5</b> HP1 Proteins - A Background</a><ul>
<li><a href="1-introduction.html#the-abgs-of-hp1-proteins">The <em>ABG’s</em> of HP1 Proteins</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-results.html"><a href="2-results.html"><i class="fa fa-check"></i><b>2</b> Results</a><ul>
<li class="chapter" data-level="2.1" data-path="2-results.html"><a href="2-results.html#HP1aT"><i class="fa fa-check"></i><b>2.1</b> HP1FEC, HP1TKO and HP1<span class="math inline">\(\alpha\)</span>T</a></li>
<li class="chapter" data-level="2.2" data-path="2-results.html"><a href="2-results.html#expression-of-hp1-proteins-in-the-brain"><i class="fa fa-check"></i><b>2.2</b> Expression of HP1 Proteins in the Brain</a></li>
<li class="chapter" data-level="2.3" data-path="2-results.html"><a href="2-results.html#mosaic-deletion-of-hp1beta-and-hp1gamma-from-cortical-progenitors"><i class="fa fa-check"></i><b>2.3</b> Mosaic Deletion of HP1<span class="math inline">\(\beta\)</span> and HP1<span class="math inline">\(\gamma\)</span> from Cortical Progenitors</a></li>
<li class="chapter" data-level="2.4" data-path="2-results.html"><a href="2-results.html#deletion-of-hp1beta-hp1gamma-from-the-emx1-lineage."><i class="fa fa-check"></i><b>2.4</b> Deletion of <em>HP1<span class="math inline">\(\beta\)</span></em> &amp; <em>HP1<span class="math inline">\(\gamma\)</span></em> from the <em>Emx1</em>-lineage.</a><ul>
<li class="chapter" data-level="2.4.1" data-path="2-results.html"><a href="2-results.html#cell-fate-in-cortical-neurons"><i class="fa fa-check"></i><b>2.4.1</b> Cell Fate in Cortical Neurons</a></li>
<li class="chapter" data-level="2.4.2" data-path="2-results.html"><a href="2-results.html#imprint"><i class="fa fa-check"></i><b>2.4.2</b> Imprinted Loci</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="2-results.html"><a href="2-results.html#perturbed-regulation-of-retroelements-in-hp1-mutants"><i class="fa fa-check"></i><b>2.5</b> Perturbed Regulation of Retroelements in HP1 Mutants</a><ul>
<li class="chapter" data-level="2.5.1" data-path="2-results.html"><a href="2-results.html#line1"><i class="fa fa-check"></i><b>2.5.1</b> LINE1</a></li>
<li class="chapter" data-level="2.5.2" data-path="2-results.html"><a href="2-results.html#class-iii-ervs"><i class="fa fa-check"></i><b>2.5.2</b> Class III ERVs</a></li>
<li class="chapter" data-level="2.5.3" data-path="2-results.html"><a href="2-results.html#class-ii-ervs"><i class="fa fa-check"></i><b>2.5.3</b> Class II ERVs</a></li>
<li class="chapter" data-level="2.5.4" data-path="2-results.html"><a href="2-results.html#de-novo-retrotransposition"><i class="fa fa-check"></i><b>2.5.4</b> <em>De Novo</em> Retrotransposition</a></li>
</ul></li>
<li class="chapter" data-level="2.6" data-path="2-results.html"><a href="2-results.html#sustained-hp1betagamma-deficiency-adversely-affects-neuronal-complexity-function-and-cognition"><i class="fa fa-check"></i><b>2.6</b> Sustained HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> Deficiency Adversely Affects Neuronal Complexity, Function and Cognition</a><ul>
<li class="chapter" data-level="2.6.1" data-path="2-results.html"><a href="2-results.html#behavior"><i class="fa fa-check"></i><b>2.6.1</b> Behavior</a></li>
<li class="chapter" data-level="2.6.2" data-path="2-results.html"><a href="2-results.html#MRIresults"><i class="fa fa-check"></i><b>2.6.2</b> Structure, Volumetrics and Histology</a></li>
</ul></li>
<li class="chapter" data-level="2.7" data-path="2-results.html"><a href="2-results.html#hp1beta-and-hp1gamma-synergistically-contribute-to-repeat-silencing"><i class="fa fa-check"></i><b>2.7</b> HP1<span class="math inline">\(\beta\)</span> and HP1<span class="math inline">\(\gamma\)</span> Synergistically Contribute to Repeat Silencing</a><ul>
<li class="chapter" data-level="2.7.1" data-path="2-results.html"><a href="2-results.html#hp1gamma-is-required-for-h4k20me3-deposition-in-neurons"><i class="fa fa-check"></i><b>2.7.1</b> HP1<span class="math inline">\(\gamma\)</span> is Required for H4K20me3 Deposition in Neurons</a></li>
<li class="chapter" data-level="2.7.2" data-path="2-results.html"><a href="2-results.html#hp1gamma-binds-suv420h2-via-the-chromoshadow-domain"><i class="fa fa-check"></i><b>2.7.2</b> HP1<span class="math inline">\(\gamma\)</span> binds SUV420H2 via the Chromoshadow Domain</a></li>
<li class="chapter" data-level="2.7.3" data-path="2-results.html"><a href="2-results.html#re-addition-of-hp1gamma-can-restore-h4k20me3-in-vivo"><i class="fa fa-check"></i><b>2.7.3</b> Re-addition of HP1<span class="math inline">\(\gamma\)</span> can restore H4K20me3 In Vivo</a></li>
<li class="chapter" data-level="2.7.4" data-path="2-results.html"><a href="2-results.html#h4k20me3-loss-is-correlated-with-iap-de-repression"><i class="fa fa-check"></i><b>2.7.4</b> H4K20me3 Loss is Correlated with IAP De-repression</a></li>
<li class="chapter" data-level="2.7.5" data-path="2-results.html"><a href="2-results.html#kap1-h3k9me3-and-h4k20me3-are-lost-at-iap-elements-in-hp1ctko-es-cells"><i class="fa fa-check"></i><b>2.7.5</b> KAP1, H3K9me3 and H4K20me3 are lost at IAP Elements in HP1cTKO ES Cells</a></li>
<li class="chapter" data-level="2.7.6" data-path="2-results.html"><a href="2-results.html#dna-methylation-is-affected-following-loss-of-hp1"><i class="fa fa-check"></i><b>2.7.6</b> DNA Methylation is affected Following loss of HP1</a></li>
<li><a href="2-results.html#kap1-is-lost-from-icrs-in-hp1ctko-cells"><em>KAP1 is lost from ICRs in HP1cTKO Cells</em></a></li>
</ul></li>
<li class="chapter" data-level="2.8" data-path="2-results.html"><a href="2-results.html#RNAresults"><i class="fa fa-check"></i><b>2.8</b> Transcriptional Dysregulation following HP1 Deficiency Drives UPR and Inflammatory Responses</a><ul>
<li class="chapter" data-level="2.8.1" data-path="2-results.html"><a href="2-results.html#genotype-related-changes"><i class="fa fa-check"></i><b>2.8.1</b> Genotype Related Changes</a></li>
<li class="chapter" data-level="2.8.2" data-path="2-results.html"><a href="2-results.html#longitudinal-changes-to-gene-expression"><i class="fa fa-check"></i><b>2.8.2</b> Longitudinal Changes To Gene Expression</a></li>
<li class="chapter" data-level="2.8.3" data-path="2-results.html"><a href="2-results.html#pathway-analysis"><i class="fa fa-check"></i><b>2.8.3</b> Pathway Analysis</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-discussion.html"><a href="3-discussion.html"><i class="fa fa-check"></i><b>3</b> Discussion</a><ul>
<li class="chapter" data-level="3.1" data-path="3-discussion.html"><a href="3-discussion.html#phenotypes"><i class="fa fa-check"></i><b>3.1</b> Phenotypes</a><ul>
<li><a href="3-discussion.html#structural-changes-in-the-hippocampus"><em>Structural Changes in the Hippocampus</em></a></li>
<li><a href="3-discussion.html#reduction-in-cortical-volume"><em>Reduction in Cortical Volume</em></a></li>
<li><a href="3-discussion.html#behavior-and-impairments-in-learning-memory"><em>Behavior and Impairments in Learning &amp; Memory</em></a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="3-discussion.html"><a href="3-discussion.html#hp1-dependent-epigenetic-systems"><i class="fa fa-check"></i><b>3.2</b> HP1-dependent Epigenetic Systems</a><ul>
<li><a href="3-discussion.html#hp1gamma"><em>HP1<span class="math inline">\(\gamma\)</span></em></a></li>
<li><a href="3-discussion.html#hp1beta"><em>HP1<span class="math inline">\(\beta\)</span></em></a></li>
<li><a href="3-discussion.html#hp1beta-x-hp1gamma"><em>HP1<span class="math inline">\(\beta\)</span></em> x <em>HP1<span class="math inline">\(\gamma\)</span></em></a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="3-discussion.html"><a href="3-discussion.html#concordance-with-aging-and-disease"><i class="fa fa-check"></i><b>3.3</b> Concordance with Aging and Disease</a><ul>
<li><a href="3-discussion.html#complement-in-cns-aging-and-disease"><em>Complement in CNS Aging and Disease</em></a></li>
<li><a href="3-discussion.html#a-pathway-to-complement"><em>A Pathway to Complement</em></a></li>
<li><a href="3-discussion.html#unfolded-protein-response-upr"><em>Unfolded Protein Response (UPR)</em></a></li>
<li><a href="3-discussion.html#viruses-transactivation-and-neurodegeneration"><em>Viruses, Transactivation and Neurodegeneration</em></a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="3-discussion.html"><a href="3-discussion.html#concluding-remarks"><i class="fa fa-check"></i><b>3.4</b> Concluding Remarks</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-methods.html"><a href="4-methods.html"><i class="fa fa-check"></i><b>4</b> Methods</a><ul>
<li class="chapter" data-level="4.1" data-path="4-methods.html"><a href="4-methods.html#mouse-lines"><i class="fa fa-check"></i><b>4.1</b> Mouse Lines</a><ul>
<li class="chapter" data-level="4.1.1" data-path="4-methods.html"><a href="4-methods.html#HP1anull"><i class="fa fa-check"></i><b>4.1.1</b> Generation of HP1<span class="math inline">\(\alpha\)</span>KO and HP1<span class="math inline">\(\alpha\)</span>T</a></li>
<li class="chapter" data-level="4.1.2" data-path="4-methods.html"><a href="4-methods.html#HP1Bgtarget"><i class="fa fa-check"></i><b>4.1.2</b> Generation of Floxed HP1<span class="math inline">\(\beta\)</span> &amp; Floxed HP1<span class="math inline">\(\gamma\)</span></a></li>
<li class="chapter" data-level="4.1.3" data-path="4-methods.html"><a href="4-methods.html#genotyping"><i class="fa fa-check"></i><b>4.1.3</b> Genotyping</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="4-methods.html"><a href="4-methods.html#cloning"><i class="fa fa-check"></i><b>4.2</b> Cloning</a><ul>
<li class="chapter" data-level="4.2.1" data-path="4-methods.html"><a href="4-methods.html#rna-isolation-cdna-generation-rnaseq"><i class="fa fa-check"></i><b>4.2.1</b> RNA Isolation, cDNA Generation, &amp; RNAseq</a></li>
<li class="chapter" data-level="4.2.2" data-path="4-methods.html"><a href="4-methods.html#cloning-of-rna-probes-for-in-situ-hybridization"><i class="fa fa-check"></i><b>4.2.2</b> Cloning of RNA Probes for <em>In-Situ Hybridization</em></a></li>
<li class="chapter" data-level="4.2.3" data-path="4-methods.html"><a href="4-methods.html#cloning-of-expression-constructs"><i class="fa fa-check"></i><b>4.2.3</b> Cloning of Expression Constructs</a></li>
<li class="chapter" data-level="4.2.4" data-path="4-methods.html"><a href="4-methods.html#site-directed-mutagenesis"><i class="fa fa-check"></i><b>4.2.4</b> Site Directed Mutagenesis</a></li>
<li class="chapter" data-level="4.2.5" data-path="4-methods.html"><a href="4-methods.html#transformation"><i class="fa fa-check"></i><b>4.2.5</b> Transformation</a></li>
<li class="chapter" data-level="4.2.6" data-path="4-methods.html"><a href="4-methods.html#clonal-selection-plasmid-production"><i class="fa fa-check"></i><b>4.2.6</b> Clonal Selection &amp; Plasmid Production</a></li>
<li class="chapter" data-level="4.2.7" data-path="4-methods.html"><a href="4-methods.html#in-vitro-transcription-ivt"><i class="fa fa-check"></i><b>4.2.7</b> In Vitro Transcription (IVT)</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="4-methods.html"><a href="4-methods.html#cell-culture"><i class="fa fa-check"></i><b>4.3</b> Cell Culture</a><ul>
<li class="chapter" data-level="4.3.1" data-path="4-methods.html"><a href="4-methods.html#primary-neuronal-culture"><i class="fa fa-check"></i><b>4.3.1</b> Primary Neuronal Culture</a></li>
<li class="chapter" data-level="4.3.2" data-path="4-methods.html"><a href="4-methods.html#hek293t-culture-transfection"><i class="fa fa-check"></i><b>4.3.2</b> HEK293T Culture &amp; Transfection</a></li>
<li class="chapter" data-level="4.3.3" data-path="4-methods.html"><a href="4-methods.html#co-immunoprecipitation-co-ip-and-western-blot-wb"><i class="fa fa-check"></i><b>4.3.3</b> Co-Immunoprecipitation (Co-IP) and Western Blot (WB)</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="4-methods.html"><a href="4-methods.html#in-utero-electroporation-tissue-processing-and-histology"><i class="fa fa-check"></i><b>4.4</b> In Utero Electroporation, Tissue Processing and Histology</a><ul>
<li class="chapter" data-level="4.4.1" data-path="4-methods.html"><a href="4-methods.html#IUEmethod"><i class="fa fa-check"></i><b>4.4.1</b> In Utero Electroporation (IUE)</a></li>
<li class="chapter" data-level="4.4.2" data-path="4-methods.html"><a href="4-methods.html#tissue-fixation-and-perfusion"><i class="fa fa-check"></i><b>4.4.2</b> Tissue Fixation and Perfusion</a></li>
<li class="chapter" data-level="4.4.3" data-path="4-methods.html"><a href="4-methods.html#cryosectioning"><i class="fa fa-check"></i><b>4.4.3</b> Cryosectioning</a></li>
<li class="chapter" data-level="4.4.4" data-path="4-methods.html"><a href="4-methods.html#immuno-histochemistry-ihc"><i class="fa fa-check"></i><b>4.4.4</b> Immuno Histochemistry (IHC)</a></li>
<li class="chapter" data-level="4.4.5" data-path="4-methods.html"><a href="4-methods.html#quantification-of-cells-in-the-dentate-gyrus"><i class="fa fa-check"></i><b>4.4.5</b> Quantification of Cells in the Dentate Gyrus</a></li>
<li class="chapter" data-level="4.4.6" data-path="4-methods.html"><a href="4-methods.html#in-situ-hybridization-ish"><i class="fa fa-check"></i><b>4.4.6</b> In Situ Hybridization (ISH)</a></li>
<li class="chapter" data-level="4.4.7" data-path="4-methods.html"><a href="4-methods.html#paraffin-sectioning"><i class="fa fa-check"></i><b>4.4.7</b> Paraffin Sectioning</a></li>
<li class="chapter" data-level="4.4.8" data-path="4-methods.html"><a href="4-methods.html#nissl"><i class="fa fa-check"></i><b>4.4.8</b> Nissl Stain</a></li>
<li class="chapter" data-level="4.4.9" data-path="4-methods.html"><a href="4-methods.html#golgi-impregnation-and-dendritic-complexity-analysis"><i class="fa fa-check"></i><b>4.4.9</b> Golgi Impregnation and Dendritic Complexity Analysis</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="4-methods.html"><a href="4-methods.html#behavioral-experiments"><i class="fa fa-check"></i><b>4.5</b> Behavioral Experiments</a><ul>
<li class="chapter" data-level="4.5.1" data-path="4-methods.html"><a href="4-methods.html#open-field-activity-1"><i class="fa fa-check"></i><b>4.5.1</b> Open Field Activity</a></li>
<li class="chapter" data-level="4.5.2" data-path="4-methods.html"><a href="4-methods.html#paired-pulse-inhibition-1"><i class="fa fa-check"></i><b>4.5.2</b> Paired-Pulse Inhibition</a></li>
<li class="chapter" data-level="4.5.3" data-path="4-methods.html"><a href="4-methods.html#context-cued-fear-conditioning"><i class="fa fa-check"></i><b>4.5.3</b> Context-Cued Fear Conditioning</a></li>
<li class="chapter" data-level="4.5.4" data-path="4-methods.html"><a href="4-methods.html#Barnesmeth"><i class="fa fa-check"></i><b>4.5.4</b> Barnes Maze</a></li>
<li class="chapter" data-level="4.5.5" data-path="4-methods.html"><a href="4-methods.html#social-activity-monitor-sam-1"><i class="fa fa-check"></i><b>4.5.5</b> Social Activity Monitor (SAM)</a></li>
<li class="chapter" data-level="4.5.6" data-path="4-methods.html"><a href="4-methods.html#micro-behaviors-homecagescan"><i class="fa fa-check"></i><b>4.5.6</b> Micro Behaviors (HomeCageScan)</a></li>
<li class="chapter" data-level="4.5.7" data-path="4-methods.html"><a href="4-methods.html#nestcomethod"><i class="fa fa-check"></i><b>4.5.7</b> Nest Construction</a></li>
</ul></li>
<li class="chapter" data-level="4.6" data-path="4-methods.html"><a href="4-methods.html#magnetic-resonance-imaging-mri-and-volumetrics"><i class="fa fa-check"></i><b>4.6</b> Magnetic Resonance Imaging (MRI) and Volumetrics</a></li>
<li class="chapter" data-level="4.7" data-path="4-methods.html"><a href="4-methods.html#reduced-read-bisulfite-sequencing-rrbseq"><i class="fa fa-check"></i><b>4.7</b> Reduced Read Bisulfite Sequencing (RRBSeq)</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html"><i class="fa fa-check"></i><b>5</b> Supplemental Data</a><ul>
<li class="chapter" data-level="5.1" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html#Behavsupplement"><i class="fa fa-check"></i><b>5.1</b> Behavior</a><ul>
<li class="chapter" data-level="5.1.1" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html#homecagescan-micro-behaviors"><i class="fa fa-check"></i><b>5.1.1</b> HomeCageScan Micro Behaviors</a></li>
<li class="chapter" data-level="5.1.2" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html#context-cued-fear-conditioning-1"><i class="fa fa-check"></i><b>5.1.2</b> Context-Cued Fear Conditioning</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html#DE"><i class="fa fa-check"></i><b>5.2</b> Differential Gene Expression</a><ul>
<li class="chapter" data-level="5.2.1" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html#differentially-expressed-genes-in-aged-hp1betako"><i class="fa fa-check"></i><b>5.2.1</b> Differentially Expressed Genes in Aged HP1<span class="math inline">\(\beta\)</span>KO</a></li>
<li class="chapter" data-level="5.2.2" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html#inflam"><i class="fa fa-check"></i><b>5.2.2</b> Genes Related to Anti-viral and Interferon Response in HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html#genomic-hotspots-sensitive-to-hp1-deficiency-in-the-hippocampus"><i class="fa fa-check"></i><b>5.3</b> Genomic Hotspots Sensitive to HP1 Deficiency in the Hippocampus</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html"><i class="fa fa-check"></i><b>6</b> Bioinformatics &amp; Data Processing</a><ul>
<li class="chapter" data-level="6.1" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#software-environment-management"><i class="fa fa-check"></i><b>6.1</b> Software Environment Management</a></li>
<li class="chapter" data-level="6.2" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#snakemake-jobs"><i class="fa fa-check"></i><b>6.2</b> Snakemake Jobs</a></li>
<li class="chapter" data-level="6.3" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#tetranscripts-pipeline"><i class="fa fa-check"></i><b>6.3</b> TETranscripts Pipeline</a></li>
<li class="chapter" data-level="6.4" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#differential-gene-expression-using-edger"><i class="fa fa-check"></i><b>6.4</b> Differential Gene Expression Using EdgeR</a></li>
<li class="chapter" data-level="6.5" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#deeptools-pipeline"><i class="fa fa-check"></i><b>6.5</b> Deeptools Pipeline</a></li>
<li class="chapter" data-level="6.6" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#lions-pipeline"><i class="fa fa-check"></i><b>6.6</b> LIONS Pipeline</a></li>
<li class="chapter" data-level="6.7" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#analysis-of-reduced-read-bisulphite-sequencing-rrbs"><i class="fa fa-check"></i><b>6.7</b> Analysis of Reduced-Read Bisulphite Sequencing (RRBS)</a><ul>
<li class="chapter" data-level="6.7.1" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#methylkit-and-circos-plot"><i class="fa fa-check"></i><b>6.7.1</b> Methylkit and Circos Plot</a></li>
<li class="chapter" data-level="6.7.2" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#annotated-ideogram"><i class="fa fa-check"></i><b>6.7.2</b> Annotated Ideogram</a></li>
</ul></li>
<li class="chapter" data-level="6.8" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#about-this-thesis"><i class="fa fa-check"></i><b>6.8</b> About this Thesis</a><ul>
<li class="chapter" data-level="" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#session-info"><i class="fa fa-check"></i>Session Info</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7" data-path="7-references.html"><a href="7-references.html"><i class="fa fa-check"></i><b>7</b> References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">HP1 Deficiency Results in Accelerated Neurodegeneration</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="introduction" class="section level1">
<h1><span class="header-section-number">Part 1</span> Introduction</h1>
<div id="epigenetics" class="section level2">
<h2><span class="header-section-number">1.1</span> Epigenetics</h2>
<p>While typically seen as a recent field of study, Epigenetics, or at least a broader form of it, was conceptualized in the classical era by perhaps the earliest embryologist, Aristotle (384-322 BCE). In his <em>Generation of Animals</em> <span class="citation">(Aristotle and Peck <a href="#ref-AristotleGenerationanimalsEnglish1943">1943</a>)</span>, Aristotle contrasts his observations of developing chick embryos against the theory of preformation, which stated the progeny of animals was derived from very tiny ‘pre-formed’ versions of the complete organism. Upon observing the gradual transformations that occur during the development of the chick embryo, Aristotle posited a theory of ‘epigenesis’ (above the source) which stated that an organism gradually develops from an undifferentiated material. The word ‘epigenesis’ re-emerged in the english language in the early 17<sup>th</sup> century <span class="citation">(Harvey <a href="#ref-HarveyAnatomicalexercitationsconcerning1653">1653</a>)</span>, where its meaning was defined by oxford as <em>‘partium super-exorientium additamentum’</em>, ‘the additament of parts budding one out of another’ or simply ‘extra-growth’.</p>
<p>At the beginning of the 20<sup>th</sup> century, pre-molecular embryologists were beginning to characterize developmental processes in the wake of Darwin and Lamarck. Several experiments performed at this time led many to the conclusion that the sequence of events that took place from ‘undifferentiated material’ into different tissues, and eventually to a fully functional organism were largely irreversible. From the common usage and its adjective <em>epigenetic</em>, Conrad Hal Waddington (1905-1975) coined the term <em>epigenetics</em> to collectively refer to these processes. Waddington ascribed an “epigenetic landscape” <span class="citation">(Waddington <a href="#ref-WaddingtonOrganisersGenes1940">1940</a>)</span> metaphor to developmental processes, where a marble at the top of a hill (undifferentiated cell) could freely roll downhill into one of many valleys (specific differentiation pathway). Once downhill, the marble could not spontaneously change valleys (transdifferentiate), as it is fate-committed. This popular metaphor remains relevant to this day. Most modern cellular reprogramming requires that you push the marble back up the hill (at least partway), before re-selecting your appropriate valley, although there are emerging cases where direct reprogramming is possible.</p>
<div class="figure" style="text-align: center"><span id="fig:landscape"></span>
<img src="figure/introduction/image_epigenetic.jpg" alt="Conrad H. Waddington's Epigenetic Landscape (Organisers and Genes, 1940)" width="100%"   />
<p class="caption">
Figure 1.1: Conrad H. Waddington’s Epigenetic Landscape (Organisers and Genes, 1940)
</p>
</div>
<p>Given its scope, in that the contemporary study of epigenetics ranges from developmental processes, to imprinted parent-of-origin effects, to transgenerational effects originating from parental stressors, a meeting at Cold Spring Harbor in 2008 formulated the consensus definition of <em>epigenetics</em> to be: “[a] stably heritable phenotype resulting from changes in a chromosome without alterations in the DNA sequence”. Here, ‘stably heritable’ is implied to be either through mitosis or meiosis. However, as we will see, changes can occur to the epigenome in fully differentiated somatic cells, which can often affect their function. While such changes are not directly heritable, they are not considered to be any less <em>epigenetic</em>. While an organism’s genome confers cellular identity through the combinatorial expression of a unique set of genes, it is the superimposed epigenome that maintains the integrity of said identity. To return to Waddington’s Epigenetic Landscape, the physical deposition of chemical modifications to histones and DNA in the creation of heterochromatin generates hills and barriers to not only facilitate cellular differentiation, but to restrict later de-differentiation.</p>
<p>The perturbation of this ‘epigenetic integrity’, which ensures cell-type specific gene transcription and splicing, can result in altered cellular states that lead to altered protein synthesis <span class="citation">(Feinberg <a href="#ref-Feinbergkeyroleepigenetics2018">2018</a>)</span> and reduced cellular metabolism <span class="citation">(Ferrari et al. <a href="#ref-FerrariEpigenomemodifiersmetabolic2018">2018</a>)</span>, loss of identity <span class="citation">(Cheloufi and Hochedlinger <a href="#ref-CheloufiEmergingroleshistone2017">2017</a>; Cheloufi et al. <a href="#ref-CheloufihistonechaperoneCAF12015">2015</a>)</span>, cell death <span class="citation">(Karlic et al. <a href="#ref-Karlicroleepigeneticsregulation2014">2014</a>)</span>, and even cancer <span class="citation">(Campbell and Wellen <a href="#ref-CampbellMetabolicSignalingNucleus2018">2018</a>; Kinnaird et al. <a href="#ref-KinnairdMetaboliccontrolepigenetics2016">2016</a>)</span>. In addition to epigenetic changes that induce strong cellular changes (i.e. those that are directly pathogenic or tumorigenic), a growing body of evidence points to a ‘heterochromatin loss’ or ‘epigenetic drift’ theory of aging - whereby epigenome integrity is slowly compromised, leading to multiple compounding cascades which eventually result in cellular failure.</p>
<p>While epigenome integrity is no doubt important in all cell types, it is especially important to understand how these epigenetic processes are vulnerable in neurons, which are not only non-dividing, but whose high metabolism leaves them highly susceptible to DNA damage.</p>
<p>The epigenome is dictated by several different molecular mechanisms such as DNA methylation and hydroxymethylation, histone variants and histone post-translational modifications, non-coding RNAs, and active chromatin remodeling and interactions with the nuclear envelope. Prior to reviewing how these processes change over the course of cellular and neuronal aging, several of these mechanisms will be explored briefly below.</p>
<div id="dna-modifications" class="section level3 unnumbered">
<h3><em>DNA Modifications</em></h3>
<p>The archetypal epigenetic modification is perhaps the methylation of cytosines at CpG dinucleotides, which has been studied extensively since it was suggested it could impact gene activity long ago <span class="citation">(Riggs <a href="#ref-RiggsinactivationdifferentiationDNA1975">1975</a>)</span>. In humans, 5-methyl-cytosine (5mC) is found in approximately 1.5% of genomic DNA <span class="citation">(Lister et al. <a href="#ref-ListerHumanDNAmethylomes2009">2009</a>)</span>. While cytosine methylation can occur in other contexts (such as CA dinucleotides), methylation occurs to 60-80% of CpG dinucleotides in mouse and human <span class="citation">(Ehrlich et al. <a href="#ref-EhrlichAmountdistribution5methylcytosine1982">1982</a>; Gruenbaum et al. <a href="#ref-GruenbaumMethylationCpGsequences1981">1981</a>)</span>. However, CpG dinucleotides that cluster in ‘CpG islands’ at promoter regions and gene bodies, generally remain unmethylated <span class="citation">(Bird <a href="#ref-BirdCpGrichislandsfunction1986">1986</a>)</span>. Methylation at CpG islands can serve as a direct mode of epigenetic control, as the DNA- binding activity of most transcription factors is sensitive to CpG methylation <span class="citation">(Fiore et al. <a href="#ref-FioreCytosinemethylationtransforms1999">1999</a>; Hark et al. <a href="#ref-HarkCTCFmediatesmethylationsensitive2000">2000</a>; Mancini et al. <a href="#ref-ManciniSitespecificDNAmethylation1999">1999</a>; Prendergast and Ziff <a href="#ref-PrendergastMethylationsensitivesequencespecificDNA1991">1991</a>; Tate and Bird <a href="#ref-TateEffectsDNAmethylation1993">1993</a>)</span>. That said, it should be noted that evidence suggests that the DNA demethylation important for normal development occurs in areas that are not CpG islands <span class="citation">(Jones <a href="#ref-JonesFunctionsDNAmethylation2012">2012</a>)</span>, and we should wary of oversimplification with regards to CpG island annotations.</p>
<p>Cytosine methylation in DNA is established and maintained by the family of DNA methyltransferases (Dnmts) of which there are five known members: <em>Dnmt1</em>, <em>Dnmt3a</em>, <em>Dnmt3b</em>, <em>Dnmt3c</em>, and <em>Dnmt3L</em>. <em>Dnmt3a</em> and <em>Dnmt3b</em> encode two closely related <em>de novo</em> methyl transferases and are responsible for generating DNA methylation patterns during gametogenesis and in early embryonic development <span class="citation">(Kaneda et al. <a href="#ref-KanedaEssentialrolenovo2004">2004</a>; Okano et al. <a href="#ref-OkanoDNAmethyltransferasesDnmt3a1999">1999</a>)</span> where Dnmt3a specifically mediates the methylation of imprinted genes in germ cells <span class="citation">(Barau et al. <a href="#ref-BarauDNAmethyltransferaseDNMT3C2016b">2016</a>)</span>. Dnmt3L lacks the catalytic domain but is still important for establishing imprinting control regions (ICRs) in gametes in addition to stimulating Dnmt3a and Dnmt3b activity <span class="citation">(Bourc’his et al. <a href="#ref-BourchisDnmt3Lestablishmentmaternal2001">2001</a>; Hata et al. <a href="#ref-HataDnmt3LcooperatesDnmt32002">2002</a>; Xie et al. <a href="#ref-XieMutationsDNAmethyltransferase2006">2006</a>)</span>. Dnmt3c has only recently been identified, arising from a duplication of Dnmt3b, and described to be important in male germ cells <span class="citation">(Jain et al. <a href="#ref-Jainrahumutantallele2017">2017</a>)</span>. Dnmt1 is set apart in that it is the maintenance methyltransferase, responsible for depositing methylation after DNA replication <span class="citation">(Bestor et al. <a href="#ref-BestorCloningsequencingcDNA1988">1988</a>; Chuang et al. <a href="#ref-ChuangHumanDNAcytosine51997">1997</a>)</span> in addition to restoring DNA methylation following DNA repair <span class="citation">(Mortusewicz et al. <a href="#ref-MortusewiczRecruitmentDNAmethyltransferase2005a">2005</a>)</span>.</p>
<p>DNA can be passively or actively demethylated. Until recently it was believed the only form of DNA demethylation was the passive loss of DNA methylation due to improper maintenance methylation by Dnmt1. However it is now understood that active demethylation can occur by at least three different pathways. The first involves deamination and base exchange by thymine DNA glycosylase (TDG) which occurs (somewhat paradoxically) in the presence of Dnmt3a and Dnmt3b <span class="citation">(M’etivier et al. <a href="#ref-MetivierCyclicalDNAmethylation2008">2008</a>)</span>. The second involves induction of the nuclear excision repair system by Gadd45 (growth arrest and DNA damage inducible protein 45) <span class="citation">(Barreto et al. <a href="#ref-BarretoGadd45apromotesepigenetic2007">2007</a>)</span>. The third pathway to demethylate cytosines involves enzymes that oxidize 5-methyl-cytosine (5mC) to 5-hydroxy-methyl-cytosine (5hmC) <span class="citation">(Tahiliani et al. <a href="#ref-TahilianiConversion5methylcytosine5hydroxymethylcytosine2009">2009</a>)</span>, which is a process now understood to be of great importance in both the development and proper functioning of the central nervous system <span class="citation">(Guo et al. <a href="#ref-GuoHydroxylation5methylcytosineTET12011">2011</a>; Hahn et al. <a href="#ref-HahnDynamics5HydroxymethylcytosineChromatin2013">2013</a>; Mell’en et al. <a href="#ref-MellenMeCP2Binds5hmC2012">2012</a>)</span>. Once converted, 5hmC may be dealkylated resulting in a normal cytosine <span class="citation">(Guo et al. <a href="#ref-GuoHydroxylation5methylcytosineTET12011">2011</a>)</span>. The enzymes that actively convert 5mC to 5hmC are called Ten-Eleven-Translocation (TET) enzymes, of which there are three members: TET1, TET2, and TET3. In addition to converting 5mC to 5hmC as an intermediate, Tet enzymes can further oxidize 5hmC residues to 5-formyl-cytosine (5fC) and 5-carboxyl-cytosine (5caC), which can be reverted to an unmodified cytosine by Base Exchange Repair (BER) <span class="citation">(Ito et al. <a href="#ref-ItoTetproteinscan2011">2011</a>; Maiti and Drohat <a href="#ref-MaitiThymineDNAglycosylase2011">2011</a>)</span>.</p>
</div>
<div id="chromatin" class="section level3 unnumbered">
<h3><em>Chromatin</em></h3>
<p>DNA in eukaryotic cells is organized by histone and non-histone proteins, which is collectively known as chromatin. Here, the fundamental unit is the nucleosome, where 146 bp of DNA wraps around a single octamer of histones <span class="citation">(Luger et al. <a href="#ref-LugerCrystalstructurenucleosome1997">1997</a>)</span>. The composition of the histone octamer, post-translational modifications to its subunits, and non-histone proteins define higher order structures and DNA accessibility. Based on both structural conformation and gene transcription, Chromatin is broadly categorized into two major compartments; transcriptionally active euchromatin, and transcriptionally silent heterochromatin.</p>
<p><strong>Euchromatin</strong>, being the transcriptionally active compartment of the nucleus, corresponds with histone modifications that facilitate open DNA-histone conformations and access to the DNA. These may involve methylation, in the case of H3K4me1 (read as histone 3 lysine 4 mono-methylation) at promoters and enhancers, or H3K4me3 (histone 3 lysine 4 trimethylation) at transcription start sites <span class="citation">(Santos-Rosa et al. <a href="#ref-Santos-RosaActivegenesare2002">2002</a>)</span>. Specific histone acetylation also favors open conformations, such as H3K9ac (histone 3 lysine 9 acetylation) and H3K18ac on promoters and H3K27ac more often seen on enhancers <span class="citation">(Jin et al. <a href="#ref-JinDistinctrolesGCN52011">2011</a>)</span>. Methylation of H3K36me3 is often seen on gene bodies and was recently shown to play a prominent role in exon inclusion and splicing <span class="citation">(Edmunds, Mahadevan, and Clayton <a href="#ref-EdmundsDynamichistoneH32008a">2008</a>; Hu et al. <a href="#ref-HuHistoneH3lysine2010">2010</a>; Kolasinska-Zwierz et al. <a href="#ref-Kolasinska-ZwierzDifferentialchromatinmarking2009">2009</a>)</span>. While DNA methylation is more common in heterochromatin, DNA methylation has also been observed in euchromatin, where it too can affect exon inclusion <span class="citation">(Yearim et al. <a href="#ref-YearimHP1InvolvedRegulating2015">2015</a>)</span>.</p>
<p>Transcriptionally inactive heterochromatin can be categorized into either constitutive or facultative.</p>
<p><strong>Constitutive Heterochromatin</strong> is comprised of peri-centromeric, telomeric and perinucleolar heterochromatin and marked by tightly bound heterochromatin protein 1 (HP1) <span class="citation">(Blasco <a href="#ref-Blascoepigeneticregulationmammalian2007">2007</a>; Santoro, Li, and Grummt <a href="#ref-Santoronucleolarremodelingcomplex2002">2002</a>; Wreggett et al. <a href="#ref-WreggettmammalianhomologueDrosophila1994">1994</a>)</span> and histone modifications H3K9me3 <span class="citation">(Bannister et al. <a href="#ref-BannisterSelectiverecognitionmethylated2001">2001</a>; Lachner et al. <a href="#ref-LachnerMethylationhistoneH32001">2001</a>)</span> and H4K20me3 <span class="citation">(Kourmouli et al. <a href="#ref-KourmouliHeterochromatintrimethylatedlysine2004">2004</a>; Schotta <a href="#ref-Schottasilencingpathwayinduce2004">2004</a>)</span>. Constitutive heterochromatin is typically densely packed, inaccessible and DNA hyper-methylated <span class="citation">(Blasco <a href="#ref-Blascoepigeneticregulationmammalian2007">2007</a>; Lubit et al. <a href="#ref-LubitLocalization5methylcytosinehuman1976">1976</a>; Santoro, Li, and Grummt <a href="#ref-Santoronucleolarremodelingcomplex2002">2002</a>)</span>.</p>
<p><strong>Facultative Heterochromatin</strong> refers to heterochromatinization of non-genic and genic areas that are developmentally regulated. In addition to H3K9me3/HP1/H4K20me3, facultative heterochromatin may contain H3K27me3, an important modification for tight temporal regulation of gene expression <span class="citation">(Hahn et al. <a href="#ref-HahnDynamics5HydroxymethylcytosineChromatin2013">2013</a>)</span>. While there appears to be some cross-talk between H3K9me3- and H3K27me3- mediated repression <span class="citation">(Riclet et al. <a href="#ref-RicletDisruptionInteractionTranscriptional2008">2008</a>; Mozzetta et al. <a href="#ref-MozzettahistoneH3lysine2014">2014</a>)</span>, both are largely regulated by different pathways. H3K27me3 is regulated by the polycomb repressive complex PRC1 and PRC2 <span class="citation">(Cao and Zhang <a href="#ref-CaofunctionsEZH2mediatedmethylation2004">2004</a>; Eskeland et al. <a href="#ref-EskelandRing1BCompactsChromatin2010">2010</a>)</span>. H3K9me3 in constitutive heterochromatin is deposited by the histone methyltransferase SUV39H1/H2 <span class="citation">(Peters et al. <a href="#ref-PetersLossSuv39hHistone2001">2001</a>; O’Carroll et al. <a href="#ref-OCarrollIsolationCharacterizationofSuv39h22000">2000</a>)</span> while in facultative heterochromatin this is typically mediated by KAP1 (KRAB Associated Protein 1, also known as TRIM28 or TIF1<span class="math inline">\(\beta\)</span>)-mediated recruitment of the methyltransferase SETDB1 (SET Domain Bifurcated 1) <span class="citation">(Schultz et al. <a href="#ref-SchultzSETDB1novelKAP1associated2002a">2002</a>)</span>, which will be discussed later.</p>
</div>
<div id="chromatin-remodelling" class="section level3 unnumbered">
<h3><em>Chromatin Remodelling</em></h3>
<p>The list of proteins and RNAs that bind, read, write, or remodel chromatin is an ever growing one. On the macromolecular scale, large-scale chromatin re-organization can be determined by the combinatorial assembly of a large ATP-dependent <em>Chromatin Remodelling Complex</em>. In this arena subunits that form such assemblies have suffered greatly from classification and nomenclature problems. The most studied classification is the differential assembly of mammalian SWItch/sucrose nonfermentable (mSWI/SNF) or Brg/Brm-associated factor (BAF) complexes <span class="citation">(Wang et al. <a href="#ref-WangPurificationbiochemicalheterogeneity1996">1996</a>)</span>. During neuronal differentiation, BAF subunits found in progenitors are substituted to neuronal subunits, which confer differential specificity to chromatin remodeling <span class="citation">(Staahl et al. <a href="#ref-StaahlKineticAnalysisnpBAF2013">2013</a>)</span>. Given the estimation that the combinatorial assembly of BAF subunits could yield at least 36 unique complexes, differential composition of this complex could in turn generate a multitude of chromatin architectures in different cell types. Once present, chromatin remodeling complexes may also recruit other structural proteins and chromatin-modifying enzymes such as Histone De-ACetylases (HDACs), Histone Methyl-Transferases (HMTs) and DNA Methyl-Transferases (DNMTs) that ensure the modified chromatin architecture persists. Ultimately however, chromatin remodeling complexes are secondary to the transcription factors or co-factors that recruit them.</p>
</div>
<div id="krab-zfps-transposable-elements-and-exaptation" class="section level3 unnumbered">
<h3><em>KRAB-ZFPs, Transposable Elements and Exaptation</em></h3>
<p>The largest and oldest family of transcription factors is the Krüppel-associated box (KRAB)-containing zinc finger proteins (KRAB-ZFPs), which have proved integral to heterochromatinization pathways and gene regulation. KRAB-ZFPs have co-evolved to silence exogenously-derived proviruses and other retrotransposons such as LINEs that at one point integrated into the host germline <span class="citation">(Thomas and Schneider <a href="#ref-ThomasCoevolutionretroelementstandem2011">2011</a>; Imbeault, Helleboid, and Trono <a href="#ref-ImbeaultKRABzincfingerproteins2017">2017</a>)</span>. While DNA transposons (‘cut and paste’) are thought to be extinct in higher eukaryotes, multiple RNA transposons (or retrotransposons, ‘copy and paste’) are known to be capable of transposition in mammals (Figure <a href="1-introduction.html#fig:TEs">1.2</a>).</p>
<p>Retrotransposons can be divided into LTR containing and non-LTR containing. Viruses that successfully integrate into the germline are henceforth termed endogenous retroviruses (ERVs) and carry 5’ and 3’ long terminal repeats (LTRs). Over evolutionary time, these foreign sequences accrue mutations and lose their transposition potential, meaning they are no longer a direct threat to genomic instability. However, because the LTRs of many viruses contain the multiple eukaryotic transcription factor binding sites (TFBSs) to ensure host-replication, upon integration these ancient ERVs can confer a new layer of transcriptional regulation, a process termed <em>exaptation</em> <span class="citation">(Thompson, Macfarlan, and Lorincz <a href="#ref-ThompsonLongTerminalRepeats2016">2016</a>; Imbeault, Helleboid, and Trono <a href="#ref-ImbeaultKRABzincfingerproteins2017">2017</a>)</span>.</p>
<div class="figure" style="text-align: center"><span id="fig:TEs"></span>
<img src="figure/introduction/TEs.png" alt="Sequence architecture of mammalian LTR and non-LTR Retrotransposons. Non-LTR transposons include Long Interspersed Nuclear Elements (LINEs), Short Interspersed Nuclear Elements (SINEs) Arthrobacter luteus (Alu) and SINE/VNTR/Alu (SVA) elements. LTR-retrotransposons are resultant of past exogenous viral integration into the germline. LTR retrotransposons are comprised of 5' and 3' long terminal repeats (LTRs) and retroviral open reading frames 'gag', 'pol', and a deficient 'env'.  Recombination between 5' and 3' LTRs deletes the internal region, leaving behind a solitary LTR which harbors regulatory regions and transcription factor binding sites." width="100%"   />
<p class="caption">
Figure 1.2: Sequence architecture of mammalian LTR and non-LTR Retrotransposons. Non-LTR transposons include Long Interspersed Nuclear Elements (LINEs), Short Interspersed Nuclear Elements (SINEs) Arthrobacter luteus (Alu) and SINE/VNTR/Alu (SVA) elements. LTR-retrotransposons are resultant of past exogenous viral integration into the germline. LTR retrotransposons are comprised of 5’ and 3’ long terminal repeats (LTRs) and retroviral open reading frames ‘gag’, ‘pol’, and a deficient ‘env’. Recombination between 5’ and 3’ LTRs deletes the internal region, leaving behind a solitary LTR which harbors regulatory regions and transcription factor binding sites.
</p>
</div>
<p>Most of the LTRs in the genome, and subsequently also the most frequently exaptated are “solo” LTRs. Such sequences are generated by homologous recombination between repetitive sequences on 5’ and 3’ LTRs of a full length ERV. Over evolutionary time, purifying selection means that an exaptated ERV may lose all identifying ‘viral’ sequences, retaining only the binding sites required for transcriptional regulation <span class="citation">(Brady et al. <a href="#ref-BradyIntegrationtargetsite2009">2009</a>)</span>. Indeed, ERVs have been more frequently exaptated compared to other transposable elements, despite comprising only 6-8% of the human genome, they have provided an estimated 25% of functional TFBSs <span class="citation">(Chuong et al. <a href="#ref-ChuongEndogenousretrovirusesfunction2013">2013</a>; Cordaux and Batzer <a href="#ref-Cordauximpactretrotransposonshuman2009">2009</a>; Sundaram et al. <a href="#ref-SundaramWidespreadcontributiontransposable2014">2014</a>)</span> although it could be more.</p>
<p>Non-LTR transposons such as LINE1 are considerably more ancient. They can be viewed as ‘molecular fossils’ of the first primitive replication systems that transitioned molecular genetics from being RNA-based to DNA-based. Based on the phylogeny of the reverse transcriptase found in LINE1, models suggest that LINE1 elements were once components of mobile introns of eubacteria and were transmitted to eukaryotes during the symbiosis that gave rise the mitochondria <span class="citation">(Zimmerly et al. <a href="#ref-ZimmerlyGroupIIintron1995">1995</a>)</span>. The subsequent acquisition of an endonuclease enzyme and promoter sequence granted LINE1 autonomy to insert into multiple locations of the genome in addition to providing the machinery to mobilize <em>in trans</em> other non-autonomous retrotransposons such as Alu and SVA <span class="citation">(Dewannieux, Esnault, and Heidmann <a href="#ref-DewannieuxLINEmediatedretrotranspositionmarked2003">2003</a>; Hancks and Kazazian <a href="#ref-HancksActivehumanretrotransposons2012">2012</a>; Kajikawa and Okada <a href="#ref-KajikawaLINEsmobilizeSINEs2002">2002</a>)</span>.</p>
<p>In addition to silencing retrotransposons to maintain genetic stability, KRAB-ZFPs coordinate spatio-temporal regulation of exaptated regulatory sequences for transcriptional regulation. The role of KRAB-ZFPs in utilizing transposon-derived sequences for transcriptional regulation in ES cells is well established <span class="citation">(Maksakova et al. <a href="#ref-MaksakovaDistinctrolesKAP12013">2013</a>; Rowe et al. <a href="#ref-RowenovoDNAmethylation2013">2013</a>)</span> as well as throughout development <span class="citation">(Friedli and Trono <a href="#ref-Friedlidevelopmentalcontroltransposable2015">2015</a>)</span>, and continues to be elucidated in adult tissues <span class="citation">(Ecco et al. <a href="#ref-EccoTransposableElementsTheir2016">2016</a>)</span>.</p>
</div>
<div id="heterochromatinization-and-hp1" class="section level3 unnumbered">
<h3><em>Heterochromatinization and HP1</em></h3>
<p>How KRAB-ZFPs initiate heterochromatinization of an ERV or an exaptated regulatory region involves the building of a large heterochromatin-like complex. First, the KRAB-ZFP recognizes a specific DNA sequence with its zinc fingers. Next, the KRAB domain of the KRAB-ZFP interacts with KAP1, which acts as the central co-factor able to bind HDACs, HMTases, DNMTs, ATRX-DAXX and Heterochromatin Protein 1 (HP1). ATRX (considered to be a BAF chromatin remodeling component) -mediated recruitment of DAXX is known to replace the H3 histone subunit with H3.3 at constitutive chromatin <span class="citation">(Voon and Wong <a href="#ref-VoonNewplayersheterochromatin2016">2016</a>)</span>. While H3.3 deposition at pericentric, constitutive and telomeric chromatin is essential for mammalian development <span class="citation">(Jang et al. <a href="#ref-JangHistoneH3maintains2015">2015</a>)</span>, how it affects H3K9me3 deposition remains unclear <span class="citation">(Udugama et al. <a href="#ref-UdugamaHistonevariantH32015">2015</a>)</span>. Sumoylation of the bromodomain of KAP1 enables binding of both HDAC1/2 and SETDB1 <span class="citation">(Ivanov et al. <a href="#ref-IvanovPHDDomainMediatedE32007">2007</a>)</span>. Histone de-acetylation prepares for tri-methylation at H3K9 by SETDB1, which creates a high-affinity binding site for HP1, aiding in the localization of ATRX-DAXX, Uhrf1 and DNMTs, all of which bind to KAP1 <span class="citation">(Ayyanathan <a href="#ref-AyyanathanRegulatedrecruitmentHP12003a">2003</a>; Rottach et al. <a href="#ref-RottachmultidomainproteinNp952010">2010</a>; Smallwood et al. <a href="#ref-SmallwoodFunctionalcooperationHP12007">2007</a>; Voon and Gibbons <a href="#ref-VoonMaintainingmemorysilencing2016">2016</a>; Zuo et al. <a href="#ref-Zuozincfingerprotein2011">2011</a>)</span>. While this nucleation event creates a heterochromatin-like domain that is strongly heritable through mitotic divisions <span class="citation">(Ayyanathan <a href="#ref-AyyanathanRegulatedrecruitmentHP12003a">2003</a>)</span>, it is spatially restricted to a few kilobases. This is often the case at imprinted differentially methylated regions (DMRs), where ZFP57 recruits the KAP1 complex to maintain methylation during the pre-implantation period in the embryo <span class="citation">(Voon and Gibbons <a href="#ref-VoonMaintainingmemorysilencing2016">2016</a>)</span>. In some cases, this small KAP1-SETDB1-HP1 heterochromatin-like domain may continue to extend further megabases through cyclical deposition and heterochromatin spreading of H3K9me3 by a SUV39H1-HP1 complex <span class="citation">(Vogel et al. <a href="#ref-VogelHumanheterochromatinproteins2006">2006</a>; Groner et al. <a href="#ref-GronerKRABzincfinger2010">2010</a>)</span>, or the heterochromatin-like domain can itself be mobilized to constitutive heterochromatin by HP1 <span class="citation">(Cammas et al. <a href="#ref-CammasDynamicselectiveinteractions2007">2007</a>)</span>, which can bridge H3K9me3 chromatin fibres <span class="citation">(Hiragami-Hamada et al. <a href="#ref-Hiragami-HamadaDynamicflexibleH3K9me32016">2016</a>)</span>. Notably, HP1 proteins are known to be required for recruitment of other methyltransferases responsible for H4K20me3 <span class="citation">(Kourmouli et al. <a href="#ref-KourmouliEpigeneticregulationmammalian2005">2005</a>; Kourmouli et al. <a href="#ref-KourmouliHeterochromatintrimethylatedlysine2004">2004</a>; Schotta <a href="#ref-Schottasilencingpathwayinduce2004">2004</a>)</span> which induces strong compaction of nucleosomes <span class="citation">(Lu et al. <a href="#ref-LueffectH3K79dimethylation2008">2008</a>)</span> and has distinct implications in cellular senescence <span class="citation">(Nelson et al. <a href="#ref-NelsonMappingH4K20me3chromatin2016">2016</a>)</span>.</p>
</div>
<div id="non-coding-rna" class="section level3 unnumbered">
<h3><em>Non-Coding RNA</em></h3>
<p>Various types of non-coding RNA (ncRNA) have now emerged to be vitally important in directing the epigenetic landscape. Since the classical example of inactivation of the entire X chromosome by the long non-coding RNA <em>Xist</em> <span class="citation">(Brown et al. <a href="#ref-BrownhumanXISTgene1992">1992</a>; Wutz <a href="#ref-WutzGenesilencingXchromosome2011">2011</a>)</span>, it is now known that ncRNAs work in various ways to profoundly impact gene regulation and genome stability. Non-coding RNAs can affect gene regulation in various ways. In some cases, long non-coding RNAs (lncRNAs) can originate from ultraconserved elements and act by directly interfering with polycomb-mediated repression <span class="citation">(Fiorenzano et al. <a href="#ref-FiorenzanoUltraconservedElementContaining2018">2018</a>)</span>. Some ncRNAs have prominent roles in splicing <span class="citation">(Ashwal-Fluss et al. <a href="#ref-Ashwal-FlusscircRNABiogenesisCompetes2014">2014</a>; Gonzalez et al. <a href="#ref-GonzalezlncRNAregulatesalternative2015">2015</a>)</span>, while others can act as sponges to micro-RNA or RNA-binding proteins <span class="citation">(Memczak et al. <a href="#ref-MemczakCircularRNAsare2013">2013</a>; Wu et al. <a href="#ref-WuUnusualProcessingGenerates2016">2016</a>)</span>. Other ncRNAs act locally in <em>cis</em> to regulate the transcription of their ‘gene neighborhood’ <span class="citation">(Joung et al. <a href="#ref-JoungGenomescaleactivationscreen2017">2017</a>)</span>, an effect that is prominently seen at imprinting control regions <span class="citation">(Mancini-DiNardo et al. <a href="#ref-Mancini-DiNardoElongationKcnq1ot1transcript2006">2006</a>; Sleutels, Zwart, and Barlow <a href="#ref-SleutelsnoncodingAirRNA2002">2002</a>; Williamson et al. <a href="#ref-WilliamsonUncouplingAntisenseMediatedSilencing2011a">2011</a>)</span>.</p>
<p>A large body of evidence has shown that ncRNAs derived from different repeat regions are often required for inducing or maintaining epigenetic modifications. Transcription of <em>PAPAS</em> (Promoter and Pre-rRNA AntiSense), an ncRNA that is antisense to ribosomal DNA (rDNA) repeats, is required for rDNA silencing by recruitment of DNMT3b and SUV420H2-mediated deposition of H4K20me3 <span class="citation">(Santoro et al. <a href="#ref-SantoroIntergenictranscriptsoriginating2010">2010</a>; Santoro, Li, and Grummt <a href="#ref-Santoronucleolarremodelingcomplex2002">2002</a>; Schmitz et al. <a href="#ref-SchmitzInteractionnoncodingRNA2010">2010</a>; Bierhoff et al. <a href="#ref-BierhoffQuiescenceInducedLncRNAsTrigger2014">2014</a>)</span>. Similarly, silencing and deposition of H4K20me3 on transposable element sequences such as ERVs and LINEs is RNA dependent <span class="citation">(Bierhoff et al. <a href="#ref-BierhoffQuiescenceInducedLncRNAsTrigger2014">2014</a>; Slotkin and Martienssen <a href="#ref-SlotkinTransposableelementsepigenetic2007">2007</a>; Yang and Kazazian Jr <a href="#ref-YangL1retrotranspositionsuppressed2006">2006</a>)</span>. In constitutive heterochromatin, RNA derived from major and minor satellite ncRNA have distinct roles in proper centromere specification and function <span class="citation">(Wong et al. <a href="#ref-WongCentromereRNAkey2007">2007</a>)</span> and their transcription is required for recruitment of HP1<span class="math inline">\(\alpha\)</span> and H3K9me3 <span class="citation">(Maison et al. <a href="#ref-MaisonHigherorderstructurepericentric2002">2002</a>)</span>. Similarly, transcription of the ncRNA <em>TERRA</em> (telomeric repeat-containing RNA) is intrinsically linked with telomere stability <span class="citation">(Nergadze et al. <a href="#ref-NergadzeCpGislandpromotersdrive2009">2009</a>)</span>.</p>
<p>Finally, non-coding RNAs have garnered much attention over their ability to confer transgenerational changes. In <em>Caoenorhabditis elegans</em>, neuronal double-stranded RNA can induce germline silencing for up to 25 generations <span class="citation">(Devanapally, Ravikumar, and Jose <a href="#ref-DevanapallyDoublestrandedRNAmade2015">2015</a>)</span>. In higher organisms such as the mouse, it has been shown that both stress and exercise can alter small non-coding RNAs in sperm, resulting in augmented anxiety responses in offspring <span class="citation">(Rodgers et al. <a href="#ref-RodgersPaternalStressExposure2013">2013</a>; Rodgers et al. <a href="#ref-RodgersTransgenerationalepigeneticprogramming2015">2015</a>; Short et al. <a href="#ref-ShortExercisealtersmouse2017a">2017</a>)</span>.</p>
<p>The above epigenetic mechanisms act in concert and are interdependent in conferring a cell-type specific genomic architecture; where a subset of genes are accessibly transcribed in the euchromatic compartment, and unused genes and regulatory regions are silenced and compacted in facultative chromatin.</p>
</div>
</div>
<div id="the-heterochromatin-loss-model-of-aging" class="section level2">
<h2><span class="header-section-number">1.2</span> The Heterochromatin Loss Model of Aging</h2>
<p>Age-related disruption of heterochromatin boundaries greatly modifies the 3D architecture of the nucleus and has profound effects on cellular function. Age-related restructuring of the chromatin landscape can occur in either of two ways. The first is <em>organismal aging</em>, where a gradual reduction in normal physiological function occurs in a time dependent manner and is typically accompanied by heterochromatin loss. Organismal aging increases susceptibility to many diseases such as metabolic disorders, cancer and neurodegenerative diseases <span class="citation">(Kennedy et al. <a href="#ref-KennedyGerosciencelinkingaging2014">2014</a>; Moskalev et al. <a href="#ref-MoskalevGeneticsepigeneticsaging2014">2014</a>)</span>. The second, which can be observed in dividing cells, is <em>cellular</em> (or <em>replicative</em>) <em>senescence</em>. In contrast to the organismal aging, cellular senescence is a specialized process that results in an irreversible cell cycle-arrest now known to be a cellular response to a variety of stresses such as DNA double strand breaks (DSBs) and oncogene activation, and is now thought to serve as an anticancer mechanism in response to oncogenic stimuli <span class="citation">(Di Leonardo et al. <a href="#ref-DiLeonardoDNAdamagetriggers1994">1994</a>; Lin et al. <a href="#ref-LinPrematuresenescenceinvolving1998">1998</a>)</span>. Until now both processes are described to be functionally distinct, where heterochromatin loss is common in aging cells, and senescent cells show large senescence associated heterochromatic foci (SAHFs). However, recent work suggests that senescence is likely to occur in a two step process, first involving heterochromatin loss followed by SAHF formation <span class="citation">(Chandra et al. <a href="#ref-ChandraGlobalReorganizationNuclear2015">2015</a>)</span>, suggesting that senescence and organismal aging may proceed by similar routes.</p>
<div class="figure" style="text-align: center"><span id="fig:aging"></span>
<img src="figure/introduction/agingepigenome.png" alt="An Emerging Consensus of the Aging Epigenome.  (Modified from Booth and Brunet 2016)" width="100%"   />
<p class="caption">
Figure 1.3: An Emerging Consensus of the Aging Epigenome. (Modified from Booth and Brunet 2016)
</p>
</div>
<p>The heterochromatin loss model of aging was first put forward by Villeponteau <span class="citation">(Villeponteau <a href="#ref-Villeponteauheterochromatinlossmodel1997">1997</a>)</span>, who proposed that the heterochromatin domains established in embryogenesis are perturbed during aging, contributing to the aberrant de-repression of silenced genes. In general, age-related decreases of histone de-acetylase components <span class="citation">(Pegoraro et al. <a href="#ref-PegoraroAgeingrelatedchromatindefects2009">2009</a>)</span>, the methyltransferase SUV39H1 and bound HP1 have been observed in various species ranging from drosophila to human <span class="citation">(Djeghloul et al. <a href="#ref-DjeghloulAgeAssociatedDecreaseHistone2016">2016</a>; Wood et al. <a href="#ref-WoodChromatinremodelingaging2010">2010</a>; Jeon et al. <a href="#ref-JeonEffectheterochromatinstability2018">2018</a>; Ryu et al. <a href="#ref-RyuTranscriptionalrepressionrepeatderived2011">2011</a>)</span>. Notably, in drosophila, increasing HP1 protein and subsequent heterochromatin formation increases longevity <span class="citation">(Larson et al. <a href="#ref-LarsonHeterochromatinformationpromotes2012">2012</a>)</span>. Age-related heterochromatin loss resulting in increased transcription of repeats and transposable elements has been observed in several species and human progeria models <span class="citation">(Larson et al. <a href="#ref-LarsonHeterochromatinformationpromotes2012">2012</a>; Smeal et al. <a href="#ref-SmealLosstranscriptionalsilencing1996">1996</a>; Sousa-Victor et al. <a href="#ref-Sousa-VictorPiwiRequiredLimit2017">2017</a>; Zhang et al. <a href="#ref-ZhangWernersyndromestem2015">2015</a>; Djeghloul et al. <a href="#ref-DjeghloulAgeAssociatedDecreaseHistone2016">2016</a>; De Cecco et al. <a href="#ref-DeCeccoGenomesreplicativelysenescent2013">2013</a>)</span>. The findings described in the following section are summarized in Figure <a href="1-introduction.html#fig:aging">1.3</a>.</p>
<p></p>
<p>Until now, most genomic work on humans has been on somatic cells derived from human patients with premature aging syndromes (‘progeria’), which in general are characterized by the loss of heterochromatin <span class="citation">(Liu et al. <a href="#ref-LiuRecapitulationprematureageing2011">2011</a>; Miller et al. <a href="#ref-MillerHumaniPSCbasedmodeling2013">2013</a>; Shumaker et al. <a href="#ref-ShumakerMutantnuclearlamin2006">2006</a>)</span>. Two diseases that mimic premature aging in humans are Hutchinson-Gilford progeria syndrome (HGPS) and atypical Werner Syndrome.</p>
<p>Individuals with HGPS on average live until 13 years of age, and typically die of artherosclerosis or other cardiac related problems <span class="citation">(Pereira et al. <a href="#ref-PereiraHGPSrelatedpremature2008b">2008</a>)</span>. Patients with HGPS express ‘progerin’, a truncated form of the Lamin A receptor, normally important for maintaining heterochromatin domains around the nuclear envelope. Patients with HGPS show reduced levels of SUV39H1, H3K9me3 and HP1 and a corresponding increase in levels of satellite repeat RNA <span class="citation">(Shumaker et al. <a href="#ref-ShumakerMutantnuclearlamin2006">2006</a>)</span>. However, somewhat paradoxically, H4K20me3 is elevated in both growth-arrested cells and HGPS cells <span class="citation">(Shumaker et al. <a href="#ref-ShumakerMutantnuclearlamin2006">2006</a>)</span>, which may be a result of de-repressed lncRNAs which direct its deposition <span class="citation">(Bierhoff et al. <a href="#ref-BierhoffQuiescenceInducedLncRNAsTrigger2014">2014</a>)</span>. By contrast, aged human diploid fibroblasts show a near complete loss of H4K20me3 despite modest increases in H4K20me1 and H4K20me2 <span class="citation">(O’Sullivan et al. <a href="#ref-OSullivanReducedhistonebiosynthesis2010">2010</a>)</span>. A study examining the brains of the SAMP8 mouse model, which displays accelerated senescence, found no discernible difference in H4K20me3 levels in aged animals <span class="citation">(Wang et al. <a href="#ref-WangIdentificationhistonemethylation2010">2010</a>)</span>.</p>
<p>Patients with Werner Syndrome express a mutant from of the <em>WRN</em> gene (Werner syndrome RecQ like helicase), which is also important in DNA replication, transcription, repair and centromeric heterochromatin. <em>WRN</em> deficient cells obtained from patients with Werner syndrome show deficits in DNA repair, loss of SUV39H1, H3K9me3 and HP1, and display accelerated senescence. This phenotype can be partially reversed by maintaining heterochromatin. Overexpression of HP1<span class="math inline">\(\alpha\)</span> in <em>WRN</em>-deficient mesenchymal stem cells can restore H3K9me3 levels and repress cellular senescence <span class="citation">(Zhang et al. <a href="#ref-ZhangWernersyndromestem2015">2015</a>)</span>.</p>
<p>Importantly, Zhang et al (2015) found that cells derived from normal healthy individuals, levels of SUV39H1, H3K9me3 and HP1 decline steadily until very old age, when levels return. This return of new heterochromatin domains is likely coincident with the formation of senescence-associated heterochromatin foci (SAHF), which are a hallmark of senescent cells. SAHFs are composed of large heterochromatin domains enriched in H3K9me3 and HP1 proteins <span class="citation">(Narita et al. <a href="#ref-NaritaRbmediatedheterochromatinformation2003">2003</a>)</span>, but are not as densely packed as healthy constitutive heterochromatin <span class="citation">(Chandra et al. <a href="#ref-ChandraGlobalReorganizationNuclear2015">2015</a>)</span>. Additionally, it has been observed that H3K27me3, found in facultative heterochromatin, is predominantly lost in <em>C. Elegans</em> and human progeria models <span class="citation">(Shah et al. <a href="#ref-ShahLaminB1depletion2013">2013</a>; Ni et al. <a href="#ref-NiTwoSETdomain2012">2012</a>; Zhang et al. <a href="#ref-ZhangWernersyndromestem2015">2015</a>)</span>. Given partial crosstalk between H3K27me3 and H3K9me3 systems, changes in H3K27me3 distribution may or may not be directly linked to HP1 loss <span class="citation">(Jamieson et al. <a href="#ref-JamiesonLossHP1causes2015">2015</a>)</span>.</p>
<p>DNA methylation shows distinct age-related changes so tightly linked with age, that it provides the most accurate measurement for human biological age <span class="citation">(Hannum et al. <a href="#ref-HannumGenomewidemethylationprofiles2013">2013</a>; Horvath <a href="#ref-HorvathDNAmethylationage2013">2013</a>)</span>. A multiplexed test can now reveal an individual’s exact age within 1-3 years of accuracy depending on tissue type <span class="citation">(Horvath <a href="#ref-HorvathDNAmethylationage2013">2013</a>)</span>. In aging humans, DNA methylation is primarily lost on repeats, such as SINEs and LTRs <span class="citation">(Jintaridth and Mutirangura <a href="#ref-JintaridthDistinctivepatternsagedependent2010">2010</a>; Heyn et al. <a href="#ref-HeynDistinctDNAmethylomes2012">2012</a>; Bollati et al. <a href="#ref-BollatiDeclinegenomicDNA2009">2009</a>)</span>, an occurrence that is not observed in immortalized cells <span class="citation">(Wilson and Jones <a href="#ref-WilsonDNAmethylationdecreases1983">1983</a>)</span>. Why DNA methylation is lost with age is speculated to be linked to age-dependent loss of heterochromatin and decrease of <em>Dnmt1</em> expression <span class="citation">(Jung and Pfeifer <a href="#ref-JungAgingDNAmethylation2015">2015</a>)</span>.</p>
<p>In aging euchromatin, the general trend is towards DNA hypermethylation, however several gene promoters also show age-dependent DNA hypomethylation. Many of the genes promoters that are hypomethylated with age are directly linked to innate immune pathways and pathogenic inflammation <span class="citation">(Gowers et al. <a href="#ref-GowersAgerelatedlossCpG2011">2011</a>; Wei et al. <a href="#ref-WeiHypomethylationIL17RCpromoter2012">2012</a>; Zhang et al. <a href="#ref-ZhangAgedependentDNAmethylation2002">2002</a>)</span>. However, as stated above, euchromatin predominantly undergoes DNA hypermethylation, which occurs preferentially at loci that are tissue specific, such as at bivalent chromatin domains <span class="citation">(Rakyan et al. <a href="#ref-RakyanHumanagingassociatedDNA2010">2010</a>)</span>, or near promoters and transcription-factor binding sites <span class="citation">(Zampieri et al. <a href="#ref-ZampieriReconfigurationDNAmethylation2015">2015</a>; Sun et al. <a href="#ref-SunEpigenomicprofilingyoung2014">2014</a>; Maegawa et al. <a href="#ref-MaegawaWidespreadtissuespecific2010">2010</a>)</span>. Methylation changes to gene promoters has been proven to have immediate consequences on distal gene regulation <span class="citation">(Avrahami et al. <a href="#ref-AvrahamiAgingdependentdemethylationregulatory2015">2015</a>)</span>. It is speculated that the loci with age-associated DNA hypermethylation are normally repressed by the Polycomb repressive complex, and abnormal recruitment of DNA methyltransferases results in their hypermethylation <span class="citation">(Beerman et al. <a href="#ref-BeermanProliferationdependentalterationsDNA2013">2013</a>; Sun et al. <a href="#ref-SunEpigenomicprofilingyoung2014">2014</a>)</span>.</p>
<p>Histone modifications in aged euchromatin also show large drift. The most important alteration is to H3K4me3, an epigenetic modification that is associated with active transcription and typically found sharply at the transcription start site (TSS). H3K4me3 becomes less specific with age in humans and mouse <span class="citation">(Shah et al. <a href="#ref-ShahLaminB1depletion2013">2013</a>; Sun et al. <a href="#ref-SunEpigenomicprofilingyoung2014">2014</a>)</span>, and knockdown of enzymes responsible for H3K4me3 deposition can either diminish or promote longevity, depending on context <span class="citation">(Booth and Brunet <a href="#ref-BoothAgingEpigenome2016a">2016</a>)</span>. H3K4me1, which typically marks promoter and enhancer sequences <span class="citation">(Kim et al. <a href="#ref-KimWidespreadtranscriptionneuronal2010">2010</a>)</span>, tends to extend further into genomic loci that also show DNA hypomethylation with age <span class="citation">(Fern’andez et al. <a href="#ref-FernandezH3K4me1marksDNA2015">2015</a>)</span>. Similarly, H3K36me3, which is important for transcriptional elongation and splicing <span class="citation">(Edmunds, Mahadevan, and Clayton <a href="#ref-EdmundsDynamichistoneH32008a">2008</a>)</span> also shows an age related decline and is associated with shorter lifespan in <em>S. Cerevisiae</em> <span class="citation">(Sen et al. <a href="#ref-SenH3K36methylationpromotes2015">2015</a>)</span>, <em>C. Elegans</em> <span class="citation">(Pu et al. <a href="#ref-PuTrimethylationLys36H32015">2015</a>)</span> and in the SAMP8 mouse model <span class="citation">(Wang et al. <a href="#ref-WangIdentificationhistonemethylation2010">2010</a>)</span>. Like other histone modifications, H3K36 methylation has a strong bearing on the assembly of non-homologous end joining (NHEJ) DNA repair components, and loss of H3K36 methylation directly reduces their assembly <span class="citation">(Fnu et al. <a href="#ref-FnuMethylationhistoneH32011">2011</a>)</span>.</p>
<p>In summary, genomic aging is associated with an overall net decrease in DNA methylation at repeats and loss of constitutive heterochromatin including H3K9me3 and HP1. In euchromatin, tissue-specific promoters are subject to DNA hypermethylation, and histone marks that typically mark promoters such as H3K4me1 extend beyond their normal territory. This reorganization is coincident with decreased transcriptional efficiency in genes and increased transcription of ribosomal DNA repeats, satellite repeats and transposable elements. Sustained transcriptional de-regulation may then result in an oncogene-induced senescence, where the cellular response is to inhibit tumorigenesis and direct cell-cycle arrest, resulting in the formation of senescence associated heterochromatic foci (SAHFs).</p>
</div>
<div id="the-aging-neuronal-epigenome" class="section level2">
<h2><span class="header-section-number">1.3</span> The Aging Neuronal Epigenome</h2>
<div id="DNAging" class="section level3">
<h3><span class="header-section-number">1.3.1</span> <em>DNA Methylation and Hydroxymethylation in Aged Neurons</em></h3>
<p>While there is some evidence as to what occurs to the epigenome in aging neurons, evidence is often conflicting and is far from consensus. Most of the earliest publications studying DNA methylation in aging and diseased human brains made contradictory claims, owing to their study designs, which suffered from low throughput, low specificity and bias (for a review see <span class="citation">Yokoyama, Rutledge, and Medici (<a href="#ref-YokoyamaDNAmethylationalterations2017">2017</a>)</span>). Most human evidence now points to an age-dependent loss of DNA methylation in the brain, which is strongly corroborated by work done in mice, such as cognitive deficits observed in mice haplosuficient for DNMT1 <span class="citation">(L. Liu et al. <a href="#ref-LiuInsufficientDNAmethylation2011">2011</a>)</span>. In patients with Alzheimer’s disease for example, this translates into a correlation where higher levels of LINE-1 DNA methylation have better cognitive performance than their counterparts <span class="citation">(Bollati et al. <a href="#ref-BollatiDNAmethylationrepetitive2011">2011</a>)</span>.</p>
<p>One vitally important discovery of the past decade is that 5-hydroxy-methyl-cytosine (5hmC) is particularly enriched in the brain, which may explain some of the early confusion, as many techniques cannot discern between DNA methylation (5mC) and DNA hydroxymethylation (5hmC). In contrast to 5mC, 5hmC is typically associated with active transcription <span class="citation">(Nestor et al. <a href="#ref-NestorTissuetypemajor2012">2012</a>)</span>. As such, during neurogenesis 5hmC is dynamically regulated and in some regions this regulation continues well into adulthood <span class="citation">(Hahn et al. <a href="#ref-HahnDynamics5HydroxymethylcytosineChromatin2013">2013</a>; Szulwach et al. <a href="#ref-Szulwach5hmCmediatedepigenetic2011">2011</a>)</span>. Unlike normal 5mC DNA methylation, which is found on CpG-rich promoters and repeats, 5hmC occurs on active gene bodies which are tissue specific. In profiling 5hmC across different brain areas such as the hippocampus and cerebellum, multiple differentially hydroxy-methylated Regions (DhMRs) have been observed corresponding to gene activation specific to that tissue <span class="citation">(Szulwach et al. <a href="#ref-Szulwach5hmCmediatedepigenetic2011">2011</a>)</span>. The general trend observed is that in neurons, 5hmC tends to increase with age <span class="citation">(Chen, Dzitoyeva, and Manev <a href="#ref-ChenEffectaging5hydroxymethylcytosine2012">2012</a>; Szulwach et al. <a href="#ref-Szulwach5hmCmediatedepigenetic2011">2011</a>; Wagner et al. <a href="#ref-WagnerAgeDependentLevels5Methyl2015">2015</a>)</span>. In Alzheimer’s disease, increased expression of TET1 has been observed in the hippocampus alongside specific increases of 5hmC at DhMRs in the frontal cortex and hippocampus <span class="citation">(Zhao et al. <a href="#ref-Zhaogenomewideprofilingbrain2017">2017</a>; Bradley-Whitman and Lovell <a href="#ref-Bradley-WhitmanEpigeneticchangesprogression2013">2013</a>)</span>. It is logical to assume the age-dependent increase in 5hmC in the brain is at the expense of 5mC, however DhMRs are highly specific to cell and tissue type, and this may not always be the case. Our understanding of age-dependent dynamics of 5mC/5hmC in the brain is still limited and will benefit from continued high-throughput sequencing approaches in the future.</p>
</div>
<div id="chraging" class="section level3">
<h3><span class="header-section-number">1.3.2</span> <em>Chromatin in the Aging Brain</em></h3>
<p>Much of what we know of specific changes to the histone code in the aging brain comes from animal models. In drosophila, the aforementioned age-related loss of heterochromatin coincides with increased transcription of several retrotransposons such as R1 and R2 (LINE-like elements) and Gypsy (an LTR retrotransposon) over the course of normal aging <span class="citation">(Li et al. <a href="#ref-LiActivationtransposableelements2013">2013</a>)</span>. This age-related activation of retrotransposons results in the accumulation of <em>de novo</em> insertions begin in 2 to 4 week old adults, which is old age for a fly. Importantly, transposon activation can be attenuated by promoting heterochromatin integrity via overexpression of HP1 <span class="citation">(Larson et al. <a href="#ref-LarsonHeterochromatinformationpromotes2012">2012</a>)</span> or by caloric restriction <span class="citation">(Wood et al. <a href="#ref-WoodChromatinmodifyinggeneticinterventions2016">2016</a>)</span>.</p>
<p>In the aged mouse brain, heterochromatin marks H3K9me3 and H4K20me3 are reduced on LINE1 and the murine endogenous retrovirus IAP, resulting in their expression <span class="citation">(Ryu et al. <a href="#ref-RyuTranscriptionalrepressionrepeatderived2011">2011</a>)</span>. A separate study looking at the aged mouse hippocampus has also reported an age-dependent decrease in SUV420H2, one of the enzymes responsible for H4K20me3 deposition <span class="citation">(Sanchez et al. <a href="#ref-SanchezAgingApolipoproteinMolecular2015a">2015</a>)</span>. The same study found an age-dependent loss of ZFP57, the KRAB-ZFP responsible for maintaining DNA methylation at imprinted loci. An age-related loss of imprinting control would result in dysregulation of a wide range of genes, such as the imprinted long non-coding RNA Gas5, which sees an age-related increase in transcription in the hippocampus and is associated with impaired learning <span class="citation">(Meier et al. <a href="#ref-MeierExpressionsnoRNAhost2010">2010</a>)</span>. Given the number and variety of maternally imprinted and paternally imprinted genes <span class="citation">(Williamson et al. <a href="#ref-WilliamsonMRCHarwellOxfordshire2013">2013</a>)</span>, careful understanding of imprinting regions altered in the aging brain would vastly help our understanding of parent-of-origin effects in age-related cognitive decline.</p>
<p>Evidence for specific changes to chromatin composition in aging human neurons is lacking, and when available, often contradictory. This may be largely due to conflation by heterogenous neurodegenerative disease states such as Alzheimer’s disease, Parkinson’s and Amyotrophic Lateral Sclerosis, or by brain region specific differences. However, there is ample evidence of increases in repetitive element transcription in several neurodegenerative disorders <span class="citation">(Coufal et al. <a href="#ref-CoufalAtaxiatelangiectasiamutated2011">2011</a>; Douville et al. <a href="#ref-DouvilleIdentificationactiveloci2011">2011</a>; Jeong et al. <a href="#ref-Jeongprevalencehumanendogenous2010">2010</a>; Wenxue Li et al. <a href="#ref-LiHumanendogenousretrovirusK2015a">2015</a>; Kaneko et al. <a href="#ref-KanekoDICER1deficitinduces2011">2011</a>; Lee et al. <a href="#ref-LeeInvolvementEndogenousRetroviruses2013">2013</a>; Li et al. <a href="#ref-LiTransposableelementsTDP43mediated2012">2012</a>; Mameli et al. <a href="#ref-MameliActivationMSRVTypeEndogenous2013a">2013</a>; Perron et al. <a href="#ref-PerronHumanendogenousretrovirus2005">2005</a>)</span>, which arise undeniably due to deficiencies in their repression. Additionally, DNMT and HDAC inhibitors, which have been tested as therapies for cancer and neurodegenerative disease, actually induce cryptic transcription start sites in LTRs <span class="citation">(Brocks et al. <a href="#ref-BrocksDNMTHDACinhibitors2017">2017</a>)</span>. Increased non-genic expression in aging and neurodegenerative states are not limited to retro-transposons, as multiple lncRNAs show neurodegenerative and age-associated increases <span class="citation">(Nishimoto et al. <a href="#ref-NishimotolongnoncodingRNA2013">2013</a>; Salvi and Mekhail <a href="#ref-SalviRloopshighlightnucleus2015">2015</a>)</span> including an age-dependent accumulation of circRNA <span class="citation">(Gruner et al. <a href="#ref-GrunerCircRNAaccumulationaging2016">2016</a>; Larsen et al. <a href="#ref-LarsenWarningSINEsAlu2018b">2018</a>)</span>.</p>
<p>In the case of Amyotrophic Lateral sclerosis (ALS) and fronto-temporal lobar degeneration, a potent risk factor are mutations to the TAR DNA-binding protein-43 ( <em>TDP-43</em> ), an aggregation-prone, multifunctional RNA-binding protein. TDP-43 shows extensive binding to intronic RNA and RNA derived from LINE, SINE and LTR transposons, often localizing in stress granules <span class="citation">(Li et al. <a href="#ref-LiTransposableelementsTDP43mediated2012">2012</a>)</span>. In cortical tissue from patients with fronto-temporal lobar degeneration this association between TDP-43 and transposon RNA was drastically reduced <span class="citation">(Li et al. <a href="#ref-LiTransposableelementsTDP43mediated2012">2012</a>)</span>. While it was established that human TDP-43 can disturb siRNA-mediated retrotransposon silencing in drosophila <span class="citation">(Krug et al. <a href="#ref-KrugRetrotransposonactivationcontributes2017a">2017</a>)</span>, further examinations suggest that increases in repetitive element transcription can not be explained by TDP-43 alone <span class="citation">(Prudencio et al. <a href="#ref-PrudencioRepetitiveelementtranscripts2017">2017</a>)</span>. It is thus conceivable that natural age-related increases in retrotransposon expression is synergistic with TDP-43 deficits in generating ALS pathogenesis. It could be that sustained recruitment of TDP-43 to stress granules by transposon RNA or abnormally processed ncRNA <span class="citation">(Wu et al. <a href="#ref-WuUnusualProcessingGenerates2016">2016</a>)</span> results in either sequestration or prolonged aggregation of TDP-43 prion-like particles <span class="citation">(Nonaka et al. <a href="#ref-NonakaPrionlikePropertiesPathological2013">2013</a>)</span>.</p>
<p>In tauopathies such as Alzheimer’s disease, a body of recent works across drosophila mouse and man has shown that both hyper-phosphorylated tau and an over abundance of WT tau induces global chromatin relaxation and heterochromatin including loss of drosophila HP1<span class="math inline">\(\alpha\)</span> and SUV39H1. This tau-induced chromatin relaxation results in the expression of previously heterochromatinized genes <span class="citation">(Frost et al. <a href="#ref-FrostTaupromotesneurodegeneration2014">2014</a>)</span> and activation of transposable elements <span class="citation">(Guo et al. <a href="#ref-GuoTauActivatesTransposable2018">2018</a>)</span>. Conversely, complete loss of tau affects the composition, structure, and repair of neuronal pericentric heterochromatin <span class="citation">(Mansuroglu et al. <a href="#ref-MansurogluLossTauprotein2016">2016</a>)</span>, suggesting that tau has distinct roles in maintaining neuronal chromatin architectures.</p>
<p>Outside of distinctly pathogenic environments, the defining feature of neurons, namely their ability to transmit electrochemical signals by transient membrane depolarization, leave them exceptionally prone to epigenetic drift. During neuronal excitation, it has been shown that serine 10 of histone 3 (H3S10ph) is phosphorylated following periods of extended neuronal depolarization <span class="citation">(Noh et al. <a href="#ref-NohATRXtoleratesactivitydependent2014">2014</a>)</span>. This was surprising, given H3S10 phosphorylation is a mark normally observed in mitosis which causes HP1 proteins to be ejected from histones due to steric hindrance with the adjacent methylated H3K9me3 <span class="citation">(Hirota et al. <a href="#ref-HirotaHistoneH3serine2005">2005</a>)</span>. In neurons, HP1 proteins are also ejected from chromatin following stimulation while ATRX is capable of binding to the combination H3K9me3S10ph. Retention of ATRX on heterochromatin is important at repetitive elements and imprinted loci during neuronal activity <span class="citation">(Noh et al. <a href="#ref-NohATRXtoleratesactivitydependent2014">2014</a>; Voon et al. <a href="#ref-VoonATRXplayskey2015">2015</a>)</span>. It is presumed that following a neuron’s return from extended excitability to the resting state, HP1 proteins re-assemble along with ATRX and other heterochromatin components.</p>
<div class="figure" style="text-align: center"><span id="fig:ADEPT"></span>
<img src="figure/introduction/ADEPT.png" alt="Age-related augmentation of Activity-DEPendent Transposition.  Left column: Young neurons correctly repair activity-dependent DSBs by the non-homologous end-joining (NHEJ) pathway (bottom panel) (Madabhushi et al., 2015). Right column: In aged neurons, epigenetic drift results in poor definition of normal chromatin compartments defined by histone modifications and DNA methylation. An 'epigenetic scar' is likely to form in the vicinity of the enhancer/promoter region, due to improper re-establishment of epigenetic marks following a lifetime of DNA repair, and repetitive sequences are no longer repressed by DNA methylation and H3K9me3. It is likely that such epigenetic drift diverts DNA repair to transcription coupled homologous recombination (TCHR), which uses the available repeat-rich lncRNA as a template.  Homologous recombination between repetitive lncRNA could result in (a) large deletions, (b) alternative TSS generation and (c) insertions at the immediate early gene promoter. (Newman et al., 2017)" width="100%"   />
<p class="caption">
Figure 1.4: Age-related augmentation of Activity-DEPendent Transposition. Left column: Young neurons correctly repair activity-dependent DSBs by the non-homologous end-joining (NHEJ) pathway (bottom panel) (Madabhushi et al., 2015). Right column: In aged neurons, epigenetic drift results in poor definition of normal chromatin compartments defined by histone modifications and DNA methylation. An ‘epigenetic scar’ is likely to form in the vicinity of the enhancer/promoter region, due to improper re-establishment of epigenetic marks following a lifetime of DNA repair, and repetitive sequences are no longer repressed by DNA methylation and H3K9me3. It is likely that such epigenetic drift diverts DNA repair to transcription coupled homologous recombination (TCHR), which uses the available repeat-rich lncRNA as a template. Homologous recombination between repetitive lncRNA could result in (a) large deletions, (b) alternative TSS generation and (c) insertions at the immediate early gene promoter. (Newman et al., 2017)
</p>
</div>
<p>The second consequence of neuronal activity is DNA damage, which consistently increases the likelihood of heterochromatin loss <span class="citation">(Oberdoerffer and Sinclair <a href="#ref-Oberdoerfferrolenucleararchitecture2007">2007</a>)</span>. The high metabolism of neurons is directly linked with higher oxidative DNA damage in normal aging and multiple neurodegenerative disorders (for a review see <span class="citation">Copped‘e and Migliore (<a href="#ref-CoppedeDNAdamageneurodegenerative2015">2015</a>)</span>). If that is not enough, a miraculous feature of neuronal chromatin is the property of activity-dependent DNA double strand breaks (DSBs). Following sustained neuronal activity, DNA double-strand breaks are required for the induction of several immediate early genes, which enables contact of the immediate early gene promoter with its respective enhancer <span class="citation">(Madabhushi et al. <a href="#ref-MadabhushiActivityInducedDNABreaks2015">2015</a>)</span>. With age, one could easily imagine a scenario where loss of heterochromatin boundaries and expression of repetitive elements adversely affect the normal repair of activity-dependent double strand breaks (Figure <a href="1-introduction.html#fig:ADEPT">1.4</a>). While neuronal activity-dependent DSBs are normally repaired by blunt non-homologous end joining (NHEJ), an altered epigenetic landscape may direct DNA repair to use transcription-coupled homologous recombination (TCHR) repair <span class="citation">(Wei, Levine, and Lan <a href="#ref-WeiTranscriptioncoupledhomologousrecombination2016">2016</a>; McDevitt et al. <a href="#ref-McDevittHowRNAtranscripts2018">2018</a>)</span>, where RNA is used as a template. In an aged neuron, elevated levels of repetitive element RNA may be incorrectly used as repair template and result in Activity DEPendent Transposition (ADEPT), generating deletions, alterations to splicing, and insertions (reviewed in <span class="citation">Newman et al. (<a href="#ref-NewmanActivityDEPendentTransposition2016">2016</a>)</span>; <span class="citation">Newman et al. (<a href="#ref-NewmanActivityDEPendentTransposition2017">2017</a>)</span>, Figure <a href="1-introduction.html#fig:ADEPT">1.4</a>).</p>

</div>
</div>
<div id="objective" class="section level2">
<h2><span class="header-section-number">1.4</span> Objective</h2>
<p>Central to heterochromatin stability, maintenance, and DNA methylation are the HP1 proteins. Here we seek to examine definitively in a mammalian system how the protracted loss of HP1 proteins directly affects neuronal fitness and cognition, and if these affects recapitulate an accelerated variant of normal aging. Specifically:</p>
<ul>
<li><ol style="list-style-type: lower-alpha">
<li>does loss of HP1 accelerate age-related decreases in neuronal complexity and cognition,</li>
</ol></li>
<li><ol start="2" style="list-style-type: lower-alpha">
<li>what are the molecular mechanism(s) behind this, and</li>
</ol></li>
<li><ol start="3" style="list-style-type: lower-alpha">
<li>can areas of the mouse genome that are sensitive to HP1 loss inform on the progression of epigenomic aging in human neurons.</li>
</ol></li>
</ul>

</div>
<div id="hp1-proteins---a-background" class="section level2">
<h2><span class="header-section-number">1.5</span> HP1 Proteins - A Background</h2>
<p>Heterochromatin Protein 1 was first identified in <em>Drosophila Melanogaster</em> as mutation Su(var)205, for its ability to dominantly suppress position-effect variegation <span class="citation">(James and Elgin <a href="#ref-JamesIdentificationnonhistonechromosomal1986">1986</a>)</span>. Following successful cloning of the full gene product, it was revealed that this protein primarily localizes to pericentromeric heterochromatin <span class="citation">(James et al. <a href="#ref-JamesDistributionpatternsHP11989">1989</a>)</span> and is highly conserved, found in both plants and animals <span class="citation">(Singh et al. <a href="#ref-Singhsequencemotiffound1991">1991</a>)</span>. In addition to binding to the globular portion of Histone 3 <span class="citation">(Canzio et al. <a href="#ref-CanzioconformationalswitchHP12013">2013</a>)</span>, HP1 proteins are famous for their high affinity binding to <em>di</em> and <em>tri</em> methylated H3K9 <span class="citation">(Bannister et al. <a href="#ref-BannisterSelectiverecognitionmethylated2001">2001</a>; Lachner et al. <a href="#ref-LachnerMethylationhistoneH32001">2001</a>)</span>, a function that is important for its role in mediating induction and spreading of heterochromatin.</p>
<div class="figure" style="text-align: center"><span id="fig:HP1structure"></span>
<img src="figure/introduction/HP1structure.png" alt="Structure of Mammalian HP1 Proteins. (A) Protein sequences and domains of mouse HP1 proteins (modified from Maison and Almouzni 2004). (B) Surface view of an HP1 CSD dimer (one monomer is pink, the other is blue) binding to a CAF-1 peptide (stick model). (C) HP1 chromodomains form an aromatic cage to bind H3K9me3. (B,C) modified from Billur et al., (2010)." width="100%"   />
<p class="caption">
Figure 1.5: Structure of Mammalian HP1 Proteins. (A) Protein sequences and domains of mouse HP1 proteins (modified from Maison and Almouzni 2004). (B) Surface view of an HP1 CSD dimer (one monomer is pink, the other is blue) binding to a CAF-1 peptide (stick model). (C) HP1 chromodomains form an aromatic cage to bind H3K9me3. (B,C) modified from Billur et al., (2010).
</p>
</div>
<div id="the-abgs-of-hp1-proteins" class="section level3 unnumbered">
<h3>The <em>ABG’s</em> of HP1 Proteins</h3>
<p>HP1 proteins are encoded by subset of chomobox ( <em>Cbx</em> ) genes so named due to the presence of a chromobox domain in their protein product. In mammals, three homologs of HP1 exist, termed HP1<span class="math inline">\(\alpha\)</span> ( <em>Cbx5</em> ), HP1<span class="math inline">\(\beta\)</span> ( <em>Cbx1</em> ) and HP1<span class="math inline">\(\gamma\)</span> ( <em>Cbx3</em> ). All three display a high degree of sequence similarity, each containing a variable hinge domain and two structured domains, the chromodomain (CD) and the chromoshadow domain (CSD). The chromodomain, which is found on many proteins, recognizes H3K9me2 and H3K9me3 with increasing affinity <span class="citation">(Bannister et al. <a href="#ref-BannisterSelectiverecognitionmethylated2001">2001</a>; Nielsen et al. <a href="#ref-NielsenStructureHP1chromodomain2002">2002</a>)</span>. The chromoshadow domain is unique to HP1 proteins, and is capable of dimerization and binding to proteins which contain a PxVxL motif <span class="citation">(Smothers and Henikoff <a href="#ref-SmothersHP1chromoshadow2000">2000</a>; Brasher et al. <a href="#ref-BrasherstructuremouseHP12000">2000</a>; Thiru et al. <a href="#ref-ThiruStructuralbasisHP12004">2004</a>)</span>. Finally, the variable hinge region connects both CD and CSD domains, and is implicated in sequence-independent RNA and DNA binding <span class="citation">(Keller et al. <a href="#ref-KellerHP1Swi6mediates2012">2012</a>; Zhao et al. <a href="#ref-ZhaoHeterochromatinproteinbinds2000">2000</a>)</span> (Figure <a href="1-introduction.html#fig:HP1structure">1.5</a>).</p>
<p>The structure of HP1 proteins enables unique interactions with methyltransferases and methylated histones resulting in a feed-forward heterochromatinization following a nucleation event. After nucleation induces deposition of H3K9me3 (typically by SETDB1) and recruitment of HP1, HP1 recruits further methyltransferases such as G9a, GLP or SUV39H1, which in turn methylate adjacent histones, recruiting more HP1, which recruits more methylransferase <em>et cetera</em>, extending the repressive heterochromatin environment (Figure <a href="1-introduction.html#fig:HP1spreading">1.6</a>). The creation of such large H3K9me3 domains is akin to building hills in Waddington’s landscape (Figure <a href="1-introduction.html#fig:landscape">1.1</a>): H3K9me3 at reprogramming resistant regions (RRRs, generated by SUV39H1/2) <span class="citation">(Matoba et al. <a href="#ref-MatobaEmbryonicDevelopmentfollowing2014">2014</a>)</span> and developmental regulators (generated by SETDB1 and bound by HP1<span class="math inline">\(\gamma\)</span>) form barriers to efficient reprogramming of murine induced pluripotent stem cells.</p>
<div class="figure" style="text-align: center"><span id="fig:HP1spreading"></span>
<img src="figure/introduction/HP1spreading.png" alt="Function of HP1 Proteins. HP1 proteins are able to bind to the globular portion of Histone 3 via their chromoshadow domain (CSD), and bind with high affinity to H3K9me3 via their chromo domain (CD).  Many proteins are known to interact with HP1, and in doing so can sustain or perturb heterochromatin (top row).  Because HP1 proteins also interact with histone methyltransferases (HMTs) such as SUV39H1, recruitment of HP1 to H3K9me3 may result in a feed-forward loop resulting further methylation of adjacent nucleosomes and spreading of heterochromatin (bottom row). Modified from Canzio et. al., (2014)" width="100%"   />
<p class="caption">
Figure 1.6: Function of HP1 Proteins. HP1 proteins are able to bind to the globular portion of Histone 3 via their chromoshadow domain (CSD), and bind with high affinity to H3K9me3 via their chromo domain (CD). Many proteins are known to interact with HP1, and in doing so can sustain or perturb heterochromatin (top row). Because HP1 proteins also interact with histone methyltransferases (HMTs) such as SUV39H1, recruitment of HP1 to H3K9me3 may result in a feed-forward loop resulting further methylation of adjacent nucleosomes and spreading of heterochromatin (bottom row). Modified from Canzio et. al., (2014)
</p>
</div>
<p>The nuclear distribution of HP1 homologs is distinct along with their functions. Mammalian HP1<span class="math inline">\(\alpha\)</span> and HP1<span class="math inline">\(\beta\)</span> associate primarily with constitutive heterochromatin while HP1<span class="math inline">\(\gamma\)</span> has a more euchromatic distribution, associating with active genes and being important for transcriptional elongation <span class="citation">(Minc et al. <a href="#ref-MincLocalizationphosphorylationHP11999">1999</a>; Smallwood et al. <a href="#ref-SmallwoodCBX3regulatesefficient2012">2012</a>; Vakoc et al. <a href="#ref-VakocHistoneH3Lysine2005">2005</a>)</span>. The number of euchromatic regions of the genome bound by HP1<span class="math inline">\(\gamma\)</span> is numerous and depends largely on cell type <span class="citation">(Huang et al. <a href="#ref-HuangCbx3maintainslineage2017">2017</a>; Ostapcuk et al. <a href="#ref-OstapcukActivitydependentneuroprotectiveprotein2018">2018</a>; Smallwood et al. <a href="#ref-SmallwoodCBX3regulatesefficient2012">2012</a>)</span>.</p>
<p>HP1 homologs perform different functions despite high sequence homology in their functional domains. The CSD and CD of HP1<span class="math inline">\(\alpha\)</span> and HP1<span class="math inline">\(\gamma\)</span> show 87% and 71% sequence identity, respectively, but nonetheless domain-swapping experiments between homologs have revealed non-redundant functions <span class="citation">(Canzio, Larson, and Narlikar <a href="#ref-CanzioMechanismsfunctionalpromiscuity2014">2014</a>; Smothers and Henikoff <a href="#ref-SmothersHingeChromoShadow2001">2001</a>)</span>. We now have some understanding of how HP1 proteins function differently, which involve several subtle differences in the CD, CSD and hinge domains.</p>
<p>For instance, HP1 binding to H3K9me2/3 can occur in different affinities ranging from 5-40<span class="math inline">\(\mu\)</span>M depending on both HP1 homolog and post-translational modifications. The phosphorylation of serine residues present only on the N-terminal extension (NTE) of mammalian HP1<span class="math inline">\(\alpha\)</span> can increase its affinity for H3K9me3 approximately five-fold, close to that of both mouse HP1<span class="math inline">\(\beta\)</span> and HP1<span class="math inline">\(\gamma\)</span> <span class="citation">(Hiragami-Hamada et al. <a href="#ref-Hiragami-HamadaNTerminalPhosphorylationHP12011">2011</a>, Nishibuchi et al. <a href="#ref-NishibuchiNterminalphosphorylationHP1a2014">2014</a>)</span>. Such a property may facilitate rapid assembly/disassembly of a heterochromatin domain in the presence of a kinase/phosphatase, which seems to be the case, as NTE phosphorylation of HP1<span class="math inline">\(\alpha\)</span> persists during cell cycle <span class="citation">(Hiragami-Hamada et al. <a href="#ref-Hiragami-HamadaNTerminalPhosphorylationHP12011">2011</a>)</span>. By contrast, negatively charged glutamate residues replace most of the serines in the NTE of HP1<span class="math inline">\(\beta\)</span> and HP1<span class="math inline">\(\gamma\)</span>, and an increased number of acidic residues in the NTE of HP1<span class="math inline">\(\gamma\)</span> make the HP1<span class="math inline">\(\gamma\)</span>-CD-H3K9me3 complex more flexible than the HP1<span class="math inline">\(\beta\)</span>-CD-H3K9me3 complex <span class="citation">(Machado, Dans, and Pantano <a href="#ref-MachadoIsoformspecificdeterminantsHP12010">2010</a>)</span>.</p>
<p>In line with such differences, recent work has shown that it is HP1<span class="math inline">\(\alpha\)</span> that interacts with CTCF in pericentric heterochromatin, and its loss can result in hypercompaction through excess spreading of H3K27me3 and H4K20me3 by HP1<span class="math inline">\(\beta\)</span> and HP1<span class="math inline">\(\gamma\)</span>. Conversely, depletion of HP1<span class="math inline">\(\beta\)</span> results in decompaction of pericentric heterochromatin <span class="citation">(Bosch-Presegu’e et al. <a href="#ref-Bosch-PresegueMammalianHP1Isoforms2017">2017</a>)</span>, and depletion of both HP1<span class="math inline">\(\alpha\)</span> and HP1<span class="math inline">\(\gamma\)</span> abolishes the recruitment of HP1<span class="math inline">\(\gamma\)</span> to pericentric heterochromatin <span class="citation">(Dialynas et al. <a href="#ref-DialynasPlasticityHP1proteins2007">2007</a>)</span>.</p>
<p>Additionally, HP1 proteins show differential binding via their CSD domains to the globular core of histone 3. While some studies have observed that HP1<span class="math inline">\(\alpha\)</span> and HP1<span class="math inline">\(\gamma\)</span> but <em>not</em> HP1<span class="math inline">\(\beta\)</span> bind to the PGTVAL motif on the <span class="math inline">\(\alpha\)</span>N helix of globular H3 <span class="citation">(Dawson et al. <a href="#ref-DawsonJAK2phosphorylateshistone2009">2009</a>; Lavigne et al. <a href="#ref-LavigneInteractionHP1Brg12009">2009</a>)</span>, elsewhere it has been reported that HP1<span class="math inline">\(\beta\)</span> does associate with the globular histone <span class="citation">(Dialynas <a href="#ref-DialynasMethylationindependentBindingHistone2006">2006</a>)</span>. Additionally, two residues away from the PGTVAL motif on the histone core, HP1<span class="math inline">\(\alpha\)</span> is sensitive to phosphorylation of tyrosine 41 (Y41), which inhibits its binding. Furthermore, HP1<span class="math inline">\(\gamma\)</span> is insensitive to Y41 phosphorylation, which is consistent with phosphorylation of this mark being correlated with transcription <span class="citation">(Dawson et al. <a href="#ref-DawsonJAK2phosphorylateshistone2009">2009</a>; Lavigne et al. <a href="#ref-LavigneInteractionHP1Brg12009">2009</a>)</span>. Essentially this means that HP1<span class="math inline">\(\beta\)</span> does not bind core H3, HP1<span class="math inline">\(\gamma\)</span> always can, and HP1<span class="math inline">\(\alpha\)</span> can conditionally bind to it in the absence of Y41 phosphorylation.</p>
<p>The largest source of functional diversity perhaps comes from the hinge region, which is both poorly conserved and variable in length between HP1 homologs. In addition to containing the nuclear localization signal (NLS), phosphorylation and sumoylation modifications have been observed at the hinge region <span class="citation">(LeRoy et al. <a href="#ref-LeRoyHeterochromatinProteinExtensively2009">2009</a>; Maison et al. <a href="#ref-MaisonSUMOylationpromotesnovo2011">2011</a>)</span>. Sumoylation of the hinge region can promote <em>de novo</em> targeting of mouse HP1<span class="math inline">\(\alpha\)</span> but not HP1<span class="math inline">\(\beta\)</span> or HP1<span class="math inline">\(\gamma\)</span> to pericentric heterochromatin by associating with RNA. More importantly, phosphorylation of serine 83 in the hinge of human HP1<span class="math inline">\(\gamma\)</span> both increases its localization to euchromatin and increases its interaction with XRCC6 (also known as Ku70), a central DNA repair protein <span class="citation">(Lomberk et al. <a href="#ref-LomberkEvidenceexistenceHP1mediated2006">2006</a>)</span>.</p>
<div class="figure" style="text-align: center"><span id="fig:HP1pip"></span>
<img src="figure/introduction/HP1pip.png" alt="HP1 Protein-Protein Interaction Network.  Protein-Protein interactions (PIPs) were queried for each HP1 homolog from published data hosted on BioGrid in 2015 (http://thebiogrid.org). Newman 2015, unpublished." width="100%"   />
<p class="caption">
Figure 1.7: HP1 Protein-Protein Interaction Network. Protein-Protein interactions (PIPs) were queried for each HP1 homolog from published data hosted on BioGrid in 2015 (<a href="http://thebiogrid.org" class="uri">http://thebiogrid.org</a>). Newman 2015, unpublished.
</p>
</div>
<p>An appreciation of such functional diversity comes naturally when looking at a network of documented protein-protein interactions that occur with HP1 proteins (Figure <a href="1-introduction.html#fig:HP1pip">1.7</a>). Generated in 2015, this protein-protein network is by no means exhaustive, nor mutually exclusive, it does serve as a starting point for documented interactions common to all three homologs, those common to two, and unique interactions. Recent high-throughput approaches such as mass-spec offer much more unbiased measurements of protein binding (for example see <span class="citation">Ostapcuk et al. (<a href="#ref-OstapcukActivitydependentneuroprotectiveprotein2018">2018</a>)</span>) that can inform on which proteins are interactors of individual HP1 homologs.<br />
</p>
<p>It has long been suspected that the essential function of HP1 proteins may lay outside its classical H3K9me3 binding ability, given double mutants of SUV39H1/H2 are viable, but HP1<span class="math inline">\(\beta\)</span> mutants are perinatal lethal <span class="citation">(Billur, Bartunik, and Singh <a href="#ref-BilluressentialfunctionHP1v2010">2010</a>)</span>. Evidence for such roles is forthcoming, as it is now understood that HP1 proteins can create locally restricted repression in the absence of H3K9me3 by associating with chromatin remodellers ADNP and CHD4 <span class="citation">(Ostapcuk et al. <a href="#ref-OstapcukActivitydependentneuroprotectiveprotein2018">2018</a>)</span>. HP1 proteins were known to recruit ADNP to H3K9me3 positive heterochromatin to silence major satellites <span class="citation">(Mosch et al. <a href="#ref-MoschHP1RecruitsActivityDependent2011">2011</a>)</span>, however Ostapcuk and colleagues observed that CHD4-ADNP-HP1 (ChAHP)-mediated repression occurred independently of H3K9me3. They observed that ChAHP-mediated repression regulates the expression of genes that are crucial for maintaining distinct cellular fates. While ChAHP complexes use primarily HP1<span class="math inline">\(\beta\)</span> and HP1<span class="math inline">\(\gamma\)</span>, deletion of all three HP1 homologs was required to perturb their role in ChAHP repression, despite HP1<span class="math inline">\(\alpha\)</span> only weakly binding to ADNP.</p>
<p>Finally, recent work has begun to shift how we perceive heterochromatinization by HP1 proteins. Biophysical studies have characterized nucleation and heterochromatinization of HP1 proteins to be consistent with the physical properties of liquid phase separation <span class="citation">(Larson et al. <a href="#ref-LarsonLiquiddropletformation2017">2017</a>; Strom et al. <a href="#ref-StromPhaseseparationdrives2017">2017</a>)</span>. These experiments examined how under certain conditions HP1 proteins oligomerize to form liquid-phase separated droplets due to electrostatic interactions (Figure <a href="1-introduction.html#fig:HP1droplet">1.8</a>). Such experiments observed that such droplet formation only occurred with drosophila HP1a and human HP1<span class="math inline">\(\alpha\)</span>, where the phosphorylation of the N-terminal extension of human HP1<span class="math inline">\(\alpha\)</span> is integral to droplet generation. This greatly contributes to our understanding of how heterochromatin domains can be both stable and dynamic. DNA and HP1-interacting proteins can be incorporated into the ‘heterochromatin phase’, while non-interacting proteins are excluded and remain in the ‘euchromatin phase’. Strom et al., (2017) could confirm that the boundary between these phases is a strong barrier to protein diffusion.</p>
<div class="figure" style="text-align: center"><span id="fig:HP1droplet"></span>
<img src="figure/introduction/droplet.png" alt="HP1$\alpha$ can induce liquid phase separation to drive heterochromatin domain formation. Modified from Strom et. al., (2017)." width="100%"   />
<p class="caption">
Figure 1.8: HP1<span class="math inline">\(\alpha\)</span> can induce liquid phase separation to drive heterochromatin domain formation. Modified from Strom et. al., (2017).
</p>
</div>

<p>Collectively, HP1 proteins maintain a diverse repertoire of functions. They have proven roles in pluripotency <span class="citation">(Sridharan et al. <a href="#ref-SridharanProteomicgenomicapproaches2013">2013</a>)</span>, spermatogenesis <span class="citation">(Brown et al. <a href="#ref-BrownHP1gfunctionrequired2010">2010</a>; Leonard et al. <a href="#ref-LeonardAuroraAHP1gpathway2015">2015</a>)</span>, mitosis <span class="citation">(Liu et al. <a href="#ref-LiuChromatinproteinHP12014">2014</a>)</span>, DNA methylation <span class="citation">(Smallwood et al. <a href="#ref-SmallwoodFunctionalcooperationHP12007">2007</a>; Rottach et al. <a href="#ref-RottachmultidomainproteinNp952010">2010</a>)</span>, DNA repair <span class="citation">(Kalousi et al. <a href="#ref-KalousiNuclearOncogeneSET2015a">2015</a>; Soria and Almouzni <a href="#ref-SoriaDifferentialcontributionHP12013a">2013</a>; Luijsterburg et al. <a href="#ref-LuijsterburgHeterochromatinproteinrecruited2009a">2009</a>)</span>, cancer <span class="citation">(Dialynas, Vitalini, and Wallrath <a href="#ref-DialynasLinkingHeterochromatinProtein2008">2008</a>)</span>, transcriptional elongation <span class="citation">(Smallwood et al. <a href="#ref-SmallwoodCBX3regulatesefficient2012">2012</a>; Vakoc et al. <a href="#ref-VakocHistoneH3Lysine2005">2005</a>)</span>, RNA splicing <span class="citation">(Yearim et al. <a href="#ref-YearimHP1InvolvedRegulating2015">2015</a>)</span>, imprinting <span class="citation">(Bosch-Presegu’e et al. <a href="#ref-Bosch-PresegueMammalianHP1Isoforms2017">2017</a>; Leung et al. <a href="#ref-LeungRegulationDNAmethylation2014a">2014</a>; Quenneville et al. <a href="#ref-Quennevilleembryonicstemcells2011">2011</a>)</span>, and the silencing of repeat, regulatory and genic elements <span class="citation">(Ayyanathan <a href="#ref-AyyanathanRegulatedrecruitmentHP12003a">2003</a>; Bilodeau et al. <a href="#ref-BilodeauSetDB1contributesrepression2009">2009</a>; Karimi et al. <a href="#ref-KarimiDNAMethylationSETDB12011">2011</a>; Maksakova et al. <a href="#ref-MaksakovaDistinctrolesKAP12013">2013</a>; Ostapcuk et al. <a href="#ref-OstapcukActivitydependentneuroprotectiveprotein2018">2018</a>; Raurell-Vila et al. <a href="#ref-Raurell-VilaHP1isoformspecificfeedback2017">2017</a>; Schultz et al. <a href="#ref-SchultzSETDB1novelKAP1associated2002a">2002</a>)</span>. Yet, only two studies so far have investigated the role of HP1 proteins in neurons. Recent in vitro experiments suggest HP1<span class="math inline">\(\gamma\)</span> to be important in specifying neuronal lineage <span class="citation">(Huang et al. <a href="#ref-HuangCbx3maintainslineage2017">2017</a>)</span>, while animals with constitutive deletion of HP1<span class="math inline">\(\beta\)</span> show mitotic deficits in cortical progenitors and the formation of neuromuscular junctions <span class="citation">(Aucott et al. <a href="#ref-AucottHP1requireddevelopment2008">2008</a>)</span>.</p>
<p>Here I examine in detail how all three HP1 proteins modulate the development, maturation and aging of pyramidal neurons of the cerebral cortex and hippocampus.</p>

</div>
</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-AristotleGenerationanimalsEnglish1943">
<p>Aristotle, and Arthur Leslie Peck. 1943. <em>Generation of Animals, with an English Translation by A.L. Peck</em>. Cambridge Harvard University Press. <a href="http://archive.org/details/generationofanim00arisuoft" class="uri">http://archive.org/details/generationofanim00arisuoft</a>.</p>
</div>
<div id="ref-HarveyAnatomicalexercitationsconcerning1653">
<p>Harvey, William. 1653. “Anatomical Exercitations Concerning the Generation of Living Creatures to Which Are Added Particular Discourses of Births and of Conceptions.” Oxford Text Archive. <a href="http://ota.ox.ac.uk/tcp/headers/A43/A43030.html" class="uri">http://ota.ox.ac.uk/tcp/headers/A43/A43030.html</a>.</p>
</div>
<div id="ref-WaddingtonOrganisersGenes1940">
<p>Waddington, C. H. 1940. “Organisers and Genes.” <em>Cambridge University Press</em>.</p>
</div>
<div id="ref-Feinbergkeyroleepigenetics2018">
<p>Feinberg, Andrew P. 2018. “The Key Role of Epigenetics in Human Disease Prevention and Mitigation.” <em>New England Journal of Medicine</em> 378 (14): 1323–34.</p>
</div>
<div id="ref-FerrariEpigenomemodifiersmetabolic2018">
<p>Ferrari, Alessandra, Raffaella Longo, Rui Silva, Nico Mitro, Donatella Caruso, Emma De Fabiani, and Maurizio Crestani. 2018. “Epigenome Modifiers and Metabolic Rewiring: New Frontiers in Therapeutics.” <em>Pharmacology &amp; Therapeutics</em>.</p>
</div>
<div id="ref-CheloufiEmergingroleshistone2017">
<p>Cheloufi, Sihem, and Konrad Hochedlinger. 2017. “Emerging Roles of the Histone Chaperone CAF-1 in Cellular Plasticity.” <em>Current Opinion in Genetics &amp; Development</em> 46: 83–94.</p>
</div>
<div id="ref-CheloufihistonechaperoneCAF12015">
<p>Cheloufi, Sihem, Ulrich Elling, Barbara Hopfgartner, Youngsook L. Jung, Jernej Murn, Maria Ninova, Maria Hubmann, Aimee I. Badeaux, Cheen Euong Ang, and Danielle Tenen. 2015. “The Histone Chaperone CAF-1 Safeguards Somatic Cell Identity.” <em>Nature</em> 528 (7581): 218.</p>
</div>
<div id="ref-Karlicroleepigeneticsregulation2014">
<p>Karlic, Heidrun, Harald Herrmann, Franz Varga, Roman Thaler, Rene Reitermaier, Silvia Spitzer, Viviane Ghanim, Katharina Blatt, Wolfgang R. Sperr, and Peter Valent. 2014. “The Role of Epigenetics in the Regulation of Apoptosis in Myelodysplastic Syndromes and Acute Myeloid Leukemia.” <em>Critical Reviews in Oncology/Hematology</em> 90 (1): 1–16.</p>
</div>
<div id="ref-CampbellMetabolicSignalingNucleus2018">
<p>Campbell, Sydney L., and Kathryn E. Wellen. 2018. “Metabolic Signaling to the Nucleus in Cancer.” <em>Molecular Cell</em> 71 (3): 398–408.</p>
</div>
<div id="ref-KinnairdMetaboliccontrolepigenetics2016">
<p>Kinnaird, Adam, Steven Zhao, Kathryn E. Wellen, and Evangelos D. Michelakis. 2016. “Metabolic Control of Epigenetics in Cancer.” <em>Nature Reviews Cancer</em> 16 (11): nrc. 2016.82.</p>
</div>
<div id="ref-RiggsinactivationdifferentiationDNA1975">
<p>Riggs, A. D. 1975. “X Inactivation, Differentiation, and DNA Methylation.” <em>Cytogenetic and Genome Research</em> 14 (1): 9–25. doi:<a href="https://doi.org/10.1159/000130315">10.1159/000130315</a>.</p>
</div>
<div id="ref-ListerHumanDNAmethylomes2009">
<p>Lister, Ryan, Mattia Pelizzola, Robert H. Dowen, R. David Hawkins, Gary Hon, Julian Tonti-Filippini, Joseph R. Nery, Leonard Lee, Zhen Ye, and Que-Minh Ngo. 2009. “Human DNA Methylomes at Base Resolution Show Widespread Epigenomic Differences.” <em>Nature</em> 462 (7271): 315.</p>
</div>
<div id="ref-EhrlichAmountdistribution5methylcytosine1982">
<p>Ehrlich, Melanie, Miguel A. Gama-Sosa, Lan-Hsiang Huang, Rose Marie Midgett, Kenneth C. Kuo, Roy A. McCune, and Charles Gehrke. 1982. “Amount and Distribution of 5-Methylcytosine in Human DNA from Different Types of Tissues or Cells.” <em>Nucleic Acids Research</em> 10 (8): 2709–21. doi:<a href="https://doi.org/10.1093/nar/10.8.2709">10.1093/nar/10.8.2709</a>.</p>
</div>
<div id="ref-GruenbaumMethylationCpGsequences1981">
<p>Gruenbaum, Yosef, Reuven Stein, Howard Cedar, and Aharon Razin. 1981. “Methylation of CpG Sequences in Eukaryotic DNA.” <em>FEBS Letters</em> 124 (1): 67–71. doi:<a href="https://doi.org/10.1016/0014-5793(81)80055-5">10.1016/0014-5793(81)80055-5</a>.</p>
</div>
<div id="ref-BirdCpGrichislandsfunction1986">
<p>Bird, Adrian P. 1986. “CpG-Rich Islands and the Function of DNA Methylation.” <em>Nature</em> 321 (6067): 209–13.</p>
</div>
<div id="ref-FioreCytosinemethylationtransforms1999">
<p>Fiore, Barbara Di, Antonella Palena, Armando Felsan, Franco Palitti, Maurizia Caruso, and Patrizia Lavia. 1999. “Cytosine Methylation Transforms an E2f Site in the Retinoblastoma Gene Promoter into a Binding Site for the General Repressor Methylcytosine-Binding Protein 2 (MeCP2).” <em>Nucleic Acids Research</em> 27 (14): 2852–9.</p>
</div>
<div id="ref-HarkCTCFmediatesmethylationsensitive2000">
<p>Hark, Amy T., Christopher J. Schoenherr, David J. Katz, Robert S. Ingram, John M. Levorse, and Shirley M. Tilghman. 2000. “CTCF Mediates Methylation-Sensitive Enhancer-Blocking Activity at the H19/Igf2 Locus.” <em>Nature</em> 405 (6785): 486.</p>
</div>
<div id="ref-ManciniSitespecificDNAmethylation1999">
<p>Mancini, Debora N., Shiva M. Singh, Trevor K. Archer, and David I. Rodenhiser. 1999. “Site-Specific DNA Methylation in the Neurofibromatosis (Nf1) Promoter Interferes with Binding of CREB and Sp1 Transcription Factors.” <em>Oncogene</em> 18 (28): 4108.</p>
</div>
<div id="ref-PrendergastMethylationsensitivesequencespecificDNA1991">
<p>Prendergast, George C., and Edward B. Ziff. 1991. “Methylation-Sensitive Sequence-Specific DNA Binding by the c-Myc Basic Region.” <em>Science</em> 251 (4990): 186–89.</p>
</div>
<div id="ref-TateEffectsDNAmethylation1993">
<p>Tate, Peri H., and Adrian P. Bird. 1993. “Effects of DNA Methylation on DNA-Binding Proteins and Gene Expression.” <em>Current Opinion in Genetics &amp; Development</em> 3 (2): 226–31.</p>
</div>
<div id="ref-JonesFunctionsDNAmethylation2012">
<p>Jones, Peter A. 2012. “Functions of DNA Methylation: Islands, Start Sites, Gene Bodies and Beyond.” <em>Nature Reviews Genetics</em> 13 (May): 484. <a href="http://dx.doi.org/10.1038/nrg3230" class="uri">http://dx.doi.org/10.1038/nrg3230</a>.</p>
</div>
<div id="ref-KanedaEssentialrolenovo2004">
<p>Kaneda, Masahiro, Masaki Okano, Kenichiro Hata, Takashi Sado, Naomi Tsujimoto, En Li, and Hiroyuki Sasaki. 2004. “Essential Role for de Novo DNA Methyltransferase Dnmt3a in Paternal and Maternal Imprinting.” <em>Nature</em> 429 (6994): 900.</p>
</div>
<div id="ref-OkanoDNAmethyltransferasesDnmt3a1999">
<p>Okano, Masaki, Daphne W. Bell, Daniel A. Haber, and En Li. 1999. “DNA Methyltransferases Dnmt3a and Dnmt3b Are Essential for de Novo Methylation and Mammalian Development.” <em>Cell</em> 99 (3): 247–57.</p>
</div>
<div id="ref-BarauDNAmethyltransferaseDNMT3C2016b">
<p>Barau, Joan, Aurelie Teissandier, Natasha Zamudio, Stephanie Roy, Valerie Nalesso, Yann Herault, Florian Guillou, and Deborah Bourchis. 2016. “The DNA Methyltransferase Dnmt3c Protects Male Germ Cells from Transposon Activity.” <em>Science</em> 354 (6314): 909–12.</p>
</div>
<div id="ref-BourchisDnmt3Lestablishmentmaternal2001">
<p>Bourc’his, D’eborah, Guo-Liang Xu, Chyuan-Sheng Lin, Brooke Bollman, and Timothy H. Bestor. 2001. “Dnmt3L and the Establishment of Maternal Genomic Imprints.” <em>Science</em> 294 (5551): 2536–9.</p>
</div>
<div id="ref-HataDnmt3LcooperatesDnmt32002">
<p>Hata, Kenichiro, Masaki Okano, Hong Lei, and En Li. 2002. “Dnmt3L Cooperates with the Dnmt3 Family of de Novo DNA Methyltransferases to Establish Maternal Imprints in Mice.” <em>Development</em> 129 (8): 1983–93.</p>
</div>
<div id="ref-XieMutationsDNAmethyltransferase2006">
<p>Xie, Zheng-Hua, Yan-Nv Huang, Zhao-Xia Chen, Arthur D. Riggs, Jian-Ping Ding, Humaira Gowher, Albert Jeltsch, Hiroyuki Sasaki, Kenichiro Hata, and Guo-Liang Xu. 2006. “Mutations in DNA Methyltransferase Dnmt3b in ICF Syndrome Affect Its Regulation by Dnmt3l.” <em>Human Molecular Genetics</em> 15 (9): 1375–85.</p>
</div>
<div id="ref-Jainrahumutantallele2017">
<p>Jain, Devanshi, Cem Meydan, Julian Lange, Corentin Claeys Bouuaert, Nathalie Lailler, Christopher E. Mason, Kathryn V. Anderson, and Scott Keeney. 2017. “Rahu Is a Mutant Allele of Dnmt3c, Encoding a DNA Methyltransferase Homolog Required for Meiosis and Transposon Repression in the Mouse Male Germline.” <em>PLoS Genetics</em> 13 (8): e1006964.</p>
</div>
<div id="ref-BestorCloningsequencingcDNA1988">
<p>Bestor, Timothy, Andrew Laudano, Robert Mattaliano, and Vernon Ingram. 1988. “Cloning and Sequencing of a cDNA Encoding DNA Methyltransferase of Mouse Cells: The Carboxyl-Terminal Domain of the Mammalian Enzymes Is Related to Bacterial Restriction Methyltransferases.” <em>Journal of Molecular Biology</em> 203 (4): 971–83.</p>
</div>
<div id="ref-ChuangHumanDNAcytosine51997">
<p>Chuang, Linda S.-H., Hang-In Ian, Tong-Wey Koh, Huck-Hui Ng, Guoliang Xu, and Benjamin FL Li. 1997. “Human DNA-(Cytosine-5) Methyltransferase-PCNA Complex as a Target for p21WAF1.” <em>Science</em> 277 (5334): 1996–2000.</p>
</div>
<div id="ref-MortusewiczRecruitmentDNAmethyltransferase2005a">
<p>Mortusewicz, Oliver, Lothar Schermelleh, Joachim Walter, M. Cristina Cardoso, and Heinrich Leonhardt. 2005. “Recruitment of DNA Methyltransferase I to DNA Repair Sites.” <em>Proceedings of the National Academy of Sciences</em> 102 (25): 8905–9.</p>
</div>
<div id="ref-MetivierCyclicalDNAmethylation2008">
<p>M’etivier, Raphaël, Rozenn Gallais, Christophe Tiffoche, Christine Le P’eron, Renata Z. Jurkowska, Richard P. Carmouche, David Ibberson, Peter Barath, Florence Demay, and George Reid. 2008. “Cyclical DNA Methylation of a Transcriptionally Active Promoter.” <em>Nature</em> 452 (7183): 45.</p>
</div>
<div id="ref-BarretoGadd45apromotesepigenetic2007">
<p>Barreto, Guillermo, Andrea Schäfer, Joachim Marhold, Dirk Stach, Suresh K. Swaminathan, Vikas Handa, Gabi Döderlein, Nicole Maltry, Wei Wu, and Frank Lyko. 2007. “Gadd45a Promotes Epigenetic Gene Activation by Repair-Mediated DNA Demethylation.” <em>Nature</em> 445 (7128): 671.</p>
</div>
<div id="ref-TahilianiConversion5methylcytosine5hydroxymethylcytosine2009">
<p>Tahiliani, Mamta, Kian Peng Koh, Yinghua Shen, William A. Pastor, Hozefa Bandukwala, Yevgeny Brudno, Suneet Agarwal, Lakshminarayan M. Iyer, David R. Liu, and L. Aravind. 2009. “Conversion of 5-Methylcytosine to 5-Hydroxymethylcytosine in Mammalian DNA by MLL Partner Tet1.” <em>Science</em> 324 (5929): 930–35.</p>
</div>
<div id="ref-GuoHydroxylation5methylcytosineTET12011">
<p>Guo, Junjie U., Yijing Su, Chun Zhong, Guo-li Ming, and Hongjun Song. 2011. “Hydroxylation of 5-Methylcytosine by Tet1 Promotes Active DNA Demethylation in the Adult Brain.” <em>Cell</em> 145 (3): 423–34.</p>
</div>
<div id="ref-HahnDynamics5HydroxymethylcytosineChromatin2013">
<p>Hahn, Maria A., Runxiang Qiu, Xiwei Wu, Arthur X. Li, Heying Zhang, Jun Wang, Jonathan Jui, et al. 2013. “Dynamics of 5-Hydroxymethylcytosine and Chromatin Marks in Mammalian Neurogenesis.” <em>Cell Reports</em> 3 (2): 291–300. doi:<a href="https://doi.org/10.1016/j.celrep.2013.01.011">10.1016/j.celrep.2013.01.011</a>.</p>
</div>
<div id="ref-MellenMeCP2Binds5hmC2012">
<p>Mell’en, Marian, Pinar Ayata, Scott Dewell, Skirmantas Kriaucionis, and Nathaniel Heintz. 2012. “MeCP2 Binds to 5hmC Enriched Within Active Genes and Accessible Chromatin in the Nervous System.” <em>Cell</em> 151 (7): 1417–30. doi:<a href="https://doi.org/10.1016/j.cell.2012.11.022">10.1016/j.cell.2012.11.022</a>.</p>
</div>
<div id="ref-ItoTetproteinscan2011">
<p>Ito, Shinsuke, Li Shen, Qing Dai, Susan C. Wu, Leonard B. Collins, James A. Swenberg, Chuan He, and Yi Zhang. 2011. “Tet Proteins Can Convert 5-Methylcytosine to 5-Formylcytosine and 5-Carboxylcytosine.” <em>Science</em> 333 (6047): 1300–1303.</p>
</div>
<div id="ref-MaitiThymineDNAglycosylase2011">
<p>Maiti, Atanu, and Alexander C. Drohat. 2011. “Thymine DNA Glycosylase Can Rapidly Excise 5-Formylcytosine and 5-Carboxylcytosine Potential Implications for Active Demethylation of CpG Sites.” <em>Journal of Biological Chemistry</em> 286 (41): 35334–8.</p>
</div>
<div id="ref-LugerCrystalstructurenucleosome1997">
<p>Luger, Karolin, Armin W. Mäder, Robin K. Richmond, David F. Sargent, and Timothy J. Richmond. 1997. “Crystal Structure of the Nucleosome Core Particle at 2.8 Å Resolution.” <em>Nature</em> 389 (6648): 251.</p>
</div>
<div id="ref-Santos-RosaActivegenesare2002">
<p>Santos-Rosa, Helena, Robert Schneider, Andrew J. Bannister, Julia Sherriff, Bradley E. Bernstein, NC Tolga Emre, Stuart L. Schreiber, Jane Mellor, and Tony Kouzarides. 2002. “Active Genes Are Tri-Methylated at K4 of Histone H3.” <em>Nature</em> 419 (6905): 407–11.</p>
</div>
<div id="ref-JinDistinctrolesGCN52011">
<p>Jin, Qihuang, Li-Rong Yu, Lifeng Wang, Zhijing Zhang, Lawryn H. Kasper, Ji-Eun Lee, Chaochen Wang, Paul K. Brindle, Sharon YR Dent, and Kai Ge. 2011. “Distinct Roles of Gcn5/PCAF‐mediated H3K9ac and CBP/P300‐mediated H3k18/27ac in Nuclear Receptor Transactivation.” <em>The EMBO Journal</em> 30 (2): 249–62.</p>
</div>
<div id="ref-EdmundsDynamichistoneH32008a">
<p>Edmunds, John W., Louis C. Mahadevan, and Alison L. Clayton. 2008. “Dynamic Histone H3 Methylation During Gene Induction: HYPB/Setd2 Mediates All H3k36 Trimethylation.” <em>The EMBO Journal</em> 27 (2): 406–20.</p>
</div>
<div id="ref-HuHistoneH3lysine2010">
<p>Hu, M., X.-J. Sun, Y.-L. Zhang, Y. Kuang, C.-Q. Hu, W.-L. Wu, S.-H. Shen, et al. 2010. “Histone H3 Lysine 36 Methyltransferase Hypb/Setd2 Is Required for Embryonic Vascular Remodeling.” <em>Proceedings of the National Academy of Sciences</em> 107 (7): 2956–61. doi:<a href="https://doi.org/10.1073/pnas.0915033107">10.1073/pnas.0915033107</a>.</p>
</div>
<div id="ref-Kolasinska-ZwierzDifferentialchromatinmarking2009">
<p>Kolasinska-Zwierz, Paulina, Thomas Down, Isabel Latorre, Tao Liu, X Shirley Liu, and Julie Ahringer. 2009. “Differential Chromatin Marking of Introns and Expressed Exons by H3K36me3.” <em>Nature Genetics</em> 41 (February): 376. <a href="http://dx.doi.org/10.1038/ng.322" class="uri">http://dx.doi.org/10.1038/ng.322</a>.</p>
</div>
<div id="ref-YearimHP1InvolvedRegulating2015">
<p>Yearim, Ahuvi, Sahar Gelfman, Ronna Shayevitch, Shai Melcer, Ohad Glaich, Jan-Philipp Mallm, Malka Nissim-Rafinia, et al. 2015. “HP1 Is Involved in Regulating the Global Impact of DNA Methylation on Alternative Splicing.” <em>Cell Reports</em> 10 (7): 1122–34. doi:<a href="https://doi.org/10.1016/j.celrep.2015.01.038">10.1016/j.celrep.2015.01.038</a>.</p>
</div>
<div id="ref-Blascoepigeneticregulationmammalian2007">
<p>Blasco, Mar’ia A. 2007. “The Epigenetic Regulation of Mammalian Telomeres.” <em>Nature Reviews Genetics</em> 8 (4): 299–309. doi:<a href="https://doi.org/10.1038/nrg2047">10.1038/nrg2047</a>.</p>
</div>
<div id="ref-Santoronucleolarremodelingcomplex2002">
<p>Santoro, Raffaella, Junwei Li, and Ingrid Grummt. 2002. “The Nucleolar Remodeling Complex NoRC Mediates Heterochromatin Formation and Silencing of Ribosomal Gene Transcription.” <em>Nature Genetics</em> 32 (3): 393–96. doi:<a href="https://doi.org/10.1038/ng1010">10.1038/ng1010</a>.</p>
</div>
<div id="ref-WreggettmammalianhomologueDrosophila1994">
<p>Wreggett, K. A., F. Hill, P. S. James, A. Hutchings, G. W. Butcher, and P. B. Singh. 1994. “A Mammalian Homologue of Drosophila Heterochromatin Protein 1 (Hp1) Is a Component of Constitutive Heterochromatin.” <em>Cytogenetic and Genome Research</em> 66 (2): 99–103.</p>
</div>
<div id="ref-BannisterSelectiverecognitionmethylated2001">
<p>Bannister, Andrew J., Philip Zegerman, Janet F. Partridge, Eric A. Miska, Jean O. Thomas, Robin C. Allshire, and Tony Kouzarides. 2001. “Selective Recognition of Methylated Lysine 9 on Histone H3 by the Hp1 Chromo Domain.” <em>Nature</em> 410 (6824): 120–24.</p>
</div>
<div id="ref-LachnerMethylationhistoneH32001">
<p>Lachner, Monika, D’onal O’Carroll, Stephen Rea, Karl Mechtler, and Thomas Jenuwein. 2001. “Methylation of Histone H3 Lysine 9 Creates a Binding Site for Hp1 Proteins.” <em>Nature</em> 410 (6824): 116–20.</p>
</div>
<div id="ref-KourmouliHeterochromatintrimethylatedlysine2004">
<p>Kourmouli, Niki, Peter Jeppesen, Shantha Mahadevhaiah, Paul Burgoyne, Rong Wu, David M. Gilbert, Silvia Bongiorni, Giorgio Prantera, Laura Fanti, and Sergio Pimpinelli. 2004. “Heterochromatin and Tri-Methylated Lysine 20 of Histone H4 in Animals.” <em>Journal of Cell Science</em> 117 (12): 2491–2501.</p>
</div>
<div id="ref-Schottasilencingpathwayinduce2004">
<p>Schotta, G. 2004. “A Silencing Pathway to Induce H3-K9 and H4-K20 Trimethylation at Constitutive Heterochromatin.” <em>Genes &amp; Development</em> 18 (11): 1251–62. doi:<a href="https://doi.org/10.1101/gad.300704">10.1101/gad.300704</a>.</p>
</div>
<div id="ref-LubitLocalization5methylcytosinehuman1976">
<p>Lubit, Beverly W., Pham Tuan Duc, Orlando J. Miller, and Bernard F. Erlanger. 1976. “Localization of 5-Methylcytosine in Human Metaphase Chromosomes by Immunoelectron Microscopy.” <em>Cell</em> 9 (4): 503–9.</p>
</div>
<div id="ref-RicletDisruptionInteractionTranscriptional2008">
<p>Riclet, Raphaël, Mariam Chendeb, Jean-Luc Vonesch, Dirk Koczan, Hans-Juergen Thiesen, R’egine Losson, Florence Cammas, and Wendy Bickmore. 2008. “Disruption of the Interaction Between Transcriptional Intermediary Factor 1b and Heterochromatin Protein 1 Leads to a Switch from DNA Hyper- to Hypomethylation and H3k9 to H3k27 Trimethylation on the MEST Promoter Correlating with Gene Reactivation.” <em>Molecular Biology of the Cell</em> 20 (1): 296–305. doi:<a href="https://doi.org/10.1091/mbc.e08-05-0510">10.1091/mbc.e08-05-0510</a>.</p>
</div>
<div id="ref-MozzettahistoneH3lysine2014">
<p>Mozzetta, Chiara, Julien Pontis, Lauriane Fritsch, Philippe Robin, Manuela Portoso, Caroline Proux, Raphaël Margueron, and Slimane Ait-Si-Ali. 2014. “The Histone H3 Lysine 9 Methyltransferases G9a and GLP Regulate Polycomb Repressive Complex 2-Mediated Gene Silencing.” <em>Molecular Cell</em> 53 (2): 277–89.</p>
</div>
<div id="ref-CaofunctionsEZH2mediatedmethylation2004">
<p>Cao, Ru, and Yi Zhang. 2004. “The Functions of E(Z)/Ezh2-Mediated Methylation of Lysine 27 in Histone H3.” <em>Current Opinion in Genetics &amp; Development</em> 14 (2): 155–64. doi:<a href="https://doi.org/10.1016/j.gde.2004.02.001">10.1016/j.gde.2004.02.001</a>.</p>
</div>
<div id="ref-EskelandRing1BCompactsChromatin2010">
<p>Eskeland, Ragnhild, Martin Leeb, Graeme R. Grimes, Cl’emence Kress, Shelagh Boyle, Duncan Sproul, Nick Gilbert, et al. 2010. “Ring1B Compacts Chromatin Structure and Represses Gene Expression Independent of Histone Ubiquitination.” <em>Molecular Cell</em> 38 (3): 452–64. doi:<a href="https://doi.org/10.1016/j.molcel.2010.02.032">10.1016/j.molcel.2010.02.032</a>.</p>
</div>
<div id="ref-PetersLossSuv39hHistone2001">
<p>Peters, Antoine H. F. M., D’onal O’Carroll, Harry Scherthan, Karl Mechtler, Stephan Sauer, Christian Schöfer, Klara Weipoltshammer, et al. 2001. “Loss of the Suv39h Histone Methyltransferases Impairs Mammalian Heterochromatin and Genome Stability.” <em>Cell</em> 107 (3): 323–37. doi:<a href="https://doi.org/10.1016/S0092-8674(01)00542-6">10.1016/S0092-8674(01)00542-6</a>.</p>
</div>
<div id="ref-OCarrollIsolationCharacterizationofSuv39h22000">
<p>O’Carroll, D’onal, Harry Scherthan, Antoine H. F. M. Peters, Susanne Opravil, Andrew R. Haynes, Götz Laible, Stephen Rea, et al. 2000. “Isolation and Characterization ofSuv39h2, a Second Histone H3 Methyltransferase Gene That Displays Testis-Specific Expression.” <em>Molecular and Cellular Biology</em> 20 (24): 9423–33. doi:<a href="https://doi.org/10.1128/MCB.20.24.9423-9433.2000">10.1128/MCB.20.24.9423-9433.2000</a>.</p>
</div>
<div id="ref-SchultzSETDB1novelKAP1associated2002a">
<p>Schultz, David C., Kasirajan Ayyanathan, Dmitri Negorev, Gerd G. Maul, and Frank J. Rauscher. 2002. “SETDB1: A Novel KAP-1-Associated Histone H3, Lysine 9-Specific Methyltransferase That Contributes to Hp1-Mediated Silencing of Euchromatic Genes by KRAB Zinc-Finger Proteins.” <em>Genes &amp; Development</em> 16 (8): 919–32.</p>
</div>
<div id="ref-WangPurificationbiochemicalheterogeneity1996">
<p>Wang, Weidong, Jacques Côt’e, Yutong Xue, Sharleen Zhou, Paul A. Khavari, Stephen R. Biggar, Christian Muchardt, Ganjam V. Kalpana, Stephen P. Goff, and Moshe Yaniv. 1996. “Purification and Biochemical Heterogeneity of the Mammalian SWI‐SNF Complex.” <em>The EMBO Journal</em> 15 (19): 5370–82.</p>
</div>
<div id="ref-StaahlKineticAnalysisnpBAF2013">
<p>Staahl, B. T., J. Tang, W. Wu, A. Sun, A. D. Gitler, A. S. Yoo, and G. R. Crabtree. 2013. “Kinetic Analysis of npBAF to nBAF Switching Reveals Exchange of Ss18 with CREST and Integration with Neural Developmental Pathways.” <em>Journal of Neuroscience</em> 33 (25): 10348–61. doi:<a href="https://doi.org/10.1523/JNEUROSCI.1258-13.2013">10.1523/JNEUROSCI.1258-13.2013</a>.</p>
</div>
<div id="ref-ThomasCoevolutionretroelementstandem2011">
<p>Thomas, J. H., and S. Schneider. 2011. “Coevolution of Retroelements and Tandem Zinc Finger Genes.” <em>Genome Research</em> 21 (11): 1800–1812. doi:<a href="https://doi.org/10.1101/gr.121749.111">10.1101/gr.121749.111</a>.</p>
</div>
<div id="ref-ImbeaultKRABzincfingerproteins2017">
<p>Imbeault, Michaël, Pierre-Yves Helleboid, and Didier Trono. 2017. “KRAB Zinc-Finger Proteins Contribute to the Evolution of Gene Regulatory Networks.” <em>Nature</em> 543 (7646): 550. doi:<a href="https://doi.org/10.1038/nature21683">10.1038/nature21683</a>.</p>
</div>
<div id="ref-ThompsonLongTerminalRepeats2016">
<p>Thompson, Peter J., Todd S. Macfarlan, and Matthew C. Lorincz. 2016. “Long Terminal Repeats: From Parasitic Elements to Building Blocks of the Transcriptional Regulatory Repertoire.” <em>Molecular Cell</em> 62 (5): 766–76. doi:<a href="https://doi.org/10.1016/j.molcel.2016.03.029">10.1016/j.molcel.2016.03.029</a>.</p>
</div>
<div id="ref-BradyIntegrationtargetsite2009">
<p>Brady, Troy, Young Nam Lee, Keshet Ronen, Nirav Malani, Charles C. Berry, Paul D. Bieniasz, and Frederic D. Bushman. 2009. “Integration Target Site Selection by a Resurrected Human Endogenous Retrovirus.” <em>Genes &amp; Development</em> 23 (5): 633–42.</p>
</div>
<div id="ref-ChuongEndogenousretrovirusesfunction2013">
<p>Chuong, Edward B., MA Karim Rumi, Michael J. Soares, and Julie C. Baker. 2013. “Endogenous Retroviruses Function as Species-Specific Enhancer Elements in the Placenta.” <em>Nature Genetics</em> 45 (3): 325–29.</p>
</div>
<div id="ref-Cordauximpactretrotransposonshuman2009">
<p>Cordaux, Richard, and Mark A. Batzer. 2009. “The Impact of Retrotransposons on Human Genome Evolution.” <em>Nature Reviews Genetics</em> 10 (10): 691–703.</p>
</div>
<div id="ref-SundaramWidespreadcontributiontransposable2014">
<p>Sundaram, Vasavi, Yong Cheng, Zhihai Ma, Daofeng Li, Xiaoyun Xing, Peter Edge, Michael P. Snyder, and Ting Wang. 2014. “Widespread Contribution of Transposable Elements to the Innovation of Gene Regulatory Networks.” <em>Genome Research</em> 24 (12): 1963–76.</p>
</div>
<div id="ref-ZimmerlyGroupIIintron1995">
<p>Zimmerly, S., H. Guo, P. S. Perlman, and A. M. Lambowitz. 1995. “Group II Intron Mobility Occurs by Target DNA-Primed Reverse Transcription.” <em>Cell</em> 82 (4): 545–54.</p>
</div>
<div id="ref-DewannieuxLINEmediatedretrotranspositionmarked2003">
<p>Dewannieux, Marie, C’ecile Esnault, and Thierry Heidmann. 2003. “LINE-Mediated Retrotransposition of Marked Alu Sequences.” <em>Nature Genetics</em> 35 (1): 41–48.</p>
</div>
<div id="ref-HancksActivehumanretrotransposons2012">
<p>Hancks, Dustin C., and Haig H. Kazazian. 2012. “Active Human Retrotransposons: Variation and Disease.” <em>Current Opinion in Genetics &amp; Development</em> 22 (3): 191–203.</p>
</div>
<div id="ref-KajikawaLINEsmobilizeSINEs2002">
<p>Kajikawa, Masaki, and Norihiro Okada. 2002. “LINEs Mobilize SINEs in the Eel Through a Shared 3’ Sequence.” <em>Cell</em> 111 (3): 433–44.</p>
</div>
<div id="ref-MaksakovaDistinctrolesKAP12013">
<p>Maksakova, IrinaA, PeterJ Thompson, Preeti Goyal, StevenJM Jones, PrimB Singh, MohammadM Karimi, and MatthewC Lorincz. 2013. “Distinct Roles of Kap1, Hp1 and G9a/GLP in Silencing of the Two-Cell-Specific Retrotransposon MERVL in Mouse ES Cells.” <em>Epigenetics &amp; Chromatin</em> 6 (1): 1–16. doi:<a href="https://doi.org/10.1186/1756-8935-6-15">10.1186/1756-8935-6-15</a>.</p>
</div>
<div id="ref-RowenovoDNAmethylation2013">
<p>Rowe, H. M., M. Friedli, S. Offner, S. Verp, D. Mesnard, J. Marquis, T. Aktas, and D. Trono. 2013. “De Novo DNA Methylation of Endogenous Retroviruses Is Shaped by KRAB-ZFPs/Kap1 and ESET.” <em>Development</em> 140 (3): 519–29. doi:<a href="https://doi.org/10.1242/dev.087585">10.1242/dev.087585</a>.</p>
</div>
<div id="ref-Friedlidevelopmentalcontroltransposable2015">
<p>Friedli, Marc, and Didier Trono. 2015. “The Developmental Control of Transposable Elements and the Evolution of Higher Species.” <em>Annual Review of Cell and Developmental Biology</em> 31: 429–51.</p>
</div>
<div id="ref-EccoTransposableElementsTheir2016">
<p>Ecco, Gabriela, Marco Cassano, Annamaria Kauzlaric, Julien Duc, Andrea Coluccio, Sandra Offner, Michaël Imbeault, Helen M. Rowe, Priscilla Turelli, and Didier Trono. 2016. “Transposable Elements and Their KRAB-ZFP Controllers Regulate Gene Expression in Adult Tissues.” <em>Developmental Cell</em> 36 (6): 611–23. doi:<a href="https://doi.org/10.1016/j.devcel.2016.02.024">10.1016/j.devcel.2016.02.024</a>.</p>
</div>
<div id="ref-VoonNewplayersheterochromatin2016">
<p>Voon, Hsiao PJ, and Lee H. Wong. 2016. “New Players in Heterochromatin Silencing: Histone Variant H3. 3 and the ATRX/DAXX Chaperone.” <em>Nucleic Acids Research</em> 44 (4): 1496–1501.</p>
</div>
<div id="ref-JangHistoneH3maintains2015">
<p>Jang, Chuan-Wei, Yoichiro Shibata, Joshua Starmer, Della Yee, and Terry Magnuson. 2015. “Histone H3. 3 Maintains Genome Integrity During Mammalian Development.” <em>Genes &amp; Development</em> 29 (13): 1377–92.</p>
</div>
<div id="ref-UdugamaHistonevariantH32015">
<p>Udugama, Maheshi, Fiona T. M. Chang, F. Lyn Chan, Michelle C. Tang, Hilda A. Pickett, James D. R. McGhie, Lynne Mayne, Philippe Collas, Jeffrey R. Mann, and Lee H. Wong. 2015. “Histone Variant H3. 3 Provides the Heterochromatic H3 Lysine 9 Tri-Methylation Mark at Telomeres.” <em>Nucleic Acids Research</em> 43 (21): 10227–37.</p>
</div>
<div id="ref-IvanovPHDDomainMediatedE32007">
<p>Ivanov, Alexey V., Hongzhuang Peng, Vyacheslav Yurchenko, Kyoko L. Yap, Dmitri G. Negorev, David C. Schultz, Elyse Psulkowski, et al. 2007. “PHD Domain-Mediated E3 Ligase Activity Directs Intramolecular Sumoylation of an Adjacent Bromodomain Required for Gene Silencing.” <em>Molecular Cell</em> 28 (5): 823–37. doi:<a href="https://doi.org/10.1016/j.molcel.2007.11.012">10.1016/j.molcel.2007.11.012</a>.</p>
</div>
<div id="ref-AyyanathanRegulatedrecruitmentHP12003a">
<p>Ayyanathan, K. 2003. “Regulated Recruitment of Hp1 to a Euchromatic Gene Induces Mitotically Heritable, Epigenetic Gene Silencing: A Mammalian Cell Culture Model of Gene Variegation.” <em>Genes &amp; Development</em> 17 (15): 1855–69. doi:<a href="https://doi.org/10.1101/gad.1102803">10.1101/gad.1102803</a>.</p>
</div>
<div id="ref-RottachmultidomainproteinNp952010">
<p>Rottach, Andrea, Carina Frauer, Garwin Pichler, Ian Marc Bonapace, Fabio Spada, and Heinrich Leonhardt. 2010. “The Multi-Domain Protein Np95 Connects DNA Methylation and Histone Modification.” <em>Nucleic Acids Research</em> 38 (6): 1796–1804. <a href="http://nar.oxfordjournals.org/content/38/6/1796.short" class="uri">http://nar.oxfordjournals.org/content/38/6/1796.short</a>.</p>
</div>
<div id="ref-SmallwoodFunctionalcooperationHP12007">
<p>Smallwood, A., P.-O. Esteve, S. Pradhan, and M. Carey. 2007. “Functional Cooperation Between Hp1 and Dnmt1 Mediates Gene Silencing.” <em>Genes &amp; Development</em> 21 (10): 1169–78. doi:<a href="https://doi.org/10.1101/gad.1536807">10.1101/gad.1536807</a>.</p>
</div>
<div id="ref-VoonMaintainingmemorysilencing2016">
<p>Voon, Hsiao PJ, and Richard J. Gibbons. 2016. “Maintaining Memory of Silencing at Imprinted Differentially Methylated Regions.” <em>Cellular and Molecular Life Sciences</em> 73 (9): 1871–9.</p>
</div>
<div id="ref-Zuozincfingerprotein2011">
<p>Zuo, Xiaopan, Jipo Sheng, Ho-Tak Lau, Carol M. McDonald, Monica Andrade, Dana E. Cullen, Fong T. Bell, Michelina Iacovino, Michael Kyba, and Guoliang Xu. 2011. “The Zinc Finger Protein Zfp57 Requires Its Cofactor to Recruit DNA Methyltransferases and Maintains the DNA Methylation Imprint in Embryonic Stem Cells via Its Transcriptional Repression Domain.” <em>Journal of Biological Chemistry</em>, jbc. M111. 322644.</p>
</div>
<div id="ref-VogelHumanheterochromatinproteins2006">
<p>Vogel, M. J., L. Guelen, E. de Wit, D. P. Hupkes, M. Loden, W. Talhout, M. Feenstra, B. Abbas, A.-K. Classen, and B. van Steensel. 2006. “Human Heterochromatin Proteins Form Large Domains Containing KRAB-ZNF Genes.” <em>Genome Research</em> 16 (12): 1493–1504. doi:<a href="https://doi.org/10.1101/gr.5391806">10.1101/gr.5391806</a>.</p>
</div>
<div id="ref-GronerKRABzincfinger2010">
<p>Groner, Anna C., Sylvain Meylan, Angela Ciuffi, Nadine Zangger, Giovanna Ambrosini, Nicolas D’enervaud, Philipp Bucher, and Didier Trono. 2010. “KRAB–Zinc Finger Proteins and Kap1 Can Mediate Long-Range Transcriptional Repression Through Heterochromatin Spreading.” <em>PLoS Genetics</em> 6 (3): e1000869.</p>
</div>
<div id="ref-CammasDynamicselectiveinteractions2007">
<p>Cammas, Florence, Agnes Janoshazi, Thierry Lerouge, and R’egine Losson. 2007. “Dynamic and Selective Interactions of the Transcriptional Corepressor TIF1β with the Heterochromatin Protein Hp1 Isotypes During Cell Differentiation.” <em>Differentiation</em> 75 (7): 627–37.</p>
</div>
<div id="ref-Hiragami-HamadaDynamicflexibleH3K9me32016">
<p>Hiragami-Hamada, Kyoko, Szabolcs Soeroes, Miroslav Nikolov, Bryan Wilkins, Sarah Kreuz, Carol Chen, Inti A. De La Rosa-Vel’azquez, et al. 2016. “Dynamic and Flexible H3K9me3 Bridging via HP1β Dimerization Establishes a Plastic State of Condensed Chromatin.” <em>Nature Communications</em> 7 (April): 11310. doi:<a href="https://doi.org/10.1038/ncomms11310">10.1038/ncomms11310</a>.</p>
</div>
<div id="ref-KourmouliEpigeneticregulationmammalian2005">
<p>Kourmouli, Niki, Yuh-man Sun, Sjaak van der Sar, Prim B. Singh, and Jeremy P. Brown. 2005. “Epigenetic Regulation of Mammalian Pericentric Heterochromatin in Vivo by Hp1.” <em>Biochemical and Biophysical Research Communications</em> 337 (3): 901–7. doi:<a href="https://doi.org/10.1016/j.bbrc.2005.09.132">10.1016/j.bbrc.2005.09.132</a>.</p>
</div>
<div id="ref-LueffectH3K79dimethylation2008">
<p>Lu, Xu, Matthew D. Simon, Jayanth V. Chodaparambil, Jeffrey C. Hansen, Kevan M. Shokat, and Karolin Luger. 2008. “The Effect of H3k79 Dimethylation and H4k20 Trimethylation on Nucleosome and Chromatin Structure.” <em>Nature Structural &amp; Molecular Biology</em> 15 (10): 1122–4. doi:<a href="https://doi.org/10.1038/nsmb.1489">10.1038/nsmb.1489</a>.</p>
</div>
<div id="ref-NelsonMappingH4K20me3chromatin2016">
<p>Nelson, David M., Farah Jaber-Hijazi, John J. Cole, Neil A. Robertson, Jeffrey S. Pawlikowski, Kevin T. Norris, Steven W. Criscione, et al. 2016. “Mapping H4K20me3 onto the Chromatin Landscape of Senescent Cells Indicates a Function in Control of Cell Senescence and Tumor Suppression Through Preservation of Genetic and Epigenetic Stability.” <em>Genome Biology</em> 17 (July). doi:<a href="https://doi.org/10.1186/s13059-016-1017-x">10.1186/s13059-016-1017-x</a>.</p>
</div>
<div id="ref-BrownhumanXISTgene1992">
<p>Brown, Carolyn J., Brian D. Hendrich, Jim L. Rupert, Ronald G. Lafreniere, Yigong Xing, Jeanne Lawrence, and Huntington F. Willard. 1992. “The Human XIST Gene: Analysis of a 17 Kb Inactive X-Specific RNA That Contains Conserved Repeats and Is Highly Localized Within the Nucleus.” <em>Cell</em> 71 (3): 527–42.</p>
</div>
<div id="ref-WutzGenesilencingXchromosome2011">
<p>Wutz, Anton. 2011. “Gene Silencing in X-Chromosome Inactivation: Advances in Understanding Facultative Heterochromatin Formation.” <em>Nature Reviews Genetics</em> 12 (8): 542.</p>
</div>
<div id="ref-FiorenzanoUltraconservedElementContaining2018">
<p>Fiorenzano, Alessandro, Emilia Pascale, Miriam Gagliardi, Sara Terreri, Mariarosaria Papa, Gennaro Andolfi, Marco Galasso, et al. 2018. “An Ultraconserved Element Containing lncRNA Preserves Transcriptional Dynamics and Maintains ESC Self-Renewal.” <em>Stem Cell Reports</em> 10 (3): 1102–14. doi:<a href="https://doi.org/10.1016/j.stemcr.2018.01.014">10.1016/j.stemcr.2018.01.014</a>.</p>
</div>
<div id="ref-Ashwal-FlusscircRNABiogenesisCompetes2014">
<p>Ashwal-Fluss, Reut, Markus Meyer, Nagarjuna Reddy Pamudurti, Andranik Ivanov, Osnat Bartok, Mor Hanan, Naveh Evantal, Sebastian Memczak, Nikolaus Rajewsky, and Sebastian Kadener. 2014. “circRNA Biogenesis Competes with Pre-mRNA Splicing.” <em>Molecular Cell</em> 56 (1): 55–66. doi:<a href="https://doi.org/10.1016/j.molcel.2014.08.019">10.1016/j.molcel.2014.08.019</a>.</p>
</div>
<div id="ref-GonzalezlncRNAregulatesalternative2015">
<p>Gonzalez, Inma, Roberto Munita, Eneritz Agirre, Travis A. Dittmer, Katia Gysling, Tom Misteli, and Reini F. Luco. 2015. “A lncRNA Regulates Alternative Splicing via Establishment of a Splicing-Specific Chromatin Signature.” <em>Nature Structural &amp; Molecular Biology</em> 22 (5): 370–76. doi:<a href="https://doi.org/10.1038/nsmb.3005">10.1038/nsmb.3005</a>.</p>
</div>
<div id="ref-MemczakCircularRNAsare2013">
<p>Memczak, Sebastian, Marvin Jens, Antigoni Elefsinioti, Francesca Torti, Janna Krueger, Agnieszka Rybak, Luisa Maier, et al. 2013. “Circular RNAs Are a Large Class of Animal RNAs with Regulatory Potency.” <em>Nature</em> 495 (7441): 333–38. doi:<a href="https://doi.org/10.1038/nature11928">10.1038/nature11928</a>.</p>
</div>
<div id="ref-WuUnusualProcessingGenerates2016">
<p>Wu, Huang, Qing-Fei Yin, Zheng Luo, Run-Wen Yao, Chuan-Chuan Zheng, Jun Zhang, Jian-Feng Xiang, Li Yang, and Ling-Ling Chen. 2016. “Unusual Processing Generates SPA LncRNAs That Sequester Multiple RNA Binding Proteins.” <em>Molecular Cell</em> 64 (3): 534–48. doi:<a href="https://doi.org/10.1016/j.molcel.2016.10.007">10.1016/j.molcel.2016.10.007</a>.</p>
</div>
<div id="ref-JoungGenomescaleactivationscreen2017">
<p>Joung, Julia, Jesse M. Engreitz, Silvana Konermann, Omar O. Abudayyeh, Vanessa K. Verdine, Francois Aguet, Jonathan S. Gootenberg, et al. 2017. “Genome-Scale Activation Screen Identifies a lncRNA Locus Regulating a Gene Neighbourhood.” <em>Nature</em> 548 (7667): 343–46. doi:<a href="https://doi.org/10.1038/nature23451">10.1038/nature23451</a>.</p>
</div>
<div id="ref-Mancini-DiNardoElongationKcnq1ot1transcript2006">
<p>Mancini-DiNardo, Debora, Scott JS Steele, John M. Levorse, Robert S. Ingram, and Shirley M. Tilghman. 2006. “Elongation of the Kcnq1ot1 Transcript Is Required for Genomic Imprinting of Neighboring Genes.” <em>Genes &amp; Development</em> 20 (10): 1268–82.</p>
</div>
<div id="ref-SleutelsnoncodingAirRNA2002">
<p>Sleutels, Frank, Ronald Zwart, and Denise P. Barlow. 2002. “The Non-Coding <em>Air</em> RNA Is Required for Silencing Autosomal Imprinted Genes.” <em>Nature</em> 415 (6873): 810–13. doi:<a href="https://doi.org/10.1038/415810a">10.1038/415810a</a>.</p>
</div>
<div id="ref-WilliamsonUncouplingAntisenseMediatedSilencing2011a">
<p>Williamson, Christine M., Simon T. Ball, Claire Dawson, Stuti Mehta, Colin V. Beechey, Martin Fray, Lydia Teboul, T. Neil Dear, Gavin Kelsey, and Jo Peters. 2011. “Uncoupling Antisense-Mediated Silencing and DNA Methylation in the Imprinted Gnas Cluster.” <em>PLOS Genetics</em> 7 (3): e1001347. doi:<a href="https://doi.org/10.1371/journal.pgen.1001347">10.1371/journal.pgen.1001347</a>.</p>
</div>
<div id="ref-SantoroIntergenictranscriptsoriginating2010">
<p>Santoro, Raffaella, Kerstin-Maike Schmitz, Juan Sandoval, and Ingrid Grummt. 2010. “Intergenic Transcripts Originating from a Subclass of Ribosomal DNA Repeats Silence Ribosomal RNA Genes in Trans.” <em>EMBO Reports</em> 11 (1): 52–58.</p>
</div>
<div id="ref-SchmitzInteractionnoncodingRNA2010">
<p>Schmitz, Kerstin-Maike, Christine Mayer, Anna Postepska, and Ingrid Grummt. 2010. “Interaction of Noncoding RNA with the rDNA Promoter Mediates Recruitment of DNMT3b and Silencing of rRNA Genes.” <em>Genes &amp; Development</em> 24 (20): 2264–9.</p>
</div>
<div id="ref-BierhoffQuiescenceInducedLncRNAsTrigger2014">
<p>Bierhoff, Holger, Marcel Andre Dammert, David Brocks, Silvia Dambacher, Gunnar Schotta, and Ingrid Grummt. 2014. “Quiescence-Induced LncRNAs Trigger H4k20 Trimethylation and Transcriptional Silencing.” <em>Molecular Cell</em> 54 (4): 675–82. doi:<a href="https://doi.org/10.1016/j.molcel.2014.03.032">10.1016/j.molcel.2014.03.032</a>.</p>
</div>
<div id="ref-SlotkinTransposableelementsepigenetic2007">
<p>Slotkin, R. Keith, and Robert Martienssen. 2007. “Transposable Elements and the Epigenetic Regulation of the Genome.” <em>Nature Reviews Genetics</em> 8 (4): 272.</p>
</div>
<div id="ref-YangL1retrotranspositionsuppressed2006">
<p>Yang, Nuo, and Haig H. Kazazian Jr. 2006. “L1 Retrotransposition Is Suppressed by Endogenously Encoded Small Interfering RNAs in Human Cultured Cells.” <em>Nature Structural and Molecular Biology</em> 13 (9): 763.</p>
</div>
<div id="ref-WongCentromereRNAkey2007">
<p>Wong, Lee H., Kate H. Brettingham-Moore, Lyn Chan, Julie M. Quach, Melisssa A. Anderson, Emma L. Northrop, Ross Hannan, Richard Saffery, Margaret L. Shaw, and Evan Williams. 2007. “Centromere RNA Is a Key Component for the Assembly of Nucleoproteins at the Nucleolus and Centromere.” <em>Genome Research</em> 17 (8): 000–000.</p>
</div>
<div id="ref-MaisonHigherorderstructurepericentric2002">
<p>Maison, Christ‘ele, Delphine Bailly, Antoine HFM Peters, Jean-Pierre Quivy, Dani‘ele Roche, Angela Taddei, Monika Lachner, Thomas Jenuwein, and Genevi‘eve Almouzni. 2002. “Higher-Order Structure in Pericentric Heterochromatin Involves a Distinct Pattern of Histone Modification and an RNA Component.” <em>Nature Genetics</em> 30 (3): 329.</p>
</div>
<div id="ref-NergadzeCpGislandpromotersdrive2009">
<p>Nergadze, Solomon G., Benjamin O. Farnung, Harry Wischnewski, Lela Khoriauli, Valerio Vitelli, Raghav Chawla, Elena Giulotto, and Claus M. Azzalin. 2009. “CpG-Island Promoters Drive Transcription of Human Telomeres.” <em>Rna</em>.</p>
</div>
<div id="ref-DevanapallyDoublestrandedRNAmade2015">
<p>Devanapally, Sindhuja, Snusha Ravikumar, and Antony M. Jose. 2015. “Double-Stranded RNA Made in C. Elegans Neurons Can Enter the Germline and Cause Transgenerational Gene Silencing.” <em>Proceedings of the National Academy of Sciences</em> 112 (7): 2133–8.</p>
</div>
<div id="ref-RodgersPaternalStressExposure2013">
<p>Rodgers, A. B., C. P. Morgan, S. L. Bronson, S. Revello, and T. L. Bale. 2013. “Paternal Stress Exposure Alters Sperm MicroRNA Content and Reprograms Offspring HPA Stress Axis Regulation.” <em>Journal of Neuroscience</em> 33 (21): 9003–12. doi:<a href="https://doi.org/10.1523/JNEUROSCI.0914-13.2013">10.1523/JNEUROSCI.0914-13.2013</a>.</p>
</div>
<div id="ref-RodgersTransgenerationalepigeneticprogramming2015">
<p>Rodgers, Ali B., Christopher P. Morgan, N. Adrian Leu, and Tracy L. Bale. 2015. “Transgenerational Epigenetic Programming via Sperm microRNA Recapitulates Effects of Paternal Stress.” <em>Proceedings of the National Academy of Sciences</em> 112 (44): 13699–13704.</p>
</div>
<div id="ref-ShortExercisealtersmouse2017a">
<p>Short, A. K., S. Yeshurun, R. Powell, V. M. Perreau, A. Fox, J. H. Kim, T. Y. Pang, and A. J. Hannan. 2017. “Exercise Alters Mouse Sperm Small Noncoding RNAs and Induces a Transgenerational Modification of Male Offspring Conditioned Fear and Anxiety.” <em>Translational Psychiatry</em> 7 (5): e1114.</p>
</div>
<div id="ref-KennedyGerosciencelinkingaging2014">
<p>Kennedy, Brian K., Shelley L. Berger, Anne Brunet, Judith Campisi, Ana Maria Cuervo, Elissa S. Epel, Claudio Franceschi, Gordon J. Lithgow, Richard I. Morimoto, and Jeffrey E. Pessin. 2014. “Geroscience: Linking Aging to Chronic Disease.” <em>Cell</em> 159 (4): 709–13.</p>
</div>
<div id="ref-MoskalevGeneticsepigeneticsaging2014">
<p>Moskalev, Alexey, Alexander Aliper, Zeljka Smit-McBride, Anton Buzdin, and Alex Zhavoronkov. 2014. “Genetics and Epigenetics of Aging and Longevity.” <em>Cell Cycle</em> 13 (7): 1063–77.</p>
</div>
<div id="ref-DiLeonardoDNAdamagetriggers1994">
<p>Di Leonardo, Aldo, Steven P. Linke, Kris Clarkin, and Geoffrey M. Wahl. 1994. “DNA Damage Triggers a Prolonged P53-Dependent G1 Arrest and Long-Term Induction of Cip1 in Normal Human Fibroblasts.” <em>Genes &amp; Development</em> 8 (21): 2540–51.</p>
</div>
<div id="ref-LinPrematuresenescenceinvolving1998">
<p>Lin, Athena W., Marta Barradas, James C. Stone, Linda van Aelst, Manuel Serrano, and Scott W. Lowe. 1998. “Premature Senescence Involving P53 and P16 Is Activated in Response to Constitutive MEK/MAPK Mitogenic Signaling.” <em>Genes &amp; Development</em> 12 (19): 3008–19.</p>
</div>
<div id="ref-ChandraGlobalReorganizationNuclear2015">
<p>Chandra, Tamir, Philip Andrew Ewels, Stefan Schoenfelder, Mayra Furlan-Magaril, Steven William Wingett, Kristina Kirschner, Jean-Yves Thuret, Simon Andrews, Peter Fraser, and Wolf Reik. 2015. “Global Reorganization of the Nuclear Landscape in Senescent Cells.” <em>Cell Reports</em> 10 (4): 471–83. doi:<a href="https://doi.org/10.1016/j.celrep.2014.12.055">10.1016/j.celrep.2014.12.055</a>.</p>
</div>
<div id="ref-Villeponteauheterochromatinlossmodel1997">
<p>Villeponteau, Bryant. 1997. “The Heterochromatin Loss Model of Aging.” <em>Experimental Gerontology</em> 32 (4-5): 383–94.</p>
</div>
<div id="ref-PegoraroAgeingrelatedchromatindefects2009">
<p>Pegoraro, Gianluca, Nard Kubben, Ute Wickert, Heike Göhler, Katrin Hoffmann, and Tom Misteli. 2009. “Ageing-Related Chromatin Defects Through Loss of the NURD Complex.” <em>Nature Cell Biology</em> 11 (10): 1261–7. doi:<a href="https://doi.org/10.1038/ncb1971">10.1038/ncb1971</a>.</p>
</div>
<div id="ref-DjeghloulAgeAssociatedDecreaseHistone2016">
<p>Djeghloul, Dounia, Klaudia Kuranda, Isabelle Kuzniak, Daniela Barbieri, Irina Naguibneva, Caroline Choisy, Jean-Christophe Bories, et al. 2016. “Age-Associated Decrease of the Histone Methyltransferase Suv39h1 in HSC Perturbs Heterochromatin and B Lymphoid Differentiation.” <em>Stem Cell Reports</em> 6 (6): 970–84. doi:<a href="https://doi.org/10.1016/j.stemcr.2016.05.007">10.1016/j.stemcr.2016.05.007</a>.</p>
</div>
<div id="ref-WoodChromatinremodelingaging2010">
<p>Wood, Jason G., Sara Hillenmeyer, Charles Lawrence, Chengyi Chang, Suzanne Hosier, Will Lightfoot, Eric Mukherjee, Nan Jiang, Christoph Schorl, and Alexander S. Brodsky. 2010. “Chromatin Remodeling in the Aging Genome of Drosophila.” <em>Aging Cell</em> 9 (6): 971–78.</p>
</div>
<div id="ref-JeonEffectheterochromatinstability2018">
<p>Jeon, Ho-Jun, Young-Shin Kim, Joong-Gook Kim, Kyu Heo, Jung-Hoon Pyo, Masamitsu Yamaguchi, Joung-Sun Park, and Mi-Ae Yoo. 2018. “Effect of Heterochromatin Stability on Intestinal Stem Cell Aging in Drosophila.” <em>Mechanisms of Ageing and Development</em> 173 (July): 50–60. doi:<a href="https://doi.org/10.1016/j.mad.2018.04.001">10.1016/j.mad.2018.04.001</a>.</p>
</div>
<div id="ref-RyuTranscriptionalrepressionrepeatderived2011">
<p>Ryu, Soo Hee, Keunsoo Kang, Taekyung Yoo, Cheol O. Joe, and Jae Hoon Chung. 2011. “Transcriptional Repression of Repeat-Derived Transcripts Correlates with Histone Hypoacetylation at Repetitive DNA Elements in Aged Mice Brain.” <em>Experimental Gerontology</em> 46 (10): 811–18. doi:<a href="https://doi.org/10.1016/j.exger.2011.07.001">10.1016/j.exger.2011.07.001</a>.</p>
</div>
<div id="ref-LarsonHeterochromatinformationpromotes2012">
<p>Larson, Kimberly, Shian-Jang Yan, Amy Tsurumi, Jacqueline Liu, Jun Zhou, Kriti Gaur, Dongdong Guo, Thomas H. Eickbush, and Willis X. Li. 2012. “Heterochromatin Formation Promotes Longevity and Represses Ribosomal RNA Synthesis.” <em>PLoS Genet</em> 8 (1): e1002473.</p>
</div>
<div id="ref-SmealLosstranscriptionalsilencing1996">
<p>Smeal, Tod, James Claus, Brian Kennedy, Francesca Cole, and Leonard Guarente. 1996. “Loss of Transcriptional Silencing Causes Sterility in Old Mother Cells of S. Cerevisiae.” <em>Cell</em> 84 (4): 633–42.</p>
</div>
<div id="ref-Sousa-VictorPiwiRequiredLimit2017">
<p>Sousa-Victor, Pedro, Arshad Ayyaz, Rippei Hayashi, Yanyan Qi, David T. Madden, Victoria V. Lunyak, and Heinrich Jasper. 2017. “Piwi Is Required to Limit Exhaustion of Aging Somatic Stem Cells.” <em>Cell Reports</em> 20 (11): 2527–37.</p>
</div>
<div id="ref-ZhangWernersyndromestem2015">
<p>Zhang, Weiqi, Jingyi Li, Keiichiro Suzuki, Jing Qu, Ping Wang, Junzhi Zhou, Xiaomeng Liu, et al. 2015. “A Werner Syndrome Stem Cell Model Unveils Heterochromatin Alterations as a Driver of Human Aging.” <em>Science</em> 348 (6239): 1160–3. doi:<a href="https://doi.org/10.1126/science.aaa1356">10.1126/science.aaa1356</a>.</p>
</div>
<div id="ref-DeCeccoGenomesreplicativelysenescent2013">
<p>De Cecco, Marco, Steven W. Criscione, Edward J. Peckham, Sara Hillenmeyer, Eliza A. Hamm, Jayameenakshi Manivannan, Abigail L. Peterson, Jill A. Kreiling, Nicola Neretti, and John M. Sedivy. 2013. “Genomes of Replicatively Senescent Cells Undergo Global Epigenetic Changes Leading to Gene Silencing and Activation of Transposable Elements.” <em>Aging Cell</em> 12 (2): 247–56. doi:<a href="https://doi.org/10.1111/acel.12047">10.1111/acel.12047</a>.</p>
</div>
<div id="ref-LiuRecapitulationprematureageing2011">
<p>Liu, Guang-Hui, Basam Z. Barkho, Sergio Ruiz, Dinh Diep, Jing Qu, Sheng-Lian Yang, Athanasia D. Panopoulos, Keiichiro Suzuki, Leo Kurian, and Christopher Walsh. 2011. “Recapitulation of Premature Ageing with iPSCs from Hutchinson–Gilford Progeria Syndrome.” <em>Nature</em> 472 (7342): 221.</p>
</div>
<div id="ref-MillerHumaniPSCbasedmodeling2013">
<p>Miller, Justine D., Yosif M. Ganat, Sarah Kishinevsky, Robert L. Bowman, Becky Liu, Edmund Y. Tu, Pankaj K. Mandal, Elsa Vera, Jae-won Shim, and Sonja Kriks. 2013. “Human iPSC-Based Modeling of Late-Onset Disease via Progerin-Induced Aging.” <em>Cell Stem Cell</em> 13 (6): 691–705.</p>
</div>
<div id="ref-ShumakerMutantnuclearlamin2006">
<p>Shumaker, Dale K., Thomas Dechat, Alexander Kohlmaier, Stephen A. Adam, Matthew R. Bozovsky, Michael R. Erdos, Maria Eriksson, Anne E. Goldman, Satya Khuon, and Francis S. Collins. 2006. “Mutant Nuclear Lamin A Leads to Progressive Alterations of Epigenetic Control in Premature Aging.” <em>Proceedings of the National Academy of Sciences</em> 103 (23): 8703–8.</p>
</div>
<div id="ref-PereiraHGPSrelatedpremature2008b">
<p>Pereira, Sandrine, Patrice Bourgeois, Claire Navarro, Vera Esteves-Vieira, Pierre Cau, Annachiara De Sandre-Giovannoli, and Nicolas L’evy. 2008. “HGPS and Related Premature Aging Disorders: From Genomic Identification to the First Therapeutic Approaches.” <em>Mechanisms of Ageing and Development</em>, DNA damage, repair, ageing and age-related disease, 129 (7): 449–59. doi:<a href="https://doi.org/10.1016/j.mad.2008.04.003">10.1016/j.mad.2008.04.003</a>.</p>
</div>
<div id="ref-OSullivanReducedhistonebiosynthesis2010">
<p>O’Sullivan, Roderick J, Stefan Kubicek, Stuart L Schreiber, and Jan Karlseder. 2010. “Reduced Histone Biosynthesis and Chromatin Changes Arising from a Damage Signal at Telomeres.” <em>Nature Structural &amp; Molecular Biology</em> 17 (10): 1218–25. doi:<a href="https://doi.org/10.1038/nsmb.1897">10.1038/nsmb.1897</a>.</p>
</div>
<div id="ref-WangIdentificationhistonemethylation2010">
<p>Wang, Chun Mei, Sau Na Tsai, Tai Wai Yew, Yiu Wa Kwan, and Sai Ming Ngai. 2010. “Identification of Histone Methylation Multiplicities Patterns in the Brain of Senescence-Accelerated Prone Mouse 8.” <em>Biogerontology</em> 11 (1): 87–102. doi:<a href="https://doi.org/10.1007/s10522-009-9231-5">10.1007/s10522-009-9231-5</a>.</p>
</div>
<div id="ref-NaritaRbmediatedheterochromatinformation2003">
<p>Narita, Masashi, Sabrina Nuñez, Edith Heard, Masako Narita, Athena W. Lin, Stephen A. Hearn, David L. Spector, Gregory J. Hannon, and Scott W. Lowe. 2003. “Rb-Mediated Heterochromatin Formation and Silencing of E2f Target Genes During Cellular Senescence.” <em>Cell</em> 113 (6): 703–16.</p>
</div>
<div id="ref-ShahLaminB1depletion2013">
<p>Shah, P. P., G. Donahue, G. L. Otte, B. C. Capell, D. M. Nelson, K. Cao, V. Aggarwala, et al. 2013. “Lamin B1 Depletion in Senescent Cells Triggers Large-Scale Changes in Gene Expression and the Chromatin Landscape.” <em>Genes &amp; Development</em> 27 (16): 1787–99. doi:<a href="https://doi.org/10.1101/gad.223834.113">10.1101/gad.223834.113</a>.</p>
</div>
<div id="ref-NiTwoSETdomain2012">
<p>Ni, Zhuoyu, Atsushi Ebata, Elham Alipanahiramandi, and Siu Sylvia Lee. 2012. “Two SET Domain Containing Genes Link Epigenetic Changes and Aging in Caenorhabditis Elegans.” <em>Aging Cell</em> 11 (2): 315–25.</p>
</div>
<div id="ref-JamiesonLossHP1causes2015">
<p>Jamieson, Kirsty, Elizabeth T. Wiles, Kevin J. McNaught, Simone Sidoli, Neena Leggett, Yanchun Shao, Benjamin A. Garcia, and Eric U. Selker. 2015. “Loss of Hp1 Causes Depletion of H3K27me3 from Facultative Heterochromatin and Gain of H3K27me2 at Constitutive Heterochromatin.” <em>Genome Research</em>, gr. 194555.115.</p>
</div>
<div id="ref-HannumGenomewidemethylationprofiles2013">
<p>Hannum, Gregory, Justin Guinney, Ling Zhao, Li Zhang, Guy Hughes, SriniVas Sadda, Brandy Klotzle, Marina Bibikova, Jian-Bing Fan, and Yuan Gao. 2013. “Genome-Wide Methylation Profiles Reveal Quantitative Views of Human Aging Rates.” <em>Molecular Cell</em> 49 (2): 359–67.</p>
</div>
<div id="ref-HorvathDNAmethylationage2013">
<p>Horvath, Steve. 2013. “DNA Methylation Age of Human Tissues and Cell Types.” <em>Genome Biology</em> 14 (10): 3156.</p>
</div>
<div id="ref-JintaridthDistinctivepatternsagedependent2010">
<p>Jintaridth, Pornrutsami, and Apiwat Mutirangura. 2010. “Distinctive Patterns of Age-Dependent Hypomethylation in Interspersed Repetitive Sequences.” <em>Physiological Genomics</em> 41 (2): 194–200.</p>
</div>
<div id="ref-HeynDistinctDNAmethylomes2012">
<p>Heyn, Holger, Ning Li, Humberto J. Ferreira, Sebastian Moran, David G. Pisano, Antonio Gomez, Javier Diez, Jose V. Sanchez-Mut, Fernando Setien, and F. Javier Carmona. 2012. “Distinct DNA Methylomes of Newborns and Centenarians.” <em>Proceedings of the National Academy of Sciences</em> 109 (26): 10522–7.</p>
</div>
<div id="ref-BollatiDeclinegenomicDNA2009">
<p>Bollati, Valentina, Joel Schwartz, Robert Wright, Augusto Litonjua, Letizia Tarantini, Helen Suh, David Sparrow, Pantel Vokonas, and Andrea Baccarelli. 2009. “Decline in Genomic DNA Methylation Through Aging in a Cohort of Elderly Subjects.” <em>Mechanisms of Ageing and Development</em> 130 (4): 234–39. doi:<a href="https://doi.org/10.1016/j.mad.2008.12.003">10.1016/j.mad.2008.12.003</a>.</p>
</div>
<div id="ref-WilsonDNAmethylationdecreases1983">
<p>Wilson, Vincent L., and Peter A. Jones. 1983. “DNA Methylation Decreases in Aging but Not in Immortal Cells.” <em>Science</em> 220 (4601): 1055–7.</p>
</div>
<div id="ref-JungAgingDNAmethylation2015">
<p>Jung, Marc, and Gerd P. Pfeifer. 2015. “Aging and DNA Methylation.” <em>BMC Biology</em> 13 (1): 7.</p>
</div>
<div id="ref-GowersAgerelatedlossCpG2011">
<p>Gowers, Isobel R., Kevin Walters, Endre Kiss-Toth, Robert C. Read, Gordon W. Duff, and Anthony G. Wilson. 2011. “Age-Related Loss of CpG Methylation in the Tumour Necrosis Factor Promoter.” <em>Cytokine</em> 56 (3): 792–97.</p>
</div>
<div id="ref-WeiHypomethylationIL17RCpromoter2012">
<p>Wei, Lai, Baoying Liu, Jingsheng Tuo, Defen Shen, Ping Chen, Zhiyu Li, Xunxian Liu, Jia Ni, Pradeep Dagur, and H. Nida Sen. 2012. “Hypomethylation of the Il17rc Promoter Associates with Age-Related Macular Degeneration.” <em>Cell Reports</em> 2 (5): 1151–8.</p>
</div>
<div id="ref-ZhangAgedependentDNAmethylation2002">
<p>Zhang, Zhiyong, Chun Deng, Qianjin Lu, and Bruce Richardson. 2002. “Age-Dependent DNA Methylation Changes in the ITGAL (CD11a) Promoter.” <em>Mechanisms of Ageing and Development</em> 123 (9): 1257–68.</p>
</div>
<div id="ref-RakyanHumanagingassociatedDNA2010">
<p>Rakyan, V. K., T. A. Down, S. Maslau, T. Andrew, T. P. Yang, H. Beyan, P. Whittaker, et al. 2010. “Human Aging-Associated DNA Hypermethylation Occurs Preferentially at Bivalent Chromatin Domains.” <em>Genome Research</em> 20 (4): 434–39. doi:<a href="https://doi.org/10.1101/gr.103101.109">10.1101/gr.103101.109</a>.</p>
</div>
<div id="ref-ZampieriReconfigurationDNAmethylation2015">
<p>Zampieri, Michele, Fabio Ciccarone, Roberta Calabrese, Claudio Franceschi, Alexander Bürkle, and Paola Caiafa. 2015. “Reconfiguration of DNA Methylation in Aging.” <em>Mechanisms of Ageing and Development</em>, Biomarkers of human ageing, 151 (November): 60–70. doi:<a href="https://doi.org/10.1016/j.mad.2015.02.002">10.1016/j.mad.2015.02.002</a>.</p>
</div>
<div id="ref-SunEpigenomicprofilingyoung2014">
<p>Sun, Deqiang, Min Luo, Mira Jeong, Benjamin Rodriguez, Zheng Xia, Rebecca Hannah, Hui Wang, Thuc Le, Kym F. Faull, and Rui Chen. 2014. “Epigenomic Profiling of Young and Aged HSCs Reveals Concerted Changes During Aging That Reinforce Self-Renewal.” <em>Cell Stem Cell</em> 14 (5): 673–88.</p>
</div>
<div id="ref-MaegawaWidespreadtissuespecific2010">
<p>Maegawa, Shinji, George Hinkal, Hyun Soo Kim, Lanlan Shen, Li Zhang, Jiexin Zhang, Nianxiang Zhang, Shoudan Liang, Lawrence A. Donehower, and Jean-Pierre J. Issa. 2010. “Widespread and Tissue Specific Age-Related DNA Methylation Changes in Mice.” <em>Genome Research</em> 20 (3): 332–40. doi:<a href="https://doi.org/10.1101/gr.096826.109">10.1101/gr.096826.109</a>.</p>
</div>
<div id="ref-AvrahamiAgingdependentdemethylationregulatory2015">
<p>Avrahami, Dana, Changhong Li, Jia Zhang, Jonathan Schug, Ran Avrahami, Shilpa Rao, Michael B. Stadler, Lukas Burger, Dirk Schübeler, and Benjamin Glaser. 2015. “Aging-Dependent Demethylation of Regulatory Elements Correlates with Chromatin State and Improved β Cell Function.” <em>Cell Metabolism</em> 22 (4): 619–32.</p>
</div>
<div id="ref-BeermanProliferationdependentalterationsDNA2013">
<p>Beerman, Isabel, Christoph Bock, Brian S. Garrison, Zachary D. Smith, Hongcang Gu, Alexander Meissner, and Derrick J. Rossi. 2013. “Proliferation-Dependent Alterations of the DNA Methylation Landscape Underlie Hematopoietic Stem Cell Aging.” <em>Cell Stem Cell</em> 12 (4): 413–25.</p>
</div>
<div id="ref-BoothAgingEpigenome2016a">
<p>Booth, Lauren N., and Anne Brunet. 2016. “The Aging Epigenome.” <em>Molecular Cell</em> 62 (5): 728–44. doi:<a href="https://doi.org/10.1016/j.molcel.2016.05.013">10.1016/j.molcel.2016.05.013</a>.</p>
</div>
<div id="ref-KimWidespreadtranscriptionneuronal2010">
<p>Kim, Tae-Kyung, Martin Hemberg, Jesse M. Gray, Allen M. Costa, Daniel M. Bear, Jing Wu, David A. Harmin, Mike Laptewicz, Kellie Barbara-Haley, and Scott Kuersten. 2010. “Widespread Transcription at Neuronal Activity-Regulated Enhancers.” <em>Nature</em> 465 (7295): 182.</p>
</div>
<div id="ref-FernandezH3K4me1marksDNA2015">
<p>Fern’andez, Agust’in F., Gustavo F. Bay’on, Roc’io G. Urdinguio, Estela G. Toraño, Mar’ia G. Garc’ia, Antonella Carella, Sandra Petrus-Reurer, et al. 2015. “H3K4me1 Marks DNA Regions Hypomethylated During Aging in Human Stem and Differentiated Cells.” <em>Genome Research</em> 25 (1): 27–40. doi:<a href="https://doi.org/10.1101/gr.169011.113">10.1101/gr.169011.113</a>.</p>
</div>
<div id="ref-SenH3K36methylationpromotes2015">
<p>Sen, Payel, Weiwei Dang, Greg Donahue, Junbiao Dai, Jean Dorsey, Xiaohua Cao, Wei Liu, Kajia Cao, Rocco Perry, and Jun Yeop Lee. 2015. “H3K36 Methylation Promotes Longevity by Enhancing Transcriptional Fidelity.” <em>Genes &amp; Development</em> 29 (13): 1362–76.</p>
</div>
<div id="ref-PuTrimethylationLys36H32015">
<p>Pu, Mintie, Zhuoyu Ni, Minghui Wang, Xiujuan Wang, Jason G. Wood, Stephen L. Helfand, Haiyuan Yu, and Siu Sylvia Lee. 2015. “Trimethylation of Lys36 on H3 Restricts Gene Expression Change During Aging and Impacts Life Span.” <em>Genes &amp; Development</em> 29 (7): 718–31.</p>
</div>
<div id="ref-FnuMethylationhistoneH32011">
<p>Fnu, Sheema, Elizabeth A. Williamson, Leyma P. De Haro, Mark Brenneman, Justin Wray, Montaser Shaheen, Krishnan Radhakrishnan, Suk-Hee Lee, Jac A. Nickoloff, and Robert Hromas. 2011. “Methylation of Histone H3 Lysine 36 Enhances DNA Repair by Nonhomologous End-Joining.” <em>Proceedings of the National Academy of Sciences</em> 108 (2): 540–45. <a href="http://www.pnas.org/content/108/2/540.short" class="uri">http://www.pnas.org/content/108/2/540.short</a>.</p>
</div>
<div id="ref-YokoyamaDNAmethylationalterations2017">
<p>Yokoyama, Amy S., John C. Rutledge, and Valentina Medici. 2017. “DNA Methylation Alterations in Alzheimer’s Disease.” <em>Environmental Epigenetics</em> 3 (2): dvx008.</p>
</div>
<div id="ref-LiuInsufficientDNAmethylation2011">
<p>Liu, Liang, Thomas van Groen, Inga Kadish, Yuanyuan Li, Deli Wang, Smitha R. James, Adam R. Karpf, and Trygve O. Tollefsbol. 2011. “Insufficient DNA Methylation Affects Healthy Aging and Promotes Age-Related Health Problems.” <em>Clinical Epigenetics</em> 2 (2): 349. doi:<a href="https://doi.org/10.1007/s13148-011-0042-6">10.1007/s13148-011-0042-6</a>.</p>
</div>
<div id="ref-BollatiDNAmethylationrepetitive2011">
<p>Bollati, V., D. Galimberti, L. Pergoli, E. Dalla Valle, F. Barretta, F. Cortini, E. Scarpini, P. A. Bertazzi, and A. Baccarelli. 2011. “DNA Methylation in Repetitive Elements and Alzheimer Disease.” <em>Brain, Behavior, and Immunity</em> 25 (6): 1078–83. doi:<a href="https://doi.org/10.1016/j.bbi.2011.01.017">10.1016/j.bbi.2011.01.017</a>.</p>
</div>
<div id="ref-NestorTissuetypemajor2012">
<p>Nestor, Colm E., Raffaele Ottaviano, James Reddington, Duncan Sproul, Diana Reinhardt, Donncha Dunican, Elad Katz, J. Michael Dixon, David J. Harrison, and Richard R. Meehan. 2012. “Tissue Type Is a Major Modifier of the 5-Hydroxymethylcytosine Content of Human Genes.” <em>Genome Research</em> 22 (3): 467–77.</p>
</div>
<div id="ref-Szulwach5hmCmediatedepigenetic2011">
<p>Szulwach, Keith E, Xuekun Li, Yujing Li, Chun-Xiao Song, Hao Wu, Qing Dai, Hasan Irier, et al. 2011. “5-hmC–Mediated Epigenetic Dynamics During Postnatal Neurodevelopment and Aging.” <em>Nature Neuroscience</em> 14 (12): 1607–16. doi:<a href="https://doi.org/10.1038/nn.2959">10.1038/nn.2959</a>.</p>
</div>
<div id="ref-ChenEffectaging5hydroxymethylcytosine2012">
<p>Chen, Hu, Svetlana Dzitoyeva, and Hari Manev. 2012. “Effect of Aging on 5-Hydroxymethylcytosine in the Mouse Hippocampus.” <em>Restorative Neurology and Neuroscience</em> 30 (3): 237–45.</p>
</div>
<div id="ref-WagnerAgeDependentLevels5Methyl2015">
<p>Wagner, Mirko, Jessica Steinbacher, Theo F. J. Kraus, Stylianos Michalakis, Benjamin Hackner, Toni Pfaffeneder, Arshan Perera, et al. 2015. “Age-Dependent Levels of 5-Methyl-, 5-Hydroxymethyl-, and 5-Formylcytosine in Human and Mouse Brain Tissues.” <em>Angewandte Chemie International Edition</em> 54 (42): 12511–4. doi:<a href="https://doi.org/10.1002/anie.201502722">10.1002/anie.201502722</a>.</p>
</div>
<div id="ref-Zhaogenomewideprofilingbrain2017">
<p>Zhao, Jinying, Yun Zhu, Jingyun Yang, Lin Li, Hao Wu, Philip L. De Jager, Peng Jin, and David A. Bennett. 2017. “A Genome-Wide Profiling of Brain DNA Hydroxymethylation in Alzheimer’s Disease.” <em>Alzheimer’s &amp; Dementia</em> 13 (6): 674–88. doi:<a href="https://doi.org/10.1016/j.jalz.2016.10.004">10.1016/j.jalz.2016.10.004</a>.</p>
</div>
<div id="ref-Bradley-WhitmanEpigeneticchangesprogression2013">
<p>Bradley-Whitman, M. A., and M. A. Lovell. 2013. “Epigenetic Changes in the Progression of Alzheimer’s Disease.” <em>Mechanisms of Ageing and Development</em>, Genome dynamics shaping neuroscience, 134 (10): 486–95. doi:<a href="https://doi.org/10.1016/j.mad.2013.08.005">10.1016/j.mad.2013.08.005</a>.</p>
</div>
<div id="ref-LiActivationtransposableelements2013">
<p>Li, Wanhe, Lisa Prazak, Nabanita Chatterjee, Servan Grüninger, Lisa Krug, Delphine Theodorou, and Josh Dubnau. 2013. “Activation of Transposable Elements During Aging and Neuronal Decline in Drosophila.” <em>Nature Neuroscience</em> 16 (5): 529–31. doi:<a href="https://doi.org/10.1038/nn.3368">10.1038/nn.3368</a>.</p>
</div>
<div id="ref-WoodChromatinmodifyinggeneticinterventions2016">
<p>Wood, Jason G., Brian C. Jones, Nan Jiang, Chengyi Chang, Suzanne Hosier, Priyan Wickremesinghe, Meyrolin Garcia, et al. 2016. “Chromatin-Modifying Genetic Interventions Suppress Age-Associated Transposable Element Activation and Extend Life Span in <em>Drosophila</em>.” <em>Proceedings of the National Academy of Sciences</em> 113 (40): 11277–82. doi:<a href="https://doi.org/10.1073/pnas.1604621113">10.1073/pnas.1604621113</a>.</p>
</div>
<div id="ref-SanchezAgingApolipoproteinMolecular2015a">
<p>Sanchez, Diego, Raquel Bajo-Grañeras, Manuela Del Caño-Espinel, Rosa Garcia-Centeno, Nadia Garcia-Mateo, Raquel Pascua-Maestro, and Maria D. Ganfornina. 2015. “Aging Without Apolipoprotein D: Molecular and Cellular Modifications in the Hippocampus and Cortex.” <em>Experimental Gerontology</em> 67 (July): 19–47. doi:<a href="https://doi.org/10.1016/j.exger.2015.04.003">10.1016/j.exger.2015.04.003</a>.</p>
</div>
<div id="ref-MeierExpressionsnoRNAhost2010">
<p>Meier, Ingo, Laetitia Fellini, Mira Jakovcevski, Melitta Schachner, and Fabio Morellini. 2010. “Expression of the snoRNA Host Gene Gas5 in the Hippocampus Is Upregulated by Age and Psychogenic Stress and Correlates with Reduced Novelty‐induced Behavior in C57bl/6 Mice.” <em>Hippocampus</em> 20 (9): 1027–36.</p>
</div>
<div id="ref-WilliamsonMRCHarwellOxfordshire2013">
<p>Williamson, C. M., A. Blake, S. Thomas, C. V. Beechey, J. Hancock, B. M. Cattanach, and J. Peters. 2013. “MRC Harwell, Oxfordshire.” <em>World Wide Web Site-Mouse Imprinting Data and References.[http://Www. Har. Mrc. Ac. Uk/Research/Genomic_imprinting]</em>.</p>
</div>
<div id="ref-CoufalAtaxiatelangiectasiamutated2011">
<p>Coufal, Nicole G., Jos‘e Luis Garcia-Perez, Grace E. Peng, Maria C. N. Marchetto, Alysson R. Muotri, Yangling Mu, Christian T. Carson, Angela Macia, John V. Moran, and Fred H. Gage. 2011. “Ataxia Telangiectasia Mutated (ATM) Modulates Long Interspersed Element-1 (L1) Retrotransposition in Human Neural Stem Cells.” <em>Proceedings of the National Academy of Sciences of the United States of America</em> 108 (51): 20382–7. doi:<a href="https://doi.org/10.1073/pnas.1100273108">10.1073/pnas.1100273108</a>.</p>
</div>
<div id="ref-DouvilleIdentificationactiveloci2011">
<p>Douville, Ren’ee, Jiankai Liu, Jeffrey Rothstein, and Avindra Nath. 2011. “Identification of Active Loci of a Human Endogenous Retrovirus in Neurons of Patients with Amyotrophic Lateral Sclerosis.” <em>Annals of Neurology</em> 69 (1): 141–51.</p>
</div>
<div id="ref-Jeongprevalencehumanendogenous2010">
<p>Jeong, Byung-Hoon, Yun-Jung Lee, Richard I. Carp, and Yong-Sun Kim. 2010. “The Prevalence of Human Endogenous Retroviruses in Cerebrospinal Fluids from Patients with Sporadic Creutzfeldt–Jakob Disease.” <em>Journal of Clinical Virology</em> 47 (2): 136–42.</p>
</div>
<div id="ref-LiHumanendogenousretrovirusK2015a">
<p>Li, Wenxue, Myoung-Hwa Lee, Lisa Henderson, Richa Tyagi, Muzna Bachani, Joseph Steiner, Emilie Campanac, et al. 2015. “Human Endogenous Retrovirus-K Contributes to Motor Neuron Disease.” <em>Science Translational Medicine</em> 7 (307): 307ra153–307ra153. <a href="http://stm.sciencemag.org/content/7/307/307ra153.short" class="uri">http://stm.sciencemag.org/content/7/307/307ra153.short</a>.</p>
</div>
<div id="ref-KanekoDICER1deficitinduces2011">
<p>Kaneko, Hiroki, Sami Dridi, Valeria Tarallo, Bradley D. Gelfand, Benjamin J. Fowler, Won Gil Cho, Mark E. Kleinman, et al. 2011. “DICER1 Deficit Induces Alu RNA Toxicity in Age-Related Macular Degeneration.” <em>Nature</em> 471 (7338): 325–30. doi:<a href="https://doi.org/10.1038/nature09830">10.1038/nature09830</a>.</p>
</div>
<div id="ref-LeeInvolvementEndogenousRetroviruses2013">
<p>Lee, Yun-Jung, Byung-Hoon Jeong, Eun-Kyung Choi, and Yong-Sun Kim. 2013. “Involvement of Endogenous Retroviruses in Prion Diseases.” <em>Pathogens</em> 2 (3): 533–43. doi:<a href="https://doi.org/10.3390/pathogens2030533">10.3390/pathogens2030533</a>.</p>
</div>
<div id="ref-LiTransposableelementsTDP43mediated2012">
<p>Li, Wanhe, Ying Jin, Lisa Prazak, Molly Hammell, and Josh Dubnau. 2012. “Transposable Elements in TDP-43-Mediated Neurodegenerative Disorders.” <em>PloS One</em> 7 (9): e44099. doi:<a href="https://doi.org/10.1371/journal.pone.0044099">10.1371/journal.pone.0044099</a>.</p>
</div>
<div id="ref-MameliActivationMSRVTypeEndogenous2013a">
<p>Mameli, Giuseppe, Giordano Madeddu, Alessandra Mei, Elena Uleri, Luciana Poddighe, Lucia G. Delogu, Ivana Maida, et al. 2013. “Activation of MSRV-Type Endogenous Retroviruses During Infectious Mononucleosis and Epstein-Barr Virus Latency: The Missing Link with Multiple Sclerosis?” <em>PLOS ONE</em> 8 (11): e78474. doi:<a href="https://doi.org/10.1371/journal.pone.0078474">10.1371/journal.pone.0078474</a>.</p>
</div>
<div id="ref-PerronHumanendogenousretrovirus2005">
<p>Perron, Herv’e, Françoise Lazarini, Klemens Ruprecht, Christine P’echoux-Longin, Danielle Seilhean, V’eronique Sazdovitch, Alain Cr’eange, et al. 2005. “Human Endogenous Retrovirus (HERV)-W ENV and GAG Proteins: Physiological Expression in Human Brain and Pathophysiological Modulation in Multiple Sclerosis Lesions.” <em>Journal of Neurovirology</em> 11 (1): 23–33. doi:<a href="https://doi.org/10.1080/13550280590901741">10.1080/13550280590901741</a>.</p>
</div>
<div id="ref-BrocksDNMTHDACinhibitors2017">
<p>Brocks, David, Christopher R Schmidt, Michael Daskalakis, Hyo Sik Jang, Nakul M Shah, Daofeng Li, Jing Li, et al. 2017. “DNMT and HDAC Inhibitors Induce Cryptic Transcription Start Sites Encoded in Long Terminal Repeats.” <em>Nature Genetics</em> 49 (7): 1052–60. doi:<a href="https://doi.org/10.1038/ng.3889">10.1038/ng.3889</a>.</p>
</div>
<div id="ref-NishimotolongnoncodingRNA2013">
<p>Nishimoto, Yoshinori, Shinichi Nakagawa, Tetsuro Hirose, Hirotaka James Okano, Masaki Takao, Shinsuke Shibata, Satoshi Suyama, Ken-ichiro Kuwako, Takao Imai, and Shigeo Murayama. 2013. “The Long Non-Coding RNA Nuclear-Enriched Abundant Transcript 1_2 Induces Paraspeckle Formation in the Motor Neuron During the Early Phase of Amyotrophic Lateral Sclerosis.” <em>Molecular Brain</em> 6 (1): 31.</p>
</div>
<div id="ref-SalviRloopshighlightnucleus2015">
<p>Salvi, Jayesh S., and Karim Mekhail. 2015. “R-Loops Highlight the Nucleus in ALS.” <em>Nucleus</em> 6 (1): 23–29.</p>
</div>
<div id="ref-GrunerCircRNAaccumulationaging2016">
<p>Gruner, Hannah, Mariela Cort’es-L’opez, Daphne A. Cooper, Matthew Bauer, and Pedro Miura. 2016. “CircRNA Accumulation in the Aging Mouse Brain.” <em>Scientific Reports</em> 6: 38907.</p>
</div>
<div id="ref-LarsenWarningSINEsAlu2018b">
<p>Larsen, Peter A., Kelsie E. Hunnicutt, Roxanne J. Larsen, Anne D. Yoder, and Ann M. Saunders. 2018. “Warning SINEs: Alu Elements, Evolution of the Human Brain, and the Spectrum of Neurological Disease.” <em>Chromosome Research</em> 26 (1-2): 93–111. doi:<a href="https://doi.org/10.1007/s10577-018-9573-4">10.1007/s10577-018-9573-4</a>.</p>
</div>
<div id="ref-KrugRetrotransposonactivationcontributes2017a">
<p>Krug, Lisa, Nabanita Chatterjee, Rebeca Borges-Monroy, Stephen Hearn, Wen-Wei Liao, Kathleen Morrill, Lisa Prazak, et al. 2017. “Retrotransposon Activation Contributes to Neurodegeneration in a Drosophila TDP-43 Model of ALS.” <em>PLOS Genetics</em> 13 (3): e1006635. doi:<a href="https://doi.org/10.1371/journal.pgen.1006635">10.1371/journal.pgen.1006635</a>.</p>
</div>
<div id="ref-PrudencioRepetitiveelementtranscripts2017">
<p>Prudencio, Mercedes, Patrick K. Gonzales, Casey N. Cook, Tania F. Gendron, Lillian M. Daughrity, Yuping Song, Mark T. W. Ebbert, et al. 2017. “Repetitive Element Transcripts Are Elevated in the Brain of C9orf72 ALS/FTLD Patients.” <em>Human Molecular Genetics</em> 26 (17): 3421–31. doi:<a href="https://doi.org/10.1093/hmg/ddx233">10.1093/hmg/ddx233</a>.</p>
</div>
<div id="ref-NonakaPrionlikePropertiesPathological2013">
<p>Nonaka, Takashi, Masami Masuda-Suzukake, Tetsuaki Arai, Yoko Hasegawa, Hiroyasu Akatsu, Tomokazu Obi, Mari Yoshida, et al. 2013. “Prion-Like Properties of Pathological TDP-43 Aggregates from Diseased Brains.” <em>Cell Reports</em> 4 (1): 124–34. doi:<a href="https://doi.org/10.1016/j.celrep.2013.06.007">10.1016/j.celrep.2013.06.007</a>.</p>
</div>
<div id="ref-FrostTaupromotesneurodegeneration2014">
<p>Frost, Bess, Martin Hemberg, Jada Lewis, and Mel B Feany. 2014. “Tau Promotes Neurodegeneration Through Global Chromatin Relaxation.” <em>Nature Neuroscience</em> 17 (3): 357–66. doi:<a href="https://doi.org/10.1038/nn.3639">10.1038/nn.3639</a>.</p>
</div>
<div id="ref-GuoTauActivatesTransposable2018">
<p>Guo, Caiwei, Hyun-Hwan Jeong, Yi-Chen Hsieh, Hans-Ulrich Klein, David A. Bennett, Philip L. De Jager, Zhandong Liu, and Joshua M. Shulman. 2018. “Tau Activates Transposable Elements in Alzheimer’s Disease.” <em>Cell Reports</em> 23 (10): 2874–80. doi:<a href="https://doi.org/10.1016/j.celrep.2018.05.004">10.1016/j.celrep.2018.05.004</a>.</p>
</div>
<div id="ref-MansurogluLossTauprotein2016">
<p>Mansuroglu, Zeyni, Houda Benhelli-Mokrani, Vasco Marcato, Audrey Sultan, Marie Violet, Alban Chauderlier, Lucie Delattre, et al. 2016. “Loss of Tau Protein Affects the Structure, Transcription and Repair of Neuronal Pericentromeric Heterochromatin.” <em>Scientific Reports</em> 6 (September): 33047. doi:<a href="https://doi.org/10.1038/srep33047">10.1038/srep33047</a>.</p>
</div>
<div id="ref-NohATRXtoleratesactivitydependent2014">
<p>Noh, Kyung-Min, Ian Maze, Dan Zhao, Bin Xiang, Wendy Wenderski, Peter W. Lewis, Li Shen, Haitao Li, and C. David Allis. 2014. “ATRX Tolerates Activity-Dependent Histone H3 Methyl/Phos Switching to Maintain Repetitive Element Silencing in Neurons.” <em>Proceedings of the National Academy of Sciences</em>, 201411258. <a href="http://www.pnas.org/content/early/2014/12/22/1411258112.short" class="uri">http://www.pnas.org/content/early/2014/12/22/1411258112.short</a>.</p>
</div>
<div id="ref-HirotaHistoneH3serine2005">
<p>Hirota, Toru, Jesse J. Lipp, Ban-Hock Toh, and Jan-Michael Peters. 2005. “Histone H3 Serine 10 Phosphorylation by Aurora B Causes Hp1 Dissociation from Heterochromatin.” <em>Nature</em> 438 (7071): 1176–80.</p>
</div>
<div id="ref-VoonATRXplayskey2015">
<p>Voon, Hsiao PJ, Jim R. Hughes, Christina Rode, A. Inti, Thomas Jenuwein, Robert Feil, Douglas R. Higgs, and Richard J. Gibbons. 2015. “ATRX Plays a Key Role in Maintaining Silencing at Interstitial Heterochromatic Loci and Imprinted Genes.” <em>Cell Reports</em> 11 (3): 405–18.</p>
</div>
<div id="ref-Oberdoerfferrolenucleararchitecture2007">
<p>Oberdoerffer, Philipp, and David A. Sinclair. 2007. “The Role of Nuclear Architecture in Genomic Instability and Ageing.” <em>Nature Reviews Molecular Cell Biology</em> 8 (9): 692.</p>
</div>
<div id="ref-CoppedeDNAdamageneurodegenerative2015">
<p>Copped‘e, Fabio, and Lucia Migliore. 2015. “DNA Damage in Neurodegenerative Diseases.” <em>Mutation Research/Fundamental and Molecular Mechanisms of Mutagenesis</em>, DNA damage in chronic diseases and aging, 776 (June): 84–97. doi:<a href="https://doi.org/10.1016/j.mrfmmm.2014.11.010">10.1016/j.mrfmmm.2014.11.010</a>.</p>
</div>
<div id="ref-MadabhushiActivityInducedDNABreaks2015">
<p>Madabhushi, Ram, Fan Gao, Andreas R. Pfenning, Ling Pan, Satoko Yamakawa, Jinsoo Seo, Richard Rueda, et al. 2015. “Activity-Induced DNA Breaks Govern the Expression of Neuronal Early-Response Genes.” <em>Cell</em> 161 (7): 1592–1605. doi:<a href="https://doi.org/10.1016/j.cell.2015.05.032">10.1016/j.cell.2015.05.032</a>.</p>
</div>
<div id="ref-WeiTranscriptioncoupledhomologousrecombination2016">
<p>Wei, Leizhen, Arthur Samuel Levine, and Li Lan. 2016. “Transcription-Coupled Homologous Recombination After Oxidative Damage.” <em>DNA Repair</em>, May. doi:<a href="https://doi.org/10.1016/j.dnarep.2016.05.009">10.1016/j.dnarep.2016.05.009</a>.</p>
</div>
<div id="ref-McDevittHowRNAtranscripts2018">
<p>McDevitt, Shane, Timur Rusanov, Tatiana Kent, Gurushankar Chandramouly, and Richard T. Pomerantz. 2018. “How RNA Transcripts Coordinate DNA Recombination and Repair.” <em>Nature Communications</em> 9 (1): 1091. doi:<a href="https://doi.org/10.1038/s41467-018-03483-7">10.1038/s41467-018-03483-7</a>.</p>
</div>
<div id="ref-NewmanActivityDEPendentTransposition2016">
<p>Newman, Andrew G., Paraskevi Bessa, V. Tarabykin, and Prim B. Singh. 2016. “Activity DEPendent Transposition (ADEPT) and the Aging Brain.” <em>Opera Medica et Physiologica</em>, no. 3-4. <a href="http://cyberleninka.ru/article/n/activity-dependent-transposition-adept-and-the-aging-brain" class="uri">http://cyberleninka.ru/article/n/activity-dependent-transposition-adept-and-the-aging-brain</a>.</p>
</div>
<div id="ref-NewmanActivityDEPendentTransposition2017">
<p>Newman, Andrew G, Paraskevi Bessa, Victor Tarabykin, and Prim B Singh. 2017. “Activity‐DEPendent Transposition.” <em>EMBO Reports</em>, February, e201643797. doi:<a href="https://doi.org/10.15252/embr.201643797">10.15252/embr.201643797</a>.</p>
</div>
<div id="ref-JamesIdentificationnonhistonechromosomal1986">
<p>James, THARAPPEL C., and S. C. Elgin. 1986. “Identification of a Nonhistone Chromosomal Protein Associated with Heterochromatin in Drosophila Melanogaster and Its Gene.” <em>Molecular and Cellular Biology</em> 6 (11): 3862–72.</p>
</div>
<div id="ref-JamesDistributionpatternsHP11989">
<p>James, T. C., J. C. Eissenberg, C. Craig, V. Dietrich, A. Hobson, and S. C. Elgin. 1989. “Distribution Patterns of Hp1, a Heterochromatin-Associated Nonhistone Chromosomal Protein of Drosophila.” <em>European Journal of Cell Biology</em> 50 (1): 170–80.</p>
</div>
<div id="ref-Singhsequencemotiffound1991">
<p>Singh, Prim B., J. Ross Miller, Jonathan Pearce, Rashmi Kothary, Robert D. Burton, Renato Paro, Tharappel C. James, and Stephen J. Gaunt. 1991. “A Sequence Motif Found in a Drosophila Heterochromatin Protein Is Conserved in Animals and Plants.” <em>Nucleic Acids Research</em> 19 (4): 789–94. <a href="http://nar.oxfordjournals.org/content/19/4/789.short" class="uri">http://nar.oxfordjournals.org/content/19/4/789.short</a>.</p>
</div>
<div id="ref-CanzioconformationalswitchHP12013">
<p>Canzio, Daniele, Maofu Liao, Nariman Naber, Edward Pate, Adam Larson, Shenping Wu, Diana B. Marina, et al. 2013. “A Conformational Switch in Hp1 Releases Auto-Inhibition to Drive Heterochromatin Assembly.” <em>Nature</em> 496 (7445): 377–81. doi:<a href="https://doi.org/10.1038/nature12032">10.1038/nature12032</a>.</p>
</div>
<div id="ref-NielsenStructureHP1chromodomain2002">
<p>Nielsen, Peter R., Daniel Nietlispach, Helen R. Mott, Juliana Callaghan, Andrew Bannister, Tony Kouzarides, Alexey G. Murzin, Natalia V. Murzina, and Ernest D. Laue. 2002. “Structure of the Hp1 Chromodomain Bound to Histone H3 Methylated at Lysine 9.” <em>Nature</em> 416 (6876): 103.</p>
</div>
<div id="ref-SmothersHP1chromoshadow2000">
<p>Smothers, James F., and Steven Henikoff. 2000. “The Hp1 Chromo Shadow Domain Binds a Consensus Peptide Pentamer.” <em>Current Biology</em> 10 (1): 27–30.</p>
</div>
<div id="ref-BrasherstructuremouseHP12000">
<p>Brasher, Sally V., Brian O. Smith, Rasmus H. Fogh, Daniel Nietlispach, Abarna Thiru, Peter R. Nielsen, R. William Broadhurst, Linda J. Ball, Natalia V. Murzina, and Ernest D. Laue. 2000. “The Structure of Mouse Hp1 Suggests a Unique Mode of Single Peptide Recognition by the Shadow Chromo Domain Dimer.” <em>The EMBO Journal</em> 19 (7): 1587–97. doi:<a href="https://doi.org/10.1093/emboj/19.7.1587">10.1093/emboj/19.7.1587</a>.</p>
</div>
<div id="ref-ThiruStructuralbasisHP12004">
<p>Thiru, Abarna, Daniel Nietlispach, Helen R. Mott, Mitsuru Okuwaki, Debbie Lyon, Peter R. Nielsen, Miriam Hirshberg, Alain Verreault, Natalia V. Murzina, and Ernest D. Laue. 2004. “Structural Basis of Hp1/PXVXL Motif Peptide Interactions and Hp1 Localisation to Heterochromatin.” <em>The EMBO Journal</em> 23 (3): 489–99. <a href="http://www.nature.com/emboj/journal/v23/n3/abs/7600088a.html" class="uri">http://www.nature.com/emboj/journal/v23/n3/abs/7600088a.html</a>.</p>
</div>
<div id="ref-KellerHP1Swi6mediates2012">
<p>Keller, Claudia, Ricardo Adaixo, Rieka Stunnenberg, Katrina J. Woolcock, Sebastian Hiller, and Marc Bühler. 2012. “HP1 Swi6 Mediates the Recognition and Destruction of Heterochromatic RNA Transcripts.” <em>Molecular Cell</em> 47 (2): 215–27.</p>
</div>
<div id="ref-ZhaoHeterochromatinproteinbinds2000">
<p>Zhao, Tao, Thomas Heyduk, C. David Allis, and Joel C. Eissenberg. 2000. “Heterochromatin Protein 1 Binds to Nucleosomes and DNA in Vitro.” <em>Journal of Biological Chemistry</em> 275 (36): 28332–8.</p>
</div>
<div id="ref-MatobaEmbryonicDevelopmentfollowing2014">
<p>Matoba, Shogo, Yuting Liu, Falong Lu, Kumiko A. Iwabuchi, Li Shen, Azusa Inoue, and Yi Zhang. 2014. “Embryonic Development Following Somatic Cell Nuclear Transfer Impeded by Persisting Histone Methylation.” <em>Cell</em> 159 (4): 884–95. doi:<a href="https://doi.org/10.1016/j.cell.2014.09.055">10.1016/j.cell.2014.09.055</a>.</p>
</div>
<div id="ref-MincLocalizationphosphorylationHP11999">
<p>Minc, Elsa, Yves Allory, Howard J. Worman, Jean-Claude Courvalin, and Brigitte Buendia. 1999. “Localization and Phosphorylation of Hp1 Proteins During the Cell Cycle in Mammalian Cells.” <em>Chromosoma</em> 108 (4): 220–34.</p>
</div>
<div id="ref-SmallwoodCBX3regulatesefficient2012">
<p>Smallwood, Andrea, Gary C. Hon, Fulai Jin, Ryan E. Henry, Joaquin E. Espinosa, and Bing Ren. 2012. “CBX3 Regulates Efficient RNA Processing Genome-Wide.” <em>Genome Research</em>, gr. 124818.111.</p>
</div>
<div id="ref-VakocHistoneH3Lysine2005">
<p>Vakoc, Christopher R., Sean A. Mandat, Benjamin A. Olenchock, and Gerd A. Blobel. 2005. “Histone H3 Lysine 9 Methylation and HP1γ Are Associated with Transcription Elongation Through Mammalian Chromatin.” <em>Molecular Cell</em> 19 (3): 381–91. doi:<a href="https://doi.org/10.1016/j.molcel.2005.06.011">10.1016/j.molcel.2005.06.011</a>.</p>
</div>
<div id="ref-HuangCbx3maintainslineage2017">
<p>Huang, Chengyang, Trent Su, Yong Xue, Chen Cheng, Fides D. Lay, Robin A. McKee, Meiyang Li, Ajay Vashisht, James Wohlschlegel, and Bennett G. Novitch. 2017. “Cbx3 Maintains Lineage Specificity During Neural Differentiation.” <em>Genes &amp; Development</em> 31 (3): 241–46.</p>
</div>
<div id="ref-OstapcukActivitydependentneuroprotectiveprotein2018">
<p>Ostapcuk, Veronika, Fabio Mohn, Sarah H. Carl, Anja Basters, Daniel Hess, Vytautas Iesmantavicius, Lisa Lampersberger, et al. 2018. “Activity-Dependent Neuroprotective Protein Recruits Hp1 and Chd4 to Control Lineage-Specifying Genes.” <em>Nature</em> 557 (7707): 739–43. doi:<a href="https://doi.org/10.1038/s41586-018-0153-8">10.1038/s41586-018-0153-8</a>.</p>
</div>
<div id="ref-CanzioMechanismsfunctionalpromiscuity2014">
<p>Canzio, Daniele, Adam Larson, and Geeta J. Narlikar. 2014. “Mechanisms of Functional Promiscuity by Hp1 Proteins.” <em>Trends in Cell Biology</em> 24 (6): 377–86. doi:<a href="https://doi.org/10.1016/j.tcb.2014.01.002">10.1016/j.tcb.2014.01.002</a>.</p>
</div>
<div id="ref-SmothersHingeChromoShadow2001">
<p>Smothers, James F., and Steven Henikoff. 2000. “The Hp1 Chromo Shadow Domain Binds a Consensus Peptide Pentamer.” <em>Current Biology</em> 10 (1): 27–30.</p> 2001. “The Hinge and Chromo Shadow Domain Impart Distinct Targeting of Hp1-Like Proteins.” <em>Molecular and Cellular Biology</em> 21 (7): 2555–69. doi:<a href="https://doi.org/10.1128/MCB.21.7.2555-2569.2001">10.1128/MCB.21.7.2555-2569.2001</a>.</p>
</div>
<div id="ref-Hiragami-HamadaNTerminalPhosphorylationHP12011">
<p>Hiragami-Hamada, K., K. Shinmyozu, D. Hamada, Y. Tatsu, K. Uegaki, S. Fujiwara, and J.-i. Nakayama. 2011. “N-Terminal Phosphorylation of Hp1 Promotes Its Chromatin Binding.” <em>Molecular and Cellular Biology</em> 31 (6): 1186–1200. doi:<a href="https://doi.org/10.1128/MCB.01012-10">10.1128/MCB.01012-10</a>.</p>
</div>
<div id="ref-NishibuchiNterminalphosphorylationHP1a2014">
<p>Nishibuchi, Gohei, Shinichi Machida, Akihisa Osakabe, Hiromu Murakoshi, Kyoko Hiragami-Hamada, Reiko Nakagawa, Wolfgang Fischle, et al. 2014. “N-Terminal Phosphorylation of HP1α Increases Its Nucleosome-Binding Specificity.” <em>Nucleic Acids Research</em> 42 (20): 12498–12511. doi:<a href="https://doi.org/10.1093/nar/gku995">10.1093/nar/gku995</a>.</p>
</div>
<div id="ref-MachadoIsoformspecificdeterminantsHP12010">
<p>Machado, Matias R., Pablo D. Dans, and Sergio Pantano. 2010. “Isoform-Specific Determinants in the Hp1 Binding to Histone 3: Insights from Molecular Simulations.” <em>Amino Acids</em> 38 (5): 1571–81. doi:<a href="https://doi.org/10.1007/s00726-009-0371-3">10.1007/s00726-009-0371-3</a>.</p>
</div>
<div id="ref-Bosch-PresegueMammalianHP1Isoforms2017">
<p>Bosch-Presegu’e, Laia, Helena Raurell-Vila, Joshua K. Thackray, Jessica Gonz’alez, Carmen Casal, Noriko Kane-Goldsmith, Miguel Vizoso, et al. 2017. “Mammalian Hp1 Isoforms Have Specific Roles in Heterochromatin Structure and Organization.” <em>Cell Reports</em> 21 (8): 2048–57. doi:<a href="https://doi.org/10.1016/j.celrep.2017.10.092">10.1016/j.celrep.2017.10.092</a>.</p>
</div>
<div id="ref-DialynasPlasticityHP1proteins2007">
<p>Dialynas, George K., Stefan Terjung, Jeremy P. Brown, Rebecca L. Aucott, Bettina Baron-Luhr, Prim B. Singh, and Spyros D. Georgatos. 2007. “Plasticity of Hp1 Proteins in Mammalian Cells.” <em>Journal of Cell Science</em> 120 (19): 3415–24.</p>
</div>
<div id="ref-DawsonJAK2phosphorylateshistone2009">
<p>Dawson, Mark A., Andrew J. Bannister, Berthold Göttgens, Samuel D. Foster, Till Bartke, Anthony R. Green, and Tony Kouzarides. 2009. “JAK2 Phosphorylates Histone H3y41 and Excludes HP1α from Chromatin.” <em>Nature</em> 461 (7265): 819–22. doi:<a href="https://doi.org/10.1038/nature08448">10.1038/nature08448</a>.</p>
</div>
<div id="ref-LavigneInteractionHP1Brg12009">
<p>Lavigne, Marc, Ragnhild Eskeland, Saliha Azebi, Violaine Saint-Andr’e, Suk Min Jang, Eric Batsch’e, Hua-Ying Fan, Robert E. Kingston, Axel Imhof, and Christian Muchardt. 2009. “Interaction of Hp1 and Brg1/Brm with the Globular Domain of Histone H3 Is Required for Hp1-Mediated Repression.” Edited by Asifa Akhtar. <em>PLoS Genetics</em> 5 (12): e1000769. doi:<a href="https://doi.org/10.1371/journal.pgen.1000769">10.1371/journal.pgen.1000769</a>.</p>
</div>
<div id="ref-DialynasMethylationindependentBindingHistone2006">
<p>Dialynas, G. K. 2006. “Methylation-Independent Binding to Histone H3 and Cell Cycle-Dependent Incorporation of HP1beta into Heterochromatin.” <em>Journal of Biological Chemistry</em> 281 (20): 14350–60. doi:<a href="https://doi.org/10.1074/jbc.M600558200">10.1074/jbc.M600558200</a>.</p>
</div>
<div id="ref-LeRoyHeterochromatinProteinExtensively2009">
<p>LeRoy, Gary, John T. Weston, Barry M. Zee, Nicolas L. Young, Mariana D. Plazas-Mayorca, and Benjamin A. Garcia. 2009. “Heterochromatin Protein 1 Is Extensively Decorated with Histone Code-Like Post-Translational Modifications.” <em>Molecular &amp; Cellular Proteomics</em> 8 (11): 2432–42. doi:<a href="https://doi.org/10.1074/mcp.M900160-MCP200">10.1074/mcp.M900160-MCP200</a>.</p>
</div>
<div id="ref-MaisonSUMOylationpromotesnovo2011">
<p>Maison, Christ‘ele, Delphine Bailly, Dani‘ele Roche, Rocio Montes de Oca, Aline V Probst, Isabelle Vassias, Florent Dingli, et al. 2011. “SUMOylation Promotes de Novo Targeting of HP1α to Pericentric Heterochromatin.” <em>Nature Genetics</em> 43 (3): 220–27. doi:<a href="https://doi.org/10.1038/ng.765">10.1038/ng.765</a>.</p>
</div>
<div id="ref-LomberkEvidenceexistenceHP1mediated2006">
<p>Lomberk, Gwen, Debora Bensi, Martín E. Fernandez-Zapico, and Raul Urrutia. 2006. “Evidence for the Existence of an Hp1-Mediated Subcode Within the Histone Code.” <em>Nature Cell Biology</em> 8 (4): 407–15. doi:<a href="https://doi.org/10.1038/ncb1383">10.1038/ncb1383</a>.</p>
</div>
<div id="ref-BilluressentialfunctionHP1v2010">
<p>Billur, Mustafa, Hans D. Bartunik, and Prim B. Singh. 2010. “The Essential Function of HP1β: A Case of the Tail Wagging the Dog?” <em>Trends in Biochemical Sciences</em> 35 (2): 115–23. doi:<a href="https://doi.org/10.1016/j.tibs.2009.09.003">10.1016/j.tibs.2009.09.003</a>.</p>
</div>
<div id="ref-MoschHP1RecruitsActivityDependent2011">
<p>Mosch, Kerstin, Henriette Franz, Szabolcs Soeroes, Prim B. Singh, and Wolfgang Fischle. 2011. “HP1 Recruits Activity-Dependent Neuroprotective Protein to H3K9me3 Marked Pericentromeric Heterochromatin for Silencing of Major Satellite Repeats.” Edited by Michael J. Pazin. <em>PLoS ONE</em> 6 (1): e15894. doi:<a href="https://doi.org/10.1371/journal.pone.0015894">10.1371/journal.pone.0015894</a>.</p>
</div>
<div id="ref-LarsonLiquiddropletformation2017">
<p>Larson, Adam G., Daniel Elnatan, Madeline M. Keenen, Michael J. Trnka, Jonathan B. Johnston, Alma L. Burlingame, David A. Agard, Sy Redding, and Geeta J. Narlikar. 2017. “Liquid Droplet Formation by HP1α Suggests a Role for Phase Separation in Heterochromatin.” <em>Nature</em> 547 (7662): 236–40. doi:<a href="https://doi.org/10.1038/nature22822">10.1038/nature22822</a>.</p>
</div>
<div id="ref-StromPhaseseparationdrives2017">
<p>Strom, Amy R., Alexander V. Emelyanov, Mustafa Mir, Dmitry V. Fyodorov, Xavier Darzacq, and Gary H. Karpen. 2017. “Phase Separation Drives Heterochromatin Domain Formation.” <em>Nature</em> 547 (7662): 241–45. doi:<a href="https://doi.org/10.1038/nature22989">10.1038/nature22989</a>.</p>
</div>
<div id="ref-SridharanProteomicgenomicapproaches2013">
<p>Sridharan, Rupa, Michelle Gonzales-Cope, Constantinos Chronis, Giancarlo Bonora, Robin McKee, Chengyang Huang, Sanjeet Patel, et al. 2013. “Proteomic and Genomic Approaches Reveal Critical Functions of H3k9 Methylation and Heterochromatin Protein-1γ in Reprogramming to Pluripotency.” <em>Nature Cell Biology</em> 15 (7): 872–82. doi:<a href="https://doi.org/10.1038/ncb2768">10.1038/ncb2768</a>.</p>
</div>
<div id="ref-BrownHP1gfunctionrequired2010">
<p>Brown, Jeremy P., Jörn Bullwinkel, Bettina Baron-Lühr, Mustafa Billur, Philipp Schneider, Heinz Winking, and Prim B. Singh. 2010. “HP1γ Function Is Required for Male Germ Cell Survival and Spermatogenesis.” <em>Epigenetics &amp; Chromatin</em> 3: 9. doi:<a href="https://doi.org/10.1186/1756-8935-3-9">10.1186/1756-8935-3-9</a>.</p>
</div>
<div id="ref-LeonardAuroraAHP1gpathway2015">
<p>Leonard, Phoebe H., Adrienne Grzenda, Angela Mathison, Dean E. Morbeck, Jolene R. Fredrickson, Thiago M. de Assuncao, Trace Christensen, et al. 2015. “The Aurora A-HP1γ Pathway Regulates Gene Expression and Mitosis in Cells from the Sperm Lineage.” <em>BMC Developmental Biology</em> 15 (1): 23. doi:<a href="https://doi.org/10.1186/s12861-015-0073-x">10.1186/s12861-015-0073-x</a>.</p>
</div>
<div id="ref-LiuChromatinproteinHP12014">
<p>Liu, Xing, Zhenwei Song, Yuda Huo, Jiahai Zhang, Tongge Zhu, Jianyu Wang, Xuannv Zhao, et al. 2014. “Chromatin Protein Hp1 Interacts with the Mitotic Regulator Borealin Protein and Specifies the Centromere Localization of the Chromosomal Passenger Complex.” <em>The Journal of Biological Chemistry</em> 289 (30): 20638–49. doi:<a href="https://doi.org/10.1074/jbc.M114.572842">10.1074/jbc.M114.572842</a>.</p>
</div>
<div id="ref-KalousiNuclearOncogeneSET2015a">
<p>Kalousi, Alkmini, Anne-Sophie Hoffbeck, Platonas N. Selemenakis, Jordan Pinder, Kienan I. Savage, Kum Kum Khanna, Laurent Brino, Graham Dellaire, Vassilis G. Gorgoulis, and Evi Soutoglou. 2015. “The Nuclear Oncogene SET Controls DNA Repair by Kap1 and Hp1 Retention to Chromatin.” <em>Cell Reports</em> 11 (1): 149–63. doi:<a href="https://doi.org/10.1016/j.celrep.2015.03.005">10.1016/j.celrep.2015.03.005</a>.</p>
</div>
<div id="ref-SoriaDifferentialcontributionHP12013a">
<p>Soria, Gaston, and Genevi‘eve Almouzni. 2013. “Differential Contribution of Hp1 Proteins to DNA End Resection and Homology-Directed Repair.” <em>Cell Cycle</em> 12 (3): 422–29. doi:<a href="https://doi.org/10.4161/cc.23215">10.4161/cc.23215</a>.</p>
</div>
<div id="ref-LuijsterburgHeterochromatinproteinrecruited2009a">
<p>Luijsterburg, Martijn S., Christoffel Dinant, Hannes Lans, Jan Stap, Elzbieta Wiernasz, Saskia Lagerwerf, Daniël O. Warmerdam, et al. 2009. “Heterochromatin Protein 1 Is Recruited to Various Types of DNA Damage.” <em>The Journal of Cell Biology</em> 185 (4): 577–86. doi:<a href="https://doi.org/10.1083/jcb.200810035">10.1083/jcb.200810035</a>.</p>
</div>
<div id="ref-DialynasLinkingHeterochromatinProtein2008">
<p>Dialynas, George K., Michael W. Vitalini, and Lori L. Wallrath. 2008. “Linking Heterochromatin Protein 1 (Hp1) to Cancer Progression.” <em>Mutation Research/Fundamental and Molecular Mechanisms of Mutagenesis</em>, Epigenetics of development and human disease, 647 (1): 13–20. doi:<a href="https://doi.org/10.1016/j.mrfmmm.2008.09.007">10.1016/j.mrfmmm.2008.09.007</a>.</p>
</div>
<div id="ref-LeungRegulationDNAmethylation2014a">
<p>Leung, Danny, Tingting Du, Ulrich Wagner, Wei Xie, Ah Young Lee, Preeti Goyal, Yujing Li, et al. 2014. “Regulation of DNA Methylation Turnover at LTR Retrotransposons and Imprinted Loci by the Histone Methyltransferase Setdb1.” <em>Proceedings of the National Academy of Sciences</em> 111 (18): 6690–5. doi:<a href="https://doi.org/10.1073/pnas.1322273111">10.1073/pnas.1322273111</a>.</p>
</div>
<div id="ref-Quennevilleembryonicstemcells2011">
<p>Quenneville, Simon, Gaetano Verde, Andrea Corsinotti, Adamandia Kapopoulou, Johan Jakobsson, Sandra Offner, Ilaria Baglivo, Paolo V. Pedone, Giovanna Grimaldi, and Andrea Riccio. 2011. “In Embryonic Stem Cells, Zfp57/Kap1 Recognize a Methylated Hexanucleotide to Affect Chromatin and DNA Methylation of Imprinting Control Regions.” <em>Molecular Cell</em> 44 (3): 361–72.</p>
</div>
<div id="ref-BilodeauSetDB1contributesrepression2009">
<p>Bilodeau, Steve, Michael H. Kagey, Garrett M. Frampton, Peter B. Rahl, and Richard A. Young. 2009. “SetDB1 Contributes to Repression of Genes Encoding Developmental Regulators and Maintenance of ES Cell State.” <em>Genes &amp; Development</em> 23 (21): 2484–9.</p>
</div>
<div id="ref-KarimiDNAMethylationSETDB12011">
<p>Karimi, Mohammad M., Preeti Goyal, Irina A. Maksakova, Misha Bilenky, Danny Leung, Jie Xin Tang, Yoichi Shinkai, et al. 2011. “DNA Methylation and Setdb1/H3K9me3 Regulate Predominantly Distinct Sets of Genes, Retroelements, and Chimeric Transcripts in mESCs.” <em>Cell Stem Cell</em> 8 (6): 676–87. doi:<a href="https://doi.org/10.1016/j.stem.2011.04.004">10.1016/j.stem.2011.04.004</a>.</p>
</div>
<div id="ref-Raurell-VilaHP1isoformspecificfeedback2017">
<p>Raurell-Vila, Helena, Laia Bosch-Presegue, Jessica Gonzalez, Noriko Kane-Goldsmith, Carmen Casal, Jeremy P. Brown, Anna Marazuela-Duque, Prim B. Singh, Lourdes Serrano, and Alejandro Vaquero. 2017. “An Hp1 Isoform-Specific Feedback Mechanism Regulates Suv39h1 Activity Under Stress Conditions.” <em>Epigenetics</em> 12 (2): 166–75. doi:<a href="https://doi.org/10.1080/15592294.2016.1278096">10.1080/15592294.2016.1278096</a>.</p>
</div>
<div id="ref-AucottHP1requireddevelopment2008">
<p>Aucott, R., J. Bullwinkel, Y. Yu, W. Shi, M. Billur, J. P. Brown, U. Menzel, et al. 2008. “HP1- Is Required for Development of the Cerebral Neocortex and Neuromuscular Junctions.” <em>The Journal of Cell Biology</em> 183 (4): 597–606. doi:<a href="https://doi.org/10.1083/jcb.200804041">10.1083/jcb.200804041</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="acknowledgements.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="2-results.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"download": null,
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
