<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Part 2 Results | HP1 Deficiency Results in Accelerated Neurodegeneration</title>
  <meta name="description" content="Part 2 Results | HP1 Deficiency Results in Accelerated Neurodegeneration">
  <meta name="generator" content="bookdown  and GitBook 2.6.7">

  <meta property="og:title" content="Part 2 Results | HP1 Deficiency Results in Accelerated Neurodegeneration" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Part 2 Results | HP1 Deficiency Results in Accelerated Neurodegeneration" />
  
  
  

<meta name="author" content="Andrew G. Newman">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="1-introduction.html">
<link rel="next" href="3-discussion.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />







<script src="libs/kePrint-0.0.1/kePrint.js"></script>


<style type="text/css">
div.sourceCode { overflow-x: auto; }
table.sourceCode, tr.sourceCode, td.lineNumbers, td.sourceCode {
  margin: 0; padding: 0; vertical-align: baseline; border: none; }
table.sourceCode { width: 100%; line-height: 100%; }
td.lineNumbers { text-align: right; padding-right: 4px; padding-left: 4px; color: #aaaaaa; border-right: 1px solid #aaaaaa; }
td.sourceCode { padding-left: 5px; }
code > span.kw { color: #007020; font-weight: bold; } /* Keyword */
code > span.dt { color: #902000; } /* DataType */
code > span.dv { color: #40a070; } /* DecVal */
code > span.bn { color: #40a070; } /* BaseN */
code > span.fl { color: #40a070; } /* Float */
code > span.ch { color: #4070a0; } /* Char */
code > span.st { color: #4070a0; } /* String */
code > span.co { color: #60a0b0; font-style: italic; } /* Comment */
code > span.ot { color: #007020; } /* Other */
code > span.al { color: #ff0000; font-weight: bold; } /* Alert */
code > span.fu { color: #06287e; } /* Function */
code > span.er { color: #ff0000; font-weight: bold; } /* Error */
code > span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
code > span.cn { color: #880000; } /* Constant */
code > span.sc { color: #4070a0; } /* SpecialChar */
code > span.vs { color: #4070a0; } /* VerbatimString */
code > span.ss { color: #bb6688; } /* SpecialString */
code > span.im { } /* Import */
code > span.va { color: #19177c; } /* Variable */
code > span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code > span.op { color: #666666; } /* Operator */
code > span.bu { } /* BuiltIn */
code > span.ex { } /* Extension */
code > span.pp { color: #bc7a00; } /* Preprocessor */
code > span.at { color: #7d9029; } /* Attribute */
code > span.do { color: #ba2121; font-style: italic; } /* Documentation */
code > span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code > span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code > span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
</style>

</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="preface.html"><a href="preface.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="" data-path="contribution-of-authors.html"><a href="contribution-of-authors.html"><i class="fa fa-check"></i>Contribution of Authors</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="1" data-path="1-introduction.html"><a href="1-introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="1-introduction.html"><a href="1-introduction.html#epigenetics"><i class="fa fa-check"></i><b>1.1</b> Epigenetics</a><ul>
<li><a href="1-introduction.html#dna-modifications"><em>DNA Modifications</em></a></li>
<li><a href="1-introduction.html#chromatin"><em>Chromatin</em></a></li>
<li><a href="1-introduction.html#chromatin-remodelling"><em>Chromatin Remodelling</em></a></li>
<li><a href="1-introduction.html#krab-zfps-transposable-elements-and-exaptation"><em>KRAB-ZFPs, Transposable Elements and Exaptation</em></a></li>
<li><a href="1-introduction.html#heterochromatinization-and-hp1"><em>Heterochromatinization and HP1</em></a></li>
<li><a href="1-introduction.html#non-coding-rna"><em>Non-Coding RNA</em></a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="1-introduction.html"><a href="1-introduction.html#the-heterochromatin-loss-model-of-aging"><i class="fa fa-check"></i><b>1.2</b> The Heterochromatin Loss Model of Aging</a></li>
<li class="chapter" data-level="1.3" data-path="1-introduction.html"><a href="1-introduction.html#the-aging-neuronal-epigenome"><i class="fa fa-check"></i><b>1.3</b> The Aging Neuronal Epigenome</a><ul>
<li class="chapter" data-level="1.3.1" data-path="1-introduction.html"><a href="1-introduction.html#DNAging"><i class="fa fa-check"></i><b>1.3.1</b> <em>DNA Methylation and Hydroxymethylation in Aged Neurons</em></a></li>
<li class="chapter" data-level="1.3.2" data-path="1-introduction.html"><a href="1-introduction.html#chraging"><i class="fa fa-check"></i><b>1.3.2</b> <em>Chromatin in the Aging Brain</em></a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="1-introduction.html"><a href="1-introduction.html#objective"><i class="fa fa-check"></i><b>1.4</b> Objective</a></li>
<li class="chapter" data-level="1.5" data-path="1-introduction.html"><a href="1-introduction.html#hp1-proteins---a-background"><i class="fa fa-check"></i><b>1.5</b> HP1 Proteins - A Background</a><ul>
<li><a href="1-introduction.html#the-abgs-of-hp1-proteins">The <em>ABG’s</em> of HP1 Proteins</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-results.html"><a href="2-results.html"><i class="fa fa-check"></i><b>2</b> Results</a><ul>
<li class="chapter" data-level="2.1" data-path="2-results.html"><a href="2-results.html#HP1aT"><i class="fa fa-check"></i><b>2.1</b> HP1FEC, HP1TKO and HP1<span class="math inline">\(\alpha\)</span>T</a></li>
<li class="chapter" data-level="2.2" data-path="2-results.html"><a href="2-results.html#expression-of-hp1-proteins-in-the-brain"><i class="fa fa-check"></i><b>2.2</b> Expression of HP1 Proteins in the Brain</a></li>
<li class="chapter" data-level="2.3" data-path="2-results.html"><a href="2-results.html#mosaic-deletion-of-hp1beta-and-hp1gamma-from-cortical-progenitors"><i class="fa fa-check"></i><b>2.3</b> Mosaic Deletion of HP1<span class="math inline">\(\beta\)</span> and HP1<span class="math inline">\(\gamma\)</span> from Cortical Progenitors</a></li>
<li class="chapter" data-level="2.4" data-path="2-results.html"><a href="2-results.html#deletion-of-hp1beta-hp1gamma-from-the-emx1-lineage."><i class="fa fa-check"></i><b>2.4</b> Deletion of <em>HP1<span class="math inline">\(\beta\)</span></em> &amp; <em>HP1<span class="math inline">\(\gamma\)</span></em> from the <em>Emx1</em>-lineage.</a><ul>
<li class="chapter" data-level="2.4.1" data-path="2-results.html"><a href="2-results.html#cell-fate-in-cortical-neurons"><i class="fa fa-check"></i><b>2.4.1</b> Cell Fate in Cortical Neurons</a></li>
<li class="chapter" data-level="2.4.2" data-path="2-results.html"><a href="2-results.html#imprint"><i class="fa fa-check"></i><b>2.4.2</b> Imprinted Loci</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="2-results.html"><a href="2-results.html#perturbed-regulation-of-retroelements-in-hp1-mutants"><i class="fa fa-check"></i><b>2.5</b> Perturbed Regulation of Retroelements in HP1 Mutants</a><ul>
<li class="chapter" data-level="2.5.1" data-path="2-results.html"><a href="2-results.html#line1"><i class="fa fa-check"></i><b>2.5.1</b> LINE1</a></li>
<li class="chapter" data-level="2.5.2" data-path="2-results.html"><a href="2-results.html#class-iii-ervs"><i class="fa fa-check"></i><b>2.5.2</b> Class III ERVs</a></li>
<li class="chapter" data-level="2.5.3" data-path="2-results.html"><a href="2-results.html#class-ii-ervs"><i class="fa fa-check"></i><b>2.5.3</b> Class II ERVs</a></li>
<li class="chapter" data-level="2.5.4" data-path="2-results.html"><a href="2-results.html#de-novo-retrotransposition"><i class="fa fa-check"></i><b>2.5.4</b> <em>De Novo</em> Retrotransposition</a></li>
</ul></li>
<li class="chapter" data-level="2.6" data-path="2-results.html"><a href="2-results.html#sustained-hp1betagamma-deficiency-adversely-affects-neuronal-complexity-function-and-cognition"><i class="fa fa-check"></i><b>2.6</b> Sustained HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> Deficiency Adversely Affects Neuronal Complexity, Function and Cognition</a><ul>
<li class="chapter" data-level="2.6.1" data-path="2-results.html"><a href="2-results.html#behavior"><i class="fa fa-check"></i><b>2.6.1</b> Behavior</a></li>
<li class="chapter" data-level="2.6.2" data-path="2-results.html"><a href="2-results.html#MRIresults"><i class="fa fa-check"></i><b>2.6.2</b> Structure, Volumetrics and Histology</a></li>
</ul></li>
<li class="chapter" data-level="2.7" data-path="2-results.html"><a href="2-results.html#hp1beta-and-hp1gamma-synergistically-contribute-to-repeat-silencing"><i class="fa fa-check"></i><b>2.7</b> HP1<span class="math inline">\(\beta\)</span> and HP1<span class="math inline">\(\gamma\)</span> Synergistically Contribute to Repeat Silencing</a><ul>
<li class="chapter" data-level="2.7.1" data-path="2-results.html"><a href="2-results.html#hp1gamma-is-required-for-h4k20me3-deposition-in-neurons"><i class="fa fa-check"></i><b>2.7.1</b> HP1<span class="math inline">\(\gamma\)</span> is Required for H4K20me3 Deposition in Neurons</a></li>
<li class="chapter" data-level="2.7.2" data-path="2-results.html"><a href="2-results.html#hp1gamma-binds-suv420h2-via-the-chromoshadow-domain"><i class="fa fa-check"></i><b>2.7.2</b> HP1<span class="math inline">\(\gamma\)</span> binds SUV420H2 via the Chromoshadow Domain</a></li>
<li class="chapter" data-level="2.7.3" data-path="2-results.html"><a href="2-results.html#re-addition-of-hp1gamma-can-restore-h4k20me3-in-vivo"><i class="fa fa-check"></i><b>2.7.3</b> Re-addition of HP1<span class="math inline">\(\gamma\)</span> can restore H4K20me3 In Vivo</a></li>
<li class="chapter" data-level="2.7.4" data-path="2-results.html"><a href="2-results.html#h4k20me3-loss-is-correlated-with-iap-de-repression"><i class="fa fa-check"></i><b>2.7.4</b> H4K20me3 Loss is Correlated with IAP De-repression</a></li>
<li class="chapter" data-level="2.7.5" data-path="2-results.html"><a href="2-results.html#kap1-h3k9me3-and-h4k20me3-are-lost-at-iap-elements-in-hp1ctko-es-cells"><i class="fa fa-check"></i><b>2.7.5</b> KAP1, H3K9me3 and H4K20me3 are lost at IAP Elements in HP1cTKO ES Cells</a></li>
<li class="chapter" data-level="2.7.6" data-path="2-results.html"><a href="2-results.html#dna-methylation-is-affected-following-loss-of-hp1"><i class="fa fa-check"></i><b>2.7.6</b> DNA Methylation is affected Following loss of HP1</a></li>
<li><a href="2-results.html#kap1-is-lost-from-icrs-in-hp1ctko-cells"><em>KAP1 is lost from ICRs in HP1cTKO Cells</em></a></li>
</ul></li>
<li class="chapter" data-level="2.8" data-path="2-results.html"><a href="2-results.html#RNAresults"><i class="fa fa-check"></i><b>2.8</b> Transcriptional Dysregulation following HP1 Deficiency Drives UPR and Inflammatory Responses</a><ul>
<li class="chapter" data-level="2.8.1" data-path="2-results.html"><a href="2-results.html#genotype-related-changes"><i class="fa fa-check"></i><b>2.8.1</b> Genotype Related Changes</a></li>
<li class="chapter" data-level="2.8.2" data-path="2-results.html"><a href="2-results.html#longitudinal-changes-to-gene-expression"><i class="fa fa-check"></i><b>2.8.2</b> Longitudinal Changes To Gene Expression</a></li>
<li class="chapter" data-level="2.8.3" data-path="2-results.html"><a href="2-results.html#pathway-analysis"><i class="fa fa-check"></i><b>2.8.3</b> Pathway Analysis</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-discussion.html"><a href="3-discussion.html"><i class="fa fa-check"></i><b>3</b> Discussion</a><ul>
<li class="chapter" data-level="3.1" data-path="3-discussion.html"><a href="3-discussion.html#phenotypes"><i class="fa fa-check"></i><b>3.1</b> Phenotypes</a><ul>
<li><a href="3-discussion.html#structural-changes-in-the-hippocampus"><em>Structural Changes in the Hippocampus</em></a></li>
<li><a href="3-discussion.html#reduction-in-cortical-volume"><em>Reduction in Cortical Volume</em></a></li>
<li><a href="3-discussion.html#behavior-and-impairments-in-learning-memory"><em>Behavior and Impairments in Learning &amp; Memory</em></a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="3-discussion.html"><a href="3-discussion.html#hp1-dependent-epigenetic-systems"><i class="fa fa-check"></i><b>3.2</b> HP1-dependent Epigenetic Systems</a><ul>
<li><a href="3-discussion.html#hp1gamma"><em>HP1<span class="math inline">\(\gamma\)</span></em></a></li>
<li><a href="3-discussion.html#hp1beta"><em>HP1<span class="math inline">\(\beta\)</span></em></a></li>
<li><a href="3-discussion.html#hp1beta-x-hp1gamma"><em>HP1<span class="math inline">\(\beta\)</span></em> x <em>HP1<span class="math inline">\(\gamma\)</span></em></a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="3-discussion.html"><a href="3-discussion.html#concordance-with-aging-and-disease"><i class="fa fa-check"></i><b>3.3</b> Concordance with Aging and Disease</a><ul>
<li><a href="3-discussion.html#complement-in-cns-aging-and-disease"><em>Complement in CNS Aging and Disease</em></a></li>
<li><a href="3-discussion.html#a-pathway-to-complement"><em>A Pathway to Complement</em></a></li>
<li><a href="3-discussion.html#unfolded-protein-response-upr"><em>Unfolded Protein Response (UPR)</em></a></li>
<li><a href="3-discussion.html#viruses-transactivation-and-neurodegeneration"><em>Viruses, Transactivation and Neurodegeneration</em></a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="3-discussion.html"><a href="3-discussion.html#concluding-remarks"><i class="fa fa-check"></i><b>3.4</b> Concluding Remarks</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-methods.html"><a href="4-methods.html"><i class="fa fa-check"></i><b>4</b> Methods</a><ul>
<li class="chapter" data-level="4.1" data-path="4-methods.html"><a href="4-methods.html#mouse-lines"><i class="fa fa-check"></i><b>4.1</b> Mouse Lines</a><ul>
<li class="chapter" data-level="4.1.1" data-path="4-methods.html"><a href="4-methods.html#HP1anull"><i class="fa fa-check"></i><b>4.1.1</b> Generation of HP1<span class="math inline">\(\alpha\)</span>KO and HP1<span class="math inline">\(\alpha\)</span>T</a></li>
<li class="chapter" data-level="4.1.2" data-path="4-methods.html"><a href="4-methods.html#HP1Bgtarget"><i class="fa fa-check"></i><b>4.1.2</b> Generation of Floxed HP1<span class="math inline">\(\beta\)</span> &amp; Floxed HP1<span class="math inline">\(\gamma\)</span></a></li>
<li class="chapter" data-level="4.1.3" data-path="4-methods.html"><a href="4-methods.html#genotyping"><i class="fa fa-check"></i><b>4.1.3</b> Genotyping</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="4-methods.html"><a href="4-methods.html#cloning"><i class="fa fa-check"></i><b>4.2</b> Cloning</a><ul>
<li class="chapter" data-level="4.2.1" data-path="4-methods.html"><a href="4-methods.html#rna-isolation-cdna-generation-rnaseq"><i class="fa fa-check"></i><b>4.2.1</b> RNA Isolation, cDNA Generation, &amp; RNAseq</a></li>
<li class="chapter" data-level="4.2.2" data-path="4-methods.html"><a href="4-methods.html#cloning-of-rna-probes-for-in-situ-hybridization"><i class="fa fa-check"></i><b>4.2.2</b> Cloning of RNA Probes for <em>In-Situ Hybridization</em></a></li>
<li class="chapter" data-level="4.2.3" data-path="4-methods.html"><a href="4-methods.html#cloning-of-expression-constructs"><i class="fa fa-check"></i><b>4.2.3</b> Cloning of Expression Constructs</a></li>
<li class="chapter" data-level="4.2.4" data-path="4-methods.html"><a href="4-methods.html#site-directed-mutagenesis"><i class="fa fa-check"></i><b>4.2.4</b> Site Directed Mutagenesis</a></li>
<li class="chapter" data-level="4.2.5" data-path="4-methods.html"><a href="4-methods.html#transformation"><i class="fa fa-check"></i><b>4.2.5</b> Transformation</a></li>
<li class="chapter" data-level="4.2.6" data-path="4-methods.html"><a href="4-methods.html#clonal-selection-plasmid-production"><i class="fa fa-check"></i><b>4.2.6</b> Clonal Selection &amp; Plasmid Production</a></li>
<li class="chapter" data-level="4.2.7" data-path="4-methods.html"><a href="4-methods.html#in-vitro-transcription-ivt"><i class="fa fa-check"></i><b>4.2.7</b> In Vitro Transcription (IVT)</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="4-methods.html"><a href="4-methods.html#cell-culture"><i class="fa fa-check"></i><b>4.3</b> Cell Culture</a><ul>
<li class="chapter" data-level="4.3.1" data-path="4-methods.html"><a href="4-methods.html#primary-neuronal-culture"><i class="fa fa-check"></i><b>4.3.1</b> Primary Neuronal Culture</a></li>
<li class="chapter" data-level="4.3.2" data-path="4-methods.html"><a href="4-methods.html#hek293t-culture-transfection"><i class="fa fa-check"></i><b>4.3.2</b> HEK293T Culture &amp; Transfection</a></li>
<li class="chapter" data-level="4.3.3" data-path="4-methods.html"><a href="4-methods.html#co-immunoprecipitation-co-ip-and-western-blot-wb"><i class="fa fa-check"></i><b>4.3.3</b> Co-Immunoprecipitation (Co-IP) and Western Blot (WB)</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="4-methods.html"><a href="4-methods.html#in-utero-electroporation-tissue-processing-and-histology"><i class="fa fa-check"></i><b>4.4</b> In Utero Electroporation, Tissue Processing and Histology</a><ul>
<li class="chapter" data-level="4.4.1" data-path="4-methods.html"><a href="4-methods.html#IUEmethod"><i class="fa fa-check"></i><b>4.4.1</b> In Utero Electroporation (IUE)</a></li>
<li class="chapter" data-level="4.4.2" data-path="4-methods.html"><a href="4-methods.html#tissue-fixation-and-perfusion"><i class="fa fa-check"></i><b>4.4.2</b> Tissue Fixation and Perfusion</a></li>
<li class="chapter" data-level="4.4.3" data-path="4-methods.html"><a href="4-methods.html#cryosectioning"><i class="fa fa-check"></i><b>4.4.3</b> Cryosectioning</a></li>
<li class="chapter" data-level="4.4.4" data-path="4-methods.html"><a href="4-methods.html#immuno-histochemistry-ihc"><i class="fa fa-check"></i><b>4.4.4</b> Immuno Histochemistry (IHC)</a></li>
<li class="chapter" data-level="4.4.5" data-path="4-methods.html"><a href="4-methods.html#quantification-of-cells-in-the-dentate-gyrus"><i class="fa fa-check"></i><b>4.4.5</b> Quantification of Cells in the Dentate Gyrus</a></li>
<li class="chapter" data-level="4.4.6" data-path="4-methods.html"><a href="4-methods.html#in-situ-hybridization-ish"><i class="fa fa-check"></i><b>4.4.6</b> In Situ Hybridization (ISH)</a></li>
<li class="chapter" data-level="4.4.7" data-path="4-methods.html"><a href="4-methods.html#paraffin-sectioning"><i class="fa fa-check"></i><b>4.4.7</b> Paraffin Sectioning</a></li>
<li class="chapter" data-level="4.4.8" data-path="4-methods.html"><a href="4-methods.html#nissl"><i class="fa fa-check"></i><b>4.4.8</b> Nissl Stain</a></li>
<li class="chapter" data-level="4.4.9" data-path="4-methods.html"><a href="4-methods.html#golgi-impregnation-and-dendritic-complexity-analysis"><i class="fa fa-check"></i><b>4.4.9</b> Golgi Impregnation and Dendritic Complexity Analysis</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="4-methods.html"><a href="4-methods.html#behavioral-experiments"><i class="fa fa-check"></i><b>4.5</b> Behavioral Experiments</a><ul>
<li class="chapter" data-level="4.5.1" data-path="4-methods.html"><a href="4-methods.html#open-field-activity-1"><i class="fa fa-check"></i><b>4.5.1</b> Open Field Activity</a></li>
<li class="chapter" data-level="4.5.2" data-path="4-methods.html"><a href="4-methods.html#paired-pulse-inhibition-1"><i class="fa fa-check"></i><b>4.5.2</b> Paired-Pulse Inhibition</a></li>
<li class="chapter" data-level="4.5.3" data-path="4-methods.html"><a href="4-methods.html#context-cued-fear-conditioning"><i class="fa fa-check"></i><b>4.5.3</b> Context-Cued Fear Conditioning</a></li>
<li class="chapter" data-level="4.5.4" data-path="4-methods.html"><a href="4-methods.html#Barnesmeth"><i class="fa fa-check"></i><b>4.5.4</b> Barnes Maze</a></li>
<li class="chapter" data-level="4.5.5" data-path="4-methods.html"><a href="4-methods.html#social-activity-monitor-sam-1"><i class="fa fa-check"></i><b>4.5.5</b> Social Activity Monitor (SAM)</a></li>
<li class="chapter" data-level="4.5.6" data-path="4-methods.html"><a href="4-methods.html#micro-behaviors-homecagescan"><i class="fa fa-check"></i><b>4.5.6</b> Micro Behaviors (HomeCageScan)</a></li>
<li class="chapter" data-level="4.5.7" data-path="4-methods.html"><a href="4-methods.html#nestcomethod"><i class="fa fa-check"></i><b>4.5.7</b> Nest Construction</a></li>
</ul></li>
<li class="chapter" data-level="4.6" data-path="4-methods.html"><a href="4-methods.html#magnetic-resonance-imaging-mri-and-volumetrics"><i class="fa fa-check"></i><b>4.6</b> Magnetic Resonance Imaging (MRI) and Volumetrics</a></li>
<li class="chapter" data-level="4.7" data-path="4-methods.html"><a href="4-methods.html#reduced-read-bisulfite-sequencing-rrbseq"><i class="fa fa-check"></i><b>4.7</b> Reduced Read Bisulfite Sequencing (RRBSeq)</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html"><i class="fa fa-check"></i><b>5</b> Supplemental Data</a><ul>
<li class="chapter" data-level="5.1" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html#Behavsupplement"><i class="fa fa-check"></i><b>5.1</b> Behavior</a><ul>
<li class="chapter" data-level="5.1.1" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html#homecagescan-micro-behaviors"><i class="fa fa-check"></i><b>5.1.1</b> HomeCageScan Micro Behaviors</a></li>
<li class="chapter" data-level="5.1.2" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html#context-cued-fear-conditioning-1"><i class="fa fa-check"></i><b>5.1.2</b> Context-Cued Fear Conditioning</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html#DE"><i class="fa fa-check"></i><b>5.2</b> Differential Gene Expression</a><ul>
<li class="chapter" data-level="5.2.1" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html#differentially-expressed-genes-in-aged-hp1betako"><i class="fa fa-check"></i><b>5.2.1</b> Differentially Expressed Genes in Aged HP1<span class="math inline">\(\beta\)</span>KO</a></li>
<li class="chapter" data-level="5.2.2" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html#inflam"><i class="fa fa-check"></i><b>5.2.2</b> Genes Related to Anti-viral and Interferon Response in HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO</a></li>
<li class="chapter" data-level="5.2.3" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html#genomic-hotspots-sensitive-to-hp1-deficiency-in-the-hippocampus"><i class="fa fa-check"></i><b>5.2.3</b> <em>Genomic Hotspots Sensitive to HP1 Deficiency in the Hippocampus</em></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html"><i class="fa fa-check"></i><b>6</b> Bioinformatics &amp; Data Processing</a><ul>
<li class="chapter" data-level="6.1" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#software-environment-management"><i class="fa fa-check"></i><b>6.1</b> Software Environment Management</a></li>
<li class="chapter" data-level="6.2" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#snakemake-jobs"><i class="fa fa-check"></i><b>6.2</b> Snakemake Jobs</a></li>
<li class="chapter" data-level="6.3" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#tetranscripts-pipeline"><i class="fa fa-check"></i><b>6.3</b> TETranscripts Pipeline</a></li>
<li class="chapter" data-level="6.4" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#differential-gene-expression-using-edger"><i class="fa fa-check"></i><b>6.4</b> Differential Gene Expression Using EdgeR</a></li>
<li class="chapter" data-level="6.5" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#deeptools-pipeline"><i class="fa fa-check"></i><b>6.5</b> Deeptools Pipeline</a></li>
<li class="chapter" data-level="6.6" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#lions-pipeline"><i class="fa fa-check"></i><b>6.6</b> LIONS Pipeline</a></li>
<li class="chapter" data-level="6.7" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#analysis-of-reduced-read-bisulphite-sequencing-rrbs"><i class="fa fa-check"></i><b>6.7</b> Analysis of Reduced-Read Bisulphite Sequencing (RRBS)</a><ul>
<li class="chapter" data-level="6.7.1" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#methylkit-and-circos-plot"><i class="fa fa-check"></i><b>6.7.1</b> Methylkit and Circos Plot</a></li>
<li class="chapter" data-level="6.7.2" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#annotated-ideogram"><i class="fa fa-check"></i><b>6.7.2</b> Annotated Ideogram</a></li>
</ul></li>
<li class="chapter" data-level="6.8" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#about-this-thesis"><i class="fa fa-check"></i><b>6.8</b> About this Thesis</a><ul>
<li class="chapter" data-level="6.8.1" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#session-info"><i class="fa fa-check"></i><b>6.8.1</b> Session Info</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7" data-path="7-references.html"><a href="7-references.html"><i class="fa fa-check"></i><b>7</b> References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">HP1 Deficiency Results in Accelerated Neurodegeneration</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="results" class="section level1">
<h1><span class="header-section-number">Part 2</span> Results</h1>
<div id="HP1aT" class="section level2">
<h2><span class="header-section-number">2.1</span> HP1FEC, HP1TKO and HP1<span class="math inline">\(\alpha\)</span>T</h2>
<p>The aim to generate HP1TKO mice from the HP1FEC line, where HP1<span class="math inline">\(\alpha\)</span> is constitutively deleted (null allele, Methods <a href="4-methods.html#HP1anull">4.1.1</a>) and HP1<span class="math inline">\(\beta\)</span> and HP1<span class="math inline">\(\gamma\)</span> are conditionally deleted in <em>Emx1</em>-lineage pyramidal neurons (Methods <a href="4-methods.html#HP1Bgtarget">4.1.2</a>), could not be realized. Over the course of one year of crossing and re-crossing, we found that HP1<span class="math inline">\(\alpha\)</span> null allele homozygosity did not follow mendelian ratios and was extremely rare. Of the few ‘HP1<span class="math inline">\(\alpha\)</span> null’ brains that could be obtained, positive staining could be observed in euchromatin using an anti-HP1<span class="math inline">\(\alpha\)</span> antibody. It was then necessary to check HP1<span class="math inline">\(\alpha\)</span> protein expression in this mutant.</p>
<div class="figure" style="text-align: center"><span id="fig:HP1aTmap"></span>
<img src="figure/results/HP1aTMap.png" alt="HP1$\alpha$ and HP1$\alpha$T Transcript Maps.  (A) Wild-type, full length HP1$\alpha$ transcript annotated with major domains and point mutations. (B) The constructed HP1$\alpha$ null allele gives rise to the truncated HP1$\alpha$T transcript that lacks exon II.  In-frame transcription is initiated at either of two ATG codons found inside exon III.  The resulting HP1$\alpha$T protein contains full length hinge and chromoshadow domains but does not have a functional chromodomain, which runs around 12 kDa, compared to the wild-type HP1$\alpha$ protein of 22 kDa found in C57BL/6J mice (C)." width="100%"   />
<p class="caption">
Figure 2.1: HP1<span class="math inline">\(\alpha\)</span> and HP1<span class="math inline">\(\alpha\)</span>T Transcript Maps. (A) Wild-type, full length HP1<span class="math inline">\(\alpha\)</span> transcript annotated with major domains and point mutations. (B) The constructed HP1<span class="math inline">\(\alpha\)</span> null allele gives rise to the truncated HP1<span class="math inline">\(\alpha\)</span>T transcript that lacks exon II. In-frame transcription is initiated at either of two ATG codons found inside exon III. The resulting HP1<span class="math inline">\(\alpha\)</span>T protein contains full length hinge and chromoshadow domains but does not have a functional chromodomain, which runs around 12 kDa, compared to the wild-type HP1<span class="math inline">\(\alpha\)</span> protein of 22 kDa found in C57BL/6J mice (C).
</p>
</div>

<p>While no HP1<span class="math inline">\(\alpha\)</span> is transcribed from the HP1<span class="math inline">\(\alpha\)</span> NeO allele containing the STOP cassette (Methods <a href="4-methods.html#HP1anull">4.1.1</a>), a truncated ~12 kDa protein could be recognized in brains with the HP1<span class="math inline">\(\alpha\)</span> null allele (Figure <a href="2-results.html#fig:HP1aTmap">2.1</a>C). To identify this protein, mRNA was isolated from these animals and reverse transcribed into cDNA. Using primers specific to the 5’UTR and 3’UTR of HP1<span class="math inline">\(\alpha\)</span>, I could amplify a truncated 548 bp isoform of HP1<span class="math inline">\(\alpha\)</span> being transcribed from the HP1<span class="math inline">\(\alpha\)</span> ‘null’ allele, which was dubbed HP1<span class="math inline">\(\alpha\)</span>T. Sequencing of this transcript confimed that an in-frame coding sequence could be initiated from either of two ATG codons found inside exon III (Figure <a href="2-results.html#fig:HP1aTmap">2.1</a>B).</p>
<p>Given HP1<span class="math inline">\(\alpha\)</span>T retains hinge and CSD domains but lacks a functional chromodomain, I suspected it could still heterodimerize with HP1<span class="math inline">\(\beta\)</span> and HP1<span class="math inline">\(\gamma\)</span> interact with other proteins but would no longer bind to H3K9me3. To test this, I cloned a c-terminal HP1<span class="math inline">\(\alpha\)</span>T-GFP fusion protein for expression in cells to observe where the protein is localized in the nucleus. Similar to mutations that inhibit H3K9me3 binding (HP1<span class="math inline">\(\alpha\)</span>F44E) or dimerization (HP1<span class="math inline">\(\alpha\)</span>I165E), HP1<span class="math inline">\(\alpha\)</span>T does not localize to H3K9me3 positive heterochromatin (Figure <a href="2-results.html#fig:HP1amut">2.2</a>).</p>
<div class="figure" style="text-align: center"><span id="fig:HP1amut"></span>
<img src="figure/results/HP1amut.png" alt="Sub-Nuclear Localization of HP1$\alpha$ Mutants.  C-terminal HP1$\alpha$-GFP fusion proteins enable clean visualization of sub-nuclear localization.  In ES cells, wild-type HP1$\alpha$ localizes to both euchromatic and H3K9me3+ heterochromatin. HP1$\alpha$F44E, HP1$\alpha$I165E, and HP1$\alpha$T are not found at H3K9me3 chromocenters." width="100%"   />
<p class="caption">
Figure 2.2: Sub-Nuclear Localization of HP1<span class="math inline">\(\alpha\)</span> Mutants. C-terminal HP1<span class="math inline">\(\alpha\)</span>-GFP fusion proteins enable clean visualization of sub-nuclear localization. In ES cells, wild-type HP1<span class="math inline">\(\alpha\)</span> localizes to both euchromatic and H3K9me3+ heterochromatin. HP1<span class="math inline">\(\alpha\)</span>F44E, HP1<span class="math inline">\(\alpha\)</span>I165E, and HP1<span class="math inline">\(\alpha\)</span>T are not found at H3K9me3 chromocenters.
</p>
</div>

<p>The presence of a functional CSD domain on HP1<span class="math inline">\(\alpha\)</span>T was a cause of greater concern, given it could conceivably not only bind to HP1<span class="math inline">\(\beta\)</span> and HP1<span class="math inline">\(\gamma\)</span>, but could act as a dominant negative to any proteins containing a PxVxL motif. This is likely, as when I over-expressed HP1<span class="math inline">\(\alpha\)</span>T, heterochromatic localization of HP1<span class="math inline">\(\beta\)</span> was markedly less (Figure <a href="2-results.html#fig:HP1aTHP1BHP1g">2.3</a>A). HP1<span class="math inline">\(\alpha\)</span>T may also function as a dominant negative for HP1<span class="math inline">\(\gamma\)</span>, sequestering its binding to H3K9me3 foci, however this effect is less obvious as HP1<span class="math inline">\(\gamma\)</span> is normally found in both euchromatin and heterochromatin (Figure <a href="2-results.html#fig:HP1aTHP1BHP1g">2.3</a>B).</p>
<div class="figure" style="text-align: center"><span id="fig:HP1aTHP1BHP1g"></span>
<img src="figure/results/HP1aTHP1BHP1g.png" alt="Sub-Nuclear Distribution of HP1$\alpha$T, HP1$\beta$ and HP1$\gamma$. Over-expression of HP1$\alpha$T-GFP in HEK293T cells (select cells marked with dashed outline) results in decreased localization of endogenous HP1$\beta$ (A) and HP1$\gamma$ (B) to heterochromatin (yellow arrows)." width="100%"   />
<p class="caption">
Figure 2.3: Sub-Nuclear Distribution of HP1<span class="math inline">\(\alpha\)</span>T, HP1<span class="math inline">\(\beta\)</span> and HP1<span class="math inline">\(\gamma\)</span>. Over-expression of HP1<span class="math inline">\(\alpha\)</span>T-GFP in HEK293T cells (select cells marked with dashed outline) results in decreased localization of endogenous HP1<span class="math inline">\(\beta\)</span> (A) and HP1<span class="math inline">\(\gamma\)</span> (B) to heterochromatin (yellow arrows).
</p>
</div>
<p>There was no doubt that if HP1<span class="math inline">\(\alpha\)</span> could act a dominant negative for other HP1 proteins it could also do so for non-HP1 proteins. Given the decreased fertility, potential toxicity and confounding nature of HP1<span class="math inline">\(\alpha\)</span>T, the HP1FEC line was bred back to homozygous wild-type HP1<span class="math inline">\(\alpha\)</span> allele. From henceforth, all future references to the HP1FEC line assume a wild-type HP1<span class="math inline">\(\alpha\)</span> allele. </p>
</div>
<div id="expression-of-hp1-proteins-in-the-brain" class="section level2">
<h2><span class="header-section-number">2.2</span> Expression of HP1 Proteins in the Brain</h2>
In the developing brain, while weak expression can be observed in the ventricular zone, HP1<span class="math inline">\(\alpha\)</span> and HP1<span class="math inline">\(\beta\)</span> are most strongly expressed in post-mitotic neurons in the cortical plate. HP1<span class="math inline">\(\gamma\)</span> is more uniformly expressed in all strata of the developing cortex, including progenitors of the ventricular and subventricular zones (VZ and SVZ respectively) (Figure <a href="2-results.html#fig:HP1brainexp">2.4</a>).
<div class="figure" style="text-align: center"><span id="fig:HP1brainexp"></span>
<img src="figure/results/HP1brainexp.png" alt="Expression of Mammalian HP1 Proteins in the Developing Cerebral Cortex. While HP1 proteins are generally considered to have ubiquitous expression in tissues throughout the body, HP1$\alpha$ and HP1$\beta$ are expressed more strongly in post-mitotic neurons of the cortical plate (CP).  HP1$\gamma$ however, which is also found in euchromatin, is observed in progenitor populations in ventricular and subventricular zones (VZ and SVZ, respectively). Scalebar = 25$\mu$m" width="90%"   />
<p class="caption">
Figure 2.4: Expression of Mammalian HP1 Proteins in the Developing Cerebral Cortex. While HP1 proteins are generally considered to have ubiquitous expression in tissues throughout the body, HP1<span class="math inline">\(\alpha\)</span> and HP1<span class="math inline">\(\beta\)</span> are expressed more strongly in post-mitotic neurons of the cortical plate (CP). HP1<span class="math inline">\(\gamma\)</span> however, which is also found in euchromatin, is observed in progenitor populations in ventricular and subventricular zones (VZ and SVZ, respectively). Scalebar = 25<span class="math inline">\(\mu\)</span>m
</p>
</div>
</div>
<div id="mosaic-deletion-of-hp1beta-and-hp1gamma-from-cortical-progenitors" class="section level2">
<h2><span class="header-section-number">2.3</span> Mosaic Deletion of HP1<span class="math inline">\(\beta\)</span> and HP1<span class="math inline">\(\gamma\)</span> from Cortical Progenitors</h2>
<p>Initial experiments were carried out to determine if cell-intrinsic deletion of HP1<span class="math inline">\(\beta\)</span> and HP1<span class="math inline">\(\gamma\)</span> in neocortical progenitors would affect fate acquisition and/or connectivity of resultant HP1 deficient neurons. To do this, I performed In Utero Electroporation (IUE) on HP1<span class="math inline">\(\beta\)</span><sup>fl/fl</sup><span class="math inline">\(\gamma\)</span><sup>fl/fl</sup> E14.5 embryos, where I could introduce plasmid DNA into the lateral ventricles of the developing brain (see methods <a href="4-methods.html#IUEmethod">4.4.1</a>). This electroporation used a mixture of two plasmids, a Cre expressed consitutively under the CAG promoter (pCAG-Cre) and pCAG-flox-STOP-flox-eGFP (pCAG-FSF-eGFP), a plasmid that expresses cytoplasmic eGFP in the presence of Cre. This plasmid combination both induces Cre-mediated recombination of floxed HP1<span class="math inline">\(\beta\)</span> and HP1<span class="math inline">\(\gamma\)</span> alleles, and allows visualization of electroporated cells via GFP.</p>
<div class="figure" style="text-align: center"><span id="fig:DKOmosaic"></span>
<img src="figure/results/DKOmosaicIUE.png" alt="Mosaic Deletion of HP1$\beta$ and HP1$\gamma$ in the Neocortex at E14.5. Mosaic deletion of HP1$\beta$ and HP1$\gamma$ in cortical progenitors by electroporation with pCAG-Cre and pCAG-FSF-GFP does not impede migration or normal contralateral axon projection (Top Row).  GFP labelled neurons in upper layers were confirmed to be deficient for HP1$\beta$ (Middle Row) and HP1$\gamma$ (Bottom Row), which corresponds with Cre expression (Bottom Row).  Draq5 was used to stain all nuclei." width="75%"   />
<p class="caption">
Figure 2.5: Mosaic Deletion of HP1<span class="math inline">\(\beta\)</span> and HP1<span class="math inline">\(\gamma\)</span> in the Neocortex at E14.5. Mosaic deletion of HP1<span class="math inline">\(\beta\)</span> and HP1<span class="math inline">\(\gamma\)</span> in cortical progenitors by electroporation with pCAG-Cre and pCAG-FSF-GFP does not impede migration or normal contralateral axon projection (Top Row). GFP labelled neurons in upper layers were confirmed to be deficient for HP1<span class="math inline">\(\beta\)</span> (Middle Row) and HP1<span class="math inline">\(\gamma\)</span> (Bottom Row), which corresponds with Cre expression (Bottom Row). Draq5 was used to stain all nuclei.
</p>
</div>
<p>Cell intrinsic deletion of HP1<span class="math inline">\(\beta\)</span> and HP1<span class="math inline">\(\gamma\)</span> at E14.5 did not affect normal positioning or axon projection (Figure <a href="2-results.html#fig:DKOmosaic">2.5</a>). This implies that either HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> are not explicitly required for the final specification of neurons cell instrinsically, or genomic deletion at E14.5 is not sufficient to induce a noticeable effect due to the stability of HP1 proteins already present in the nucleus at E14.5.</p>
</div>
<div id="deletion-of-hp1beta-hp1gamma-from-the-emx1-lineage." class="section level2">
<h2><span class="header-section-number">2.4</span> Deletion of <em>HP1<span class="math inline">\(\beta\)</span></em> &amp; <em>HP1<span class="math inline">\(\gamma\)</span></em> from the <em>Emx1</em>-lineage.</h2>
<div id="cell-fate-in-cortical-neurons" class="section level3">
<h3><span class="header-section-number">2.4.1</span> Cell Fate in Cortical Neurons</h3>
<p><em>Emx1<sup>Cre</sup></em> mediated recombination results in the successful deletion of HP1 proteins from pyramidal cells of the developing cortex (Figure <a href="2-results.html#fig:FECctx">2.6</a>A).</p>
<div class="figure" style="text-align: center"><span id="fig:FECctx"></span>
<img src="figure/results/FECctxHP1Bg.png" alt="Emx1-Cre -Mediated Deletion of HP1 Proteins from the Cerebral Cortex.  (A): Staining of P0 somatosensory cortex reveals that HP1 proteins are deleted from pyramidal cells in the presence of Emx1-Cre. (B): Cell Fate and Laminarization is Unaffected in HP1$\beta$/$\gamma$ DKO Cortices. Specification of upper layer neurons marked by SATB2 and deep layer neurons marked by CTIP2 in the cortical plate (CP) is unperturbed following deletion of HP1$\beta$, HP1$\gamma$, or both." width="100%"   />
<p class="caption">
Figure 2.6: Emx1-Cre -Mediated Deletion of HP1 Proteins from the Cerebral Cortex. (A): Staining of P0 somatosensory cortex reveals that HP1 proteins are deleted from pyramidal cells in the presence of Emx1-Cre. (B): Cell Fate and Laminarization is Unaffected in HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO Cortices. Specification of upper layer neurons marked by SATB2 and deep layer neurons marked by CTIP2 in the cortical plate (CP) is unperturbed following deletion of HP1<span class="math inline">\(\beta\)</span>, HP1<span class="math inline">\(\gamma\)</span>, or both.
</p>
</div>
<p>Surprisingly, cell fate and laminarization is unaffected in <em>HP1<span class="math inline">\(\beta\)</span><sup>fl/fl</sup>Emx1<sup>Cre</sup></em>, <em>HP1<span class="math inline">\(\gamma\)</span><sup>fl/fl</sup>Emx1<sup>Cre</sup></em> and <em>HP1<span class="math inline">\(\beta\)</span><sup>fl/fl</sup><span class="math inline">\(\gamma\)</span><sup>fl/fl</sup>Emx1<sup>Cre</sup></em> cortices, as evidenced by normal distribution of upper layer neurons marked by SATB2 and deep layer neurons marked by CTIP2 (Figure <a href="2-results.html#fig:FECctx">2.6</a>B).</p>
<p>Cells remaining positive for HP1<span class="math inline">\(\beta\)</span> or HP1<span class="math inline">\(\gamma\)</span> are negative for SATB2, and could be interneurons, epithelial or other cell types that are non- <em>Emx1</em> lineage (Figure <a href="2-results.html#fig:Satb2exclude">2.7</a>).</p>
<div class="figure" style="text-align: center"><span id="fig:Satb2exclude"></span>
<img src="figure/results/FECCTXcellfate.png" alt="Cells in upper layers of the cerebral cortex remaining positive for HP1$\beta$ or HP1$\gamma$ in the HP1DKO mutant are negative for SATB2 (white circles), excluding them from pyramidal lineage. Scalebar = 25$\mu$m" width="100%"   />
<p class="caption">
Figure 2.7: Cells in upper layers of the cerebral cortex remaining positive for HP1<span class="math inline">\(\beta\)</span> or HP1<span class="math inline">\(\gamma\)</span> in the HP1DKO mutant are negative for SATB2 (white circles), excluding them from pyramidal lineage. Scalebar = 25<span class="math inline">\(\mu\)</span>m
</p>
</div>

</div>
<div id="imprint" class="section level3">
<h3><span class="header-section-number">2.4.2</span> Imprinted Loci</h3>
<p>Given the known involvement of HP1 with the ZFP57-KAP1 repressive complex in controlling imprinted loci <span class="citation">(Voon and Gibbons <a href="#ref-VoonMaintainingmemorysilencing2016">2016</a>)</span>, the expression of several imprinted loci was profiled by in situ hybridization. While some imprinted genes maintain expression in mature neurons such as <em>Plagl1</em>, no differences could be detected in HP1 mutants at P0 (Figure <a href="2-results.html#fig:Plagl1">2.8</a>). Additionally, probes designed for canonically imprinted <em>Impact</em>, <em>Inpp5f</em>, <em>Gnas</em>, <em>Grb10</em>, <em>Mdga1</em>, <em>Rasgrf1</em> or <em>Pip5k1c</em> did not reveal any changes in tissue expression following single or double deletion of HP1 proteins in the <em>Emx1</em> lineage (data not shown).</p>
<div class="figure" style="text-align: center"><span id="fig:Plagl1"></span>
<img src="figure/results/Plagl1.png" alt="Imprinted Loci Are unaffected in HP1 Mutants at P0.  Shown here is the in situ-hybridization signal for Plagl1." width="100%"   />
<p class="caption">
Figure 2.8: Imprinted Loci Are unaffected in HP1 Mutants at P0. Shown here is the in situ-hybridization signal for Plagl1.
</p>
</div>
</div>
</div>
<div id="perturbed-regulation-of-retroelements-in-hp1-mutants" class="section level2">
<h2><span class="header-section-number">2.5</span> Perturbed Regulation of Retroelements in HP1 Mutants</h2>
<p>Given the known association of HP1 proteins with the KAP1 repressive complex and its role in silencing repetitive elements, I hypothesized that their absence may affect the normal regulation of repetitive elements. To examine this in the brain, I designed consensus RNA probes to detect all copies of a particular retrotransposon by <em>in situ hybridization</em> (ISH).</p>
<div id="line1" class="section level3">
<h3><span class="header-section-number">2.5.1</span> LINE1</h3>
<p>An RNA probe designed against a 700 bp region of open reading frame 2 (ORF2) of LINE1 revealed robust, ubiquitous expression in post-mitotic neurons of the cortex and hippocampus in both wild-type and mutant (Figure <a href="2-results.html#fig:LINEish">2.9</a>). While high expression in the dentate gyrus is concordant with the large body of work examining LINE-driven somatic mosaicism there <span class="citation">(McConnell et al. <a href="#ref-McConnellMosaiccopynumber2013">2013</a>; Muotri et al. <a href="#ref-MuotriEnvironmentalinfluenceL12009">2009</a>; Thomas, Paquola, and Muotri <a href="#ref-ThomasLINE1RetrotranspositionNervous2012">2012</a>)</span>, it was surprising to observe such high abundance of transcripts in all post-mitotic neurons of the cortex. Given such abundance in wild-type, it was not feasible to discern any changes in LINE transcription in HP1 mutant brains by ISH. Notably, in the hippocampus, LINE1 expression is evident in the dentate gyrus as well as CA1 and CA3 fields.</p>
<div class="figure" style="text-align: center"><span id="fig:LINEish"></span>
<img src="figure/results/LINEish.png" alt="In Situ Hybridization for LINE transcripts in adult wild-type and HP1 $\beta$/$\gamma$ DKO brains.  LINE transcripts are already highly abundant in wild-type  post-mitotic neurons." width="100%"   />
<p class="caption">
Figure 2.9: In Situ Hybridization for LINE transcripts in adult wild-type and HP1 <span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO brains. LINE transcripts are already highly abundant in wild-type post-mitotic neurons.
</p>
</div>

</div>
<div id="class-iii-ervs" class="section level3">
<h3><span class="header-section-number">2.5.2</span> Class III ERVs</h3>
<p>To examine possible dysregulation of Class III murine endogenous retroviruses, I designed a 450bp consensus probe to test all transcripts derived from MERVL-MT2, a murine ERVL/Class III endogenous retrovirus. The probe for MERVL that would hybridize to RNA transcripts originating from approximately ~205 MERVL loci. MERVL-derived transcripts are important at the 2 cell (2C) stage of embryonic development and both their standalone expression and of MERVL-derived chimeras is tightly associated with naive pluripotency <span class="citation">(Macfarlan et al. <a href="#ref-MacfarlanEmbryonicstemcell2012">2012</a>)</span>. Modest expression of MERVL derived transcripts can be observed in wild-type brains, and similar levels could be observed in HP1 mutants (Figure <a href="2-results.html#fig:MERVLish">2.10</a>).</p>
<div class="figure" style="text-align: center"><span id="fig:MERVLish"></span>
<img src="figure/results/MERVL.png" alt="In Situ Hybridization Using a Consensus Probe for MERVL.  A probe was designed around a consensus sequence common to all MERVL elements.  MERVL transcripts appear unaltered in HP1 mutant brains." width="100%"   />
<p class="caption">
Figure 2.10: In Situ Hybridization Using a Consensus Probe for MERVL. A probe was designed around a consensus sequence common to all MERVL elements. MERVL transcripts appear unaltered in HP1 mutant brains.
</p>
</div>

</div>
<div id="class-ii-ervs" class="section level3">
<h3><span class="header-section-number">2.5.3</span> Class II ERVs</h3>
<p>To examine possible dysregulation of Class II endogenous retroviruses, I designed a consensus probe for Intracisternal Alpha Particle (IAP), an evolutionary recent murine endogenous retrovirus of the ERVK family. To meet this end I created an RNA probe for a section of the <em>gag</em> domain of IAP. A BLAST query predicted that such a sequence would hybridize to RNA transcripts originating from at least ~196 IAP loci. While IAP transcripts are normally silenced in both wild-type and single mutant brains, a robust de-repression of IAP elements could be observed in the HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> double mutant (Figure <a href="2-results.html#fig:IAPcons">2.11</a>).</p>
<div class="figure" style="text-align: center"><span id="fig:IAPcons"></span>
<img src="figure/results/IAPcons.png" alt="Intracisternal Alpha Particle (IAP) elements are de-repressed in HP1$\beta$/$\gamma$ DKO.  In situ hybridization on various HP1 mutants using an RNA probe for a sequence common to IAP familly members.  IAP transcripts are strongly de-repressed in HP1$\beta$/$\gamma$ DKO, especially in the hippocampus." width="100%"   />
<p class="caption">
Figure 2.11: Intracisternal Alpha Particle (IAP) elements are de-repressed in HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO. In situ hybridization on various HP1 mutants using an RNA probe for a sequence common to IAP familly members. IAP transcripts are strongly de-repressed in HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO, especially in the hippocampus.
</p>
</div>

<p>To ensure the signal observed by the consensus probe was only hybridizing to IAP elements, I designed a separate probe that recognized a unique sequence on an IAP element found on chromosome 2. This RNA probe for a unique IAP sequence recapitulated the findings observed with the consensus probe, where robust de-repression could be observed in post-mitotic neurons of the cerebral cortex and hippocampus (Figure <a href="2-results.html#fig:IAPchr2">2.12</a>). IAP de-repression was confirmed to continue into adulthood (<a href="2-results.html#fig:IAPadult">2.13</a>), where it can be clearly observed that IAP de-repression in the hippocampus is restricted to CA1 and CA3 fields and absent from the dentate gyrus. This is a notable contrast from the expression of LINE1, which can be normally observed in all pyramidal fields of the hippocampus (Figure <a href="2-results.html#fig:LINEish">2.9</a>). Differences in cortical thickness in caudal portions of HP1DKO animals are addressed in section <a href="2-results.html#MRIresults">2.6.2</a>.</p>
<div class="figure" style="text-align: center"><span id="fig:IAPchr2"></span>
<img src="figure/results/IAPchr2.png" alt="ISH of unique IAP Element on P0 brains.  A probe designed around a unique region on an evolutionary recent IAP element on chromosome 2 confirms specificity of the IAP consensus probe." width="100%"   />
<p class="caption">
Figure 2.12: ISH of unique IAP Element on P0 brains. A probe designed around a unique region on an evolutionary recent IAP element on chromosome 2 confirms specificity of the IAP consensus probe.
</p>
</div>
<div class="figure" style="text-align: center"><span id="fig:IAPadult"></span>
<img src="figure/results/IAPadult.png" alt="Sustained IAP De-Repression in the Adult HP1β/γ DKO Brain. In situ hybridization using the IAP consensus probe in 3 month old adult brains." width="100%"   />
<p class="caption">
Figure 2.13: Sustained IAP De-Repression in the Adult HP1β/γ DKO Brain. In situ hybridization using the IAP consensus probe in 3 month old adult brains.
</p>
</div>

</div>
<div id="de-novo-retrotransposition" class="section level3">
<h3><span class="header-section-number">2.5.4</span> <em>De Novo</em> Retrotransposition</h3>
<p>Given multiple full-length copies of IAP elements capable of autonomous retrotransposition exist in the mouse <span class="citation">(Dewannieux et al. <a href="#ref-DewannieuxIdentificationautonomousIAP2004">2004</a>; Maksakova et al. <a href="#ref-MaksakovaRetroviralElementsTheir2006">2006</a>; Saito et al. <a href="#ref-SaitoTranslationnonautonomoustype2008">2008</a>)</span>, I sought to determine if rates of <em>de novo</em> transpositions were increased in HP1 mutant brains where IAP elements are de-repressed. Namely, do a subset of the IAP transcripts we observe encode for full-length autonomous retrotransposons? To examine this, GFP sensors encoding autonomous and non-autonomous IAP elements (pFL and pDE1 respectively, Figure <a href="2-results.html#fig:IAPsensors">2.14</a>, <span class="citation">(Horie et al. <a href="#ref-HorieRetrotransposonsInfluenceMouse2007a">2007</a>)</span>) were introduced by in utero electroporation (IUE) into wild-type and HP1 mutant cortices (Figure <a href="2-results.html#fig:IAPIUE">2.15</a>).</p>
<div class="figure" style="text-align: center"><span id="fig:IAPsensors"></span>
<img src="figure/results/IAPsensors.png" alt="Autonomous and Non-Autonomous IAP Sensors used to detect De Novo Retrotransposition. (Top): The pFL plasmid encodes a full length IAP element followed by an antisense GFP containing an intron.  This construct can autonomously integrate into the genome, and will only express GFP upon integration and is thus an autonomous retrotransposon. The pDE1 plasmid encodes an IAP element with an internal deletion rendering it incapable to retrotranspose on its own (it is nonautonomous).  In the presence of a functional (autonomous) IAP provided in trans, retrotransposition of pDE1 can be made possible. (Bottom): Only following successful transcription, splicing and genomic integration can GFP be expressed, thus the presence of GFP informs on de novo genomic integration(s)." width="80%"   />
<p class="caption">
Figure 2.14: Autonomous and Non-Autonomous IAP Sensors used to detect De Novo Retrotransposition. (Top): The pFL plasmid encodes a full length IAP element followed by an antisense GFP containing an intron. This construct can autonomously integrate into the genome, and will only express GFP upon integration and is thus an autonomous retrotransposon. The pDE1 plasmid encodes an IAP element with an internal deletion rendering it incapable to retrotranspose on its own (it is nonautonomous). In the presence of a functional (autonomous) IAP provided in trans, retrotransposition of pDE1 can be made possible. (Bottom): Only following successful transcription, splicing and genomic integration can GFP be expressed, thus the presence of GFP informs on de novo genomic integration(s).
</p>
</div>
<p>Following exogenous introduction of the autonomous IAP sensor (pFL), <em>de novo</em> genomic integrations could be observed in both wild type and HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO cortices with similar frequencies (Figure <a href="2-results.html#fig:IAPIUE">2.15</a>A). While introduction of the defunct IAP sensor pDE1 into wildtype never resulted in any integrations measured by GFP positive cells (n = 6), when introduced into HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO cortex several integration events could be observed (Figure <a href="2-results.html#fig:IAPIUE">2.15</a>B).</p>
<div class="figure" style="text-align: center"><span id="fig:IAPIUE"></span>
<img src="figure/results/IAPIUE.png" alt="De-Repressed IAP Transcripts Facilitate De Novo Retrotransposition in Trans. Introduction by in utero electroporation (IUE) of autonomous and non-autonomous IAP integration sensors at E14 and observed at E18.  The introduction of a non-autonomous IAP pDE1 into wildtype does not result in integration, however integrations can be observed when introduced into HP1$\beta$/$\gamma$ DKO brains, where IAP elements are de-repressed.  Because pDE1 requires the presence of a functional IAP for genomic integration, this infers a subset of IAP elements de-repressed in the HP1$\beta$/$\gamma$ DKO are functionally autonomous retrotransposons." width="100%"   />
<p class="caption">
Figure 2.15: De-Repressed IAP Transcripts Facilitate De Novo Retrotransposition in Trans. Introduction by in utero electroporation (IUE) of autonomous and non-autonomous IAP integration sensors at E14 and observed at E18. The introduction of a non-autonomous IAP pDE1 into wildtype does not result in integration, however integrations can be observed when introduced into HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO brains, where IAP elements are de-repressed. Because pDE1 requires the presence of a functional IAP for genomic integration, this infers a subset of IAP elements de-repressed in the HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO are functionally autonomous retrotransposons.
</p>
</div>

</div>
</div>
<div id="sustained-hp1betagamma-deficiency-adversely-affects-neuronal-complexity-function-and-cognition" class="section level2">
<h2><span class="header-section-number">2.6</span> Sustained HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> Deficiency Adversely Affects Neuronal Complexity, Function and Cognition</h2>
<p>The <em>HP1<span class="math inline">\(\beta\)</span><sup>fl/fl</sup>HP1<span class="math inline">\(\gamma\)</span><sup>fl/fl</sup>Emx1<sup>Cre</sup></em> mutants provided an interesting model. While neuronal differentiation proceeds relatively normally -at least in terms of classical identity, HP1 double mutants exhibit precocious de-repression of endogenous retroviruses, a phenotype that emerges gradually with normal aging. The question was then posed “What is the functional consequence to the organism when sustained de-repression of such elements occurs in neurons?” To answer this, behavioral phenotyping was performed in addition to descriptive volumetrics and histology.</p>
<div id="behavior" class="section level3">
<h3><span class="header-section-number">2.6.1</span> Behavior</h3>
<div id="seizures" class="section level4 unnumbered">
<h4><em>Seizures</em></h4>
Upon initial handling, it was observed that many HP1FEC animals descended into brief tonic-clonic seizures following novel stimulus, such as a cage change or routine handling. Such seizures typically included a violent convulsion stage lasting about 5 seconds, where in the most extreme cases included excessive salivation. Such convulsions were followed by a 15-30 second period of stupor where the animal was non-responsive. Following this period of stupor, animals exhibited entirely normal behavior. After these initial observations, I installed a seizure log where seizures could be documented by animal staff and myself. Seizures were recorded from April 2015 until February 2017 (Figure <a href="2-results.html#fig:seizures">2.16</a>). While seizures occurred primarily in HP1<span class="math inline">\(\beta\)</span> animals, some were observed in the other genotypes. It should be noted that because these observations rely on animal staff reporting the correct earcode of the animal, rather than a concrete genotype, it is possible that some observations were inaccurately reported, such as those seen in wildtype.<br />

<div class="figure" style="text-align: center"><span id="fig:seizures"></span>
<img src="figure/results/Seizures.png" alt="Seizures in HP1 Mutants.  Seizures were documented in HP1FEC animals in their home cages in the animal facility as well as those undergoing behavioral testing. " width="100%"   />
<p class="caption">
Figure 2.16: Seizures in HP1 Mutants. Seizures were documented in HP1FEC animals in their home cages in the animal facility as well as those undergoing behavioral testing.
</p>
</div>
<p>Several behavioral tests were used to test for alterations to exploratory behavior, anxiety, and learning and memory in young and aged HP1FEC animals. Young (3 months) and Aged (13-14 months) male mice were tested in <em>Open Field Activity</em>, <em>Cued Context Fear Learning</em>, <em>Paired-Pulse Inhibition</em>, the <em>Barnes Maze</em>, <em>Social Activity Monitor</em>, <em>HomeCageScan</em> and <em>Nest Construction</em>. Detailed results of <em>HomeCageScan</em> and <em>Cued Context Fear Learning</em> can be found in Appendix <a href="5-supplemental-data.html#Behavsupplement">5.1</a>. Prior to all behavioral phenotyping, animals were assessed using a modified SHIRPA to ensure animals did not have any physical limitations that could bias behavioral testing, such as deficits in vision, audition, grip strength or locomotion. A total of 42 animals (15 WT, 11 HP1<span class="math inline">\(\beta\)</span>KO, 6 HP1<span class="math inline">\(\gamma\)</span>KO, 10 HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span>DKO) completed the aged time point, where 4 died (2 WT, 1 HP1<span class="math inline">\(\beta\)</span>KO, 1 HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span>DKO) between young and aged testing. All of the following box and whisker plots are comprised of median and 25<sup>th</sup> and 75<sup>th</sup> percentiles, where whiskers extend no further than 1.5X the interquartile range. All line charts plot the mean and standard error as a ribbon surrounding the line.</p>

</div>
<div id="open-field-activity" class="section level4 unnumbered">
<h4><em>Open Field activity</em></h4>
<p>Anxiety and exploration was measured using a simple open field test, where animals are placed in the center of a square open field and their activity in both the center and the periphery is tracked over the course of 10 minutes (Figure <a href="2-results.html#fig:openfield">2.17</a>).</p>
<p>In young animals, significant decreases in average ambulation velocity and percent activity were observed in HP1<span class="math inline">\(\beta\)</span>KO animals. This effect was due primarily to a strong initial freezing response in roughly half of the HP1<span class="math inline">\(\beta\)</span> animals following entrance of the animal into the center of the arena at the beginning of the test. This initial freezing response in HP1<span class="math inline">\(\beta\)</span>KO animals was akin to non-convulsive status epilepticus rather than a fear response. Typically midway through the test HP1<span class="math inline">\(\beta\)</span> animals began normal exploration and avoidance of the center. In aged animals, the initial freezing response could be observed in a subset of HP1<span class="math inline">\(\beta\)</span>KO animals, however the strongest effect was seen in aged DKO animals, where a majority showed a prolonged stupor in the center of the open field at the beginning of the test.</p>
<div class="figure" style="text-align: center"><span id="fig:openfield"></span>
<img src="figure/results/Open_field.png" alt="Open Field Activity in HP1 Mutants. Ribbons surrounding per-minute line plots denote SEM. Statistics: two-way ANOVA with bonferroni corrections on multiple comparisons. Asterisks (*) denote tests within age for differences between genotype: *** = p &lt; 0.001, ** = p &lt; 0.01, * = p &lt; 0.05. All statistically significant changes associated with age within genotype are represented by a pound sign (\#): \#\#\# = p &lt; 0.001,  \#\# = p &lt; 0.01, \# = p &lt; 0.05)." width="100%"   />
<p class="caption">
Figure 2.17: Open Field Activity in HP1 Mutants. Ribbons surrounding per-minute line plots denote SEM. Statistics: two-way ANOVA with bonferroni corrections on multiple comparisons. Asterisks (<em>) denote tests within age for differences between genotype: </em>** = p &lt; 0.001, ** = p &lt; 0.01, * = p &lt; 0.05. All statistically significant changes associated with age within genotype are represented by a pound sign (#): ### = p &lt; 0.001, ## = p &lt; 0.01, # = p &lt; 0.05).
</p>
</div>
</div>
<div id="social-activity-monitor-sam" class="section level4 unnumbered">
<h4><em>Social Activity Monitor (SAM)</em></h4>
<p>To examine social activity and circadian rhythm, animal activity was measured in their home cages by RFID chip movement. Animals were monitored over 14 days and data was averaged first within animal by hour of the day across all days, then by genotype (Figure <a href="2-results.html#fig:SAM">2.18</a>).</p>
<div class="figure" style="text-align: center"><span id="fig:SAM"></span>
<img src="figure/results/SAMalt.png" alt="Social Activity of HP1 mutants by Time of Day.  The activity of each animal was recorded over a 14-day period.  Individual animal activity was averaged by each hour of the day over the duration, then grouped by genotype and age.  Plotted here is the mean activity per hour (thick line) along with the standard error of the mean (ribbon) per genotype by age.  Statistics: two-way ANOVA with bonferroni corrections on multiple comparisons. Asterisks (*) denote tests within age for differences between genotype: *** = p &lt; 0.001, ** = p &lt; 0.01, * = p &lt; 0.05. All statistically significant changes associated with age within genotype are represented by a pound sign (\#); (\#\#\# = p &lt; 0.001, \#\# = p &lt; 0.01, \# = p &lt; 0.05)." width="85%"   />
<p class="caption">
Figure 2.18: Social Activity of HP1 mutants by Time of Day. The activity of each animal was recorded over a 14-day period. Individual animal activity was averaged by each hour of the day over the duration, then grouped by genotype and age. Plotted here is the mean activity per hour (thick line) along with the standard error of the mean (ribbon) per genotype by age. Statistics: two-way ANOVA with bonferroni corrections on multiple comparisons. Asterisks (<em>) denote tests within age for differences between genotype: </em>** = p &lt; 0.001, ** = p &lt; 0.01, * = p &lt; 0.05. All statistically significant changes associated with age within genotype are represented by a pound sign (#); (### = p &lt; 0.001, ## = p &lt; 0.01, # = p &lt; 0.05).
</p>
</div>

</div>
<div id="homecagescan-animal-microbehaviors" class="section level4 unnumbered">
<h4><em>HomeCageScan &amp; Animal Microbehaviors</em></h4>
<p>Following a 24 hr observation period under active scanning, the frequency and number of discrete bouts of various microbehaviors was quantified. These behaviors are summarized in supplemental Figures <a href="5-supplemental-data.html#fig:HCSpie">5.1</a>, <a href="5-supplemental-data.html#fig:HCSpercent">5.2</a> and <a href="5-supplemental-data.html#fig:HCSbouts">5.3</a>. Several behaviors associated with increased anxiety could be observed in HP1<span class="math inline">\(\gamma\)</span> and HP1 double mutants. Notable changes were observed in both the bouts and percent of time spent grooming in HP1<span class="math inline">\(\gamma\)</span> and HP1DKO young mice, which is sustained with age (Figure <a href="2-results.html#fig:GroomEatWeight">2.19</a>A-B). Additionally, aged HP1DKO animals spend significantly more time eating (Figure <a href="2-results.html#fig:GroomEatWeight">2.19</a>C-D), which corresponds to a tendency for the HP1DKOs to gain weight (Figure <a href="2-results.html#fig:GroomEatWeight">2.19</a>E).</p>
<div class="figure" style="text-align: center"><span id="fig:GroomEatWeight"></span>
<img src="figure/results/GroomEatWeight.png" alt="HP1 $\beta$/$\gamma$ DKO Mutants Display Microbehaviors associated with increased Anxiety. Statistics: two-way ANOVA with bonferroni corrections on multiple comparisons. Asterisks (*) denote tests within age for differences between genotype: *** = p &lt; 0.001, ** = p &lt; 0.01, * = p &lt; 0.05. All statistically significant changes associated with age within genotype are represented by a pound sign (\#); (\#\#\# = p &lt; 0.001, \#\# = p &lt; 0.01, \# = p &lt; 0.05)." width="100%"   />
<p class="caption">
Figure 2.19: HP1 <span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO Mutants Display Microbehaviors associated with increased Anxiety. Statistics: two-way ANOVA with bonferroni corrections on multiple comparisons. Asterisks (<em>) denote tests within age for differences between genotype: </em>** = p &lt; 0.001, ** = p &lt; 0.01, * = p &lt; 0.05. All statistically significant changes associated with age within genotype are represented by a pound sign (#); (### = p &lt; 0.001, ## = p &lt; 0.01, # = p &lt; 0.05).
</p>
</div>

</div>
<div id="paired-pulse-inhibition" class="section level4 unnumbered">
<h4><em>Paired-Pulse Inhibition</em></h4>
<p>An examination for alterations in paired-pulse inhibition revealed an age-dependent deficit in HP1DKO animals (Figure <a href="2-results.html#fig:PPI">2.20</a>). While all aged animals displayed a tendency towards weaker paired-pulse inhibition, aged HP1DKO animals showed little change in startle response with increasing amplitude of pre-pulse.</p>
<div class="figure" style="text-align: center"><span id="fig:PPI"></span>
<img src="figure/results/PPI.png" alt="Percent Paired-Pulse Inhibition In HP1 Mutants.  Percent paired-pulse inhibition was measured by percent normalizing the measured startle responses in each pre-pulse condition (69 dB, 71 dB and 81 dB) to the measured startle response to the pulse (120 dB). Statistics: two-way ANOVA with bonferroni corrections on multiple comparisons. Asterisks (*) denote tests within age for differences between genotype: *** = p &lt; 0.001, ** = p &lt; 0.01, * = p &lt; 0.05. All statistically significant changes associated with age within genotype are represented by a pound sign (\#); (\#\#\# = p &lt; 0.001, \#\# = p &lt; 0.01, \# = p &lt; 0.05)." width="100%"   />
<p class="caption">
Figure 2.20: Percent Paired-Pulse Inhibition In HP1 Mutants. Percent paired-pulse inhibition was measured by percent normalizing the measured startle responses in each pre-pulse condition (69 dB, 71 dB and 81 dB) to the measured startle response to the pulse (120 dB). Statistics: two-way ANOVA with bonferroni corrections on multiple comparisons. Asterisks (<em>) denote tests within age for differences between genotype: </em>** = p &lt; 0.001, ** = p &lt; 0.01, * = p &lt; 0.05. All statistically significant changes associated with age within genotype are represented by a pound sign (#); (### = p &lt; 0.001, ## = p &lt; 0.01, # = p &lt; 0.05).
</p>
</div>

</div>
<div id="nest-construction" class="section level4 unnumbered">
<h4><em>Nest Construction</em></h4>
<p>Pilot experiments revealed that following the two day separation of male animals from their homecages, return to their homecage resulted in excessive aggression and violence between former cohabitants. For this reason the nest construction test was only tested at the aged timepoint. Nest construction was adversely affected in HP1DKO mutants both in terms of complexity (complexity score, methods <a href="4-methods.html#nestcomethod">4.5.7</a>) and completion (% nestlet used in constructing the nest) (Figure <a href="2-results.html#fig:Nestconstruction">2.21</a>). In most cases, HP1DKO animals barely touched the cotton nestlet, leaving it mostly intact (Figure <a href="2-results.html#fig:Nestconstruction">2.21</a>C).</p>
<div class="figure" style="text-align: center"><span id="fig:Nestconstruction"></span>
<img src="figure/results/Nest.png" alt="Nest Construction and Complexity.  (A) plots the amount of nest construction, where any remaining solid nestlet is percent normalized to its initial weight.  Nest complexity is qualified in (B), which was measured using a standardized rubrick (see methods). (C) Representative images of nests constructed by each genotype. Statistics: one-way ANOVA with bonferroni corrections on multiple comparisons: *** = p &lt; 0.001, ** = p &lt; 0.01, * = p &lt; 0.05." width="100%"   />
<p class="caption">
Figure 2.21: Nest Construction and Complexity. (A) plots the amount of nest construction, where any remaining solid nestlet is percent normalized to its initial weight. Nest complexity is qualified in (B), which was measured using a standardized rubrick (see methods). (C) Representative images of nests constructed by each genotype. Statistics: one-way ANOVA with bonferroni corrections on multiple comparisons: *** = p &lt; 0.001, ** = p &lt; 0.01, * = p &lt; 0.05.
</p>
</div>

</div>
<div id="barnes-maze" class="section level4 unnumbered">
<h4><em>Barnes Maze</em></h4>
<p>Spatial memory of HP1 animals was tested using the Barnes maze (methods <a href="4-methods.html#Barnesmeth">4.5.4</a>). Rather than employing a direct strategy, which would involve directly navigating to the perceived target hole, HP1DKO animals often employed serial search patterns. HP1DKO animals walked clockwise or counter-clockwise around the periphery of the Barnes maze checking every hole (Figure <a href="2-results.html#fig:barnes">2.22</a>A). Such poor performance coincided with a significantly longer latency to target and significant decreases in the number of correct visits to target (Figure <a href="2-results.html#fig:barnes">2.22</a>B,C).</p>
<div class="figure" style="text-align: center"><span id="fig:barnes"></span>
<img src="figure/results/barnes.png" alt="HP1 Mutant Performance in the Barnes Maze. Histograms and inset radar plots displaying mean headpokes +SEM per hole (A) reveal search patterns of HP1 mutants in each test by age.  Total percent correct visits (B) and latency to target (C) summarize effects by genotype. Statistics: two-way ANOVA with bonferroni corrections on multiple comparisons. Asterisks (*) denote tests between specified genotype and wildtype within age group: *** = p &lt; 0.001, ** = p &lt; 0.01, * = p &lt; 0.05. All statistically significant changes associated with age within genotype are represented by a pound sign (\#); (\#\#\# = p &lt; 0.001, \#\# = p &lt; 0.01, \# = p &lt; 0.05)." width="100%"   />
<p class="caption">
Figure 2.22: HP1 Mutant Performance in the Barnes Maze. Histograms and inset radar plots displaying mean headpokes +SEM per hole (A) reveal search patterns of HP1 mutants in each test by age. Total percent correct visits (B) and latency to target (C) summarize effects by genotype. Statistics: two-way ANOVA with bonferroni corrections on multiple comparisons. Asterisks (<em>) denote tests between specified genotype and wildtype within age group: </em>** = p &lt; 0.001, ** = p &lt; 0.01, * = p &lt; 0.05. All statistically significant changes associated with age within genotype are represented by a pound sign (#); (### = p &lt; 0.001, ## = p &lt; 0.01, # = p &lt; 0.05).
</p>
</div>

</div>
</div>
<div id="MRIresults" class="section level3">
<h3><span class="header-section-number">2.6.2</span> Structure, Volumetrics and Histology</h3>
<p>Given strong behavioral deficits in spatial memory and the aforementioned strong de-repression of IAP elements in the developing hippocampus, further attention was paid to determine any alterations in hippocampal circuitry or structure.</p>
<div id="reduced-dendritic-complexity-in-hippocampal-ca3-neurons" class="section level4 unnumbered">
<h4><em>Reduced Dendritic Complexity in Hippocampal CA3 Neurons</em></h4>
<p>Dendritic complexity of CA3 neurons was quantified in young and aged HP1 mutants by golgi impregnation (Figure <a href="2-results.html#fig:golgiquant">2.23</a>). Young CA3 neurons from both HP1<span class="math inline">\(\beta\)</span>KO and HP1DKO animals showed marked reduction in the complexity of their basal dendritic trees (Figure <a href="2-results.html#fig:golgiquant">2.23</a>C, E). This trend continues into aged animals, most prominently in HP1DKO animals, where a drastic loss of basal dendrite complexity can be observed (Figure <a href="2-results.html#fig:golgiquant">2.23</a>D, F).</p>
<div class="figure" style="text-align: center"><span id="fig:golgiquant"></span>
<img src="figure/results/golgiCA3quant5.png" alt="Dendritic Complexity of Hippocampal CA3 Neurons in Young and Aged HP1 Mutants.  Dendritic complexity was quantified by golgi impregnation followed by Scholl analysis, where intersections of dendrites in each of thirty 10 $\mu$m concentric circles were quantified (A).  (B) displays representative reconstructions of CA3 neurons.  Scholl analysis is plotted for neurons from young (C) and aged animals (D) by genotype, where the thickness of the line corresponds with the standard error around the mean and n refers to the number of neurons counted accross 3 hippocampi from each condition.  Box and whisker plots display total apical and basal interactions for young (E) and aged (F) animals. Statistics: two-way ANOVA with bonferroni corrections on multiple comparisons. Asterisks (*) denote tests within age for differences between genotype: *** = p &lt; 0.001, ** = p &lt; 0.01, * = p &lt; 0.05. All statistically significant changes associated with age within genotype are represented by a pound sign (\#); (\#\#\# = p &lt; 0.001, \#\# = p &lt; 0.01, \# = p &lt; 0.05)." width="100%"   />
<p class="caption">
Figure 2.23: Dendritic Complexity of Hippocampal CA3 Neurons in Young and Aged HP1 Mutants. Dendritic complexity was quantified by golgi impregnation followed by Scholl analysis, where intersections of dendrites in each of thirty 10 <span class="math inline">\(\mu\)</span>m concentric circles were quantified (A). (B) displays representative reconstructions of CA3 neurons. Scholl analysis is plotted for neurons from young (C) and aged animals (D) by genotype, where the thickness of the line corresponds with the standard error around the mean and n refers to the number of neurons counted accross 3 hippocampi from each condition. Box and whisker plots display total apical and basal interactions for young (E) and aged (F) animals. Statistics: two-way ANOVA with bonferroni corrections on multiple comparisons. Asterisks (<em>) denote tests within age for differences between genotype: </em>** = p &lt; 0.001, ** = p &lt; 0.01, * = p &lt; 0.05. All statistically significant changes associated with age within genotype are represented by a pound sign (#); (### = p &lt; 0.001, ## = p &lt; 0.01, # = p &lt; 0.05).
</p>
</div>
</div>
<div id="malformation-of-the-dentate-gyrus" class="section level4 unnumbered">
<h4><em>Malformation of the Dentate Gyrus</em></h4>
<p>It became apparent that the infrapyramidal blade of the dentate gyrus was missing in HP1<span class="math inline">\(\beta\)</span>KO and HP1DKO animals (Figure <a href="2-results.html#fig:nissl">2.24</a>). In order to understand this deficit, I stained for dividing cells (Ki-67) and post-mitotic granule neurons (Prox1) at early postnatal timepoints when the dentate gyrus is formed (Figure <a href="2-results.html#fig:prox1ki67">2.25</a>). At P0, HP1DKO dentate gyrii show a statistically significant decrease in Ki-67 positive cells, and already show a trend towards a decrease in Prox1 positive cells. By postnatal day 8, HP1<span class="math inline">\(\beta\)</span>KO and HP1DKO animals fail to generate the granular and molecular layers of the infrapyramidal blade of the dentate gyrus, which corresponds with a decrease in total area, and in the number of Prox1 positive and proliferative Ki-67 positive cells (Figure <a href="2-results.html#fig:prox1ki67">2.25</a>B &amp; C). Given that a subset of DG granule cells in the infrapyramidal blade project to the basal dendrites of CA3 cells, it is likely that dentate gyrus malformations seen in HP1<span class="math inline">\(\beta\)</span>KO and HP1DKO animals are directly linked to the progressive loss of CA3 basal dendritic fields in those animals. It should also be noted that while the infrapyramidal blade of the dentate gyrus is lost in HP1<span class="math inline">\(\beta\)</span>KO animals, they do not exhibit the same behavioral abnormalities seen in HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span>DKO animals. </p>
<div class="figure" style="text-align: center"><span id="fig:nissl"></span>
<img src="figure/results/_Nissl.png" alt="Young and Aged Adult HP1 Mutant Hippocampi. A nissl stain using cresyl violet on paraffin sections." width="100%"   />
<p class="caption">
Figure 2.24: Young and Aged Adult HP1 Mutant Hippocampi. A nissl stain using cresyl violet on paraffin sections.
</p>
</div>
<div class="figure" style="text-align: center"><span id="fig:prox1ki67"></span>
<img src="figure/results/DGprox1ki67.png" alt="Dentate Gyrus Malformations in HP1 Mutants. (A) Representative images of HP1 mutant dentate gyrii at postnatal day 0 (P0) and day 8 (P8) stained for Prox1, Ki-67 and DAPI. (B) Measured areas of dentate gyrii filling a standard field of view. (C) Plots Prox1 and Ki-67 positive cells by genotype.  Statistics: One-way ANOVA within age for area or marker with Dunnett's multiple comparison test. *** = p &lt; 0.001, ** = p &lt; 0.01, * = p &lt; 0.05. P0: WT n = 10, HP1$\beta$KO n = 4, HP1$\gamma$KO n = 7, HP1DKO n = 11. P8: WT n = 4, HP1$\beta$KO n = 3, HP1$\gamma$KO n = 4, HP1DKO n = 5, where n is a biological replicate" width="100%"   />
<p class="caption">
Figure 2.25: Dentate Gyrus Malformations in HP1 Mutants. (A) Representative images of HP1 mutant dentate gyrii at postnatal day 0 (P0) and day 8 (P8) stained for Prox1, Ki-67 and DAPI. (B) Measured areas of dentate gyrii filling a standard field of view. (C) Plots Prox1 and Ki-67 positive cells by genotype. Statistics: One-way ANOVA within age for area or marker with Dunnett’s multiple comparison test. *** = p &lt; 0.001, ** = p &lt; 0.01, * = p &lt; 0.05. P0: WT n = 10, HP1<span class="math inline">\(\beta\)</span>KO n = 4, HP1<span class="math inline">\(\gamma\)</span>KO n = 7, HP1DKO n = 11. P8: WT n = 4, HP1<span class="math inline">\(\beta\)</span>KO n = 3, HP1<span class="math inline">\(\gamma\)</span>KO n = 4, HP1DKO n = 5, where n is a biological replicate
</p>
</div>
<p></p>
</div>
<div id="volumetrics" class="section level4 unnumbered">
<h4><em>Volumetrics</em></h4>
<p>A formal characterization of volumetric changes to the cortex and hippocampus in HP1 mutants was obtained by structural Magnetic Resonance Imaging (MRI). A notable reduction in the volume of the total brain, isocortex, hippocampal region and dentate gyrus was confirmed in HP1DKO animals (Figure <a href="2-results.html#fig:MRIresults">2.26</a>). As previously reported <span class="citation">(Aucott et al. <a href="#ref-AucottHP1requireddevelopment2008">2008</a>)</span>, the constitutive deletion of <em>HP1<span class="math inline">\(\beta\)</span></em> results in mitotic deficits, suggesting that while cells deficient for HP1<span class="math inline">\(\beta\)</span> can divide, they have a finite number of divisions that can occur. In the <em>Emx1<sup>Cre</sup></em> mouse line, Cre is expressed coincident with <em>Emx1</em>, which begins at E10.5. Normally, the radial glia and subsequent intermediate progenitors undergo a finite amount of divisions in generating the post-mitotic neurons of the cortex and hippocampus. Given the dentate gyrus is one of the last regions to be formed, it stands to reason that HP1<span class="math inline">\(\beta\)</span>-deficient neural progenitors prematurely exhaust their division potential. In parallel to DG loss, when normalizing to overall brain volume, young animals show specific reductions in slices 10 &amp; 11, which corresponds to the parietal and somatosensory cortices surrounding the hippocampus, and also that seen in Figure <a href="2-results.html#fig:IAPadult">2.13</a>. Notably, when normalized to total brain volume, cortical volumes are not statistically different from wildtype.</p>
<div class="figure" style="text-align: center"><span id="fig:MRIresults"></span>
<img src="figure/results/_MRI.png" alt="Volumetric Analysis of HP1 mutant brains by Magnetic Resonance Imaging. (A) Representative MRI volume reconstructions with isocortex (cyan), hippocampus (green) and dentate gyrus (red) region masks applied. Total volume of the brain (B), hippocampus (C), dentate gyrus (D) and detailed analysis of the volume of the isocortex (E) is plotted by genotype for both ages. Ribbons around line plots in (E) account for standard deviation per slice per genotype. Statistics: one-way ANOVA within age with Bonferroni multiple comparison corrections. *** = p &lt; 0.001, ** = p &lt; 0.01, * = p &lt; 0.05." width="100%"   />
<p class="caption">
Figure 2.26: Volumetric Analysis of HP1 mutant brains by Magnetic Resonance Imaging. (A) Representative MRI volume reconstructions with isocortex (cyan), hippocampus (green) and dentate gyrus (red) region masks applied. Total volume of the brain (B), hippocampus (C), dentate gyrus (D) and detailed analysis of the volume of the isocortex (E) is plotted by genotype for both ages. Ribbons around line plots in (E) account for standard deviation per slice per genotype. Statistics: one-way ANOVA within age with Bonferroni multiple comparison corrections. *** = p &lt; 0.001, ** = p &lt; 0.01, * = p &lt; 0.05.
</p>
</div>

</div>
</div>
</div>
<div id="hp1beta-and-hp1gamma-synergistically-contribute-to-repeat-silencing" class="section level2">
<h2><span class="header-section-number">2.7</span> HP1<span class="math inline">\(\beta\)</span> and HP1<span class="math inline">\(\gamma\)</span> Synergistically Contribute to Repeat Silencing</h2>
<p>The robust de-repression of ERVs in HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO cells is in stark contrast to only a modest de-repression observed in HP1<span class="math inline">\(\alpha\)</span>/<span class="math inline">\(\beta\)</span> DKO ES cells <span class="citation">(Maksakova et al. <a href="#ref-MaksakovaH3K9me3bindingproteinsare2011">2011</a>)</span>, which was mainly attributed to deficiency in HP1<span class="math inline">\(\beta\)</span>. This suggested that the de-repression we observe in HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO cells is likely because HP1<span class="math inline">\(\beta\)</span> and HP1<span class="math inline">\(\gamma\)</span> repress ERVs by some synergism between separate molecular pathways, rather than be due to HP1 dosage.</p>
<div id="hp1gamma-is-required-for-h4k20me3-deposition-in-neurons" class="section level3">
<h3><span class="header-section-number">2.7.1</span> HP1<span class="math inline">\(\gamma\)</span> is Required for H4K20me3 Deposition in Neurons</h3>
<p>While HP1 proteins collectively have been shown to be important for secondary deposition of H4K20me3 following H3K9 trimethylation <span class="citation">(Kourmouli et al. <a href="#ref-KourmouliEpigeneticregulationmammalian2005">2005</a>; Schotta <a href="#ref-Schottasilencingpathwayinduce2004">2004</a>)</span>, it was surprising to observe that H4K20me3 is completely lost following single deletion of HP1<span class="math inline">\(\gamma\)</span> (Figure <a href="2-results.html#fig:H4K20me3loss">2.27</a>B). It was confirmed that this effect is specific to HP1<span class="math inline">\(\gamma\)</span>, as H3K9me3, a pre-requisite for H4K20me3, is unaffected (Figure <a href="2-results.html#fig:H4K20me3loss">2.27</a>A).</p>
<div class="figure" style="text-align: center"><span id="fig:H4K20me3loss"></span>
<img src="figure/results/H3K9H4K20landscape.png" alt="H4K20me3 is lost following deletion of HP1$\gamma$. (A) Trimethylation of histone 3 lysine 9 (H3K9me3) is unaffected in HP1$\beta$ and HP1$\gamma$ single and double mutants. (B) Trimethylation of histone 4 lysine 20 (H4K20me3) is lost following deletion of HP1$\gamma$. Pia - Pia matter, CP - Cortical Plate." width="100%"   />
<p class="caption">
Figure 2.27: H4K20me3 is lost following deletion of HP1<span class="math inline">\(\gamma\)</span>. (A) Trimethylation of histone 3 lysine 9 (H3K9me3) is unaffected in HP1<span class="math inline">\(\beta\)</span> and HP1<span class="math inline">\(\gamma\)</span> single and double mutants. (B) Trimethylation of histone 4 lysine 20 (H4K20me3) is lost following deletion of HP1<span class="math inline">\(\gamma\)</span>. Pia - Pia matter, CP - Cortical Plate.
</p>
</div>

</div>
<div id="hp1gamma-binds-suv420h2-via-the-chromoshadow-domain" class="section level3">
<h3><span class="header-section-number">2.7.2</span> HP1<span class="math inline">\(\gamma\)</span> binds SUV420H2 via the Chromoshadow Domain</h3>
<p>In characterizing this phenotype, I tested if HP1<span class="math inline">\(\gamma\)</span> could bind to SUV420H2, one of the primary enzymes responsible for deposition of H4K20me3, and tested which residues are necessary for this interaction. To do this I generated HP1<span class="math inline">\(\gamma\)</span> point mutants (Methods, Table <a href="4-methods.html#tab:CSDpointmutations">4.4</a>), and observed their ability to Co-ImmunoPrecipitate (Co-IP) and co-localize with myc-tagged SUV420H2 (Figure <a href="2-results.html#fig:HP1gSUValt">2.28</a>). In addition to residues known to be important for HP1<span class="math inline">\(\gamma\)</span> dimerization, such as I165 and W174, I could determine that F167 and L172 residues are required for HP1<span class="math inline">\(\gamma\)</span> binding with SUV420H2.</p>
<div class="figure" style="text-align: center"><span id="fig:HP1gSUValt"></span>
<img src="figure/results/HP1gSUV420H2alt.png" alt="Localization and Binding of HP1$\gamma$ with SUV420H2 Requires the Chromoshadow Domain (CSD). Sub-nuclear colocalization of SUV420H2 with wild type HP1-GFP fusion proteins and HP1$\gamma$-GFP CSD mutations (A).  Co-immunoprecipitation of HP1$\gamma$ and SUV420H2 requires dimerization of HP1$\gamma$ (B). Deletion of the entire HP1$\gamma$ CSD domain or mutation of residues I165, F167, L172 on HP1$\gamma$ inhibits localization with SUV420H2 (A) and binding (B)." width="100%"   />
<p class="caption">
Figure 2.28: Localization and Binding of HP1<span class="math inline">\(\gamma\)</span> with SUV420H2 Requires the Chromoshadow Domain (CSD). Sub-nuclear colocalization of SUV420H2 with wild type HP1-GFP fusion proteins and HP1<span class="math inline">\(\gamma\)</span>-GFP CSD mutations (A). Co-immunoprecipitation of HP1<span class="math inline">\(\gamma\)</span> and SUV420H2 requires dimerization of HP1<span class="math inline">\(\gamma\)</span> (B). Deletion of the entire HP1<span class="math inline">\(\gamma\)</span> CSD domain or mutation of residues I165, F167, L172 on HP1<span class="math inline">\(\gamma\)</span> inhibits localization with SUV420H2 (A) and binding (B).
</p>
</div>
</div>
<div id="re-addition-of-hp1gamma-can-restore-h4k20me3-in-vivo" class="section level3">
<h3><span class="header-section-number">2.7.3</span> Re-addition of HP1<span class="math inline">\(\gamma\)</span> can restore H4K20me3 In Vivo</h3>
<p>To test for sufficiency of HP1<span class="math inline">\(\gamma\)</span>’s requirement for deposition of H4K20me3, I re-introduced HP1<span class="math inline">\(\gamma\)</span> into HP1DKO cortices by in-utero electroporation. Re-addition of HP1<span class="math inline">\(\gamma\)</span> was able to restore H4K20me3 heterochromatic puncta (Figure <a href="2-results.html#fig:H4K20me3rescue">2.29</a>), although such puncta appear much smaller than those seen in adjacent wild-type interneurons.</p>
<div class="figure" style="text-align: center"><span id="fig:H4K20me3rescue"></span>
<img src="figure/results/H4K20me3rescue.png" alt="Re-introduction of HP1$\gamma$ can restore H4K20me3.  Following Emx1-mediated deletion of HP1$\beta$ and HP1$\gamma$ at E10.5, re-introduction of HP1$\gamma$-GFP by in utero electroporation (IUE) at E14 restores H4K20me3 (magnification, white circles)." width="100%"   />
<p class="caption">
Figure 2.29: Re-introduction of HP1<span class="math inline">\(\gamma\)</span> can restore H4K20me3. Following Emx1-mediated deletion of HP1<span class="math inline">\(\beta\)</span> and HP1<span class="math inline">\(\gamma\)</span> at E10.5, re-introduction of HP1<span class="math inline">\(\gamma\)</span>-GFP by in utero electroporation (IUE) at E14 restores H4K20me3 (magnification, white circles).
</p>
</div>
</div>
<div id="h4k20me3-loss-is-correlated-with-iap-de-repression" class="section level3">
<h3><span class="header-section-number">2.7.4</span> H4K20me3 Loss is Correlated with IAP De-repression</h3>
<p>Given H4K20me3 is known to be important in silencing repeats, it followed that we could observe an exact correlation with loss of H4K20me3 and the de-repression of IAP elements there. Specifically, we observe that H4K20me3 is highly enriched in post-mitotic neurons of the cortex and hippocampus, which is also where we observe IAP de-repression in the HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO (Figure <a href="2-results.html#fig:H4K20me3corr">2.30</a>).</p>
<div class="figure" style="text-align: center"><span id="fig:H4K20me3corr"></span>
<img src="figure/results/H4K20me3_4.png" alt="H4K20me3 loss is Correlated with IAP De-Repression. (A) In situ hybridization using the IAP consensus probe on P0 sections followed by IHC for H4K20me3 and DAPI on the next serial section with magnification on the cortex. (B) Combined fluorescent ISH/IHC of IAP and H4K20me3 and HP1$\gamma$ in the developing CA3 field.  Robust IAP signal can be observed in the stratum pyramidale (Str. Pyr.) where HP1$\gamma$ and H4K20me3 are lost in post-mitotic CA3 neurons." width="85%"   />
<p class="caption">
Figure 2.30: H4K20me3 loss is Correlated with IAP De-Repression. (A) In situ hybridization using the IAP consensus probe on P0 sections followed by IHC for H4K20me3 and DAPI on the next serial section with magnification on the cortex. (B) Combined fluorescent ISH/IHC of IAP and H4K20me3 and HP1<span class="math inline">\(\gamma\)</span> in the developing CA3 field. Robust IAP signal can be observed in the stratum pyramidale (Str. Pyr.) where HP1<span class="math inline">\(\gamma\)</span> and H4K20me3 are lost in post-mitotic CA3 neurons.
</p>
</div>

<p>However, as evidenced by ISH (Figure <a href="2-results.html#fig:IAPcons">2.11</a>) and RNAseq from HP1<span class="math inline">\(\gamma\)</span> deficient neurons (Figure <a href="2-results.html#fig:RNAsum">2.34</a>), loss of H4K20me3 alone is not sufficient to induce de-repression of ERVK elements such as IAPs. Clearly, HP1<span class="math inline">\(\beta\)</span> contributes an additional mechanism to the repression of these elements.</p>
</div>
<div id="kap1-h3k9me3-and-h4k20me3-are-lost-at-iap-elements-in-hp1ctko-es-cells" class="section level3">
<h3><span class="header-section-number">2.7.5</span> KAP1, H3K9me3 and H4K20me3 are lost at IAP Elements in HP1cTKO ES Cells</h3>
<div class="figure" style="text-align: center"><span id="fig:IAPH3K9"></span>
<img src="figure/results/IAPEz-int_h3k9me3_H4K20me3.png" alt="KAP1, H3K9me3 and H4K20me3 are Reduced at IAP Elements in HP1cTKO ES Cells. (Top): Read coverage profiles of KAP1, H3K9me3 and H4K20me3 ChIP signals split by cluster. (Below): Heatmaps of raw ChIPseq signal (RPKM) over annotated IAPEz internal (-int) fragments, grouped by 3 kmeans clusters." width="100%"   />
<p class="caption">
Figure 2.31: KAP1, H3K9me3 and H4K20me3 are Reduced at IAP Elements in HP1cTKO ES Cells. (Top): Read coverage profiles of KAP1, H3K9me3 and H4K20me3 ChIP signals split by cluster. (Below): Heatmaps of raw ChIPseq signal (RPKM) over annotated IAPEz internal (-int) fragments, grouped by 3 kmeans clusters.
</p>
</div>

<p>A collaboration with Dr. Jafar Sharif (RIKEN, Yokohama, Japan) enabled interrogation into the effects of the loss of HP1 proteins in a single cell population. HP1cTKO ES cells are transgenic ES cells where all three HP1’s (<span class="math inline">\(\alpha\)</span>, <span class="math inline">\(\beta\)</span>, <span class="math inline">\(\gamma\)</span>) are floxed and can be excised following activation of an ERT2-Cre transgene. In these cells, following the addition of tamoxifen, Cre translocates to the nucleus and excises the targeted alleles, resulting in triple deletion of HP1 proteins. Following 6 days of culture in the presence of tamoxifen, HP1 proteins were verified to be deleted from these cells. Here we were able to observe the effect of complete loss of HP1 proteins on DNA methylation as well as any changes in the distribution of H3K9me3, H4K20me3 and KAP1.</p>
<p>In addition to a large number of changes in gene transcription (data not shown), HP1cTKO ES cells show a de-repression of several ERVs, including IAPs, similar to that seen in HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKOs. ChIP-seq signal of KAP1, H3K9me3, and H4K20me3 was noticeably reduced from IAP elements (figure <a href="2-results.html#fig:IAPH3K9">2.31</a>). Consistent with the brain data, H4K20me3 appears to be completely lost, owing to deletion of HP1<span class="math inline">\(\gamma\)</span>. Noticeably, H3K9me3 and KAP1 are also reduced at IAP elements in HP1cTKO cells, implicating HP1 proteins to be necessary for assembly of the KAP1/SETDB1/SUV39H1 repressive complex at these repeats.</p>
</div>
<div id="dna-methylation-is-affected-following-loss-of-hp1" class="section level3">
<h3><span class="header-section-number">2.7.6</span> DNA Methylation is affected Following loss of HP1</h3>
<div id="non-cannonically-imprinted-genes-are-upregulated-in-hp1dko-hippocampi" class="section level4 unnumbered">
<h4><em>Non-Cannonically Imprinted Genes are Upregulated in HP1DKO Hippocampi</em></h4>
<p>While transcriptional changes to a subset of canonically imprinted genes could not be detected by in situ hybridization (section <a href="2-results.html#imprint">2.4.2</a>), the neuronally expressed Neuronatin ( <em>Nnat</em> ) and other non-canonically imprinted genes such as <em>Xlr3a</em> and <em>Xlr3c</em> are dysregulated in the HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO and aged HP1<span class="math inline">\(\beta\)</span>KO (clusters 4 &amp; 5, Figure <a href="2-results.html#fig:RNAsum">2.34</a>), and are summarized in table <a href="2-results.html#tab:imprintedtable">2.1</a>. While the majority of such genes are changed in DKO animals, Neuronatin shows de-repression in aged HP1<span class="math inline">\(\beta\)</span> KO, suggesting that this may be due to a deficiency in DNMT1 maintaining the methylation imprint over a lifespan without HP1<span class="math inline">\(\beta\)</span>.</p>
<table class="table table-striped table-hover table-condensed table-responsive" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:imprintedtable">Table 2.1: </span>Imprinted Loci showing Elevated Transcription in HP1DKO Hippocampi
</caption>
<thead>
<tr>
<th style="text-align:left;">
Gene
</th>
<th style="text-align:left;">
Class
</th>
<th style="text-align:left;">
Chromosome
</th>
<th style="text-align:right;">
LogFC
</th>
<th style="text-align:right;">
logCPM
</th>
<th style="text-align:right;">
Pvalue
</th>
<th style="text-align:right;">
FDR
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Xlr3a
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:right;">
5.778093
</td>
<td style="text-align:right;">
1.2524118
</td>
<td style="text-align:right;">
0.0001989
</td>
<td style="text-align:right;">
0.0272010
</td>
</tr>
<tr>
<td style="text-align:left;">
Xlr3c
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:right;">
8.440795
</td>
<td style="text-align:right;">
1.7659862
</td>
<td style="text-align:right;">
0.0000003
</td>
<td style="text-align:right;">
0.0002928
</td>
</tr>
<tr>
<td style="text-align:left;">
Gm364
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:right;">
6.553524
</td>
<td style="text-align:right;">
0.5700350
</td>
<td style="text-align:right;">
0.0000035
</td>
<td style="text-align:right;">
0.0018841
</td>
</tr>
<tr>
<td style="text-align:left;">
Mkrn3
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
7
</td>
<td style="text-align:right;">
3.510904
</td>
<td style="text-align:right;">
2.4471193
</td>
<td style="text-align:right;">
0.0000000
</td>
<td style="text-align:right;">
0.0000146
</td>
</tr>
<tr>
<td style="text-align:left;">
Tsga8
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:right;">
5.900168
</td>
<td style="text-align:right;">
0.3040211
</td>
<td style="text-align:right;">
0.0002019
</td>
<td style="text-align:right;">
0.0420649
</td>
</tr>
<tr>
<td style="text-align:left;">
Uba1y
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
Y
</td>
<td style="text-align:right;">
4.554711
</td>
<td style="text-align:right;">
1.6450681
</td>
<td style="text-align:right;">
0.0000002
</td>
<td style="text-align:right;">
0.0002569
</td>
</tr>
<tr>
<td style="text-align:left;">
Nnat
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:right;">
1.385290
</td>
<td style="text-align:right;">
7.4718652
</td>
<td style="text-align:right;">
0.0000006
</td>
<td style="text-align:right;">
0.0004993
</td>
</tr>
<tr>
<td style="text-align:left;">
Mdga1
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
17
</td>
<td style="text-align:right;">
1.176271
</td>
<td style="text-align:right;">
6.6211071
</td>
<td style="text-align:right;">
0.0001667
</td>
<td style="text-align:right;">
0.0357879
</td>
</tr>
</tbody>
</table>
</div>
<div id="hp1ctko-cells-show-abberant-dna-methylation" class="section level4 unnumbered">
<h4><em>HP1cTKO Cells Show Abberant DNA Methylation</em></h4>
<p>DNA methylation in HP1cTKO ES cells was measured by Reduced Read Bisulfite Sequencing (RRBSeq) (Figure <a href="2-results.html#fig:RRBScircos">2.32</a>). Following deletion of all HP1 proteins we could observe loss of DNA methylation at LINE and LTR repeats, and partial hypermethylation of promoters, exons and SINEs.</p>
<div class="figure" style="text-align: center"><span id="fig:RRBScircos"></span>
<img src="figure/results/HP1cTKO_RRBS.png" alt="HP1cTKO ES Cells Show Abberant DNA Methylation. (A) Scatterplot of methylation in HP1cTKO and WT cells. Hypermethylation of exons and promoters occurs, while intergenic regions become hypomethylated (B). A Circos plot (C) visualizes all changes occuring in the HP1cTKO RRBS data relative to WT by chromosome. The magnitude of change is represented on the radial y axis. The outermost ring represents methylation change density.  Inner rings annotate methylation changes occuring to CpG islands, exons, LTRs, LINEs and SINEs respectively." width="100%"   />
<p class="caption">
Figure 2.32: HP1cTKO ES Cells Show Abberant DNA Methylation. (A) Scatterplot of methylation in HP1cTKO and WT cells. Hypermethylation of exons and promoters occurs, while intergenic regions become hypomethylated (B). A Circos plot (C) visualizes all changes occuring in the HP1cTKO RRBS data relative to WT by chromosome. The magnitude of change is represented on the radial y axis. The outermost ring represents methylation change density. Inner rings annotate methylation changes occuring to CpG islands, exons, LTRs, LINEs and SINEs respectively.
</p>
</div>
</div>
</div>
<div id="kap1-is-lost-from-icrs-in-hp1ctko-cells" class="section level3 unnumbered">
<h3><em>KAP1 is lost from ICRs in HP1cTKO Cells</em></h3>
<p>To understand how imprinted loci may be affected in HP1cTKO ES cells, I examined Imprinting Control Regions (ICRs) that overlap with ZFP57 binding (data from <span class="citation">(Riso et al. <a href="#ref-RisoZFP57maintainsparentoforiginspecific2016">2016</a>)</span>). Figure <a href="2-results.html#fig:KAP1ICR">2.33</a>A shows the average binding of KAP1, H3K9me3 and H4K20me3 across these ICRs, centered at the ZFP57 peak. We observe a distinct loss of KAP1 at ICRs, one example of which is Neuronatin ( <em>Nnat</em> , Figure <a href="2-results.html#fig:KAP1ICR">2.33</a>B) where KAP1 binding near the CpG island is lost and RRBS shows a shift towards DNA hypermethylation at the CpG. Notably, like HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO hippocampi, <em>Nnat</em> expression is also elevated in HP1cTKO ES cells (Figure <a href="2-results.html#fig:KAP1ICR">2.33</a>B, bottom track). As methylation of the CpG would normally be associated with transcriptional repression, it is likely the signal observed here is the transcriptionally permissive 5-hydroxymethylation (5hmC) modification, which cannot be distinguished from 5mC in bisulfite sequencing.</p>
<div class="figure" style="text-align: center"><span id="fig:KAP1ICR"></span>
<img src="figure/results/KAP1_ICR_profile.png" alt="KAP1 binding to Imprinting Control Regions is Lost in HP1cTKO ES Cells. (A) KAP1 binding around ZFP57 peaks (Riso 2014) at 20 Imprinting Control Regions (ICRs) is lost in HP1cTKO ES cells. (B) Epigenetic Disorder at the Nnat ICR in HP1cTKO ES cells. RRBS track: Blue if methylation is 0-40\%, red if methylation is 60-100\%, and white if methylation is 40\%-60\%." width="75%"   />
<p class="caption">
Figure 2.33: KAP1 binding to Imprinting Control Regions is Lost in HP1cTKO ES Cells. (A) KAP1 binding around ZFP57 peaks (Riso 2014) at 20 Imprinting Control Regions (ICRs) is lost in HP1cTKO ES cells. (B) Epigenetic Disorder at the Nnat ICR in HP1cTKO ES cells. RRBS track: Blue if methylation is 0-40%, red if methylation is 60-100%, and white if methylation is 40%-60%.
</p>
</div>
<p></p>
</div>
</div>
<div id="RNAresults" class="section level2">
<h2><span class="header-section-number">2.8</span> Transcriptional Dysregulation following HP1 Deficiency Drives UPR and Inflammatory Responses</h2>
<p>To gain insight into any transcriptional de-regulation occurring in HP1 mutant hippocampi, 32bp paired-end total RNAseq was performed to detect any changes in both coding and non-coding RNA, including repetitive elements.</p>
<div id="genotype-related-changes" class="section level3">
<h3><span class="header-section-number">2.8.1</span> Genotype Related Changes</h3>
<div id="differential-gene-expression" class="section level4 unnumbered">
<h4><em>Differential Gene Expression</em></h4>
<p>Surprisingly, relatively few genes were significantly changed in HP1 mutant hippocampi. In examining such changes both a coarse analysis (Benjamin-Hochberg FDR &lt; 0.05) and a stringent analysis (FDR &lt; 0.01) was undertaken. To understand the primary effects the stringent analysis will first be discussed, followed by pathway analysis using the coarse analysis.</p>
<p>Testing for significant changes between genotypes in the aged timepoint yields the heatmap seen in Figure <a href="2-results.html#fig:RNAsum">2.34</a>A where data from young animals is included (Figure <a href="2-results.html#fig:RNAsum">2.34</a>B. Notably, testing within batch for differential gene expression between WT and DKO genotypes in young animals largely gives the same gene set (data not shown), suggesting transcriptional differences are minimal between the 3 month DKO and the 13 month DKO.</p>
<p>Hierarchical clustering of differentially expressed genes reveals distinct sets associated with genotype (Figure <a href="2-results.html#fig:RNAsum">2.34</a>B). Clusters 1 &amp; 2 include genes that are underrepresented compared to WT. It is clear that cluster 1, which shows underrepresentation in HP1<span class="math inline">\(\beta\)</span>KO and HP1DKO, comprises genes associated with the dentate gyrus, so their underrepresentation is not due to altered expression, rather a complete lack of the tissue that normally expresses them. Cluster 2 comprises a handful of genes that are in the vicinity of the HP1<span class="math inline">\(\gamma\)</span> gene locus, which infers their expression was affected due to local genomic changes following deletion of the HP1<span class="math inline">\(\gamma\)</span> gene.</p>
<p>Genes and repeats in clusters 3, 4 and 5 have elevated expression compared to wildtype, due to either singular loss of HP1<span class="math inline">\(\gamma\)</span> (cluster 3), HP1<span class="math inline">\(\beta\)</span> (cluster 4) or both (cluster 5). Strikingly, from cluster 3 we can observe that the primary effect due to loss of HP1<span class="math inline">\(\gamma\)</span> is elevated transcription of the protocadherin cluster on chromosome 18, which will be discussed later. Cluster 4 contains genes which are elevated in aged HP1<span class="math inline">\(\beta\)</span> animals and HP1DKO animals, one of which is the known imprinted gene Neuronatin ( <em>Nnat</em> ).</p>
<p>Cluster 5 contains protein coding genes, lncRNAs and repeats which show elevated transcription only in the HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> double mutant and will be discussed in the next section.</p>
<div class="figure" style="text-align: center"><span id="fig:RNAsum"></span>
<img src="figure/results/RNA_summary.png" alt="High Confidence Differential Gene Expression in HP1 Mutant Adult Hippocampi. Volcano plots (A) display log fold changes with respect to corrected p value (aged).  (B) Hierarchical clustering of differentially expressed genes (corrected p &lt; 0.01)  across all samples." width="100%"   />
<p class="caption">
Figure 2.34: High Confidence Differential Gene Expression in HP1 Mutant Adult Hippocampi. Volcano plots (A) display log fold changes with respect to corrected p value (aged). (B) Hierarchical clustering of differentially expressed genes (corrected p &lt; 0.01) across all samples.
</p>
</div>
<p></p>
</div>
<div id="read-coverage-of-lncrnas" class="section level4 unnumbered">
<h4><em>Read Coverage of lncRNAs</em></h4>
<p>Given the statistically significant de-repression of several lncRNAs, I looked at lncRNA read coverage across the entire genome in each of the mutants (Figure <a href="2-results.html#fig:lncRNAcov">2.35</a>). Global de-repression of annotated long non-coding RNAs infers significant alterations to heterochromatin boundaries. Notably, transcription of lncRNAs increases steadily with age, in an effect that appears to be dependent on HP1<span class="math inline">\(\beta\)</span>.</p>
<div class="figure" style="text-align: center"><span id="fig:lncRNAcov"></span>
<img src="figure/results/lncRNAcov.png" alt="LncRNA Transcription is Elevated in HP1 Mutants.  Read coverage heatmaps of all lncRNAs genome-wide (bottom) plot all alignments per 50bp bin for each lncRNA locus including 1 kilobase upstream of the transcription start site (TSS) and 1 kilobase downstream of the transcription end site (TES), which are scaled to be the same size.  Read coverage profiles (top) profide an overview of genome-wide changes in lncRNA transcription." width="100%"   />
<p class="caption">
Figure 2.35: LncRNA Transcription is Elevated in HP1 Mutants. Read coverage heatmaps of all lncRNAs genome-wide (bottom) plot all alignments per 50bp bin for each lncRNA locus including 1 kilobase upstream of the transcription start site (TSS) and 1 kilobase downstream of the transcription end site (TES), which are scaled to be the same size. Read coverage profiles (top) profide an overview of genome-wide changes in lncRNA transcription.
</p>
</div>

</div>
<div id="read-coverage-at-repeat-flanks" class="section level4 unnumbered">
<h4><em>Read Coverage at Repeat Flanks</em></h4>
<p>A similar analysis was undertaken for LINE, SINE and LTR repeat classes (Figure <a href="2-results.html#fig:readcov">2.36</a>). In the case of repeats, observing transcription at sequences that are upstream or downstream (ie sequences that ‘flank’ the repeat) can be informative for cases where the repeat cannot be mapped. From (Figure <a href="2-results.html#fig:readcov">2.36</a>A), where the read coverage in repeat flanks is markedly higher than the internal repeat, we can assume that our measured counts of repeat transcripts are an underestimate of what is actually occurring <em>in vivo</em>. Significant de-repression of repeat type plotted by class (Figure <a href="2-results.html#fig:readcov">2.36</a>B) reveals the most significant changes occur in the ERVK and ERV1 families.</p>
<div class="figure" style="text-align: center"><span id="fig:readcov"></span>
<img src="figure/results/Repeatcovsig.png" alt="Read Coverage at Repeat Flanks in HP1 Mutants and Significant Changes by Repeat Class. (A) Read coverage profiles and heatmaps of LINE, SINE and LTR repeat classes.  Repeats are scaled to be aligned by their 5' end and their 3' end in addition to 1kb upstream and downstream. (B) Significant changes in repeats by repeat familly.  The most significant changes in HP1DKO belong to ERVK and ERV1 families." width="100%"   />
<p class="caption">
Figure 2.36: Read Coverage at Repeat Flanks in HP1 Mutants and Significant Changes by Repeat Class. (A) Read coverage profiles and heatmaps of LINE, SINE and LTR repeat classes. Repeats are scaled to be aligned by their 5’ end and their 3’ end in addition to 1kb upstream and downstream. (B) Significant changes in repeats by repeat familly. The most significant changes in HP1DKO belong to ERVK and ERV1 families.
</p>
</div>
<p></p>
</div>
<div id="chimeric-transcription" class="section level4 unnumbered">
<h4><em>Chimeric Transcription</em></h4>
<p>Given the elevated transcription of ERV elements, I examined how this may contribute to chimeric transcription events-where transcription begins in a repeat-such as an LTR, and ‘reads into’ a coding gene or exon. The resultant transcript is a ‘chimera’ where it contains both the repeat and exon sequence. This may result in alternative splicing events or direct activation of entire genes. To find such transcripts I implemented a customized version of the LIONS pipeline <span class="citation">(Babaian et al. <a href="#ref-BabaianLIONSAnalysisSuite2018">2018</a>)</span> to run on the BIH CUBI cluster. A strong trend can be observed in LTR-derived chimeric transcripts in DKO animals (Figure <a href="2-results.html#fig:chimeras">2.37</a>), where increased transcription of the repeat correlates with increased transcription of the adjacent exon. While most chimeric events are small, pertaining to exon inclusion or retention, elevated transcription in a subset of genes in Cluster 5 is due to direct activation by an adjacent repeat (Table <a href="2-results.html#tab:chimeratable">2.2</a> &amp; Figure <a href="2-results.html#fig:chimeras">2.37</a>). </p>
<table class="table table-striped table-hover table-condensed table-responsive" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:chimeratable">Table 2.2: </span>Gene Expression Changes in HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO Hippocampi due to Chimeric Transcription Originating in an Adjacent Repeat
</caption>
<thead>
<tr>
<th style="text-align:left;">
Name
</th>
<th style="text-align:left;">
Class
</th>
<th style="text-align:left;">
AdjacentRepeat
</th>
<th style="text-align:right;">
logFC
</th>
<th style="text-align:right;">
PValue
</th>
<th style="text-align:right;">
FDR
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Gm48619
</td>
<td style="text-align:left;">
lncRNA
</td>
<td style="text-align:left;">
IAP
</td>
<td style="text-align:right;">
6.809431
</td>
<td style="text-align:right;">
0.0000000
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
Kank4os
</td>
<td style="text-align:left;">
antisense
</td>
<td style="text-align:left;">
RLTR31C_MM
</td>
<td style="text-align:right;">
8.334549
</td>
<td style="text-align:right;">
0.0000000
</td>
<td style="text-align:right;">
0.0000052
</td>
</tr>
<tr>
<td style="text-align:left;">
Gm28321
</td>
<td style="text-align:left;">
lncRNA
</td>
<td style="text-align:left;">
IAP
</td>
<td style="text-align:right;">
9.281399
</td>
<td style="text-align:right;">
0.0000000
</td>
<td style="text-align:right;">
0.0000052
</td>
</tr>
<tr>
<td style="text-align:left;">
Got1l1
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
multiple LTRs
</td>
<td style="text-align:right;">
3.685565
</td>
<td style="text-align:right;">
0.0000002
</td>
<td style="text-align:right;">
0.0002764
</td>
</tr>
<tr>
<td style="text-align:left;">
E230025N22Rik
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
mixed
</td>
<td style="text-align:right;">
3.852652
</td>
<td style="text-align:right;">
0.0000004
</td>
<td style="text-align:right;">
0.0003757
</td>
</tr>
<tr>
<td style="text-align:left;">
Akr1c13
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
ERVB7?
</td>
<td style="text-align:right;">
6.067095
</td>
<td style="text-align:right;">
0.0000005
</td>
<td style="text-align:right;">
0.0004745
</td>
</tr>
<tr>
<td style="text-align:left;">
Gramd1c
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
IAP
</td>
<td style="text-align:right;">
1.901830
</td>
<td style="text-align:right;">
0.0000042
</td>
<td style="text-align:right;">
0.0021810
</td>
</tr>
<tr>
<td style="text-align:left;">
G730013B05Rik
</td>
<td style="text-align:left;">
lncRNA
</td>
<td style="text-align:left;">
RLTR31B2
</td>
<td style="text-align:right;">
6.867196
</td>
<td style="text-align:right;">
0.0000046
</td>
<td style="text-align:right;">
0.0022764
</td>
</tr>
<tr>
<td style="text-align:left;">
Olfr316
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
RLTR44C
</td>
<td style="text-align:right;">
2.337525
</td>
<td style="text-align:right;">
0.0000083
</td>
<td style="text-align:right;">
0.0038010
</td>
</tr>
<tr>
<td style="text-align:left;">
Gm15398
</td>
<td style="text-align:left;">
lncRNA
</td>
<td style="text-align:left;">
MTA_mm
</td>
<td style="text-align:right;">
6.430798
</td>
<td style="text-align:right;">
0.0000148
</td>
<td style="text-align:right;">
0.0063698
</td>
</tr>
<tr>
<td style="text-align:left;">
Serpinb1c
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
IAP
</td>
<td style="text-align:right;">
7.012703
</td>
<td style="text-align:right;">
0.0000343
</td>
<td style="text-align:right;">
0.0116408
</td>
</tr>
<tr>
<td style="text-align:left;">
Smarca5-ps
</td>
<td style="text-align:left;">
processed_pseudogene
</td>
<td style="text-align:left;">
IAPEY3-int
</td>
<td style="text-align:right;">
6.451726
</td>
<td style="text-align:right;">
0.0000365
</td>
<td style="text-align:right;">
0.0120489
</td>
</tr>
<tr>
<td style="text-align:left;">
Cyp2b13
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
IAP
</td>
<td style="text-align:right;">
7.271137
</td>
<td style="text-align:right;">
0.0000601
</td>
<td style="text-align:right;">
0.0125165
</td>
</tr>
<tr>
<td style="text-align:left;">
Mnd1
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
IAP
</td>
<td style="text-align:right;">
5.634051
</td>
<td style="text-align:right;">
0.0000676
</td>
<td style="text-align:right;">
0.0189726
</td>
</tr>
<tr>
<td style="text-align:left;">
Umodl1
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
RLTR17B_Mm
</td>
<td style="text-align:right;">
2.841678
</td>
<td style="text-align:right;">
0.0001121
</td>
<td style="text-align:right;">
0.0274364
</td>
</tr>
<tr>
<td style="text-align:left;">
Gsta2
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
MT2B
</td>
<td style="text-align:right;">
5.357106
</td>
<td style="text-align:right;">
0.0001169
</td>
<td style="text-align:right;">
0.0190674
</td>
</tr>
<tr>
<td style="text-align:left;">
Patj
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
IAP
</td>
<td style="text-align:right;">
1.513902
</td>
<td style="text-align:right;">
0.0001566
</td>
<td style="text-align:right;">
0.0346798
</td>
</tr>
<tr>
<td style="text-align:left;">
Dmp1
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
RMER19B and IAPLTR2b
</td>
<td style="text-align:right;">
1.824824
</td>
<td style="text-align:right;">
0.0001642
</td>
<td style="text-align:right;">
0.0243289
</td>
</tr>
<tr>
<td style="text-align:left;">
Btnl7-ps
</td>
<td style="text-align:left;">
unprocessed_pseudogene
</td>
<td style="text-align:left;">
IAP1-MM_I-int
</td>
<td style="text-align:right;">
4.094741
</td>
<td style="text-align:right;">
0.0001794
</td>
<td style="text-align:right;">
0.0255045
</td>
</tr>
<tr>
<td style="text-align:left;">
Gm13110
</td>
<td style="text-align:left;">
lncRNA
</td>
<td style="text-align:left;">
MTEa
</td>
<td style="text-align:right;">
1.889639
</td>
<td style="text-align:right;">
0.0001809
</td>
<td style="text-align:right;">
0.0255511
</td>
</tr>
<tr>
<td style="text-align:left;">
Platr25
</td>
<td style="text-align:left;">
protein_coding
</td>
<td style="text-align:left;">
mixed
</td>
<td style="text-align:right;">
1.979744
</td>
<td style="text-align:right;">
0.0004374
</td>
<td style="text-align:right;">
0.0457926
</td>
</tr>
</tbody>
</table>

<div class="figure" style="text-align: center"><span id="fig:chimeras"></span>
<img src="figure/results/Hexplot_thesis2.png" alt="Chimeric Transcription Events in HP1 Mutant Hippocampi.  (A) Co-transcription plots, where each point represents a chimeric transcript. Hexplots were used to correct for overplotting at low RPKM.  As can be observed in LTR-derived chimeras in aged HP1DKO animals (yellow square), higher transcription of the repeat (Reads Per Kilobase per Million reads (RPKM)), results in higher transcription of the exon portion of the chimera.  Black axes in each plot represent the mean Exon-RPKM (horizontal line) and the mean Repeat-RPKM (vertical line) in chimeras observed in wild-type of that repeat class and age. (B) An example chimeric transcript. An IAP element inside the first intron of Mnd1 results in transcriptional activation of the entire gene." width="100%"   />
<p class="caption">
Figure 2.37: Chimeric Transcription Events in HP1 Mutant Hippocampi. (A) Co-transcription plots, where each point represents a chimeric transcript. Hexplots were used to correct for overplotting at low RPKM. As can be observed in LTR-derived chimeras in aged HP1DKO animals (yellow square), higher transcription of the repeat (Reads Per Kilobase per Million reads (RPKM)), results in higher transcription of the exon portion of the chimera. Black axes in each plot represent the mean Exon-RPKM (horizontal line) and the mean Repeat-RPKM (vertical line) in chimeras observed in wild-type of that repeat class and age. (B) An example chimeric transcript. An IAP element inside the first intron of Mnd1 results in transcriptional activation of the entire gene.
</p>
</div>

</div>
</div>
<div id="longitudinal-changes-to-gene-expression" class="section level3">
<h3><span class="header-section-number">2.8.2</span> Longitudinal Changes To Gene Expression</h3>
<p>Following initial data exploration, a non-negligible batch effect (see methods <a href="6-bioinformatics-data-processing.html#tab:colData">6.1</a>) dictated that it was unfavorable to conduct direct longitudinal comparisons of WT and DKO (Young WT vs Aged WT and Young DKO vs Aged DKO). However, testing for differential expression in HP1<span class="math inline">\(\beta\)</span> and HP1<span class="math inline">\(\gamma\)</span> single mutants across age (within batch) yielded minimal results. For HP1<span class="math inline">\(\gamma\)</span>KO for instance, no differential expression passes an FDR &lt; 0.01 or 0.05. For age-related changes in HP1<span class="math inline">\(\beta\)</span>KO, no genes pass an FDR &lt; 0.01, and only a small number (184 genes) pass FDR &lt; 0.05. The top ten upregulated genes by ascending p value are presented in Table <a href="2-results.html#tab:BKOtop">2.3</a>, with C4b being a notable top hit. The complete gene list of age-related expression changes in HP1<span class="math inline">\(\beta\)</span>KO hippocampi is found in supplementary data <a href="5-supplemental-data.html#DE">5.2</a>.</p>
<table class="table table-striped table-hover table-condensed table-responsive" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:BKOtop">Table 2.3: </span>Top 10 Upregulated Genes due to Age in HP1<span class="math inline">\(\beta\)</span>KO
</caption>
<thead>
<tr>
<th style="text-align:left;">
Gene
</th>
<th style="text-align:left;">
Gene.Biotype
</th>
<th style="text-align:right;">
logFC
</th>
<th style="text-align:right;">
logCPM
</th>
<th style="text-align:right;">
F
</th>
<th style="text-align:right;">
p.Value
</th>
<th style="text-align:right;">
FDR
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
C4b
</td>
<td style="text-align:left;">
protein coding
</td>
<td style="text-align:right;">
1.968703
</td>
<td style="text-align:right;">
6.167731
</td>
<td style="text-align:right;">
40.90800
</td>
<td style="text-align:right;">
6.00e-07
</td>
<td style="text-align:right;">
0.0120851
</td>
</tr>
<tr>
<td style="text-align:left;">
Aldh16a1
</td>
<td style="text-align:left;">
protein coding
</td>
<td style="text-align:right;">
3.383482
</td>
<td style="text-align:right;">
2.733340
</td>
<td style="text-align:right;">
35.91748
</td>
<td style="text-align:right;">
1.70e-06
</td>
<td style="text-align:right;">
0.0132210
</td>
</tr>
<tr>
<td style="text-align:left;">
Rprm
</td>
<td style="text-align:left;">
protein coding
</td>
<td style="text-align:right;">
1.742288
</td>
<td style="text-align:right;">
4.544562
</td>
<td style="text-align:right;">
33.25780
</td>
<td style="text-align:right;">
3.20e-06
</td>
<td style="text-align:right;">
0.0132210
</td>
</tr>
<tr>
<td style="text-align:left;">
Ethe1
</td>
<td style="text-align:left;">
protein coding
</td>
<td style="text-align:right;">
2.724653
</td>
<td style="text-align:right;">
2.706537
</td>
<td style="text-align:right;">
32.55687
</td>
<td style="text-align:right;">
3.80e-06
</td>
<td style="text-align:right;">
0.0132210
</td>
</tr>
<tr>
<td style="text-align:left;">
Dmap1
</td>
<td style="text-align:left;">
protein coding
</td>
<td style="text-align:right;">
1.433555
</td>
<td style="text-align:right;">
4.434852
</td>
<td style="text-align:right;">
32.53388
</td>
<td style="text-align:right;">
3.80e-06
</td>
<td style="text-align:right;">
0.0132210
</td>
</tr>
<tr>
<td style="text-align:left;">
Gm26917
</td>
<td style="text-align:left;">
lincRNA
</td>
<td style="text-align:right;">
2.189478
</td>
<td style="text-align:right;">
5.305955
</td>
<td style="text-align:right;">
31.10771
</td>
<td style="text-align:right;">
5.30e-06
</td>
<td style="text-align:right;">
0.0160351
</td>
</tr>
<tr>
<td style="text-align:left;">
Caly
</td>
<td style="text-align:left;">
protein coding
</td>
<td style="text-align:right;">
1.699563
</td>
<td style="text-align:right;">
6.281637
</td>
<td style="text-align:right;">
29.16404
</td>
<td style="text-align:right;">
8.70e-06
</td>
<td style="text-align:right;">
0.0213195
</td>
</tr>
<tr>
<td style="text-align:left;">
Gstm4
</td>
<td style="text-align:left;">
protein coding
</td>
<td style="text-align:right;">
2.351643
</td>
<td style="text-align:right;">
3.047190
</td>
<td style="text-align:right;">
28.51131
</td>
<td style="text-align:right;">
1.03e-05
</td>
<td style="text-align:right;">
0.0216079
</td>
</tr>
<tr>
<td style="text-align:left;">
Alad
</td>
<td style="text-align:left;">
protein coding
</td>
<td style="text-align:right;">
1.680305
</td>
<td style="text-align:right;">
3.891168
</td>
<td style="text-align:right;">
27.85071
</td>
<td style="text-align:right;">
1.22e-05
</td>
<td style="text-align:right;">
0.0233221
</td>
</tr>
<tr>
<td style="text-align:left;">
Podxl2
</td>
<td style="text-align:left;">
protein coding
</td>
<td style="text-align:right;">
1.496893
</td>
<td style="text-align:right;">
6.484461
</td>
<td style="text-align:right;">
27.43499
</td>
<td style="text-align:right;">
1.36e-05
</td>
<td style="text-align:right;">
0.0238435
</td>
</tr>
</tbody>
</table>
</div>
<div id="pathway-analysis" class="section level3">
<h3><span class="header-section-number">2.8.3</span> Pathway Analysis</h3>
<div id="innate-immune-pathways-are-activated-from-prolonged-absence-of-hp1betako" class="section level4 unnumbered">
<h4><em>Innate Immune Pathways are Activated from Prolonged Absence of HP1<span class="math inline">\(\beta\)</span>KO</em></h4>
<p>In accordance with the changes observed in HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO hippocampi, the majority of the changes that occur due to sustained absence of HP1<span class="math inline">\(\beta\)</span>KO occurs in the realm of upregulation. Of the 184 genes differentially expressed in aged HP1<span class="math inline">\(\beta\)</span>KO hippocampi compared to their youthful counterparts, 43 genes (24%) are downregulated, such as Prox1 and others that may suggest a progressive loss of the dentate gyrus. The majority of changes lies in 141 genes (76%) which are upregulated. Upon testing for age-related changes to gene transcription in HP1<span class="math inline">\(\beta\)</span>KO hippocampi, aged HP1<span class="math inline">\(\beta\)</span>KO cluster strongly with HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO samples (Figure <a href="2-results.html#fig:BKOreactome">2.38</a>, left). Following a reactome pathway analysis (<a href="https://reactome.org" class="uri">https://reactome.org</a>, <span class="citation">Fabregat et al. (<a href="#ref-FabregatReactomePathwayKnowledgebase2018">2018</a>)</span>) it became clear that the sustained absence of HP1<span class="math inline">\(\beta\)</span> results in cellular stress and immune responses akin to viral infection (Figure <a href="2-results.html#fig:BKOreactome">2.38</a>, right).</p>
<div class="figure" style="text-align: center"><span id="fig:BKOreactome"></span>
<img src="figure/results/BKO_long_reactome.png" alt="Pathway Analysis of Longitudinal Changes in HP1$\beta$KO. (Left): Volcano plot and hierachical clustering following testing between aged HP1$\beta$KO and young HP1$\beta$KO. (Right): Top 25 reactome pathways from genes upregulated in aged HP1$\beta$KO." width="100%"   />
<p class="caption">
Figure 2.38: Pathway Analysis of Longitudinal Changes in HP1<span class="math inline">\(\beta\)</span>KO. (Left): Volcano plot and hierachical clustering following testing between aged HP1<span class="math inline">\(\beta\)</span>KO and young HP1<span class="math inline">\(\beta\)</span>KO. (Right): Top 25 reactome pathways from genes upregulated in aged HP1<span class="math inline">\(\beta\)</span>KO.
</p>
</div>
</div>
<div id="upr-and-innate-immune-pathways-are-activated-in-hp1betagamma-dko" class="section level4 unnumbered">
<h4><em>UPR and Innate Immune Pathways are Activated in HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO</em></h4>
<div class="figure" style="text-align: center"><span id="fig:reactome"></span>
<img src="figure/results/reactome_out.png" alt="Analysis of Pathways Upregulated in HP1$\beta$/$\gamma$ DKO Hippocampi.  (Left): Volcano plot and hierachical clustering following testing between all HP1$\beta$/$\gamma$ DKO (young and aged) and all WT (young and aged).  (Right): Top 25 rectome pathways from genes upregulated in HP1$\beta$/$\gamma$ DKO across ages." width="100%"   />
<p class="caption">
Figure 2.39: Analysis of Pathways Upregulated in HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO Hippocampi. (Left): Volcano plot and hierachical clustering following testing between all HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO (young and aged) and all WT (young and aged). (Right): Top 25 rectome pathways from genes upregulated in HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO across ages.
</p>
</div>
<p>To examine the cellular response to loss of HP1<span class="math inline">\(\beta\)</span> and HP1<span class="math inline">\(\gamma\)</span> and the de-repression of non-coding regions, the up-regulated gene set from the ‘coarse’ analysis (FDR &lt; 0.05) was analysed on reactome (Figure <a href="2-results.html#fig:reactome">2.39</a>). Hierarchical clustering of the coarse set reveals that transcriptomic signatures of HP1<span class="math inline">\(\gamma\)</span>KO hippocampi to be most similar to WT, excepting dysregulation of the protocadherin cluster (Figure <a href="2-results.html#fig:reactome">2.39</a>, left). In HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO hippocampi, upregulated pathways comprise the unfolded protein response, innate immune responses including complement activation, neuromodulatory pathways such as dopamine and serotonin release, amongst other pathways important for synaptic function. It should be noted that protocadherins (Pcdhs) do not contribute to neurotransmitter pathway over-representation, as they are not presently annotated in reactome.org.</p>
<p>If one compares genes differentially expressed in aged HP1<span class="math inline">\(\beta\)</span>KO with those changed in HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO, a small subset of genes overlap which include complement components <em>C4b</em> and <em>C1qa</em> (Figure <a href="2-results.html#fig:C4">2.40</a>A). While <em>Nnat</em> is highly significant in the HP1DKO dataset, it narrowly fails FDR in the young vs aged HP1<span class="math inline">\(\beta\)</span>KO test statistic. C4 protein can be observed in CA3 cell bodies as well as GFAP-positive astrocytes in Aged HP1<span class="math inline">\(\beta\)</span>KO and HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO hippocampi (Figure <a href="2-results.html#fig:C4">2.40</a>B).</p>
<div class="figure" style="text-align: center"><span id="fig:C4"></span>
<img src="figure/results/C4_GFAP.png" alt="(A) Intersection of Differential expression in HP1$\beta$/$\gamma$ DKO and Aged HP1$\beta$KO. (B) HP1$\beta$ Deficiency results in Elevated C4 Expression in CA3 Pyramidal Neurons" width="100%"   />
<p class="caption">
Figure 2.40: (A) Intersection of Differential expression in HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO and Aged HP1<span class="math inline">\(\beta\)</span>KO. (B) HP1<span class="math inline">\(\beta\)</span> Deficiency results in Elevated C4 Expression in CA3 Pyramidal Neurons
</p>
</div>
<p>The age-dependent upregulation of ncRNAs, repeats and ERVs in HP1<span class="math inline">\(\beta\)</span>KO, coincides with pathway analysis that suggests an anti-viral response (Figure <a href="2-results.html#fig:BKOreactome">2.38</a>) is instigated following prolonged de-repression of endogenous retroviruses in the hippocampus. As ERV de-repression is even more robust in HP1DKO hippocampi, the 689 transcripts differentially regulated in HP1DKO were cross checked against the interferome database (<a href="http://www.interferome.org" class="uri">http://www.interferome.org</a>), where ~15% (88/565 protein coding genes) were found to overlap with cellular responses to interferons. Notably, the majority of the overlap (55) occurred from a previously published experiment where cortical neurons were treated with Interferon <span class="math inline">\(\beta\)</span> ( <em>IFN<span class="math inline">\(\beta\)</span></em> ) and their transcriptome analysed. The immune response occurring in HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO neurons involves upregulation of innate immunity related <em>Oas3</em>, <em>Lyk6</em>, <em>Wdfy4</em>, <em>Zap70</em>, <em>Ly6e</em>, <em>Ctsz</em>, <em>Il34</em>, <em>Reep6</em>, <em>Crlf2</em>, complement <em>C3</em>, <em>C4b</em>, <em>C1qa</em>, apoptosis-related <em>Hrk</em> and <em>Aatk</em>, interferon response genes <em>Ifitm2</em>, <em>Ifi27</em>, and the regulatory component of the Interferon gamma receptor, <em>Ifngr2</em>. The complete list of genes overlapping with the interferome is found in supplementary table <a href="#tab:BKOinflam"><strong>??</strong></a>.</p>
<div class="figure" style="text-align: center"><span id="fig:EIF3"></span>
<img src="figure/results/EIF3.png" alt="HP1$\beta$/$\gamma$ DKO Hippocampi show Evidence of Stress Granules and increaseed Infiltration of Microglia. IHC of Aged CA3 fields using Hoechst (nuclei, Blue) EIF3$\eta$ (stress granules, green) and Iba1 (microglia, red).  HP1$\beta$KO and HP1$\beta$/$\gamma$ DKO show increased invasion of microglia (white arrows) into the Stratum pyramidale. The third row shows a single channel of EIF3$\eta$ staining intensity coded to the viridis colour palette on the 0-255 colour scale." width="100%"   />
<p class="caption">
Figure 2.41: HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO Hippocampi show Evidence of Stress Granules and increaseed Infiltration of Microglia. IHC of Aged CA3 fields using Hoechst (nuclei, Blue) EIF3<span class="math inline">\(\eta\)</span> (stress granules, green) and Iba1 (microglia, red). HP1<span class="math inline">\(\beta\)</span>KO and HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO show increased invasion of microglia (white arrows) into the Stratum pyramidale. The third row shows a single channel of EIF3<span class="math inline">\(\eta\)</span> staining intensity coded to the viridis colour palette on the 0-255 colour scale.
</p>
</div>
<p>Additionally, multiple genes normally attributed to chemotaxis/invasion ( <em>Mmp14</em>, <em>Mmp17</em>, <em>Mpped1</em>, <em>Adam11</em>, <em>Cck</em>) and phagocytosis ( <em>Cd302</em> and the synapse-specific <em>C1ql2</em>) were differentially affected. This suggested that the end result of the inflammatory cascade could be resulting in the activation of microglia. Recent work has elucidated a role for microglia in synaptic pruning which appears to rely on complement proteins in the CNS <span class="citation">(Stevens et al. <a href="#ref-StevensClassicalComplementCascade2007">2007</a>)</span>, and dysregulation of complement proteins is now understood to underlie a wide range of inflammatory diseases in the CNS including neurodegeneration <span class="citation">(Bodea et al. <a href="#ref-BodeaNeurodegenerationActivationMicroglial2014">2014</a>)</span>, Multiple sclerosis <span class="citation">(Christensen et al. <a href="#ref-ChristensenGeneenvironmentinteractions2007a">2007</a>)</span>, Alzheimer’s’ <span class="citation">(Fischer et al. <a href="#ref-FischerComplementC1qC31995">1995</a>)</span>, ALS <span class="citation">(Lee et al. <a href="#ref-LeeComplementcomponentsare2018">2018</a>)</span>, and normal aging <span class="citation">(Stephan et al. <a href="#ref-StephanDramaticIncreaseC1q2013">2013</a>)</span>. The sustained elevated transcription of <em>C3</em>, <em>C4</em> and <em>C1qa</em> in HP1<span class="math inline">\(\beta\)</span>KO and HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO hippocampi is likely to result in increased infiltration and phagocytosis of neuronal synapses by microglia. This can be summarily seen in figure <a href="2-results.html#fig:EIF3">2.41</a>, where increased transcription of lncRNA and retroelements due to loss of HP1<span class="math inline">\(\beta\)</span> and HP1<span class="math inline">\(\gamma\)</span> has resulted in innate immune and unfolded protein responses. In the <em>Stratum pyramidale</em> where the CA3 cell bodies reside, an innate immune response has induced invasion of microglia while condensation of EIF3<span class="math inline">\(\eta\)</span>+ stress granules is visible in CA3 cell bodies, indicative of UPR activation there.</p>
<p>All of the players in the aforementioned pathway have known links to inflammatory disease states, neurodegeneration, and normal aging. The potential causes of these cascades, along with their relevance to human aging and disease, will be examined in the next section.</p>

</div>
</div>
</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-VoonMaintainingmemorysilencing2016">
<p>Voon, Hsiao PJ, and Richard J. Gibbons. 2016. “Maintaining Memory of Silencing at Imprinted Differentially Methylated Regions.” <em>Cellular and Molecular Life Sciences</em> 73 (9): 1871–9.</p>
</div>
<div id="ref-McConnellMosaiccopynumber2013">
<p>McConnell, Michael J., Michael R. Lindberg, Kristen J. Brennand, Julia C. Piper, Thierry Voet, Chris Cowing-Zitron, Svetlana Shumilina, Roger S. Lasken, Joris R. Vermeesch, and Ira M. Hall. 2013. “Mosaic Copy Number Variation in Human Neurons.” <em>Science</em> 342 (6158): 632–37.</p>
</div>
<div id="ref-MuotriEnvironmentalinfluenceL12009">
<p>Muotri, Alysson R., Chunmei Zhao, Maria C.N. Marchetto, and Fred H. Gage. 2009. “Environmental Influence on L1 Retrotransposons in the Adult Hippocampus.” <em>Hippocampus</em> 19 (10): 1002–7. doi:<a href="https://doi.org/10.1002/hipo.20564">10.1002/hipo.20564</a>.</p>
</div>
<div id="ref-ThomasLINE1RetrotranspositionNervous2012">
<p>Thomas, Charles A., Apuã C.M. Paquola, and Alysson R. Muotri. 2012. “LINE-1 Retrotransposition in the Nervous System.” <em>Annual Review of Cell and Developmental Biology</em> 28 (1): 555–73. doi:<a href="https://doi.org/10.1146/annurev-cellbio-101011-155822">10.1146/annurev-cellbio-101011-155822</a>.</p>
</div>
<div id="ref-MacfarlanEmbryonicstemcell2012">
<p>Macfarlan, Todd S., Wesley D. Gifford, Shawn Driscoll, Karen Lettieri, Helen M. Rowe, Dario Bonanomi, Amy Firth, Oded Singer, Didier Trono, and Samuel L. Pfaff. 2012. “Embryonic Stem Cell Potency Fluctuates with Endogenous Retrovirus Activity.” <em>Nature</em> 487 (June): 57. <a href="http://dx.doi.org/10.1038/nature11244" class="uri">http://dx.doi.org/10.1038/nature11244</a>.</p>
</div>
<div id="ref-DewannieuxIdentificationautonomousIAP2004">
<p>Dewannieux, Marie, Anne Dupressoir, Francis Harper, G’erard Pierron, and Thierry Heidmann. 2004. “Identification of Autonomous IAP LTR Retrotransposons Mobile in Mammalian Cells.” <em>Nature Genetics</em> 36 (5): 534–39. doi:<a href="https://doi.org/10.1038/ng1353">10.1038/ng1353</a>.</p>
</div>
<div id="ref-MaksakovaRetroviralElementsTheir2006">
<p>Maksakova, Irina A., Mark T. Romanish, Liane Gagnier, Catherine A. Dunn, Louie N. van de Lagemaat, and Dixie L. Mager. 2006. “Retroviral Elements and Their Hosts: Insertional Mutagenesis in the Mouse Germ Line.” <em>PLoS Genetics</em> 2 (1): e2. doi:<a href="https://doi.org/10.1371/journal.pgen.0020002">10.1371/journal.pgen.0020002</a>.</p>
</div>
<div id="ref-SaitoTranslationnonautonomoustype2008">
<p>Saito, E.-s., V. W. Keng, J. Takeda, and K. Horie. 2008. “Translation from Nonautonomous Type IAP Retrotransposon Is a Critical Determinant of Transposition Activity: Implication for Retrotransposon-Mediated Genome Evolution.” <em>Genome Research</em> 18 (6): 859–68. doi:<a href="https://doi.org/10.1101/gr.069310.107">10.1101/gr.069310.107</a>.</p>
</div>
<div id="ref-HorieRetrotransposonsInfluenceMouse2007a">
<p>Horie, Kyoji, Ei-suke Saito, Vincent W. Keng, Ryuji Ikeda, Hiroshi Ishihara, and Junji Takeda. 2007. “Retrotransposons Influence the Mouse Transcriptome: Implication for the Divergence of Genetic Traits.” <em>Genetics</em> 176 (2): 815–27. doi:<a href="https://doi.org/10.1534/genetics.107.071647">10.1534/genetics.107.071647</a>.</p>
</div>
<div id="ref-AucottHP1requireddevelopment2008">
<p>Aucott, R., J. Bullwinkel, Y. Yu, W. Shi, M. Billur, J. P. Brown, U. Menzel, et al. 2008. “HP1- Is Required for Development of the Cerebral Neocortex and Neuromuscular Junctions.” <em>The Journal of Cell Biology</em> 183 (4): 597–606. doi:<a href="https://doi.org/10.1083/jcb.200804041">10.1083/jcb.200804041</a>.</p>
</div>
<div id="ref-MaksakovaH3K9me3bindingproteinsare2011">
<p>Maksakova, Irina A., Preeti Goyal, Jörn Bullwinkel, Jeremy P. Brown, Misha Bilenky, Dixie L. Mager, Prim B. Singh, and Matthew C. Lorincz. 2011. “H3K9me3-Binding Proteins Are Dispensable for Setdb1/H3K9me3-Dependent Retroviral Silencing.” <em>Epigenetics Chromatin</em> 4 (1): 12. <a href="http://www.biomedcentral.com/content/pdf/1756-8935-4-12.pdf" class="uri">http://www.biomedcentral.com/content/pdf/1756-8935-4-12.pdf</a>.</p>
</div>
<div id="ref-KourmouliEpigeneticregulationmammalian2005">
<p>Kourmouli, Niki, Yuh-man Sun, Sjaak van der Sar, Prim B. Singh, and Jeremy P. Brown. 2005. “Epigenetic Regulation of Mammalian Pericentric Heterochromatin in Vivo by Hp1.” <em>Biochemical and Biophysical Research Communications</em> 337 (3): 901–7. doi:<a href="https://doi.org/10.1016/j.bbrc.2005.09.132">10.1016/j.bbrc.2005.09.132</a>.</p>
</div>
<div id="ref-Schottasilencingpathwayinduce2004">
<p>Schotta, G. 2004. “A Silencing Pathway to Induce H3-K9 and H4-K20 Trimethylation at Constitutive Heterochromatin.” <em>Genes &amp; Development</em> 18 (11): 1251–62. doi:<a href="https://doi.org/10.1101/gad.300704">10.1101/gad.300704</a>.</p>
</div>
<div id="ref-RisoZFP57maintainsparentoforiginspecific2016">
<p>Riso, Vincenzo, Marco Cammisa, Harpreet Kukreja, Zahra Anvar, Gaetano Verde, Angela Sparago, Basilia Acurzio, et al. 2016. “ZFP57 Maintains the Parent-of-Origin-Specific Expression of the Imprinted Genes and Differentially Affects Non-Imprinted Targets in Mouse Embryonic Stem Cells.” <em>Nucleic Acids Research</em> 44 (17): 8165–78. doi:<a href="https://doi.org/10.1093/nar/gkw505">10.1093/nar/gkw505</a>.</p>
</div>
<div id="ref-BabaianLIONSAnalysisSuite2018">
<p>Babaian, Artem, Richard Thompson, Jake Lever, Liane Gagnier, Mohammad M. Karimi, and Dixie L. Mager. 2018. “LIONS: Analysis Suite for Detecting and Quantifying Transposable Element Initiated Transcription from RNA-Seq.” <em>bioRxiv</em>, August, 149864. doi:<a href="https://doi.org/10.1101/149864">10.1101/149864</a>.</p>
</div>
<div id="ref-FabregatReactomePathwayKnowledgebase2018">
<p>Fabregat, Antonio, Steven Jupe, Lisa Matthews, Konstantinos Sidiropoulos, Marc Gillespie, Phani Garapati, Robin Haw, et al. 2018. “The Reactome Pathway Knowledgebase.” <em>Nucleic Acids Research</em> 46 (D1): D649–D655. doi:<a href="https://doi.org/10.1093/nar/gkx1132">10.1093/nar/gkx1132</a>.</p>
</div>
<div id="ref-StevensClassicalComplementCascade2007">
<p>Stevens, Beth, Nicola J. Allen, Luis E. Vazquez, Gareth R. Howell, Karen S. Christopherson, Navid Nouri, Kristina D. Micheva, et al. 2007. “The Classical Complement Cascade Mediates CNS Synapse Elimination.” <em>Cell</em> 131 (6): 1164–78. doi:<a href="https://doi.org/10.1016/j.cell.2007.10.036">10.1016/j.cell.2007.10.036</a>.</p>
</div>
<div id="ref-BodeaNeurodegenerationActivationMicroglial2014">
<p>Bodea, Liviu-Gabriel, Yiner Wang, Bettina Linnartz-Gerlach, Jens Kopatz, Lasse Sinkkonen, Ruth Musgrove, Tony Kaoma, et al. 2014. “Neurodegeneration by Activation of the Microglial ComplementPhagosome Pathway.” <em>Journal of Neuroscience</em> 34 (25): 8546–56. doi:<a href="https://doi.org/10.1523/JNEUROSCI.5002-13.2014">10.1523/JNEUROSCI.5002-13.2014</a>.</p>
</div>
<div id="ref-ChristensenGeneenvironmentinteractions2007a">
<p>Christensen, Tove, Thor Petersen, Steffen Thiel, Tomasz Brudek, Svend Ellermann-Eriksen, and Ann’e Møller-Larsen. 2007. “GeneEnvironment Interactions in Multiple Sclerosis: Innate and Adaptive Immune Responses to Human Endogenous Retrovirus and Herpesvirus Antigens and the Lectin Complement Activation Pathway.” <em>Journal of Neuroimmunology</em> 183 (1): 175–88. doi:<a href="https://doi.org/10.1016/j.jneuroim.2006.09.014">10.1016/j.jneuroim.2006.09.014</a>.</p>
</div>
<div id="ref-FischerComplementC1qC31995">
<p>Fischer, B., H. Schmoll, D. Platt, A. Popa-Wagner, P. Riederer, and J. Bauer. 1995. “Complement C1q and C3 mRNA Expression in the Frontal Cortex of Alzheimer’s Patients.” <em>Journal of Molecular Medicine</em> 73 (9): 465–71. doi:<a href="https://doi.org/10.1007/BF00202265">10.1007/BF00202265</a>.</p>
</div>
<div id="ref-LeeComplementcomponentsare2018">
<p>Lee, John D., Samantha C. Levin, Emily F. Willis, Rui Li, Trent M. Woodruff, and Peter G. Noakes. 2018. “Complement Components Are Upregulated and Correlate with Disease Progression in the TDP-43q331k Mouse Model of Amyotrophic Lateral Sclerosis.” <em>Journal of Neuroinflammation</em> 15 (1): 171. doi:<a href="https://doi.org/10.1186/s12974-018-1217-2">10.1186/s12974-018-1217-2</a>.</p>
</div>
<div id="ref-StephanDramaticIncreaseC1q2013">
<p>Stephan, Alexander H., Daniel V. Madison, Jos’e María Mateos, Deborah A. Fraser, Emilie A. Lovelett, Laurence Coutellier, Leo Kim, et al. 2013. “A Dramatic Increase of C1q Protein in the CNS During Normal Aging.” <em>The Journal of Neuroscience</em> 33 (33): 13460–74. doi:<a href="https://doi.org/10.1523/JNEUROSCI.1333-13.2013">10.1523/JNEUROSCI.1333-13.2013</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="1-introduction.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="3-discussion.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"download": null,
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
