<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Part 3 Discussion | HP1 Deficiency Results in Accelerated Neurodegeneration</title>
  <meta name="description" content="Part 3 Discussion | HP1 Deficiency Results in Accelerated Neurodegeneration">
  <meta name="generator" content="bookdown  and GitBook 2.6.7">

  <meta property="og:title" content="Part 3 Discussion | HP1 Deficiency Results in Accelerated Neurodegeneration" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Part 3 Discussion | HP1 Deficiency Results in Accelerated Neurodegeneration" />
  
  
  

<meta name="author" content="Andrew G. Newman">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="2-results.html">
<link rel="next" href="4-methods.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />







<script src="libs/kePrint-0.0.1/kePrint.js"></script>


<style type="text/css">
div.sourceCode { overflow-x: auto; }
table.sourceCode, tr.sourceCode, td.lineNumbers, td.sourceCode {
  margin: 0; padding: 0; vertical-align: baseline; border: none; }
table.sourceCode { width: 100%; line-height: 100%; }
td.lineNumbers { text-align: right; padding-right: 4px; padding-left: 4px; color: #aaaaaa; border-right: 1px solid #aaaaaa; }
td.sourceCode { padding-left: 5px; }
code > span.kw { color: #007020; font-weight: bold; } /* Keyword */
code > span.dt { color: #902000; } /* DataType */
code > span.dv { color: #40a070; } /* DecVal */
code > span.bn { color: #40a070; } /* BaseN */
code > span.fl { color: #40a070; } /* Float */
code > span.ch { color: #4070a0; } /* Char */
code > span.st { color: #4070a0; } /* String */
code > span.co { color: #60a0b0; font-style: italic; } /* Comment */
code > span.ot { color: #007020; } /* Other */
code > span.al { color: #ff0000; font-weight: bold; } /* Alert */
code > span.fu { color: #06287e; } /* Function */
code > span.er { color: #ff0000; font-weight: bold; } /* Error */
code > span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
code > span.cn { color: #880000; } /* Constant */
code > span.sc { color: #4070a0; } /* SpecialChar */
code > span.vs { color: #4070a0; } /* VerbatimString */
code > span.ss { color: #bb6688; } /* SpecialString */
code > span.im { } /* Import */
code > span.va { color: #19177c; } /* Variable */
code > span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code > span.op { color: #666666; } /* Operator */
code > span.bu { } /* BuiltIn */
code > span.ex { } /* Extension */
code > span.pp { color: #bc7a00; } /* Preprocessor */
code > span.at { color: #7d9029; } /* Attribute */
code > span.do { color: #ba2121; font-style: italic; } /* Documentation */
code > span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code > span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code > span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
</style>

</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="preface.html"><a href="preface.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="" data-path="contribution-of-authors.html"><a href="contribution-of-authors.html"><i class="fa fa-check"></i>Contribution of Authors</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="1" data-path="1-introduction.html"><a href="1-introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="1-introduction.html"><a href="1-introduction.html#epigenetics"><i class="fa fa-check"></i><b>1.1</b> Epigenetics</a><ul>
<li><a href="1-introduction.html#dna-modifications"><em>DNA Modifications</em></a></li>
<li><a href="1-introduction.html#chromatin"><em>Chromatin</em></a></li>
<li><a href="1-introduction.html#chromatin-remodelling"><em>Chromatin Remodelling</em></a></li>
<li><a href="1-introduction.html#krab-zfps-transposable-elements-and-exaptation"><em>KRAB-ZFPs, Transposable Elements and Exaptation</em></a></li>
<li><a href="1-introduction.html#heterochromatinization-and-hp1"><em>Heterochromatinization and HP1</em></a></li>
<li><a href="1-introduction.html#non-coding-rna"><em>Non-Coding RNA</em></a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="1-introduction.html"><a href="1-introduction.html#the-heterochromatin-loss-model-of-aging"><i class="fa fa-check"></i><b>1.2</b> The Heterochromatin Loss Model of Aging</a></li>
<li class="chapter" data-level="1.3" data-path="1-introduction.html"><a href="1-introduction.html#the-aging-neuronal-epigenome"><i class="fa fa-check"></i><b>1.3</b> The Aging Neuronal Epigenome</a><ul>
<li class="chapter" data-level="1.3.1" data-path="1-introduction.html"><a href="1-introduction.html#DNAging"><i class="fa fa-check"></i><b>1.3.1</b> <em>DNA Methylation and Hydroxymethylation in Aged Neurons</em></a></li>
<li class="chapter" data-level="1.3.2" data-path="1-introduction.html"><a href="1-introduction.html#chraging"><i class="fa fa-check"></i><b>1.3.2</b> <em>Chromatin in the Aging Brain</em></a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="1-introduction.html"><a href="1-introduction.html#objective"><i class="fa fa-check"></i><b>1.4</b> Objective</a></li>
<li class="chapter" data-level="1.5" data-path="1-introduction.html"><a href="1-introduction.html#hp1-proteins---a-background"><i class="fa fa-check"></i><b>1.5</b> HP1 Proteins - A Background</a><ul>
<li><a href="1-introduction.html#the-abgs-of-hp1-proteins">The <em>ABG’s</em> of HP1 Proteins</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-results.html"><a href="2-results.html"><i class="fa fa-check"></i><b>2</b> Results</a><ul>
<li class="chapter" data-level="2.1" data-path="2-results.html"><a href="2-results.html#HP1aT"><i class="fa fa-check"></i><b>2.1</b> HP1FEC, HP1TKO and HP1<span class="math inline">\(\alpha\)</span>T</a></li>
<li class="chapter" data-level="2.2" data-path="2-results.html"><a href="2-results.html#expression-of-hp1-proteins-in-the-brain"><i class="fa fa-check"></i><b>2.2</b> Expression of HP1 Proteins in the Brain</a></li>
<li class="chapter" data-level="2.3" data-path="2-results.html"><a href="2-results.html#mosaic-deletion-of-hp1beta-and-hp1gamma-from-cortical-progenitors"><i class="fa fa-check"></i><b>2.3</b> Mosaic Deletion of HP1<span class="math inline">\(\beta\)</span> and HP1<span class="math inline">\(\gamma\)</span> from Cortical Progenitors</a></li>
<li class="chapter" data-level="2.4" data-path="2-results.html"><a href="2-results.html#deletion-of-hp1beta-hp1gamma-from-the-emx1-lineage."><i class="fa fa-check"></i><b>2.4</b> Deletion of <em>HP1<span class="math inline">\(\beta\)</span></em> &amp; <em>HP1<span class="math inline">\(\gamma\)</span></em> from the <em>Emx1</em>-lineage.</a><ul>
<li class="chapter" data-level="2.4.1" data-path="2-results.html"><a href="2-results.html#cell-fate-in-cortical-neurons"><i class="fa fa-check"></i><b>2.4.1</b> Cell Fate in Cortical Neurons</a></li>
<li class="chapter" data-level="2.4.2" data-path="2-results.html"><a href="2-results.html#imprint"><i class="fa fa-check"></i><b>2.4.2</b> Imprinted Loci</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="2-results.html"><a href="2-results.html#perturbed-regulation-of-retroelements-in-hp1-mutants"><i class="fa fa-check"></i><b>2.5</b> Perturbed Regulation of Retroelements in HP1 Mutants</a><ul>
<li class="chapter" data-level="2.5.1" data-path="2-results.html"><a href="2-results.html#line1"><i class="fa fa-check"></i><b>2.5.1</b> LINE1</a></li>
<li class="chapter" data-level="2.5.2" data-path="2-results.html"><a href="2-results.html#class-iii-ervs"><i class="fa fa-check"></i><b>2.5.2</b> Class III ERVs</a></li>
<li class="chapter" data-level="2.5.3" data-path="2-results.html"><a href="2-results.html#class-ii-ervs"><i class="fa fa-check"></i><b>2.5.3</b> Class II ERVs</a></li>
<li class="chapter" data-level="2.5.4" data-path="2-results.html"><a href="2-results.html#de-novo-retrotransposition"><i class="fa fa-check"></i><b>2.5.4</b> <em>De Novo</em> Retrotransposition</a></li>
</ul></li>
<li class="chapter" data-level="2.6" data-path="2-results.html"><a href="2-results.html#sustained-hp1betagamma-deficiency-adversely-affects-neuronal-complexity-function-and-cognition"><i class="fa fa-check"></i><b>2.6</b> Sustained HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> Deficiency Adversely Affects Neuronal Complexity, Function and Cognition</a><ul>
<li class="chapter" data-level="2.6.1" data-path="2-results.html"><a href="2-results.html#behavior"><i class="fa fa-check"></i><b>2.6.1</b> Behavior</a></li>
<li class="chapter" data-level="2.6.2" data-path="2-results.html"><a href="2-results.html#MRIresults"><i class="fa fa-check"></i><b>2.6.2</b> Structure, Volumetrics and Histology</a></li>
</ul></li>
<li class="chapter" data-level="2.7" data-path="2-results.html"><a href="2-results.html#hp1beta-and-hp1gamma-synergistically-contribute-to-repeat-silencing"><i class="fa fa-check"></i><b>2.7</b> HP1<span class="math inline">\(\beta\)</span> and HP1<span class="math inline">\(\gamma\)</span> Synergistically Contribute to Repeat Silencing</a><ul>
<li class="chapter" data-level="2.7.1" data-path="2-results.html"><a href="2-results.html#hp1gamma-is-required-for-h4k20me3-deposition-in-neurons"><i class="fa fa-check"></i><b>2.7.1</b> HP1<span class="math inline">\(\gamma\)</span> is Required for H4K20me3 Deposition in Neurons</a></li>
<li class="chapter" data-level="2.7.2" data-path="2-results.html"><a href="2-results.html#hp1gamma-binds-suv420h2-via-the-chromoshadow-domain"><i class="fa fa-check"></i><b>2.7.2</b> HP1<span class="math inline">\(\gamma\)</span> binds SUV420H2 via the Chromoshadow Domain</a></li>
<li class="chapter" data-level="2.7.3" data-path="2-results.html"><a href="2-results.html#re-addition-of-hp1gamma-can-restore-h4k20me3-in-vivo"><i class="fa fa-check"></i><b>2.7.3</b> Re-addition of HP1<span class="math inline">\(\gamma\)</span> can restore H4K20me3 In Vivo</a></li>
<li class="chapter" data-level="2.7.4" data-path="2-results.html"><a href="2-results.html#h4k20me3-loss-is-correlated-with-iap-de-repression"><i class="fa fa-check"></i><b>2.7.4</b> H4K20me3 Loss is Correlated with IAP De-repression</a></li>
<li class="chapter" data-level="2.7.5" data-path="2-results.html"><a href="2-results.html#kap1-h3k9me3-and-h4k20me3-are-lost-at-iap-elements-in-hp1ctko-es-cells"><i class="fa fa-check"></i><b>2.7.5</b> KAP1, H3K9me3 and H4K20me3 are lost at IAP Elements in HP1cTKO ES Cells</a></li>
<li class="chapter" data-level="2.7.6" data-path="2-results.html"><a href="2-results.html#dna-methylation-is-affected-following-loss-of-hp1"><i class="fa fa-check"></i><b>2.7.6</b> DNA Methylation is affected Following loss of HP1</a></li>
<li><a href="2-results.html#kap1-is-lost-from-icrs-in-hp1ctko-cells"><em>KAP1 is lost from ICRs in HP1cTKO Cells</em></a></li>
</ul></li>
<li class="chapter" data-level="2.8" data-path="2-results.html"><a href="2-results.html#RNAresults"><i class="fa fa-check"></i><b>2.8</b> Transcriptional Dysregulation following HP1 Deficiency Drives UPR and Inflammatory Responses</a><ul>
<li class="chapter" data-level="2.8.1" data-path="2-results.html"><a href="2-results.html#genotype-related-changes"><i class="fa fa-check"></i><b>2.8.1</b> Genotype Related Changes</a></li>
<li class="chapter" data-level="2.8.2" data-path="2-results.html"><a href="2-results.html#longitudinal-changes-to-gene-expression"><i class="fa fa-check"></i><b>2.8.2</b> Longitudinal Changes To Gene Expression</a></li>
<li class="chapter" data-level="2.8.3" data-path="2-results.html"><a href="2-results.html#pathway-analysis"><i class="fa fa-check"></i><b>2.8.3</b> Pathway Analysis</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-discussion.html"><a href="3-discussion.html"><i class="fa fa-check"></i><b>3</b> Discussion</a><ul>
<li class="chapter" data-level="3.1" data-path="3-discussion.html"><a href="3-discussion.html#phenotypes"><i class="fa fa-check"></i><b>3.1</b> Phenotypes</a><ul>
<li><a href="3-discussion.html#structural-changes-in-the-hippocampus"><em>Structural Changes in the Hippocampus</em></a></li>
<li><a href="3-discussion.html#reduction-in-cortical-volume"><em>Reduction in Cortical Volume</em></a></li>
<li><a href="3-discussion.html#behavior-and-impairments-in-learning-memory"><em>Behavior and Impairments in Learning &amp; Memory</em></a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="3-discussion.html"><a href="3-discussion.html#hp1-dependent-epigenetic-systems"><i class="fa fa-check"></i><b>3.2</b> HP1-dependent Epigenetic Systems</a><ul>
<li><a href="3-discussion.html#hp1gamma"><em>HP1<span class="math inline">\(\gamma\)</span></em></a></li>
<li><a href="3-discussion.html#hp1beta"><em>HP1<span class="math inline">\(\beta\)</span></em></a></li>
<li><a href="3-discussion.html#hp1beta-x-hp1gamma"><em>HP1<span class="math inline">\(\beta\)</span></em> x <em>HP1<span class="math inline">\(\gamma\)</span></em></a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="3-discussion.html"><a href="3-discussion.html#concordance-with-aging-and-disease"><i class="fa fa-check"></i><b>3.3</b> Concordance with Aging and Disease</a><ul>
<li><a href="3-discussion.html#complement-in-cns-aging-and-disease"><em>Complement in CNS Aging and Disease</em></a></li>
<li><a href="3-discussion.html#a-pathway-to-complement"><em>A Pathway to Complement</em></a></li>
<li><a href="3-discussion.html#unfolded-protein-response-upr"><em>Unfolded Protein Response (UPR)</em></a></li>
<li><a href="3-discussion.html#viruses-transactivation-and-neurodegeneration"><em>Viruses, Transactivation and Neurodegeneration</em></a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="3-discussion.html"><a href="3-discussion.html#concluding-remarks"><i class="fa fa-check"></i><b>3.4</b> Concluding Remarks</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-methods.html"><a href="4-methods.html"><i class="fa fa-check"></i><b>4</b> Methods</a><ul>
<li class="chapter" data-level="4.1" data-path="4-methods.html"><a href="4-methods.html#mouse-lines"><i class="fa fa-check"></i><b>4.1</b> Mouse Lines</a><ul>
<li class="chapter" data-level="4.1.1" data-path="4-methods.html"><a href="4-methods.html#HP1anull"><i class="fa fa-check"></i><b>4.1.1</b> Generation of HP1<span class="math inline">\(\alpha\)</span>KO and HP1<span class="math inline">\(\alpha\)</span>T</a></li>
<li class="chapter" data-level="4.1.2" data-path="4-methods.html"><a href="4-methods.html#HP1Bgtarget"><i class="fa fa-check"></i><b>4.1.2</b> Generation of Floxed HP1<span class="math inline">\(\beta\)</span> &amp; Floxed HP1<span class="math inline">\(\gamma\)</span></a></li>
<li class="chapter" data-level="4.1.3" data-path="4-methods.html"><a href="4-methods.html#genotyping"><i class="fa fa-check"></i><b>4.1.3</b> Genotyping</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="4-methods.html"><a href="4-methods.html#cloning"><i class="fa fa-check"></i><b>4.2</b> Cloning</a><ul>
<li class="chapter" data-level="4.2.1" data-path="4-methods.html"><a href="4-methods.html#rna-isolation-cdna-generation-rnaseq"><i class="fa fa-check"></i><b>4.2.1</b> RNA Isolation, cDNA Generation, &amp; RNAseq</a></li>
<li class="chapter" data-level="4.2.2" data-path="4-methods.html"><a href="4-methods.html#cloning-of-rna-probes-for-in-situ-hybridization"><i class="fa fa-check"></i><b>4.2.2</b> Cloning of RNA Probes for <em>In-Situ Hybridization</em></a></li>
<li class="chapter" data-level="4.2.3" data-path="4-methods.html"><a href="4-methods.html#cloning-of-expression-constructs"><i class="fa fa-check"></i><b>4.2.3</b> Cloning of Expression Constructs</a></li>
<li class="chapter" data-level="4.2.4" data-path="4-methods.html"><a href="4-methods.html#site-directed-mutagenesis"><i class="fa fa-check"></i><b>4.2.4</b> Site Directed Mutagenesis</a></li>
<li class="chapter" data-level="4.2.5" data-path="4-methods.html"><a href="4-methods.html#transformation"><i class="fa fa-check"></i><b>4.2.5</b> Transformation</a></li>
<li class="chapter" data-level="4.2.6" data-path="4-methods.html"><a href="4-methods.html#clonal-selection-plasmid-production"><i class="fa fa-check"></i><b>4.2.6</b> Clonal Selection &amp; Plasmid Production</a></li>
<li class="chapter" data-level="4.2.7" data-path="4-methods.html"><a href="4-methods.html#in-vitro-transcription-ivt"><i class="fa fa-check"></i><b>4.2.7</b> In Vitro Transcription (IVT)</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="4-methods.html"><a href="4-methods.html#cell-culture"><i class="fa fa-check"></i><b>4.3</b> Cell Culture</a><ul>
<li class="chapter" data-level="4.3.1" data-path="4-methods.html"><a href="4-methods.html#primary-neuronal-culture"><i class="fa fa-check"></i><b>4.3.1</b> Primary Neuronal Culture</a></li>
<li class="chapter" data-level="4.3.2" data-path="4-methods.html"><a href="4-methods.html#hek293t-culture-transfection"><i class="fa fa-check"></i><b>4.3.2</b> HEK293T Culture &amp; Transfection</a></li>
<li class="chapter" data-level="4.3.3" data-path="4-methods.html"><a href="4-methods.html#co-immunoprecipitation-co-ip-and-western-blot-wb"><i class="fa fa-check"></i><b>4.3.3</b> Co-Immunoprecipitation (Co-IP) and Western Blot (WB)</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="4-methods.html"><a href="4-methods.html#in-utero-electroporation-tissue-processing-and-histology"><i class="fa fa-check"></i><b>4.4</b> In Utero Electroporation, Tissue Processing and Histology</a><ul>
<li class="chapter" data-level="4.4.1" data-path="4-methods.html"><a href="4-methods.html#IUEmethod"><i class="fa fa-check"></i><b>4.4.1</b> In Utero Electroporation (IUE)</a></li>
<li class="chapter" data-level="4.4.2" data-path="4-methods.html"><a href="4-methods.html#tissue-fixation-and-perfusion"><i class="fa fa-check"></i><b>4.4.2</b> Tissue Fixation and Perfusion</a></li>
<li class="chapter" data-level="4.4.3" data-path="4-methods.html"><a href="4-methods.html#cryosectioning"><i class="fa fa-check"></i><b>4.4.3</b> Cryosectioning</a></li>
<li class="chapter" data-level="4.4.4" data-path="4-methods.html"><a href="4-methods.html#immuno-histochemistry-ihc"><i class="fa fa-check"></i><b>4.4.4</b> Immuno Histochemistry (IHC)</a></li>
<li class="chapter" data-level="4.4.5" data-path="4-methods.html"><a href="4-methods.html#quantification-of-cells-in-the-dentate-gyrus"><i class="fa fa-check"></i><b>4.4.5</b> Quantification of Cells in the Dentate Gyrus</a></li>
<li class="chapter" data-level="4.4.6" data-path="4-methods.html"><a href="4-methods.html#in-situ-hybridization-ish"><i class="fa fa-check"></i><b>4.4.6</b> In Situ Hybridization (ISH)</a></li>
<li class="chapter" data-level="4.4.7" data-path="4-methods.html"><a href="4-methods.html#paraffin-sectioning"><i class="fa fa-check"></i><b>4.4.7</b> Paraffin Sectioning</a></li>
<li class="chapter" data-level="4.4.8" data-path="4-methods.html"><a href="4-methods.html#nissl"><i class="fa fa-check"></i><b>4.4.8</b> Nissl Stain</a></li>
<li class="chapter" data-level="4.4.9" data-path="4-methods.html"><a href="4-methods.html#golgi-impregnation-and-dendritic-complexity-analysis"><i class="fa fa-check"></i><b>4.4.9</b> Golgi Impregnation and Dendritic Complexity Analysis</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="4-methods.html"><a href="4-methods.html#behavioral-experiments"><i class="fa fa-check"></i><b>4.5</b> Behavioral Experiments</a><ul>
<li class="chapter" data-level="4.5.1" data-path="4-methods.html"><a href="4-methods.html#open-field-activity-1"><i class="fa fa-check"></i><b>4.5.1</b> Open Field Activity</a></li>
<li class="chapter" data-level="4.5.2" data-path="4-methods.html"><a href="4-methods.html#paired-pulse-inhibition-1"><i class="fa fa-check"></i><b>4.5.2</b> Paired-Pulse Inhibition</a></li>
<li class="chapter" data-level="4.5.3" data-path="4-methods.html"><a href="4-methods.html#context-cued-fear-conditioning"><i class="fa fa-check"></i><b>4.5.3</b> Context-Cued Fear Conditioning</a></li>
<li class="chapter" data-level="4.5.4" data-path="4-methods.html"><a href="4-methods.html#Barnesmeth"><i class="fa fa-check"></i><b>4.5.4</b> Barnes Maze</a></li>
<li class="chapter" data-level="4.5.5" data-path="4-methods.html"><a href="4-methods.html#social-activity-monitor-sam-1"><i class="fa fa-check"></i><b>4.5.5</b> Social Activity Monitor (SAM)</a></li>
<li class="chapter" data-level="4.5.6" data-path="4-methods.html"><a href="4-methods.html#micro-behaviors-homecagescan"><i class="fa fa-check"></i><b>4.5.6</b> Micro Behaviors (HomeCageScan)</a></li>
<li class="chapter" data-level="4.5.7" data-path="4-methods.html"><a href="4-methods.html#nestcomethod"><i class="fa fa-check"></i><b>4.5.7</b> Nest Construction</a></li>
</ul></li>
<li class="chapter" data-level="4.6" data-path="4-methods.html"><a href="4-methods.html#magnetic-resonance-imaging-mri-and-volumetrics"><i class="fa fa-check"></i><b>4.6</b> Magnetic Resonance Imaging (MRI) and Volumetrics</a></li>
<li class="chapter" data-level="4.7" data-path="4-methods.html"><a href="4-methods.html#reduced-read-bisulfite-sequencing-rrbseq"><i class="fa fa-check"></i><b>4.7</b> Reduced Read Bisulfite Sequencing (RRBSeq)</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html"><i class="fa fa-check"></i><b>5</b> Supplemental Data</a><ul>
<li class="chapter" data-level="5.1" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html#Behavsupplement"><i class="fa fa-check"></i><b>5.1</b> Behavior</a><ul>
<li class="chapter" data-level="5.1.1" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html#homecagescan-micro-behaviors"><i class="fa fa-check"></i><b>5.1.1</b> HomeCageScan Micro Behaviors</a></li>
<li class="chapter" data-level="5.1.2" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html#context-cued-fear-conditioning-1"><i class="fa fa-check"></i><b>5.1.2</b> Context-Cued Fear Conditioning</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html#DE"><i class="fa fa-check"></i><b>5.2</b> Differential Gene Expression</a><ul>
<li class="chapter" data-level="5.2.1" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html#differentially-expressed-genes-in-aged-hp1betako"><i class="fa fa-check"></i><b>5.2.1</b> Differentially Expressed Genes in Aged HP1<span class="math inline">\(\beta\)</span>KO</a></li>
<li class="chapter" data-level="5.2.2" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html#inflam"><i class="fa fa-check"></i><b>5.2.2</b> Genes Related to Anti-viral and Interferon Response in HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO</a></li>
<li class="chapter" data-level="5.2.3" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html#genomic-hotspots-sensitive-to-hp1-deficiency-in-the-hippocampus"><i class="fa fa-check"></i><b>5.2.3</b> <em>Genomic Hotspots Sensitive to HP1 Deficiency in the Hippocampus</em></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html"><i class="fa fa-check"></i><b>6</b> Bioinformatics &amp; Data Processing</a><ul>
<li class="chapter" data-level="6.1" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#software-environment-management"><i class="fa fa-check"></i><b>6.1</b> Software Environment Management</a></li>
<li class="chapter" data-level="6.2" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#snakemake-jobs"><i class="fa fa-check"></i><b>6.2</b> Snakemake Jobs</a></li>
<li class="chapter" data-level="6.3" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#tetranscripts-pipeline"><i class="fa fa-check"></i><b>6.3</b> TETranscripts Pipeline</a></li>
<li class="chapter" data-level="6.4" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#differential-gene-expression-using-edger"><i class="fa fa-check"></i><b>6.4</b> Differential Gene Expression Using EdgeR</a></li>
<li class="chapter" data-level="6.5" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#deeptools-pipeline"><i class="fa fa-check"></i><b>6.5</b> Deeptools Pipeline</a></li>
<li class="chapter" data-level="6.6" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#lions-pipeline"><i class="fa fa-check"></i><b>6.6</b> LIONS Pipeline</a></li>
<li class="chapter" data-level="6.7" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#analysis-of-reduced-read-bisulphite-sequencing-rrbs"><i class="fa fa-check"></i><b>6.7</b> Analysis of Reduced-Read Bisulphite Sequencing (RRBS)</a><ul>
<li class="chapter" data-level="6.7.1" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#methylkit-and-circos-plot"><i class="fa fa-check"></i><b>6.7.1</b> Methylkit and Circos Plot</a></li>
<li class="chapter" data-level="6.7.2" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#annotated-ideogram"><i class="fa fa-check"></i><b>6.7.2</b> Annotated Ideogram</a></li>
</ul></li>
<li class="chapter" data-level="6.8" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#about-this-thesis"><i class="fa fa-check"></i><b>6.8</b> About this Thesis</a><ul>
<li class="chapter" data-level="6.8.1" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#session-info"><i class="fa fa-check"></i><b>6.8.1</b> Session Info</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7" data-path="7-references.html"><a href="7-references.html"><i class="fa fa-check"></i><b>7</b> References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">HP1 Deficiency Results in Accelerated Neurodegeneration</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="discussion" class="section level1">
<h1><span class="header-section-number">Part 3</span> Discussion</h1>
<p>Given that epigenetic drift occurs over the course of normal aging, the effect of deletion of two non-histone proteins HP1<span class="math inline">\(\beta\)</span> and HP1<span class="math inline">\(\gamma\)</span> was examined to determine if heterochromatin loss is causal in the intrinsic aging of the mammalian brain. <em>Emx1</em>-mediated deletion of HPI<span class="math inline">\(\beta\)</span> impairs neurogenesis resulting in malformation of the infrapyramidal blade of the dentate gyrus and a subsequent progressive reduction in the complexity of CA3 basal dendrites. This malformation of the dentate gyrus and reduction in CA3 basal dendrites is stronger in the HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO.</p>
<p>At the molecular level, distinct and synergistic contributions of HP1<span class="math inline">\(\beta\)</span> and HP1<span class="math inline">\(\gamma\)</span> in mediating genome stability in neurons can be observed. In addition to detrimental effects on mitosis, deletion of HP1<span class="math inline">\(\beta\)</span> appears to affect tissue-specific imprinted loci in an age dependent manner. Deletion of HP1<span class="math inline">\(\gamma\)</span> in neurons results in complete loss of the histone post-translational modification H4K20me3 and up-regulation of the protocadherin cluster of genes on chromosome 18. Synergistic effects of HP1<span class="math inline">\(\beta\)</span> and HP1<span class="math inline">\(\gamma\)</span> include further activation of imprinted loci and de-repression of endogenous retroviruses and ncRNAs. Subsequently, this transcriptional de-regulation activates pathways associated with innate immunity and the unfolded protein response.</p>
<p>Collectively, the effects described above contribute to cognitive effects that mirror aspects of human age-related cognitive decline. HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span>DKO mice display profound impairments in spatial memory which become progressively worse with age, in addition to age-dependent deficiencies in paired-pulse inhibition. HP1DKO mice also display cognitive deficits in nest construction, and show distinct abnormalities in circadian activity. The molecular mechanism(s) behind the multiple above phenotypes in addition to how they are connected is explored below.</p>
<div id="phenotypes" class="section level2">
<h2><span class="header-section-number">3.1</span> Phenotypes</h2>
<div id="structural-changes-in-the-hippocampus" class="section level3 unnumbered">
<h3><em>Structural Changes in the Hippocampus</em></h3>
<p>Shortly after discovering strong de-repression of ERVs in HP1DKOs, we observed seizures in the colony. This provided the first pointer that functional changes were occuring in the hippocampus. How partial loss of the infrapyramidal blade (IFB) of the dentate gyrus (DG) in HP1<span class="math inline">\(\beta\)</span>KO animals may have resulted in altered circuitry that favours seizure activity is unclear. It is possible that in the absence of the IFB, all inputs from the entorhinal cortex synapse on the suprapyramidal blade, which may render it hyper-excitable. However, it was unusual that the seizures occured more often in HP1<span class="math inline">\(\beta\)</span>KO compared to DKO, which has a smaller DG and may have even higher convergence of entorhinal input.</p>
<p>The degree of malformation of the IFB in HP1<span class="math inline">\(\beta\)</span>KO and HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO correlates strongly with degree of age-dependent loss in CA3 basal dendrite complexity. Given proximal CA3 basal dendrites receive a large amount of input from the infrapyramidal blade, it is likely that CA3 basal dendrites remodel or degenerate based on this absence of input from the DG.</p>
<p>The malformation of the dentate gyrus appears to be due to mitotic exhaustion as the number of Ki67+ cells in the developing DG is already reduced at P0 and progressively decreased at P8 (Figure <a href="2-results.html#fig:prox1ki67">2.25</a>). This corresponds with a reduction in the number of Prox1+ cells at P8, an effect that appears to be dependent on HP1<span class="math inline">\(\beta\)</span> and consistent with previous reports of mitotic deficits following its deletion <span class="citation">(Aucott et al. <a href="#ref-AucottHP1requireddevelopment2008">2008</a>)</span>.</p>
</div>
<div id="reduction-in-cortical-volume" class="section level3 unnumbered">
<h3><em>Reduction in Cortical Volume</em></h3>
<p>While the total brain volume is noticibly reduced in young HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO animals, this difference fails to reach statistical significance in aged animals (Figure <a href="2-results.html#fig:MRIresults">2.26</a>). Similarly, the raw volume of the young HP1DKO cortex is reduced, however if normalized to total brain volume, cortical volume is only significantly reduced in specific caudal regions (slices 3-5, 10-11 Figure <a href="2-results.html#fig:MRIresults">2.26</a>). This selective reduction in cortical volume appears to be a developmental phenotype, as the aged HP1DKO cortex, while still smaller than WT, does not show region-specific reductions in cortical volume. Why such a rostral-caudal difference in cortical volume exists would require further analysis, but it may be related to the onset of <em>Emx1</em> expression occuring slightly earlier in the caudal cortex <span class="citation">(Theil et al. <a href="#ref-TheilGli3requiredEmx1999">1999</a>)</span>, resulting in an earlier onset of HP1<span class="math inline">\(\beta\)</span>-related mitotic deficiencies in caudal neural progenitors.</p>
</div>
<div id="behavior-and-impairments-in-learning-memory" class="section level3 unnumbered">
<h3><em>Behavior and Impairments in Learning &amp; Memory</em></h3>
<p>HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO mutants show several age-dependent behavioral abnormalities.</p>
<p>The simplest test, where animal exploratory activity and anxiety is measured in an open field, reveals that aged HP1DKO mutants show extended initial freezing in the open field. For the first half of the open field test, aged HP1DKO animals show an absence of thigmotaxis –the tendency to remain close to the walls, and remain partially active in the centre zone for the first half of the test before further exploration (Figure <a href="2-results.html#fig:openfield">2.17</a>). The prolonged activity of HP1DKO mutants in the centre of the open field occurs at the beginning of the test similar to in young HP1<span class="math inline">\(\beta\)</span>KO animals.</p>
<p>Conversely, analysis of circadian activity reveals that HP1 double and single mutants are hyperactive compared to their wild-type brothers (Figure <a href="2-results.html#fig:SAM">2.18</a>). Such increased activity is accompanied by increased bouts of grooming, and in aged HP1DKO animals increased bouts of eating (Figure <a href="2-results.html#fig:GroomEatWeight">2.19</a>), which may be indicative of increased anxiety and stress. HP1DKO animals also show pronounced deficits in the species-specific behavior of nest construction (Figure <a href="2-results.html#fig:Nestconstruction">2.21</a>), which is thought to measure both animal wellbeing and cognitive function <span class="citation">(Lin et al. <a href="#ref-LinNeuralencodingconcept2007">2007</a>)</span>.</p>
<p>As to be expected, the most prominent age-dependent behavioral phenotypes are those closely aligned with proper hippocampal function; namely Paired Pulse Inhibition (PPI) and spatial memory as measured in the Barnes Maze. Deficits in PPI appear to be unique to aged HP1DKO animals (Figure <a href="2-results.html#fig:PPI">2.20</a>). Similar deficits in PPI in humans with dementia of Alzheimer’s type has been attributed to damage to the enthorhinal cortex and the loss of parahippocampal volume <span class="citation">(Ueki et al. <a href="#ref-UekiPrepulseinhibitionacoustic2006">2006</a>; Kesslak, Nalcioglu, and Cotman <a href="#ref-KesslakQuantificationmagneticresonance1991">1991</a>)</span>. As the entorhinal cortex is the source of the perforant path which projects to the DG, and the dentate gyrus projects to CA3 neurons, it is natural to infer deficits in PPI in HP1DKO aged animals is directly related to the age-dependent atrophy of CA3 basal dendrites (Figure <a href="2-results.html#fig:golgiquant">2.23</a>F).</p>
<p>The relationship between HP1 function and CA3 basal dendrite complexity is not a simple one. First, the complexity of CA3 basal dendrites in HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO animals is half that of younger animals, yet the CA3 complexity in young HP1<span class="math inline">\(\beta\)</span>KO mice is reduced but does not change with age. Second, the spatial memory deficits seen in HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO animals may be explained by DG malformation and reduced CA3 basal dendrite complexity but must be set against the observation that HP1<span class="math inline">\(\beta\)</span>KO exhibit similar structural malformations without any deficits in spatial memory (Figure <a href="2-results.html#fig:barnes">2.22</a>). Lastly, HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO CA3 basal dendrite complexity shows a reduction with age, whereas the complexity of basal dendrites in HP1<span class="math inline">\(\beta\)</span>KO neurons remains the same.</p>
<p>It is thus more likely that the gross transcriptional dysregulation affects functional hippocampal output, either due to the observed altered transcription of neuromodulatory (such as dopamine &amp; serotonin) receptors (Figure <a href="2-results.html#fig:reactome">2.39</a> &amp; Supplementary Table <a href="5-supplemental-data.html#tab:DEtable">5.2</a>) or potential synapse remodelling due to elevated levels of complement pathway components <span class="citation">(Liddelow et al. <a href="#ref-LiddelowNeurotoxicreactiveastrocytes2017">2017</a>; Stevens et al. <a href="#ref-StevensClassicalComplementCascade2007">2007</a>)</span>.</p>
</div>
</div>
<div id="hp1-dependent-epigenetic-systems" class="section level2">
<h2><span class="header-section-number">3.2</span> HP1-dependent Epigenetic Systems</h2>
<p>The phenotypes observed due to HP1 deficiency do overlap partially with the phenotypes induced by deficiencies of related epigenetic pathways.</p>
<div id="hp1gamma" class="section level3 unnumbered">
<h3><em>HP1<span class="math inline">\(\gamma\)</span></em></h3>
<p>A surprising finding was that the absence of HP1<span class="math inline">\(\gamma\)</span> alone could abolish all Histone 4 Lysine 20 trimethylation (H4K20me3) in <em>Emx1<sup>Cre</sup></em> expressing cells. Even more surprising is that in both structural quantifications and behavioral tests, <em>HP1<span class="math inline">\(\gamma\)</span> Emx1<sup>Cre</sup></em> mice are largely indistinguishable from wild-type. While one would expect H4K20me3 loss to be directly linked to ERV activation, HP1<span class="math inline">\(\gamma\)</span>KO hippocampi only show elevated levels of MuRRS-int from the ERV1 family (section <a href="2-results.html#RNAresults">2.8</a>). The only notable change detected in <em>HP1<span class="math inline">\(\gamma\)</span> Emx1<sup>Cre</sup></em> hippocampi is dysregulation of the protocadherin cluster of genes on chromosome 18. The protocadherin cluster contains three subclusters of protocadherins; <span class="math inline">\(\alpha\)</span>, <span class="math inline">\(\beta\)</span> and <span class="math inline">\(\gamma\)</span>, which are arranged in tandem. <span class="math inline">\(\alpha\)</span>- and <span class="math inline">\(\gamma\)</span>- protocadherins are formed by variable first exons and a static C terminus, while <span class="math inline">\(\beta\)</span>-protocadherins are single exon genes. Their combinatorial assembly enables an immense range of unique surface molecules, enabling unique ‘barcoding’ of neurons, which is thought to be important for dendrite branching, synaptic connection, and neurite self-avoidance <span class="citation">(Molumby et al. <a href="#ref-MolumbygProtocadherinsInteractNeuroligin12017">2017</a>; Rubinstein et al. <a href="#ref-RubinsteinMolecularLogicNeuronal2015">2015</a>; Toyoda et al. <a href="#ref-ToyodaDevelopmentalEpigeneticModification2014">2014</a>)</span>. Despite the previously ascribed importance of the <span class="math inline">\(\alpha\)</span>, <span class="math inline">\(\beta\)</span> and <span class="math inline">\(\gamma\)</span> protocadherins in neuronal function, no pronounced differences could be observed in the morphological phenotyping or behavioral assays undertaken here.</p>
<div class="figure" style="text-align: center"><span id="fig:cadclust"></span>
<img src="figure/discussion/Pcdh_clust3.png" alt="HP1$\gamma$ is Required for Transcriptional Regulation of the Protocadherin Cluster. (Top): Reference Gene annotation of the mouse protocadherin $\alpha$, $\beta$, and $\gamma$ clusters on chromosome 18 along with previously published ChIPseq of H3K4me3, H3K9me3, and H4K20me3 performed in cortical neurons. (Bottom): mRNA and LTR reference annotation along with RNA tracks from young and aged HP1 mutants." width="100%"   />
<p class="caption">
Figure 3.1: HP1<span class="math inline">\(\gamma\)</span> is Required for Transcriptional Regulation of the Protocadherin Cluster. (Top): Reference Gene annotation of the mouse protocadherin <span class="math inline">\(\alpha\)</span>, <span class="math inline">\(\beta\)</span>, and <span class="math inline">\(\gamma\)</span> clusters on chromosome 18 along with previously published ChIPseq of H3K4me3, H3K9me3, and H4K20me3 performed in cortical neurons. (Bottom): mRNA and LTR reference annotation along with RNA tracks from young and aged HP1 mutants.
</p>
</div>
<p>However, the HP1<span class="math inline">\(\gamma\)</span>KO does provide a new insight into regulation of the protocadherin cluster. It has been previously reported that regulation of the protocadherin cluster is dependent on Wiz-mediated recruitment of G9a/GLP <span class="citation">(Isbel et al. <a href="#ref-IsbelWizbindsactive2016">2016</a>)</span> and H3K9 trimethylation by SETDB1 <span class="citation">(Jiang et al. <a href="#ref-JiangmethyltransferaseSETDB1regulates2017">2017</a>)</span>. In humans, excessive H3K9me3 at the protocadherin cluster is associated with cognitive deficits in Kleefstra syndrome <span class="citation">(Iacono et al. <a href="#ref-IaconoIncreasedH3K9methylation2018">2018</a>)</span>. Additionally, altered DNA methylation at the protocadherin cluster is associated with several developmental and psychiatric diseases <span class="citation">(El Hajj, Dittrich, and Haaf <a href="#ref-ElHajjEpigeneticdysregulationprotocadherins2017d">2017</a>)</span>. Age-dependent differences in DNA methylation have been observed at all <span class="math inline">\(\alpha\)</span>, <span class="math inline">\(\beta\)</span> and <span class="math inline">\(\gamma\)</span> subclusters <span class="citation">(Bell et al. <a href="#ref-BellEpigenomeWideScansIdentify2012a">2012</a>; El Hajj, Dittrich, and Haaf <a href="#ref-ElHajjEpigeneticdysregulationprotocadherins2017d">2017</a>; Salpea et al. <a href="#ref-SalpeaPostnataldevelopmentagerelated2012">2012</a>)</span>, however again these analyses could not distinguish between 5mC and 5hmC.</p>
<div class="figure" style="text-align: center"><span id="fig:EScadclust"></span>
<img src="figure/discussion/ES_Pcdh_H3K9_H4K20.png" alt="H3K9me3 is unnaffected at the Protocadherin Cluster in HP1cTKO ES cells. While HP1cTKO ES cells show a loss of H3K9me3 at IAP elements proximal to the protocadherin cluster, H3K9me3 regulatory enrichment at the cluster is otherwise unaffected." width="100%"   />
<p class="caption">
Figure 3.2: H3K9me3 is unnaffected at the Protocadherin Cluster in HP1cTKO ES cells. While HP1cTKO ES cells show a loss of H3K9me3 at IAP elements proximal to the protocadherin cluster, H3K9me3 regulatory enrichment at the cluster is otherwise unaffected.
</p>
</div>
<p>H4K20me3 loss may cause the elevated transcription at the protocadherin cluster. H3K9me3 and H4K20me3 normally show parallel enrichment over the protocadherin cluster in cortical neurons (Figure <a href="3-discussion.html#fig:cadclust">3.1</a>). As H3K9me3 is considered a pre-requisite for H4K20me3 deposition <span class="citation">(Kourmouli et al. <a href="#ref-KourmouliHeterochromatintrimethylatedlysine2004">2004</a>; Schotta <a href="#ref-Schottasilencingpathwayinduce2004">2004</a>)</span>, it stands to reason that the function of SETDB1/H3K9me3-mediated transcriptional regulation of the protocadherin cluster <span class="citation">(Jiang et al. <a href="#ref-JiangmethyltransferaseSETDB1regulates2017">2017</a>)</span> functions by H4K20me3 deposition. This raises two possibilities; 1) that HP1<span class="math inline">\(\gamma\)</span> is required for SETDB1 localization, or that 2) elevated protocadherin transcription in HP1<span class="math inline">\(\gamma\)</span>KO is due to the absence of H4K20me3.</p>
<p>It is known that H4K20me3 could play a direct role in gene regulation. It has recently been reported that H4K20me3 is found to coincide with activating marks in ES cells <span class="citation">(Xu and Kidder <a href="#ref-XuH4K20me3colocalizesactivating2018">2018</a>)</span>, which may be in line with the known propensity of HP1<span class="math inline">\(\gamma\)</span> to associate with actively transcribed chromatin and its requirement for ES cell self-renewal <span class="citation">(Caillier et al. <a href="#ref-CaillierRoleEpigeneticRegulator2010">2010</a>)</span>.</p>
<p>In HP1cTKO embryonic stem cells, the binding profile of H3K9me3 and H4K20me3 at the protocadherin cluster is slightly different from that in neurons. In neurons, where protocadherins are highly expressed, H4K20me3 strongly mirrors H3K9me3 signal (Figure <a href="3-discussion.html#fig:cadclust">3.1</a>). In ES cells, where protocadherins have very low expression, H4K20me3 primarily marks IAP elements within the protocadherin cluster (Figure <a href="3-discussion.html#fig:EScadclust">3.2</a>). Importantly, H4K20me3 is lost in HP1cTKO ES cells while H3K9me3 appears unchanged (Figure <a href="3-discussion.html#fig:EScadclust">3.2</a>). This indicates that contrary to the previous report that the protocadherin cluster is regulated by SETDB1 <span class="citation">(Jiang et al. <a href="#ref-JiangmethyltransferaseSETDB1regulates2017">2017</a>)</span>, it is likely the loss of H4K20me3 (in both SETDB1 and HP1<span class="math inline">\(\gamma\)</span> mutants) that results in elevated transcription at the protocadherin cluster in neurons.</p>
</div>
<div id="hp1beta" class="section level3 unnumbered">
<h3><em>HP1<span class="math inline">\(\beta\)</span></em></h3>
<p>The stronger phenotypes observed are rooted in molecular mechanisms mediated by HP1<span class="math inline">\(\beta\)</span>, which is consistent with its requisite roles in mitotic stability <span class="citation">(Aucott et al. <a href="#ref-AucottHP1requireddevelopment2008">2008</a>)</span>, splicing <span class="citation">(Yearim et al. <a href="#ref-YearimHP1InvolvedRegulating2015">2015</a>)</span>, DNA repair <span class="citation">(Ayoub, Jeyasekharan, and Venkitaraman <a href="#ref-AyoubMobilizationrecruitmentHP1v2009">2009</a>; Trembecka-Lucas et al. <a href="#ref-Trembecka-LucasDynamicsHP1vPCNAcontainingcomplexes2013">2013</a>)</span>, and lethality <span class="citation">(Billur, Bartunik, and Singh <a href="#ref-BilluressentialfunctionHP1v2010">2010</a>)</span>. Notably, unbiased exome data predicts HP1<span class="math inline">\(\beta\)</span> to have the highest probability of loss of function intolerance (pLI) out of the HP1 proteins<a href="#fn1" class="footnoteRef" id="fnref1"><sup>1</sup></a>. In <em>HP1<span class="math inline">\(\beta\)</span><sup>fl/fl</sup>Emx1<sup>Cre</sup></em> animals, loss of HP1<span class="math inline">\(\beta\)</span> results in mitotic deficits of the dentate gyrus, described above, as well as numerous age-related changes that speak to its essential role in the stability of mammalian epigenetic systems that will be discussed below.</p>
<p>Knockout studies of related epigenetic effectors show similar phenotypes to those seen in <em>HP1<span class="math inline">\(\beta\)</span><sup>fl/fl</sup>Emx1<sup>Cre</sup></em> animals. Previous work examining <em>Dnmt1</em> loss in the brain, where <em>Dnmt1</em> is deleted by tamoxifen induced <em>Nestin</em>-Cre, report a reduction in the formation of the dentate gyrus <span class="citation">(Noguchi et al. <a href="#ref-NoguchiDNAMethyltransferaseIndispensable2016">2016</a>)</span> similar to that seen here in HP1<span class="math inline">\(\beta\)</span>KOs. In the <em>Dnmt1<sup>fl/fl</sup>Emx1<sup>Cre</sup></em> mutant, robust activation of IAP elements phenocopies what occurs in the <em>HP1<span class="math inline">\(\beta\)</span><sup>fl/fl</sup>HP1<span class="math inline">\(\gamma\)</span><sup>fl/fl</sup>Emx1<sup>Cre</sup></em> mutant, however, the <em>Dnmt1<sup>fl/fl</sup>Emx1<sup>Cre</sup></em> mutant also displays a dramatic, progressive degeneration of the cortex <span class="citation">(Hutnick et al. <a href="#ref-HutnickDNAhypomethylationrestricted2009">2009</a>)</span> which is not seen in the <em>HP1<span class="math inline">\(\beta\)</span><sup>fl/fl</sup>Emx1<sup>Cre</sup></em> or <em>HP1<span class="math inline">\(\beta\)</span><sup>fl/fl</sup>HP1<span class="math inline">\(\gamma\)</span><sup>fl/fl</sup>Emx1<sup>Cre</sup></em> brain. While there appears to be a tendency towards HP1<span class="math inline">\(\beta\)</span> being important in Dnmt1 function, given an age-dependent elevation of both repetitive elements, inflammatory <em>C4</em> and <em>C1qa</em> and <em>Nnat</em> transcription in <em>HP1<span class="math inline">\(\beta\)</span><sup>fl/fl</sup>Emx1<sup>Cre</sup></em> brains, dysregulation of non-cannonical imprinted loci is most pronounced in <em>HP1<span class="math inline">\(\beta\)</span><sup>fl/fl</sup>HP1<span class="math inline">\(\gamma\)</span><sup>fl/fl</sup>Emx1<sup>Cre</sup></em> mutants.</p>
</div>
<div id="hp1beta-x-hp1gamma" class="section level3 unnumbered">
<h3><em>HP1<span class="math inline">\(\beta\)</span></em> x <em>HP1<span class="math inline">\(\gamma\)</span></em></h3>
<p>Some of the strongest changes in <em>HP1<span class="math inline">\(\beta\)</span><sup>fl/fl</sup>HP1<span class="math inline">\(\gamma\)</span><sup>fl/fl</sup>Emx1<sup>Cre</sup></em> hippocampi occur at known imprinted loci such as <em>Nnat</em> <span class="citation">(Evans <a href="#ref-EvansComparativePhylogeneticAnalysis2005">2005</a>)</span>, <em>Mkrn3</em> <span class="citation">(Glenn et al. <a href="#ref-GlennModification15q11q13DNA1993">1993</a>)</span>, and non-cannonical loci such as <em>Xlr3c</em> and <em>Xlr3a</em> <span class="citation">(Bonthuis et al. <a href="#ref-BonthuisNoncanonicalGenomicImprinting2015">2015</a>)</span>. This effect is in line with the known role of HP1 proteins to stimulate Dnmt1 activity in vitro <span class="citation">(Smallwood et al. <a href="#ref-SmallwoodFunctionalcooperationHP12007">2007</a>)</span>. However, the ~20 genes normally imprinted in mouse ( <em>Plagl1</em>, <em>H19</em>, <em>Igfr2</em> etc.) appear to be unnaffected in the hippocampus. What can be observed is elevated transcription of non-cannonically imprinted genes that appear to be tissue-specific, <em>Nnat</em>, which is normally expressed in the CA3 region of the hippocampus <span class="citation">(Oyang et al. <a href="#ref-OyangFunctionalCharacterizationDendritically2011">2011</a>)</span>.</p>
<p>A recent work, which restored Dnmt1 to <em>Dnmt1 Null</em> ESCs resulting in viable chimeras, identified 2468 No restored DMRs (NORED) regions; regions where Dnmt1-dependent methylation is not recovered once abolished <span class="citation">(Martos et al. <a href="#ref-MartosTwoapproachesreveal2017">2017</a>)</span>. Tellingly, many genes changed in <em>HP1<span class="math inline">\(\beta\)</span><sup>fl/fl</sup>HP1<span class="math inline">\(\gamma\)</span><sup>fl/fl</sup>Emx1<sup>Cre</sup></em> hippocampi–including some of the strongest changes, are related to NORED regions. These include <em>Oas3</em>, <em>Nnat</em>, <em>Xbp1</em>, <em>Got1l1</em>, and <em>Pcdha1</em>, <em>Pcdha2</em>, <em>Pcdha8</em>, <em>Pcdha11</em>, <em>Pcdhb9</em>. Because <em>Emx1<sup>Cre</sup></em> mediated excision of HP1 genes occurs from E10.5 onwards, it follows that the sustained absence of HP1<span class="math inline">\(\beta\)</span> and HP1<span class="math inline">\(\gamma\)</span> is resulting in a loss of normal DNA maintenance methylation, and this may be the root cause of the majority of the genic and non-genic effects observed. Because the genes that are de-regulated are predominantly tissue-specific (ie. neuronal), it is likely that their de-repression is coupled with their initial transcription. This very phenomena has previously been observed with IAP elements in aging mice, where their periodic activation results in progressive demethylation and complete de-silencing <span class="citation">(Barbot et al. <a href="#ref-BarbotEpigeneticregulationIAP2002">2002</a>)</span>.</p>
<p>The known role of DNA methylation in the regulation of the protocadherin cluster <span class="citation">(Kaneko et al. <a href="#ref-KanekoExpressionlevelsProtocadherina2009">2009</a>)</span> means it is entirely possible that the loss of H4K20me3 in HP1<span class="math inline">\(\gamma\)</span>KO is unrelated to their elevated transcription. In this scenario, despite recruitment of DNMT1 to the SETDB1/H3K9me3 complex, its enzymatic activity is not stimulated in the absence of HP1<span class="math inline">\(\gamma\)</span>. As protocadherins are strongly expressed in neurons during their maturation, sustained transcriptional activation results in greater demethylation and subsequently, higher transcription. Concomitant to this is the fact that H4K20me3 is completely lost in the HP1<span class="math inline">\(\gamma\)</span>KO but de-repression of repetitive elements is minimal. While it cannot be excluded that SETDB1/H3K9me3 distribution at the protocadherin locus is unaffected in HP1<span class="math inline">\(\gamma\)</span>KO neurons, this is unlikely given H3K9me3 appears to be largely unaffected at <em>Pcdh</em> regulatory regions in HP1cTKO cells (Figure <a href="3-discussion.html#fig:EScadclust">3.2</a>). Ultimately, why genic and non-genic regions normally regulated by DNA methylation are specifically affected in the HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO is unclear. The fact that the same effect is not observed in single mutants could make an argument for HP1 dosage, where the presence of HP1<span class="math inline">\(\alpha\)</span> alone is insufficient to stimulate Dnmt1-mediated maintenance methylation. However if this were true one would expect similar partial transcriptional dysregulation in HP1<span class="math inline">\(\beta\)</span>KO and HP1<span class="math inline">\(\gamma\)</span>KO single mutants, but this is not the case, as HP1<span class="math inline">\(\beta\)</span> and HP1<span class="math inline">\(\gamma\)</span> are not entirely redundant in neurons.</p>
<p>It is worth considering how the embryonic expression pattern of HP1<span class="math inline">\(\beta\)</span>, HP1<span class="math inline">\(\gamma\)</span>, and de-repressed IAP elements in the cortex may inform on the molecular mechanism. While HP1<span class="math inline">\(\gamma\)</span> is more ubiquitously expressed in the developing dorsal telencephalon (Figure <a href="2-results.html#fig:HP1brainexp">2.4</a>), HP1<span class="math inline">\(\beta\)</span> is more robustly expressed in post-mitotic neurons, where repeats are exclusively de-repressed (Figure <a href="2-results.html#fig:IAPcons">2.11</a>). This expression of repetitive elements in neurons of the cortical plate overlaps completely with the known wave of transcriptionally permissive 5-hydroxymethylation (5hmC) that occurs upon cell cycle exit <span class="citation">(Hahn et al. <a href="#ref-HahnDynamics5HydroxymethylcytosineChromatin2013">2013</a>)</span>. HP1<span class="math inline">\(\beta\)</span> and HP1<span class="math inline">\(\gamma\)</span> no doubt have a major role in protecting repeat silencing during this wave of hydroxymethylation, as repeat activation is immediate in their absence. This may be related to HP1 proteins stimulating Dnmt1 activity via Uhrf1, as <em>Uhrf1<sup>fl/fl</sup>Emx1<sup>Cre</sup></em> mice show similar deficiencies in DNA methylation, upregulation of IAP elements, and increased 5hmC on IAP elements <span class="citation">(Ramesh et al. <a href="#ref-RameshLossUhrf1neural2016">2016</a>)</span>. HP1 proteins may be central to insulate repeats and imprinted loci from TET-mediated hydroxylation, as Uhrf1 binds 5mC and 5hmC with equal affinity <span class="citation">(Frauer et al. <a href="#ref-FrauerRecognition5HydroxymethylcytosineUhrf12011">2011</a>)</span>. How this occurs is yet to be determined, and remains a suitable course for future experiments.</p>
</div>
</div>
<div id="concordance-with-aging-and-disease" class="section level2">
<h2><span class="header-section-number">3.3</span> Concordance with Aging and Disease</h2>
<p>The central question remains: how does heterochromatin loss and HP1<span class="math inline">\(\beta\)</span> &amp; HP1<span class="math inline">\(\gamma\)</span> deficiency contribute to disease states and normal aging in the human brain? Multiple studies have established that genetic associations for age-related traits may be mediated by DNA methylation <span class="citation">(Bell et al. <a href="#ref-BellEpigenomeWideScansIdentify2012a">2012</a>; Horvath <a href="#ref-HorvathDNAmethylationage2013">2013</a>; Verschoor et al. <a href="#ref-VerschoorDNAmethylationpatterns2018">2018</a>)</span>. However, <em>DNMT1<sup>fl/fl</sup>Emx1<sup>Cre</sup></em> and <em>Uhrf1<sup>fl/fl</sup>Emx1<sup>Cre</sup></em> mutants show aggressive degenerative phenotypes which void their ability to model age-related changes in DNA methylation. Despite the minor malformation of the dentate gyrus, the <em>HP1<span class="math inline">\(\beta\)</span><sup>fl/fl</sup>HP1<span class="math inline">\(\gamma\)</span><sup>fl/fl</sup>Emx1<sup>Cre</sup></em> mutant is relatively stable, and provides a unique model to study both how heterochromatin loss may be a driver of aging and age-related disease risk, by affecting DNA methylation.</p>
<p>Aging studies often attribute the loss of plasticity in aging hippocampi to a decreased splicing capacity <span class="citation">(Stilling et al. <a href="#ref-StillingDeregulationgeneexpression2014">2014</a>)</span>. Deficiency of HP1 proteins is known to negatively affect splicing by affecting DNA methylation of exons <span class="citation">(Yearim et al. <a href="#ref-YearimHP1InvolvedRegulating2015">2015</a>)</span>, and while mRNA splicing was not explicitly analyzed here, it is likely that aberrations in mRNA splicing occur in <em>HP1<span class="math inline">\(\beta\)</span><sup>fl/fl</sup>HP1<span class="math inline">\(\gamma\)</span><sup>fl/fl</sup>Emx1<sup>Cre</sup></em> mutants, and this further contributes to decreased neuronal function. What is clear, is that the primary changes in expression of genic and non genic entities in HP1DKO hippocampi is concordant with many effects seen during normal aging.</p>
<p>The present study compared baseline adult mice (3 months) with older adult mice (13-14 months), however mice can live much longer, upwards of 24-29 months where the mouse is considered geriatric. The transcriptomes of cortex and hippocampi in these ‘geriatric’ mice show uncanny commonalities with <em>HP1<span class="math inline">\(\beta\)</span><sup>fl/fl</sup>HP1<span class="math inline">\(\gamma\)</span><sup>fl/fl</sup>Emx1<sup>Cre</sup></em> hippocampi. They show downregulation of <em>ZFP57</em> and <em>SUV420H2</em>, along with upregulation of much of the protocadherins <span class="citation">(Sanchez et al. <a href="#ref-SanchezAgingApolipoproteinMolecular2015a">2015</a>)</span>. Some of the strongest changes in <em>HP1<span class="math inline">\(\beta\)</span><sup>fl/fl</sup>HP1<span class="math inline">\(\gamma\)</span><sup>fl/fl</sup>Emx1<sup>Cre</sup></em> are also the strongest observed in very aged hippocampi. These include the upregulation of <em>Pisd-ps1</em> and <em>Pisd-ps2</em>, two mysterious ncRNAs consistently observed with aging, and complement components <em>C1qa</em>, <em>C3</em> and <em>C4b</em> <span class="citation">(Sanchez et al. <a href="#ref-SanchezAgingApolipoproteinMolecular2015a">2015</a>; Stilling et al. <a href="#ref-StillingDeregulationgeneexpression2014">2014</a>; Ianov et al. <a href="#ref-IanovHippocampalTranscriptomicProfiles2017">2017</a>; de Magalhães, Curado, and Church <a href="#ref-deMagalhaesMetaanalysisagerelatedgene2009">2009</a>)</span> whose relevance will be discussed in the next section.</p>
<div id="complement-in-cns-aging-and-disease" class="section level3 unnumbered">
<h3><em>Complement in CNS Aging and Disease</em></h3>
<p>Over the past decade new roles for the Complement system in the central nervous system (CNS) have been described, and are now understood to play pivotal roles in synaptic development, aging and neurodegenerative disease. Historically, Complement proteins were known for their role in the immune system in opsonizing (or ‘tagging’) pathogens for rapid elimination by phagocytic macrophages. During CNS development, Complement proteins are widely expressed in neurons and glia in the postnatal brain and are localized to synapses during the remodelling period, where microglia mediate synapse removal <span class="citation">(Stevens et al. <a href="#ref-StevensClassicalComplementCascade2007">2007</a>)</span>. Since that time, dysregulation of Complement components in the CNS have become increasingly associated with disease states. Complement pathways have been observed to be reactivated in depression <span class="citation">(Wei et al. <a href="#ref-WeiPlasmacomplementcomponent2017">2017</a>)</span>, Huntington’s disease <span class="citation">(Singhrao et al. <a href="#ref-SinghraoIncreasedcomplementbiosynthesis1999">1999</a>)</span>, ALS <span class="citation">(Lee et al. <a href="#ref-LeeComplementcomponentsare2018">2018</a>)</span>, Alzheimer’s disease <span class="citation">(Fischer et al. <a href="#ref-FischerComplementC1qC31995">1995</a>; Lian et al. <a href="#ref-LianNFkBActivatedAstroglialRelease2015">2015</a>; Hong et al. <a href="#ref-HongComplementmicrogliamediate2016">2016</a>; Trouw et al. <a href="#ref-TrouwC4bbindingproteinAlzheimer2008">2008</a>)</span>, tauopathies <span class="citation">(Sharma et al. <a href="#ref-SharmaPresymptomaticchangemicroRNAs2018">2018</a>)</span>, and over the course of normal aging <span class="citation">(Stephan et al. <a href="#ref-StephanDramaticIncreaseC1q2013">2013</a>; de Magalhães, Curado, and Church <a href="#ref-deMagalhaesMetaanalysisagerelatedgene2009">2009</a>)</span>. In Alzheimer’s disease, microglial activation coincides with an increase in complement C1q and resulting synapse loss that precedes plaque pathology in the hippocampus <span class="citation">(Hong et al. <a href="#ref-HongComplementmicrogliamediate2016">2016</a>)</span>. Additionally, intragenic duplication of complement receptor 1 ( <em>CR1</em> ) is known to be one of the most potent risk factors for late-onset Alzheimer’s disease <span class="citation">(Lambert et al. <a href="#ref-LambertMetaanalysis740462013">2013</a>; Kucukkilic et al. <a href="#ref-KucukkilicComplementreceptorgene2018">2018</a>)</span>. Similarly, variation at the <em>C4b</em> locus serves as a potent risk factor for schizophrenia <span class="citation">(Sekar et al. <a href="#ref-SekarSchizophreniariskcomplex2016">2016</a>; Nimgaonkar et al. <a href="#ref-Nimgaonkarcomplementsystemgateway2017">2017</a>)</span>.</p>
<p>Normal aging is associated with a dramatic increase in C1q in the brain <span class="citation">(Stephan et al. <a href="#ref-StephanDramaticIncreaseC1q2013">2013</a>)</span>, which in addition to its inflammatory roles, may promote aging -related phenotypes by activation of the <em>Wnt</em> pathway <span class="citation">(Naito et al. <a href="#ref-NaitoComplementC1qActivates2012a">2012</a>)</span>. How astrocytes and microglia coordinate the complement cascade is not fully understood, but it has been suggested that activated microglia induce A1-like reactive astrocytes by secreting TNF<span class="math inline">\(\alpha\)</span>, Il-1<span class="math inline">\(\alpha\)</span> and C1q <span class="citation">(Liddelow et al. <a href="#ref-LiddelowNeurotoxicreactiveastrocytes2017">2017</a>)</span>. Subsequently, as normal aging induces activated microglia <span class="citation">(Norden and Godbout <a href="#ref-NordenMicrogliaagedbrain2013">2013</a>)</span>, this in turn induces chronic A1 astrocyte reactivity which is detrimental to neuronal function <span class="citation">(Clarke et al. <a href="#ref-ClarkeNormalaginginduces2018">2018</a>)</span>.</p>
<p>Complement activation in the aging brain may have as many triggers as there are pathways to inflammation. What the present study indicates is that the primary inducer of age-related increases in complement cascades may be of endogenous origin, namely the activation of an innate immune response via by endogenous retroviruses.</p>
</div>
<div id="a-pathway-to-complement" class="section level3 unnumbered">
<h3><em>A Pathway to Complement</em></h3>
<p>How heterochromatin loss results in complement activation is likely to operate by an initial innate immune response to elevated ERV transcripts. This is likely initiated by a member of the RIG-1 family of cytosolic pattern recognition receptors (PRRs), such as RIG-1 (Retinoic-acid Inducible Gene 1, official gene name <em>DDX58</em>) or MDA5 (Melanoma Differentiation-Associated protein 5, official gene name <em>IFIH1</em>). RIG-1 and MDA5 have been observed to recognize dsRNA from exogenous and endogenous viruses and initiate immune responses by associating with the Mitochondrial antiviral-signaling protein (MAVS), which induces an inflammatory response by signalling via IRF3 or IRF7 (Interferon regulatory factor 3 or 7 respectively) <span class="citation">(Cuellar et al. <a href="#ref-CuellarSilencingretrotransposonsSETDB12017">2017</a>; Loo et al. <a href="#ref-LooDistinctRIGIMDA52008">2008</a>; Dai et al. <a href="#ref-DaicytosolicdsRNAsensingpathway2018">2018</a>)</span>.</p>
<p>This innate immune response to ERVs has been elucidated in several cancer studies. For example, treatment with drugs that inhibit DNA methylation (DNMTi) results in de-repression of endogenous retroviruses and activation of an interferon response via MDA5/MAVS/IRF7 <span class="citation">(Chiappinelli et al. <a href="#ref-ChiappinelliInhibitingDNAMethylation2015">2015</a>; Roulois et al. <a href="#ref-RouloisDNADemethylatingAgentsTarget2015a">2015</a>)</span>. In another case <em>SETDB1</em> is found upregulated in a number of cancers, and upon deletion of <em>SETDB1</em>, elevated transcripts of dsRNAs derived from retrotransposons induce an interferon response via RIG-1/MDA5/MAVS/IRF7 <span class="citation">(Cuellar et al. <a href="#ref-CuellarSilencingretrotransposonsSETDB12017">2017</a>)</span>.</p>
<p><em>HP1<span class="math inline">\(\beta\)</span><sup>fl/fl</sup>HP1<span class="math inline">\(\gamma\)</span><sup>fl/fl</sup>Emx1<sup>Cre</sup></em> hippocampi show various signatures of activation of this innate immune pathway and is represented in Figure <a href="3-discussion.html#fig:finalmodel">3.3</a>. They show elevated transcription of <em>WDFY4</em>, which is known to augment MDA5 signalling <span class="citation">(Kochi et al. <a href="#ref-KochiSplicingvariantWDFY42018a">2018</a>)</span>. Activation of the MDA5/MAVS/IRF3/7 pathway stimulates expression of 2’-5’-Oligoadenylate Synthetase 3 ( <em>OAS3</em> , also strongly upregulated in <em>HP1<span class="math inline">\(\beta\)</span><sup>fl/fl</sup>HP1<span class="math inline">\(\gamma\)</span><sup>fl/fl</sup>Emx1<sup>Cre</sup></em> hippocampi) <span class="citation">(Rutherford, Hannigan, and Williams <a href="#ref-RutherfordInterferoninducedbinding1988">1988</a>)</span> whose expression controls the activity of RNase L <span class="citation">(Li et al. <a href="#ref-LiActivationRNasedependent2016">2016</a>)</span>. Upon contact with dsRNA, OAS3 produces 2’,5’-oligoadenylate (2-5A) <span class="citation">(Kristiansen et al. <a href="#ref-Kristiansenoligoadenylatesynthetasefamily2011">2011</a>)</span>, a molecule required for the activity of RNase L. Once activated, RNase L is able to digest dsRNA in the cytoplasm.</p>
<p>It is difficult to pinpoint the interferon pathway from transcriptome data, as induced genes from type I and type II interferon (IFN) can be largely redundant <span class="citation">(Crotta et al. <a href="#ref-CrottaTypetypeIII2013">2013</a>)</span>. In <em>HP1<span class="math inline">\(\beta\)</span><sup>fl/fl</sup>HP1<span class="math inline">\(\gamma\)</span><sup>fl/fl</sup>Emx1<sup>Cre</sup></em> hippocampi the initial innate immune response due to ERV activation appears to be related to IFN-<span class="math inline">\(\gamma\)</span>. IFN-<span class="math inline">\(\gamma\)</span> is the key regulator of macrophage activation via the JAK/STAT pathway <span class="citation">(Cherwinski et al. <a href="#ref-CherwinskiTwotypesmouse1987">1987</a>)</span> and its regulation is dependent on assembly of its receptor, comprised of IFNGR1 and IFNGR2. The upregulation of IFNGR2 in HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO hippocampi is a strong indication of IFN-<span class="math inline">\(\gamma\)</span> pathway activation, as IFNGR1 is normally produced in excess, and IFNGR2 is more tightly regulated in most cell types <span class="citation">(Bernabei et al. <a href="#ref-BernabeiInterferongreceptorexpression2001">2001</a>; Regis et al. <a href="#ref-RegisIFNgR2traffickingtunes2006">2006</a>)</span>, such that its surface expression dictates responsiveness to IFN-<span class="math inline">\(\gamma\)</span> <span class="citation">(Bach et al. <a href="#ref-BachLigandInducedAutoregulationIFNg1995">1995</a>)</span>. In multiple sclerosis, a disease where human ERVs are suspect, surface localization of IFNGR2 in T cells has been shown to be dependent on JAK2, and that this pathway is required for IFN-<span class="math inline">\(\beta\)</span> -based therapy to be effective <span class="citation">(Conti et al. <a href="#ref-ContiTh17CellsMultiple2012">2012</a>)</span>. In otherwords, elevated expression of IFNGR2 makes cells sensitive to both IFN-<span class="math inline">\(\gamma\)</span> (type II) and IFN<span class="math inline">\(\beta\)</span> (type I) signalling. Activated astrocytes in many neurodegenerative diseases show upregulation of IFNGRs <span class="citation">(Hashioka et al. <a href="#ref-HashiokaInterferongdependentcytotoxicactivation2009">2009</a>)</span>, suggesting increased sensitivity to IFN-<span class="math inline">\(\gamma\)</span> signalling. Once effected, IFN-<span class="math inline">\(\gamma\)</span> signalling has been found to up-regulate expression of the complement components <em>C3</em> and <em>C4</em> by stabilization of their mRNA <span class="citation">(Mitchell et al. <a href="#ref-MitchellIFNgammaupregulatesexpression1996">1996</a>)</span>, an effect that has been observed in the brain <span class="citation">(Chakrabarty et al. <a href="#ref-ChakrabartyIFNgPromotesComplement2010">2010</a>)</span>. Notably, in addition to being described in several other non-inflammatory roles, IFN<span class="math inline">\(\gamma\)</span> signalling has been shown to promote tau hyperphosphorylation <span class="citation">(A. Li et al. <a href="#ref-LiIFNgpromotesphosphorylation2015">2015</a>)</span>, a feature commonly observed in Alzheimer’s Disease.</p>
<div class="figure" style="text-align: center"><span id="fig:finalmodel"></span>
<img src="figure/discussion/final_model_black_arrows.png" alt="Heterochromatin Loss Results in De-repression of ERVs and Activaton of Innate Immune Pathways and Unfolded Protein Response (UPR). (A): Deficiency of HP1 proteins results in loss of DNA methylation at tissue specific Differentially Methylated Regions (tDMRs) and Endogenous Retroviruses (ERVs). (B): De-repressed ERVs are (1) detected in the cytoplasm by MDA5/RIG-1, which (2) signals via MAVS to (3a) activate IRF3/7 to induce 4) interferon stimulated genes (ISGs) in the nucleus, such as OAS3 and IFNGR2.  OAS3 bound to dsRNA produces 2',5'-oligoadenylate (2-5A), which activates RNase L to digest dsRNA.  Upregulation of IFNGR2 sensitizes the cell to IFN$\gamma$ signalling, which is known to stabilize C3 and C4b mRNAs. (3b) Activation of PKR results in phosphorylation of eIF2$\alpha$, which results in translation of ATF4, further inducing unfolded protein response genes (UPRGs) (5). All induced genes marked with green upward arrows show upregulation in HP1$\beta$/$\gamma$ DKO hippocampi." width="100%"   />
<p class="caption">
Figure 3.3: Heterochromatin Loss Results in De-repression of ERVs and Activaton of Innate Immune Pathways and Unfolded Protein Response (UPR). (A): Deficiency of HP1 proteins results in loss of DNA methylation at tissue specific Differentially Methylated Regions (tDMRs) and Endogenous Retroviruses (ERVs). (B): De-repressed ERVs are (1) detected in the cytoplasm by MDA5/RIG-1, which (2) signals via MAVS to (3a) activate IRF3/7 to induce 4) interferon stimulated genes (ISGs) in the nucleus, such as OAS3 and IFNGR2. OAS3 bound to dsRNA produces 2’,5’-oligoadenylate (2-5A), which activates RNase L to digest dsRNA. Upregulation of IFNGR2 sensitizes the cell to IFN<span class="math inline">\(\gamma\)</span> signalling, which is known to stabilize C3 and C4b mRNAs. (3b) Activation of PKR results in phosphorylation of eIF2<span class="math inline">\(\alpha\)</span>, which results in translation of ATF4, further inducing unfolded protein response genes (UPRGs) (5). All induced genes marked with green upward arrows show upregulation in HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO hippocampi.
</p>
</div>

</div>
<div id="unfolded-protein-response-upr" class="section level3 unnumbered">
<h3><em>Unfolded Protein Response (UPR)</em></h3>
<p>Major neurodegenerative diseases such as Alzheimer’s (AD), Parkinson’s (PD), Huntington’s (HT), frontotemporal dementia and Amyotrophic Lateral Sclerosis (ALS) are characterized by the accumulation of misfolded proteins or protein aggregates. Notable players in the <em>HP1<span class="math inline">\(\beta\)</span><sup>fl/fl</sup>HP1<span class="math inline">\(\gamma\)</span><sup>fl/fl</sup>Emx1<sup>Cre</sup></em> unfolded protein response include elevated transcription of <em>ATF5</em>, <em>Pdia6</em>, and <em>XBP1</em>. Several <em>XBP1</em> target genes are also upregulated, such as <em>WFS1</em> and <em>Creb3l1</em> a gene whose function has already been implicated in dendrite loss <span class="citation">(Chung et al. <a href="#ref-ChungGolgiOutpostSynthesis2017">2017</a>)</span>.</p>
<p>Because extensive cross-talk is known to occur between innate immune pathways and the unfolded protein response <span class="citation">(Smith <a href="#ref-Smithnewparadigminnate2014">2014</a>)</span>, UPR activation in <em>HP1<span class="math inline">\(\beta\)</span><sup>fl/fl</sup>HP1<span class="math inline">\(\gamma\)</span><sup>fl/fl</sup>Emx1<sup>Cre</sup></em> hippocampi is inseparable from ERV de-repression. Despite the demonstrated synergies between innate immune and UPR pathways, how UPR machinery becomes active after induction by the RIG-1/MDA5 remains unclear. Cross-talk occurs between innate immune responses but appears to be selective. For instance, it was observed that toll-like-receptor (TLR) activation can induce XBP1 transcription to activate UPR in macrophages, but this occurs only during engagement of TLR2 and 4, and not TLR3,7 or 9 <span class="citation">(Martinon et al. <a href="#ref-MartinonTLRactivationtranscription2010">2010</a>)</span>. Furthermore it was found that engagement of the UPR response (in the form of spliced XBP1) was required for optimal TLR stimulated induction of cytokines and inflammatory mediators <span class="citation">(Martinon et al. <a href="#ref-MartinonTLRactivationtranscription2010">2010</a>)</span>. Other arms of the UPR response tend to be as equally important in initiating an immune response, as deficiency of GADD34 ( <em>PRR1R15A</em> , a UPR gene) severely impairs IFN-<span class="math inline">\(\beta\)</span> response to virus, which was thought to act in parallel with MDA5 pathway for dsRNA sensing <span class="citation">(Clavarino et al. <a href="#ref-ClavarinoInductionGADD34Necessary2012">17AD–2012</a>)</span>.</p>
<p>Many exogenous viruses are known to activate the unfolded protein response, such as Hepatitis B <span class="citation">(Pager et al. <a href="#ref-PagerModulationhepatitisvirus2013">2013</a>; Mastroeni et al. <a href="#ref-MastroeniLasercapturedmicrogliaAlzheimer2018">2018</a>)</span>, West Nile <span class="citation">(Medigeshi et al. <a href="#ref-MedigeshiWestNilevirus2007">2007</a>)</span>, Dengue <span class="citation">(Umareddy et al. <a href="#ref-UmareddyDenguevirusserotype2007">2007</a>)</span>, respiratory syncytial virus <span class="citation">(Cervantes-Ortiz, Zamorano Cuervo, and Grandvaux <a href="#ref-Cervantes-OrtizRespiratorySyncytialVirus2016">2016</a>)</span>, and Zika <span class="citation">(Gladwyn-Ng et al. <a href="#ref-Gladwyn-NgStressinducedunfoldedprotein2018">2018</a>)</span>. Many other viruses act by modulating the UPR response for their benefit, such as hepatitis C <span class="citation">(Ke and Chen <a href="#ref-KeActivationunfoldedprotein2011">2011</a>, <span class="citation">Tardif et al. (<a href="#ref-TardifHepatitisvirussuppresses2004">2004</a>)</span>)</span>. Recently, many endogenous repetitive elements have been observed in stress granules, such as Alu <span class="citation">(Fitzpatrick and Huang <a href="#ref-FitzpatrickUTRlocatedinvertedAlu2012">2012</a>)</span>, and the LINE1 open reading frame <span class="citation">(Goodier et al. <a href="#ref-GoodierLINE1ORF1Protein2007">2007</a>)</span>. In pro-B cells, deletion of <em>SETDB1</em> resulted in de-repression of ERVs and a transcriptional profile indicating a similar unfolded protein response <span class="citation">(Pasquarella et al. <a href="#ref-PasquarellaRetrotransposonderepressionleads2016a">2016</a>)</span>.</p>
<p>Given induction of innate immunity to ERVs via MAVS at the mitochondrial membrane, it is likely that in <em>HP1<span class="math inline">\(\beta\)</span><sup>fl/fl</sup>HP1<span class="math inline">\(\gamma\)</span><sup>fl/fl</sup>Emx1<sup>Cre</sup></em> hippocampi, UPR signaling originates there. This is also concordant with <em>HP1<span class="math inline">\(\beta\)</span><sup>fl/fl</sup>HP1<span class="math inline">\(\gamma\)</span><sup>fl/fl</sup>Emx1<sup>Cre</sup></em> hippocampi showing elevated transcription of <em>ATF5</em>, a transcription factor associated with the mitochondrial unfolded protein response (UPR<sup>mt</sup>). UPR originating from MAVS is suspected to involve MAVS-mediated activation of protein kinase R (PKR), although the mechanism is unknown. An alternative is that PKR is activated by IFN<span class="math inline">\(\gamma\)</span> mRNA <span class="citation">(Ben-Asouli et al. <a href="#ref-Ben-AsouliHumaninterferongmRNA2002">2002</a>)</span> or directly by ERV dsRNA <span class="citation">(Lemaire et al. <a href="#ref-LemaireMechanismPKRActivation2008">2008</a>; Williams <a href="#ref-WilliamsPKRsentinelkinase1999">1999</a>)</span>. Once activated, PKR phosphorylates c-Jun N-terminal kinase JNK2, which activates c-Jun, initiating a nuclear transcriptional response <span class="citation">(Rath et al. <a href="#ref-RathInductiondsRNAactivatedprotein2012">2012</a>)</span>. In terms of classical UPR pathways, activated PKR can directly phosphorylate eIF2<span class="math inline">\(\alpha\)</span> <span class="citation">(Rojas, Arias, and L’opez <a href="#ref-RojasProteinKinaseResponsible2010">2010</a>)</span>. Once phosphorylated, eIF2<span class="math inline">\(\alpha\)</span> initates a repression of global translation initiation along with a selective enhancement of ATF4 translation <span class="citation">(Vattem and Wek <a href="#ref-VattemReinitiationinvolvingupstream2004">2004</a>)</span>, which then initiates transcription of UPR genes in the nucleus (Figure <a href="3-discussion.html#fig:finalmodel">3.3</a>).</p>
<p>Elevated levels of Neuronatin ( <em>Nnat</em> ) is also likely to contribute to the unfolded protein response. Strongly expressed in the CA3 region of the hippocampus, neuronatin is a proteolipid, a class of proteins normally found as subunits of membrane channels in the endoplasmic reticulum (ER) which facilitate calcium influx during the normal folding and assembly of proteins. Neuronatin also has proven functions for simulating the release of calcium ions in the dendrites of hippocampal neurons, a process which is necessary for <em>in-situ</em> translation <span class="citation">(Oyang et al. <a href="#ref-OyangFunctionalCharacterizationDendritically2011">2011</a>)</span>. Neuronatin is notably aggregate prone, as evidenced in patients with Lafora disease. Under normal physiological conditions, neuronatin is marked for degradation by the E3-Ubiquitin ligase malin. In Lafora disease, a mutation in the malin gene results in the accumulation of cytotoxic misfolded neuronatin aggregates which disrupt the endoplasmic reticulum <span class="citation">(Sharma et al. <a href="#ref-SharmaNeuronatinmediatedaberrantcalcium2013">2013</a>)</span>. Thus, loss of imprinting and bi-allelic expression of <em>Neuronatin</em> due to heterochromatin loss marks it as an important risk factor in age-related diseases of the hippocampus.</p>
</div>
<div id="viruses-transactivation-and-neurodegeneration" class="section level3 unnumbered">
<h3><em>Viruses, Transactivation and Neurodegeneration</em></h3>
<p>Given the anti-viral inflammatory responses observed following loss of repeat silencing in <em>HP1<span class="math inline">\(\beta\)</span><sup>fl/fl</sup>HP1<span class="math inline">\(\gamma\)</span><sup>fl/fl</sup>Emx1<sup>Cre</sup></em> hippocampi, it is natural to draw parallels to exogenous viral infections of the brain. Indeed, a recent study found that exogenous infection by west nile virus phenocopies what occurs in <em>HP1<span class="math inline">\(\beta\)</span><sup>fl/fl</sup>HP1<span class="math inline">\(\gamma\)</span><sup>fl/fl</sup>Emx1<sup>Cre</sup></em> hippocampi, where synapse loss in the CA3 region of the hippocampus occurs via the activation of complement <span class="citation">(Vasek et al. <a href="#ref-Vasekcomplementmicroglialaxis2016">2016</a>)</span>. In fact, evidence is mounting to implicate infection with comon exogenous viruses to be causative in several neurodegenerative diseases.</p>
<p>In Alzheimer’s Disease (AD), evidence for exogenous infection was first put forward in 1991 where a low-throughput study found that brains from AD patients contain Herpes simplex virus type 1 (HSV1) DNA <span class="citation">(Jamieson et al. <a href="#ref-JamiesonLatentherpessimplex1991">1991</a>)</span>. By 2016, over 100 publications have supported either direct or indirect associations making HSV1 the pathogen most strongly linked to AD <span class="citation">(Itzhaki et al. <a href="#ref-ItzhakiMicrobesAlzheimerDisease2016">2016</a>)</span>. HSV1 DNA has been found to be localized to amyloid plaques <span class="citation">(Wozniak, Mee, and Itzhaki <a href="#ref-WozniakHerpessimplexvirus2009">2009</a>)</span>, and it has recently been understood that following HSV1 or HHV-6A infection, amyloid-<span class="math inline">\(\beta\)</span> (A<span class="math inline">\(\beta\)</span>) is rapidly induced to surround and contain viral DNA <span class="citation">(Eimer et al. <a href="#ref-EimerAlzheimerDiseaseAssociatedvAmyloid2018">2018</a>)</span>. Recent sequencing experiments have profiled viral content in the aging brain and have found that many common viral species are frequently observed in the normal aging brain, but that elevated levels of human herpesvirus 6A and 7 (HHV-6A and HHV-7) was unique to Alzheimer’s brains <span class="citation">(Readhead et al. <a href="#ref-ReadheadMultiscaleAnalysisIndependent2018">2018</a>)</span>. The same study found that HHV-6A interacts directly or indirectly with most of the known Alzheimer’s disease risk genes. Furthermore, a sixteen-year clinical study in 33,448 patients found that HSV increases the risk of AD 2.5-fold and that antiviral medication could drastically reduce risk ten-fold <span class="citation">(Tzeng et al. <a href="#ref-TzengAntiherpeticMedicationsReduced2018">2018</a>)</span>. Finally, the fact that AD brain homogenates injected into healthy primates can induce AD pathology <span class="citation">(Baker et al. <a href="#ref-BakerInductionA4amyloid1994">1994</a>; Ridley et al. <a href="#ref-RidleyVerylongterm2006">2006</a>)</span> speaks to an infectious origin.</p>
<p>Aside from the negative effects of acute inflammatory response from exogenous viral infection, the greater consequence is how a lifetime of viral infection can affect the epigenome. Infection from common viruses such as HSV and even influenza are known to induce the expression of human endogenous retroviruses <span class="citation">(Nellåker et al. <a href="#ref-NellakerTransactivationelementshuman2006">2006</a>)</span>, a process termed viral transactivation. This particular phenomena has strong links to the inducement of Multiple Sclerosis (MS), where both HSV and Epstein Barr Virus (EBV) have been linked to activation of human endogenous retrovirus W (HERV-W) and subsequent activation of the immune system <span class="citation">(Brudek et al. <a href="#ref-BrudekSynergisticimmuneresponses2008">2008</a>; Mameli et al. <a href="#ref-MameliActivationMSRVTypeEndogenous2013a">2013</a>; Mameli et al. <a href="#ref-MameliExpressionactivationEpstein2012">2012</a>)</span>.</p>
<p>The activation of endogenous retroelements can also be driven by pharmacological agents and disease genes. As mentioned above, cancer studies have shown that ERVs can be de-repressed by drugs such as DNMT inhibitors. Other drugs such as Chloroquine, which is normally used to treat malaria, has been found to trigger EBV replication by phosphorylation of KAP1 <span class="citation">(Li, Burton, and Bhaduri-McIntosh <a href="#ref-LiChloroquinetriggersEpsteinBarr2017a">2017</a>)</span>, which is then likely to have knock-on effects by transactivating HERVs. Several disease-risk genes are implicated in direct ERV activation, such as TDP-43 in ALS or tau-mediated chromatin relaxation in Alzheimer’s (section <a href="1-introduction.html#chraging">1.3.2</a>). Other risk genes may act more indirectly, such as <span class="math inline">\(\alpha\)</span>-synuclein (AD, PD, Lewy-body dementia) which has been found to sequester DNMT1 from the nucleus <span class="citation">(Desplats et al. <a href="#ref-DesplatsaSynucleinSequestersDnmt12011">2011</a>)</span>. Ultimately, our understanding of how various pharmacological and genetic interventions affect ERV activation is currently very limited. If the multiscale HSV AD study was any indication, it appears that a large majority of genes associated with neurodegenerative disease risk have essential roles in either perturbing heterochromatin stability, innate immunity, or ERV activation –pathways now understood to be intrinsically linked.</p>
</div>
</div>
<div id="concluding-remarks" class="section level2">
<h2><span class="header-section-number">3.4</span> Concluding Remarks</h2>
<p>The present study is the first to establish an uninterrupted link between heterochromatin loss, de-repression of endogenous retroviruses, and activation of immune and complement pathways resulting in decreased neuronal complexity and accelerated cognitive decline. Whats more, the concordance of the phenotype observed here with established features of normal aging and neurodegenerative disease further substantiates the heterochromatin loss model of aging.</p>
<p>It is worth considering that in addition to a great diversity of cell subtypes and gene programs, neurons have <em>exapted</em> several epigenetic features that appear to enable their function during excitation at the cost of genomic stability:</p>
<ul>
<li>From DNA repair neurons have exapted activity-dependent DNA double-strand breaks for immediate-early gene activation <span class="citation">(Madabhushi et al. <a href="#ref-MadabhushiActivityInducedDNABreaks2015">2015</a>; Madabhushi, Pan, and Tsai <a href="#ref-MadabhushiDNADamageIts2014">2014</a>)</span>.</li>
<li>From mitosis they have exapted the phosphorylation of H3S10, which ejects HP1 from H3K9me3-chromatin during periods of excitation <span class="citation">(Noh et al. <a href="#ref-NohATRXtoleratesactivitydependent2014">2014</a>)</span>.</li>
<li>From the DNA methylation cycle neurons have exapted high levels of 5hmC, perhaps to enable rapid transcription to facilitate plasticity <span class="citation">(Szulwach et al. <a href="#ref-Szulwach5hmCmediatedepigenetic2011">2011</a>)</span>.</li>
</ul>
<p>All of the above, which no doubt enable neurons to act as they do, also expose the genome to risk in the form of recombination, repeat activation or any combination of the two <span class="citation">(Newman et al. <a href="#ref-NewmanActivityDEPendentTransposition2017">2017</a>)</span>.</p>
<p>While the possibility may exist to waylay neuronal aging by halting viral transactivation, the intrinsic properties of neurons may align with universal rules; just as one cannot choose both <em>speed</em> and <em>stability</em>, perhaps one cannot choose both <em>function</em> and <em>longevity</em>.</p>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-AucottHP1requireddevelopment2008">
<p>Aucott, R., J. Bullwinkel, Y. Yu, W. Shi, M. Billur, J. P. Brown, U. Menzel, et al. 2008. “HP1- Is Required for Development of the Cerebral Neocortex and Neuromuscular Junctions.” <em>The Journal of Cell Biology</em> 183 (4): 597–606. doi:<a href="https://doi.org/10.1083/jcb.200804041">10.1083/jcb.200804041</a>.</p>
</div>
<div id="ref-TheilGli3requiredEmx1999">
<p>Theil, T., G. Alvarez-Bolado, A. Walter, and U. Ruther. 1999. “Gli3 Is Required for Emx Gene Expression During Dorsal Telencephalon Development.” <em>Development</em> 126 (16): 3561–71. <a href="http://dev.biologists.org/content/126/16/3561" class="uri">http://dev.biologists.org/content/126/16/3561</a>.</p>
</div>
<div id="ref-LinNeuralencodingconcept2007">
<p>Lin, Longnian, Guifen Chen, Hui Kuang, Dong Wang, and Joe Z. Tsien. 2007. “Neural Encoding of the Concept of Nest in the Mouse Brain.” <em>Proceedings of the National Academy of Sciences</em> 104 (14): 6066–71. doi:<a href="https://doi.org/10.1073/pnas.0701106104">10.1073/pnas.0701106104</a>.</p>
</div>
<div id="ref-UekiPrepulseinhibitionacoustic2006">
<p>Ueki, Akinori, Kyoko Goto, Noriko Sato, Hiroyuki Iso, and Yoshio Morita. 2006. “Prepulse Inhibition of Acoustic Startle Response in Mild Cognitive Impairment and Mild Dementia of Alzheimer Type.” <em>Psychiatry and Clinical Neurosciences</em> 60 (1): 55–62. doi:<a href="https://doi.org/10.1111/j.1440-1819.2006.01460.x">10.1111/j.1440-1819.2006.01460.x</a>.</p>
</div>
<div id="ref-KesslakQuantificationmagneticresonance1991">
<p>Kesslak, J. Patrick, Orhan Nalcioglu, and Carl W. Cotman. 1991. “Quantification of Magnetic Resonance Scans for Hippocampal and Parahippocampal Atrophy in Alzheimer’s Disease.” <em>Neurology</em> 41 (1): 51–51.</p>
</div>
<div id="ref-LiddelowNeurotoxicreactiveastrocytes2017">
<p>Liddelow, Shane A., Kevin A. Guttenplan, Laura E. Clarke, Frederick C. Bennett, Christopher J. Bohlen, Lucas Schirmer, Mariko L. Bennett, et al. 2017. “Neurotoxic Reactive Astrocytes Are Induced by Activated Microglia.” <em>Nature</em> 541 (7638): 481–87. doi:<a href="https://doi.org/10.1038/nature21029">10.1038/nature21029</a>.</p>
</div>
<div id="ref-StevensClassicalComplementCascade2007">
<p>Stevens, Beth, Nicola J. Allen, Luis E. Vazquez, Gareth R. Howell, Karen S. Christopherson, Navid Nouri, Kristina D. Micheva, et al. 2007. “The Classical Complement Cascade Mediates CNS Synapse Elimination.” <em>Cell</em> 131 (6): 1164–78. doi:<a href="https://doi.org/10.1016/j.cell.2007.10.036">10.1016/j.cell.2007.10.036</a>.</p>
</div>
<div id="ref-MolumbygProtocadherinsInteractNeuroligin12017">
<p>Molumby, Michael J., Rachel M. Anderson, Dillan J. Newbold, Norah K. Koblesky, Andrew M. Garrett, Dietmar Schreiner, Jason J. Radley, and Joshua A. Weiner. 2017. “Γ-Protocadherins Interact with Neuroligin-1 and Negatively Regulate Dendritic Spine Morphogenesis.” <em>Cell Reports</em> 18 (11): 2702–14. doi:<a href="https://doi.org/10.1016/j.celrep.2017.02.060">10.1016/j.celrep.2017.02.060</a>.</p>
</div>
<div id="ref-RubinsteinMolecularLogicNeuronal2015">
<p>Rubinstein, Rotem, Chan Aye Thu, Kerry Marie Goodman, Holly Noelle Wolcott, Fabiana Bahna, Seetha Mannepalli, Goran Ahlsen, et al. 2015. “Molecular Logic of Neuronal Self-Recognition Through Protocadherin Domain Interactions.” <em>Cell</em> 163 (3): 629–42. doi:<a href="https://doi.org/10.1016/j.cell.2015.09.026">10.1016/j.cell.2015.09.026</a>.</p>
</div>
<div id="ref-ToyodaDevelopmentalEpigeneticModification2014">
<p>Toyoda, Shunsuke, Masahumi Kawaguchi, Toshihiro Kobayashi, Etsuko Tarusawa, Tomoko Toyama, Masaki Okano, Masaaki Oda, et al. 2014. “Developmental Epigenetic Modification Regulates Stochastic Expression of Clustered Protocadherin Genes, Generating Single Neuron Diversity.” <em>Neuron</em> 82 (1): 94–108. doi:<a href="https://doi.org/10.1016/j.neuron.2014.02.005">10.1016/j.neuron.2014.02.005</a>.</p>
</div>
<div id="ref-IsbelWizbindsactive2016">
<p>Isbel, Luke, Lexie Prokopuk, Haoyu Wu, Lucia Daxinger, Harald Oey, Alex Spurling, Adam J Lawther, Matthew W Hale, and Emma Whitelaw. 2016. “Wiz Binds Active Promoters and CTCF-Binding Sites and Is Required for Normal Behaviour in the Mouse.” <em>eLife</em> 5. doi:<a href="https://doi.org/10.7554/eLife.15082">10.7554/eLife.15082</a>.</p>
</div>
<div id="ref-JiangmethyltransferaseSETDB1regulates2017">
<p>Jiang, Yan, Yong-Hwee Eddie Loh, Prashanth Rajarajan, Teruyoshi Hirayama, Will Liao, Bibi S Kassim, Behnam Javidfar, et al. 2017. “The Methyltransferase Setdb1 Regulates a Large Neuron-Specific Topological Chromatin Domain.” <em>Nature Genetics</em> 49 (8): 1239–50. doi:<a href="https://doi.org/10.1038/ng.3906">10.1038/ng.3906</a>.</p>
</div>
<div id="ref-IaconoIncreasedH3K9methylation2018">
<p>Iacono, Giovanni, Aline Dubos, Hamid M’eziane, Marco Benevento, Ehsan Habibi, Amit Mandoli, Fabrice Riet, et al. 2018. “Increased H3k9 Methylation and Impaired Expression of Protocadherins Are Associated with the Cognitive Dysfunctions of the Kleefstra Syndrome.” <em>Nucleic Acids Research</em>, March. doi:<a href="https://doi.org/10.1093/nar/gky196">10.1093/nar/gky196</a>.</p>
</div>
<div id="ref-ElHajjEpigeneticdysregulationprotocadherins2017d">
<p>El Hajj, Nady, Marcus Dittrich, and Thomas Haaf. 2017. “Epigenetic Dysregulation of Protocadherins in Human Disease.” <em>Seminars in Cell &amp; Developmental Biology</em>, Spectraplakins, versatile roles in physiology and pathology, 69 (September): 172–82. doi:<a href="https://doi.org/10.1016/j.semcdb.2017.07.007">10.1016/j.semcdb.2017.07.007</a>.</p>
</div>
<div id="ref-BellEpigenomeWideScansIdentify2012a">
<p>Bell, Jordana T., Pei-Chien Tsai, Tsun-Po Yang, Ruth Pidsley, James Nisbet, Daniel Glass, Massimo Mangino, et al. 2012. “Epigenome-Wide Scans Identify Differentially Methylated Regions for Age and Age-Related Phenotypes in a Healthy Ageing Population.” <em>PLOS Genetics</em> 8 (4): e1002629. doi:<a href="https://doi.org/10.1371/journal.pgen.1002629">10.1371/journal.pgen.1002629</a>.</p>
</div>
<div id="ref-SalpeaPostnataldevelopmentagerelated2012">
<p>Salpea, Paraskevi, Valya R. Russanova, Tazuko H. Hirai, Thomae G. Sourlingas, Kalliope E. Sekeri-Pataryas, Roberto Romero, Jonathan Epstein, and Bruce H. Howard. 2012. “Postnatal Development- and Age-Related Changes in DNA-Methylation Patterns in the Human Genome.” <em>Nucleic Acids Research</em> 40 (14): 6477–94. doi:<a href="https://doi.org/10.1093/nar/gks312">10.1093/nar/gks312</a>.</p>
</div>
<div id="ref-KourmouliHeterochromatintrimethylatedlysine2004">
<p>Kourmouli, Niki, Peter Jeppesen, Shantha Mahadevhaiah, Paul Burgoyne, Rong Wu, David M. Gilbert, Silvia Bongiorni, Giorgio Prantera, Laura Fanti, and Sergio Pimpinelli. 2004. “Heterochromatin and Tri-Methylated Lysine 20 of Histone H4 in Animals.” <em>Journal of Cell Science</em> 117 (12): 2491–2501.</p>
</div>
<div id="ref-Schottasilencingpathwayinduce2004">
<p>Schotta, G. 2004. “A Silencing Pathway to Induce H3-K9 and H4-K20 Trimethylation at Constitutive Heterochromatin.” <em>Genes &amp; Development</em> 18 (11): 1251–62. doi:<a href="https://doi.org/10.1101/gad.300704">10.1101/gad.300704</a>.</p>
</div>
<div id="ref-XuH4K20me3colocalizesactivating2018">
<p>Xu, Jian, and Benjamin L. Kidder. 2018. “H4K20me3 Co-Localizes with Activating Histone Modifications at Transcriptionally Dynamic Regions in Embryonic Stem Cells.” <em>BMC Genomics</em> 19 (1): 514. doi:<a href="https://doi.org/10.1186/s12864-018-4886-4">10.1186/s12864-018-4886-4</a>.</p>
</div>
<div id="ref-CaillierRoleEpigeneticRegulator2010">
<p>Caillier, Maïa, Sandrine Th’enot, Violaine Tribollet, Anne-Marie Birot, Jacques Samarut, and Anne Mey. 2010. “Role of the Epigenetic Regulator Hp1<span class="math inline">\(\gamma\)</span> in the Control of Embryonic Stem Cell Properties.” Edited by Michael J. Pazin. <em>PLoS ONE</em> 5 (11): e15507. doi:<a href="https://doi.org/10.1371/journal.pone.0015507">10.1371/journal.pone.0015507</a>.</p>
</div>
<div id="ref-YearimHP1InvolvedRegulating2015">
<p>Yearim, Ahuvi, Sahar Gelfman, Ronna Shayevitch, Shai Melcer, Ohad Glaich, Jan-Philipp Mallm, Malka Nissim-Rafinia, et al. 2015. “HP1 Is Involved in Regulating the Global Impact of DNA Methylation on Alternative Splicing.” <em>Cell Reports</em> 10 (7): 1122–34. doi:<a href="https://doi.org/10.1016/j.celrep.2015.01.038">10.1016/j.celrep.2015.01.038</a>.</p>
</div>
<div id="ref-AyoubMobilizationrecruitmentHP1v2009">
<p>Ayoub, Nabieh, Anand D. Jeyasekharan, and Ashok R. Venkitaraman. 2009. “Mobilization and Recruitment of HP1β: A Bimodal Response to DNA Breakage.” <em>Cell Cycle</em> 8 (18): 2946–51. doi:<a href="https://doi.org/10.4161/cc.8.18.9486">10.4161/cc.8.18.9486</a>.</p>
</div>
<div id="ref-Trembecka-LucasDynamicsHP1vPCNAcontainingcomplexes2013">
<p>Trembecka-Lucas, Dominika O., Dominika O. Trembecka-Lucas, Aleksander T. Szczurek, Aleksander T. Szczurek, Jurek W. Dobrucki, and Jurek W. Dobrucki. 2013. “Dynamics of the Hp1<span class="math inline">\(\beta\)</span>-PCNA-Containing Complexes in DNA Replication and Repair.” <em>Nucleus</em> 4 (1): 74–82. doi:<a href="https://doi.org/10.4161/nucl.23683">10.4161/nucl.23683</a>.</p>
</div>
<div id="ref-BilluressentialfunctionHP1v2010">
<p>Billur, Mustafa, Hans D. Bartunik, and Prim B. Singh. 2010. “The Essential Function of HP1β: A Case of the Tail Wagging the Dog?” <em>Trends in Biochemical Sciences</em> 35 (2): 115–23. doi:<a href="https://doi.org/10.1016/j.tibs.2009.09.003">10.1016/j.tibs.2009.09.003</a>.</p>
</div>
<div id="ref-NoguchiDNAMethyltransferaseIndispensable2016">
<p>Noguchi, Hirofumi, Naoya Murao, Ayaka Kimura, Taito Matsuda, Masakazu Namihira, and Kinichi Nakashima. 2016. “DNA Methyltransferase 1 Is Indispensable for Development of the Hippocampal Dentate Gyrus.” <em>The Journal of Neuroscience</em> 36 (22): 6050–68. doi:<a href="https://doi.org/10.1523/JNEUROSCI.0512-16.2016">10.1523/JNEUROSCI.0512-16.2016</a>.</p>
</div>
<div id="ref-HutnickDNAhypomethylationrestricted2009">
<p>Hutnick, L. K., P. Golshani, M. Namihira, Z. Xue, A. Matynia, X. W. Yang, A. J. Silva, F. E. Schweizer, and G. Fan. 2009. “DNA Hypomethylation Restricted to the Murine Forebrain Induces Cortical Degeneration and Impairs Postnatal Neuronal Maturation.” <em>Human Molecular Genetics</em> 18 (15): 2875–88. doi:<a href="https://doi.org/10.1093/hmg/ddp222">10.1093/hmg/ddp222</a>.</p>
</div>
<div id="ref-EvansComparativePhylogeneticAnalysis2005">
<p>Evans, H. K. 2005. “Comparative Phylogenetic Analysis of Blcap/Nnat Reveals Eutherian-Specific Imprinted Gene.” <em>Molecular Biology and Evolution</em> 22 (8): 1740–8. doi:<a href="https://doi.org/10.1093/molbev/msi165">10.1093/molbev/msi165</a>.</p>
</div>
<div id="ref-GlennModification15q11q13DNA1993">
<p>Glenn, C. C., R. D. Nicholls, W. P. Robinson, S. Saitoh, N. Niikawa, A. Schinzel, B. Horsthemke, and D. J. Driscoll. 1993. “Modification of 15q11-Q13 DNA Methylation Imprints in Unique Angelman and Prader-Willi Patients.” <em>Human Molecular Genetics</em> 2 (9): 1377–82.</p>
</div>
<div id="ref-BonthuisNoncanonicalGenomicImprinting2015">
<p>Bonthuis, Paul J., Wei-Chao Huang, Cornelia N. Stacher Hörndli, Elliott Ferris, Tong Cheng, and Christopher Gregg. 2015. “Noncanonical Genomic Imprinting Effects in Offspring.” <em>Cell Reports</em> 12 (6): 979–91. doi:<a href="https://doi.org/10.1016/j.celrep.2015.07.017">10.1016/j.celrep.2015.07.017</a>.</p>
</div>
<div id="ref-SmallwoodFunctionalcooperationHP12007">
<p>Smallwood, A., P.-O. Esteve, S. Pradhan, and M. Carey. 2007. “Functional Cooperation Between Hp1 and Dnmt1 Mediates Gene Silencing.” <em>Genes &amp; Development</em> 21 (10): 1169–78. doi:<a href="https://doi.org/10.1101/gad.1536807">10.1101/gad.1536807</a>.</p>
</div>
<div id="ref-OyangFunctionalCharacterizationDendritically2011">
<p>Oyang, Elaine L., Bonnie C. Davidson, Winfong Lee, and Michael M. Poon. 2011. “Functional Characterization of the Dendritically Localized mRNA Neuronatin in Hippocampal Neurons.” <em>PLOS ONE</em> 6 (9): e24879. doi:<a href="https://doi.org/10.1371/journal.pone.0024879">10.1371/journal.pone.0024879</a>.</p>
</div>
<div id="ref-MartosTwoapproachesreveal2017">
<p>Martos, Suzanne N, Teng Li, Ramon Bossardi Ramos, Dan Lou, Hongzheng Dai, Jin-Chong Xu, Ganglong Gao, et al. 2017. “Two Approaches Reveal a New Paradigm of ‘Switchable or Genetics-Influenced Allele-Specific DNA Methylation’ with Potential in Human Disease.” <em>Cell Discovery</em> 3 (November): 17038. doi:<a href="https://doi.org/10.1038/celldisc.2017.38">10.1038/celldisc.2017.38</a>.</p>
</div>
<div id="ref-BarbotEpigeneticregulationIAP2002">
<p>Barbot, Willy, Anne Dupressoir, Vladimir Lazar, and Thierry Heidmann. 2002. “Epigenetic Regulation of an IAP Retrotransposon in the Aging Mouse: Progressive Demethylation and de-Silencing of the Element by Its Repetitive Induction.” <em>Nucleic Acids Research</em> 30 (11): 2365–73. doi:<a href="https://doi.org/10.1093/nar/30.11.2365">10.1093/nar/30.11.2365</a>.</p>
</div>
<div id="ref-KanekoExpressionlevelsProtocadherina2009">
<p>Kaneko, Ryosuke, Masahumi Kawaguchi, Tomoko Toyama, Yusuke Taguchi, and Takeshi Yagi. 2009. “Expression Levels of Protocadherin-<span class="math inline">\(\alpha\)</span> Transcripts Are Decreased by Nonsense-Mediated mRNA Decay with Frameshift Mutations and by High DNA Methylation in Their Promoter Regions.” <em>Gene</em> 430 (1): 86–94. doi:<a href="https://doi.org/10.1016/j.gene.2008.10.018">10.1016/j.gene.2008.10.018</a>.</p>
</div>
<div id="ref-HahnDynamics5HydroxymethylcytosineChromatin2013">
<p>Hahn, Maria A., Runxiang Qiu, Xiwei Wu, Arthur X. Li, Heying Zhang, Jun Wang, Jonathan Jui, et al. 2013. “Dynamics of 5-Hydroxymethylcytosine and Chromatin Marks in Mammalian Neurogenesis.” <em>Cell Reports</em> 3 (2): 291–300. doi:<a href="https://doi.org/10.1016/j.celrep.2013.01.011">10.1016/j.celrep.2013.01.011</a>.</p>
</div>
<div id="ref-RameshLossUhrf1neural2016">
<p>Ramesh, Vidya, Efil Bayam, Filippo M. Cernilogar, Ian M. Bonapace, Markus Schulze, Markus J. Riemenschneider, Gunnar Schotta, and Magdalena Götz. 2016. “Loss of Uhrf1 in Neural Stem Cells Leads to Activation of Retroviral Elements and Delayed Neurodegeneration.” <em>Genes &amp; Development</em> 30 (19): 2199–2212. doi:<a href="https://doi.org/10.1101/gad.284992.116">10.1101/gad.284992.116</a>.</p>
</div>
<div id="ref-FrauerRecognition5HydroxymethylcytosineUhrf12011">
<p>Frauer, Carina, Thomas Hoffmann, Sebastian Bultmann, Valentina Casa, M. Cristina Cardoso, Iris Antes, and Heinrich Leonhardt. 2011. “Recognition of 5-Hydroxymethylcytosine by the Uhrf1 SRA Domain.” <em>PLOS ONE</em> 6 (6): e21306. doi:<a href="https://doi.org/10.1371/journal.pone.0021306">10.1371/journal.pone.0021306</a>.</p>
</div>
<div id="ref-HorvathDNAmethylationage2013">
<p>Horvath, Steve. 2013. “DNA Methylation Age of Human Tissues and Cell Types.” <em>Genome Biology</em> 14 (10): 3156.</p>
</div>
<div id="ref-VerschoorDNAmethylationpatterns2018">
<p>Verschoor, Chris P., Lisa M. McEwen, Michael S. Kobor, Mark B. Loeb, and Dawn M. E. Bowdish. 2018. “DNA Methylation Patterns Are Related to Co-Morbidity Status and Circulating C-Reactive Protein Levels in the Nursing Home Elderly.” <em>Experimental Gerontology</em>, Aging and Immunity 2017, 105 (May): 47–52. doi:<a href="https://doi.org/10.1016/j.exger.2017.10.010">10.1016/j.exger.2017.10.010</a>.</p>
</div>
<div id="ref-StillingDeregulationgeneexpression2014">
<p>Stilling, Roman M., Eva Benito, Michael Gertig, Jonas Barth, Vincenzo Capece, Susanne Burkhardt, Stefan Bonn, and Andre Fischer. 2014. “De-Regulation of Gene Expression and Alternative Splicing Affects Distinct Cellular Pathways in the Aging Hippocampus.” <em>Frontiers in Cellular Neuroscience</em> 8. doi:<a href="https://doi.org/10.3389/fncel.2014.00373">10.3389/fncel.2014.00373</a>.</p>
</div>
<div id="ref-SanchezAgingApolipoproteinMolecular2015a">
<p>Sanchez, Diego, Raquel Bajo-Grañeras, Manuela Del Caño-Espinel, Rosa Garcia-Centeno, Nadia Garcia-Mateo, Raquel Pascua-Maestro, and Maria D. Ganfornina. 2015. “Aging Without Apolipoprotein D: Molecular and Cellular Modifications in the Hippocampus and Cortex.” <em>Experimental Gerontology</em> 67 (July): 19–47. doi:<a href="https://doi.org/10.1016/j.exger.2015.04.003">10.1016/j.exger.2015.04.003</a>.</p>
</div>
<div id="ref-IanovHippocampalTranscriptomicProfiles2017">
<p>Ianov, Lara, Matt De Both, Monica K. Chawla, Asha Rani, Andrew J. Kennedy, Ignazio Piras, Jeremy J. Day, et al. 2017. “Hippocampal Transcriptomic Profiles: Subfield Vulnerability to Age and Cognitive Impairment.” <em>Frontiers in Aging Neuroscience</em> 9 (December). doi:<a href="https://doi.org/10.3389/fnagi.2017.00383">10.3389/fnagi.2017.00383</a>.</p>
</div>
<div id="ref-deMagalhaesMetaanalysisagerelatedgene2009">
<p>de Magalhães, João Pedro, João Curado, and George M. Church. 2009. “Meta-Analysis of Age-Related Gene Expression Profiles Identifies Common Signatures of Aging.” <em>Bioinformatics</em> 25 (7): 875–81. doi:<a href="https://doi.org/10.1093/bioinformatics/btp073">10.1093/bioinformatics/btp073</a>.</p>
</div>
<div id="ref-WeiPlasmacomplementcomponent2017">
<p>Wei, Jinxue, Ye Liu, Liansheng Zhao, Xiao Yang, Peiyan Ni, Yingcheng Wang, Tao Li, and Xiaohong Ma. 2017. “Plasma Complement Component 4 Increases in Patients with Major Depressive Disorder.” <em>Neuropsychiatric Disease and Treatment</em> 14 (December): 37–41. doi:<a href="https://doi.org/10.2147/NDT.S151238">10.2147/NDT.S151238</a>.</p>
</div>
<div id="ref-SinghraoIncreasedcomplementbiosynthesis1999">
<p>Singhrao, S. K., J. W. Neal, B. P. Morgan, and P. Gasque. 1999. “Increased Complement Biosynthesis by Microglia and Complement Activation on Neurons in Huntington’s Disease.” <em>Experimental Neurology</em> 159 (2): 362–76. doi:<a href="https://doi.org/10.1006/exnr.1999.7170">10.1006/exnr.1999.7170</a>.</p>
</div>
<div id="ref-LeeComplementcomponentsare2018">
<p>Lee, John D., Samantha C. Levin, Emily F. Willis, Rui Li, Trent M. Woodruff, and Peter G. Noakes. 2018. “Complement Components Are Upregulated and Correlate with Disease Progression in the TDP-43q331k Mouse Model of Amyotrophic Lateral Sclerosis.” <em>Journal of Neuroinflammation</em> 15 (1): 171. doi:<a href="https://doi.org/10.1186/s12974-018-1217-2">10.1186/s12974-018-1217-2</a>.</p>
</div>
<div id="ref-FischerComplementC1qC31995">
<p>Fischer, B., H. Schmoll, D. Platt, A. Popa-Wagner, P. Riederer, and J. Bauer. 1995. “Complement C1q and C3 mRNA Expression in the Frontal Cortex of Alzheimer’s Patients.” <em>Journal of Molecular Medicine</em> 73 (9): 465–71. doi:<a href="https://doi.org/10.1007/BF00202265">10.1007/BF00202265</a>.</p>
</div>
<div id="ref-LianNFkBActivatedAstroglialRelease2015">
<p>Lian, Hong, Li Yang, Allysa Cole, Lu Sun, Angie C. -A. Chiang, Stephanie W. Fowler, David J. Shim, et al. 2015. “NF<span class="math inline">\(\kappa\)</span>B-Activated Astroglial Release of Complement C3 Compromises Neuronal Morphology and Function Associated with Alzheimer’s Disease.” <em>Neuron</em> 85 (1): 101–15. doi:<a href="https://doi.org/10.1016/j.neuron.2014.11.018">10.1016/j.neuron.2014.11.018</a>.</p>
</div>
<div id="ref-HongComplementmicrogliamediate2016">
<p>Hong, Soyon, Victoria F. Beja-Glasser, Bianca M. Nfonoyim, Arnaud Frouin, Shaomin Li, Saranya Ramakrishnan, Katherine M. Merry, Qiaoqiao Shi, Arnon Rosenthal, and Ben A. Barres. 2016. “Complement and Microglia Mediate Early Synapse Loss in Alzheimer Mouse Models.” <em>Science</em> 352 (6286): 712–16.</p>
</div>
<div id="ref-TrouwC4bbindingproteinAlzheimer2008">
<p>Trouw, Leendert. A., Henrietta M. Nielsen, Lennart Minthon, Elisabet Londos, Göran Landberg, Robert Veerhuis, Sabina Janciauskiene, and Anna M. Blom. 2008. “C4b-Binding Protein in Alzheimer’s Disease: Binding to A<span class="math inline">\(\beta\)</span>142 and to Dead Cells.” <em>Molecular Immunology</em> 45 (13): 3649–60. doi:<a href="https://doi.org/10.1016/j.molimm.2008.04.025">10.1016/j.molimm.2008.04.025</a>.</p>
</div>
<div id="ref-SharmaPresymptomaticchangemicroRNAs2018">
<p>Sharma, Salil, Ines Khadimallah, Adam Williamson Corya, Yousuf Omar Ali, Xi Rao, Yunlong Liu, and Hui-Chen Lu. 2018. “Presymptomatic Change in microRNAs Modulates Tau Pathology.” <em>Scientific Reports</em> 8 (1): 9251. doi:<a href="https://doi.org/10.1038/s41598-018-27527-6">10.1038/s41598-018-27527-6</a>.</p>
</div>
<div id="ref-StephanDramaticIncreaseC1q2013">
<p>Stephan, Alexander H., Daniel V. Madison, Jos’e María Mateos, Deborah A. Fraser, Emilie A. Lovelett, Laurence Coutellier, Leo Kim, et al. 2013. “A Dramatic Increase of C1q Protein in the CNS During Normal Aging.” <em>The Journal of Neuroscience</em> 33 (33): 13460–74. doi:<a href="https://doi.org/10.1523/JNEUROSCI.1333-13.2013">10.1523/JNEUROSCI.1333-13.2013</a>.</p>
</div>
<div id="ref-LambertMetaanalysis740462013">
<p>Lambert, Jean-Charles, Carla A. Ibrahim-Verbaas, Denise Harold, Adam C. Naj, Rebecca Sims, C’eline Bellenguez, Gyungah Jun, et al. 2013. “Meta-Analysis of 74,046 Individuals Identifies 11 New Susceptibility Loci for Alzheimer’s Disease.” <em>Nature Genetics</em> 45 (12): 1452–8. doi:<a href="https://doi.org/10.1038/ng.2802">10.1038/ng.2802</a>.</p>
</div>
<div id="ref-KucukkilicComplementreceptorgene2018">
<p>Kucukkilic, Ezgi, Keeley Brookes, Imelda Barber, Tamar Guetta-Baranes, ARUK Consortium, Kevin Morgan, and Edward J. Hollox. 2018. “Complement Receptor 1 Gene (Cr1) Intragenic Duplication and Risk of Alzheimer’s Disease.” <em>Human Genetics</em> 137 (4): 305–14. doi:<a href="https://doi.org/10.1007/s00439-018-1883-2">10.1007/s00439-018-1883-2</a>.</p>
</div>
<div id="ref-SekarSchizophreniariskcomplex2016">
<p>Sekar, Aswin, Allison R. Bialas, Heather de Rivera, Avery Davis, Timothy R. Hammond, Nolan Kamitaki, Katherine Tooley, et al. 2016. “Schizophrenia Risk from Complex Variation of Complement Component 4.” <em>Nature</em> 530 (7589): 177–83. doi:<a href="https://doi.org/10.1038/nature16549">10.1038/nature16549</a>.</p>
</div>
<div id="ref-Nimgaonkarcomplementsystemgateway2017">
<p>Nimgaonkar, V. L., K. M. Prasad, K. V. Chowdari, E. G. Severance, and R. H. Yolken. 2017. “The Complement System: A Gateway to GeneEnvironment Interactions in Schizophrenia Pathogenesis.” <em>Molecular Psychiatry</em> 22 (11): 1554–61. doi:<a href="https://doi.org/10.1038/mp.2017.151">10.1038/mp.2017.151</a>.</p>
</div>
<div id="ref-NaitoComplementC1qActivates2012a">
<p>Naito, Atsuhiko T., Tomokazu Sumida, Seitaro Nomura, Mei-Lan Liu, Tomoaki Higo, Akito Nakagawa, Katsuki Okada, et al. 2012. “Complement C1q Activates Canonical Wnt Signaling and Promotes Aging-Related Phenotypes.” <em>Cell</em> 149 (6): 1298–1313. doi:<a href="https://doi.org/10.1016/j.cell.2012.03.047">10.1016/j.cell.2012.03.047</a>.</p>
</div>
<div id="ref-NordenMicrogliaagedbrain2013">
<p>Norden, Diana M., and Jonathan P. Godbout. 2013. “Microglia of the Aged Brain: Primed to Be Activated and Resistant to Regulation.” <em>Neuropathology and Applied Neurobiology</em> 39 (1): 19–34.</p>
</div>
<div id="ref-ClarkeNormalaginginduces2018">
<p>Clarke, Laura E., Shane A. Liddelow, Chandrani Chakraborty, Alexandra E. Münch, Myriam Heiman, and Ben A. Barres. 2018. “Normal Aging Induces A1-Like Astrocyte Reactivity.” <em>Proceedings of the National Academy of Sciences</em> 115 (8): E1896–E1905. doi:<a href="https://doi.org/10.1073/pnas.1800165115">10.1073/pnas.1800165115</a>.</p>
</div>
<div id="ref-CuellarSilencingretrotransposonsSETDB12017">
<p>Cuellar, Trinna L., Anna-Maria Herzner, Xiaotian Zhang, Yogesh Goyal, Colin Watanabe, Brad A. Friedman, Vasantharajan Janakiraman, et al. 2017. “<span class="math inline">\(\-\-\-\)</span>Silencing of Retrotransposons by Setdb1 Inhibits the Interferon Response in Acute Myeloid Leukemia<span class="math inline">\(\-\-\)</span>.” <em>The Journal of Cell Biology</em> 216 (11): 3535–49. doi:<a href="https://doi.org/10.1083/jcb.201612160">10.1083/jcb.201612160</a>.</p>
</div>
<div id="ref-LooDistinctRIGIMDA52008">
<p>Loo, Yueh-Ming, Jamie Fornek, Nanette Crochet, Gagan Bajwa, Olivia Perwitasari, Luis Martinez-Sobrido, Shizuo Akira, et al. 2008. “Distinct RIG-I and Mda5 Signaling by RNA Viruses in Innate Immunity.” <em>Journal of Virology</em> 82 (1): 335–45. doi:<a href="https://doi.org/10.1128/JVI.01080-07">10.1128/JVI.01080-07</a>.</p>
</div>
<div id="ref-DaicytosolicdsRNAsensingpathway2018">
<p>Dai, P., C. Meyer, K. Shaw, Y. Wang, L. Anderson, S. Shuman, T. Tuschl, and L. Deng. 2018. “The Cytosolic dsRNA-Sensing Pathway Mediated by Mda5/MAVS/Irf3 Is Critical for the Induction of Type I and III IFNs After Viral Infection of Skin Keratinocytes.” <em>Journal of Investigative Dermatology</em>, International Investigative Dermatology (IID) 2018 Meeting Abstract Supplement, 138 (5, Supplement): S154. doi:<a href="https://doi.org/10.1016/j.jid.2018.03.916">10.1016/j.jid.2018.03.916</a>.</p>
</div>
<div id="ref-ChiappinelliInhibitingDNAMethylation2015">
<p>Chiappinelli, Katherine B., Pamela L. Strissel, Alexis Desrichard, Huili Li, Christine Henke, Benjamin Akman, Alexander Hein, et al. 2015. “Inhibiting DNA Methylation Causes an Interferon Response in Cancer via dsRNA Including Endogenous Retroviruses.” <em>Cell</em> 162 (5): 974–86. doi:<a href="https://doi.org/10.1016/j.cell.2015.07.011">10.1016/j.cell.2015.07.011</a>.</p>
</div>
<div id="ref-RouloisDNADemethylatingAgentsTarget2015a">
<p>Roulois, David, Helen Loo Yau, Rajat Singhania, Yadong Wang, Arnavaz Danesh, Shu Yi Shen, Han Han, et al. 2015. “DNA-Demethylating Agents Target Colorectal Cancer Cells by Inducing Viral Mimicry by Endogenous Transcripts.” <em>Cell</em> 162 (5): 961–73. doi:<a href="https://doi.org/10.1016/j.cell.2015.07.056">10.1016/j.cell.2015.07.056</a>.</p>
</div>
<div id="ref-KochiSplicingvariantWDFY42018a">
<p>Kochi, Yuta, Yoichiro Kamatani, Yuya Kondo, Akari Suzuki, Eiryo Kawakami, Ryosuke Hiwa, Yukihide Momozawa, Manabu Fujimoto, Masatoshi Jinnin, and Yoshiya Tanaka. 2018. “Splicing Variant of Wdfy4 Augments Mda5 Signalling and the Risk of Clinically Amyopathic Dermatomyositis.” <em>Annals of the Rheumatic Diseases</em>, annrheumdis–2017–212149.</p>
</div>
<div id="ref-RutherfordInterferoninducedbinding1988">
<p>Rutherford, M. N., G. E. Hannigan, and B. R. Williams. 1988. “Interferon-Induced Binding of Nuclear Factors to Promoter Elements of the 2-5a Synthetase Gene.” <em>The EMBO Journal</em> 7 (3): 751–59.</p>
</div>
<div id="ref-LiActivationRNasedependent2016">
<p>Li, Yize, Shuvojit Banerjee, Yuyan Wang, Stephen A. Goldstein, Beihua Dong, Christina Gaughan, Robert H. Silverman, and Susan R. Weiss. 2016. “Activation of RNase L Is Dependent on Oas3 Expression During Infection with Diverse Human Viruses.” <em>Proceedings of the National Academy of Sciences</em> 113 (8): 2241–6. doi:<a href="https://doi.org/10.1073/pnas.1519657113">10.1073/pnas.1519657113</a>.</p>
</div>
<div id="ref-Kristiansenoligoadenylatesynthetasefamily2011">
<p>Kristiansen, Helle, Hans Henrik Gad, Signe Eskildsen-Larsen, Philippe Despres, and Rune Hartmann. 2011. “The Oligoadenylate Synthetase Family: An Ancient Protein Family with Multiple Antiviral Activities.” <em>Journal of Interferon &amp; Cytokine Research</em> 31 (1): 41–47.</p>
</div>
<div id="ref-CrottaTypetypeIII2013">
<p>Crotta, Stefania, Sophia Davidson, Tanel Mahlakoiv, Christophe J. Desmet, Matthew R. Buckwalter, Matthew L. Albert, Peter Staeheli, and Andreas Wack. 2013. “Type I and Type III Interferons Drive Redundant Amplification Loops to Induce a Transcriptional Signature in Influenza-Infected Airway Epithelia.” <em>PLoS Pathogens</em> 9 (11): e1003773.</p>
</div>
<div id="ref-CherwinskiTwotypesmouse1987">
<p>Cherwinski, H. M., J. H. Schumacher, K. D. Brown, and T. R. Mosmann. 1987. “Two Types of Mouse Helper T Cell Clone. III. Further Differences in Lymphokine Synthesis Between Th1 and Th2 Clones Revealed by RNA Hybridization, Functionally Monospecific Bioassays, and Monoclonal Antibodies.” <em>Journal of Experimental Medicine</em> 166 (5): 1229–44. doi:<a href="https://doi.org/10.1084/jem.166.5.1229">10.1084/jem.166.5.1229</a>.</p>
</div>
<div id="ref-BernabeiInterferongreceptorexpression2001">
<p>Bernabei, Paola, Eliana M. Coccia, Laura Rigamonti, Marita Bosticardo, Guido Forni, Sidney Pestka, Christopher D. Krause, Angela Battistini, and Francesco Novelli. 2001. “Interferon-<span class="math inline">\(\gamma\)</span> Receptor 2 Expression as the Deciding Factor in Human T, B, and Myeloid Cell Proliferation or Death.” <em>Journal of Leukocyte Biology</em> 70 (6): 950–60. doi:<a href="https://doi.org/10.1189/jlb.70.6.950">10.1189/jlb.70.6.950</a>.</p>
</div>
<div id="ref-RegisIFNgR2traffickingtunes2006">
<p>Regis, Gabriella, Laura Conti, Daniela Boselli, and Francesco Novelli. 2006. “IFN<span class="math inline">\(\gamma\)</span>R2 Trafficking Tunes IFN<span class="math inline">\(\gamma\)</span>STAT1 Signaling in T Lymphocytes.” <em>Trends in Immunology</em> 27 (2): 96–101. doi:<a href="https://doi.org/10.1016/j.it.2005.12.002">10.1016/j.it.2005.12.002</a>.</p>
</div>
<div id="ref-BachLigandInducedAutoregulationIFNg1995">
<p>Bach, Erika A., Susanne J. Szabo, Anand S. Dighe, Avi Ashkenazi, Michel Aguet, Kenneth M. Murphy, and Robert D. Schreiber. 1995. “Ligand-Induced Autoregulation of IFN-<span class="math inline">\(\gamma\)</span> Receptor <span class="math inline">\(\beta\)</span> Chain Expression in T Helper Cell Subsets.” <em>Science</em> 270 (5239): 1215–8. doi:<a href="https://doi.org/10.1126/science.270.5239.1215">10.1126/science.270.5239.1215</a>.</p>
</div>
<div id="ref-ContiTh17CellsMultiple2012">
<p>Conti, Laura, Raffaele De Palma, Simona Rolla, Daniela Boselli, Gabriella Rodolico, Surinder Kaur, Olli Silvennoinen, et al. 2012. “Th17 Cells in Multiple Sclerosis Express Higher Levels of Jak2, Which Increases Their Surface Expression of IFN-<span class="math inline">\(\gamma\)</span>R2.” <em>The Journal of Immunology</em>, January, 1004013. doi:<a href="https://doi.org/10.4049/jimmunol.1004013">10.4049/jimmunol.1004013</a>.</p>
</div>
<div id="ref-HashiokaInterferongdependentcytotoxicactivation2009">
<p>Hashioka, Sadayuki, Andis Klegeris, Claudia Schwab, and Patrick L. McGeer. 2009. “Interferon-<span class="math inline">\(\gamma\)</span>-Dependent Cytotoxic Activation of Human Astrocytes and Astrocytoma Cells.” <em>Neurobiology of Aging</em> 30 (12): 1924–35. doi:<a href="https://doi.org/10.1016/j.neurobiolaging.2008.02.019">10.1016/j.neurobiolaging.2008.02.019</a>.</p>
</div>
<div id="ref-MitchellIFNgammaupregulatesexpression1996">
<p>Mitchell, T. J., M. Naughton, P. Norsworthy, K. A. Davies, M. J. Walport, and B. J. Morley. 1996. “IFN-Gamma up-Regulates Expression of the Complement Components C3 and C4 by Stabilization of mRNA.” <em>Journal of Immunology (Baltimore, Md.: 1950)</em> 156 (11): 4429–34.</p>
</div>
<div id="ref-ChakrabartyIFNgPromotesComplement2010">
<p>Chakrabarty, Paramita, Carolina Ceballos-Diaz, Amanda Beccard, Christopher Janus, Dennis Dickson, Todd E. Golde, and Pritam Das. 2010. “IFN-<span class="math inline">\(\gamma\)</span> Promotes Complement Expression and Attenuates Amyloid Plaque Deposition in Amyloid <span class="math inline">\(\beta\)</span> Precursor Protein Transgenic Mice.” <em>The Journal of Immunology</em> 184 (9): 5333–43. doi:<a href="https://doi.org/10.4049/jimmunol.0903382">10.4049/jimmunol.0903382</a>.</p>
</div>
<div id="ref-LiIFNgpromotesphosphorylation2015">
<p>Li, Andrew, Carolina Ceballos-Diaz, Nadia DiNunno, Yona Levites, Pedro E. Cruz, Jada Lewis, Todd E. Golde, and Paramita Chakrabarty. 2015. “IFN-<span class="math inline">\(\gamma\)</span> Promotes <span class="math inline">\(\tau\)</span> Phosphorylation Without Affecting Mature Tangles.” <em>The FASEB Journal</em> 29 (10): 4384–98. doi:<a href="https://doi.org/10.1096/fj.15-275834">10.1096/fj.15-275834</a>.</p>
</div>
<div id="ref-ChungGolgiOutpostSynthesis2017">
<p>Chung, Chang Geon, Min Jee Kwon, Keun Hye Jeon, Do Young Hyeon, Myeong Hoon Han, Jeong Hyang Park, In Jun Cha, et al. 2017. “Golgi Outpost Synthesis Impaired by Toxic Polyglutamine Proteins Contributes to Dendritic Pathology in Neurons.” <em>Cell Reports</em> 20 (2): 356–69. doi:<a href="https://doi.org/10.1016/j.celrep.2017.06.059">10.1016/j.celrep.2017.06.059</a>.</p>
</div>
<div id="ref-Smithnewparadigminnate2014">
<p>Smith, Judith A. 2014. “A New Paradigm: Innate Immune Sensing of Viruses via the Unfolded Protein Response.” <em>Frontiers in Microbiology</em> 5. doi:<a href="https://doi.org/10.3389/fmicb.2014.00222">10.3389/fmicb.2014.00222</a>.</p>
</div>
<div id="ref-MartinonTLRactivationtranscription2010">
<p>Martinon, Fabio, Xi Chen, Ann-Hwee Lee, and Laurie H Glimcher. 2010. “TLR Activation of the Transcription Factor Xbp1 Regulates Innate Immune Responses in Macrophages.” <em>Nature Immunology</em> 11 (March): 411.</p>
</div>
<div id="ref-ClavarinoInductionGADD34Necessary2012">
<p>Clavarino, Giovanna, Nuno Cl’audio, Th’er‘ese Couderc, Alexandre Dalet, Delphine Judith, Voahirana Camosseto, Enrico K. Schmidt, et al. 17AD–2012. “Induction of Gadd34 Is Necessary for dsRNA-Dependent Interferon-<span class="math inline">\(\beta\)</span> Production and Participates in the Control of Chikungunya Virus Infection.” <em>PLOS Pathogens</em> 8 (5): e1002708. doi:<a href="https://doi.org/10.1371/journal.ppat.1002708">10.1371/journal.ppat.1002708</a>.</p>
</div>
<div id="ref-PagerModulationhepatitisvirus2013">
<p>Pager, Cara T., Sylvia Schütz, Teresa M. Abraham, Guangxiang Luo, and Peter Sarnow. 2013. “Modulation of Hepatitis C Virus RNA Abundance and Virus Release by Dispersion of Processing Bodies and Enrichment of Stress Granules.” <em>Virology</em> 435 (2): 472–84. doi:<a href="https://doi.org/10.1016/j.virol.2012.10.027">10.1016/j.virol.2012.10.027</a>.</p>
</div>
<div id="ref-MastroeniLasercapturedmicrogliaAlzheimer2018">
<p>Mastroeni, Diego, Jennifer Nolz, Shobana Sekar, Elaine Delvaux, Geidy Serrano, Lori Cuyugan, Winnie S. Liang, Thomas G. Beach, Joseph Rogers, and Paul D. Coleman. 2018. “Laser-Captured Microglia in the Alzheimer’s and Parkinson’s Brain Reveal Unique Regional Expression Profiles and Suggest a Potential Role for Hepatitis B in the Alzheimer’s Brain.” <em>Neurobiology of Aging</em> 63 (March): 12–21. doi:<a href="https://doi.org/10.1016/j.neurobiolaging.2017.10.019">10.1016/j.neurobiolaging.2017.10.019</a>.</p>
</div>
<div id="ref-MedigeshiWestNilevirus2007">
<p>Medigeshi, Guruprasad R., Alissa M. Lancaster, Alec J. Hirsch, Thomas Briese, W. Ian Lipkin, Victor DeFilippis, Klaus Früh, Peter W. Mason, Janko Nikolich-Zugich, and Jay A. Nelson. 2007. “West Nile Virus Infection Activates the Unfolded Protein Response, Leading to CHOP Induction and Apoptosis.” <em>Journal of Virology</em> 81 (20): 10849–60.</p>
</div>
<div id="ref-UmareddyDenguevirusserotype2007">
<p>Umareddy, Indira, Olivier Pluquet, Qing Yin Wang, Subhash G. Vasudevan, Eric Chevet, and Feng Gu. 2007. “Dengue Virus Serotype Infection Specifies the Activation of the Unfolded Protein Response.” <em>Virology Journal</em> 4 (1): 91.</p>
</div>
<div id="ref-Cervantes-OrtizRespiratorySyncytialVirus2016">
<p>Cervantes-Ortiz, Sandra L., Natalia Zamorano Cuervo, and Nathalie Grandvaux. 2016. “Respiratory Syncytial Virus and Cellular Stress Responses: Impact on Replication and Physiopathology.” <em>Viruses</em> 8 (5): 124. doi:<a href="https://doi.org/10.3390/v8050124">10.3390/v8050124</a>.</p>
</div>
<div id="ref-Gladwyn-NgStressinducedunfoldedprotein2018">
<p>Gladwyn-Ng, Ivan, Lluís Cord’on-Barris, Christian Alfano, Catherine Creppe, Therese Couderc, Giovanni Morelli, Nicolas Thelen, Michelle America, Bettina Bessi‘eres, and F’erecht’e Encha-Razavi. 2018. “Stress-Induced Unfolded Protein Response Contributes to Zika VirusAssociated Microcephaly.” <em>Nature Neuroscience</em> 21 (1): 63.</p>
</div>
<div id="ref-KeActivationunfoldedprotein2011">
<p>Ke, Po-Yuan, and Steve S.-L. Chen. 2011. “Activation of the Unfolded Protein Response and Autophagy After Hepatitis C Virus Infection Suppresses Innate Antiviral Immunity in Vitro.” <em>The Journal of Clinical Investigation</em> 121 (1): 37–56. doi:<a href="https://doi.org/10.1172/JCI41474">10.1172/JCI41474</a>.</p>
</div>
<div id="ref-TardifHepatitisvirussuppresses2004">
<p>Tardif, Keith D., Kazutoshi Mori, Randal J. Kaufman, and Aleem Siddiqui. 2004. “Hepatitis C Virus Suppresses the Ire1-Xbp1 Pathway of the Unfolded Protein Response.” <em>Journal of Biological Chemistry</em>.</p>
</div>
<div id="ref-FitzpatrickUTRlocatedinvertedAlu2012">
<p>Fitzpatrick, Terry, and Sui Huang. 2012. “3’-UTR-Located Inverted Alu Repeats Facilitate mRNA Translational Repression and Stress Granule Accumulation.” <em>Nucleus</em> 3 (4): 359–69. doi:<a href="https://doi.org/10.4161/nucl.20827">10.4161/nucl.20827</a>.</p>
</div>
<div id="ref-GoodierLINE1ORF1Protein2007">
<p>Goodier, John L., Lili Zhang, Melissa R. Vetter, and Haig H. Kazazian. 2007. “LINE-1 Orf1 Protein Localizes in Stress Granules with Other RNA-Binding Proteins, Including Components of RNA Interference RNA-Induced Silencing Complex.” <em>Molecular and Cellular Biology</em> 27 (18): 6469–83. doi:<a href="https://doi.org/10.1128/MCB.00332-07">10.1128/MCB.00332-07</a>.</p>
</div>
<div id="ref-PasquarellaRetrotransposonderepressionleads2016a">
<p>Pasquarella, Alessandra, Anja Ebert, Gustavo Pereira de Almeida, Maria Hinterberger, Maryam Kazerani, Alexander Nuber, Joachim Ellwart, Ludger Klein, Meinrad Busslinger, and Gunnar Schotta. 2016. “Retrotransposon Derepression Leads to Activation of the Unfolded Protein Response and Apoptosis in Pro-B Cells.” <em>Development</em> 143 (10): 1788–99. doi:<a href="https://doi.org/10.1242/dev.130203">10.1242/dev.130203</a>.</p>
</div>
<div id="ref-Ben-AsouliHumaninterferongmRNA2002">
<p>Ben-Asouli, Yitzhak, Yona Banai, Yehuda Pel-Or, Alexei Shir, and Raymond Kaempfer. 2002. “Human Interferon-<span class="math inline">\(\gamma\)</span> mRNA Autoregulates Its Translation Through a Pseudoknot That Activates the Interferon-Inducible Protein Kinase PKR.” <em>Cell</em> 108 (2): 221–32.</p>
</div>
<div id="ref-LemaireMechanismPKRActivation2008">
<p>Lemaire, Peter A., Eric Anderson, Jeffrey Lary, and James L. Cole. 2008. “Mechanism of PKR Activation by dsRNA.” <em>Journal of Molecular Biology</em> 381 (2): 351–60. doi:<a href="https://doi.org/10.1016/j.jmb.2008.05.056">10.1016/j.jmb.2008.05.056</a>.</p>
</div>
<div id="ref-WilliamsPKRsentinelkinase1999">
<p>Williams, Bryan RG. 1999. “PKR; a Sentinel Kinase for Cellular Stress.” <em>Oncogene</em> 18 (45): 6112–20. doi:<a href="https://doi.org/10.1038/sj.onc.1203127">10.1038/sj.onc.1203127</a>.</p>
</div>
<div id="ref-RathInductiondsRNAactivatedprotein2012">
<p>Rath, Eva, Emanuel Berger, Anja Messlik, Tiago Nunes, Bo Liu, Sandy C. Kim, Nick Hoogenraad, Miquel Sans, R. Balfour Sartor, and Dirk Haller. 2012. “Induction of dsRNA-Activated Protein Kinase Links Mitochondrial Unfolded Protein Response to the Pathogenesis of Intestinal Inflammation.” <em>Gut</em> 61 (9): 1269–78.</p>
</div>
<div id="ref-RojasProteinKinaseResponsible2010">
<p>Rojas, Margarito, Carlos F. Arias, and Susana L’opez. 2010. “Protein Kinase R Is Responsible for the Phosphorylation of eIF2<span class="math inline">\(\alpha\)</span> in Rotavirus Infection.” <em>Journal of Virology</em> 84 (20): 10457–66. doi:<a href="https://doi.org/10.1128/JVI.00625-10">10.1128/JVI.00625-10</a>.</p>
</div>
<div id="ref-VattemReinitiationinvolvingupstream2004">
<p>Vattem, Krishna M., and Ronald C. Wek. 2004. “Reinitiation Involving Upstream ORFs Regulates Atf4 mRNA Translation in Mammalian Cells.” <em>Proceedings of the National Academy of Sciences of the United States of America</em> 101 (31): 11269–74. doi:<a href="https://doi.org/10.1073/pnas.0400541101">10.1073/pnas.0400541101</a>.</p>
</div>
<div id="ref-SharmaNeuronatinmediatedaberrantcalcium2013">
<p>Sharma, Jaiprakash, Diptendu Mukherjee, Sudheendra NR Rao, Soumya Iyengar, Susarla Krishna Shankar, Parthasarathy Satishchandra, and Nihar Ranjan Jana. 2013. “Neuronatin-Mediated Aberrant Calcium Signaling and Endoplasmic Reticulum Stress Underlie Neuropathology in Lafora Disease.” <em>Journal of Biological Chemistry</em> 288 (13): 9482–90.</p>
</div>
<div id="ref-Vasekcomplementmicroglialaxis2016">
<p>Vasek, Michael J., Charise Garber, Denise Dorsey, Douglas M. Durrant, Bryan Bollman, Allison Soung, Jinsheng Yu, et al. 2016. “A ComplementMicroglial Axis Drives Synapse Loss During Virus-Induced Memory Impairment.” <em>Nature</em> 534 (7608): 538–43. doi:<a href="https://doi.org/10.1038/nature18283">10.1038/nature18283</a>.</p>
</div>
<div id="ref-JamiesonLatentherpessimplex1991">
<p>Jamieson, Gordon A., Norman J. Maitland, Gordon K. Wilcock, John Craske, and Ruth F. Itzhaki. 1991. “Latent Herpes Simplex Virus Type 1 in Normal and Alzheimer’s Disease Brains.” <em>Journal of Medical Virology</em> 33 (4): 224–27. doi:<a href="https://doi.org/10.1002/jmv.1890330403">10.1002/jmv.1890330403</a>.</p>
</div>
<div id="ref-ItzhakiMicrobesAlzheimerDisease2016">
<p>Itzhaki, Ruth F., Richard Lathe, Brian J. Balin, Melvyn J. Ball, Elaine L. Bearer, Heiko Braak, Maria J. Bullido, et al. 2016. “Microbes and Alzheimer’s Disease.” <em>Journal of Alzheimer’s Disease</em> 51 (4): 979–84. doi:<a href="https://doi.org/10.3233/JAD-160152">10.3233/JAD-160152</a>.</p>
</div>
<div id="ref-WozniakHerpessimplexvirus2009">
<p>Wozniak, M. A., A. P. Mee, and R. F. Itzhaki. 2009. “Herpes Simplex Virus Type 1 DNA Is Located Within Alzheimer’s Disease Amyloid Plaques.” <em>The Journal of Pathology: A Journal of the Pathological Society of Great Britain and Ireland</em> 217 (1): 131–38.</p>
</div>
<div id="ref-EimerAlzheimerDiseaseAssociatedvAmyloid2018">
<p>Eimer, William A., Deepak Kumar Vijaya Kumar, Nanda Kumar Navalpur Shanmugam, Alex S. Rodriguez, Teryn Mitchell, Kevin J. Washicosky, Bence György, Xandra O. Breakefield, Rudolph E. Tanzi, and Robert D. Moir. 2018. “Alzheimer’s Disease-Associated <span class="math inline">\(\beta\)</span>-Amyloid Is Rapidly Seeded by Herpesviridae to Protect Against Brain Infection.” <em>Neuron</em> 99 (1): 56–63.e3. doi:<a href="https://doi.org/10.1016/j.neuron.2018.06.030">10.1016/j.neuron.2018.06.030</a>.</p>
</div>
<div id="ref-ReadheadMultiscaleAnalysisIndependent2018">
<p>Readhead, Ben, Jean-Vianney Haure-Mirande, Cory C. Funk, Matthew A. Richards, Paul Shannon, Vahram Haroutunian, Mary Sano, et al. 2018. “Multiscale Analysis of Independent Alzheimer’s Cohorts Finds Disruption of Molecular, Genetic, and Clinical Networks by Human Herpesvirus.” <em>Neuron</em> 99 (1): 64–82.e7. doi:<a href="https://doi.org/10.1016/j.neuron.2018.05.023">10.1016/j.neuron.2018.05.023</a>.</p>
</div>
<div id="ref-TzengAntiherpeticMedicationsReduced2018">
<p>Tzeng, Nian-Sheng, Chi-Hsiang Chung, Fu-Huang Lin, Chien-Ping Chiang, Chin-Bin Yeh, San-Yuan Huang, Ru-Band Lu, et al. 2018. “Anti-Herpetic Medications and Reduced Risk of Dementia in Patients with Herpes Simplex Virus Infectionsa Nationwide, Population-Based Cohort Study in Taiwan.” <em>Neurotherapeutics</em> 15 (2): 417–29. doi:<a href="https://doi.org/10.1007/s13311-018-0611-x">10.1007/s13311-018-0611-x</a>.</p>
</div>
<div id="ref-BakerInductionA4amyloid1994">
<p>Baker, H. F., R. M. Ridley, L. W. Duchen, T. J. Crow, and C. J. Bruton. 1994. “Induction of <span class="math inline">\(\beta\)</span> (A4)-Amyloid in Primates by Injection of Alzheimer’s Disease Brain Homogenate.” <em>Molecular Neurobiology</em> 8 (1): 25–39.</p>
</div>
<div id="ref-RidleyVerylongterm2006">
<p>Ridley, R. M., H. F. Baker, C. P. Windle, and R. M. Cummings. 2006. “Very Long Term Studies of the Seeding of <span class="math inline">\(\beta\)</span>-Amyloidosis in Primates.” <em>Journal of Neural Transmission</em> 113 (9): 1243.</p>
</div>
<div id="ref-NellakerTransactivationelementshuman2006">
<p>Nellåker, Christoffer, Yuanrong Yao, Lorraine Jones-Brando, François Mallet, Robert H. Yolken, and H<span class="math inline">\(\backslash\)</span>a akan Karlsson. 2006. “Transactivation of Elements in the Human Endogenous Retrovirus W Family by Viral Infection.” <em>Retrovirology</em> 3 (1): 1.</p>
</div>
<div id="ref-BrudekSynergisticimmuneresponses2008">
<p>Brudek, T., T. Christensen, H. J. Hansen, T. Petersen, and A. Møller-Larsen. 2008. “Synergistic Immune Responses Induced by Endogenous Retrovirus and Herpesvirus Antigens Result in Increased Production of Inflammatory Cytokines in Multiple Sclerosis Patients.” <em>Scandinavian Journal of Immunology</em> 67 (3): 295–303. doi:<a href="https://doi.org/10.1111/j.1365-3083.2007.02067.x">10.1111/j.1365-3083.2007.02067.x</a>.</p>
</div>
<div id="ref-MameliActivationMSRVTypeEndogenous2013a">
<p>Mameli, Giuseppe, Giordano Madeddu, Alessandra Mei, Elena Uleri, Luciana Poddighe, Lucia G. Delogu, Ivana Maida, et al. 2013. “Activation of MSRV-Type Endogenous Retroviruses During Infectious Mononucleosis and Epstein-Barr Virus Latency: The Missing Link with Multiple Sclerosis?” <em>PLOS ONE</em> 8 (11): e78474. doi:<a href="https://doi.org/10.1371/journal.pone.0078474">10.1371/journal.pone.0078474</a>.</p>
</div>
<div id="ref-MameliExpressionactivationEpstein2012">
<p>Mameli, Giuseppe, Luciana Poddighe, Alessandra Mei, Elena Uleri, Stefano Sotgiu, Caterina Serra, Roberto Manetti, and Antonina Dolei. 2012. “Expression and Activation by Epstein Barr Virus of Human Endogenous Retroviruses-W in Blood Cells and Astrocytes: Inference for Multiple Sclerosis.” <em>PloS One</em> 7 (9): e44991. doi:<a href="https://doi.org/10.1371/journal.pone.0044991">10.1371/journal.pone.0044991</a>.</p>
</div>
<div id="ref-LiChloroquinetriggersEpsteinBarr2017a">
<p>Li, Xiaofan, Eric M. Burton, and Sumita Bhaduri-McIntosh. 2017. “Chloroquine Triggers Epstein-Barr Virus Replication Through Phosphorylation of Kap1/Trim28 in Burkitt Lymphoma Cells.” <em>PLOS Pathogens</em> 13 (3): e1006249. doi:<a href="https://doi.org/10.1371/journal.ppat.1006249">10.1371/journal.ppat.1006249</a>.</p>
</div>
<div id="ref-DesplatsaSynucleinSequestersDnmt12011">
<p>Desplats, Paula, Brian Spencer, Elizabeth Coffee, Pruthul Patel, Sarah Michael, Christina Patrick, Anthony Adame, Edward Rockenstein, and Eliezer Masliah. 2011. “<span class="math inline">\(\alpha\)</span>-Synuclein Sequesters Dnmt1 from the Nucleus: A NOVEL MECHANISM FOR EPIGENETIC ALTERATIONS IN LEWY BODY DISEASES.” <em>Journal of Biological Chemistry</em> 286 (11): 9031–7. doi:<a href="https://doi.org/10.1074/jbc.C110.212589">10.1074/jbc.C110.212589</a>.</p>
</div>
<div id="ref-MadabhushiActivityInducedDNABreaks2015">
<p>Madabhushi, Ram, Fan Gao, Andreas R. Pfenning, Ling Pan, Satoko Yamakawa, Jinsoo Seo, Richard Rueda, et al. 2015. “Activity-Induced DNA Breaks Govern the Expression of Neuronal Early-Response Genes.” <em>Cell</em> 161 (7): 1592–1605. doi:<a href="https://doi.org/10.1016/j.cell.2015.05.032">10.1016/j.cell.2015.05.032</a>.</p>
</div>
<div id="ref-MadabhushiDNADamageIts2014">
<p>Madabhushi, Ram, Ling Pan, and Li-Huei Tsai. 2014. “DNA Damage and Its Links to Neurodegeneration.” <em>Neuron</em> 83 (2): 266–82. doi:<a href="https://doi.org/10.1016/j.neuron.2014.06.034">10.1016/j.neuron.2014.06.034</a>.</p>
</div>
<div id="ref-NohATRXtoleratesactivitydependent2014">
<p>Noh, Kyung-Min, Ian Maze, Dan Zhao, Bin Xiang, Wendy Wenderski, Peter W. Lewis, Li Shen, Haitao Li, and C. David Allis. 2014. “ATRX Tolerates Activity-Dependent Histone H3 Methyl/Phos Switching to Maintain Repetitive Element Silencing in Neurons.” <em>Proceedings of the National Academy of Sciences</em>, 201411258. <a href="http://www.pnas.org/content/early/2014/12/22/1411258112.short" class="uri">http://www.pnas.org/content/early/2014/12/22/1411258112.short</a>.</p>
</div>
<div id="ref-Szulwach5hmCmediatedepigenetic2011">
<p>Szulwach, Keith E, Xuekun Li, Yujing Li, Chun-Xiao Song, Hao Wu, Qing Dai, Hasan Irier, et al. 2011. “5-hmC–Mediated Epigenetic Dynamics During Postnatal Neurodevelopment and Aging.” <em>Nature Neuroscience</em> 14 (12): 1607–16. doi:<a href="https://doi.org/10.1038/nn.2959">10.1038/nn.2959</a>.</p>
</div>
<div id="ref-NewmanActivityDEPendentTransposition2017">
<p>Newman, Andrew G, Paraskevi Bessa, Victor Tarabykin, and Prim B Singh. 2017. “Activity‐DEPendent Transposition.” <em>EMBO Reports</em>, February, e201643797. doi:<a href="https://doi.org/10.15252/embr.201643797">10.15252/embr.201643797</a>.</p>
</div>
</div>
<div class="footnotes">
<hr />
<ol start="1">
<li id="fn1"><p>pLI is a probablity calculation of how well loss of function mutations can be tolerated, where 1.00 means the loss of function of the protein cannot be tolerated. HP1<span class="math inline">\(\beta\)</span> holds a pLI of 0.95 while HP1<span class="math inline">\(\alpha\)</span> a lower pLI of 0.84, and HP1<span class="math inline">\(\gamma\)</span> a pLI of 0.63. <a href="http://exac.broadinstitute.org/" class="uri">http://exac.broadinstitute.org/</a><a href="3-discussion.html#fnref1">↩</a></p></li>
</ol>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="2-results.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="4-methods.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"download": null,
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
