<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Part 4 Methods | HP1 Deficiency Results in Accelerated Neurodegeneration</title>
  <meta name="description" content="Part 4 Methods | HP1 Deficiency Results in Accelerated Neurodegeneration">
  <meta name="generator" content="bookdown  and GitBook 2.6.7">

  <meta property="og:title" content="Part 4 Methods | HP1 Deficiency Results in Accelerated Neurodegeneration" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Part 4 Methods | HP1 Deficiency Results in Accelerated Neurodegeneration" />
  
  
  

<meta name="author" content="Andrew G. Newman">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="3-discussion.html">
<link rel="next" href="5-supplemental-data.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />







<script src="libs/kePrint-0.0.1/kePrint.js"></script>


<style type="text/css">
div.sourceCode { overflow-x: auto; }
table.sourceCode, tr.sourceCode, td.lineNumbers, td.sourceCode {
  margin: 0; padding: 0; vertical-align: baseline; border: none; }
table.sourceCode { width: 100%; line-height: 100%; }
td.lineNumbers { text-align: right; padding-right: 4px; padding-left: 4px; color: #aaaaaa; border-right: 1px solid #aaaaaa; }
td.sourceCode { padding-left: 5px; }
code > span.kw { color: #007020; font-weight: bold; } /* Keyword */
code > span.dt { color: #902000; } /* DataType */
code > span.dv { color: #40a070; } /* DecVal */
code > span.bn { color: #40a070; } /* BaseN */
code > span.fl { color: #40a070; } /* Float */
code > span.ch { color: #4070a0; } /* Char */
code > span.st { color: #4070a0; } /* String */
code > span.co { color: #60a0b0; font-style: italic; } /* Comment */
code > span.ot { color: #007020; } /* Other */
code > span.al { color: #ff0000; font-weight: bold; } /* Alert */
code > span.fu { color: #06287e; } /* Function */
code > span.er { color: #ff0000; font-weight: bold; } /* Error */
code > span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
code > span.cn { color: #880000; } /* Constant */
code > span.sc { color: #4070a0; } /* SpecialChar */
code > span.vs { color: #4070a0; } /* VerbatimString */
code > span.ss { color: #bb6688; } /* SpecialString */
code > span.im { } /* Import */
code > span.va { color: #19177c; } /* Variable */
code > span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code > span.op { color: #666666; } /* Operator */
code > span.bu { } /* BuiltIn */
code > span.ex { } /* Extension */
code > span.pp { color: #bc7a00; } /* Preprocessor */
code > span.at { color: #7d9029; } /* Attribute */
code > span.do { color: #ba2121; font-style: italic; } /* Documentation */
code > span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code > span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code > span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
</style>

</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="preface.html"><a href="preface.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="" data-path="contribution-of-authors.html"><a href="contribution-of-authors.html"><i class="fa fa-check"></i>Contribution of Authors</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="1" data-path="1-introduction.html"><a href="1-introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="1-introduction.html"><a href="1-introduction.html#epigenetics"><i class="fa fa-check"></i><b>1.1</b> Epigenetics</a><ul>
<li><a href="1-introduction.html#dna-modifications"><em>DNA Modifications</em></a></li>
<li><a href="1-introduction.html#chromatin"><em>Chromatin</em></a></li>
<li><a href="1-introduction.html#chromatin-remodelling"><em>Chromatin Remodelling</em></a></li>
<li><a href="1-introduction.html#krab-zfps-transposable-elements-and-exaptation"><em>KRAB-ZFPs, Transposable Elements and Exaptation</em></a></li>
<li><a href="1-introduction.html#heterochromatinization-and-hp1"><em>Heterochromatinization and HP1</em></a></li>
<li><a href="1-introduction.html#non-coding-rna"><em>Non-Coding RNA</em></a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="1-introduction.html"><a href="1-introduction.html#the-heterochromatin-loss-model-of-aging"><i class="fa fa-check"></i><b>1.2</b> The Heterochromatin Loss Model of Aging</a></li>
<li class="chapter" data-level="1.3" data-path="1-introduction.html"><a href="1-introduction.html#the-aging-neuronal-epigenome"><i class="fa fa-check"></i><b>1.3</b> The Aging Neuronal Epigenome</a><ul>
<li class="chapter" data-level="1.3.1" data-path="1-introduction.html"><a href="1-introduction.html#DNAging"><i class="fa fa-check"></i><b>1.3.1</b> <em>DNA Methylation and Hydroxymethylation in Aged Neurons</em></a></li>
<li class="chapter" data-level="1.3.2" data-path="1-introduction.html"><a href="1-introduction.html#chraging"><i class="fa fa-check"></i><b>1.3.2</b> <em>Chromatin in the Aging Brain</em></a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="1-introduction.html"><a href="1-introduction.html#objective"><i class="fa fa-check"></i><b>1.4</b> Objective</a></li>
<li class="chapter" data-level="1.5" data-path="1-introduction.html"><a href="1-introduction.html#hp1-proteins---a-background"><i class="fa fa-check"></i><b>1.5</b> HP1 Proteins - A Background</a><ul>
<li><a href="1-introduction.html#the-abgs-of-hp1-proteins">The <em>ABG’s</em> of HP1 Proteins</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-results.html"><a href="2-results.html"><i class="fa fa-check"></i><b>2</b> Results</a><ul>
<li class="chapter" data-level="2.1" data-path="2-results.html"><a href="2-results.html#HP1aT"><i class="fa fa-check"></i><b>2.1</b> HP1FEC, HP1TKO and HP1<span class="math inline">\(\alpha\)</span>T</a></li>
<li class="chapter" data-level="2.2" data-path="2-results.html"><a href="2-results.html#expression-of-hp1-proteins-in-the-brain"><i class="fa fa-check"></i><b>2.2</b> Expression of HP1 Proteins in the Brain</a></li>
<li class="chapter" data-level="2.3" data-path="2-results.html"><a href="2-results.html#mosaic-deletion-of-hp1beta-and-hp1gamma-from-cortical-progenitors"><i class="fa fa-check"></i><b>2.3</b> Mosaic Deletion of HP1<span class="math inline">\(\beta\)</span> and HP1<span class="math inline">\(\gamma\)</span> from Cortical Progenitors</a></li>
<li class="chapter" data-level="2.4" data-path="2-results.html"><a href="2-results.html#deletion-of-hp1beta-hp1gamma-from-the-emx1-lineage."><i class="fa fa-check"></i><b>2.4</b> Deletion of <em>HP1<span class="math inline">\(\beta\)</span></em> &amp; <em>HP1<span class="math inline">\(\gamma\)</span></em> from the <em>Emx1</em>-lineage.</a><ul>
<li class="chapter" data-level="2.4.1" data-path="2-results.html"><a href="2-results.html#cell-fate-in-cortical-neurons"><i class="fa fa-check"></i><b>2.4.1</b> Cell Fate in Cortical Neurons</a></li>
<li class="chapter" data-level="2.4.2" data-path="2-results.html"><a href="2-results.html#imprint"><i class="fa fa-check"></i><b>2.4.2</b> Imprinted Loci</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="2-results.html"><a href="2-results.html#perturbed-regulation-of-retroelements-in-hp1-mutants"><i class="fa fa-check"></i><b>2.5</b> Perturbed Regulation of Retroelements in HP1 Mutants</a><ul>
<li class="chapter" data-level="2.5.1" data-path="2-results.html"><a href="2-results.html#line1"><i class="fa fa-check"></i><b>2.5.1</b> LINE1</a></li>
<li class="chapter" data-level="2.5.2" data-path="2-results.html"><a href="2-results.html#class-iii-ervs"><i class="fa fa-check"></i><b>2.5.2</b> Class III ERVs</a></li>
<li class="chapter" data-level="2.5.3" data-path="2-results.html"><a href="2-results.html#class-ii-ervs"><i class="fa fa-check"></i><b>2.5.3</b> Class II ERVs</a></li>
<li class="chapter" data-level="2.5.4" data-path="2-results.html"><a href="2-results.html#de-novo-retrotransposition"><i class="fa fa-check"></i><b>2.5.4</b> <em>De Novo</em> Retrotransposition</a></li>
</ul></li>
<li class="chapter" data-level="2.6" data-path="2-results.html"><a href="2-results.html#sustained-hp1betagamma-deficiency-adversely-affects-neuronal-complexity-function-and-cognition"><i class="fa fa-check"></i><b>2.6</b> Sustained HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> Deficiency Adversely Affects Neuronal Complexity, Function and Cognition</a><ul>
<li class="chapter" data-level="2.6.1" data-path="2-results.html"><a href="2-results.html#behavior"><i class="fa fa-check"></i><b>2.6.1</b> Behavior</a></li>
<li class="chapter" data-level="2.6.2" data-path="2-results.html"><a href="2-results.html#MRIresults"><i class="fa fa-check"></i><b>2.6.2</b> Structure, Volumetrics and Histology</a></li>
</ul></li>
<li class="chapter" data-level="2.7" data-path="2-results.html"><a href="2-results.html#hp1beta-and-hp1gamma-synergistically-contribute-to-repeat-silencing"><i class="fa fa-check"></i><b>2.7</b> HP1<span class="math inline">\(\beta\)</span> and HP1<span class="math inline">\(\gamma\)</span> Synergistically Contribute to Repeat Silencing</a><ul>
<li class="chapter" data-level="2.7.1" data-path="2-results.html"><a href="2-results.html#hp1gamma-is-required-for-h4k20me3-deposition-in-neurons"><i class="fa fa-check"></i><b>2.7.1</b> HP1<span class="math inline">\(\gamma\)</span> is Required for H4K20me3 Deposition in Neurons</a></li>
<li class="chapter" data-level="2.7.2" data-path="2-results.html"><a href="2-results.html#hp1gamma-binds-suv420h2-via-the-chromoshadow-domain"><i class="fa fa-check"></i><b>2.7.2</b> HP1<span class="math inline">\(\gamma\)</span> binds SUV420H2 via the Chromoshadow Domain</a></li>
<li class="chapter" data-level="2.7.3" data-path="2-results.html"><a href="2-results.html#re-addition-of-hp1gamma-can-restore-h4k20me3-in-vivo"><i class="fa fa-check"></i><b>2.7.3</b> Re-addition of HP1<span class="math inline">\(\gamma\)</span> can restore H4K20me3 In Vivo</a></li>
<li class="chapter" data-level="2.7.4" data-path="2-results.html"><a href="2-results.html#h4k20me3-loss-is-correlated-with-iap-de-repression"><i class="fa fa-check"></i><b>2.7.4</b> H4K20me3 Loss is Correlated with IAP De-repression</a></li>
<li class="chapter" data-level="2.7.5" data-path="2-results.html"><a href="2-results.html#kap1-h3k9me3-and-h4k20me3-are-lost-at-iap-elements-in-hp1ctko-es-cells"><i class="fa fa-check"></i><b>2.7.5</b> KAP1, H3K9me3 and H4K20me3 are lost at IAP Elements in HP1cTKO ES Cells</a></li>
<li class="chapter" data-level="2.7.6" data-path="2-results.html"><a href="2-results.html#dna-methylation-is-affected-following-loss-of-hp1"><i class="fa fa-check"></i><b>2.7.6</b> DNA Methylation is affected Following loss of HP1</a></li>
<li><a href="2-results.html#kap1-is-lost-from-icrs-in-hp1ctko-cells"><em>KAP1 is lost from ICRs in HP1cTKO Cells</em></a></li>
</ul></li>
<li class="chapter" data-level="2.8" data-path="2-results.html"><a href="2-results.html#RNAresults"><i class="fa fa-check"></i><b>2.8</b> Transcriptional Dysregulation following HP1 Deficiency Drives UPR and Inflammatory Responses</a><ul>
<li class="chapter" data-level="2.8.1" data-path="2-results.html"><a href="2-results.html#genotype-related-changes"><i class="fa fa-check"></i><b>2.8.1</b> Genotype Related Changes</a></li>
<li class="chapter" data-level="2.8.2" data-path="2-results.html"><a href="2-results.html#longitudinal-changes-to-gene-expression"><i class="fa fa-check"></i><b>2.8.2</b> Longitudinal Changes To Gene Expression</a></li>
<li class="chapter" data-level="2.8.3" data-path="2-results.html"><a href="2-results.html#pathway-analysis"><i class="fa fa-check"></i><b>2.8.3</b> Pathway Analysis</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-discussion.html"><a href="3-discussion.html"><i class="fa fa-check"></i><b>3</b> Discussion</a><ul>
<li class="chapter" data-level="3.1" data-path="3-discussion.html"><a href="3-discussion.html#phenotypes"><i class="fa fa-check"></i><b>3.1</b> Phenotypes</a><ul>
<li><a href="3-discussion.html#structural-changes-in-the-hippocampus"><em>Structural Changes in the Hippocampus</em></a></li>
<li><a href="3-discussion.html#reduction-in-cortical-volume"><em>Reduction in Cortical Volume</em></a></li>
<li><a href="3-discussion.html#behavior-and-impairments-in-learning-memory"><em>Behavior and Impairments in Learning &amp; Memory</em></a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="3-discussion.html"><a href="3-discussion.html#hp1-dependent-epigenetic-systems"><i class="fa fa-check"></i><b>3.2</b> HP1-dependent Epigenetic Systems</a><ul>
<li><a href="3-discussion.html#hp1gamma"><em>HP1<span class="math inline">\(\gamma\)</span></em></a></li>
<li><a href="3-discussion.html#hp1beta"><em>HP1<span class="math inline">\(\beta\)</span></em></a></li>
<li><a href="3-discussion.html#hp1beta-x-hp1gamma"><em>HP1<span class="math inline">\(\beta\)</span></em> x <em>HP1<span class="math inline">\(\gamma\)</span></em></a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="3-discussion.html"><a href="3-discussion.html#concordance-with-aging-and-disease"><i class="fa fa-check"></i><b>3.3</b> Concordance with Aging and Disease</a><ul>
<li><a href="3-discussion.html#complement-in-cns-aging-and-disease"><em>Complement in CNS Aging and Disease</em></a></li>
<li><a href="3-discussion.html#a-pathway-to-complement"><em>A Pathway to Complement</em></a></li>
<li><a href="3-discussion.html#unfolded-protein-response-upr"><em>Unfolded Protein Response (UPR)</em></a></li>
<li><a href="3-discussion.html#viruses-transactivation-and-neurodegeneration"><em>Viruses, Transactivation and Neurodegeneration</em></a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="3-discussion.html"><a href="3-discussion.html#concluding-remarks"><i class="fa fa-check"></i><b>3.4</b> Concluding Remarks</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-methods.html"><a href="4-methods.html"><i class="fa fa-check"></i><b>4</b> Methods</a><ul>
<li class="chapter" data-level="4.1" data-path="4-methods.html"><a href="4-methods.html#mouse-lines"><i class="fa fa-check"></i><b>4.1</b> Mouse Lines</a><ul>
<li class="chapter" data-level="4.1.1" data-path="4-methods.html"><a href="4-methods.html#HP1anull"><i class="fa fa-check"></i><b>4.1.1</b> Generation of HP1<span class="math inline">\(\alpha\)</span>KO and HP1<span class="math inline">\(\alpha\)</span>T</a></li>
<li class="chapter" data-level="4.1.2" data-path="4-methods.html"><a href="4-methods.html#HP1Bgtarget"><i class="fa fa-check"></i><b>4.1.2</b> Generation of Floxed HP1<span class="math inline">\(\beta\)</span> &amp; Floxed HP1<span class="math inline">\(\gamma\)</span></a></li>
<li class="chapter" data-level="4.1.3" data-path="4-methods.html"><a href="4-methods.html#genotyping"><i class="fa fa-check"></i><b>4.1.3</b> Genotyping</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="4-methods.html"><a href="4-methods.html#cloning"><i class="fa fa-check"></i><b>4.2</b> Cloning</a><ul>
<li class="chapter" data-level="4.2.1" data-path="4-methods.html"><a href="4-methods.html#rna-isolation-cdna-generation-rnaseq"><i class="fa fa-check"></i><b>4.2.1</b> RNA Isolation, cDNA Generation, &amp; RNAseq</a></li>
<li class="chapter" data-level="4.2.2" data-path="4-methods.html"><a href="4-methods.html#cloning-of-rna-probes-for-in-situ-hybridization"><i class="fa fa-check"></i><b>4.2.2</b> Cloning of RNA Probes for <em>In-Situ Hybridization</em></a></li>
<li class="chapter" data-level="4.2.3" data-path="4-methods.html"><a href="4-methods.html#cloning-of-expression-constructs"><i class="fa fa-check"></i><b>4.2.3</b> Cloning of Expression Constructs</a></li>
<li class="chapter" data-level="4.2.4" data-path="4-methods.html"><a href="4-methods.html#site-directed-mutagenesis"><i class="fa fa-check"></i><b>4.2.4</b> Site Directed Mutagenesis</a></li>
<li class="chapter" data-level="4.2.5" data-path="4-methods.html"><a href="4-methods.html#transformation"><i class="fa fa-check"></i><b>4.2.5</b> Transformation</a></li>
<li class="chapter" data-level="4.2.6" data-path="4-methods.html"><a href="4-methods.html#clonal-selection-plasmid-production"><i class="fa fa-check"></i><b>4.2.6</b> Clonal Selection &amp; Plasmid Production</a></li>
<li class="chapter" data-level="4.2.7" data-path="4-methods.html"><a href="4-methods.html#in-vitro-transcription-ivt"><i class="fa fa-check"></i><b>4.2.7</b> In Vitro Transcription (IVT)</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="4-methods.html"><a href="4-methods.html#cell-culture"><i class="fa fa-check"></i><b>4.3</b> Cell Culture</a><ul>
<li class="chapter" data-level="4.3.1" data-path="4-methods.html"><a href="4-methods.html#primary-neuronal-culture"><i class="fa fa-check"></i><b>4.3.1</b> Primary Neuronal Culture</a></li>
<li class="chapter" data-level="4.3.2" data-path="4-methods.html"><a href="4-methods.html#hek293t-culture-transfection"><i class="fa fa-check"></i><b>4.3.2</b> HEK293T Culture &amp; Transfection</a></li>
<li class="chapter" data-level="4.3.3" data-path="4-methods.html"><a href="4-methods.html#co-immunoprecipitation-co-ip-and-western-blot-wb"><i class="fa fa-check"></i><b>4.3.3</b> Co-Immunoprecipitation (Co-IP) and Western Blot (WB)</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="4-methods.html"><a href="4-methods.html#in-utero-electroporation-tissue-processing-and-histology"><i class="fa fa-check"></i><b>4.4</b> In Utero Electroporation, Tissue Processing and Histology</a><ul>
<li class="chapter" data-level="4.4.1" data-path="4-methods.html"><a href="4-methods.html#IUEmethod"><i class="fa fa-check"></i><b>4.4.1</b> In Utero Electroporation (IUE)</a></li>
<li class="chapter" data-level="4.4.2" data-path="4-methods.html"><a href="4-methods.html#tissue-fixation-and-perfusion"><i class="fa fa-check"></i><b>4.4.2</b> Tissue Fixation and Perfusion</a></li>
<li class="chapter" data-level="4.4.3" data-path="4-methods.html"><a href="4-methods.html#cryosectioning"><i class="fa fa-check"></i><b>4.4.3</b> Cryosectioning</a></li>
<li class="chapter" data-level="4.4.4" data-path="4-methods.html"><a href="4-methods.html#immuno-histochemistry-ihc"><i class="fa fa-check"></i><b>4.4.4</b> Immuno Histochemistry (IHC)</a></li>
<li class="chapter" data-level="4.4.5" data-path="4-methods.html"><a href="4-methods.html#quantification-of-cells-in-the-dentate-gyrus"><i class="fa fa-check"></i><b>4.4.5</b> Quantification of Cells in the Dentate Gyrus</a></li>
<li class="chapter" data-level="4.4.6" data-path="4-methods.html"><a href="4-methods.html#in-situ-hybridization-ish"><i class="fa fa-check"></i><b>4.4.6</b> In Situ Hybridization (ISH)</a></li>
<li class="chapter" data-level="4.4.7" data-path="4-methods.html"><a href="4-methods.html#paraffin-sectioning"><i class="fa fa-check"></i><b>4.4.7</b> Paraffin Sectioning</a></li>
<li class="chapter" data-level="4.4.8" data-path="4-methods.html"><a href="4-methods.html#nissl"><i class="fa fa-check"></i><b>4.4.8</b> Nissl Stain</a></li>
<li class="chapter" data-level="4.4.9" data-path="4-methods.html"><a href="4-methods.html#golgi-impregnation-and-dendritic-complexity-analysis"><i class="fa fa-check"></i><b>4.4.9</b> Golgi Impregnation and Dendritic Complexity Analysis</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="4-methods.html"><a href="4-methods.html#behavioral-experiments"><i class="fa fa-check"></i><b>4.5</b> Behavioral Experiments</a><ul>
<li class="chapter" data-level="4.5.1" data-path="4-methods.html"><a href="4-methods.html#open-field-activity-1"><i class="fa fa-check"></i><b>4.5.1</b> Open Field Activity</a></li>
<li class="chapter" data-level="4.5.2" data-path="4-methods.html"><a href="4-methods.html#paired-pulse-inhibition-1"><i class="fa fa-check"></i><b>4.5.2</b> Paired-Pulse Inhibition</a></li>
<li class="chapter" data-level="4.5.3" data-path="4-methods.html"><a href="4-methods.html#context-cued-fear-conditioning"><i class="fa fa-check"></i><b>4.5.3</b> Context-Cued Fear Conditioning</a></li>
<li class="chapter" data-level="4.5.4" data-path="4-methods.html"><a href="4-methods.html#Barnesmeth"><i class="fa fa-check"></i><b>4.5.4</b> Barnes Maze</a></li>
<li class="chapter" data-level="4.5.5" data-path="4-methods.html"><a href="4-methods.html#social-activity-monitor-sam-1"><i class="fa fa-check"></i><b>4.5.5</b> Social Activity Monitor (SAM)</a></li>
<li class="chapter" data-level="4.5.6" data-path="4-methods.html"><a href="4-methods.html#micro-behaviors-homecagescan"><i class="fa fa-check"></i><b>4.5.6</b> Micro Behaviors (HomeCageScan)</a></li>
<li class="chapter" data-level="4.5.7" data-path="4-methods.html"><a href="4-methods.html#nestcomethod"><i class="fa fa-check"></i><b>4.5.7</b> Nest Construction</a></li>
</ul></li>
<li class="chapter" data-level="4.6" data-path="4-methods.html"><a href="4-methods.html#magnetic-resonance-imaging-mri-and-volumetrics"><i class="fa fa-check"></i><b>4.6</b> Magnetic Resonance Imaging (MRI) and Volumetrics</a></li>
<li class="chapter" data-level="4.7" data-path="4-methods.html"><a href="4-methods.html#reduced-read-bisulfite-sequencing-rrbseq"><i class="fa fa-check"></i><b>4.7</b> Reduced Read Bisulfite Sequencing (RRBSeq)</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html"><i class="fa fa-check"></i><b>5</b> Supplemental Data</a><ul>
<li class="chapter" data-level="5.1" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html#Behavsupplement"><i class="fa fa-check"></i><b>5.1</b> Behavior</a><ul>
<li class="chapter" data-level="5.1.1" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html#homecagescan-micro-behaviors"><i class="fa fa-check"></i><b>5.1.1</b> HomeCageScan Micro Behaviors</a></li>
<li class="chapter" data-level="5.1.2" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html#context-cued-fear-conditioning-1"><i class="fa fa-check"></i><b>5.1.2</b> Context-Cued Fear Conditioning</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html#DE"><i class="fa fa-check"></i><b>5.2</b> Differential Gene Expression</a><ul>
<li class="chapter" data-level="5.2.1" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html#differentially-expressed-genes-in-aged-hp1betako"><i class="fa fa-check"></i><b>5.2.1</b> Differentially Expressed Genes in Aged HP1<span class="math inline">\(\beta\)</span>KO</a></li>
<li class="chapter" data-level="5.2.2" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html#inflam"><i class="fa fa-check"></i><b>5.2.2</b> Genes Related to Anti-viral and Interferon Response in HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO</a></li>
<li class="chapter" data-level="5.2.3" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html#genomic-hotspots-sensitive-to-hp1-deficiency-in-the-hippocampus"><i class="fa fa-check"></i><b>5.2.3</b> <em>Genomic Hotspots Sensitive to HP1 Deficiency in the Hippocampus</em></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html"><i class="fa fa-check"></i><b>6</b> Bioinformatics &amp; Data Processing</a><ul>
<li class="chapter" data-level="6.1" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#software-environment-management"><i class="fa fa-check"></i><b>6.1</b> Software Environment Management</a></li>
<li class="chapter" data-level="6.2" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#snakemake-jobs"><i class="fa fa-check"></i><b>6.2</b> Snakemake Jobs</a></li>
<li class="chapter" data-level="6.3" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#tetranscripts-pipeline"><i class="fa fa-check"></i><b>6.3</b> TETranscripts Pipeline</a></li>
<li class="chapter" data-level="6.4" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#differential-gene-expression-using-edger"><i class="fa fa-check"></i><b>6.4</b> Differential Gene Expression Using EdgeR</a></li>
<li class="chapter" data-level="6.5" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#deeptools-pipeline"><i class="fa fa-check"></i><b>6.5</b> Deeptools Pipeline</a></li>
<li class="chapter" data-level="6.6" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#lions-pipeline"><i class="fa fa-check"></i><b>6.6</b> LIONS Pipeline</a></li>
<li class="chapter" data-level="6.7" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#analysis-of-reduced-read-bisulphite-sequencing-rrbs"><i class="fa fa-check"></i><b>6.7</b> Analysis of Reduced-Read Bisulphite Sequencing (RRBS)</a><ul>
<li class="chapter" data-level="6.7.1" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#methylkit-and-circos-plot"><i class="fa fa-check"></i><b>6.7.1</b> Methylkit and Circos Plot</a></li>
<li class="chapter" data-level="6.7.2" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#annotated-ideogram"><i class="fa fa-check"></i><b>6.7.2</b> Annotated Ideogram</a></li>
</ul></li>
<li class="chapter" data-level="6.8" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#about-this-thesis"><i class="fa fa-check"></i><b>6.8</b> About this Thesis</a><ul>
<li class="chapter" data-level="6.8.1" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#session-info"><i class="fa fa-check"></i><b>6.8.1</b> Session Info</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7" data-path="7-references.html"><a href="7-references.html"><i class="fa fa-check"></i><b>7</b> References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">HP1 Deficiency Results in Accelerated Neurodegeneration</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="methods" class="section level1">
<h1><span class="header-section-number">Part 4</span> Methods</h1>
<p>All animal experiments were conducted in compliance with animal welfare guidelines put in place by the Landesamt für Gedundheit und Soziales Berlin (LaGeSo). HP1 mouse lines (HP1<span class="math inline">\(\alpha\)</span>KO, HP1<span class="math inline">\(\alpha\)</span>T, HP1<span class="math inline">\(\beta\)</span><sup>fl/fl</sup>, HP1<span class="math inline">\(\gamma\)</span><sup>fl/fl</sup>) were cloned by Dr. Jeremy Brown. The Emx1<sup>Cre</sup> mouse was obtained from the Jackson laboratory (<a href="http://www.jax.org/" class="uri">http://www.jax.org/</a>).</p>
<div id="mouse-lines" class="section level2">
<h2><span class="header-section-number">4.1</span> Mouse Lines</h2>
<p>The <strong>H</strong>eterochromatin <strong>P</strong>rotein <strong>1</strong> <strong>F</strong>loxed <strong>E</strong>mx1<strong>C</strong>re (HP1FEC) mouse line was generated by combining mouse lines containing floxed alleles of HP1<span class="math inline">\(\beta\)</span> (Cbx1, chromosome 11) and HP1<span class="math inline">\(\gamma\)</span> (Cbx3, chromosome 6) with the Emx1-IRES-Cre mouse line, which drives cre-mediated recombination in pyramidal neurons of the neocortex and hippocampus <span class="citation">(Gorski et al. <a href="#ref-GorskiCorticalExcitatoryNeurons2002">2002</a>)</span>. In this animal, the presence of the Cre transgene will induce excision of any and all floxed HP1 alleles in Emx1+ neuronal progenitors around E10.5. Initially, HP1<span class="math inline">\(\alpha\)</span> (Cbx5, chromosome 15) was also intended to be deleted, which was accomplished by crossing with an HP1<span class="math inline">\(\alpha\)</span> KO animal. However, as previously described, the HP1<span class="math inline">\(\alpha\)</span> KO allele was able to express using an alternate ATG start, and generated the truncated ‘HP1<span class="math inline">\(\alpha\)</span>T’, which we found to be infertile.</p>
<div id="HP1anull" class="section level3">
<h3><span class="header-section-number">4.1.1</span> Generation of HP1<span class="math inline">\(\alpha\)</span>KO and HP1<span class="math inline">\(\alpha\)</span>T</h3>
<div class="figure" style="text-align: center"><span id="fig:HP1atargeting"></span>
<img src="data/methods/HP1aTargeting.png" alt="Genetic Design for Generation of HP1$\alpha$ Null Mutation" width="100%" />
<p class="caption">
Figure 4.1: Genetic Design for Generation of HP1<span class="math inline">\(\alpha\)</span> Null Mutation
</p>
</div>
<p>The strategy to generate an HP1<span class="math inline">\(\alpha\)</span> null allele involved using a targeting cassette that inserts a STOP cassette preceding the canonical ATG of HP1<span class="math inline">\(\alpha\)</span> in addition to a cryptic ATG (Figure <a href="4-methods.html#fig:HP1atargeting">4.1</a>. Targetting cassettes were introduced into ES cells by homologous recombination. Successful integration was detected by neomycin selection. Following insertion of the targeting casette (NeO allele of Figure <a href="4-methods.html#fig:HP1atargeting">4.1</a>) all transcription of HP1<span class="math inline">\(\alpha\)</span> is halted, resulting in complete absence of the protein (HP1<span class="math inline">\(\alpha\)</span>KO). This design was to enable conditional reconstitution of the HP1<span class="math inline">\(\alpha\)</span> allele, where addition of Cre to the NeO allele induces recombination at loxP sites, removing the NeO STOP cassette, and transcription is allowed to proceed (FLPe Allele, Figure <a href="4-methods.html#fig:HP1atargeting">4.1</a>). The FLPe allele was not used in these studies. Crossing the FLPe allele with a flip deleter mouse results in flippase-mediated recombination at FRT sites, which was intended to remove all transcription start sites to give an HP1<span class="math inline">\(\alpha\)</span> null allele, but in fact resulted in HP1<span class="math inline">\(\alpha\)</span>T via downstream transcription start sites inside exon III (see Results <a href="2-results.html#HP1aT">2.1</a>).</p>
</div>
<div id="HP1Bgtarget" class="section level3">
<h3><span class="header-section-number">4.1.2</span> Generation of Floxed HP1<span class="math inline">\(\beta\)</span> &amp; Floxed HP1<span class="math inline">\(\gamma\)</span></h3>
<div class="figure" style="text-align: center"><span id="fig:HP1Btargeting"></span>
<img src="data/methods/HP1BTargeting.png" alt="Genetic Design for Generation of Floxed HP1$\beta$ Allele" width="100%" />
<p class="caption">
Figure 4.2: Genetic Design for Generation of Floxed HP1<span class="math inline">\(\beta\)</span> Allele
</p>
</div>
The targeting of HP1<span class="math inline">\(\beta\)</span> (Figure <a href="4-methods.html#fig:HP1Btargeting">4.2</a>) and HP1<span class="math inline">\(\gamma\)</span> (Figure <a href="4-methods.html#fig:HP1gtargeting">4.3</a>) used a similar strategy. This involved introduction of a targeting cassette into ES cells by homologous recombination that inserted loxP sites surrounding exons II and III. Succesfful integrations were detected by neomycin selection via an FRT flanked neomycin cassette inserted between exons III and IV (NeO allele, Figure <a href="4-methods.html#fig:HP1Btargeting">4.2</a> &amp; <a href="4-methods.html#fig:HP1gtargeting">4.3</a>). Following crossing to a flip-deleter mouse, the NeO cassette is removed by flippase-mediated recombination, giving the conditional (HP1<span class="math inline">\(\beta\)</span><sup>fl/fl</sup>) allele. In the presence of cre recombinase (either by ectopic expression or by crossing with a cre mouse such as Emx1<sup>Cre</sup>), recombination occurs at loxP sites, resuting in excision of exons II and III and loss of expression (Cre allele, Figure <a href="4-methods.html#fig:HP1Btargeting">4.2</a> &amp; <a href="4-methods.html#fig:HP1gtargeting">4.3</a>).
<div class="figure" style="text-align: center"><span id="fig:HP1gtargeting"></span>
<img src="data/methods/HP1gTargeting.png" alt="Genetic Design for Generation of Floxed HP1$\gamma$ Allele" width="100%" />
<p class="caption">
Figure 4.3: Genetic Design for Generation of Floxed HP1<span class="math inline">\(\gamma\)</span> Allele
</p>
</div>
</div>
<div id="genotyping" class="section level3">
<h3><span class="header-section-number">4.1.3</span> Genotyping</h3>
<div id="dna-isolation" class="section level4 unnumbered">
<h4><em>DNA Isolation</em></h4>
<p>For Genotyping, a simple ispropropanol DNA isolation protocol was utilized. This involved mixing 0.3 cm tailcuts or earcuts in 1.5ml tubes with 300 <span class="math inline">\(\mu\)</span>l of lysis buffer (100 mM NaCl, 10 mM EDTA pH 8.0, 10mM Tris pH 8.0, 0.5% SDS) each supplemented with 6 <span class="math inline">\(\mu\)</span>l Proteinase K (10 mg/ml). Lysis proceeded on a heatblock shaker and lysed at 55<span class="math inline">\(^\circ\)</span>C for a minimum of 2hrs. Following lysis, tubes were centrifuged at 13,000rpm on a tabletop centrifuge for 2 minutes to pellet hair and unlysed debris. The supernatant was then moved to a new tube where it was mixed with an equal volume (300 <span class="math inline">\(\mu\)</span>l) of 100% isopropanol to precipitate the DNA. Following thorough mixing, DNA was pelleted by centrifugation at 13000rpm for 15mins. The supernatant was then discarded and the DNA pellet was washed twice with 70% ethanol, spinning down for 5 minutes at 13,000rpm between washes. Following removal of the final ethanol wash, the DNA pellet is allowed to air-dry. DNA is then resuspended in 50 <span class="math inline">\(\mu\)</span>l of ddH<sub>2</sub>O.</p>
<p>Genotyping was then perfomed using GoTaq Polymerase (Promega) using the appropriate primers listed in table <a href="4-methods.html#tab:GenotypingPCRs">4.1</a> in a multiple of the following reaction:</p>
<blockquote>
<p>(For five 20 <span class="math inline">\(\mu\)</span>l reactions)<br />
20 <span class="math inline">\(\mu\)</span>l 5X GoTaq Buffer<br />
2 <span class="math inline">\(\mu\)</span>l dNTPs (10 mM)<br />
X <span class="math inline">\(\mu\)</span>l Forward Primer (10 <span class="math inline">\(\mu\)</span>M) (depending on the PCR)<br />
X <span class="math inline">\(\mu\)</span>l Reverse Primer (10 <span class="math inline">\(\mu\)</span>M) (depending on the PCR)<br />
1 <span class="math inline">\(\mu\)</span>l genomic DNA<br />
0.5 <span class="math inline">\(\mu\)</span>l GoTaq Polymerase<br />
ddH<sub>2</sub>0 (to 100 <span class="math inline">\(\mu\)</span>l)</p>
</blockquote>
<table class="table table-striped table-hover table-condensed table-responsive" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:GenotypingPCRs">Table 4.1: </span>HP1FEC Genotyping Primers and PCR Programs
</caption>
<thead>
<tr>
<th style="text-align:left;">
PCR
</th>
<th style="text-align:left;">
Primers
</th>
<th style="text-align:left;">
Program
</th>
<th style="text-align:left;">
Amplicons
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
HP1<span class="math inline">\(\alpha\)</span>
</td>
<td style="text-align:left;">
18319: gacagccaaggctacacag
</td>
<td style="text-align:left;">
95<span class="math inline">\(^\circ\)</span>C 1’
</td>
<td style="text-align:left;">
KO: 347b.p.
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
23267: ggtgttggacaggcgcatgg
</td>
<td style="text-align:left;">
95<span class="math inline">\(^\circ\)</span>C 60’’ —
</td>
<td style="text-align:left;">
WT: 450b.p.
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
22436: cccaagcagagtcagtctca
</td>
<td style="text-align:left;">
62<span class="math inline">\(^\circ\)</span>C 30’’ x39
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
72<span class="math inline">\(^\circ\)</span>C 45’’ —
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
72<span class="math inline">\(^\circ\)</span>C 1’
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
12<span class="math inline">\(^\circ\)</span>C
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
HP1<span class="math inline">\(\beta\)</span>
</td>
<td style="text-align:left;">
22427: gagtacgaactaccatactggac
</td>
<td style="text-align:left;">
95<span class="math inline">\(^\circ\)</span>C 1’
</td>
<td style="text-align:left;">
Targeted: 500b.p.
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Intr3Rev B: ctgcaccatggatgaaaag
</td>
<td style="text-align:left;">
95<span class="math inline">\(^\circ\)</span>C 60’’ —
</td>
<td style="text-align:left;">
WT: 200b.p.
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
58<span class="math inline">\(^\circ\)</span>C 30’’ x39
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
72<span class="math inline">\(^\circ\)</span>C 45’’ —
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
72<span class="math inline">\(^\circ\)</span>C 1’
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
12<span class="math inline">\(^\circ\)</span>C
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
HP1<span class="math inline">\(\gamma\)</span>
</td>
<td style="text-align:left;">
22431: agtcccagtactgagagttc
</td>
<td style="text-align:left;">
95<span class="math inline">\(^\circ\)</span>C 1’
</td>
<td style="text-align:left;">
Targeted: 400b.p.
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
22432: ctctacctcctgagtactag
</td>
<td style="text-align:left;">
95<span class="math inline">\(^\circ\)</span>C 60’’ —
</td>
<td style="text-align:left;">
WT: 200 b.p.
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
58<span class="math inline">\(^\circ\)</span>C 30’’ x39
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
72<span class="math inline">\(^\circ\)</span>C 45’’ —
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
72<span class="math inline">\(^\circ\)</span>C 1’
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
12<span class="math inline">\(^\circ\)</span>C
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Emx1Cre
</td>
<td style="text-align:left;">
4170: aaggtgtggttccagaatcg
</td>
<td style="text-align:left;">
95<span class="math inline">\(^\circ\)</span>C 2’
</td>
<td style="text-align:left;">
CRE: 450b.p.
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
4171: ctctccaccagaaggctgag
</td>
<td style="text-align:left;">
95<span class="math inline">\(^\circ\)</span>C 10’’ —
</td>
<td style="text-align:left;">
WT: 378b.p.
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
2 X 159: tcgatgcaacgagtgatgag
</td>
<td style="text-align:left;">
55<span class="math inline">\(^\circ\)</span>C 30’’ x39
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
2 X 160: ttcggctatacgtaacaggg
</td>
<td style="text-align:left;">
72<span class="math inline">\(^\circ\)</span>C 45’’ —
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
72<span class="math inline">\(^\circ\)</span>C 1’
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
12<span class="math inline">\(^\circ\)</span>C
</td>
<td style="text-align:left;">
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="cloning" class="section level2">
<h2><span class="header-section-number">4.2</span> Cloning</h2>
<p>For all cloning, construct design and annotation was done using either ApE plasmid editor (M. Wayne Davis; <a href="http://biologylabs.utah.edu/jorgensen/wayned/ape/" class="uri">http://biologylabs.utah.edu/jorgensen/wayned/ape/</a>) or Snapgene Viewer (GSL biotech; <a href="http://www.snapgene.com/products/snapgene_viewer/" class="uri">http://www.snapgene.com/products/snapgene_viewer/</a>).</p>
<div id="rna-isolation-cdna-generation-rnaseq" class="section level3">
<h3><span class="header-section-number">4.2.1</span> RNA Isolation, cDNA Generation, &amp; RNAseq</h3>
<div id="rna-isolation" class="section level4 unnumbered">
<h4><em>RNA Isolation</em></h4>
<p>RNA was isolated from E17, P0 and adult C57BL/6J mouse cerebral cortex using TRIzol (Invitrogen) according to the manufacturer’s instructions. Briefly, this involved homogenizing cortices in 500 <span class="math inline">\(\mu\)</span>l TRIzol using a tissue homogenizer, adding 100 <span class="math inline">\(\mu\)</span>l 100% chloroform, shaking vigorously for 15 seconds, and allowing to incubate 5 mins RT. Following a second thorough mixing, the homogenate was then spun for 15 minutes at 13,000rpm, 4<span class="math inline">\(^\circ\)</span>C on a tabletop centrifuge to allow phase separation. The upper aqueous phase (~200 <span class="math inline">\(\mu\)</span>l) was moved to a new tube, and 250 <span class="math inline">\(\mu\)</span>l of isoporpoanol was added to precipitate RNA at -20<span class="math inline">\(^\circ\)</span>C for 1 hour. Precipitaed RNA was then pelleted by spinning for 10 minutes at 13,000rpm, 4<span class="math inline">\(^\circ\)</span>C. Supernatant was decanted and RNA pellet was washed once with 75% ethanol (molecular grade ethanol, diluted with RNase-free ddH<sub>2</sub>0. Following removal of the ethanol wash, RNA pellet was allowed to air dry for 15 minutes and resuspended in 50 <span class="math inline">\(\mu\)</span>l of RNase-free water and 1 <span class="math inline">\(\mu\)</span>l RNAsin RNase inhibitor (Promega).</p>
<p>For RNAseq, RNA was isolated using the Relia-prep RNA mini kit (Promega) according to the manufacturer’s instructions.</p>
</div>
<div id="cdna-library-generation" class="section level4 unnumbered">
<h4><em>cDNA Library Generation</em></h4>
<p>cDNA was generated from the obtained RNA using SuperScript II reverse transcriptase (ThermoFischer). Briefly, 5 <span class="math inline">\(\mu\)</span>g of purified total RNA was mixed with 250 ng of random primers and 1 <span class="math inline">\(\mu\)</span>l of 10 mM dNTP mix, and the final volume brought to 12 <span class="math inline">\(\mu\)</span>l with RNAase-free water. Mixture was then heated to 65<span class="math inline">\(^\circ\)</span>C for 5 min and quickly chilled on ice. 4 <span class="math inline">\(\mu\)</span>l of 5X First-Strand Buffer was then added alongside 2 <span class="math inline">\(\mu\)</span>l 0.1 M DTT, and incubated at 25<span class="math inline">\(^\circ\)</span>C for 2 minutes. To begin the reverse transcription reaction, 1 <span class="math inline">\(\mu\)</span>l of SuperScript II Reverse Transcriptase was added, mixed by pipetting, and the final volume was brought to 20 <span class="math inline">\(\mu\)</span>l with RNase-free water. Because random primers were used, the reaction was incubated at 25<span class="math inline">\(^\circ\)</span>C for 10 minutes, and then 42<span class="math inline">\(^\circ\)</span>C for 50 minutes. Reaction was inactivated by heating at 70<span class="math inline">\(^\circ\)</span>C for 15 minutes.</p>
</div>
</div>
<div id="cloning-of-rna-probes-for-in-situ-hybridization" class="section level3">
<h3><span class="header-section-number">4.2.2</span> Cloning of RNA Probes for <em>In-Situ Hybridization</em></h3>
<div id="probe-design" class="section level4 unnumbered">
<h4><em>Probe Design</em></h4>
<p>For genic probes, gene coding sequence was obtained from the UCSC gene sorter and submitted to NCBI’s mouse-specific BLAST to confirm unique transcript regions. A unique ~1,000 bp region was selected to generate an amplicon of 600-900 bp. Primers for this amplicon were designed by submitting the unique region to Primer3 (<a href="http://bioinfo.ut.ee/primer3" class="uri">http://bioinfo.ut.ee/primer3</a>) and specifying the size range of 600-900 bp. Primers were typically designed to anneal around 58<span class="math inline">\(^\circ\)</span>C.</p>
<p>For non-genic probes, primers were designed around consensus sequences obtained from RepBase. Potential amplicons were then queried using UCSC’s BLAT or NCBI BLAST to determine the estimated diversity of transcripts corresponding to the probe. A probe unique for an IAP element on chromosome 2 was inferred from searching (UCSC in silico PCR) the location of previously used qPCR primers (Julie Brind’Amour, UBC). Primers used to clone in situ probes are listed in table <a href="4-methods.html#tab:probeprimers">4.2</a>.</p>
</div>
<div id="probe-pcr-vector-ligation" class="section level4 unnumbered">
<h4><em>Probe PCR &amp; Vector Ligation</em></h4>
<p>Probe templates were generated by amplifying from cDNA libraries using GoTaq polymerase in the following reaction:</p>
<blockquote>
<p>10 <span class="math inline">\(\mu\)</span>l 5X GoTaq Buffer<br />
1 <span class="math inline">\(\mu\)</span>l dNTPs (10 mM)<br />
1 <span class="math inline">\(\mu\)</span>l Forward Primer (10 <span class="math inline">\(\mu\)</span>M)<br />
1 <span class="math inline">\(\mu\)</span>l Reverse Primer (10 <span class="math inline">\(\mu\)</span>M)<br />
1 <span class="math inline">\(\mu\)</span>l cDNA<br />
0.5 <span class="math inline">\(\mu\)</span>l GoTaq Polymerase<br />
35.5 <span class="math inline">\(\mu\)</span>l ddH<sub>2</sub>0 (to 50 <span class="math inline">\(\mu\)</span>l)</p>
</blockquote>
<p>PCR Program:</p>
<table>
<thead>
<tr class="header">
<th>Temperature</th>
<th>Duration</th>
<th>Step</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>95<span class="math inline">\(^\circ\)</span>C</td>
<td>1 minute initial denaturation</td>
<td></td>
</tr>
<tr class="even">
<td>95<span class="math inline">\(^\circ\)</span>C</td>
<td>30 seconds denaturation</td>
<td>—</td>
</tr>
<tr class="odd">
<td>58<span class="math inline">\(^\circ\)</span>C</td>
<td>30 seconds annealing</td>
<td>x39 Cycles</td>
</tr>
<tr class="even">
<td>72<span class="math inline">\(^\circ\)</span>C</td>
<td>45 seconds elongation</td>
<td>—</td>
</tr>
<tr class="odd">
<td>72<span class="math inline">\(^\circ\)</span>C</td>
<td>1 minute final elongation</td>
<td></td>
</tr>
<tr class="even">
<td>20<span class="math inline">\(^\circ\)</span>C</td>
<td>END</td>
<td></td>
</tr>
</tbody>
</table>
<p>PCR amplicons were then directly loaded into a 1.5% agarose gel cast with TAE buffer (40mM Tris, 20mM acetic acid, 1mM EDTA, pH 8.0) and run at 150mV. Amplicons were gel-extracted using a QIAquick Gel extraction Kit (QIAGEN). Purified amplicon DNA was then ligated overnight at 4<span class="math inline">\(^\circ\)</span>C into the pGEMT easy vector (Promega) using T4 ligase in the following reaction:</p>
<blockquote>
<p>2 <span class="math inline">\(\mu\)</span>l 10X Rapid Ligation Buffer<br />
8 <span class="math inline">\(\mu\)</span>l Gel Extracted PCR Product<br />
0.5 <span class="math inline">\(\mu\)</span>l pGEMT vector 1 <span class="math inline">\(\mu\)</span>l T4 Ligase<br />
9.5 <span class="math inline">\(\mu\)</span>l H<sub>2</sub>O<br />
———————————–<br />
20 <span class="math inline">\(\mu\)</span>l Total</p>
</blockquote>
<table class="table table-striped table-hover table-condensed table-responsive" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:probeprimers">Table 4.2: </span>Primers used for In Situ Probes
</caption>
<thead>
<tr>
<th style="text-align:left;">
Probe
</th>
<th style="text-align:left;">
Primer
</th>
<th style="text-align:left;">
Sequence
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
IAP chr2 unique
</td>
<td style="text-align:left;">
fw:
</td>
<td style="text-align:left;">
CCAGCGGTGACCACTTAGA
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
rv:
</td>
<td style="text-align:left;">
AAGACAACACAGAGCCTCCT
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
IAP consensus
</td>
<td style="text-align:left;">
fw:
</td>
<td style="text-align:left;">
ATGTGCCGAGGGTGGTTCTT
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
rv:
</td>
<td style="text-align:left;">
TGAATCCTTCTTAACAGTCTGCT
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
MERVL consensus
</td>
<td style="text-align:left;">
fw:
</td>
<td style="text-align:left;">
CTTCCATTCACAGCTGCGACTG
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
rv:
</td>
<td style="text-align:left;">
TTTCCACAGATACTAAGAGCAGC
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
LINE1 ORF2
</td>
<td style="text-align:left;">
fw:
</td>
<td style="text-align:left;">
CAAGATCCAACACCCATTCATGA
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
rv:
</td>
<td style="text-align:left;">
TTCCGCCAGAAGTTCTTTTATCC
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Impact
</td>
<td style="text-align:left;">
fw:
</td>
<td style="text-align:left;">
TTTATGGCGAGGAGTGGTGT
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
rv:
</td>
<td style="text-align:left;">
TCTCACACAGGTTCACGTCTTA
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Inpp5f
</td>
<td style="text-align:left;">
fw:
</td>
<td style="text-align:left;">
TGGGTAATAAGGTCCAGGGG
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
rv:
</td>
<td style="text-align:left;">
GGGTTCTGCTGGACTCTTCT
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Gnas
</td>
<td style="text-align:left;">
fw:
</td>
<td style="text-align:left;">
ATGGATCGCAGGTCCCGG
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
rv:
</td>
<td style="text-align:left;">
TTAGTGACGCCGGATGGG
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Grb10
</td>
<td style="text-align:left;">
fw:
</td>
<td style="text-align:left;">
TACCATGCCAAGTGAGAGCA
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
rv:
</td>
<td style="text-align:left;">
CGCCTTTGGATTACTCTGGC
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Mdga1
</td>
<td style="text-align:left;">
fw:
</td>
<td style="text-align:left;">
CAGTACCACCCACCATCAGT
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
rv:
</td>
<td style="text-align:left;">
TTGTGCTGGTTCAACTGTCG
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Pip5k1c
</td>
<td style="text-align:left;">
fw:
</td>
<td style="text-align:left;">
TGCACAACATCGATCAGCAG
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
rv:
</td>
<td style="text-align:left;">
GGCAGCGTCTATTTCCACAG
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Rasgrf1
</td>
<td style="text-align:left;">
fw:
</td>
<td style="text-align:left;">
ATCGTGTGCTGAATGTGCTC
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
rv:
</td>
<td style="text-align:left;">
TAGTCTGCTGAAACTGGCGA
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Plagl1
</td>
<td style="text-align:left;">
fw:
</td>
<td style="text-align:left;">
ACCCACGATCCCAACAAGAT
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
rv:
</td>
<td style="text-align:left;">
GAGGAGAGGTGGGAACTACG
</td>
</tr>
</tbody>
</table>

</div>
</div>
<div id="cloning-of-expression-constructs" class="section level3">
<h3><span class="header-section-number">4.2.3</span> Cloning of Expression Constructs</h3>
<p>To clone HP1 and SUV420H2 constructs, two rounds of PCR were conducted. Primers used for clonding of expression constructs are listed in table <a href="4-methods.html#tab:HP1cloningprimers">4.3</a>. The first round involved amplifying the ‘naked’ coding region of the gene, which utilized *_110 and *_111 primers. After succesful cloning of ‘naked’ inserts into pGEMT plasmids, these plasmids were then used as templates in a second PCR reaction, using primers containing restriction sites suitable for ligation into fusion constructs. In the case of SUV420H2, the second round of PCR added a C-terminal myc tag.</p>
<table class="table table-striped table-hover table-condensed table-responsive" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:HP1cloningprimers">Table 4.3: </span>Primers Used for Generating Expression Plasmids
</caption>
<thead>
<tr>
<th style="text-align:left;">
Gene
</th>
<th style="text-align:left;">
Primer
</th>
<th style="text-align:left;">
Sequence
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
HP1<span class="math inline">\(\alpha\)</span>T
</td>
<td style="text-align:left;">
HP1aT1_F:
</td>
<td style="text-align:left;">
GCCATAGCCACAATGCTTAACT
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
HP1a_At_R:
</td>
<td style="text-align:left;">
GACTGCCTCCTCCTTTAGCTC
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
HP1aT_kpnI_F:
</td>
<td style="text-align:left;">
aaggtaccGCCATAGCCACAATGCTTAA
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
HP1aT_AgeI_R:
</td>
<td style="text-align:left;">
taACCGGTaaGCTCTTCGCGCTTTCTTTT
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
HP1<span class="math inline">\(\alpha\)</span>
</td>
<td style="text-align:left;">
HP1a_110_F:
</td>
<td style="text-align:left;">
CACGACATGGGAAAGAAGACCAA
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
HP1a_111_R:
</td>
<td style="text-align:left;">
GCTCTTCGCGCTTTCTTTTTCTTTG
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
HP1a_kpn1_F:
</td>
<td style="text-align:left;">
aaggtaccCACGACATGGGAAAGAAG
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
HP1a_Age1_R:
</td>
<td style="text-align:left;">
taACCGGTaaGCTCTTCGCGCTTTC
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
HP1<span class="math inline">\(\beta\)</span>
</td>
<td style="text-align:left;">
HP1B_110_F:
</td>
<td style="text-align:left;">
ATGGGGAAAAAGCAAAACAAGAAG
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
HP1B_111_R:
</td>
<td style="text-align:left;">
ATTCTTGTCGTCTTTTTTGTCATCAT
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
HP1B_kpn1_F:
</td>
<td style="text-align:left;">
AGGGTACCCGTACTATGGGGAAAAAGC
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
HP1B_Age1_R:
</td>
<td style="text-align:left;">
GGACCGGTaAATTCTTGTCGTCTTTTTTGT
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
HP1<span class="math inline">\(\gamma\)</span>
</td>
<td style="text-align:left;">
HP1g_110_F:
</td>
<td style="text-align:left;">
ATGGCCTCCAATAAAACTACATTG
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
HP1g_110_R:
</td>
<td style="text-align:left;">
TTGTGCTTCATCTTCAGGACAAG
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
HP1g_kpn1_F:
</td>
<td style="text-align:left;">
AAGGTACCTAAAAAATGGCCTCCAATA
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
HP1g_Age1_R:
</td>
<td style="text-align:left;">
AGACCGGTAATTGTGCTTCATCTTCAG
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
SUV420H2
</td>
<td style="text-align:left;">
SUV420H2_110_F:
</td>
<td style="text-align:left;">
AGCACCATGGGGCCTGATC
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
SUV420H2_110_R:
</td>
<td style="text-align:left;">
TGGCTCACCACTATTGATGTCCA
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
kpnI-kzkSUV420_F:
</td>
<td style="text-align:left;">
AAGGTACCGCCATGGGGCCTGATCGAGTG
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
xhoI-SUV420_R:
</td>
<td style="text-align:left;">
TGCTCGAGTGGCTCACCACTATTGATGTCCAG
</td>
</tr>
</tbody>
</table>
<p>‘Naked’ coding sequences for HP1 and SUV420H2 expression plasmids were amplified using the high-fidelity Q5 Polymerase (New England Biolabs) in the following reaction:</p>
<blockquote>
<p>12.5 <span class="math inline">\(\mu\)</span>l 2X Q5 Hot-Start Master Mix<br />
1.25 <span class="math inline">\(\mu\)</span>l Forward Primer (10 <span class="math inline">\(\mu\)</span>M)<br />
1.25 <span class="math inline">\(\mu\)</span>l Reverse Primer (10 <span class="math inline">\(\mu\)</span>M)<br />
1 <span class="math inline">\(\mu\)</span>l cDNA<br />
9 <span class="math inline">\(\mu\)</span>l H<sub>2</sub>O<br />
———————————–<br />
25 <span class="math inline">\(\mu\)</span>l Total</p>
</blockquote>
<p>PCR Program:</p>
<table>
<thead>
<tr class="header">
<th>Temperature</th>
<th>Duration</th>
<th>Step</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>98<span class="math inline">\(^\circ\)</span>C</td>
<td>30 seconds initial denaturation</td>
<td></td>
</tr>
<tr class="even">
<td>98<span class="math inline">\(^\circ\)</span>C</td>
<td>10 seconds denaturation</td>
<td>—</td>
</tr>
<tr class="odd">
<td>Variable</td>
<td>30 seconds annealing</td>
<td>x39 Cycles</td>
</tr>
<tr class="even">
<td>72<span class="math inline">\(^\circ\)</span>C</td>
<td>45 seconds elongation</td>
<td>—</td>
</tr>
<tr class="odd">
<td>72<span class="math inline">\(^\circ\)</span>C</td>
<td>5 minutes final elongation</td>
<td></td>
</tr>
<tr class="even">
<td>20<span class="math inline">\(^\circ\)</span>C</td>
<td>END</td>
<td></td>
</tr>
</tbody>
</table>
<p>Annealing temperature for HP1<span class="math inline">\(\beta\)</span> and HP1<span class="math inline">\(\gamma\)</span> was 64<span class="math inline">\(^\circ\)</span>C whereas for HP1<span class="math inline">\(\alpha\)</span> it was 68<span class="math inline">\(^\circ\)</span>C. PCR amplicons were then gel-extracted and A-tailed (GoTaq &amp; 1 <span class="math inline">\(\mu\)</span>M dNTPs in GoTaq buffer, 72<span class="math inline">\(^\circ\)</span>C for 1 hour) prior to overnight ligation into pGEMT backbone (as described above) yielding pGEMT-HP1x (where x is <span class="math inline">\(\alpha\)</span>, <span class="math inline">\(\beta\)</span>, or <span class="math inline">\(\gamma\)</span>). To generate eGFP fusions, HP1 coding sequences were re-amplified using the second set of primers which remove the stop codon and add KpnI and AgeI restriction sites. These amplicons were then A-tailed and ligated into pGEMT backbones yielding pGEMT-kpnI-HP1x. To create pCAG-HP1x-eGFP fusion constructs, pGEMT-kpnI-HP1x plasmids and pCAG-eGFP (Clontech) were digested with KpnI and AgeI for 4 hours at 37<span class="math inline">\(^\circ\)</span>C, and purified by gel extraction. HP1 inserts were then ligated into pCAG-eGFP overnight at 4<span class="math inline">\(^\circ\)</span>C using T4 ligase as described above, which yielded pCAG-HP1x-eGFP fusion constructs.</p>
<p>Prior to cloning SUV420H2, pCAG-mycDKK was created by substituting the CAG promoter from pCAG-eGFP with the CMV promoter of pCMV6-Entry though double digestion with SpeI and EcoRI. To clone SUV420H2, a ‘naked’ SUV420H2 amplicon was first amplified using 110 and 111 primers. Then , 1 <span class="math inline">\(\mu\)</span>l of this PCR product was used as a template in a second PCR using KpnI and XhoI primers. The PCR product from KpnI and XhoI primers was then gel-extracted and digested for 20 minutes using Fast Digest KpnI and XhoI (Fermentas). This digested product was then ligated to a KpnI XhoI digested pCAG-mycDKK vector, resulting in an in-frame C-terminal myc-DKK (Myc tag and Flag tag).</p>

</div>
<div id="site-directed-mutagenesis" class="section level3">
<h3><span class="header-section-number">4.2.4</span> Site Directed Mutagenesis</h3>
<p>For SUV420H2 IP experiments, mutations were designed in chromoshadow domain (CSD) of HP1<span class="math inline">\(\gamma\)</span>. The created CSD mutations are summarized in table <a href="4-methods.html#tab:CSDpointmutations">4.4</a>. All HP1 point mutations were created using the Q5 site-directed mutagenesis protocol (New England Biolabs). This involved designing primers using the online NEBaseChanger tool (<a href="http://nebasechanger.neb.com" class="uri">http://nebasechanger.neb.com</a>), using pGEMT-kpnI-HP1x plasmids as templates.</p>
<table class="table table-striped table-hover table-condensed table-responsive" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:CSDpointmutations">Table 4.4: </span>Chromoshadow Domain Mutations
</caption>
<thead>
<tr>
<th style="text-align:left;">
Position ( HP1<span class="math inline">\(\gamma\)</span> )
</th>
<th style="text-align:left;">
158
</th>
<th style="text-align:left;">
…
</th>
<th style="text-align:left;">
165
</th>
<th style="text-align:left;">
166
</th>
<th style="text-align:left;">
167
</th>
<th style="text-align:left;">
168
</th>
<th style="text-align:left;">
169
</th>
<th style="text-align:left;">
170
</th>
<th style="text-align:left;">
171
</th>
<th style="text-align:left;">
172
</th>
<th style="text-align:left;">
173
</th>
<th style="text-align:left;">
174
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
HP1<span class="math inline">\(\beta\)</span>
</td>
<td style="text-align:left;">
V
</td>
<td style="text-align:left;">
…
</td>
<td style="text-align:left;">
I
</td>
<td style="text-align:left;">
S
</td>
<td style="text-align:left;">
F
</td>
<td style="text-align:left;">
Y
</td>
<td style="text-align:left;">
E
</td>
<td style="text-align:left;">
E
</td>
<td style="text-align:left;">
R
</td>
<td style="text-align:left;">
L
</td>
<td style="text-align:left;">
T
</td>
<td style="text-align:left;">
W
</td>
</tr>
<tr>
<td style="text-align:left;">
HP1<span class="math inline">\(\alpha\)</span>
</td>
<td style="text-align:left;">
V
</td>
<td style="text-align:left;">
…
</td>
<td style="text-align:left;">
I
</td>
<td style="text-align:left;">
A
</td>
<td style="text-align:left;">
F
</td>
<td style="text-align:left;">
Y
</td>
<td style="text-align:left;">
E
</td>
<td style="text-align:left;">
E
</td>
<td style="text-align:left;">
R
</td>
<td style="text-align:left;">
L
</td>
<td style="text-align:left;">
T
</td>
<td style="text-align:left;">
W
</td>
</tr>
<tr>
<td style="text-align:left;">
HP1<span class="math inline">\(\gamma\)</span>
</td>
<td style="text-align:left;">
M
</td>
<td style="text-align:left;">
…
</td>
<td style="text-align:left;">
I
</td>
<td style="text-align:left;">
A
</td>
<td style="text-align:left;">
F
</td>
<td style="text-align:left;">
Y
</td>
<td style="text-align:left;">
E
</td>
<td style="text-align:left;">
E
</td>
<td style="text-align:left;">
R
</td>
<td style="text-align:left;">
L
</td>
<td style="text-align:left;">
T
</td>
<td style="text-align:left;">
W
</td>
</tr>
<tr>
<td style="text-align:left;">
codon :
</td>
<td style="text-align:left;">
ATG
</td>
<td style="text-align:left;">
…
</td>
<td style="text-align:left;">
ATT
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
TTC
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
GAG
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
CTG
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
TGG
</td>
</tr>
<tr>
<td style="text-align:left;">
a.a.
</td>
<td style="text-align:left;">
Methionine
</td>
<td style="text-align:left;">
…
</td>
<td style="text-align:left;">
Isoleucine
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Phenylalanine
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Glutamate
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Leucine
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Tryptophan
</td>
</tr>
<tr>
<td style="text-align:left;">
mutate to:
</td>
<td style="text-align:left;">
V
</td>
<td style="text-align:left;">
…
</td>
<td style="text-align:left;">
E
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
R
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
V
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
H
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
A
</td>
</tr>
<tr>
<td style="text-align:left;">
new codon
</td>
<td style="text-align:left;">
GTG
</td>
<td style="text-align:left;">
…
</td>
<td style="text-align:left;">
GAG
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
CGC
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
GTG
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
CAC
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
GCG
</td>
</tr>
<tr>
<td style="text-align:left;">
new a.a.
</td>
<td style="text-align:left;">
Valine
</td>
<td style="text-align:left;">
…
</td>
<td style="text-align:left;">
Glutamate
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Arginine
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Valine
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Histidine
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Alanine
</td>
</tr>
<tr>
<td style="text-align:left;">
name
</td>
<td style="text-align:left;">
M158V
</td>
<td style="text-align:left;">
…
</td>
<td style="text-align:left;">
I165E
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
F167R
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
E170V
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
L172H
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
W174A
</td>
</tr>
</tbody>
</table>
<p>Q5 site-directed mutagenesis was performed by first performing PCRs on pGEMT-knpI-HP1x plasmids using mutagenic primers listed in table <a href="4-methods.html#tab:pointmutprimertable">4.5</a>, in a reaction similar to that specified in <a href="4-methods.html#cloning-of-expression-constructs">4.2.3</a>. Notably, the duration of the elongation step of each PCR program was sufficient to amplify the length of the entire plasmid. Following 25 cycles of PCR, 5 <span class="math inline">\(\mu\)</span>l of product was run on a gel to check for a successful PCR, which should yield a linear product the length of the entire plasmid.</p>
<table class="table table-striped table-hover table-condensed table-responsive" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:pointmutprimertable">Table 4.5: </span>Primers Used for Generating Point Mutations and Deletions
</caption>
<thead>
<tr>
<th style="text-align:left;">
Gene
</th>
<th style="text-align:left;">
Mutation
</th>
<th style="text-align:left;">
Domain
</th>
<th style="text-align:left;">
Primers
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
CBX5
</td>
<td style="text-align:left;">
F44E
</td>
<td style="text-align:left;">
CD
</td>
<td style="text-align:left;">
GTGGAAAGGCgagTCTGAGGAGCAC
</td>
</tr>
<tr>
<td style="text-align:left;">
(HP1<span class="math inline">\(\alpha\)</span>)
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
TTCAACAGATATTCCACTTG
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
I165E
</td>
<td style="text-align:left;">
CSD
</td>
<td style="text-align:left;">
ACAGATTGTGgagGCATTTTATGAAGAGAGAC
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
GGACACTTCACGTTAGCTTC
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
CBX1
</td>
<td style="text-align:left;">
F45E
</td>
<td style="text-align:left;">
CD
</td>
<td style="text-align:left;">
GTGGAAGGGTgagTCAGATGAGGAC
</td>
</tr>
<tr>
<td style="text-align:left;">
(HP1<span class="math inline">\(\beta\)</span>)
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
TTTAGAAGATATTCCACCTTG
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
I161E
</td>
<td style="text-align:left;">
CSD
</td>
<td style="text-align:left;">
ACAGGTTGTCgaaTCCTTCTATGAGGAAAG
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
GGGCACTTGACATTGGCT
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
CBX3
</td>
<td style="text-align:left;">
F48E
</td>
<td style="text-align:left;">
CD
</td>
<td style="text-align:left;">
GGTGGAGTATgagCTGAAGTGGAAG
</td>
</tr>
<tr>
<td style="text-align:left;">
(HP1<span class="math inline">\(\gamma\)</span>)
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
TTCCCATTCACTACACGAC
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
M158V
</td>
<td style="text-align:left;">
CSD
</td>
<td style="text-align:left;">
GGAGGCGAACgtgAAGTGTCCTC
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
TTTGCCAGCACCAAGTCG
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
I164E
</td>
<td style="text-align:left;">
CSD
</td>
<td style="text-align:left;">
TCAGATTGTCgagGCCTTCTACGAGG
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
GGACACTTCATGTTCGCC
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
F167R
</td>
<td style="text-align:left;">
CSD
</td>
<td style="text-align:left;">
TGTCATTGCCcgcTACGAGGAGC
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
ATCTGAGGACACTTCATGTTC
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
E170V
</td>
<td style="text-align:left;">
CSD
</td>
<td style="text-align:left;">
CTTCTACGAGgtgCGGCTGACTT
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
GCAATGACAATCTGAGGACAC
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
L172H
</td>
<td style="text-align:left;">
CSD
</td>
<td style="text-align:left;">
CGAGGAGCGGcacACTTGGCATT
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
TAGAAGGCAATGACAATCTGAGG
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
W174A
</td>
<td style="text-align:left;">
CSD
</td>
<td style="text-align:left;">
GCGGCTGACTgcgCATTCTTGTC
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
TCCTCGTAGAAGGCAATG
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
<span class="math inline">\(\Delta\)</span>CSD
</td>
<td style="text-align:left;">
CSD
</td>
<td style="text-align:left;">
CATTCTTGTCCTGAAGATGAAGC
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
GTTCGCCTCCTTTGCCAG
</td>
</tr>
</tbody>
</table>
<p>Following a successful PCR, the PCR product was used in a Kinase, Ligase, and DpnI reaction, which phosphorylates the ends of the PCR product, ligates (circularizes) the PCR product into a plasmid, and DpnI digests any remaining pGEMT template:</p>
<blockquote>
<p>KLD Reaction:<br />
1 <span class="math inline">\(\mu\)</span>l PCR Product<br />
5 <span class="math inline">\(\mu\)</span>l 2X KLD Buffer<br />
1 <span class="math inline">\(\mu\)</span>l 10X KLD Enzyme mix<br />
3 <span class="math inline">\(\mu\)</span>l ddH<sub>2</sub>O</p>
</blockquote>
<p>KLD reaction was then allowed to proceed for 30 minutes at room temperature. 5 <span class="math inline">\(\mu\)</span>l of the KLD reaction was then transformed, yielding pGEMT-kpnI-HP1x(mut). After sequence verification, mutated HP1 inserts were digested from pGEMT using KpnI and AgeI and ligated into pCAG-eGFP, yielding HP1 point mutant-GFP fusions in pCAG-HP1x(mut)-eGFP. </p>
</div>
<div id="transformation" class="section level3">
<h3><span class="header-section-number">4.2.5</span> Transformation</h3>
<p>For pGEMT-ligations, prior to transformations, ampicilin (100 <span class="math inline">\(\mu\)</span>g/ml) agar plates were coated with 40 <span class="math inline">\(\mu\)</span>l XGal (40 mg/ml in DMF) and 40 <span class="math inline">\(\mu\)</span>l Isopropyl <span class="math inline">\(\beta\)</span>-D-1-thiogalactopyranoside (IPTG, 100 mM) to allow for blue/white selection. Plates were then warmed at 37<span class="math inline">\(^\circ\)</span>C for 15 minutes to ensure evaporation of DMSO.</p>
<p>Ligations were transformed into XL10-Gold heat-competent e.coli by thawing 50 <span class="math inline">\(\mu\)</span>l cells on ice for 10 minutes and adding 5 <span class="math inline">\(\mu\)</span>l of ligation mixture to the cells and applying a heat shock of 42<span class="math inline">\(^\circ\)</span>C for 45 seconds before returning to ice. After a 5 minute incubation on ice, 500 <span class="math inline">\(\mu\)</span>l of Lysogeny Broth (LB) transformation mixture was added and ligation was shaken vigorously for 15 minutes at 37<span class="math inline">\(^\circ\)</span>C before plating. Plates where then incubated overnight at 37<span class="math inline">\(^\circ\)</span>C.</p>
</div>
<div id="clonal-selection-plasmid-production" class="section level3">
<h3><span class="header-section-number">4.2.6</span> Clonal Selection &amp; Plasmid Production</h3>
<p>Following transforamtion, single colonies were picked and grown as minipreps in 4 ml LB + ampicilin (100 <span class="math inline">\(\mu\)</span>g/ml) overnight. Typically 3 colonies are sufficient to find a positive clone. The following day plasmid DNA was purified from miniprep cultures by using a QIAprep Spin Miniprep kit according to the manufacturer’s instructions. Following test digestions to determine the orientation of the insert, clones were sequenced to confirm template sequence fidelity. For expression constructs, clones deemed desirable were then re-transformed and grown in 200 ml of LB and purfied using endotoxin-free maxiprep kit (Machery Nagel), which can yield very high plasmid concentrations.</p>
</div>
<div id="in-vitro-transcription-ivt" class="section level3">
<h3><span class="header-section-number">4.2.7</span> In Vitro Transcription (IVT)</h3>
<p>To generate antisense RNA from pGEMT- templates, plasmids were linearized using a restriction site on the opposite side of the template from the in vitro transcription promoter (either T7 or SP6). Linearization of pGEMT- plasmids for IVT was typically performed overnight at 37<span class="math inline">\(^\circ\)</span>C in the following reaction:</p>
<blockquote>
<p>50 <span class="math inline">\(\mu\)</span>l pGEMT- vector containing probe template (~2.5 <span class="math inline">\(\mu\)</span>g of DNA)<br />
3 <span class="math inline">\(\mu\)</span>l Restriction enzyme (New England Biolabs)<br />
10 <span class="math inline">\(\mu\)</span>l 10X Restriction buffer (New England Biolabs)<br />
37 <span class="math inline">\(\mu\)</span>l H<sub>2</sub>0</p>
</blockquote>
<p>Linearized plasmid was then purified by gel-extracted using a QIAquick Gel Extraction kit (QIAGEN). To perform in vitro transcription, the following reaction was allowed to proceed at 37<span class="math inline">\(^\circ\)</span>C for 3 hours:</p>
<blockquote>
<p>14.5 <span class="math inline">\(\mu\)</span>l linearized DNA template (or ~2<span class="math inline">\(\mu\)</span>g of DNA)<br />
2 <span class="math inline">\(\mu\)</span>l 10X Transcription Buffer (Roche)<br />
2 <span class="math inline">\(\mu\)</span>l RNA Polymerase (T7 or SP6 depending on orientation of the insert) (Roche)<br />
0.5 <span class="math inline">\(\mu\)</span>l RNAse Inhibitor, Human Placenta (New England Biolabs)<br />
2 <span class="math inline">\(\mu\)</span>l dNTP DIG labeling mix (Roche)</p>
</blockquote>
<p>Following the 3 hours of incubation, the following was added to the reaction to digest DNA template:</p>
<blockquote>
<p>2.5 <span class="math inline">\(\mu\)</span>l 10X DNase buffer (New England Biolabs)<br />
2.5 <span class="math inline">\(\mu\)</span>l DNase (RNAse free) (New England Biolabs)</p>
</blockquote>
<p>and incubated for a further 30 minutes at 37<span class="math inline">\(^\circ\)</span>C. To precipitate the transcribed RNA,</p>
<blockquote>
<p>2 <span class="math inline">\(\mu\)</span>l 0.2M EDTA (pH 8.0)<br />
2.7 <span class="math inline">\(\mu\)</span>l 4M LiCl<br />
75 <span class="math inline">\(\mu\)</span>l 100% ultrapure ethanol</p>
</blockquote>
<p>was added, briefly vortexed, and RNA was allowed to precipitate overnight at -20<span class="math inline">\(^\circ\)</span>C.</p>
<p>The following day, RNA was pelleted by centrifugation at 13,000 rpm at 4<span class="math inline">\(^\circ\)</span>C on a tabletop centrifuge for 30 minutes. After decanting the supernatant, RNA pellet was washed with 75 <span class="math inline">\(\mu\)</span>l of 70% ethanol (Ultrapure, diluted with RNase-free H<sub>2</sub>O). Following removal of the ethanol wash, the RNA pellet was allowed to briefly airdry, before resolubilizing RNA in 20 <span class="math inline">\(\mu\)</span>l of water treated with Diethyl Pyrocarbonate (DEPC water) and 1 <span class="math inline">\(\mu\)</span>l of RNase inhibitor. After allowing pellet to disolve in DEPC water, 180 <span class="math inline">\(\mu\)</span>l of Hybmix Buffer was added (50% formamide, 5X SSC pH 7.0, 1% Boehringer block, 5mM EDTA, 0.1% Tween-20, 0.1% CHAPS, 0.1 mg/ml Heparin, 100 <span class="math inline">\(\mu\)</span>g/ml Yeast tRNA) and mixed thoroughly by vortexing. RNA probes were then stored at -20<span class="math inline">\(^\circ\)</span>C until needed.</p>
</div>
</div>
<div id="cell-culture" class="section level2">
<h2><span class="header-section-number">4.3</span> Cell Culture</h2>
<div id="primary-neuronal-culture" class="section level3">
<h3><span class="header-section-number">4.3.1</span> Primary Neuronal Culture</h3>
<div id="coverslip-preparation" class="section level4 unnumbered">
<h4><em>Coverslip Preparation</em></h4>
<p>Prior to neuronal culture, circular coverslips for 24-well plates were treated by ethanol wash. For all washes coverslips were placed in a large sealed container and left to shake, first with 70% ethanol (Ultrapure EtOH diluted in ddH<sub>2</sub>O) overnight, followed by three 1-hour washes with ddH<sub>2</sub>O, followed by two 90 minute washes in 100% ethanol (Ultrapure). Finally, coverslips were washed in 70% ethanol for 15 minutes before storing in 100% Ethanol. Prior to use, ethanol was allowed to evaporate from coverslips inside the cell culture hood. To allow neuronal adhesion, coverslips were then coated with poly-L-lysine (Sigma, 0.1mg/ml in ddH<sub>2</sub>O) by covering the entirety of the coverslip and incubating at 37<span class="math inline">\(^\circ\)</span>C overnight. The following morning, excess poly-L-lysine was removed, coverslips rinsed twice in ddH<sub>2</sub>O and left to dry in the cell culture hood.</p>
</div>
<div id="isolation-of-primary-cortical-and-hippocampal-neurons" class="section level4 unnumbered">
<h4><em>Isolation of Primary Cortical and Hippocampal Neurons</em></h4>
<p>Mouse brains of E17.5 embryos were isolated in ice-cold HBSS containing MgCl<sub>2</sub> and CaCl<sub>2</sub> (HBSS+/+, Gibco). Following removal of the meninges, hippocampi and cortices were dissected and processed separately. Collected tissue was washed 3 times with HBSS without MgCl<sub>2</sub> and CaCl<sub>2</sub> (HBSS-/-, Gibco). After the final wash, tissue was enzymatically digested at 37<span class="math inline">\(^\circ\)</span>C by resuspension in 3.5 ml HBSS-/- and 500 <span class="math inline">\(\mu\)</span>l 0.25% trypsin (Gibco). Periodically during digestion, tube was genlty flicked to facilitate dissociation. Cortical tissue was digested for ~30 minutes while hippocampal tissue required typically 15 minutes. Trypsin digestion was inactivated by addition of 2 ml of heat inactivated Fetal Bovine Serum (FBS, Gibco). Tissue was then washed three times with 10 ml of cold HBSS+/+, before incubation with 100 <span class="math inline">\(\mu\)</span>l of 1mg/ml DNase (Worthington) for 1 minute. Tissue was then washed with 10 ml Neurobasal media (Gibco), centrifuged once at 1200 rpm for 1 minute before an additional wash with Neurobasal growth media (Neurobasal media supplemented with 1X Glutamax, 1% penicilin/streptomycin, 1X B27 (Gibco)). 2 ml of Neurobasal media was left in the tube and tissue was gently dissociated using a 10 ml pipette. Tissue was then carefully titurated 10 times using a P1000 pipette tip and then 10 times using a P200 pipette tip. Cells were then counted using a haemocytometer and plated on coated coverslips at a density of 50,000 cells per well. The day following plating neuronal media was changed to BrainPhys media containing SM1 supplement (StemCell) and cultured in this media henceforth.</p>
</div>
<div id="nucleofection" class="section level4 unnumbered">
<h4><em>Nucleofection</em></h4>
<p>As post-mitotic neurons are infamously difficult to transfect, nucleofection, a method of <em>in vitro</em> electroporation, was used to introduce DNA plasmids into primary post-mitotic neurons. Following dissociation of primary neurons into a single-cell suspension, neurons were nucleofected using a Mouse neuron Nucleofector kit (VPG-1001) with an Amaxa 2b Nucleofection system (Lonza) according to the manufacturer’s instructions. Briefly, this involved mixing a total amount of 1 <span class="math inline">\(\mu\)</span>g of plasmid DNA with 1 million cells in 100 <span class="math inline">\(\mu\)</span>l of provided Nucleofector solution in the nucleocuvette, followed by nucleofection using program O-005. Following nucleofection, neurons were allowed to recover immediately in 1 ml of RPMI medium for 5 minutes at 37<span class="math inline">\(^\circ\)</span>C. Neurons were then resuspended in Neurobasal growth media and plated at the appropriate density. Note: as nucleofection induces death of 40% of cells, plating density should be calculated accordingly. The day following plating neuronal media was changed to BrainPhys media containing SM1 supplement (StemCell) and cultured in this media henceforth.</p>
</div>
</div>
<div id="hek293t-culture-transfection" class="section level3">
<h3><span class="header-section-number">4.3.2</span> HEK293T Culture &amp; Transfection</h3>
<p>Under normal growing conditions, HEK293T cells are cultured in DMEM+Glutamax (Gibco) supplemented with 10% FBS and 1% penicilin/streptomycin. HEK293T cells were seeded in 6 well plates at a density of 0.3X10^6 cells per well for Co-IP, and seeded onto glass coverslips in 24-well plates at a density of 100,000 cells per well for IHC. Plasmid DNA was introduced into cells by chemical transfection using Lipofectamine 2000 or Turbofect (ThermoFisher) according to the manufacturer’s instructions. Briefly, DNA was diluted in a volume of Opti-MEM (ThermoFisher) while transfection reagent was diluted in an equal volume of Opti-MEM and incubated for 5 minutes. Diluted DNA and diluted transfection reagent were then mixed and allowed to incubate for 30 minutes before being applied directly to cells. Cells are then harvested one day following transfection. Cells to be used for IHC are gently rinsed once in PBS and fixed for 10 minutes (1% PFA in PBS). Harvesting protein is described in the following section.</p>
</div>
<div id="co-immunoprecipitation-co-ip-and-western-blot-wb" class="section level3">
<h3><span class="header-section-number">4.3.3</span> Co-Immunoprecipitation (Co-IP) and Western Blot (WB)</h3>
<div id="lysis" class="section level4 unnumbered">
<h4><em>Lysis</em></h4>
<p>To harvest protein from 6 well plate transfections, cells are rinsed once in PBS, then lysed on ice in 300 <span class="math inline">\(\mu\)</span>l of Radioimmunoprecipitation assay buffer (RIPA, 50 mM Tris, pH 8.0, 150 mM NaCl, 1% Triton X-100, 0.5% Sodium Deoxycholate, 0.1% SDS) with 1X Protease Inhibitor Cocktail (PIC, Roche). Given our interest in nuclear proteins, lysates were sonicated for 15 pulses on ice using a probe sonicator. Insoluble cellular debris was then precipitated by centrifugation at 13,000 rpm, 4 <span class="math inline">\(^\circ\)</span>C and lysate was decanted into a new tube.</p>
</div>
<div id="protein-concentration" class="section level4 unnumbered">
<h4><em>Protein Concentration</em></h4>
<p>Protein concentration was measured using a standard Bicinchroninic acid (BCA) assay. BCA assay was carried out by diluting 10 <span class="math inline">\(\mu\)</span>l of protein sample 1:10, and plating 25 <span class="math inline">\(\mu\)</span>l onto a 96 well plate in triplicate. A Bovine Serum Albumin (BSA) standard (0, 0.1, 0.2, 0.5, 0.75, 1.0, 1.5, 2.0 <span class="math inline">\(\mu\)</span>g/<span class="math inline">\(\mu\)</span>l) was also plated. 200 <span class="math inline">\(\mu\)</span>l of working reagent (Reagent A: Reagent B = 1:50) was added to each well and allowed to incubate 20 minutes at 37<span class="math inline">\(^\circ\)</span>C in the dark. BCA assay was then measured on a nanophotometer 1374 plate reader, and protein concentration was determined by regression to the BSA standard curve.</p>
</div>
<div id="co-immunoprecipitation-co-ip" class="section level4 unnumbered">
<h4><em>Co-Immunoprecipitation (Co-IP)</em></h4>
<p>Prior to immunoprecipitation, each sample was adjusted to a 300 <span class="math inline">\(\mu\)</span>l volume at a protein concentration of 1.5 <span class="math inline">\(\mu\)</span>g/<span class="math inline">\(\mu\)</span>l using lysis (RIPA) buffer. The appropriate amount of lysate was also set aside for input. To perform immunoprecipitation, 1.5 <span class="math inline">\(\mu\)</span>l of the appropriate antibody was incubated per sample on a rocker for 2 hours at 4<span class="math inline">\(^\circ\)</span>C. During this time, protein G sepharose beads (GE Healthcare, 25 <span class="math inline">\(\mu\)</span>l per sample) were rinsed 3 x 15 minutes in 1ml of cold TBS (50 mM Tris pH 7.5, 150 mM NaCl), rocking at 4<span class="math inline">\(^\circ\)</span>C. Between washes beads were spun down using a short 5 second spin on a tabletop centrifuge (no greater than 9,000 rpm). Following the 2 hour antibody/lysate incubation, washed beads are added to each sample and incubated on a rocker for an additional hour at 4<span class="math inline">\(\circ\)</span>C. Following IP, beads are washed twice using lysis buffer and twice with TBS. On the final wash, as much buffer was removed as possible before addition of 25 <span class="math inline">\(\mu\)</span>l 2.5X lammeli buffer. IP was then boiled at 95<span class="math inline">\(^\circ\)</span>C for 5 minutes. For input samples, 5 <span class="math inline">\(\mu\)</span>g of total lysate was used in a volume of 25 <span class="math inline">\(\mu\)</span>l, adjusted with 5 <span class="math inline">\(\mu\)</span>l 5X lammeli buffer and the appropriate amount of lysis buffer, and boiled at 95<span class="math inline">\(^\circ\)</span>C for 5 minutes.</p>
</div>
<div id="western-blot" class="section level4 unnumbered">
<h4><em>Western Blot</em></h4>
<p>SDS polyacrylamide gel electrophoresis (SDS-PAGE) was performed to separate and analyse protein samples. The composition of the running gel was as follows:</p>
<blockquote>
<p><em>For two 10% Running gels (20ml):</em><br />
7.78 ml ddH<sub>2</sub>O<br />
6.6 ml 40% Acrylamide/Bisacrylamide 29:1<br />
5.2 ml 1.5M Tris pH 9.1<br />
0.2 ml 10% SDS<br />
0.2 ml 10% Ammonium Persulfate (APS)<br />
16 <span class="math inline">\(\mu\)</span>l Tetramethylethylenediamine (TEMED)</p>
</blockquote>
<p>Running gel was cast, and 1 ml of isopropanol was added to the top of the gel to enable a level gel top. Following running gel polymerization, the stacking gel was prepared and added with loading combs:</p>
<blockquote>
<p><em>For two 5% stacking gels (6.1ml):</em><br />
4 ml ddH<sub>2</sub>O<br />
1.2 ml 40% Acrylamide/Bisacrylamide 29:1<br />
800 <span class="math inline">\(\mu\)</span>l 1.0M Tris pH 6.7<br />
92 <span class="math inline">\(\mu\)</span>l 10% SDS<br />
48 <span class="math inline">\(\mu\)</span>l 10% Ammonium Persulfate (APS)<br />
8 <span class="math inline">\(\mu\)</span>l Tetramethylethylenediamine (TEMED)</p>
</blockquote>
<p>The internal chamber was filled with running buffer (25 mM Tris, 190 mM glycine, 0.1% SDS, pH 8.3) and both marker (PageRuler, Fermentas) and samples were loaded using a Hamilton microliter syringe. Protein samples were run at 60 V until clear of the stacking buffer, and then run at 120 V until finished (typically 90 minutes). Proteins were then transfered onto an Immobilon-P transfer membrane using the wet technique. Membranes were activated by briefly rinsing in methanol and then in transfer buffer (25 mM Tris, 190 mM glycine, 20 % methanol, pH 8.3). A transfer casette was set up by stacking sponges, Whatmann paper, and Polyvinylidene difluoride (PVDF) membrane sandwich. Proteins were transfered for 90 minutes at 100 amps while being packed with ice to minimize resistance.</p>
<p>Following transfer, PVDF membrane was washed three times with TBST (50 mM Tris, 150 mM NaCl, 0.1% Tween 20, pH 7.4) before a one hour incubation with blocking solution (3% Bovine Serum Albumin in TBST), shaking at room temperature. Primary antibodies were then applied to membranes (typically 1:1000 in TBST), shaking overnight at 4<span class="math inline">\(^\circ\)</span>C. The following day, membranes were washed (3 x 15 minutes) in TBST before incubating for 2 hours with HRP-conjugated secondary antibodies (Dianova, 1:20,000 in TBST). Proteins were then visualized using ECL on a BioRad Transluminator using ImageLab v4.0 software. To re-image the same membrane using a different antibody, membranes were stripped with two 10 minute washes in glycine-SDS (15% glycine, 1% SDS, 10% Tween 20, pH 2.2) followed by three 10 minute washes in PBS. Membranes could then be washed in TBST and blocked for incubation with a new antibody.</p>
</div>
</div>
</div>
<div id="in-utero-electroporation-tissue-processing-and-histology" class="section level2">
<h2><span class="header-section-number">4.4</span> In Utero Electroporation, Tissue Processing and Histology</h2>
<div id="IUEmethod" class="section level3">
<h3><span class="header-section-number">4.4.1</span> In Utero Electroporation (IUE)</h3>
<p>In Utero Electroporation was performed according to the initial protocol <span class="citation">(Saito <a href="#ref-Saitovivoelectroporationembryonic2006a">2006</a>)</span> with minor modifications and is described here. All surgical procedures were performed in accordance with LaGeSo experimental licenses G0079/11 and G0206/16.</p>
<p>Animal surgery was performed in a sterile asceptic environment and all instruments were sterilized using 70% ethanol and a Germinator 500 glass bead sterilizer. Prior to surgery, in utero electroporation buffer (IUE buffer; 1 X PBS, pH 7.4, supplemented with 1% penicilin/streptomycin) was warmed to 37<span class="math inline">\(^\circ\)</span>C. Glass capillaries were pulled using a HEK-PIP6 capillary puller and filled with a mixture of 30 <span class="math inline">\(\mu\)</span>l of endotoxin-free plasmid DNA (~2<span class="math inline">\(\mu\)</span>g/<span class="math inline">\(\mu\)</span>l) and 1.5 <span class="math inline">\(\mu\)</span>l filter-sterilized FastGreen.</p>
<p>Induction of anesthesia and maintenence of anesthesia was mediated by a supply of isoflurane (set at 2.5 flow) mixed with medical air supplied at 1 atmosphere of pressure. Following induction of anesthesia in a sealed chamber, pregnant females were transfered to a heating pad where continuous isoflurane was supplied via a fitted nose cone. Bepanthen was applied to the eyes to prevent corneal desiccation during surgery, and 150 <span class="math inline">\(\mu\)</span>l of analgesic (Buprenorphine 300 <span class="math inline">\(\mu\)</span>g/ml, Buscopan 30 <span class="math inline">\(\mu\)</span>g/ml) was injected subcutaneously as analgesic.</p>
<p>Once under stable anesthesia, determined by the pinch test, the abdomen of the mouse was disinfected using iodine spray and wiped clean with 70% ethanol. A midline incision, around 3 cm in length was made in the skin of the abdomen. After freeing adjacent connective fibres, the incision was repeated on the muscle wall of the abdomen, exposing the uterine horns. Embryos were then carefully pulled out of the abdominal cavity using blunt forceps. Periodically during the surgery, embryos were bathed in IUE buffer using a pasteur pipette. To perform the electroporation the embryo was held gently using the operators fingers, and the tip of the capillary containing the plasmid DNA was inserted into the embryo’s lateral ventricle near the brain midline. DNA was then pushed into the ventricle using a picopump, controlled by a foot pedal. After sufficient pulses (visible by the FastGreen turning the ventricle dark green), platinum electrodes were placed around the head of the embryo (+ve electrode to the injection site) to apply an electric pulse using a BTX electroporator. An electroporation program of six 35 mV wave pulses, each 50 ms long, with a 950 ms interval, was applied. Once all accessible embryos are electroporated, the abdominal wall was stitched with thread, and the skin was stapled using wound-clip staples (Autoclip).</p>
<p>Following the surgery the animal was returned to her clean home cage to recover. Post-operative drinking water was changed to contain a final concentration of 25 <span class="math inline">\(\mu\)</span>g/ml Tramadol and 12 <span class="math inline">\(\mu\)</span>g/ml Buscopan for 2 days to prevent abdominal cramps.</p>
</div>
<div id="tissue-fixation-and-perfusion" class="section level3">
<h3><span class="header-section-number">4.4.2</span> Tissue Fixation and Perfusion</h3>
<p>For embryonic tissue, the date of vaginal plug was counted as embryonic day 0.5 (E0.5). Pregnant females at the desired stage were killed by a lethal dose of Avertin or Pentobarbital and death was confirmed via cervical dislocation. Following a midline incision, uterine horns were excised and placed in a dish containing ice-cold PBS where embryonic brains were isolated using microforceps. Brains were then immediately fixed in paraformaldehyde (PFA, 4 % in PBS) overnight at 4<span class="math inline">\(^\circ\)</span>C. Embryos younger than E16 it is preferrable to fix no longer than 4 hours.</p>
<p>For animals older than P6, animals were given a lethal injection of pentobarbital and transcardially perfused with first PBS (10-20ml, depending on age) until the liver was clear, followed by perfusion with 4% PFA (5-20ml, depending on age). The head was then decapitated, skull carefully removed, and brain fixed overnight in 4% PFA at 4<span class="math inline">\(^\circ\)</span>C. If brains were not to be processed immediately, they were transfered to PBS containing 0.001% sodium azide.</p>
</div>
<div id="cryosectioning" class="section level3">
<h3><span class="header-section-number">4.4.3</span> Cryosectioning</h3>
<p>If brains were to be used for in situ hybridization, all solutions were based using ddH<sub>2</sub>O treated with Diethyl pyrocarbonate (Sigma, prepare by shaking 1 ml DEPC in 1 L ddH<sub>2</sub>O at 37<span class="math inline">\(^\circ\)</span>C o/n) and cryosections are the weapon of choice. For the best results cryopreservation should occur immediately after fixation without delay. Following fixation, brains were briefly rinsed 5 minutes in PBS prior to transfering to 15 % sucrose (in DEPC-PBS). Brains are allowed to dehydrate in 15% sucrose for 3 hours or until they sink to the bottom of the well. Brains were then moved to 30% sucrose (in DEPC-PBS) and incubated a minimum of 6 hours, but typically left overnight. Following sucrose dehydration, cerebellum of brains was removed using a razerblade and briefly swilled in Optimal cutting temperature compound (OCT, Tissue-Tek). Brains were then embedded, brainstem down, on in tissue blocks filled with OCT. Tissue blocks were then snap frozen by placing on a brick of dry ice until frozen. Tissue blocks were then stored at -20<span class="math inline">\(^\circ\)</span>C until sectioning. Blocks that require longer term storage were placed at -80<span class="math inline">\(^\circ\)</span>C.</p>
<p>Cryoblocks were manually sectioned with a thickness of 14 <span class="math inline">\(\mu\)</span>m on a cryostat at -20<span class="math inline">\(^\circ\)</span>C and collected on positively charged slides (Superfrost, ThermoFisher). Slides with sections were allowed to air dry for 2 hours before being stored at -20<span class="math inline">\(^\circ\)</span>C. Alternatively, cryoblocks were thrown in the air, and cut latterally with a katana. This yielded random sized sections, but when you are aiming at a mouse brain in the air you get what you get.</p>
</div>
<div id="immuno-histochemistry-ihc" class="section level3">
<h3><span class="header-section-number">4.4.4</span> Immuno Histochemistry (IHC)</h3>
<p>IHC was performed on either cryosections or paraffin sections. For paraffin sections, an antigen retrieval step (Boiling 3 x 5 minutes in Antigen Unmasking Solution, Vector Labs) was performed prior to blocking.</p>
<p>Immunohistochemistry was primarily performed on cryosections. Prior to IHC, paraffin slides are rehydrated as in <a href="4-methods.html#nissl">4.4.8</a> and an antigen retrieval step was performed were the sections were boiled in a citrate buffer (Vectorlabs antigen unmasking solution, 3 boils 5 minutes each). Henceforth the protocol for immunohistochemical staining of paraffin or cyrosections is identical.<br />
To perform IHC, slides were washed 2 x 5 minutes in PBS, and then blocked and permeabilized in blocking solution (2% BSA, 1% Triton X100 in PBS), where slides are covered with a thin piece of parafilm to inhibit dessication. All antibody steps use this same blocking solution as diluent. Primary antibodies were then diluted (typically ~1:200-500) in blocking solution and incubated on the sections at 4<span class="math inline">\(^\circ\)</span>C overnight. The following day, slides were washed 3 x 10 minutes in PBS to remove excess antibody. Sections were then incubated with the appropriate secondary antibody (ie Donkey anti-Rabbit, Donkey anti-rat etc. (Dianova)) diluted 1:500 in blocking buffer for 2 hours at room temperature. If Draq5 far-red nuclear stain was to be used, it was included in the secondary antibody incubation step. Following secondary antibody incubation, slides are rinsed 2 X 5 minutes in PBS, and stained with Hoechst/DAPI (1:5000 in PBS) for 5 minutes at room temperature. When using adult sections, lipofuscin autofluorescence was quenched by a 10 minute incubation with a solution containing 10mM CuS0<sub>4</sub> &amp; 50mM NH<sub>4</sub>Cl. Sections were then mounted aqueously using Immu-Mount (Shandon).</p>
<table class="table table-striped table-hover table-condensed table-responsive" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:primaryantibodies">Table 4.6: </span>Primary Antibodies
</caption>
<thead>
<tr>
<th style="text-align:left;">
Species
</th>
<th style="text-align:left;">
Antibody
</th>
<th style="text-align:left;">
Cat…
</th>
<th style="text-align:left;">
Source
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
rabbit
</td>
<td style="text-align:left;">
anti-HP1<span class="math inline">\(\alpha\)</span> (N term)
</td>
<td style="text-align:left;">
LS-C286285
</td>
<td style="text-align:left;">
LSBio
</td>
</tr>
<tr>
<td style="text-align:left;">
mouse
</td>
<td style="text-align:left;">
anti-HP1<span class="math inline">\(\alpha\)</span>
</td>
<td style="text-align:left;">
MAB3446
</td>
<td style="text-align:left;">
Millipore
</td>
</tr>
<tr>
<td style="text-align:left;">
mouse
</td>
<td style="text-align:left;">
anti-HP1<span class="math inline">\(\beta\)</span>
</td>
<td style="text-align:left;">
MAB3448
</td>
<td style="text-align:left;">
Millipore
</td>
</tr>
<tr>
<td style="text-align:left;">
rat
</td>
<td style="text-align:left;">
anti-HP1<span class="math inline">\(\beta\)</span>
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Self-Made by Singh laboratory
</td>
</tr>
<tr>
<td style="text-align:left;">
mouse
</td>
<td style="text-align:left;">
anti-HP1<span class="math inline">\(\gamma\)</span>
</td>
<td style="text-align:left;">
MAB3450
</td>
<td style="text-align:left;">
Millipore
</td>
</tr>
<tr>
<td style="text-align:left;">
rabbit
</td>
<td style="text-align:left;">
anti-H3K9me3
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Self-Made by Singh laboratory
</td>
</tr>
<tr>
<td style="text-align:left;">
rabbit
</td>
<td style="text-align:left;">
anti-H4K20me3
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Self-Made by Singh laboratory
</td>
</tr>
<tr>
<td style="text-align:left;">
rabbit
</td>
<td style="text-align:left;">
anti-ki67
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Self-Made by Singh laboratory
</td>
</tr>
<tr>
<td style="text-align:left;">
rabbit
</td>
<td style="text-align:left;">
anti-Satb2
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Self-Made by Tarabykin laboratory
</td>
</tr>
<tr>
<td style="text-align:left;">
rat
</td>
<td style="text-align:left;">
anti-Ctip2
</td>
<td style="text-align:left;">
ab18465
</td>
<td style="text-align:left;">
Abcam
</td>
</tr>
<tr>
<td style="text-align:left;">
goat
</td>
<td style="text-align:left;">
anti-GFP
</td>
<td style="text-align:left;">
600-101-215
</td>
<td style="text-align:left;">
Rockland
</td>
</tr>
<tr>
<td style="text-align:left;">
chicken
</td>
<td style="text-align:left;">
anti-GFP
</td>
<td style="text-align:left;">
ab13970
</td>
<td style="text-align:left;">
Abcam
</td>
</tr>
<tr>
<td style="text-align:left;">
goat
</td>
<td style="text-align:left;">
anti-Prox1
</td>
<td style="text-align:left;">
AF2727
</td>
<td style="text-align:left;">
R&amp;amp;D Systems
</td>
</tr>
<tr>
<td style="text-align:left;">
goat
</td>
<td style="text-align:left;">
anti-tdTomato
</td>
<td style="text-align:left;">
AB8181-200
</td>
<td style="text-align:left;">
SICGEN Antibodies
</td>
</tr>
<tr>
<td style="text-align:left;">
rat
</td>
<td style="text-align:left;">
anti-C4 antibody [16D2]
</td>
<td style="text-align:left;">
ab11863
</td>
<td style="text-align:left;">
Abcam
</td>
</tr>
<tr>
<td style="text-align:left;">
mouse
</td>
<td style="text-align:left;">
anti-Myc (9B11)
</td>
<td style="text-align:left;">
2276 S
</td>
<td style="text-align:left;">
Cell Signalling
</td>
</tr>
<tr>
<td style="text-align:left;">
rabbit
</td>
<td style="text-align:left;">
anti-Iba1
</td>
<td style="text-align:left;">
019-19741
</td>
<td style="text-align:left;">
wako
</td>
</tr>
<tr>
<td style="text-align:left;">
mouse
</td>
<td style="text-align:left;">
anti-GFAP
</td>
<td style="text-align:left;">
MAB360
</td>
<td style="text-align:left;">
Millipore
</td>
</tr>
<tr>
<td style="text-align:left;">
goat
</td>
<td style="text-align:left;">
anti-EIF3<span class="math inline">\(\eta\)</span> (N-20)
</td>
<td style="text-align:left;">
sc-16377
</td>
<td style="text-align:left;">
Santa Cruz
</td>
</tr>
</tbody>
</table>

</div>
<div id="quantification-of-cells-in-the-dentate-gyrus" class="section level3">
<h3><span class="header-section-number">4.4.5</span> Quantification of Cells in the Dentate Gyrus</h3>
<p>The number of Prox1+ and Ki-67+ cells in P0 and P8 dentate gyrii were quantified by creating a pipeline in CellProfiler (<a href="http://cellprofiler.org/" class="uri">http://cellprofiler.org/</a>). First, images to be quantified were masked such that only the dentate gyrus was visible, then split into individual files for RGB. Upon file import into cellprofiler, metadata from split-RGB files, each containing DAPI (blue), ki-67 (green) and Prox1 (red). Metadata extraction of each was done by the regular expression:</p>
<div class="sourceCode"><pre class="sourceCode bash"><code class="sourceCode bash">^<span class="kw">(</span><span class="ex">?P</span><span class="op">&lt;</span>Genotype<span class="op">&gt;</span>[A-Z,a-z]<span class="dt">{2,3}</span><span class="kw">)</span><span class="ex">_</span> (?P<span class="op">&lt;</span>animal<span class="op">&gt;</span>[a-z]<span class="dt">{2,3}[0-9]{3}</span>)<span class="ex">_</span>(?P<span class="op">&lt;</span>Slide<span class="op">&gt;</span>.*)<span class="ex">_</span>(?P<span class="op">&lt;</span>Channel<span class="op">&gt;</span>.)<span class="ex">.tif</span></code></pre></div>
<p>Primary objects were identified for each channel using the following parameters: For nuclei, min-diameter 8, max-diameter 14, threshold correction 1.5, distinguishing by shape. For ki67, min-diameter 5, max-diameter 25, threshold correction 1, distinguishing by intensity. Prox1 primary objects were identified using the same settings as nuclei. Primary objects were then related to nuclei, removing false signal, and count was exported to a csv file for analysis in R. To calculate cells/<span class="math inline">\(\mu\)</span>m, the area of the dentate gyrus measured was quantified manually in imageJ (<span class="citation">Schindelin et al. (<a href="#ref-SchindelinFijiopensourceplatform2012">2012</a>)</span>, <a href="https://imagej.nih.gov/ij/" class="uri">https://imagej.nih.gov/ij/</a> ).</p>
</div>
<div id="in-situ-hybridization-ish" class="section level3">
<h3><span class="header-section-number">4.4.6</span> In Situ Hybridization (ISH)</h3>
<p>In situ hybridization was carried out largely in line with the protocol outlined in <span class="citation">(Bormuth et al. <a href="#ref-BormuthNeuronalBasicHelixLoopHelix2013a">2013</a>)</span> and is summarized here. Prior to beginning the protocol, forceps used to handle slides were washed for 30 minutes in 3% H<sub>2</sub>O<sub>2</sub> and glass cuvettes were baked at 200<span class="math inline">\(^\circ\)</span>C for two hours. All buffers used prior to the hybridization step were made using DEPC-ddH<sub>2</sub>O and autoclaved. Gloves are worn throughout the protocol and all washing steps include gentle shaking unless specified otherwise.</p>
<p><em>Day 1</em></p>
<p>Cryosections were allowed to thaw and air dry at room temperature for 20 minutes prior to beginning the protocol. During this time the sections were labelled with a pencil with the RNA probe to be applied to that section. Sections were fixed for 5 minutes (4% PFA in PBS) at room temperature without shaking. After this step PFA is saved for post-fixation step after proteinase K digestion.</p>
<p>Following fixation, slides were washed 2 x 5 minutes with DEPC-PBS. During this time proteinase K buffer (30mM Tris, 10mM EDTA, 0.5% SDS pH 8.0) was pre-warmed to 37<span class="math inline">\(^\circ\)</span>C. Slides were washed once in proteinase K buffer, and then incubated with proteinase K buffer containing 20<span class="math inline">\(\mu\)</span>g/ml proteinase K at 37<span class="math inline">\(^\circ\)</span>C for <em>exactly</em> 2 minutes. Proteinase K digestion needs to be a tightly controlled incubation, as too long will result in overdigested tissue. (If performing a combination ISH/IHC, substitute the proteinase K step with a single wash of 5X SSC pH 7.0).</p>
<p>Proteinase K digestion was inactivated by immediate washing in 0.2% Glycine in DEPC-PBS for 5 minutes at room temperature. Slides were then washed 2 x 5 minutes in PBS prior to post-fixation. Post-fixation was carried out by adding glutaraldehyde (final concentration 0.2%) to the previously used 4% PFA in DEPC-PBS, and incubating for 20 minutes at RT. Slides were then washed twice in DEPC-PBS before pre-hybridization.</p>
<p>Pre-hybridization was carried out by applying 200 <span class="math inline">\(\mu\)</span>l hybmix buffer to each slide and covering with parafilm, sealing in a humid chamber (50% FA 50% ddH<sub>2</sub>O) and incubating at the same temperature to be used for the hybridization (typically 68<span class="math inline">\(^\circ\)</span>C) for 2 hours. Towards the end of the pre-hybridization incubation, RNA probes are diluted in hybmix (10-16 <span class="math inline">\(\mu\)</span>l of probe per 150 <span class="math inline">\(\mu\)</span>l hybmix) and denatured at 65<span class="math inline">\(^\circ\)</span>C for 5 minutes. RNA probes are then kept on ice until being applied to slides. RNA probes are then applied to slides, covered with parafilm, sealed and incubated at 68<span class="math inline">\(^\circ\)</span>C overnight.</p>
<p><em>Day 2</em></p>
<p>The next day, probe solution was discarded and slides are rinsed in 2X SSC pH 4.5 for 5 minutes at RT. Slides were then pre-washed in warmed RNase buffer (15 mM NaCl, 10 mM Tris pH) for 5 minutes. To remove background by digesting un-hybridized RNA, slides were then treated with RNase (20 <span class="math inline">\(^\mu\)</span>g/ml) in RNase buffer at 37<span class="math inline">\(^\circ\)</span>C for 30 minutes.</p>
<p>After RNase treatment, slides were washed with 2X SSC, formamide washes and KTBT (50 mM Tris pH 7.5, 150 mM NaCl, 10 mM KCl, 1% Triton X-100) in the following fashion:</p>
<blockquote>
<p>2X SSC pH 4.5: 5 minutes, RT<br />
(2X SSC, 50% formamide): 30 minutes, 65<span class="math inline">\(^\circ\)</span>C<br />
(2X SSC, 50% formamide): 30 minutes, 65<span class="math inline">\(^\circ\)</span>C<br />
(2X SSC, 50% formamide): 30 minutes, 65<span class="math inline">\(^\circ\)</span>C<br />
KTBT: 10 minutes, RT<br />
KTBT: 10 minutes, RT<br />
KTBT: 10 minutes, RT</p>
</blockquote>
<p>Slides were then incubated in ISH blocking solution (20% sheep serum in KTBT) for 2 hours at room temperature. Following blocking, slides were incubated with anti-DIG alkaline phosphatase antibody (Roche, 1:1000 in ISH blocking solution) overnight at 4<span class="math inline">\(^\circ\)</span>C. If perfoming a combination IHC/ISH, additional primary antibodies were included at this time.</p>
<p><em>Day 3</em></p>
<p>On the final day, sections were washed with KTBT and NT(M)T (100 mM Tris pH 9.5, 100 mM NaCl, 50 mM MgCl<sub>2</sub>, 0.05% Tween-20) in the following fashion:</p>
<blockquote>
<p>KTBT: 5 minutes, RT<br />
KTBT: 5 minutes, RT<br />
KTBT: 5 minutes, RT<br />
KTBT: 30 minutes, RT<br />
KTBT: 30 minutes, RT<br />
KTBT: 30 minutes, RT<br />
(If performing a combination IHC/ISH, slides were incubated here with secondary antibodies in KTBT.)<br />
NT(M)T: 5 minutes, RT<br />
NT(M)T: 5 minutes, RT<br />
NT(M)T: 5 minutes, RT</p>
</blockquote>
<p>After the third NT(M)T wash, slides were developed until stained by the addition of NBT/BCIP substrate (Roche, 1:50 in NT(M)T) or in the case of IHC/ISH, liquid FastRed (Abcam, ab64254). Slides were then washed <em>ad libitum</em> in PBST (1% Triton X-100 in PBS) to clear excess background staining. For fluorescent IHC/ISH slides, sections were washed once in PBS and mounted using Immu-Mount. For normal in situ, slides are then fixed for 5 minutes in 4% PFA in PBS before dehydration by ethanol row and non-aqueous mounting using Entellan (Merck).</p>
</div>
<div id="paraffin-sectioning" class="section level3">
<h3><span class="header-section-number">4.4.7</span> Paraffin Sectioning</h3>
<p>Following fixation, brains were washed (3 x 15 minutes) with PBS and transfered to paraffin cassettes and dehydrated through increasing ethanol concentrations while shaking:</p>
<blockquote>
<p>30% Ethanol: 2 hours<br />
50% Ethanol: 2 hours<br />
70% Ethanol: Overnight<br />
80% Ethanol: 2 hours<br />
90% Ethanol: 2 hours<br />
100% Ethanol: 1 hour<br />
Xylol I: 2 hours<br />
Xylol II: 1 hour</p>
</blockquote>
<p>Following Xylol II, paraffin cassettes were incubated in paraplast I at 60<span class="math inline">\(^\circ\)</span>C overnight, followed by a second incubation in paraplast II for 3 hours. Brains were then cast in paraffin in metal embedding molds and allowed to solidify on the 4<span class="math inline">\(^\circ\)</span>C surface for 1 hour.</p>
<p>For sectioning, a Microtome (Leica) was used, and 14 <span class="math inline">\(\mu\)</span>m thick sections collected in 37<span class="math inline">\(^\circ\)</span>C ddH<sub>2</sub>O. Sections were collected on positively charged slides (Superfrost, ThermoFisher). Following sectioning, slides were incubated overnight at 37<span class="math inline">\(^\circ\)</span>C.</p>
</div>
<div id="nissl" class="section level3">
<h3><span class="header-section-number">4.4.8</span> Nissl Stain</h3>
<p>Cresyl Violet acetate was used to perform the Nissl stain. To make the cresyl violet staining solution, 0.1 g of cresul violet acetate was disolved in 100 ml ddH<sub>2</sub>O, covering with parafilm and stirring overnight at room temperature. The following morning, 10 drops (~0.3ml) of glacial acetic acid was added, stirring 5-10 additional minutes. The cresyl violet staining solution was then filtered to remove any precipitate and stored in a light-protective bottle at room temperature. Prior to any staining, the cresyl violet staining solution was filtered again.</p>
<p>Nissl staining was performed on rehydrated parrafin sections. The following series was used to rehydrate parrafin sections:</p>
<blockquote>
<p>Xylol II: 5 minutes<br />
Xylol I: 5 minutes 100% Ethanol: 3 minutes 100% Ethanol: 3 minutes 95% Ethanol: 5 minutes</p>
</blockquote>
<p>Sections were immediately stained in 0.1% Cresyl Violet for 3-10 minutes. Staining with pre-warmed (37-50<span class="math inline">\(^\circ\)</span>C) Cresyl violet can vastly improve penetration and enhance staining). Sections were then rinsed quickyl in distilled water, removing excess cresyl violet. Stain is then differentiated in 95% ethanol for 2-30 minutes, checking microscopically for optimal staining.</p>
<p>Sections were then dehydrated by two 5 minute washes in 100% ethanol, and cleared in Xylol (2 x 5 minutes) before mounting with entellan (Sigma).</p>
<!-- ### Bielschowsky's Silver Stain -->
<!-- Prior to commencing Bielschowky's stain, glass cuvettes were autoclaved and rinsed thoroughly in 2M HCl followed by extensive rinsing in ddH~2~O. -->
<!-- The following solutions were used for the Bielschowsky's stain: -->
<!-- Developer Stock Solution (Made fresh): -->
<!-- \t 20 ml 37% formaldehyde -->
<!-- \t 0.5 g Citric Acid -->
<!-- \t 2 drops concentrated ammonium hydroxide -->
<!-- \t 100 ml ddH~2~O -->
<!-- Developer Working Solution (Made Fresh immediately before use): -->
<!-- \t 8 drops Developer stock solution -->
<!-- \t 8 drops Concentrated Ammonium hydroxide -->
<!-- \t 50 ml ddH~2~O -->
<!-- 14 $\mu$m thick paraffin sections were dewaxed by two 5 minute changes in Xylol and two 5 minute changes in 100% Ethanol. Following one 5 minute wash in 95% ethanol, secionts were rinsed three times in ddH~2~O.  Sections were placed into pre-warmed (40$^|circ$C) 10 % silver nitrate solution (5% Silver nitrate in ddH~2~O) and allowed to stain for 15 minutes.  Sections were then rinsed three times in distilled water.  To the silver nitrate solution, concentrated ammonium hydroxide was added dropwise until the precipitate formed was _just_ clear.  (Excess ammonia may result in poor impregnation of the fibers or excess precipitate; excess ammonia could be compensated by addition of a few more drops of 10% silver nitrate).  Slides were placed directly back into the ammonia-silver nitrate solution and left to stain for 30 minutes at 40$^\circ$C.  Slides were then placed into the developer working solution for 30-60 seconds, as reaction was held under microscopic observation.  To halt the reaction, slides were placed in 1% ammonium hydroxide solution for 1 minute.  Slides were then rinsed three times in ddH~2~O before incubating 5 minutes in 5% sodium thiosulfate (5 g sodium thiosulphate in 100 ml ddH~2~O).  Slides were then rinsed three times in ddH~2~O before dehydration by ethanol row and Xylol, followed by mounting with entellan (Sigma). -->
</div>
<div id="golgi-impregnation-and-dendritic-complexity-analysis" class="section level3">
<h3><span class="header-section-number">4.4.9</span> Golgi Impregnation and Dendritic Complexity Analysis</h3>
<p>Fresh brain samples from WT, HP1<span class="math inline">\(\beta\)</span><sup><em>f/f</em></sup>Emx1<sup><em>Cre</em></sup>, HP1<span class="math inline">\(\gamma\)</span><sup><em>f/f</em></sup>Emx1<sup><em>Cre</em></sup> and HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span><sup><em>f/f</em></sup>Emx1<sup><em>Cre</em></sup> of young and aged mice were cut into two hemispheres and impregnated in Golgi-Cox solution for 2 weeks as described in <span class="citation">(Zaqout and Kaindl <a href="#ref-ZaqoutGolgiCoxStainingStep2016">2016</a>)</span>. Sholl analysis <span class="citation">(Sholl <a href="#ref-ShollDendriticorganizationneurons1953">1953</a>)</span> was performed on CA3 hippocampal neurons to assess the dendritic complexity for both apical and basal dendrites. Concentric circles and cell counter plug-ins available for ImageJ were used to quantify the total intersections number of the dendritic tree with 30 10-<span class="math inline">\(\mu\)</span>m spaced concentric circles. Simple neurite tracer plug-in available for Fiji/ImageJ was used to draw representative neurons.</p>
</div>
</div>
<div id="behavioral-experiments" class="section level2">
<h2><span class="header-section-number">4.5</span> Behavioral Experiments</h2>
<p>All behavioral experiments were undertaken with the support of the Animal Outcome Core Facility (AOCF) in house at the Charité Cross Over building (CCO). Behavioral tests were performed within the guidelines granted by the Landesamt für Gedundheit und Soziales Berlin (LaGeSo) under an extension to experimental license G0079/11 and G0206/16. Prior to behavioral testing, male HP1FEC mice were implanted with subdermal RFID transponders to ensure accurate identification. Behavioral experiments were performed at two different time points. The ‘Young’ time point (2.5 - 3 Months) and the ‘Aged’ time point (13-14 Months). Prior to each cohort of behavioral testing, all animals were subjected to a modified <strong>S</strong>mithKline Beecham, <strong>H</strong>arwell, <strong>I</strong>mperial College, <strong>R</strong>oyal London Hospital, <strong>p</strong>henotype <strong>a</strong>ssessment (SHIRPA), which ensured animals did not have any gross deficits in vision, audition, grip strength, pina reflex and normal exploratory locomotion.</p>
<p>Following SHIRPA assessment on animals in any given cohort, behavioral tests were always carried out at the same time of day, with tests occurring in the following order:</p>
<ul>
<li>Open Field activity</li>
<li>Pre-Pulse Inhibition</li>
<li>Context-Cued Fear Conditioning</li>
<li>Barnes Maze</li>
<li>Social Activity Monitor (SAM)</li>
<li>HomeCageScan</li>
<li>Next Construction</li>
</ul>
<div id="open-field-activity-1" class="section level3">
<h3><span class="header-section-number">4.5.1</span> Open Field Activity</h3>
<p>The open field test is a common test of general activity and exploratory behavior. While movement is the primary metric, this can be influenced by motor output, exploratory drive, circadian cycle, and freezing or fear-related behaviors. The Open Field test was performed largely according to reccomendations by <span class="citation">(D. Gould <a href="#ref-Gouldbook2009">2009</a>)</span>.</p>
<p>To conduct the Open Field Test, animals were placed in the centre of a square enclosure while an overhead camera tracks movement for 10 minutes using the Biobserve Viewer Software. Following a single trial, feces and urine was quantified and the box rigourously cleaned with 5% ethanol. Activity in the ‘centre zone’ and activity in the periphery was binned per minute and exported for later analysis in R.</p>
</div>
<div id="paired-pulse-inhibition-1" class="section level3">
<h3><span class="header-section-number">4.5.2</span> Paired-Pulse Inhibition</h3>
<p>Many organisms display an innate reflexive reaction to an intense unexpected acoustic stimulus, which is typically termed the ‘acoustic startle response’ or ‘startle reflex’. In rodents this is manifested as a rapid muscle contraction resembling a ‘flinch-like’ action. A test for paired-pulse inhibition involves measuring how a weaker pre-stimulus (pre-pulse) diminishes the reaction to the intense stimulus (startle pulse). The normal response is that larger and larger pre-pulses result in smaller startle responses. This reduction in amplitude of startle response reflects temporary adaptations afforded by the nervous system. Deficits of pre-pulse inhibition are thought to be linked to abnormalities of sensorimotor gating. Abrogations in PPI are most commonly observed in schizophrenia <span class="citation">(Braff, Geyer, and Swerdlow <a href="#ref-BraffHumanstudiesprepulse2001">2001</a>)</span>, but have also been observed in neurodegenerative diseases such as Parkinson’s <span class="citation">(Zoetmulder et al. <a href="#ref-ZoetmulderPrepulseInhibitionAssociated2014">2014</a>)</span> and Alzheimer’s disease <span class="citation">(Ueki et al. <a href="#ref-UekiPrepulseinhibitionacoustic2006">2006</a>)</span>.</p>
<p>Paired-pulse inhibition was measured two animals at a time in a 2-box startle box apparatus (TSE Systems), which consisted of black soundproofed plexiglass boxes (49cm x 49cm x 49cm). Metal bars on the floors of internal cages allow for precise measurement of startle response via built-in pressure sensors. Background noise was set at 65dB. Upon program start, animals acclimatized for 5 minutes, followed by a program that randomized by trial induction of a startle pulse alone (120dB, for 40ms), or the startle pulse that had been preceded by a pre-pulse (either 69dB, 73dB or 81dB for 20ms). Following testing data was exported for later analysis in R.</p>
</div>
<div id="context-cued-fear-conditioning" class="section level3">
<h3><span class="header-section-number">4.5.3</span> Context-Cued Fear Conditioning</h3>
<p>To measure context and cue dependent memory, mice were analysed two at a time in two adjacent multi-conditioning boxes from TSE Systems. The fear learning boxes are designed to be sound-proofed and have a sound probe, camera, and an context arena with metal-grated floor and walls high enough to contain the animal. Context cued fear learning is tested over 5 phases:</p>
<p><em>Phase 1 (Shock):</em> Animals are allowed to acclimatize in the chamber for 4 minutes (Context), which is followed by a 30 second sound (Cue), which is immediately followed by a 2 second weak electric shock on the floor of the context arena.</p>
<p><em>Phase 2 (Context 24h):</em> The following day (24h later), the animals are placed back into the context arena and freezing behavior was recorded for 3 minutes.</p>
<p><em>Phase 3 (Cue 24h):</em> Two hours following the Context 24h test, a cover is placed over the metal floor of the context arena and animals are put back under observation. Following a 3 minute habituation, the cue sound is played for 3 minutes and freezing behavior is recorded.</p>
<p><em>Phase 4 (Context 7 Days):</em> Exactly as in <em>phase 2</em>, context is tested 7 days following <em>phase 1</em>.</p>
<p><em>Phase 5 (Cue 7 Days):</em> Two hours following the Context 7 day test, animals are tested for cue, exactly as done in <em>phase 3</em>.</p>
<p>Following completion of the Fear Learning paradigm, automated freezes and freeze intervals measured by TSE System software was cross-referenced with the obtained videos. Freezes measured by the software with duration smaller than 2 seconds were removed and data was plotted in R.</p>
</div>
<div id="Barnesmeth" class="section level3">
<h3><span class="header-section-number">4.5.4</span> Barnes Maze</h3>
<p>Similar to the Morris Water Maze, the Barnes Maze evaluates visual-spatial learning and memory in rodents <span class="citation">(Barnes <a href="#ref-BarnesMemorydeficitsassociated1979">1979</a>)</span>. This is corroborated by many studies where rodents with hippocampal damage show markedly impaired performance in the test <span class="citation">(Deacon and Rawlins <a href="#ref-DeaconLearningimpairmentshippocampallesioned2002">2002</a>; Fox et al. <a href="#ref-FoxEffectTraumaticBrain1998">1998</a>; Harrison et al. <a href="#ref-HarrisonSpatialnonspatialescape2006">2006</a>)</span>. It carries advantages over the Morris Water Maze in that no strong aversive stimuli or deprivation is utilized, rather a weak averzive stimulus (bright light, sound and open area) is utilized. It is also a lot cleaner. In our case, Barnes Maze was performed with short-term and long-term reference memory in mind <span class="citation">(Sunyer et al. <a href="#ref-SunyerBarnesmazeuseful2007">2007</a>)</span>. Both training and testing of the Barnes maze was recorded using Biobserve Viewer software with the Barnes maze plugin.</p>
<p>The Barnes maze consists of a circular platform that contains 20 holes around the circumference of the platform. Only one hole contains a submerged nest to serve as an escape from the open environment. To perform the Barnes Maze test, animals are trained on the location of the submerged test for the first 4 days. This involves similar conditions to the test paradigm, where the mouse is placed in the centre of the platform and loud static noise is played. Mice were trained to find the hidden nest (target hole) over four 3-minute trials over 4 training days. Once the mouse finds the hole the loud static noise is immediately discontinued. If the mouse does not find the nest by the end of the 3 minute trial, it is shown the hole by the experimenter. Once the mouse is in the nest it is allowed to remain there for 30 seconds to allow positive reinforcement. Following 4 days of training, the 5th day involves the 24h test, where mice are placed in the centre of the platform and their hole seeking behavior is recorded in terms of time to the target hole, primary errors (errors before target hole), and the number of head pokes in each hole.</p>
</div>
<div id="social-activity-monitor-sam-1" class="section level3">
<h3><span class="header-section-number">4.5.5</span> Social Activity Monitor (SAM)</h3>
<p>To measure basic interaction and circadian activity, animals in their home cages were placed on top of RF sensors that detect motion where animal activity was recorded for 14 days using Phenosoft control software. SAM data was binned by hour and following data export, RFIDs were decoded to corresponding animal ids and genotypes.</p>
</div>
<div id="micro-behaviors-homecagescan" class="section level3">
<h3><span class="header-section-number">4.5.6</span> Micro Behaviors (HomeCageScan)</h3>
<p>To examine any changes in stereotyped murine behavior, mice were recorded individually over 24hrs using CleverSys software. Prior to recording, background cage registration, night/day and transition calibrations were set according to each cage. For data export, data was binned by minute and by hour. Individual animal data was then combined into one large dataframe for analysis in R.</p>
</div>
<div id="nestcomethod" class="section level3">
<h3><span class="header-section-number">4.5.7</span> Nest Construction</h3>
<p>A behavior exhibited solely by male mice is their innate desire to construct nests. This behaviour can be easily pertubed by distress, pain and cognitive distruptions <span class="citation">(Deacon <a href="#ref-DeaconAssessingBurrowingNest2012">2012</a>)</span>. To test nest construction ability, mice were housed individually and given a square piece of densely woven cotton called a ‘nestlet’ (Ancare). The first 24hrs (Habituation) allowed the animal to become familiar with the piece of cotton as a nesting material. Prior to the second 24hrs (Nest Construction Test), the cotton was removed and replaced with a new cotton square, which was weighed prior to adding it to the cage. The following morning, what remained of the cotton square was weighed, and the complexity of the nest was given a score of 1-5 based on criteria from <span class="citation">(Deacon <a href="#ref-DeaconAssessingBurrowingNest2012">2012</a>)</span>:</p>
<blockquote>
<ol style="list-style-type: decimal">
<li>The Nestlet is largely untouched (&gt;90% intact).</li>
<li>The Nestlet is partially torn up (50-90% remaining intact).</li>
<li>The Nestlet is mostly shredded but often there is no identifiable nest site: &lt; 50% of the Nestlet remains intact but &lt; 90% is within a quarter of the cage floor area, i.e. the cotton is not gathered into a nest but spread around the cage. Note: the material may sometimes be in a broadly defined nest area but the critical definition is that 50-90% has been shredded.</li>
<li>An identifiable, but flat nest: &gt; 90% of the Nestlet is torn up, the material is gathered into a nest within a quarter of the cage floor area, but the nest is flat, with walls higher than mouse body height (curled up on its side) on less than 50% of its circumference.</li>
<li>A (near) perfect nest: &gt; 90% of the Nestlet is torn up, the nest is a crater, with walls higher than mouse body height on more than 50% of its circumference</li>
</ol>
</blockquote>
</div>
</div>
<div id="magnetic-resonance-imaging-mri-and-volumetrics" class="section level2">
<h2><span class="header-section-number">4.6</span> Magnetic Resonance Imaging (MRI) and Volumetrics</h2>
<p>MRI was performed at a 7 Tesla rodent scanner (Pharmascan 70 ⁄ 16, Bruker BioSpin, Ettlingen, Germany) with a 20mm - 1H-RF quadratur-volume resonator. For imaging the mouse brain a T2-weighted 2D turbo spin-echo sequence was used (imaging parameters TR / TE = 5505 / 36 ms, rare factor 8, 6 averages, 46 axial slices with a slice thickness of 0.350 mm, field of view of 2.56 x 2.56 cm, matrix size 256 x 256; scan time 13m12s). With an in-house developed MATLAB toolbox MRI data were registered on the Allen mouse brain atlas (ABA). The volumes of each single ABA brain structure were calculated fully automated followed by label based statistics for defined regions <span class="citation">(Koch et al. <a href="#ref-KochAtlasregistrationedemacorrected2017">2017</a>)</span>. For section-wise analysis, the mean isocortex volume per section per genotype per age was calculated and tested using two-way ANOVA.</p>
</div>
<div id="reduced-read-bisulfite-sequencing-rrbseq" class="section level2">
<h2><span class="header-section-number">4.7</span> Reduced Read Bisulfite Sequencing (RRBSeq)</h2>
<p>RRBS-seq was carried out following a previously described method <span class="citation">(Boyle et al. <a href="#ref-BoyleGelfreemultiplexedreduced2012">2012</a>)</span> with minor modifications. 500 ng of genomic DNA was used as a starting material. Bisulfite conversion was done by the EZ DNA Methylation Gold Kit (Zymo Research, #D5005) with 50 ng of DNA, per sample. 2x KAPA HiFi Hot Start Uracil+ Ready Mix (KAPA Biosystems, #KK2801) was utilized for library amplification. PCR amplification was done for 10 cycles.</p>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-GorskiCorticalExcitatoryNeurons2002">
<p>Gorski, Jessica A., Tiffany Talley, Mengsheng Qiu, Luis Puelles, John L. R. Rubenstein, and Kevin R. Jones. 2002. “Cortical Excitatory Neurons and Glia, but Not GABAergic Neurons, Are Produced in the Emx1-Expressing Lineage.” <em>Journal of Neuroscience</em> 22 (15): 6309–14. doi:<a href="https://doi.org/10.1523/JNEUROSCI.22-15-06309.2002">10.1523/JNEUROSCI.22-15-06309.2002</a>.</p>
</div>
<div id="ref-Saitovivoelectroporationembryonic2006a">
<p>Saito, Tetsuichiro. 2006. “In Vivo Electroporation in the Embryonic Mouse Central Nervous System.” <em>Nature Protocols</em> 1 (November): 1552. <a href="http://dx.doi.org/10.1038/nprot.2006.276" class="uri">http://dx.doi.org/10.1038/nprot.2006.276</a>.</p>
</div>
<div id="ref-SchindelinFijiopensourceplatform2012">
<p>Schindelin, Johannes, Ignacio Arganda-Carreras, Erwin Frise, Verena Kaynig, Mark Longair, Tobias Pietzsch, Stephan Preibisch, et al. 2012. “Fiji: An Open-Source Platform for Biological-Image Analysis.” <em>Nature Methods</em> 9 (7): 676–82. doi:<a href="https://doi.org/10.1038/nmeth.2019">10.1038/nmeth.2019</a>.</p>
</div>
<div id="ref-BormuthNeuronalBasicHelixLoopHelix2013a">
<p>Bormuth, I., K. Yan, T. Yonemasu, M. Gummert, M. Zhang, S. Wichert, O. Grishina, et al. 2013. “Neuronal Basic Helix-Loop-Helix Proteins Neurod2/6 Regulate Cortical Commissure Formation Before Midline Interactions.” <em>Journal of Neuroscience</em> 33 (2): 641–51. doi:<a href="https://doi.org/10.1523/JNEUROSCI.0899-12.2013">10.1523/JNEUROSCI.0899-12.2013</a>.</p>
</div>
<div id="ref-ZaqoutGolgiCoxStainingStep2016">
<p>Zaqout, Sami, and Angela M. Kaindl. 2016. “Golgi-Cox Staining Step by Step.” <em>Frontiers in Neuroanatomy</em> 10 (March). doi:<a href="https://doi.org/10.3389/fnana.2016.00038">10.3389/fnana.2016.00038</a>.</p>
</div>
<div id="ref-ShollDendriticorganizationneurons1953">
<p>Sholl, D. A. 1953. “Dendritic Organization in the Neurons of the Visual and Motor Cortices of the Cat.” <em>Journal of Anatomy</em> 87 (Pt 4): 387–406.1. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1244622/" class="uri">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1244622/</a>.</p>
</div>
<div id="ref-Gouldbook2009">
<p>D. Gould, Todd. 2009. <em>Mood and Anxiety Related Phenotypes in Mice: Characterization Using Behavioral Tests</em>.</p>
</div>
<div id="ref-BraffHumanstudiesprepulse2001">
<p>Braff, D. L., M. A. Geyer, and N. R. Swerdlow. 2001. “Human Studies of Prepulse Inhibition of Startle: Normal Subjects, Patient Groups, and Pharmacological Studies.” <em>Psychopharmacology</em> 156 (2-3): 234–58.</p>
</div>
<div id="ref-ZoetmulderPrepulseInhibitionAssociated2014">
<p>Zoetmulder, Marielle, Heidi B. Biernat, Miki Nikolic, Lise Korbo, Lars Friberg, and Poul J. Jennum. 2014. “Prepulse Inhibition Is Associated with Attention, Processing Speed, and 123i-FP-CIT SPECT in Parkinson’s Disease.” <em>Journal of Parkinson’s Disease</em> 4 (1): 77–87. doi:<a href="https://doi.org/10.3233/JPD-130307">10.3233/JPD-130307</a>.</p>
</div>
<div id="ref-UekiPrepulseinhibitionacoustic2006">
<p>Ueki, Akinori, Kyoko Goto, Noriko Sato, Hiroyuki Iso, and Yoshio Morita. 2006. “Prepulse Inhibition of Acoustic Startle Response in Mild Cognitive Impairment and Mild Dementia of Alzheimer Type.” <em>Psychiatry and Clinical Neurosciences</em> 60 (1): 55–62. doi:<a href="https://doi.org/10.1111/j.1440-1819.2006.01460.x">10.1111/j.1440-1819.2006.01460.x</a>.</p>
</div>
<div id="ref-BarnesMemorydeficitsassociated1979">
<p>Barnes, C. A. 1979. “Memory Deficits Associated with Senescence: A Neurophysiological and Behavioral Study in the Rat.” <em>Journal of Comparative and Physiological Psychology</em> 93 (1): 74–104. doi:<a href="https://doi.org/10.1037/h0077579">10.1037/h0077579</a>.</p>
</div>
<div id="ref-DeaconLearningimpairmentshippocampallesioned2002">
<p>Deacon, Robert M. J., and J. Nicholas P. Rawlins. 2002. “Learning Impairments of Hippocampal-Lesioned Mice in a Paddling Pool.” <em>Behavioral Neuroscience</em> 116 (3): 472–78. doi:<a href="https://doi.org/10.1037/0735-7044.116.3.472">10.1037/0735-7044.116.3.472</a>.</p>
</div>
<div id="ref-FoxEffectTraumaticBrain1998">
<p>Fox, Gerard B., Lei Fan, Ryan A. LeVASSEUR, and Alan I. Faden. 1998. “Effect of Traumatic Brain Injury on Mouse Spatial and Nonspatial Learning in the Barnes Circular Maze.” <em>Journal of Neurotrauma</em> 15 (12): 1037–46. doi:<a href="https://doi.org/10.1089/neu.1998.15.1037">10.1089/neu.1998.15.1037</a>.</p>
</div>
<div id="ref-HarrisonSpatialnonspatialescape2006">
<p>Harrison, Fiona E, Randall S Reiserer, Andrew J Tomarken, and Michael P McDonald. 2006. “Spatial and Nonspatial Escape Strategies in the Barnes Maze.” <em>Learning &amp; Memory</em> 13 (6): 809–19. doi:<a href="https://doi.org/10.1101/lm.334306">10.1101/lm.334306</a>.</p>
</div>
<div id="ref-SunyerBarnesmazeuseful2007">
<p>Sunyer, Berta, Sudarshan Patil, Harald Höger, and Gert Lubec. 2007. “Barnes Maze, a Useful Task to Assess Spatial Reference Memory in the Mice.” <em>Nature Protocols</em>. October 4. <a href="http://dx.doi.org/10.1038/nprot.2007.390" class="uri">http://dx.doi.org/10.1038/nprot.2007.390</a>.</p>
</div>
<div id="ref-DeaconAssessingBurrowingNest2012">
<p>Deacon, Robert. 2012. “Assessing Burrowing, Nest Construction, and Hoarding in Mice.” <em>Journal of Visualized Experiments : JoVE</em>, no. 59 (January). doi:<a href="https://doi.org/10.3791/2607">10.3791/2607</a>.</p>
</div>
<div id="ref-KochAtlasregistrationedemacorrected2017">
<p>Koch, Stefan, Susanne Mueller, Marco Foddis, Thomas Bienert, Dominik von Elverfeldt, Felix Knab, Tracy D. Farr, et al. 2017. “Atlas Registration for Edema-Corrected MRI Lesion Volume in Mouse Stroke Models:” <em>Journal of Cerebral Blood Flow &amp; Metabolism</em>, August. doi:<a href="https://doi.org/10.1177/0271678X17726635">10.1177/0271678X17726635</a>.</p>
</div>
<div id="ref-BoyleGelfreemultiplexedreduced2012">
<p>Boyle, Patrick, Kendell Clement, Hongcang Gu, Zachary D. Smith, Michael Ziller, Jennifer L. Fostel, Laurie Holmes, et al. 2012. “Gel-Free Multiplexed Reduced Representation Bisulfite Sequencing for Large-Scale DNA Methylation Profiling.” <em>Genome Biology</em> 13 (10): R92. doi:<a href="https://doi.org/10.1186/gb-2012-13-10-r92">10.1186/gb-2012-13-10-r92</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="3-discussion.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="5-supplemental-data.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"download": null,
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
