<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Part 6 Bioinformatics &amp; Data Processing | HP1 Deficiency Results in Accelerated Neurodegeneration</title>
  <meta name="description" content="Part 6 Bioinformatics &amp; Data Processing | HP1 Deficiency Results in Accelerated Neurodegeneration">
  <meta name="generator" content="bookdown  and GitBook 2.6.7">

  <meta property="og:title" content="Part 6 Bioinformatics &amp; Data Processing | HP1 Deficiency Results in Accelerated Neurodegeneration" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Part 6 Bioinformatics &amp; Data Processing | HP1 Deficiency Results in Accelerated Neurodegeneration" />
  
  
  

<meta name="author" content="Andrew G. Newman">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="5-supplemental-data.html">
<link rel="next" href="7-references.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />







<script src="libs/kePrint-0.0.1/kePrint.js"></script>


<style type="text/css">
div.sourceCode { overflow-x: auto; }
table.sourceCode, tr.sourceCode, td.lineNumbers, td.sourceCode {
  margin: 0; padding: 0; vertical-align: baseline; border: none; }
table.sourceCode { width: 100%; line-height: 100%; }
td.lineNumbers { text-align: right; padding-right: 4px; padding-left: 4px; color: #aaaaaa; border-right: 1px solid #aaaaaa; }
td.sourceCode { padding-left: 5px; }
code > span.kw { color: #007020; font-weight: bold; } /* Keyword */
code > span.dt { color: #902000; } /* DataType */
code > span.dv { color: #40a070; } /* DecVal */
code > span.bn { color: #40a070; } /* BaseN */
code > span.fl { color: #40a070; } /* Float */
code > span.ch { color: #4070a0; } /* Char */
code > span.st { color: #4070a0; } /* String */
code > span.co { color: #60a0b0; font-style: italic; } /* Comment */
code > span.ot { color: #007020; } /* Other */
code > span.al { color: #ff0000; font-weight: bold; } /* Alert */
code > span.fu { color: #06287e; } /* Function */
code > span.er { color: #ff0000; font-weight: bold; } /* Error */
code > span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
code > span.cn { color: #880000; } /* Constant */
code > span.sc { color: #4070a0; } /* SpecialChar */
code > span.vs { color: #4070a0; } /* VerbatimString */
code > span.ss { color: #bb6688; } /* SpecialString */
code > span.im { } /* Import */
code > span.va { color: #19177c; } /* Variable */
code > span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code > span.op { color: #666666; } /* Operator */
code > span.bu { } /* BuiltIn */
code > span.ex { } /* Extension */
code > span.pp { color: #bc7a00; } /* Preprocessor */
code > span.at { color: #7d9029; } /* Attribute */
code > span.do { color: #ba2121; font-style: italic; } /* Documentation */
code > span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code > span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code > span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
</style>

</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="1" data-path="1-introduction.html"><a href="1-introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="1-introduction.html"><a href="1-introduction.html#epigenetics"><i class="fa fa-check"></i><b>1.1</b> Epigenetics</a><ul>
<li><a href="1-introduction.html#dna-modifications"><em>DNA Modifications</em></a></li>
<li><a href="1-introduction.html#chromatin"><em>Chromatin</em></a></li>
<li><a href="1-introduction.html#chromatin-remodelling"><em>Chromatin Remodelling</em></a></li>
<li><a href="1-introduction.html#krab-zfps-transposable-elements-and-exaptation"><em>KRAB-ZFPs, Transposable Elements and Exaptation</em></a></li>
<li><a href="1-introduction.html#heterochromatinization-and-hp1"><em>Heterochromatinization and HP1</em></a></li>
<li><a href="1-introduction.html#non-coding-rna"><em>Non-Coding RNA</em></a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="1-introduction.html"><a href="1-introduction.html#the-heterochromatin-loss-model-of-aging"><i class="fa fa-check"></i><b>1.2</b> The Heterochromatin Loss Model of Aging</a></li>
<li class="chapter" data-level="1.3" data-path="1-introduction.html"><a href="1-introduction.html#the-aging-neuronal-epigenome"><i class="fa fa-check"></i><b>1.3</b> The Aging Neuronal Epigenome</a><ul>
<li class="chapter" data-level="1.3.1" data-path="1-introduction.html"><a href="1-introduction.html#DNAging"><i class="fa fa-check"></i><b>1.3.1</b> <em>DNA Methylation and Hydroxymethylation in Aged Neurons</em></a></li>
<li class="chapter" data-level="1.3.2" data-path="1-introduction.html"><a href="1-introduction.html#chraging"><i class="fa fa-check"></i><b>1.3.2</b> <em>Chromatin in the Aging Brain</em></a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="1-introduction.html"><a href="1-introduction.html#objective"><i class="fa fa-check"></i><b>1.4</b> Objective</a></li>
<li class="chapter" data-level="1.5" data-path="1-introduction.html"><a href="1-introduction.html#hp1-proteins---a-background"><i class="fa fa-check"></i><b>1.5</b> HP1 Proteins - A Background</a><ul>
<li><a href="1-introduction.html#the-abgs-of-hp1-proteins">The <em>ABG’s</em> of HP1 Proteins</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-results.html"><a href="2-results.html"><i class="fa fa-check"></i><b>2</b> Results</a><ul>
<li class="chapter" data-level="2.1" data-path="2-results.html"><a href="2-results.html#HP1aT"><i class="fa fa-check"></i><b>2.1</b> HP1FEC, HP1TKO and HP1<span class="math inline">\(\alpha\)</span>T</a></li>
<li class="chapter" data-level="2.2" data-path="2-results.html"><a href="2-results.html#expression-of-hp1-proteins-in-the-brain"><i class="fa fa-check"></i><b>2.2</b> Expression of HP1 Proteins in the Brain</a></li>
<li class="chapter" data-level="2.3" data-path="2-results.html"><a href="2-results.html#mosaic-deletion-of-hp1beta-and-hp1gamma-from-cortical-progenitors"><i class="fa fa-check"></i><b>2.3</b> Mosaic Deletion of HP1<span class="math inline">\(\beta\)</span> and HP1<span class="math inline">\(\gamma\)</span> from Cortical Progenitors</a></li>
<li class="chapter" data-level="2.4" data-path="2-results.html"><a href="2-results.html#deletion-of-hp1beta-hp1gamma-from-the-emx1-lineage."><i class="fa fa-check"></i><b>2.4</b> Deletion of <em>HP1<span class="math inline">\(\beta\)</span></em> &amp; <em>HP1<span class="math inline">\(\gamma\)</span></em> from the <em>Emx1</em>-lineage.</a><ul>
<li class="chapter" data-level="2.4.1" data-path="2-results.html"><a href="2-results.html#cell-fate-in-cortical-neurons"><i class="fa fa-check"></i><b>2.4.1</b> Cell Fate in Cortical Neurons</a></li>
<li class="chapter" data-level="2.4.2" data-path="2-results.html"><a href="2-results.html#imprint"><i class="fa fa-check"></i><b>2.4.2</b> Imprinted Loci</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="2-results.html"><a href="2-results.html#perturbed-regulation-of-retroelements-in-hp1-mutants"><i class="fa fa-check"></i><b>2.5</b> Perturbed Regulation of Retroelements in HP1 Mutants</a><ul>
<li class="chapter" data-level="2.5.1" data-path="2-results.html"><a href="2-results.html#line1"><i class="fa fa-check"></i><b>2.5.1</b> LINE1</a></li>
<li class="chapter" data-level="2.5.2" data-path="2-results.html"><a href="2-results.html#class-iii-ervs"><i class="fa fa-check"></i><b>2.5.2</b> Class III ERVs</a></li>
<li class="chapter" data-level="2.5.3" data-path="2-results.html"><a href="2-results.html#class-ii-ervs"><i class="fa fa-check"></i><b>2.5.3</b> Class II ERVs</a></li>
<li class="chapter" data-level="2.5.4" data-path="2-results.html"><a href="2-results.html#de-novo-retrotransposition"><i class="fa fa-check"></i><b>2.5.4</b> <em>De Novo</em> Retrotransposition</a></li>
</ul></li>
<li class="chapter" data-level="2.6" data-path="2-results.html"><a href="2-results.html#sustained-hp1betagamma-deficiency-adversely-affects-neuronal-complexity-function-and-cognition"><i class="fa fa-check"></i><b>2.6</b> Sustained HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> Deficiency Adversely Affects Neuronal Complexity, Function and Cognition</a><ul>
<li class="chapter" data-level="2.6.1" data-path="2-results.html"><a href="2-results.html#behavior"><i class="fa fa-check"></i><b>2.6.1</b> Behavior</a></li>
<li class="chapter" data-level="2.6.2" data-path="2-results.html"><a href="2-results.html#MRIresults"><i class="fa fa-check"></i><b>2.6.2</b> Structure, Volumetrics and Histology</a></li>
</ul></li>
<li class="chapter" data-level="2.7" data-path="2-results.html"><a href="2-results.html#hp1beta-and-hp1gamma-synergistically-contribute-to-repeat-silencing"><i class="fa fa-check"></i><b>2.7</b> HP1<span class="math inline">\(\beta\)</span> and HP1<span class="math inline">\(\gamma\)</span> Synergistically Contribute to Repeat Silencing</a><ul>
<li class="chapter" data-level="2.7.1" data-path="2-results.html"><a href="2-results.html#hp1gamma-is-required-for-h4k20me3-deposition-in-neurons"><i class="fa fa-check"></i><b>2.7.1</b> HP1<span class="math inline">\(\gamma\)</span> is Required for H4K20me3 Deposition in Neurons</a></li>
<li class="chapter" data-level="2.7.2" data-path="2-results.html"><a href="2-results.html#hp1gamma-binds-suv420h2-via-the-chromoshadow-domain"><i class="fa fa-check"></i><b>2.7.2</b> HP1<span class="math inline">\(\gamma\)</span> binds SUV420H2 via the Chromoshadow Domain</a></li>
<li class="chapter" data-level="2.7.3" data-path="2-results.html"><a href="2-results.html#re-addition-of-hp1gamma-can-restore-h4k20me3-in-vivo"><i class="fa fa-check"></i><b>2.7.3</b> Re-addition of HP1<span class="math inline">\(\gamma\)</span> can restore H4K20me3 In Vivo</a></li>
<li class="chapter" data-level="2.7.4" data-path="2-results.html"><a href="2-results.html#h4k20me3-loss-is-correlated-with-iap-de-repression"><i class="fa fa-check"></i><b>2.7.4</b> H4K20me3 Loss is Correlated with IAP De-repression</a></li>
<li class="chapter" data-level="2.7.5" data-path="2-results.html"><a href="2-results.html#kap1-h3k9me3-and-h4k20me3-are-lost-at-iap-elements-in-hp1ctko-es-cells"><i class="fa fa-check"></i><b>2.7.5</b> KAP1, H3K9me3 and H4K20me3 are lost at IAP Elements in HP1cTKO ES Cells</a></li>
<li class="chapter" data-level="2.7.6" data-path="2-results.html"><a href="2-results.html#dna-methylation-is-affected-following-loss-of-hp1"><i class="fa fa-check"></i><b>2.7.6</b> DNA Methylation is affected Following loss of HP1</a></li>
<li><a href="2-results.html#kap1-is-lost-from-icrs-in-hp1ctko-cells"><em>KAP1 is lost from ICRs in HP1cTKO Cells</em></a></li>
</ul></li>
<li class="chapter" data-level="2.8" data-path="2-results.html"><a href="2-results.html#RNAresults"><i class="fa fa-check"></i><b>2.8</b> Transcriptional Dysregulation following HP1 Deficiency Drives UPR and Inflammatory Responses</a><ul>
<li class="chapter" data-level="2.8.1" data-path="2-results.html"><a href="2-results.html#genotype-related-changes"><i class="fa fa-check"></i><b>2.8.1</b> Genotype Related Changes</a></li>
<li class="chapter" data-level="2.8.2" data-path="2-results.html"><a href="2-results.html#longitudinal-changes-to-gene-expression"><i class="fa fa-check"></i><b>2.8.2</b> Longitudinal Changes To Gene Expression</a></li>
<li class="chapter" data-level="2.8.3" data-path="2-results.html"><a href="2-results.html#pathway-analysis"><i class="fa fa-check"></i><b>2.8.3</b> Pathway Analysis</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-discussion.html"><a href="3-discussion.html"><i class="fa fa-check"></i><b>3</b> Discussion</a><ul>
<li class="chapter" data-level="3.1" data-path="3-discussion.html"><a href="3-discussion.html#phenotypes"><i class="fa fa-check"></i><b>3.1</b> Phenotypes</a><ul>
<li><a href="3-discussion.html#structural-changes-in-the-hippocampus"><em>Structural Changes in the Hippocampus</em></a></li>
<li><a href="3-discussion.html#reduction-in-cortical-volume"><em>Reduction in Cortical Volume</em></a></li>
<li><a href="3-discussion.html#behavior-and-impairments-in-learning-memory"><em>Behavior and Impairments in Learning &amp; Memory</em></a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="3-discussion.html"><a href="3-discussion.html#hp1-dependent-epigenetic-systems"><i class="fa fa-check"></i><b>3.2</b> HP1-dependent Epigenetic Systems</a><ul>
<li><a href="3-discussion.html#hp1gamma"><em>HP1<span class="math inline">\(\gamma\)</span></em></a></li>
<li><a href="3-discussion.html#hp1beta"><em>HP1<span class="math inline">\(\beta\)</span></em></a></li>
<li><a href="3-discussion.html#hp1beta-x-hp1gamma"><em>HP1<span class="math inline">\(\beta\)</span></em> x <em>HP1<span class="math inline">\(\gamma\)</span></em></a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="3-discussion.html"><a href="3-discussion.html#concordance-with-aging-and-disease"><i class="fa fa-check"></i><b>3.3</b> Concordance with Aging and Disease</a><ul>
<li><a href="3-discussion.html#complement-in-cns-aging-and-disease"><em>Complement in CNS Aging and Disease</em></a></li>
<li><a href="3-discussion.html#a-pathway-to-complement"><em>A Pathway to Complement</em></a></li>
<li><a href="3-discussion.html#unfolded-protein-response-upr"><em>Unfolded Protein Response (UPR)</em></a></li>
<li><a href="3-discussion.html#viruses-transactivation-and-neurodegeneration"><em>Viruses, Transactivation and Neurodegeneration</em></a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="3-discussion.html"><a href="3-discussion.html#concluding-remarks"><i class="fa fa-check"></i><b>3.4</b> Concluding Remarks</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-methods.html"><a href="4-methods.html"><i class="fa fa-check"></i><b>4</b> Methods</a><ul>
<li class="chapter" data-level="4.1" data-path="4-methods.html"><a href="4-methods.html#mouse-lines"><i class="fa fa-check"></i><b>4.1</b> Mouse Lines</a><ul>
<li class="chapter" data-level="4.1.1" data-path="4-methods.html"><a href="4-methods.html#HP1anull"><i class="fa fa-check"></i><b>4.1.1</b> Generation of HP1<span class="math inline">\(\alpha\)</span>KO and HP1<span class="math inline">\(\alpha\)</span>T</a></li>
<li class="chapter" data-level="4.1.2" data-path="4-methods.html"><a href="4-methods.html#HP1Bgtarget"><i class="fa fa-check"></i><b>4.1.2</b> Generation of Floxed HP1<span class="math inline">\(\beta\)</span> &amp; Floxed HP1<span class="math inline">\(\gamma\)</span></a></li>
<li class="chapter" data-level="4.1.3" data-path="4-methods.html"><a href="4-methods.html#genotyping"><i class="fa fa-check"></i><b>4.1.3</b> Genotyping</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="4-methods.html"><a href="4-methods.html#cloning"><i class="fa fa-check"></i><b>4.2</b> Cloning</a><ul>
<li class="chapter" data-level="4.2.1" data-path="4-methods.html"><a href="4-methods.html#rna-isolation-cdna-generation-rnaseq"><i class="fa fa-check"></i><b>4.2.1</b> RNA Isolation, cDNA Generation, &amp; RNAseq</a></li>
<li class="chapter" data-level="4.2.2" data-path="4-methods.html"><a href="4-methods.html#cloning-of-rna-probes-for-in-situ-hybridization"><i class="fa fa-check"></i><b>4.2.2</b> Cloning of RNA Probes for <em>In-Situ Hybridization</em></a></li>
<li class="chapter" data-level="4.2.3" data-path="4-methods.html"><a href="4-methods.html#cloning-of-expression-constructs"><i class="fa fa-check"></i><b>4.2.3</b> Cloning of Expression Constructs</a></li>
<li class="chapter" data-level="4.2.4" data-path="4-methods.html"><a href="4-methods.html#site-directed-mutagenesis"><i class="fa fa-check"></i><b>4.2.4</b> Site Directed Mutagenesis</a></li>
<li class="chapter" data-level="4.2.5" data-path="4-methods.html"><a href="4-methods.html#transformation"><i class="fa fa-check"></i><b>4.2.5</b> Transformation</a></li>
<li class="chapter" data-level="4.2.6" data-path="4-methods.html"><a href="4-methods.html#clonal-selection-plasmid-production"><i class="fa fa-check"></i><b>4.2.6</b> Clonal Selection &amp; Plasmid Production</a></li>
<li class="chapter" data-level="4.2.7" data-path="4-methods.html"><a href="4-methods.html#in-vitro-transcription-ivt"><i class="fa fa-check"></i><b>4.2.7</b> In Vitro Transcription (IVT)</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="4-methods.html"><a href="4-methods.html#cell-culture"><i class="fa fa-check"></i><b>4.3</b> Cell Culture</a><ul>
<li class="chapter" data-level="4.3.1" data-path="4-methods.html"><a href="4-methods.html#primary-neuronal-culture"><i class="fa fa-check"></i><b>4.3.1</b> Primary Neuronal Culture</a></li>
<li class="chapter" data-level="4.3.2" data-path="4-methods.html"><a href="4-methods.html#hek293t-culture-transfection"><i class="fa fa-check"></i><b>4.3.2</b> HEK293T Culture &amp; Transfection</a></li>
<li class="chapter" data-level="4.3.3" data-path="4-methods.html"><a href="4-methods.html#co-immunoprecipitation-co-ip-and-western-blot-wb"><i class="fa fa-check"></i><b>4.3.3</b> Co-Immunoprecipitation (Co-IP) and Western Blot (WB)</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="4-methods.html"><a href="4-methods.html#in-utero-electroporation-tissue-processing-and-histology"><i class="fa fa-check"></i><b>4.4</b> In Utero Electroporation, Tissue Processing and Histology</a><ul>
<li class="chapter" data-level="4.4.1" data-path="4-methods.html"><a href="4-methods.html#IUEmethod"><i class="fa fa-check"></i><b>4.4.1</b> In Utero Electroporation (IUE)</a></li>
<li class="chapter" data-level="4.4.2" data-path="4-methods.html"><a href="4-methods.html#tissue-fixation-and-perfusion"><i class="fa fa-check"></i><b>4.4.2</b> Tissue Fixation and Perfusion</a></li>
<li class="chapter" data-level="4.4.3" data-path="4-methods.html"><a href="4-methods.html#cryosectioning"><i class="fa fa-check"></i><b>4.4.3</b> Cryosectioning</a></li>
<li class="chapter" data-level="4.4.4" data-path="4-methods.html"><a href="4-methods.html#immuno-histochemistry-ihc"><i class="fa fa-check"></i><b>4.4.4</b> Immuno Histochemistry (IHC)</a></li>
<li class="chapter" data-level="4.4.5" data-path="4-methods.html"><a href="4-methods.html#quantification-of-cells-in-the-dentate-gyrus"><i class="fa fa-check"></i><b>4.4.5</b> Quantification of Cells in the Dentate Gyrus</a></li>
<li class="chapter" data-level="4.4.6" data-path="4-methods.html"><a href="4-methods.html#in-situ-hybridization-ish"><i class="fa fa-check"></i><b>4.4.6</b> In Situ Hybridization (ISH)</a></li>
<li class="chapter" data-level="4.4.7" data-path="4-methods.html"><a href="4-methods.html#paraffin-sectioning"><i class="fa fa-check"></i><b>4.4.7</b> Paraffin Sectioning</a></li>
<li class="chapter" data-level="4.4.8" data-path="4-methods.html"><a href="4-methods.html#nissl"><i class="fa fa-check"></i><b>4.4.8</b> Nissl Stain</a></li>
<li class="chapter" data-level="4.4.9" data-path="4-methods.html"><a href="4-methods.html#golgi-impregnation-and-dendritic-complexity-analysis"><i class="fa fa-check"></i><b>4.4.9</b> Golgi Impregnation and Dendritic Complexity Analysis</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="4-methods.html"><a href="4-methods.html#behavioral-experiments"><i class="fa fa-check"></i><b>4.5</b> Behavioral Experiments</a><ul>
<li class="chapter" data-level="4.5.1" data-path="4-methods.html"><a href="4-methods.html#open-field-activity-1"><i class="fa fa-check"></i><b>4.5.1</b> Open Field Activity</a></li>
<li class="chapter" data-level="4.5.2" data-path="4-methods.html"><a href="4-methods.html#paired-pulse-inhibition-1"><i class="fa fa-check"></i><b>4.5.2</b> Paired-Pulse Inhibition</a></li>
<li class="chapter" data-level="4.5.3" data-path="4-methods.html"><a href="4-methods.html#context-cued-fear-conditioning"><i class="fa fa-check"></i><b>4.5.3</b> Context-Cued Fear Conditioning</a></li>
<li class="chapter" data-level="4.5.4" data-path="4-methods.html"><a href="4-methods.html#Barnesmeth"><i class="fa fa-check"></i><b>4.5.4</b> Barnes Maze</a></li>
<li class="chapter" data-level="4.5.5" data-path="4-methods.html"><a href="4-methods.html#social-activity-monitor-sam-1"><i class="fa fa-check"></i><b>4.5.5</b> Social Activity Monitor (SAM)</a></li>
<li class="chapter" data-level="4.5.6" data-path="4-methods.html"><a href="4-methods.html#micro-behaviors-homecagescan"><i class="fa fa-check"></i><b>4.5.6</b> Micro Behaviors (HomeCageScan)</a></li>
<li class="chapter" data-level="4.5.7" data-path="4-methods.html"><a href="4-methods.html#nestcomethod"><i class="fa fa-check"></i><b>4.5.7</b> Nest Construction</a></li>
</ul></li>
<li class="chapter" data-level="4.6" data-path="4-methods.html"><a href="4-methods.html#magnetic-resonance-imaging-mri-and-volumetrics"><i class="fa fa-check"></i><b>4.6</b> Magnetic Resonance Imaging (MRI) and Volumetrics</a></li>
<li class="chapter" data-level="4.7" data-path="4-methods.html"><a href="4-methods.html#reduced-read-bisulfite-sequencing-rrbseq"><i class="fa fa-check"></i><b>4.7</b> Reduced Read Bisulfite Sequencing (RRBSeq)</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html"><i class="fa fa-check"></i><b>5</b> Supplemental Data</a><ul>
<li class="chapter" data-level="5.1" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html#Behavsupplement"><i class="fa fa-check"></i><b>5.1</b> Behavior</a><ul>
<li class="chapter" data-level="5.1.1" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html#homecagescan-micro-behaviors"><i class="fa fa-check"></i><b>5.1.1</b> HomeCageScan Micro Behaviors</a></li>
<li class="chapter" data-level="5.1.2" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html#context-cued-fear-conditioning-1"><i class="fa fa-check"></i><b>5.1.2</b> Context-Cued Fear Conditioning</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html#DE"><i class="fa fa-check"></i><b>5.2</b> Differential Gene Expression</a><ul>
<li class="chapter" data-level="5.2.1" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html#differentially-expressed-genes-in-aged-hp1betako"><i class="fa fa-check"></i><b>5.2.1</b> Differentially Expressed Genes in Aged HP1<span class="math inline">\(\beta\)</span>KO</a></li>
<li class="chapter" data-level="5.2.2" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html#inflam"><i class="fa fa-check"></i><b>5.2.2</b> Genes Related to Anti-viral and Interferon Response in HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO</a></li>
<li class="chapter" data-level="5.2.3" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html#genomic-hotspots-sensitive-to-hp1-deficiency-in-the-hippocampus"><i class="fa fa-check"></i><b>5.2.3</b> <em>Genomic Hotspots Sensitive to HP1 Deficiency in the Hippocampus</em></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html"><i class="fa fa-check"></i><b>6</b> Bioinformatics &amp; Data Processing</a><ul>
<li class="chapter" data-level="6.1" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#software-environment-management"><i class="fa fa-check"></i><b>6.1</b> Software Environment Management</a></li>
<li class="chapter" data-level="6.2" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#snakemake-jobs"><i class="fa fa-check"></i><b>6.2</b> Snakemake Jobs</a></li>
<li class="chapter" data-level="6.3" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#tetranscripts-pipeline"><i class="fa fa-check"></i><b>6.3</b> TETranscripts Pipeline</a></li>
<li class="chapter" data-level="6.4" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#differential-gene-expression-using-edger"><i class="fa fa-check"></i><b>6.4</b> Differential Gene Expression Using EdgeR</a></li>
<li class="chapter" data-level="6.5" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#deeptools-pipeline"><i class="fa fa-check"></i><b>6.5</b> Deeptools Pipeline</a></li>
<li class="chapter" data-level="6.6" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#lions-pipeline"><i class="fa fa-check"></i><b>6.6</b> LIONS Pipeline</a></li>
<li class="chapter" data-level="6.7" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#analysis-of-reduced-read-bisulphite-sequencing-rrbs"><i class="fa fa-check"></i><b>6.7</b> Analysis of Reduced-Read Bisulphite Sequencing (RRBS)</a><ul>
<li class="chapter" data-level="6.7.1" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#methylkit-and-circos-plot"><i class="fa fa-check"></i><b>6.7.1</b> Methylkit and Circos Plot</a></li>
<li class="chapter" data-level="6.7.2" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#annotated-ideogram"><i class="fa fa-check"></i><b>6.7.2</b> Annotated Ideogram</a></li>
</ul></li>
<li class="chapter" data-level="6.8" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#about-this-thesis"><i class="fa fa-check"></i><b>6.8</b> About this Thesis</a><ul>
<li class="chapter" data-level="6.8.1" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#session-info"><i class="fa fa-check"></i><b>6.8.1</b> Session Info</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7" data-path="7-references.html"><a href="7-references.html"><i class="fa fa-check"></i><b>7</b> References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">HP1 Deficiency Results in Accelerated Neurodegeneration</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="bioinformatics-data-processing" class="section level1">
<h1><span class="header-section-number">Part 6</span> Bioinformatics &amp; Data Processing</h1>
<p>Three methods were used to run multiple jobs in parallel on the cluster hosted at the Core Unit Bioinformatics (CUBI) of the Berlin Institute of Health (BIH):</p>
<ul>
<li><p>For single test jobs, typically a single bash script was writen and submitted using qsub.</p></li>
<li><p>Larger pipelines utilized the Snakemake pipeline building system (Such as TETranscripts and deeptools)</p></li>
<li><p>Pre-built self-contained pipeline was customized for running on CUBI (Such as the LIONS chimeric analysis)</p></li>
</ul>
<p>All work on CUBI was done using <code>screen/ssh</code>, and data was transfered to and from the cluster using <code>rsync</code>.</p>
<div id="software-environment-management" class="section level2">
<h2><span class="header-section-number">6.1</span> Software Environment Management</h2>
<p>Prior to running any jobs on the CUBI cluster, software dependencies were managed using a miniconda environment.<br />
For example, if a particular pipeline required python 2.7, and samtools0.18, an environment containing these dependencies with the name ‘py2’ could be created</p>
<div class="sourceCode"><pre class="sourceCode bash"><code class="sourceCode bash"><span class="ex">conda</span> create -n py2 python=2.7 samtools=0.18</code></pre></div>
<p>and later sourced</p>
<div class="sourceCode"><pre class="sourceCode bash"><code class="sourceCode bash"><span class="bu">source</span> activate py2</code></pre></div>
<p>prior to running any pipelines using the py2 software environment.</p>
</div>
<div id="snakemake-jobs" class="section level2">
<h2><span class="header-section-number">6.2</span> Snakemake Jobs</h2>
<p>To run large batch jobs on the CUBI cluster, pipelines were written in Snakemake <a href="https://snakemake.readthedocs.io/en/stable/" class="uri">https://snakemake.readthedocs.io/en/stable/</a> , which serves as python wrapper for bash scripts. For parallel distribution of jobs on the cluster using <code>qsub</code>, a bash script such as <code>submit_snakejob.sh</code> was written, which includes all the relevant qsub flags which specify requested CPU per job and memory requirements. <code>submit_snakejob.sh</code>:</p>
<div class="sourceCode"><pre class="sourceCode bash"><code class="sourceCode bash"><span class="co">#!/bin/bash</span>
<span class="co">#$ -V</span>
<span class="co">#$ -j y</span>
<span class="co">#$ -o logs</span>
<span class="co">#$ -r yes</span>
<span class="co">#$ -cwd</span>
<span class="co">#$ -S /bin/bash</span>
<span class="co">#$ -P control</span>
<span class="bu">export</span> <span class="va">TMPDIR=</span>/fast/users/newmana_c/scratch/tmp
<span class="bu">export</span> <span class="va">LOGDIR=</span>logs/<span class="va">${JOB_ID}</span>
<span class="fu">mkdir</span> -p <span class="va">$LOGDIR</span>
<span class="kw">set</span> <span class="ex">-x</span>
<span class="ex">snakemake</span> \
    --drmaa <span class="st">&quot; \</span>
<span class="st">        -V \</span>
<span class="st">        -cwd \</span>
<span class="st">        -P medium \</span>
<span class="st">        -l h_vmem=8g \</span>
<span class="st">        -l h_rt=72:00:00 \</span>
<span class="st">        -pe smp 8 \</span>
<span class="st">        -j yes \</span>
<span class="st">    -m e \</span>
<span class="st">        -M andrew.newman@charite.de \</span>
<span class="st">        -o </span><span class="va">$LOGDIR</span><span class="st">/&quot;</span> \
    -j 2 \
    -k \
    -p</code></pre></div>
<p>Calling Snakemake from the shell script directs to the Snakefile, where the pipeline is specified. Snakemake uses the snakefile to look for rules used to create the desired outputs from the firs rule. For this reason <code>rule all</code> is placed first, and snakemake works to find the inputs for this rule. In this way the pipeline is built backwards based on what outputs are desired and what wildcards are specified.</p>
</div>
<div id="tetranscripts-pipeline" class="section level2">
<h2><span class="header-section-number">6.3</span> TETranscripts Pipeline</h2>
<p>To run the TETranscripts pipeline, a conda environment containing python 3.5, STAR, samtools and TEtranscripts was sourced. The following Snakefile was written using the aid of some existing templates (such as <a href="https://github.com/leipzig/snakemake-example/blob/master/Snakefile" class="uri">https://github.com/leipzig/snakemake-example/blob/master/Snakefile</a>) in order to run TETranscripts on all HP1 paired end RNAseq data.</p>
<p><code>Snakefile</code>:</p>
<div class="sourceCode"><pre class="sourceCode python"><code class="sourceCode python"><span class="co">#! bin/env/python</span>
<span class="co">#TETranscripts Pipeline</span>
<span class="co">#Invoked by submit_TE.sh in ~/scratch/TEToolkit</span>

WT_YOUNG <span class="op">=</span> <span class="st">&quot;WT_young_S10422 WT_young_S10424 WT_young_S10425 \</span>
<span class="st">WT_young_S10432 WT_young_S10433 WT_young_S10602&quot;</span>
WT_OLD <span class="op">=</span> <span class="st">&quot;WT_old_S10604 WT_old_S10631 WT_old_S10632&quot;</span>
BKO_YOUNG <span class="op">=</span> <span class="st">&quot;BKO_young_S10625 BKO_young_S10626 BKO_young_S10627&quot;</span>
BKO_OLD <span class="op">=</span> <span class="st">&quot;BKO_old_S10428 BKO_old_S10633&quot;</span>
GKO_YOUNG <span class="op">=</span> <span class="st">&quot;gKO_young_S10426 gKO_young_S10628 gKO_young_S10630&quot;</span>
GKO_OLD <span class="op">=</span> <span class="st">&quot;gKO_old_S10634 gKO_old_S10635 gKO_old_S10636&quot;</span>
DKO_YOUNG <span class="op">=</span> <span class="st">&quot;DKO_young_S10423 DKO_young_S10431 DKO_young_S10430 \</span>
<span class="st">DKO_young_S10603 DKO_young_S10429 DKO_young_S10629 &quot;</span>
DKO_OLD <span class="op">=</span> <span class="st">&quot;DKO_old_S10605 DKO_old_S10606 DKO_old_S10607&quot;</span>
SAMPLES <span class="op">=</span> <span class="st">&#39; &#39;</span>.join([WT_YOUNG, WT_OLD, BKO_YOUNG, BKO_OLD, <span class="op">\</span>
GKO_YOUNG, GKO_OLD, DKO_YOUNG, DKO_OLD]).split()
TE_GTFFILE <span class="op">=</span> <span class="st">&quot;annotations/mm10_rmsk_TE.gtf&quot;</span>
GTFFILE <span class="op">=</span> <span class="st">&quot;~/scratch/LIONS-master/resources/mm10/annotation/&quot;</span>
<span class="op">+</span><span class="st">&quot;gencode.vM16.basic.annotation.gtf&quot;</span>
STAR_INDEX <span class="op">=</span> <span class="st">&quot;/fast/projects/cubit/current/static_data/&quot;</span><span class="op">+</span>
<span class="co">&quot;precomputed/STAR/2.4.1d/default/GRCm38/p5/none&quot;</span>
STAR <span class="op">=</span> [<span class="st">&#39;alignment/&#39;</span><span class="op">+</span>f<span class="op">+</span><span class="st">&#39;.sorted.bam&#39;</span> <span class="cf">for</span> f <span class="kw">in</span> SAMPLES]
SORT <span class="op">=</span> [<span class="st">&#39;alignment/&#39;</span><span class="op">+</span>f<span class="op">+</span><span class="st">&#39;.sorted.bam&#39;</span> <span class="cf">for</span> f <span class="kw">in</span> SAMPLES]
FLAGSTAT <span class="op">=</span> [<span class="st">&#39;alignment/&#39;</span><span class="op">+</span>f<span class="op">+</span><span class="st">&#39;.sorted.bam.flagstat&#39;</span> <span class="cf">for</span> f <span class="kw">in</span> SAMPLES]
TE_OUT <span class="op">=</span> <span class="st">&quot;TE_OUT/all/all&quot;</span>


rule <span class="bu">all</span>:
    <span class="bu">input</span>:TE_OUT, SORT, FLAGSTAT,  STAR

rule flagstat_bam:
    <span class="bu">input</span>:
        <span class="co">&quot;alignment/{sample}.sorted.bam&quot;</span>
    output:
        <span class="co">&quot;alignment/{sample}.sorted.bam.flagstat&quot;</span>
    log:
        <span class="co">&quot;logs/{sample}.flagstat_bam&quot;</span>
    shell:
        <span class="co">&quot;&quot;&quot;</span>
<span class="co">        samtools flagstat {input} &gt; {output} 2&gt; {log}</span>
<span class="co">        &quot;&quot;&quot;</span>
        
rule index_bam:
    <span class="bu">input</span>:
        <span class="co">&quot;alignment/{sample}.sorted.bam&quot;</span>
    output:
        <span class="co">&quot;alignment/{sample}.sorted.bam.bai&quot;</span>
    log:
                <span class="co">&quot;logs/{sample}.index_bam&quot;</span>
    threads: <span class="dv">1</span>
    shell:
                <span class="co">&quot;&quot;&quot;</span>
<span class="co">                samtools index {input} 2&gt; {log}</span>
<span class="co">                &quot;&quot;&quot;</span>
                
rule align:
    <span class="bu">input</span>:
        fq1<span class="op">=</span><span class="st">&quot;fastq/</span><span class="sc">{sample}</span><span class="st">_R1.fastq.gz&quot;</span>,
        fq2<span class="op">=</span><span class="st">&quot;fastq/</span><span class="sc">{sample}</span><span class="st">_R2.fastq.gz&quot;</span>,
        genome<span class="op">=</span>STAR_INDEX,
        gtf<span class="op">=</span>GTFFILE
    output:<span class="st">&quot;</span><span class="sc">{sample}</span><span class="st">_Aligned.out.bam&quot;</span>
    log: <span class="st">&quot;logs/</span><span class="sc">{sample}</span><span class="st">_align_sort.log&quot;</span>
    threads: <span class="dv">8</span>
    shell:<span class="st">&quot;&quot;&quot;</span>
<span class="st">        STAR --genomeDir </span><span class="sc">{input.genome}</span><span class="st"> \</span>
<span class="st">        --outFileNamePrefix </span><span class="sc">{wildcards.sample}</span><span class="st">_ \</span>
<span class="st">        --readFilesCommand zcat \</span>
<span class="st">        --readFilesIn </span><span class="sc">{input.fq1}</span><span class="st"> </span><span class="sc">{input.fq2}</span><span class="st"> \</span>
<span class="st">        --runThreadN </span><span class="sc">{threads}</span><span class="st"> \</span>
<span class="st">        --genomeLoad NoSharedMemory \</span>
<span class="st">        --outSAMattributes All \</span>
<span class="st">        --outFilterMultimapNmax 100 \</span>
<span class="st">        --winAnchorMultimapNmax 100 \</span>
<span class="st">        --outSAMstrandField intronMotif \</span>
<span class="st">        --outSAMtype BAM Unsorted \</span>
<span class="st">        --sjdbGTFfile </span><span class="sc">{input.gtf}</span>
<span class="st">        mkdir -p starlogs</span>
<span class="st">        mv </span><span class="sc">{wildcards.sample}</span><span class="st">_Log.final.out </span><span class="sc">{wildcards.sample}</span><span class="st">_Log.out \</span>
<span class="st">        </span><span class="sc">{wildcards.sample}</span><span class="st">_Log.progress.out </span><span class="sc">{wildcards.sample}</span><span class="st">_SJ.out.tab \</span>
<span class="st">        </span><span class="sc">{wildcards.sample}</span><span class="st">__STARgenome starlogs</span>
<span class="st">        &quot;&quot;&quot;</span>
        
<span class="co"># Bam files must be sorted by read ( &#39;-n&#39; flag sorts by read names) </span>
<span class="co"># prior to TEtranscripts analysis</span>
rule sort:
      <span class="bu">input</span>: <span class="st">&quot;</span><span class="sc">{sample}</span><span class="st">_Aligned.out.bam&quot;</span>
    output: <span class="st">&quot;alignment/</span><span class="sc">{sample}</span><span class="st">.sorted.bam&quot;</span>
      threads: <span class="dv">4</span>
    shell:<span class="st">&quot;&quot;&quot;</span>
<span class="st">        samtools sort -m 1G -@ </span><span class="sc">{threads}</span><span class="st"> -O bam -n -T </span><span class="sc">{output}</span><span class="st">.tmp </span><span class="sc">{input}</span><span class="st"> -o </span><span class="sc">{output}</span>
<span class="st">        &quot;&quot;&quot;</span>
        
YOUNG_WT_BAMS <span class="op">=</span> <span class="op">\</span>
<span class="co">&#39; &#39;</span>.join([<span class="st">&#39;alignment/</span><span class="sc">{0}</span><span class="st">.sorted.bam&#39;</span>.<span class="bu">format</span>(f) <span class="cf">for</span> f <span class="kw">in</span> WT_YOUNG.split()])
YOUNG_BKO_BAMS <span class="op">=</span> <span class="op">\</span>
<span class="co">&#39; &#39;</span>.join([<span class="st">&#39;alignment/</span><span class="sc">{0}</span><span class="st">.sorted.bam&#39;</span>.<span class="bu">format</span>(f) <span class="cf">for</span> f <span class="kw">in</span> BKO_YOUNG.split()])
YOUNG_GKO_BAMS <span class="op">=</span> <span class="op">\</span>
<span class="co">&#39; &#39;</span>.join([<span class="st">&#39;alignment/</span><span class="sc">{0}</span><span class="st">.sorted.bam&#39;</span>.<span class="bu">format</span>(f) <span class="cf">for</span> f <span class="kw">in</span> GKO_YOUNG.split()])
YOUNG_DKO_BAMS <span class="op">=</span> <span class="op">\</span>
<span class="co">&#39; &#39;</span>.join([<span class="st">&#39;alignment/</span><span class="sc">{0}</span><span class="st">.sorted.bam&#39;</span>.<span class="bu">format</span>(f) <span class="cf">for</span> f <span class="kw">in</span> DKO_YOUNG.split()])
OLD_WT_BAMS <span class="op">=</span> <span class="op">\</span>
<span class="co">&#39; &#39;</span>.join([<span class="st">&#39;alignment/</span><span class="sc">{0}</span><span class="st">.sorted.bam&#39;</span>.<span class="bu">format</span>(f) <span class="cf">for</span> f <span class="kw">in</span> WT_OLD.split()])
OLD_BKO_BAMS <span class="op">=</span> <span class="op">\</span>
<span class="co">&#39; &#39;</span>.join([<span class="st">&#39;alignment/</span><span class="sc">{0}</span><span class="st">.sorted.bam&#39;</span>.<span class="bu">format</span>(f) <span class="cf">for</span> f <span class="kw">in</span> BKO_OLD.split()])
OLD_GKO_BAMS <span class="op">=</span> <span class="op">\</span>
<span class="co">&#39; &#39;</span>.join([<span class="st">&#39;alignment/</span><span class="sc">{0}</span><span class="st">.sorted.bam&#39;</span>.<span class="bu">format</span>(f) <span class="cf">for</span> f <span class="kw">in</span> GKO_OLD.split()])
OLD_DKO_BAMS <span class="op">=</span> <span class="op">\</span>
<span class="co">&#39; &#39;</span>.join([<span class="st">&#39;alignment/</span><span class="sc">{0}</span><span class="st">.sorted.bam&#39;</span>.<span class="bu">format</span>(f) <span class="cf">for</span> f <span class="kw">in</span> DKO_OLD.split()])

rule TET_all_counts:
        <span class="bu">input</span>:
                gtf <span class="op">=</span> GTFFILE,
                te <span class="op">=</span> TE_GTFFILE,
        sortedbams <span class="op">=</span> SORT
        output: <span class="st">&quot;TE_OUT/all/all&quot;</span>
        shell:<span class="st">&quot;&quot;&quot;</span>
<span class="st">                TEtranscripts \</span>
<span class="st">                -t </span><span class="sc">{YOUNG_DKO_BAMS}</span><span class="st"> </span><span class="sc">{YOUNG_GKO_BAMS}</span><span class="st"> \</span>
<span class="st">                </span><span class="sc">{YOUNG_BKO_BAMS}</span><span class="st"> </span><span class="sc">{OLD_WT_BAMS}</span><span class="st"> </span><span class="sc">{OLD_GKO_BAMS}</span><span class="st"> \</span>
<span class="st">                </span><span class="sc">{OLD_BKO_BAMS}</span><span class="st"> </span><span class="sc">{OLD_DKO_BAMS}</span><span class="st"> \</span>
<span class="st">                -c </span><span class="sc">{YOUNG_WT_BAMS}</span><span class="st"> \</span>
<span class="st">                --GTF </span><span class="sc">{input.gtf}</span><span class="st"> \</span>
<span class="st">                --TE </span><span class="sc">{input.te}</span><span class="st"> \</span>
<span class="st">                --project </span><span class="sc">{output}</span>
<span class="st">                &quot;&quot;&quot;</span></code></pre></div>
</div>
<div id="differential-gene-expression-using-edger" class="section level2">
<h2><span class="header-section-number">6.4</span> Differential Gene Expression Using EdgeR</h2>
<p>The output of the <code>rule TET_all_counts</code> in the TETranscripts pipeline is a count table with genes or repetitive elements as rows and each sequencing sample as columns, which was renamed to <code>counts.clean.tab</code>. The metadata file <code>coldata.txt</code> was then prepared where each row in it corresponds to each column of expression data in counts.clean.tab. In this way lane and batch effects can easily be observed in addition to easy grouping by age and genotype.</p>
<table>
<caption>
<span id="tab:colData">Table 6.1: </span>colData.txt
</caption>
<thead>
<tr>
<th style="text-align:left;">
coldata
</th>
<th style="text-align:left;">
Batch
</th>
<th style="text-align:left;">
Lane
</th>
<th style="text-align:left;">
Genotype
</th>
<th style="text-align:left;">
Age
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
WT_young_S10424
</td>
<td style="text-align:left;">
b1
</td>
<td style="text-align:left;">
L1
</td>
<td style="text-align:left;">
WT
</td>
<td style="text-align:left;">
Young
</td>
</tr>
<tr>
<td style="text-align:left;">
WT_young_S10425
</td>
<td style="text-align:left;">
b1
</td>
<td style="text-align:left;">
L1
</td>
<td style="text-align:left;">
WT
</td>
<td style="text-align:left;">
Young
</td>
</tr>
<tr>
<td style="text-align:left;">
WT_young_S10602
</td>
<td style="text-align:left;">
b1
</td>
<td style="text-align:left;">
L2
</td>
<td style="text-align:left;">
WT
</td>
<td style="text-align:left;">
Young
</td>
</tr>
<tr>
<td style="text-align:left;">
WT_young_S10422
</td>
<td style="text-align:left;">
b1
</td>
<td style="text-align:left;">
L1
</td>
<td style="text-align:left;">
WT
</td>
<td style="text-align:left;">
Young
</td>
</tr>
<tr>
<td style="text-align:left;">
DKO_young_S10430
</td>
<td style="text-align:left;">
b1
</td>
<td style="text-align:left;">
L1
</td>
<td style="text-align:left;">
DKO
</td>
<td style="text-align:left;">
Young
</td>
</tr>
<tr>
<td style="text-align:left;">
DKO_young_S10431
</td>
<td style="text-align:left;">
b1
</td>
<td style="text-align:left;">
L1
</td>
<td style="text-align:left;">
DKO
</td>
<td style="text-align:left;">
Young
</td>
</tr>
<tr>
<td style="text-align:left;">
DKO_young_S10423
</td>
<td style="text-align:left;">
b1
</td>
<td style="text-align:left;">
L1
</td>
<td style="text-align:left;">
DKO
</td>
<td style="text-align:left;">
Young
</td>
</tr>
<tr>
<td style="text-align:left;">
DKO_young_S10603
</td>
<td style="text-align:left;">
b1
</td>
<td style="text-align:left;">
L2
</td>
<td style="text-align:left;">
DKO
</td>
<td style="text-align:left;">
Young
</td>
</tr>
<tr>
<td style="text-align:left;">
BKO_young_S10626
</td>
<td style="text-align:left;">
b2
</td>
<td style="text-align:left;">
L3
</td>
<td style="text-align:left;">
BKO
</td>
<td style="text-align:left;">
Young
</td>
</tr>
<tr>
<td style="text-align:left;">
BKO_young_S10625
</td>
<td style="text-align:left;">
b2
</td>
<td style="text-align:left;">
L3
</td>
<td style="text-align:left;">
BKO
</td>
<td style="text-align:left;">
Young
</td>
</tr>
<tr>
<td style="text-align:left;">
BKO_young_S10627
</td>
<td style="text-align:left;">
b2
</td>
<td style="text-align:left;">
L3
</td>
<td style="text-align:left;">
BKO
</td>
<td style="text-align:left;">
Young
</td>
</tr>
<tr>
<td style="text-align:left;">
gKO_young_S10426
</td>
<td style="text-align:left;">
b2
</td>
<td style="text-align:left;">
L1
</td>
<td style="text-align:left;">
gKO
</td>
<td style="text-align:left;">
Young
</td>
</tr>
<tr>
<td style="text-align:left;">
gKO_young_S10630
</td>
<td style="text-align:left;">
b2
</td>
<td style="text-align:left;">
L3
</td>
<td style="text-align:left;">
gKO
</td>
<td style="text-align:left;">
Young
</td>
</tr>
<tr>
<td style="text-align:left;">
gKO_young_S10628
</td>
<td style="text-align:left;">
b2
</td>
<td style="text-align:left;">
L3
</td>
<td style="text-align:left;">
gKO
</td>
<td style="text-align:left;">
Young
</td>
</tr>
<tr>
<td style="text-align:left;">
WT_old_S10604
</td>
<td style="text-align:left;">
b2
</td>
<td style="text-align:left;">
L2
</td>
<td style="text-align:left;">
WT
</td>
<td style="text-align:left;">
Aged
</td>
</tr>
<tr>
<td style="text-align:left;">
WT_old_S10632
</td>
<td style="text-align:left;">
b2
</td>
<td style="text-align:left;">
L3
</td>
<td style="text-align:left;">
WT
</td>
<td style="text-align:left;">
Aged
</td>
</tr>
<tr>
<td style="text-align:left;">
WT_old_S10631
</td>
<td style="text-align:left;">
b2
</td>
<td style="text-align:left;">
L3
</td>
<td style="text-align:left;">
WT
</td>
<td style="text-align:left;">
Aged
</td>
</tr>
<tr>
<td style="text-align:left;">
BKO_old_S10633
</td>
<td style="text-align:left;">
b2
</td>
<td style="text-align:left;">
L3
</td>
<td style="text-align:left;">
BKO
</td>
<td style="text-align:left;">
Aged
</td>
</tr>
<tr>
<td style="text-align:left;">
BKO_old_S10428
</td>
<td style="text-align:left;">
b2
</td>
<td style="text-align:left;">
L3
</td>
<td style="text-align:left;">
BKO
</td>
<td style="text-align:left;">
Aged
</td>
</tr>
<tr>
<td style="text-align:left;">
gKO_old_S10636
</td>
<td style="text-align:left;">
b2
</td>
<td style="text-align:left;">
L3
</td>
<td style="text-align:left;">
gKO
</td>
<td style="text-align:left;">
Aged
</td>
</tr>
<tr>
<td style="text-align:left;">
gKO_old_S10635
</td>
<td style="text-align:left;">
b2
</td>
<td style="text-align:left;">
L3
</td>
<td style="text-align:left;">
gKO
</td>
<td style="text-align:left;">
Aged
</td>
</tr>
<tr>
<td style="text-align:left;">
gKO_old_S10634
</td>
<td style="text-align:left;">
b2
</td>
<td style="text-align:left;">
L3
</td>
<td style="text-align:left;">
gKO
</td>
<td style="text-align:left;">
Aged
</td>
</tr>
<tr>
<td style="text-align:left;">
DKO_old_S10606
</td>
<td style="text-align:left;">
b2
</td>
<td style="text-align:left;">
L2
</td>
<td style="text-align:left;">
DKO
</td>
<td style="text-align:left;">
Aged
</td>
</tr>
<tr>
<td style="text-align:left;">
DKO_old_S10607
</td>
<td style="text-align:left;">
b2
</td>
<td style="text-align:left;">
L2
</td>
<td style="text-align:left;">
DKO
</td>
<td style="text-align:left;">
Aged
</td>
</tr>
<tr>
<td style="text-align:left;">
DKO_old_S10605
</td>
<td style="text-align:left;">
b2
</td>
<td style="text-align:left;">
L2
</td>
<td style="text-align:left;">
DKO
</td>
<td style="text-align:left;">
Aged
</td>
</tr>
</tbody>
</table>
<p>The following R script was used to detect differential gene expression in HP1 mutants using edgeR and plot the results:</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw">setwd</span>(<span class="st">&quot;~/Bench/&quot;</span>)
<span class="kw">library</span>(edgeR)
<span class="kw">library</span>(ggplot2)
<span class="kw">library</span>(biomaRt)
<span class="kw">library</span>(dplyr)
<span class="co"># biomaRt Annotations (to add more attributes use</span>
<span class="co"># listAttributes(mart))</span>
mart &lt;-<span class="st"> </span><span class="kw">useDataset</span>(<span class="st">&quot;mmusculus_gene_ensembl&quot;</span>, <span class="kw">useMart</span>(<span class="st">&quot;ensembl&quot;</span>))

<span class="co"># listAttributes(mart) # - to add more attributes</span>
ID_QueryAttributes =<span class="st"> </span><span class="kw">c</span>(<span class="st">&quot;ensembl_gene_id&quot;</span>, <span class="st">&quot;external_gene_name&quot;</span>, 
    <span class="st">&quot;gene_biotype&quot;</span>, <span class="st">&quot;chromosome_name&quot;</span>)
ID_QueryFilters =<span class="st"> &quot;ensembl_gene_id&quot;</span>

<span class="co"># Metadata</span>
metadata =<span class="st"> </span><span class="kw">read.delim</span>(<span class="st">&quot;colData.txt&quot;</span>)
batch =<span class="st"> </span><span class="kw">factor</span>(metadata<span class="op">$</span>Batch, <span class="dt">levels =</span> <span class="kw">c</span>(<span class="st">&quot;b1&quot;</span>, <span class="st">&quot;b2&quot;</span>))
genotype =<span class="st"> </span><span class="kw">factor</span>(metadata<span class="op">$</span>Genotype, <span class="dt">levels =</span> <span class="kw">c</span>(<span class="st">&quot;WT&quot;</span>, <span class="st">&quot;BKO&quot;</span>, 
    <span class="st">&quot;gKO&quot;</span>, <span class="st">&quot;DKO&quot;</span>))
age =<span class="st"> </span><span class="kw">factor</span>(metadata<span class="op">$</span>Age, <span class="dt">levels =</span> <span class="kw">c</span>(<span class="st">&quot;Young&quot;</span>, <span class="st">&quot;Aged&quot;</span>))
group =<span class="st"> </span><span class="kw">factor</span>(<span class="kw">paste0</span>(metadata<span class="op">$</span>Genotype, <span class="st">&quot;.&quot;</span>, metadata<span class="op">$</span>Age), 
    <span class="dt">levels =</span> <span class="kw">c</span>(<span class="st">&quot;WT.Young&quot;</span>, <span class="st">&quot;BKO.Young&quot;</span>, <span class="st">&quot;gKO.Young&quot;</span>, <span class="st">&quot;DKO.Young&quot;</span>, 
        <span class="st">&quot;WT.Aged&quot;</span>, <span class="st">&quot;BKO.Aged&quot;</span>, <span class="st">&quot;gKO.Aged&quot;</span>, <span class="st">&quot;DKO.Aged&quot;</span>))

<span class="co"># Read in Count table</span>
counts =<span class="st"> </span><span class="kw">read.delim</span>(<span class="st">&quot;counts.clean.tab&quot;</span>)
<span class="co"># Remove suffixes on ENSEMBL IDs</span>
counts<span class="op">$</span>Symbol &lt;-<span class="st"> </span><span class="kw">gsub</span>(counts<span class="op">$</span>Symbol, <span class="dt">pattern =</span> <span class="st">&quot;</span><span class="ch">\\</span><span class="st">.[0-9]+$&quot;</span>, 
    <span class="dt">replacement =</span> <span class="st">&quot;&quot;</span>)
<span class="co"># biomaRt get gene names</span>
tmp1 &lt;-<span class="st"> </span><span class="kw">data.frame</span>(<span class="dt">ensembl_gene_id =</span> counts<span class="op">$</span>Symbol)
ids &lt;-<span class="st"> </span><span class="kw">getBM</span>(<span class="dt">attributes =</span> ID_QueryAttributes, <span class="dt">filters =</span> ID_QueryFilters, 
    <span class="dt">values =</span> tmp1[, <span class="st">&quot;ensembl_gene_id&quot;</span>], <span class="dt">mart =</span> mart)

<span class="co"># Rename symbol column containing ensembl ids to</span>
<span class="co"># &#39;ensembl_gene_id&#39;</span>
<span class="kw">colnames</span>(counts)[<span class="dv">1</span>] &lt;-<span class="st"> &quot;ensembl_gene_id&quot;</span>
counts =<span class="st"> </span><span class="kw">left_join</span>(<span class="dt">x =</span> counts, <span class="dt">y =</span> ids, <span class="dt">by =</span> <span class="st">&quot;ensembl_gene_id&quot;</span>)

<span class="co"># sometimes annotation duplicates rows</span>
counts &lt;-<span class="st"> </span><span class="kw">distinct</span>(counts)

<span class="co"># Repeats aren&#39;t annotated in biomaRT so move them over to</span>
<span class="co"># external_gene_name column</span>
counts<span class="op">$</span>external_gene_name[<span class="kw">is.na</span>(counts<span class="op">$</span>external_gene_name)] &lt;-<span class="st"> </span><span class="kw">as.character</span>(counts<span class="op">$</span>ensembl_gene_id[<span class="kw">is.na</span>(counts<span class="op">$</span>external_gene_name)])

<span class="co"># and add corresponding Repeat Class to gene_biotype</span>
counts<span class="op">$</span>gene_biotype[<span class="kw">is.na</span>(counts<span class="op">$</span>gene_biotype)] &lt;-<span class="st"> </span><span class="kw">gsub</span>(<span class="st">&quot;.*:&quot;</span>, 
    <span class="st">&quot;&quot;</span>, counts<span class="op">$</span>ensembl_gene_id[<span class="kw">is.na</span>(counts<span class="op">$</span>gene_biotype)])

<span class="co"># move gene annotation columns to beginning</span>
counts &lt;-<span class="st"> </span>counts <span class="op">%&gt;%</span><span class="st"> </span><span class="kw">select</span>(gene_biotype, <span class="kw">everything</span>())
counts &lt;-<span class="st"> </span>counts <span class="op">%&gt;%</span><span class="st"> </span><span class="kw">select</span>(external_gene_name, <span class="kw">everything</span>())
counts &lt;-<span class="st"> </span>counts <span class="op">%&gt;%</span><span class="st"> </span><span class="kw">select</span>(chromosome_name, <span class="kw">everything</span>())
counts &lt;-<span class="st"> </span>counts <span class="op">%&gt;%</span><span class="st"> </span><span class="kw">select</span>(ensembl_gene_id, <span class="kw">everything</span>())
<span class="co"># Put Count table in DGElist</span>
dge &lt;-<span class="st"> </span><span class="kw">DGEList</span>(<span class="dt">counts =</span> counts[, <span class="dv">5</span><span class="op">:</span><span class="dv">29</span>], <span class="dt">genes =</span> counts[, <span class="dv">1</span><span class="op">:</span><span class="dv">4</span>], 
    <span class="dt">group =</span> group)
<span class="co"># Some ENSEMBL ids are gene duplicates, keep the ones with</span>
<span class="co"># the highest counts.</span>
o &lt;-<span class="st"> </span><span class="kw">order</span>(<span class="kw">rowSums</span>(dge<span class="op">$</span>counts), <span class="dt">decreasing =</span> <span class="ot">TRUE</span>)
dge &lt;-<span class="st"> </span>dge[o, ]
dups &lt;-<span class="st"> </span><span class="kw">duplicated</span>(dge<span class="op">$</span>genes<span class="op">$</span>external_gene_name)
dge &lt;-<span class="st"> </span>dge[<span class="op">!</span>dups, ]
<span class="co"># Remove lowly expressed Genes</span>
keep &lt;-<span class="st"> </span><span class="kw">rowSums</span>(<span class="kw">cpm</span>(dge) <span class="op">&gt;</span><span class="st"> </span><span class="dv">1</span>) <span class="op">&gt;=</span><span class="st"> </span><span class="dv">2</span>
dge &lt;-<span class="st"> </span>dge[keep, , keep.lib.sizes =<span class="st"> </span><span class="ot">FALSE</span>]

<span class="co"># Change rownames to gene name</span>
<span class="kw">rownames</span>(dge<span class="op">$</span>counts) &lt;-<span class="st"> </span>dge<span class="op">$</span>genes<span class="op">$</span>external_gene_name

<span class="co"># Experimental Design</span>
design =<span class="st"> </span><span class="kw">model.matrix</span>(<span class="op">~</span><span class="dv">0</span> <span class="op">+</span><span class="st"> </span>group)
<span class="kw">colnames</span>(design) &lt;-<span class="st"> </span><span class="kw">gsub</span>(<span class="st">&quot;group&quot;</span>, <span class="st">&quot;&quot;</span>, <span class="kw">colnames</span>(design))

<span class="co"># Set up contrasts</span>
contr.matrix &lt;-<span class="st"> </span><span class="kw">makeContrasts</span>(<span class="dt">Young.BKO_vs_Young.WT =</span> BKO.Young <span class="op">-</span><span class="st"> </span>
<span class="st">    </span>WT.Young, <span class="dt">Young.gKO_vs_Young.WT =</span> gKO.Young <span class="op">-</span><span class="st"> </span>WT.Young, <span class="dt">Young.DKO_vs_Young.WT =</span> DKO.Young <span class="op">-</span><span class="st"> </span>
<span class="st">    </span>WT.Young, <span class="dt">Aged.BKO_vs_Aged.WT =</span> BKO.Aged <span class="op">-</span><span class="st"> </span>WT.Aged, <span class="dt">Aged.gKO_vs_Aged.WT =</span> gKO.Aged <span class="op">-</span><span class="st"> </span>
<span class="st">    </span>WT.Aged, <span class="dt">Aged.DKO_vs_Aged.WT =</span> DKO.Aged <span class="op">-</span><span class="st"> </span>WT.Aged, <span class="dt">Young.WT_vs_Aged.WT =</span> WT.Aged <span class="op">-</span><span class="st"> </span>
<span class="st">    </span>WT.Young, <span class="dt">Young.BKO_vs_Aged.BKO =</span> BKO.Aged <span class="op">-</span><span class="st"> </span>BKO.Young, <span class="dt">Young.gKO_vs_Aged.gKO =</span> gKO.Aged <span class="op">-</span><span class="st"> </span>
<span class="st">    </span>gKO.Young, <span class="dt">Young.DKO_vs_Aged.DKO =</span> DKO.Aged <span class="op">-</span><span class="st"> </span>DKO.Young, 
    <span class="dt">Aged.gKO_vs_Aged.DKO =</span> DKO.Aged <span class="op">-</span><span class="st"> </span>gKO.Aged, <span class="dt">Aged.BKO_vs_Aged.DKO =</span> DKO.Aged <span class="op">-</span><span class="st"> </span>
<span class="st">        </span>BKO.Aged, <span class="dt">levels =</span> <span class="kw">colnames</span>(design))
contr.matrix

dge &lt;-<span class="st"> </span><span class="kw">calcNormFactors</span>(dge, <span class="dt">method =</span> <span class="st">&quot;TMM&quot;</span>)
dge &lt;-<span class="st"> </span><span class="kw">estimateDisp</span>(dge, <span class="dt">design =</span> design)
fit &lt;-<span class="st"> </span><span class="kw">glmQLFit</span>(dge, <span class="dt">design =</span> design)

Y.BKO.WT &lt;-<span class="st"> </span><span class="kw">glmQLFTest</span>(fit, <span class="dt">contrast =</span> contr.matrix[, <span class="st">&quot;Young.BKO_vs_Young.WT&quot;</span>])
Y.GKO.WT &lt;-<span class="st"> </span><span class="kw">glmQLFTest</span>(fit, <span class="dt">contrast =</span> contr.matrix[, <span class="st">&quot;Young.gKO_vs_Young.WT&quot;</span>])
Y.DKO.WT &lt;-<span class="st"> </span><span class="kw">glmQLFTest</span>(fit, <span class="dt">contrast =</span> contr.matrix[, <span class="st">&quot;Young.DKO_vs_Young.WT&quot;</span>])
A.BKO.WT &lt;-<span class="st"> </span><span class="kw">glmQLFTest</span>(fit, <span class="dt">contrast =</span> contr.matrix[, <span class="st">&quot;Aged.BKO_vs_Aged.WT&quot;</span>])
A.GKO.WT &lt;-<span class="st"> </span><span class="kw">glmQLFTest</span>(fit, <span class="dt">contrast =</span> contr.matrix[, <span class="st">&quot;Aged.gKO_vs_Aged.WT&quot;</span>])
A.DKO.WT &lt;-<span class="st"> </span><span class="kw">glmQLFTest</span>(fit, <span class="dt">contrast =</span> contr.matrix[, <span class="st">&quot;Aged.DKO_vs_Aged.WT&quot;</span>])
A.DKO.BKO &lt;-<span class="st"> </span><span class="kw">glmQLFTest</span>(fit, <span class="dt">contrast =</span> contr.matrix[, <span class="st">&quot;Aged.BKO_vs_Aged.DKO&quot;</span>])
A.DKO.gKO &lt;-<span class="st"> </span><span class="kw">glmQLFTest</span>(fit, <span class="dt">contrast =</span> contr.matrix[, <span class="st">&quot;Aged.gKO_vs_Aged.DKO&quot;</span>])
Y.WT.A.WT &lt;-<span class="st"> </span><span class="kw">glmQLFTest</span>(fit, <span class="dt">contrast =</span> contr.matrix[, <span class="st">&quot;Young.WT_vs_Aged.WT&quot;</span>])
Y.BKO.A.BKO &lt;-<span class="st"> </span><span class="kw">glmQLFTest</span>(fit, <span class="dt">contrast =</span> contr.matrix[, <span class="st">&quot;Young.BKO_vs_Aged.BKO&quot;</span>])
Y.GKO.A.GKO &lt;-<span class="st"> </span><span class="kw">glmQLFTest</span>(fit, <span class="dt">contrast =</span> contr.matrix[, <span class="st">&quot;Young.gKO_vs_Aged.gKO&quot;</span>])
Y.DKO.A.DKO &lt;-<span class="st"> </span><span class="kw">glmQLFTest</span>(fit, <span class="dt">contrast =</span> contr.matrix[, <span class="st">&quot;Young.DKO_vs_Aged.DKO&quot;</span>])

<span class="kw">topTags</span>(Y.WT.A.WT)
<span class="kw">topTags</span>(Y.DKO.WT, <span class="dt">n =</span> <span class="dv">100</span>, <span class="dt">sort.by =</span> <span class="st">&quot;logFC&quot;</span>)

<span class="co"># YOUNG DKO</span>
<span class="kw">summary</span>(de &lt;-<span class="st"> </span><span class="kw">decideTestsDGE</span>(Y.DKO.WT))
detags &lt;-<span class="st"> </span><span class="kw">rownames</span>(Y.DKO.WT)[<span class="kw">as.logical</span>(de)]
<span class="kw">plotSmear</span>(Y.DKO.WT, <span class="dt">de.tags =</span> detags)

<span class="co"># YOUNG BKO</span>
<span class="kw">summary</span>(de &lt;-<span class="st"> </span><span class="kw">decideTestsDGE</span>(Y.BKO.WT))
detags &lt;-<span class="st"> </span><span class="kw">rownames</span>(Y.BKO.WT)[<span class="kw">as.logical</span>(de)]
<span class="kw">plotSmear</span>(Y.BKO.WT, <span class="dt">de.tags =</span> detags)
<span class="co"># OLD DKO</span>
<span class="kw">summary</span>(de &lt;-<span class="st"> </span><span class="kw">decideTestsDGE</span>(A.DKO.WT))
detags &lt;-<span class="st"> </span><span class="kw">rownames</span>(A.DKO.WT)[<span class="kw">as.logical</span>(de)]
<span class="kw">plotSmear</span>(A.DKO.WT, <span class="dt">de.tags =</span> detags)
<span class="kw">summary</span>(de &lt;-<span class="st"> </span><span class="kw">decideTestsDGE</span>(A.GKO.WT))
<span class="co"># For tests, should only compare within batch.  Can collate</span>
<span class="co"># together later to look at trends.</span>
PVALUE =<span class="st"> </span><span class="fl">0.05</span>
Y.DKO.WT.top100 &lt;-<span class="st"> </span><span class="kw">topTags</span>(Y.DKO.WT, <span class="dt">n =</span> <span class="dv">100</span>, <span class="dt">adjust.method =</span> <span class="st">&quot;BH&quot;</span>, 
    <span class="dt">sort.by =</span> <span class="st">&quot;logFC&quot;</span>, <span class="dt">p.value =</span> PVALUE)
A.BKO.WT.top100 &lt;-<span class="st"> </span><span class="kw">topTags</span>(A.BKO.WT, <span class="dt">n =</span> <span class="dv">100</span>, <span class="dt">adjust.method =</span> <span class="st">&quot;BH&quot;</span>, 
    <span class="dt">sort.by =</span> <span class="st">&quot;logFC&quot;</span>, <span class="dt">p.value =</span> PVALUE)
A.GKO.WT.top100 &lt;-<span class="st"> </span><span class="kw">topTags</span>(A.GKO.WT, <span class="dt">n =</span> <span class="dv">100</span>, <span class="dt">adjust.method =</span> <span class="st">&quot;BH&quot;</span>, 
    <span class="dt">sort.by =</span> <span class="st">&quot;logFC&quot;</span>, <span class="dt">p.value =</span> PVALUE)
A.DKO.WT.top100 &lt;-<span class="st"> </span><span class="kw">topTags</span>(A.DKO.WT, <span class="dt">n =</span> <span class="dv">100</span>, <span class="dt">adjust.method =</span> <span class="st">&quot;BH&quot;</span>, 
    <span class="dt">sort.by =</span> <span class="st">&quot;logFC&quot;</span>, <span class="dt">p.value =</span> PVALUE)
<span class="co"># There is an uncorrectable batch effect between Y.WT and</span>
<span class="co"># A.WT, so don&#39;t use results from this test: Y.WT.A.WT.top100</span>
<span class="co"># &lt;- topTags(Y.WT.A.WT, n = 100, adjust.method = &#39;BH&#39;,</span>
<span class="co"># sort.by = &#39;logFC&#39;, p.value = PVALUE)</span>
Y.BKO.A.BKO.top100 &lt;-<span class="st"> </span><span class="kw">topTags</span>(Y.BKO.A.BKO, <span class="dt">n =</span> <span class="dv">100</span>, <span class="dt">adjust.method =</span> <span class="st">&quot;BH&quot;</span>, 
    <span class="dt">sort.by =</span> <span class="st">&quot;logFC&quot;</span>, <span class="dt">p.value =</span> PVALUE)
<span class="co"># no significant genes between young BKO and aged BKO</span>
Y.GKO.A.GKO.top100 &lt;-<span class="st"> </span><span class="kw">topTags</span>(Y.GKO.A.GKO, <span class="dt">n =</span> <span class="dv">100</span>, <span class="dt">adjust.method =</span> <span class="st">&quot;BH&quot;</span>, 
    <span class="dt">sort.by =</span> <span class="st">&quot;logFC&quot;</span>, <span class="dt">p.value =</span> PVALUE)
<span class="co"># no significant genes between young gKO and aged gKO There</span>
<span class="co"># is an uncorrectable batch effect between Y.DKO and A.DKO,</span>
<span class="co"># so don&#39;t use results from this test: Y.DKO.A.DKO.top100 &lt;-</span>
<span class="co"># topTags(Y.DKO.A.DKO, n = 100, adjust.method = &#39;BH&#39;, sort.by</span>
<span class="co"># = &#39;logFC&#39;, p.value = PVALUE)</span>

<span class="co"># Other within batch tests between single and double mutant:</span>
A.DKO.BKO.top100 &lt;-<span class="st"> </span><span class="kw">topTags</span>(A.DKO.BKO, <span class="dt">n =</span> <span class="dv">100</span>, <span class="dt">adjust.method =</span> <span class="st">&quot;BH&quot;</span>, 
    <span class="dt">sort.by =</span> <span class="st">&quot;logFC&quot;</span>, <span class="dt">p.value =</span> PVALUE)
A.DKO.gKO.top100 &lt;-<span class="st"> </span><span class="kw">topTags</span>(A.DKO.gKO, <span class="dt">n =</span> <span class="dv">100</span>, <span class="dt">adjust.method =</span> <span class="st">&quot;BH&quot;</span>, 
    <span class="dt">sort.by =</span> <span class="st">&quot;logFC&quot;</span>, <span class="dt">p.value =</span> PVALUE)

<span class="co"># Make list of Genes differentially expressed in each</span>
<span class="co"># condition</span>

y.diff.list &lt;-<span class="st"> </span><span class="kw">rbind</span>(Y.DKO.WT.top100<span class="op">$</span>table)
y.diff.list &lt;-<span class="st"> </span><span class="kw">unique</span>(<span class="kw">rownames</span>(y.diff.list))
<span class="kw">str</span>(y.diff.list)
a.diff.list &lt;-<span class="st"> </span><span class="kw">rbind</span>(A.BKO.WT.top100<span class="op">$</span>table, A.GKO.WT.top100<span class="op">$</span>table, 
    A.DKO.WT.top100<span class="op">$</span>table)
a.diff.list &lt;-<span class="st"> </span><span class="kw">unique</span>(<span class="kw">rownames</span>(a.diff.list))
<span class="kw">str</span>(a.diff.list)  <span class="co">#Check</span>
combined.diff.list &lt;-<span class="st"> </span><span class="kw">c</span>(y.diff.list, a.diff.list)
combined.diff.list &lt;-<span class="st"> </span><span class="kw">unique</span>(combined.diff.list)
<span class="kw">str</span>(combined.diff.list)  <span class="co"># Check</span>

<span class="co"># Can show batch effects</span>
logCPM &lt;-<span class="st"> </span><span class="kw">cpm</span>(dge, <span class="dt">log =</span> <span class="ot">TRUE</span>, <span class="dt">prior.count =</span> <span class="dv">5</span>)
<span class="kw">plotMDS</span>(logCPM, <span class="dt">col =</span> <span class="kw">as.numeric</span>(batch))
<span class="co"># Remove Batch effect for Heatmap using limma</span>
logCPM_norm &lt;-<span class="st"> </span><span class="kw">removeBatchEffect</span>(logCPM, <span class="dt">batch =</span> batch)
<span class="kw">plotMDS</span>(logCPM_norm, <span class="dt">col =</span> <span class="kw">as.numeric</span>(batch))

<span class="co"># Subset by Differential Gene lists for plotting</span>
y.logCPM_norm_geno &lt;-<span class="st"> </span><span class="kw">as.data.frame</span>(<span class="kw">subset</span>(logCPM_norm, <span class="kw">rownames</span>(logCPM_norm) <span class="op">%in%</span><span class="st"> </span>
<span class="st">    </span>y.diff.list))
a.logCPM_norm_geno &lt;-<span class="st"> </span><span class="kw">as.data.frame</span>(<span class="kw">subset</span>(logCPM_norm, <span class="kw">rownames</span>(logCPM_norm) <span class="op">%in%</span><span class="st"> </span>
<span class="st">    </span>a.diff.list))
alt.logCPM_norm_geno &lt;-<span class="st"> </span><span class="kw">as.data.frame</span>(<span class="kw">subset</span>(logCPM_norm, <span class="kw">rownames</span>(logCPM_norm) <span class="op">%in%</span><span class="st"> </span>
<span class="st">    </span>alt.diff.list))
c.logCPM_norm_geno &lt;-<span class="st"> </span><span class="kw">as.data.frame</span>(<span class="kw">subset</span>(logCPM_norm, <span class="kw">rownames</span>(logCPM_norm) <span class="op">%in%</span><span class="st"> </span>
<span class="st">    </span>combined.diff.list))

y.geno.heat &lt;-<span class="st"> </span><span class="kw">t</span>(<span class="kw">scale</span>(<span class="kw">t</span>(y.logCPM_norm_geno), <span class="dt">scale =</span> T))
a.geno.heat &lt;-<span class="st"> </span><span class="kw">t</span>(<span class="kw">scale</span>(<span class="kw">t</span>(a.logCPM_norm_geno), <span class="dt">scale =</span> T))
alt.geno.heat &lt;-<span class="st"> </span><span class="kw">t</span>(<span class="kw">scale</span>(<span class="kw">t</span>(alt.logCPM_norm_geno), <span class="dt">scale =</span> T))
c.geno.heat &lt;-<span class="st"> </span><span class="kw">t</span>(<span class="kw">scale</span>(<span class="kw">t</span>(c.logCPM_norm_geno), <span class="dt">scale =</span> T))

<span class="kw">library</span>(RColorBrewer)
<span class="kw">library</span>(gplots)

<span class="co"># RColorBrewer sequential palletes: Blues BuGn BuPu GnBu</span>
<span class="co"># Greens Greys Oranges OrRd PuBu PuBuGn PuRd Purples RdPu</span>
<span class="co"># Reds YlGn YlGnBu YlOrBr YlOrRd Pick Colours:</span>
hmcol &lt;-<span class="st"> </span><span class="kw">brewer.pal</span>(<span class="dv">9</span>, <span class="st">&quot;PuBu&quot;</span>)
<span class="co"># Plot Heatmap</span>
<span class="kw">heatmap.2</span>(c.geno.heat, <span class="dt">col =</span> hmcol, <span class="dt">tracecol =</span> <span class="ot">NA</span>)

<span class="co"># Volcano plot</span>
BKOdf &lt;-<span class="st"> </span><span class="kw">cbind</span>(A.BKO.WT<span class="op">$</span>table, A.BKO.WT<span class="op">$</span>genes)
BKOdf[<span class="st">&quot;gt&quot;</span>] &lt;-<span class="st"> &quot;HP1B KO&quot;</span>
GKOdf &lt;-<span class="st"> </span><span class="kw">cbind</span>(A.GKO.WT<span class="op">$</span>table, A.GKO.WT<span class="op">$</span>genes)
GKOdf[<span class="st">&quot;gt&quot;</span>] &lt;-<span class="st"> &quot;HP1g KO&quot;</span>
DKO.y.df &lt;-<span class="st"> </span><span class="kw">cbind</span>(Y.DKO.WT<span class="op">$</span>table, Y.DKO.WT<span class="op">$</span>genes)
DKO.y.df[<span class="st">&quot;gt&quot;</span>] &lt;-<span class="st"> &quot;HP1DKO&quot;</span>
DKO.a.df &lt;-<span class="st"> </span><span class="kw">cbind</span>(A.DKO.WT<span class="op">$</span>table, A.DKO.WT<span class="op">$</span>genes)
DKO.a.df[<span class="st">&quot;gt&quot;</span>] &lt;-<span class="st"> &quot;HP1DKO&quot;</span>

scatter_df &lt;-<span class="st"> </span><span class="kw">rbind</span>(BKOdf, GKOdf, DKO.a.df)
scatter_df<span class="op">$</span>gt_f &lt;-<span class="st"> </span><span class="kw">factor</span>(scatter_df<span class="op">$</span>gt, <span class="dt">levels =</span> <span class="kw">c</span>(<span class="st">&quot;HP1B KO&quot;</span>, 
    <span class="st">&quot;HP1g KO&quot;</span>, <span class="st">&quot;HP1DKO&quot;</span>))

volcano.all &lt;-<span class="st"> </span><span class="kw">ggplot</span>(scatter_df, <span class="kw">aes</span>(<span class="dt">x =</span> logFC, <span class="dt">y =</span> <span class="op">-</span><span class="kw">log10</span>(PValue), 
    <span class="dt">color =</span> <span class="kw">ifelse</span>(scatter_df<span class="op">$</span>external_gene_name <span class="op">%in%</span><span class="st"> </span>combined.diff.list, 
        scatter_df<span class="op">$</span>gene_biotype, <span class="ot">NA</span>))) <span class="op">+</span><span class="st"> </span><span class="kw">geom_point</span>(<span class="dt">size =</span> <span class="fl">0.8</span>) <span class="op">+</span><span class="st"> </span>
<span class="st">    </span><span class="kw">xlab</span>(<span class="st">&quot;LogFC&quot;</span>) <span class="op">+</span><span class="st"> </span><span class="kw">ylab</span>(<span class="st">&quot;-log10 (p value)&quot;</span>) <span class="op">+</span><span class="st"> </span><span class="kw">geom_text</span>(<span class="kw">aes</span>(<span class="dt">label =</span> <span class="kw">ifelse</span>(scatter_df<span class="op">$</span>external_gene_name <span class="op">%in%</span><span class="st"> </span>
<span class="st">    </span>combined.diff.list <span class="op">&amp;</span><span class="st"> </span>scatter_df<span class="op">$</span>PValue <span class="op">&lt;</span><span class="st"> </span><span class="fl">0.01</span> <span class="op">&amp;</span><span class="st"> </span><span class="kw">abs</span>(scatter_df<span class="op">$</span>logFC) <span class="op">&gt;</span><span class="st"> </span>
<span class="st">    </span><span class="dv">2</span> <span class="op">&amp;</span><span class="st"> </span>scatter_df<span class="op">$</span>logCPM <span class="op">&gt;</span><span class="st"> </span><span class="dv">2</span>, <span class="kw">as.character</span>(scatter_df<span class="op">$</span>external_gene_name), 
    <span class="st">&quot;&quot;</span>)), <span class="dt">hjust =</span> <span class="dv">0</span>, <span class="dt">vjust =</span> <span class="op">-</span><span class="fl">0.1</span>, <span class="dt">size =</span> <span class="dv">3</span>) <span class="op">+</span><span class="st"> </span><span class="kw">theme_minimal</span>() <span class="op">+</span><span class="st"> </span>
<span class="st">    </span><span class="kw">ggtitle</span>(<span class="kw">expression</span>(<span class="kw">paste</span>(<span class="st">&quot;Differential Gene Expression in HP1 Mutant Adult Hippocampi&quot;</span>))) <span class="op">+</span><span class="st"> </span>
<span class="st">    </span><span class="kw">theme</span>(<span class="dt">legend.title =</span> <span class="kw">element_blank</span>(), <span class="dt">plot.title =</span> <span class="kw">element_text</span>(<span class="dt">hjust =</span> <span class="fl">0.5</span>)) <span class="op">+</span><span class="st"> </span>
<span class="st">    </span><span class="kw">facet_grid</span>(gt_f <span class="op">~</span><span class="st"> </span>.)
volcano.all</code></pre></div>
</div>
<div id="deeptools-pipeline" class="section level2">
<h2><span class="header-section-number">6.5</span> Deeptools Pipeline</h2>
<p>Prior to running deeptools, repeat annotation files were downloaded from the ucsc table browser (<a href="https://genome.ucsc.edu/cgi-bin/hgTables" class="uri">https://genome.ucsc.edu/cgi-bin/hgTables</a>) in bed format, and cleaned for use in deeptools using the following code:</p>
<div class="sourceCode"><pre class="sourceCode bash"><code class="sourceCode bash"><span class="co">#sort</span>
<span class="fu">sort</span> -k1,1V -k2,2n mm10.LINE.ucsc <span class="op">&gt;</span> *mm10.LINE.sorted.bed
<span class="fu">sort</span> -k1,1V -k2,2n mm10.SINE.ucsc <span class="op">&gt;</span> *mm10.SINE.sorted.bed
<span class="fu">sort</span> -k1,1V -k2,2n mm10.LTR.ucsc <span class="op">&gt;</span> *mm10.LTR.sorted.bed
            
<span class="co">#Merge overlapping or adjacent entries</span>
<span class="ex">mergeBed</span> -i mm10.LINE.sorted.bed <span class="op">&gt;</span> mm10.LINE.merged.bed
<span class="ex">mergeBed</span> -i mm10.SINE.sorted.bed <span class="op">&gt;</span> mm10.SINE.merged.bed
<span class="ex">mergeBed</span> -i mm10.LTR.sorted.bed <span class="op">&gt;</span> mm10.LTR.merged.bed

<span class="co">#Rename to work with written snakefile</span>
<span class="fu">mv</span> mm10.LINE.merged.bed clean/mm10.LINE.bed
<span class="fu">mv</span> mm10.SINE.merged.bed clean/mm10.SINE.bed
<span class="fu">mv</span> mm10.LTR.merged.bed clean/mm10.LTR.bed</code></pre></div>
<p>To generate read coverage plots of repetitive elements, a snakemake pipeline was written to automate steps beginning from generating coverage files (bigwigs) to calculating genome wide coverage matrices by repeat by genotype before finally plotting the results. Following the first genome-wide plot, the upper 50 percent (<code>rule subsample</code>) of the comatrix was subset and re-plot yielding _top50 plots. Sample and annotation wildcards were specified using a <code>config.yaml</code> file.</p>
<p><code>config.yaml</code>:</p>
<pre><code>---
samples:
  WT_young_2: bams/WT_young_2.bam
  WT_old_2: bams/WT_old_2.bam
  BKO_young_2: bams/BKO_young_2.bam
  BKO_old_2: bams/BKO_old_2.bam
  gKO_young_2: bams/gKO_young_2.bam
  gKO_old_2: bams/gKO_old_2.bam
  DKO_young_2: bams/DKO_young_2.bam
  DKO_old_2: bams/DKO_old_2.bam

annotations:
  LINE: annotations/mm10.LINE.bed
  SINE: annotations/mm10.SINE.bed
  LTR: annotations/mm10.LTR.bed</code></pre>
<p>The Deeptools pipeline was run according to the follwoing <code>Snakefile</code>:</p>
<div class="sourceCode"><pre class="sourceCode python"><code class="sourceCode python"><span class="co">#! bin/env/python</span>
<span class="co"># deeptools Pipeline</span>
<span class="co"># Invoked by submit_cov.sh in ~/scratch/deeptools/HP1/</span>

shell.prefix(<span class="st">&quot;set -eo pipefail; echo BEGIN at $(date); &quot;</span>)
shell.suffix(<span class="st">&quot;; exitstat=$?; echo END at $(date); \</span>
<span class="st">echo exit status was $exitstat; exit $exitstat&quot;</span>)

configfile: <span class="st">&quot;config.yaml&quot;</span>

<span class="im">import</span> os

rule <span class="bu">all</span>:
    <span class="bu">input</span>:
        ALL_BW<span class="op">=</span> expand(<span class="st">&quot;bigwigs/</span><span class="sc">{sample}</span><span class="st">.bw&quot;</span>, sample <span class="op">=</span> config[<span class="st">&quot;samples&quot;</span>]),
        ALL_IND_COMATRIX<span class="op">=</span> expand(<span class="st">&quot;comatrices/</span><span class="sc">{annotation}</span><span class="st">_comatrix.gz&quot;</span>, <span class="op">\</span>
          annotation <span class="op">=</span> config[<span class="st">&quot;annotations&quot;</span>] ),
        GIANT_HEATMAP<span class="op">=</span> <span class="st">&quot;plots/giant_heatmap.pdf&quot;</span>,
        GIANT_TOP50_PLOT<span class="op">=</span><span class="st">&quot;plots/giant_top50_heatmap.pdf&quot;</span>,
        GLOBAL_PLOTS<span class="op">=</span> expand(<span class="st">&quot;plots/</span><span class="sc">{annotation}</span><span class="st">_wholegenome_heatmap.pdf&quot;</span>, <span class="op">\</span>
          annotation <span class="op">=</span> config[<span class="st">&quot;annotations&quot;</span>]),
        LNCRNA_3<span class="op">=</span><span class="st">&quot;comatrices/lncRNA3_comatrix.gz&quot;</span>,
        LNCRNA_BOD<span class="op">=</span><span class="st">&quot;plots/lncRNA_wg_body_heatmap.pdf&quot;</span>,
        SUBSAMPLE_PLOT<span class="op">=</span> expand(<span class="st">&quot;plots/</span><span class="sc">{annotation}</span><span class="st">_Top50p_heatmap.pdf&quot;</span>, <span class="op">\</span>
          annotation <span class="op">=</span> config[<span class="st">&quot;annotations&quot;</span>]),
        LNCRNA_3_PLOT<span class="op">=</span> <span class="st">&quot;plots/lncRNA3_heatmap.pdf&quot;</span>,
        ALL_SAMPLE_COMATRICES <span class="op">=</span> expand(
          <span class="st">&quot;comatrices/</span><span class="sc">{sample}</span><span class="st">_repeat_comatrix.gz&quot;</span>, 
          sample <span class="op">=</span> config[<span class="st">&quot;samples&quot;</span>]
          ),
        SAMPLE_PROFILES <span class="op">=</span> expand(
          <span class="st">&quot;plots/</span><span class="sc">{sample}</span><span class="st">_repeat_profile.pdf&quot;</span>, 
          sample <span class="op">=</span> config[<span class="st">&quot;samples&quot;</span>]
          )

rule bw_coverage: <span class="co">#converts bams to bigwigs (.bw)</span>
    <span class="bu">input</span>: <span class="kw">lambda</span> wildcards: config[<span class="st">&quot;samples&quot;</span>][wildcards.sample]
    output: <span class="st">&quot;bigwigs/</span><span class="sc">{sample}</span><span class="st">.bw&quot;</span>
    threads: <span class="dv">12</span>
    log: <span class="st">&quot;logs/</span><span class="sc">{sample}</span><span class="st">.log&quot;</span>
    shell:
            <span class="co">&quot;&quot;&quot;</span>
<span class="co">            bamCoverage -b {input} \</span>
<span class="co">        -p {threads} \</span>
<span class="co">        -o {output} \</span>
<span class="co">        --normalizeUsing RPKM </span>
<span class="co">            &quot;&quot;&quot;</span>

rule Annotation_Comatrices:  <span class="co">#one comatrix per annotation</span>
    <span class="bu">input</span>:
        bigwigs_list<span class="op">=</span>expand(<span class="st">&quot;bigwigs/</span><span class="sc">{sample}</span><span class="st">.bw&quot;</span>, sample <span class="op">=</span> config[<span class="st">&quot;samples&quot;</span>]),
        annotation_list<span class="op">=</span><span class="kw">lambda</span> wildcards: config[<span class="st">&quot;annotations&quot;</span>][wildcards.annotation]
    output: <span class="st">&quot;comatrices/</span><span class="sc">{annotation}</span><span class="st">_comatrix.gz&quot;</span>
    threads: <span class="dv">12</span>
    log: <span class="st">&quot;logs/deeptools_logs/</span><span class="sc">{annotation}</span><span class="st">_repeat_comatrix.log&quot;</span>
    run:
        bigwig_string <span class="op">=</span> <span class="st">&#39; &#39;</span>.join(<span class="bu">map</span>(<span class="bu">str</span>, <span class="bu">input</span>.bigwigs_list))
        shell(<span class="st">&quot;&quot;&quot;</span>
<span class="st">            computeMatrix scale-regions \</span>
<span class="st">            -S </span><span class="sc">{bigwig_string}</span><span class="st"> \</span>
<span class="st">            -p </span><span class="sc">{threads}</span><span class="st"> \</span>
<span class="st">            -R </span><span class="sc">{input.annotation_list}</span><span class="st"> \</span>
<span class="st">            -o </span><span class="sc">{output}</span><span class="st"> \</span>
<span class="st">        --binSize 10 \</span>
<span class="st">            --regionBodyLength 500 \</span>
<span class="st">            --missingDataAsZero \</span>
<span class="st">            --beforeRegionStartLength 1000 \</span>
<span class="st">            --afterRegionStartLength 1000</span>
<span class="st">            &quot;&quot;&quot;</span>
        )

REPEATS <span class="op">=</span> [<span class="st">&quot;LINE, SINE, LTR&quot;</span>]

rule plot_profiles:
    <span class="bu">input</span>: <span class="st">&quot;comatrices/</span><span class="sc">{sample}</span><span class="st">_repeat_comatrix.gz&quot;</span>
    output: <span class="st">&quot;plots/</span><span class="sc">{sample}</span><span class="st">_repeat_profile.pdf&quot;</span>
    shell:<span class="st">&quot;&quot;&quot;</span>
<span class="st">        plotProfile -m </span><span class="sc">{input}</span><span class="st"> \</span>
<span class="st">        --plotType lines \</span>
<span class="st">        -out </span><span class="sc">{output}</span><span class="st"> \</span>
<span class="st">        --yMin 0 \</span>
<span class="st">        --yMax 5 </span>
<span class="st">        &quot;&quot;&quot;</span>
rule Sample_profile_comatrices:
    <span class="bu">input</span>:
        annotation_list<span class="op">=</span> expand(<span class="st">&quot;annotations/mm10.</span><span class="sc">{annotation}</span><span class="st">.bed&quot;</span>, annotation <span class="op">=</span> config[<span class="st">&quot;annotations&quot;</span>]),
        bigwig<span class="op">=</span> <span class="st">&quot;bigwigs/</span><span class="sc">{sample}</span><span class="st">.bw&quot;</span>
    output: <span class="st">&quot;comatrices/</span><span class="sc">{sample}</span><span class="st">_repeat_comatrix.gz&quot;</span>
    threads: <span class="dv">12</span>
    log: <span class="st">&quot;logs/deeptools_logs/</span><span class="sc">{sample}</span><span class="st">_repeat_comatrix.log&quot;</span>
    run:
        annotation_string <span class="op">=</span> <span class="st">&#39; &#39;</span>.join(<span class="bu">map</span>(<span class="bu">str</span>, <span class="bu">input</span>.annotation_list))
        shell(<span class="st">&quot;&quot;&quot;</span>
<span class="st">            computeMatrix scale-regions \</span>
<span class="st">            -S </span><span class="sc">{input.bigwig}</span><span class="st"> \</span>
<span class="st">            -p </span><span class="sc">{threads}</span><span class="st"> \</span>
<span class="st">            -R </span><span class="sc">{annotation_string}</span><span class="st"> \</span>
<span class="st">            -o </span><span class="sc">{output}</span><span class="st"> \</span>
<span class="st">            --binSize 10 \</span>
<span class="st">            --regionBodyLength 500 \</span>
<span class="st">            --missingDataAsZero \</span>
<span class="st">            --beforeRegionStartLength 1000 \</span>
<span class="st">            --afterRegionStartLength 1000</span>
<span class="st">            &quot;&quot;&quot;</span>
        )

rule Giant_comatrix:
    <span class="bu">input</span>:
        bigwigs_list<span class="op">=</span>expand(<span class="st">&quot;bigwigs/</span><span class="sc">{sample}</span><span class="st">.bw&quot;</span>, <span class="op">\</span>
          sample <span class="op">=</span> config[<span class="st">&quot;samples&quot;</span>]
          ),
        annotation_list<span class="op">=</span>expand(<span class="st">&quot;annotations/mm10.</span><span class="sc">{annotation}</span><span class="st">.bed&quot;</span>, <span class="op">\</span>
          annotation <span class="op">=</span> config[<span class="st">&quot;annotations&quot;</span>]
          )
    output: <span class="st">&quot;comatrices/all_comatrix.gz&quot;</span>
    threads: <span class="dv">12</span>
    log: <span class="st">&quot;logs/deeptools_logs/all_repeat_comatrix.log&quot;</span>
    run:
        bigwig_string <span class="op">=</span> <span class="st">&#39; &#39;</span>.join(<span class="bu">map</span>(<span class="bu">str</span>, <span class="bu">input</span>.bigwigs_list))
        annotation_string <span class="op">=</span> <span class="st">&#39; &#39;</span>.join(<span class="bu">map</span>(<span class="bu">str</span>, <span class="bu">input</span>.annotation_list))
        shell(<span class="st">&quot;&quot;&quot;</span>
<span class="st">            computeMatrix scale-regions \</span>
<span class="st">            -S </span><span class="sc">{bigwig_string}</span><span class="st"> \</span>
<span class="st">            -p </span><span class="sc">{threads}</span><span class="st"> \</span>
<span class="st">            -R </span><span class="sc">{annotation_string}</span><span class="st"> \</span>
<span class="st">            -o </span><span class="sc">{output}</span><span class="st"> \</span>
<span class="st">        --binSize 10 \</span>
<span class="st">        --missingDataAsZero \</span>
<span class="st">            --regionBodyLength 500 \</span>
<span class="st">            --beforeRegionStartLength 1000 \</span>
<span class="st">            --afterRegionStartLength 1000</span>
<span class="st">            &quot;&quot;&quot;</span>
            )


rule Giant_heatmap_plot:
    <span class="bu">input</span>: <span class="st">&quot;comatrices/all_comatrix.gz&quot;</span>
        output: <span class="st">&quot;plots/giant_heatmap.pdf&quot;</span>
        log: <span class="st">&quot;logs/deeptools_logs/giant_heatmap.log&quot;</span>
        shell:
            <span class="co">&quot;&quot;&quot;</span>
<span class="co">            plotHeatmap -m {input} \</span>
<span class="co">            --dpi 300 \</span>
<span class="co">            -o {output} \</span>
<span class="co">            --heatmapHeight 20 \</span>
<span class="co">            --zMin 0 \</span>
<span class="co">        --zMax 5 \</span>
<span class="co">        --colorMap viridis </span>
<span class="co">        &quot;&quot;&quot;</span>

rule Giant_top50_comatrix:
    <span class="bu">input</span>:
        bigwigs_list<span class="op">=</span>expand(<span class="st">&quot;bigwigs/</span><span class="sc">{sample}</span><span class="st">.bw&quot;</span>, <span class="op">\</span>
          sample <span class="op">=</span> config[<span class="st">&quot;samples&quot;</span>]
          ),
        annotation_list<span class="op">=</span>expand(<span class="st">&quot;annotations/</span><span class="sc">{annotation}</span><span class="st">_tophalf.bed&quot;</span>, <span class="op">\</span>
          annotation <span class="op">=</span> config[<span class="st">&quot;annotations&quot;</span>]
          )
    output: <span class="st">&quot;comatrices/all_top50_comatrix.gz&quot;</span>
    threads: <span class="dv">12</span>
    log: <span class="st">&quot;logs/deeptools_logs/all_repeat_comatrix.log&quot;</span>
    run:
        bigwig_string <span class="op">=</span> <span class="st">&#39; &#39;</span>.join(<span class="bu">map</span>(<span class="bu">str</span>, <span class="bu">input</span>.bigwigs_list))
        annotation_string <span class="op">=</span> <span class="st">&#39; &#39;</span>.join(<span class="bu">map</span>(<span class="bu">str</span>, <span class="bu">input</span>.annotation_list))
        shell(<span class="st">&quot;&quot;&quot;</span>
<span class="st">            computeMatrix scale-regions \</span>
<span class="st">            -S </span><span class="sc">{bigwig_string}</span><span class="st"> \</span>
<span class="st">            -p </span><span class="sc">{threads}</span><span class="st"> \</span>
<span class="st">            -R </span><span class="sc">{annotation_string}</span><span class="st"> \</span>
<span class="st">            -o </span><span class="sc">{output}</span><span class="st"> \</span>
<span class="st">            --binSize 10 \</span>
<span class="st">            --missingDataAsZero \</span>
<span class="st">            --regionBodyLength 500 \</span>
<span class="st">            --beforeRegionStartLength 1000 \</span>
<span class="st">            --afterRegionStartLength 1000</span>
<span class="st">            &quot;&quot;&quot;</span>
            )


rule Giant_top50_plot:
        <span class="bu">input</span>: <span class="st">&quot;comatrices/all_top50_comatrix.gz&quot;</span>
        output: <span class="st">&quot;plots/giant_top50_heatmap.pdf&quot;</span>
        log: <span class="st">&quot;logs/deeptools_logs/giant_heatmap.log&quot;</span>
        shell:
            <span class="co">&quot;&quot;&quot;</span>
<span class="co">            plotHeatmap -m {input} \</span>
<span class="co">            --dpi 300 \</span>
<span class="co">            -o {output} \</span>
<span class="co">            --heatmapHeight 10 \</span>
<span class="co">            --zMin 0 \</span>
<span class="co">            --zMax 5 \</span>
<span class="co">            --colorMap viridis </span>
<span class="co">            &quot;&quot;&quot;</span>

rule Annotation_global_plot:
    <span class="bu">input</span>: <span class="st">&quot;comatrices/</span><span class="sc">{annotation}</span><span class="st">_comatrix.gz&quot;</span>
    output: 
        sorted_regions <span class="op">=</span> <span class="st">&quot;plots/</span><span class="sc">{annotation}</span><span class="st">_sortreg.bed&quot;</span>,
        wg_plot <span class="op">=</span> <span class="st">&quot;plots/</span><span class="sc">{annotation}</span><span class="st">_wholegenome_heatmap.pdf&quot;</span>
    log: <span class="st">&quot;logs/deeptools_logs/</span><span class="sc">{annotation}</span><span class="st">_heatmap.log&quot;</span>
    run:
        <span class="cf">if</span> wildcards.annotation <span class="op">==</span> <span class="st">&quot;LINE&quot;</span>:
            color<span class="op">=</span><span class="st">&quot;Blues&quot;</span>
        <span class="cf">elif</span> wildcards.annotation  <span class="op">==</span> <span class="st">&quot;SINE&quot;</span>:
            color<span class="op">=</span><span class="st">&quot;Reds&quot;</span>
        <span class="cf">elif</span> wildcards.annotation  <span class="op">==</span> <span class="st">&quot;LTR&quot;</span>:
            color<span class="op">=</span><span class="st">&quot;Greens&quot;</span>
        <span class="cf">else</span>:
            color<span class="op">=</span><span class="st">&quot;magma&quot;</span>
        shell(<span class="st">&quot;&quot;&quot;</span>
<span class="st">            plotHeatmap -m </span><span class="sc">{input}</span><span class="st"> \</span>
<span class="st">                    --dpi 300 \</span>
<span class="st">                    -o </span><span class="sc">{output.wg_plot}</span><span class="st"> \</span>
<span class="st">                    --colorMap </span><span class="sc">{color}</span><span class="st"> \</span>
<span class="st">                    --outFileSortedRegions </span><span class="sc">{output.sorted_regions}</span><span class="st"> \</span>
<span class="st">            --heatmapHeight 10 \</span>
<span class="st">            --zMin 0 \</span>
<span class="st">            --zMax 3 </span>
<span class="st">                    &quot;&quot;&quot;</span>)

rule Annotation_50p_plot:
    <span class="bu">input</span>: <span class="st">&quot;comatrices/</span><span class="sc">{annotation}</span><span class="st">_subsample_comatrix.gz&quot;</span>
    output: <span class="st">&quot;plots/</span><span class="sc">{annotation}</span><span class="st">_Top50p_heatmap.pdf&quot;</span>
    log: <span class="st">&quot;logs/deeptools_logs/</span><span class="sc">{annotation}</span><span class="st">_heatmap.log&quot;</span>
    run:
        <span class="cf">if</span> wildcards.annotation <span class="op">==</span> <span class="st">&quot;LINE&quot;</span>:
            color<span class="op">=</span><span class="st">&quot;Blues&quot;</span>
        <span class="cf">elif</span> wildcards.annotation  <span class="op">==</span> <span class="st">&quot;SINE&quot;</span>:
            color<span class="op">=</span><span class="st">&quot;Reds&quot;</span>
        <span class="cf">elif</span> wildcards.annotation  <span class="op">==</span> <span class="st">&quot;LTR&quot;</span>:
            color<span class="op">=</span><span class="st">&quot;Greens&quot;</span>
        <span class="cf">else</span>:
            color<span class="op">=</span><span class="st">&quot;Spectral&quot;</span>
            
        shell(<span class="st">&quot;&quot;&quot;plotHeatmap -m </span><span class="sc">{input}</span><span class="st"> \</span>
<span class="st">            --dpi 300 \</span>
<span class="st">            -o </span><span class="sc">{output}</span><span class="st"> \</span>
<span class="st">            --colorMap </span><span class="sc">{color}</span><span class="st"> \</span>
<span class="st">            --outFileSortedRegions \</span>
<span class="st">            plots/</span><span class="sc">{wildcards.annotation}</span><span class="st">_plotsubsample.bed \</span>
<span class="st">            --heatmapHeight 5 \</span>
<span class="st">            --zMin 0 \</span>
<span class="st">            --zMax 3</span>
<span class="st">            &quot;&quot;&quot;</span>)


rule subsample_matrix:
    <span class="bu">input</span>:
        bigwigs_list<span class="op">=</span>expand(<span class="st">&quot;bigwigs/</span><span class="sc">{sample}</span><span class="st">.bw&quot;</span>, sample <span class="op">=</span> config[<span class="st">&quot;samples&quot;</span>]),
        annotation_list<span class="op">=</span><span class="st">&quot;annotations/</span><span class="sc">{annotation}</span><span class="st">_tophalf.bed&quot;</span>
    output: 
        <span class="co">&quot;comatrices/{annotation}_subsample_comatrix.gz&quot;</span>
    threads: <span class="dv">12</span>
    log: <span class="st">&quot;logs/deeptools_logs/</span><span class="sc">{annotation}</span><span class="st">_repeat_comatrix.log&quot;</span>
    run:
        bigwig_string <span class="op">=</span> <span class="st">&#39; &#39;</span>.join(<span class="bu">map</span>(<span class="bu">str</span>, <span class="bu">input</span>.bigwigs_list))
        shell(<span class="st">&quot;&quot;&quot;</span>
<span class="st">            computeMatrix scale-regions \</span>
<span class="st">            -S </span><span class="sc">{bigwig_string}</span><span class="st"> \</span>
<span class="st">            -p </span><span class="sc">{threads}</span><span class="st"> \</span>
<span class="st">            -R </span><span class="sc">{input.annotation_list}</span><span class="st"> \</span>
<span class="st">            -o </span><span class="sc">{output}</span><span class="st"> \</span>
<span class="st">            --binSize 50 \</span>
<span class="st">            --regionBodyLength 500 \</span>
<span class="st">            --missingDataAsZero \</span>
<span class="st">            --beforeRegionStartLength 1000 \</span>
<span class="st">            --afterRegionStartLength 1000</span>
<span class="st">            &quot;&quot;&quot;</span>)

rule subsample: <span class="co">#this gives us a comatrix with the top 50% of the genome so we don&#39;t create massive plots</span>
    <span class="bu">input</span>: <span class="st">&quot;plots/</span><span class="sc">{annotation}</span><span class="st">_sortreg.bed&quot;</span>
    output: <span class="st">&quot;annotations/</span><span class="sc">{annotation}</span><span class="st">_tophalf.bed&quot;</span>
    shell:
        <span class="co">&quot;&quot;&quot;</span>
<span class="co">        head -$(expr $(wc -l plots/{wildcards.annotation}_sortreg.bed \</span>
<span class="co">        | xargs echo | cut -d&quot; &quot; -f1 ) / 2 ) \</span>
<span class="co">        plots/{wildcards.annotation}_sortreg.bed &gt; {output}</span>
<span class="co">        &quot;&quot;&quot;</span>

rule lncRNA3_plot:
    <span class="bu">input</span>: <span class="st">&quot;comatrices/lncRNA3_comatrix.gz&quot;</span>
    output: <span class="st">&quot;plots/lncRNA3_heatmap.pdf&quot;</span>
    shell:<span class="st">&quot;&quot;&quot;</span>
<span class="st">        plotHeatmap -m comatrices/lncRNA3_comatrix.gz \</span>
<span class="st">        --dpi 300 \</span>
<span class="st">        -o </span><span class="sc">{output}</span><span class="st"> \</span>
<span class="st">        --colorMap viridis \</span>
<span class="st">        --heatmapHeight 20 \</span>
<span class="st">        --kmeans 3 \</span>
<span class="st">        --zMin 0 \</span>
<span class="st">        --zMax 5 \</span>
<span class="st">        --outFileSortedRegions plots/lncRNA3_sortreg.bed \</span>
<span class="st">        --outFileNameMatrix plots/lncRNA_wg_matrix.tab</span>
<span class="st">        &quot;&quot;&quot;</span>


rule lncRNA3_comatrix:
    <span class="bu">input</span>:
        bigwigs_list<span class="op">=</span>expand(<span class="st">&quot;bigwigs/</span><span class="sc">{sample}</span><span class="st">.bw&quot;</span>, sample <span class="op">=</span> config[<span class="st">&quot;samples&quot;</span>]),
        annotation<span class="op">=</span> <span class="st">&quot;annotations/mm10.lncAndGene.bed&quot;</span>
    output: <span class="st">&quot;comatrices/lncRNA3_comatrix.gz&quot;</span>
    threads: <span class="dv">12</span>
    log: <span class="st">&quot;logs/deeptools_logs/lncRNA3_comatrix.log&quot;</span>
    run:
        bigwig_string <span class="op">=</span> <span class="st">&#39; &#39;</span>.join(<span class="bu">map</span>(<span class="bu">str</span>, <span class="bu">input</span>.bigwigs_list))
        shell(<span class="st">&quot;&quot;&quot;</span>
<span class="st">            computeMatrix scale-regions \</span>
<span class="st">            -S </span><span class="sc">{input.bigwigs_list}</span><span class="st"> \</span>
<span class="st">            -p </span><span class="sc">{threads}</span><span class="st"> \</span>
<span class="st">            -R </span><span class="sc">{input.annotation}</span><span class="st"> \</span>
<span class="st">            -o </span><span class="sc">{output}</span><span class="st"> \</span>
<span class="st">            --binSize 10 \</span>
<span class="st">        --missingDataAsZero \</span>
<span class="st">            --regionBodyLength 1000 \</span>
<span class="st">            --beforeRegionStartLength 500 \</span>
<span class="st">            --afterRegionStartLength 5000</span>
<span class="st">            &quot;&quot;&quot;</span>)

rule lncRNA_body_plot:
    <span class="bu">input</span>: <span class="st">&quot;comatrices/lncRNA_body_comatrix.gz&quot;</span>
    output: <span class="st">&quot;plots/lncRNA_wg_body_heatmap.pdf&quot;</span>
    shell:<span class="st">&quot;&quot;&quot;</span>
<span class="st">        plotHeatmap -m </span><span class="sc">{input}</span><span class="st"> \</span>
<span class="st">        --dpi 300 \</span>
<span class="st">        -o </span><span class="sc">{output}</span><span class="st"> \</span>
<span class="st">        --colorMap viridis \</span>
<span class="st">        --heatmapHeight 20 \</span>
<span class="st">        --zMin 0 \</span>
<span class="st">        --zMax 5 \</span>
<span class="st">        --outFileSortedRegions plots/lncRNA_bod_sortreg.bed \</span>
<span class="st">        --outFileNameMatrix plots/lncRNA_bod_wg_matrix.tab</span>
<span class="st">        &quot;&quot;&quot;</span>

rule lncRNA_body_comatrix:
    <span class="bu">input</span>:
        bigwigs_list<span class="op">=</span>expand(<span class="st">&quot;bigwigs/</span><span class="sc">{sample}</span><span class="st">.bw&quot;</span>, sample <span class="op">=</span> config[<span class="st">&quot;samples&quot;</span>]),
        annotation<span class="op">=</span> <span class="st">&quot;annotations/mm10.lncAndGene.bed&quot;</span>
    output: <span class="st">&quot;comatrices/lncRNA_body_comatrix.gz&quot;</span>
    threads: <span class="dv">12</span>
    log: <span class="st">&quot;logs/deeptools_logs/lncRNAbody_comatrix.log&quot;</span>
    run:
        bigwig_string <span class="op">=</span> <span class="st">&#39; &#39;</span>.join(<span class="bu">map</span>(<span class="bu">str</span>, <span class="bu">input</span>.bigwigs_list))
        shell(<span class="st">&quot;&quot;&quot;</span>
<span class="st">            computeMatrix scale-regions \</span>
<span class="st">            -S </span><span class="sc">{bigwig_string}</span><span class="st"> \</span>
<span class="st">            -p </span><span class="sc">{threads}</span><span class="st"> \</span>
<span class="st">            -R </span><span class="sc">{input.annotation}</span><span class="st"> \</span>
<span class="st">            -o </span><span class="sc">{output}</span><span class="st"> \</span>
<span class="st">        --missingDataAsZero \</span>
<span class="st">        --binSize 10 \</span>
<span class="st">            --regionBodyLength 1000 \</span>
<span class="st">            --beforeRegionStartLength 1000 \</span>
<span class="st">            --afterRegionStartLength 1000</span>
<span class="st">            &quot;&quot;&quot;</span>
            )</code></pre></div>
</div>
<div id="lions-pipeline" class="section level2">
<h2><span class="header-section-number">6.6</span> LIONS Pipeline</h2>
<p>LIONS was cloned from <a href="https://github.com/ababaian/LIONS" class="uri">https://github.com/ababaian/LIONS</a> to run on CUBI. As LIONS was written for a different cluster environment, both the loop command and qsub command was modified in the <code>eastlion.sh</code> script, in addition to setting hardware-specific parameters in <code>system.ctrl</code>. Total chimeric transcripts were plotted by plotting data from the resulting <code>.lions</code> output file using the following Rscript:</p>
<p><code>ExploreChimeras.R</code>:</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw">Sys.setenv</span>(<span class="dt">http_proxy =</span> <span class="st">&quot;http://proxy.charite.de:8080&quot;</span>)
<span class="kw">setwd</span>(<span class="st">&quot;~/Bench/&quot;</span>)

<span class="kw">library</span>(ggplot2)
<span class="kw">library</span>(biomaRt)
<span class="kw">library</span>(dplyr)

## Read in Data
lions &lt;-<span class="st"> </span><span class="kw">read.table</span>(<span class="st">&quot;HP1_master.lions&quot;</span>, <span class="dt">sep =</span> <span class="st">&quot;</span><span class="ch">\t</span><span class="st">&quot;</span>, <span class="dt">header =</span> <span class="ot">TRUE</span>)

lions<span class="op">$</span>RepeatClass_F &lt;-<span class="st"> </span><span class="kw">factor</span>(<span class="kw">sub</span>(<span class="st">&quot;.*?:([a-zA-Z0-9]+):.*&quot;</span>, <span class="st">&quot;</span><span class="ch">\\</span><span class="st">1&quot;</span>, 
    lions<span class="op">$</span>repeatName, <span class="dt">perl =</span> <span class="ot">TRUE</span>), <span class="dt">levels =</span> <span class="kw">c</span>(<span class="st">&quot;LINE&quot;</span>, <span class="st">&quot;SINE&quot;</span>, 
    <span class="st">&quot;LTR&quot;</span>, <span class="st">&quot;DNA&quot;</span>))
lions<span class="op">$</span>ensembl_transcript_id &lt;-<span class="st"> </span><span class="kw">sub</span>(<span class="st">&quot;.*:&quot;</span>, <span class="st">&quot;&quot;</span>, lions<span class="op">$</span>transcriptID)
lions<span class="op">$</span>ensembl_transcript_id &lt;-<span class="st"> </span><span class="kw">sub</span>(<span class="st">&quot;([A-Za-z0-9]+).*&quot;</span>, <span class="st">&quot;</span><span class="ch">\\</span><span class="st">1&quot;</span>, 
    lions<span class="op">$</span>ensembl_transcript_id)
lions<span class="op">$</span>ensembl_transcript_id &lt;-<span class="st"> </span><span class="kw">sub</span>(<span class="st">&quot;BKO&quot;</span>, <span class="st">&quot;&quot;</span>, lions<span class="op">$</span>ensembl_transcript_id)
lions<span class="op">$</span>ensembl_transcript_id &lt;-<span class="st"> </span><span class="kw">sub</span>(<span class="st">&quot;gKO&quot;</span>, <span class="st">&quot;&quot;</span>, lions<span class="op">$</span>ensembl_transcript_id)
lions<span class="op">$</span>ensembl_transcript_id &lt;-<span class="st"> </span><span class="kw">sub</span>(<span class="st">&quot;WT&quot;</span>, <span class="st">&quot;&quot;</span>, lions<span class="op">$</span>ensembl_transcript_id)
lions<span class="op">$</span>ensembl_transcript_id &lt;-<span class="st"> </span><span class="kw">sub</span>(<span class="st">&quot;DKO&quot;</span>, <span class="st">&quot;&quot;</span>, lions<span class="op">$</span>ensembl_transcript_id)

<span class="co"># Genotype &amp; Age</span>
lions[<span class="st">&quot;Genotype_Age&quot;</span>] &lt;-<span class="st"> </span><span class="ot">NA</span>
lions[<span class="st">&quot;GT&quot;</span>] &lt;-<span class="st"> </span><span class="ot">NA</span>
lions[<span class="st">&quot;Condition&quot;</span>] &lt;-<span class="st"> &quot;unstimulated&quot;</span>
lions<span class="op">$</span>Condition[<span class="kw">grepl</span>(<span class="st">&quot;stim&quot;</span>, lions<span class="op">$</span>LIBRARY)] &lt;-<span class="st"> &quot;stimulated&quot;</span>
lions<span class="op">$</span>Condition &lt;-<span class="st"> </span><span class="kw">factor</span>(lions<span class="op">$</span>Condition, <span class="dt">levels =</span> <span class="kw">c</span>(<span class="st">&quot;unstimulated&quot;</span>, 
    <span class="st">&quot;stimulated&quot;</span>))

WT_young &lt;-<span class="st"> </span><span class="kw">subset</span>(lions, <span class="kw">grepl</span>(<span class="st">&quot;WT_young&quot;</span>, lions<span class="op">$</span>LIBRARY))
WT_young[<span class="st">&quot;Genotype_Age&quot;</span>] &lt;-<span class="st"> &quot;WT Young&quot;</span>
WT_young[<span class="st">&quot;GT&quot;</span>] &lt;-<span class="st"> &quot;WT&quot;</span>

WT_old &lt;-<span class="st"> </span><span class="kw">subset</span>(lions, <span class="kw">grepl</span>(<span class="st">&quot;WT_old&quot;</span>, lions<span class="op">$</span>LIBRARY))
WT_old[<span class="st">&quot;Genotype_Age&quot;</span>] &lt;-<span class="st"> &quot;WT Aged&quot;</span>
WT_old[<span class="st">&quot;GT&quot;</span>] &lt;-<span class="st"> &quot;WT&quot;</span>

HP1B_young &lt;-<span class="st"> </span><span class="kw">subset</span>(lions, <span class="kw">grepl</span>(<span class="st">&quot;BKO_young&quot;</span>, lions<span class="op">$</span>LIBRARY))
HP1B_young[<span class="st">&quot;Genotype_Age&quot;</span>] &lt;-<span class="st"> &quot;HP1BKO Young&quot;</span>
HP1B_young[<span class="st">&quot;GT&quot;</span>] &lt;-<span class="st"> &quot;HP1BKO&quot;</span>

HP1B_old &lt;-<span class="st"> </span><span class="kw">subset</span>(lions, <span class="kw">grepl</span>(<span class="st">&quot;BKO_old&quot;</span>, lions<span class="op">$</span>LIBRARY))
HP1B_old[<span class="st">&quot;Genotype_Age&quot;</span>] &lt;-<span class="st"> &quot;HP1BKO Aged&quot;</span>
HP1B_old[<span class="st">&quot;GT&quot;</span>] &lt;-<span class="st"> &quot;HP1BKO&quot;</span>

HP1g_young &lt;-<span class="st"> </span><span class="kw">subset</span>(lions, <span class="kw">grepl</span>(<span class="st">&quot;gKO_young&quot;</span>, lions<span class="op">$</span>LIBRARY))
HP1g_young[<span class="st">&quot;Genotype_Age&quot;</span>] &lt;-<span class="st"> &quot;HP1gKO Young&quot;</span>
HP1g_young[<span class="st">&quot;GT&quot;</span>] &lt;-<span class="st"> &quot;HP1gKO&quot;</span>

HP1g_old &lt;-<span class="st"> </span><span class="kw">subset</span>(lions, <span class="kw">grepl</span>(<span class="st">&quot;gKO_old&quot;</span>, lions<span class="op">$</span>LIBRARY))
HP1g_old[<span class="st">&quot;Genotype_Age&quot;</span>] &lt;-<span class="st"> &quot;HP1gKO Aged&quot;</span>
HP1g_old[<span class="st">&quot;GT&quot;</span>] &lt;-<span class="st"> &quot;HP1gKO&quot;</span>

DKO_young &lt;-<span class="st"> </span><span class="kw">subset</span>(lions, <span class="kw">grepl</span>(<span class="st">&quot;DKO_young&quot;</span>, lions<span class="op">$</span>LIBRARY))
DKO_young[<span class="st">&quot;Genotype_Age&quot;</span>] &lt;-<span class="st"> &quot;HP1DKO Young&quot;</span>
DKO_young[<span class="st">&quot;GT&quot;</span>] &lt;-<span class="st"> &quot;HP1DKO&quot;</span>

DKO_old &lt;-<span class="st"> </span><span class="kw">subset</span>(lions, <span class="kw">grepl</span>(<span class="st">&quot;DKO_old&quot;</span>, lions<span class="op">$</span>LIBRARY))
DKO_old[<span class="st">&quot;Genotype_Age&quot;</span>] &lt;-<span class="st"> &quot;HP1DKO Aged&quot;</span>
DKO_old[<span class="st">&quot;GT&quot;</span>] &lt;-<span class="st"> &quot;HP1DKO&quot;</span>

lions &lt;-<span class="st"> </span><span class="kw">rbind</span>(WT_young, WT_old, HP1B_young, HP1B_old, HP1g_young, 
    HP1g_old, DKO_young, DKO_old)
lions<span class="op">$</span>GT_AGE_F &lt;-<span class="st"> </span><span class="kw">factor</span>(lions<span class="op">$</span>Genotype_Age, <span class="dt">levels =</span> <span class="kw">c</span>(<span class="st">&quot;WT Young&quot;</span>, 
    <span class="st">&quot;WT Aged&quot;</span>, <span class="st">&quot;HP1BKO Young&quot;</span>, <span class="st">&quot;HP1BKO Aged&quot;</span>, <span class="st">&quot;HP1gKO Young&quot;</span>, 
    <span class="st">&quot;HP1gKO Aged&quot;</span>, <span class="st">&quot;HP1DKO Young&quot;</span>, <span class="st">&quot;HP1DKO Aged&quot;</span>))
lions<span class="op">$</span>GT_F &lt;-<span class="st"> </span><span class="kw">factor</span>(lions<span class="op">$</span>GT, <span class="dt">levels =</span> <span class="kw">c</span>(<span class="st">&quot;WT&quot;</span>, <span class="st">&quot;HP1BKO&quot;</span>, <span class="st">&quot;HP1gKO&quot;</span>, 
    <span class="st">&quot;HP1DKO&quot;</span>))


<span class="co"># AGE</span>
lions[<span class="st">&quot;Age&quot;</span>] &lt;-<span class="st"> </span><span class="ot">NA</span>
OLD &lt;-<span class="st"> </span><span class="kw">subset</span>(lions, <span class="kw">grepl</span>(<span class="st">&quot;old&quot;</span>, lions<span class="op">$</span>LIBRARY))
OLD[<span class="st">&quot;Age&quot;</span>] &lt;-<span class="st"> &quot;Aged&quot;</span>
YOUNG &lt;-<span class="st"> </span><span class="kw">subset</span>(lions, <span class="kw">grepl</span>(<span class="st">&quot;young&quot;</span>, lions<span class="op">$</span>LIBRARY))
YOUNG[<span class="st">&quot;Age&quot;</span>] &lt;-<span class="st"> &quot;Young&quot;</span>
lions &lt;-<span class="st"> </span><span class="kw">rbind</span>(OLD, YOUNG)

lions<span class="op">$</span>AGE_F &lt;-<span class="st"> </span><span class="kw">factor</span>(lions<span class="op">$</span>Age, <span class="dt">levels =</span> <span class="kw">c</span>(<span class="st">&quot;Young&quot;</span>, <span class="st">&quot;Aged&quot;</span>))
lions<span class="op">$</span>ensembl_gene_id &lt;-<span class="st"> </span><span class="kw">as.character</span>(lions<span class="op">$</span>ensembl_transcript_id)

## biomaRt Annotations ( to add more attributes use
## listAttributes(mart) )
mart &lt;-<span class="st"> </span><span class="kw">useDataset</span>(<span class="st">&quot;mmusculus_gene_ensembl&quot;</span>, <span class="kw">useMart</span>(<span class="st">&quot;ensembl&quot;</span>))
ID_QueryAttributes =<span class="st"> </span><span class="kw">c</span>(<span class="st">&quot;ensembl_transcript_id&quot;</span>, <span class="st">&quot;external_gene_name&quot;</span>, 
    <span class="st">&quot;gene_biotype&quot;</span>, <span class="st">&quot;description&quot;</span>)
ID_QueryFilters =<span class="st"> &quot;ensembl_transcript_id&quot;</span>


tmp2 &lt;-<span class="st"> </span><span class="kw">data.frame</span>(<span class="dt">ensembl_transcript_id =</span> <span class="kw">as.character</span>(lions<span class="op">$</span>ensembl_transcript_id))

long_combined_ids &lt;-<span class="st"> </span><span class="kw">getBM</span>(<span class="dt">attributes =</span> ID_QueryAttributes, <span class="dt">filters =</span> ID_QueryFilters, 
    <span class="dt">values =</span> tmp2[, <span class="st">&quot;ensembl_transcript_id&quot;</span>], <span class="dt">mart =</span> mart)


lions =<span class="st"> </span><span class="kw">left_join</span>(<span class="dt">x =</span> lions, <span class="dt">y =</span> long_combined_ids, <span class="dt">by =</span> <span class="st">&quot;ensembl_transcript_id&quot;</span>)

<span class="co"># sometimes annotation duplicates rows</span>
lions &lt;-<span class="st"> </span><span class="kw">distinct</span>(lions)

<span class="co"># Move DeNovo Annotations over</span>
lions<span class="op">$</span>external_gene_name[<span class="kw">is.na</span>(lions<span class="op">$</span>external_gene_name)] &lt;-<span class="st"> </span><span class="kw">as.character</span>(lions<span class="op">$</span>RefID[<span class="kw">is.na</span>(lions<span class="op">$</span>external_gene_name)])

<span class="co"># When no external_gene_name, use coordinates to identify</span>
lions<span class="op">$</span>external_gene_name[<span class="kw">grepl</span>(<span class="st">&quot;[.]&quot;</span>, lions<span class="op">$</span>external_gene_name)] &lt;-<span class="st"> </span><span class="kw">as.character</span>(lions<span class="op">$</span>coordinates[<span class="kw">grepl</span>(<span class="st">&quot;[.]&quot;</span>, 
    lions<span class="op">$</span>external_gene_name)])

<span class="co"># Remove NA in RepeatClass_F</span>
lions &lt;-<span class="st"> </span><span class="kw">subset</span>(lions, RepeatClass_F <span class="op">%in%</span><span class="st"> </span><span class="kw">c</span>(<span class="st">&quot;LINE&quot;</span>, <span class="st">&quot;SINE&quot;</span>, <span class="st">&quot;LTR&quot;</span>, 
    <span class="st">&quot;DNA&quot;</span>))

<span class="co"># Some external_gene_name</span>

## Make one table for stimulation experiment and one table for
## aging STIMTABLE
lions.stim &lt;-<span class="st"> </span><span class="kw">subset</span>(lions, GT_AGE_F <span class="op">%in%</span><span class="st"> </span><span class="kw">c</span>(<span class="st">&quot;WT Young&quot;</span>, <span class="st">&quot;HP1DKO Young&quot;</span>))

<span class="co"># REMOVE STIM CONDITIONS FROM LIONS table</span>
lions &lt;-<span class="st"> </span><span class="kw">subset</span>(lions, Condition <span class="op">==</span><span class="st"> &quot;unstimulated&quot;</span>)

<span class="co"># Filter out exons not transcribed</span>
lions &lt;-<span class="st"> </span><span class="kw">subset</span>(lions, lions<span class="op">$</span>ExonRPKM <span class="op">&gt;</span><span class="st"> </span><span class="dv">0</span>)


<span class="co"># Reference lines for Comatrices</span>
ExonRPKM_youngWT_mean &lt;-<span class="st"> </span><span class="kw">tapply</span>(lions<span class="op">$</span>ExonRPKM[lions<span class="op">$</span>GT_AGE_F <span class="op">==</span><span class="st"> </span>
<span class="st">    &quot;WT Young&quot;</span>], lions<span class="op">$</span>RepeatClass_F[lions<span class="op">$</span>GT_AGE_F <span class="op">==</span><span class="st"> &quot;WT Young&quot;</span>], 
    mean)
RepeatRPKM_youngWT_mean &lt;-<span class="st"> </span><span class="kw">tapply</span>(lions<span class="op">$</span>RepeatRPKM[lions<span class="op">$</span>GT_AGE_F <span class="op">==</span><span class="st"> </span>
<span class="st">    &quot;WT Young&quot;</span>], lions<span class="op">$</span>RepeatClass_F[lions<span class="op">$</span>GT_AGE_F <span class="op">==</span><span class="st"> &quot;WT Young&quot;</span>], 
    mean)

LINE_mean_ExonRPKM_Y_WT &lt;-<span class="st"> </span>ExonRPKM_youngWT_mean[<span class="dv">1</span>]
SINE_mean_ExonRPKM_Y_WT &lt;-<span class="st"> </span>ExonRPKM_youngWT_mean[<span class="dv">2</span>]
LTR_mean_ExonRPKM_Y_WT &lt;-<span class="st"> </span>ExonRPKM_youngWT_mean[<span class="dv">3</span>]
DNA_mean_ExonRPKM_Y_WT &lt;-<span class="st"> </span>ExonRPKM_youngWT_mean[<span class="dv">4</span>]

LINE_mean_RepeatRPKM_Y_WT &lt;-<span class="st"> </span>RepeatRPKM_youngWT_mean[<span class="dv">1</span>]
SINE_mean_RepeatRPKM_Y_WT &lt;-<span class="st"> </span>RepeatRPKM_youngWT_mean[<span class="dv">2</span>]
LTR_mean_RepeatRPKM_Y_WT &lt;-<span class="st"> </span>RepeatRPKM_youngWT_mean[<span class="dv">3</span>]
DNA_mean_RepeatRPKM_Y_MT &lt;-<span class="st"> </span>RepeatRPKM_youngWT_mean[<span class="dv">4</span>]

<span class="co"># Plot Chimera Repeat-Exon Co-expression</span>
Superplot_hex &lt;-<span class="st"> </span><span class="kw">ggplot</span>(lions, <span class="kw">aes</span>(<span class="dt">x =</span> RepeatRPKM, <span class="dt">y =</span> ExonRPKM, 
    <span class="dt">colour =</span> GT_F, <span class="dt">label =</span> RefID)) <span class="op">+</span><span class="st"> </span><span class="co"># geom_point(size = 0.1) +</span>
<span class="kw">geom_hex</span>() <span class="op">+</span><span class="st"> </span><span class="co"># geom_density_2d() + xlim(c(0,200)) +</span>
<span class="kw">scale_y_log10</span>(<span class="dt">limits =</span> <span class="kw">c</span>(<span class="fl">0.001</span>, <span class="dv">1000</span>)) <span class="op">+</span><span class="st"> </span><span class="kw">scale_x_log10</span>(<span class="dt">limits =</span> <span class="kw">c</span>(<span class="fl">0.001</span>, 
    <span class="dv">1000</span>)) <span class="op">+</span><span class="st"> </span><span class="kw">scale_color_manual</span>(<span class="dt">values =</span> <span class="kw">c</span>(<span class="st">&quot;black&quot;</span>, <span class="st">&quot;orange&quot;</span>, 
    <span class="st">&quot;grey&quot;</span>, <span class="st">&quot;maroon&quot;</span>)) <span class="op">+</span><span class="st"> </span><span class="kw">facet_grid</span>(GT_F <span class="op">~</span><span class="st"> </span>RepeatClass_F <span class="op">+</span><span class="st"> </span>AGE_F) <span class="op">+</span><span class="st"> </span>
<span class="st">    </span><span class="kw">geom_hline</span>(<span class="dt">data =</span> <span class="kw">filter</span>(lions, RepeatClass_F <span class="op">==</span><span class="st"> &quot;LINE&quot;</span>), 
        <span class="kw">aes</span>(<span class="dt">yintercept =</span> LINE_mean_ExonRPKM_Y_WT)) <span class="op">+</span><span class="st"> </span><span class="kw">geom_hline</span>(<span class="dt">data =</span> <span class="kw">filter</span>(lions, 
    RepeatClass_F <span class="op">==</span><span class="st"> &quot;SINE&quot;</span>), <span class="kw">aes</span>(<span class="dt">yintercept =</span> SINE_mean_ExonRPKM_Y_WT)) <span class="op">+</span><span class="st"> </span>
<span class="st">    </span><span class="kw">geom_hline</span>(<span class="dt">data =</span> <span class="kw">filter</span>(lions, RepeatClass_F <span class="op">==</span><span class="st"> &quot;LTR&quot;</span>), 
        <span class="kw">aes</span>(<span class="dt">yintercept =</span> LTR_mean_ExonRPKM_Y_WT)) <span class="op">+</span><span class="st"> </span><span class="kw">geom_hline</span>(<span class="dt">data =</span> <span class="kw">filter</span>(lions, 
    RepeatClass_F <span class="op">==</span><span class="st"> &quot;DNA&quot;</span>), <span class="kw">aes</span>(<span class="dt">yintercept =</span> DNA_mean_ExonRPKM_Y_WT)) <span class="op">+</span><span class="st"> </span>
<span class="st">    </span><span class="kw">geom_vline</span>(<span class="dt">data =</span> <span class="kw">filter</span>(lions, RepeatClass_F <span class="op">==</span><span class="st"> &quot;LINE&quot;</span>), 
        <span class="kw">aes</span>(<span class="dt">xintercept =</span> LINE_mean_RepeatRPKM_Y_WT)) <span class="op">+</span><span class="st"> </span><span class="kw">geom_vline</span>(<span class="dt">data =</span> <span class="kw">filter</span>(lions, 
    RepeatClass_F <span class="op">==</span><span class="st"> &quot;SINE&quot;</span>), <span class="kw">aes</span>(<span class="dt">xintercept =</span> SINE_mean_RepeatRPKM_Y_WT)) <span class="op">+</span><span class="st"> </span>
<span class="st">    </span><span class="kw">geom_vline</span>(<span class="dt">data =</span> <span class="kw">filter</span>(lions, RepeatClass_F <span class="op">==</span><span class="st"> &quot;LTR&quot;</span>), 
        <span class="kw">aes</span>(<span class="dt">xintercept =</span> LTR_mean_RepeatRPKM_Y_WT)) <span class="op">+</span><span class="st"> </span><span class="kw">geom_vline</span>(<span class="dt">data =</span> <span class="kw">filter</span>(lions, 
    RepeatClass_F <span class="op">==</span><span class="st"> &quot;DNA&quot;</span>), <span class="kw">aes</span>(<span class="dt">xintercept =</span> DNA_mean_RepeatRPKM_Y_MT)) <span class="op">+</span><span class="st"> </span>
<span class="st">    </span><span class="co"># scale_fill_distiller() +</span>
<span class="kw">theme_minimal</span>()
<span class="co"># geom_text(aes(label=ifelse(ExonRPKM &gt; 75 &amp; RepeatRPKM &gt; 12,</span>
<span class="co"># as.character(RefID),&#39;&#39;)),hjust=0.1,vjust = -0.5)</span>
Superplot_hex</code></pre></div>
</div>
<div id="analysis-of-reduced-read-bisulphite-sequencing-rrbs" class="section level2">
<h2><span class="header-section-number">6.7</span> Analysis of Reduced-Read Bisulphite Sequencing (RRBS)</h2>
<p>RRBS data was processed locally using Bismark <span class="citation">(Krueger and Andrews <a href="#ref-KruegerBismarkflexiblealigner2011a">2011</a>)</span> and methylkit <span class="citation">(Akalin et al. <a href="#ref-AkalinmethylKitcomprehensivepackage2012">2012</a>)</span>. As so far there have only been two RRBS samples to analyse, preprocessing by Bismark was done using a single shell script:</p>
<div class="sourceCode"><pre class="sourceCode bash"><code class="sourceCode bash"><span class="ex">trim_galore</span> --rrbs S07665.Hp1_WT.fastq<span class="kw">;</span> 
<span class="ex">trim_galore</span> --rrbs S07666.HP1_cTKO.fastq

<span class="co"># index mm10 for bisulfite genome</span>
<span class="co"># default of bismark now bowtie 2, </span>
<span class="co"># didn’t have enough memory at the time for bowtie2, </span>
<span class="co"># and dataset is 50bp short reads ...</span>
<span class="co"># so used bowtie1 ;) </span>

<span class="ex">./bismark_genome_preparation</span> --verbose \
  --bowtie1 ~/mm10/Mus_musculus.GRCm38.dna.primary_assembly.fa

<span class="co">## align and call methylation with bismark</span>

<span class="ex">./bismark</span> --bowtie1 -n 1 -l 45 \
  --genome_folder ~/mm10/ S07666.HP1_cTKO_trimmed.fq \
  -o ~/HP1cTKO/Bismark_Output/HP1cTKO/ <span class="kw">;</span> 
  <span class="ex">./bismark</span> --bowtie1 -n 1 -l 45 \
  --genome_folder ~/mm10/ S07665.Hp1_WT_trimmed.fq \
  -o ~/Documents/Genomics/HP1cTKO/Bismark_Output/WT/ </code></pre></div>
<div id="methylkit-and-circos-plot" class="section level3">
<h3><span class="header-section-number">6.7.1</span> Methylkit and Circos Plot</h3>
<p>To annotate and quantify methylation changes, methylKit was used in the following R script:</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw">setwd</span>(<span class="st">&quot;~/Bench/&quot;</span>)

<span class="co"># SETPROXY Sys.setenv(&#39;http_proxy&#39; =</span>
<span class="co"># &#39;http://proxy.charite.de:8080&#39;) UNSET</span>
<span class="co"># Sys.setenv(&#39;http_proxy&#39; = &#39;&#39;)</span>

<span class="co"># source(&#39;http://www.bioconductor.org/biocLite.R&#39;)</span>
<span class="co"># biocLite(c(&#39;methylKit&#39;, &#39;ggbio&#39;, &#39;genomation&#39;, &#39;circlize&#39;))</span>

<span class="kw">library</span>(methylKit)

file.list =<span class="st"> </span><span class="kw">list</span>(<span class="st">&quot;S07665.Hp1_WT_trimmed_bismark.cov&quot;</span>, <span class="st">&quot;S07666.HP1_cTKO_trimmed_bismark.cov&quot;</span>)

rrbs &lt;-<span class="st"> </span><span class="kw">methRead</span>(file.list, <span class="dt">sample.id =</span> <span class="kw">list</span>(<span class="st">&quot;WT&quot;</span>, <span class="st">&quot;HP1cTKO&quot;</span>), 
    <span class="dt">assembly =</span> <span class="st">&quot;mm10&quot;</span>, <span class="dt">treatment =</span> <span class="kw">c</span>(<span class="dv">0</span>, <span class="dv">1</span>), <span class="dt">pipeline =</span> <span class="st">&quot;bismarkCoverage&quot;</span>)

WT.hist &lt;-<span class="st"> </span><span class="kw">getMethylationStats</span>(rrbs[[<span class="dv">1</span>]], <span class="dt">plot =</span> <span class="ot">TRUE</span>, <span class="dt">both.strands =</span> <span class="ot">FALSE</span>)
HP1cTKO.hist &lt;-<span class="st"> </span><span class="kw">getMethylationStats</span>(rrbs[[<span class="dv">2</span>]], <span class="dt">plot =</span> <span class="ot">TRUE</span>, <span class="dt">both.strands =</span> <span class="ot">FALSE</span>)
rrbs.a &lt;-<span class="st"> </span><span class="kw">unite</span>(rrbs, <span class="dt">destrand =</span> <span class="ot">FALSE</span>)
<span class="kw">getCorrelation</span>(rrbs.a, <span class="dt">plot =</span> <span class="ot">TRUE</span>)
rrbs.diff &lt;-<span class="st"> </span><span class="kw">calculateDiffMeth</span>(rrbs.a, <span class="dt">num.cores =</span> <span class="dv">4</span>)

<span class="co"># get hyper methylated bases that change more than 25% that</span>
<span class="co"># are statistically significant after correction to 0.01</span>
diff25p.hyper &lt;-<span class="st"> </span><span class="kw">getMethylDiff</span>(rrbs.diff, <span class="dt">difference =</span> <span class="dv">25</span>, <span class="dt">qvalue =</span> <span class="fl">0.01</span>, 
    <span class="dt">type =</span> <span class="st">&quot;hyper&quot;</span>)

<span class="co"># get hypo methylated bases that change more than 25% that</span>
<span class="co"># are statistically significant after correction to 0.01</span>
diff25p.hypo &lt;-<span class="st"> </span><span class="kw">getMethylDiff</span>(rrbs.diff, <span class="dt">difference =</span> <span class="dv">25</span>, <span class="dt">qvalue =</span> <span class="fl">0.01</span>, 
    <span class="dt">type =</span> <span class="st">&quot;hypo&quot;</span>)

<span class="co"># get all differentially methylated bases that change more</span>
<span class="co"># than 25% that are statistically significant after</span>
<span class="co"># correction to 0.01</span>
diff25p.all &lt;-<span class="st"> </span><span class="kw">getMethylDiff</span>(rrbs.diff, <span class="dt">difference =</span> <span class="dv">25</span>, <span class="dt">qvalue =</span> <span class="fl">0.01</span>)

<span class="kw">diffMethPerChr</span>(rrbs.diff, <span class="dt">plot =</span> <span class="ot">FALSE</span>, <span class="dt">qvalue.cutoff =</span> <span class="fl">0.01</span>, 
    <span class="dt">meth.cutoff =</span> <span class="dv">25</span>)

diff25p.hyper.gR &lt;-<span class="st"> </span><span class="kw">as</span>(diff25p.hyper, <span class="st">&quot;GRanges&quot;</span>)
diff25p.hypo.gR &lt;-<span class="st"> </span><span class="kw">as</span>(diff25p.hypo, <span class="st">&quot;GRanges&quot;</span>)
diff25p.all.gR &lt;-<span class="st"> </span><span class="kw">as</span>(diff25p.all, <span class="st">&quot;GRanges&quot;</span>)

<span class="kw">library</span>(ggbio)
<span class="kw">library</span>(genomation)

##################### ANNOTATIONS##############################

<span class="co"># RefSeq</span>
gene.obj &lt;-<span class="st"> </span>genomation<span class="op">::</span><span class="kw">readTranscriptFeatures</span>(<span class="st">&quot;mm10refseqgene.bed&quot;</span>, 
    <span class="dt">remove.unusual =</span> <span class="ot">TRUE</span>, <span class="dt">up.flank =</span> <span class="dv">2000</span>, <span class="dt">down.flank =</span> <span class="dv">2000</span>, 
    <span class="dt">unique.prom =</span> <span class="ot">FALSE</span>)
<span class="co"># convert to NCBI style (that used by RRBS data)</span>
newStyle &lt;-<span class="st"> </span><span class="kw">mapSeqlevels</span>(<span class="kw">seqlevels</span>(gene.obj), <span class="st">&quot;NCBI&quot;</span>)
gene.obj &lt;-<span class="st"> </span><span class="kw">renameSeqlevels</span>(gene.obj, newStyle)

<span class="co"># annotate differentially methylated CpGs with</span>
<span class="co"># promoter/exon/intron using annotation data</span>

<span class="co"># all differentially methylated annotated with Promoter,</span>
<span class="co"># intron, exon, TSS with flanks 2000-2000</span>
diff25p.anno.Refseq &lt;-<span class="st"> </span><span class="kw">annotateWithGeneParts</span>(<span class="kw">as</span>(diff25p.all, 
    <span class="st">&quot;GRanges&quot;</span>), gene.obj)
<span class="co"># all hypermethylated in HP1cTKO annotated with Promoter,</span>
<span class="co"># intron, exon, TSS with flanks 2000-2000</span>
diff25p.hyper.Refseq &lt;-<span class="st"> </span><span class="kw">annotateWithGeneParts</span>(<span class="kw">as</span>(diff25p.hyper, 
    <span class="st">&quot;GRanges&quot;</span>), gene.obj)
<span class="co"># all hypomethylated in HP1cTKO annotated with Promoter,</span>
<span class="co"># intron, exon, TSS with flanks 2000-2000</span>
diff25p.hypo.Refseq &lt;-<span class="st"> </span><span class="kw">annotateWithGeneParts</span>(<span class="kw">as</span>(diff25p.hypo, 
    <span class="st">&quot;GRanges&quot;</span>), gene.obj)

<span class="co"># promoter df for later</span>

promoter.hyper &lt;-<span class="st"> </span><span class="kw">subsetByOverlaps</span>(diff25p.hyper.gR, gene.obj<span class="op">$</span>promoters)
promoter.hyper &lt;-<span class="st"> </span><span class="kw">annotateWithGeneParts</span>(promoter.hyper, gene.obj)
promoter.hyper.df &lt;-<span class="st"> </span><span class="kw">getAssociationWithTSS</span>(promoter.hyper)
<span class="kw">write.csv</span>(promoter.hyper.df, <span class="dt">file =</span> <span class="st">&quot;HP1cTKO.promoter.hyper.disttoTSS.csv&quot;</span>)

promoter.hypo &lt;-<span class="st"> </span><span class="kw">subsetByOverlaps</span>(diff25p.hypo.gR, gene.obj<span class="op">$</span>promoters)
promoter.hypo &lt;-<span class="st"> </span><span class="kw">annotateWithGeneParts</span>(promoter.hypo, gene.obj)
promoter.hypo.df &lt;-<span class="st"> </span><span class="kw">getAssociationWithTSS</span>(promoter.hypo)
<span class="kw">write.csv</span>(promoter.hypo.df, <span class="dt">file =</span> <span class="st">&quot;HP1cTKO.promoter.hypo.disttoTSS.csv&quot;</span>)

p.diff25p.anno.Refseq &lt;-<span class="st"> </span><span class="kw">getTargetAnnotationStats</span>(diff25p.anno.Refseq, 
    <span class="dt">percentage =</span> <span class="ot">TRUE</span>, <span class="dt">precedence =</span> <span class="ot">TRUE</span>)
p.diff25p.hyper.Refseq &lt;-<span class="st"> </span><span class="kw">getTargetAnnotationStats</span>(diff25p.hyper.Refseq, 
    <span class="dt">percentage =</span> <span class="ot">TRUE</span>, <span class="dt">precedence =</span> <span class="ot">TRUE</span>)
p.diff25p.hypo.Refseq &lt;-<span class="st"> </span><span class="kw">getTargetAnnotationStats</span>(diff25p.hypo.Refseq, 
    <span class="dt">percentage =</span> <span class="ot">TRUE</span>, <span class="dt">precedence =</span> <span class="ot">TRUE</span>)

<span class="co"># Get total changes</span>
d &lt;-<span class="st"> </span><span class="kw">rbind</span>(p.diff25p.anno.Refseq, p.diff25p.hyper.Refseq, p.diff25p.hypo.Refseq, 
    <span class="kw">row.names</span>(<span class="kw">c</span>(<span class="st">&quot;Total&quot;</span>, <span class="st">&quot;Hypermethylation&quot;</span>, <span class="st">&quot;Hypomethylation&quot;</span>)))
row.vec &lt;-<span class="st"> </span><span class="kw">c</span>(<span class="st">&quot;Total&quot;</span>, <span class="st">&quot;Hypermethylation&quot;</span>, <span class="st">&quot;Hypomethylation&quot;</span>)
<span class="kw">row.names</span>(d) &lt;-<span class="st"> </span>row.vec
d &lt;-<span class="st"> </span><span class="kw">t</span>(<span class="kw">as.data.frame</span>(d))
RefseqFeat &lt;-<span class="st"> </span><span class="kw">c</span>(<span class="st">&quot;promoter&quot;</span>, <span class="st">&quot;exon&quot;</span>, <span class="st">&quot;intron&quot;</span>, <span class="st">&quot;intergenic&quot;</span>)
d &lt;-<span class="st"> </span><span class="kw">data.frame</span>(d, <span class="dt">Feature =</span> RefseqFeat)

RRBSbar &lt;-<span class="st"> </span><span class="kw">ggplot</span>(d, <span class="kw">aes</span>(<span class="dt">y =</span> Total, <span class="dt">Fill =</span> Feature)) <span class="op">+</span><span class="st"> </span><span class="kw">geom_col</span>()
RRBSbar

<span class="co"># Full Repeat Masker</span>
COLNAMES &lt;-<span class="st"> </span><span class="kw">c</span>(<span class="st">&quot;bin&quot;</span>, <span class="st">&quot;swScore&quot;</span>, <span class="st">&quot;milliDiv&quot;</span>, <span class="st">&quot;milliDel&quot;</span>, <span class="st">&quot;milliIns&quot;</span>, 
    <span class="st">&quot;genoName&quot;</span>, <span class="st">&quot;genoStart&quot;</span>, <span class="st">&quot;genoEnd&quot;</span>, <span class="st">&quot;genoLeft&quot;</span>, <span class="st">&quot;strand&quot;</span>, 
    <span class="st">&quot;repName&quot;</span>, <span class="st">&quot;repClass&quot;</span>, <span class="st">&quot;repFamily&quot;</span>, <span class="st">&quot;repStart&quot;</span>, <span class="st">&quot;repEnd&quot;</span>, 
    <span class="st">&quot;repLeft&quot;</span>, <span class="st">&quot;id&quot;</span>)
df.temp &lt;-<span class="st"> </span><span class="kw">read.table</span>(<span class="st">&quot;rmsk.txt&quot;</span>, <span class="dt">col.names =</span> COLNAMES)
rmsk &lt;-<span class="st"> </span><span class="kw">makeGRangesListFromDataFrame</span>(df.temp, <span class="dt">keep.extra.columns =</span> <span class="ot">TRUE</span>, 
    <span class="dt">seqnames.field =</span> <span class="st">&quot;genoName&quot;</span>, <span class="dt">start.field =</span> <span class="st">&quot;genoStart&quot;</span>, <span class="dt">end.field =</span> <span class="st">&quot;genoEnd&quot;</span>, 
    <span class="dt">strand.field =</span> <span class="st">&quot;strand&quot;</span>, <span class="dt">starts.in.df.are.0based =</span> <span class="ot">TRUE</span>)
<span class="co"># throw away random chromosomes</span>
chr.vec &lt;-<span class="st"> </span><span class="kw">c</span>(<span class="st">&quot;chr1&quot;</span>, <span class="st">&quot;chr2&quot;</span>, <span class="st">&quot;chr3&quot;</span>, <span class="st">&quot;chr4&quot;</span>, <span class="st">&quot;chr5&quot;</span>, <span class="st">&quot;chr6&quot;</span>, 
    <span class="st">&quot;chr7&quot;</span>, <span class="st">&quot;chr8&quot;</span>, <span class="st">&quot;chr9&quot;</span>, <span class="st">&quot;chr10&quot;</span>, <span class="st">&quot;chr11&quot;</span>, <span class="st">&quot;chr12&quot;</span>, <span class="st">&quot;chr13&quot;</span>, 
    <span class="st">&quot;chr14&quot;</span>, <span class="st">&quot;chr15&quot;</span>, <span class="st">&quot;chr16&quot;</span>, <span class="st">&quot;chr17&quot;</span>, <span class="st">&quot;chr18&quot;</span>, <span class="st">&quot;chr19&quot;</span>, <span class="st">&quot;chrX&quot;</span>, 
    <span class="st">&quot;chrY&quot;</span>, <span class="st">&quot;chrM&quot;</span>)
rmsk &lt;-<span class="st"> </span><span class="kw">keepSeqlevels</span>(rmsk, chr.vec)

<span class="co"># rename to NCBI style</span>
rmskStyle &lt;-<span class="st"> </span><span class="kw">mapSeqlevels</span>(<span class="kw">seqlevels</span>(rmsk), <span class="st">&quot;NCBI&quot;</span>)
rmsk &lt;-<span class="st"> </span><span class="kw">renameSeqlevels</span>(rmsk, rmskStyle)

### CIRCOS PLOT####

chr.vec &lt;-<span class="st"> </span><span class="kw">c</span>(<span class="st">&quot;chr1&quot;</span>, <span class="st">&quot;chr2&quot;</span>, <span class="st">&quot;chr3&quot;</span>, <span class="st">&quot;chr4&quot;</span>, <span class="st">&quot;chr5&quot;</span>, <span class="st">&quot;chr6&quot;</span>, 
    <span class="st">&quot;chr7&quot;</span>, <span class="st">&quot;chr8&quot;</span>, <span class="st">&quot;chr9&quot;</span>, <span class="st">&quot;chr10&quot;</span>, <span class="st">&quot;chr11&quot;</span>, <span class="st">&quot;chr12&quot;</span>, <span class="st">&quot;chr13&quot;</span>, 
    <span class="st">&quot;chr14&quot;</span>, <span class="st">&quot;chr15&quot;</span>, <span class="st">&quot;chr16&quot;</span>, <span class="st">&quot;chr17&quot;</span>, <span class="st">&quot;chr18&quot;</span>, <span class="st">&quot;chr19&quot;</span>, <span class="st">&quot;chrX&quot;</span>, 
    <span class="st">&quot;chrY&quot;</span>)
df.col.name &lt;-<span class="st"> </span><span class="kw">c</span>(<span class="st">&quot;chr&quot;</span>, <span class="st">&quot;start&quot;</span>, <span class="st">&quot;end&quot;</span>, <span class="st">&quot;width&quot;</span>, <span class="st">&quot;strand&quot;</span>, <span class="st">&quot;pvalue&quot;</span>, 
    <span class="st">&quot;qvalue&quot;</span>, <span class="st">&quot;meth.diff&quot;</span>)
drops &lt;-<span class="st"> </span><span class="kw">c</span>(<span class="st">&quot;width&quot;</span>, <span class="st">&quot;strand&quot;</span>, <span class="st">&quot;name&quot;</span>, <span class="st">&quot;pvalue&quot;</span>, <span class="st">&quot;qvalue&quot;</span>)

<span class="co"># Intersect with CpGs</span>
CpG.anno &lt;-<span class="st"> </span><span class="kw">read.table</span>(<span class="st">&quot;mm10.CpG.bed&quot;</span>, <span class="dt">sep =</span> <span class="st">&quot;&quot;</span>)
<span class="kw">colnames</span>(CpG.anno) &lt;-<span class="st"> </span><span class="kw">c</span>(<span class="st">&quot;chr&quot;</span>, <span class="st">&quot;start&quot;</span>, <span class="st">&quot;end&quot;</span>, <span class="st">&quot;CpG&quot;</span>)
CpG.anno &lt;-<span class="st"> </span>CpG.anno[, <span class="kw">names</span>(CpG.anno) <span class="op">%in%</span><span class="st"> </span><span class="kw">c</span>(<span class="st">&quot;chr&quot;</span>, <span class="st">&quot;start&quot;</span>, 
    <span class="st">&quot;end&quot;</span>)]
CpG.anno &lt;-<span class="st"> </span><span class="kw">makeGRangesFromDataFrame</span>(CpG.anno)
CpG.anno &lt;-<span class="st"> </span><span class="kw">keepSeqlevels</span>(CpG.anno, chr.vec, <span class="dt">pruning.mode =</span> <span class="st">&quot;coarse&quot;</span>)

<span class="co"># rename to NCBI style for intersect with rrbs GRange objects</span>
CpGStyle &lt;-<span class="st"> </span><span class="kw">mapSeqlevels</span>(<span class="kw">seqlevels</span>(CpG.anno), <span class="st">&quot;NCBI&quot;</span>)
CpG.anno &lt;-<span class="st"> </span><span class="kw">renameSeqlevels</span>(CpG.anno, CpGStyle)

<span class="co"># Hypomethylated overlapping with CpG</span>
CpG.25p.hypo &lt;-<span class="st"> </span><span class="kw">subsetByOverlaps</span>(diff25p.hypo.gR, CpG.anno)
CpG.25p.hypo.df &lt;-<span class="st"> </span><span class="kw">data.frame</span>(CpG.25p.hypo)
<span class="kw">colnames</span>(CpG.25p.hypo.df) &lt;-<span class="st"> </span>df.col.name
CpG.25p.hypo.df<span class="op">$</span>chr &lt;-<span class="st"> </span><span class="kw">paste</span>(<span class="st">&quot;chr&quot;</span>, CpG.25p.hypo.df<span class="op">$</span>chr, <span class="dt">sep =</span> <span class="st">&quot;&quot;</span>)
CpG.25p.hypo.df &lt;-<span class="st"> </span>CpG.25p.hypo.df[, <span class="op">!</span>(<span class="kw">names</span>(CpG.25p.hypo.df) <span class="op">%in%</span><span class="st"> </span>
<span class="st">    </span>drops)]

<span class="co"># Hypermethylated overlapping with CpG</span>
CpG.25p.hyper &lt;-<span class="st"> </span><span class="kw">subsetByOverlaps</span>(diff25p.hyper.gR, CpG.anno)
CpG.25p.hyper.df &lt;-<span class="st"> </span><span class="kw">data.frame</span>(CpG.25p.hyper)
<span class="kw">colnames</span>(CpG.25p.hyper.df) &lt;-<span class="st"> </span>df.col.name
CpG.25p.hyper.df<span class="op">$</span>chr &lt;-<span class="st"> </span><span class="kw">paste</span>(<span class="st">&quot;chr&quot;</span>, CpG.25p.hyper.df<span class="op">$</span>chr, <span class="dt">sep =</span> <span class="st">&quot;&quot;</span>)
CpG.25p.hyper.df &lt;-<span class="st"> </span>CpG.25p.hyper.df[, <span class="op">!</span>(<span class="kw">names</span>(CpG.25p.hyper.df) <span class="op">%in%</span><span class="st"> </span>
<span class="st">    </span>drops)]
CpG &lt;-<span class="st"> </span><span class="kw">rbind</span>(CpG.25p.hyper.df, CpG.25p.hypo.df)

<span class="co"># Intersect with LTRs Prepare LTR annotation</span>

LTRrmsk &lt;-<span class="st"> </span>genomation<span class="op">::</span><span class="kw">readBed</span>(<span class="st">&quot;mm10.rmsk.LTR.bed&quot;</span>)
<span class="co"># Throw away random chromosomes</span>
LTRrmsk &lt;-<span class="st"> </span><span class="kw">keepSeqlevels</span>(LTRrmsk, chr.vec, <span class="dt">pruning.mode =</span> <span class="st">&quot;coarse&quot;</span>)

<span class="co"># Rename to NCBI style for intersect with rrbs GRange objects</span>
LTRrmskStyle &lt;-<span class="st"> </span><span class="kw">mapSeqlevels</span>(<span class="kw">seqlevels</span>(LTRrmsk), <span class="st">&quot;NCBI&quot;</span>)
LTRrmsk &lt;-<span class="st"> </span><span class="kw">renameSeqlevels</span>(LTRrmsk, LTRrmskStyle)

<span class="co"># Hypomethylated overlapping with LTRs</span>
LTR.25p.hypo &lt;-<span class="st"> </span><span class="kw">subsetByOverlaps</span>(diff25p.hypo.gR, LTRrmsk)
LTR.25p.hypo.df &lt;-<span class="st"> </span><span class="kw">data.frame</span>(LTR.25p.hypo)
<span class="kw">colnames</span>(LTR.25p.hypo.df) &lt;-<span class="st"> </span>df.col.name
LTR.25p.hypo.df<span class="op">$</span>chr &lt;-<span class="st"> </span><span class="kw">paste</span>(<span class="st">&quot;chr&quot;</span>, LTR.25p.hypo.df<span class="op">$</span>chr, <span class="dt">sep =</span> <span class="st">&quot;&quot;</span>)
LTR.25p.hypo.df &lt;-<span class="st"> </span>LTR.25p.hypo.df[, <span class="op">!</span>(<span class="kw">names</span>(LTR.25p.hypo.df) <span class="op">%in%</span><span class="st"> </span>
<span class="st">    </span>drops)]

<span class="co"># Hypermethylated overlapping with LTRs</span>
LTR.25p.hyper &lt;-<span class="st"> </span><span class="kw">subsetByOverlaps</span>(diff25p.hyper.gR, LTRrmsk)
LTR.25p.hyper.df &lt;-<span class="st"> </span><span class="kw">data.frame</span>(LTR.25p.hyper)
<span class="kw">colnames</span>(LTR.25p.hyper.df) &lt;-<span class="st"> </span>df.col.name
LTR.25p.hyper.df<span class="op">$</span>chr &lt;-<span class="st"> </span><span class="kw">paste</span>(<span class="st">&quot;chr&quot;</span>, LTR.25p.hyper.df<span class="op">$</span>chr, <span class="dt">sep =</span> <span class="st">&quot;&quot;</span>)
LTR.25p.hyper.df &lt;-<span class="st"> </span>LTR.25p.hyper.df[, <span class="op">!</span>(<span class="kw">names</span>(LTR.25p.hyper.df) <span class="op">%in%</span><span class="st"> </span>
<span class="st">    </span>drops)]
LTRlist &lt;-<span class="st"> </span><span class="kw">rbind</span>(LTR.25p.hyper.df, LTR.25p.hypo.df)

<span class="co"># Intersect with LINEs Prepare LINE annotation</span>
LINErmsk &lt;-<span class="st"> </span>genomation<span class="op">::</span><span class="kw">readBed</span>(<span class="st">&quot;mm10.rmsk.LINE.bed&quot;</span>)
<span class="co"># Throw away random chromosomes</span>
LINErmsk &lt;-<span class="st"> </span><span class="kw">keepSeqlevels</span>(LINErmsk, chr.vec, <span class="dt">pruning.mode =</span> <span class="st">&quot;coarse&quot;</span>)

<span class="co"># Rename to NCBI style</span>
LINErmskStyle &lt;-<span class="st"> </span><span class="kw">mapSeqlevels</span>(<span class="kw">seqlevels</span>(LINErmsk), <span class="st">&quot;NCBI&quot;</span>)
LINErmsk &lt;-<span class="st"> </span><span class="kw">renameSeqlevels</span>(LINErmsk, LINErmskStyle)

<span class="co"># Hypomethylated overlapping with LINEs</span>
LINE.25p.hypo &lt;-<span class="st"> </span><span class="kw">subsetByOverlaps</span>(diff25p.hypo.gR, LINErmsk)
LINE.25p.hypo.df &lt;-<span class="st"> </span><span class="kw">data.frame</span>(LINE.25p.hypo)
<span class="kw">colnames</span>(LINE.25p.hypo.df) &lt;-<span class="st"> </span>df.col.name
LINE.25p.hypo.df<span class="op">$</span>chr &lt;-<span class="st"> </span><span class="kw">paste</span>(<span class="st">&quot;chr&quot;</span>, LINE.25p.hypo.df<span class="op">$</span>chr, <span class="dt">sep =</span> <span class="st">&quot;&quot;</span>)
LINE.25p.hypo.df &lt;-<span class="st"> </span>LINE.25p.hypo.df[, <span class="op">!</span>(<span class="kw">names</span>(LINE.25p.hypo.df) <span class="op">%in%</span><span class="st"> </span>
<span class="st">    </span>drops)]

<span class="co"># Hypermethylated overlapping with LINEs</span>
LINE.25p.hyper &lt;-<span class="st"> </span><span class="kw">subsetByOverlaps</span>(diff25p.hyper.gR, LINErmsk)
LINE.25p.hyper.df &lt;-<span class="st"> </span><span class="kw">data.frame</span>(LINE.25p.hyper)
<span class="kw">colnames</span>(LINE.25p.hyper.df) &lt;-<span class="st"> </span>df.col.name
LINE.25p.hyper.df<span class="op">$</span>chr &lt;-<span class="st"> </span><span class="kw">paste</span>(<span class="st">&quot;chr&quot;</span>, LINE.25p.hyper.df<span class="op">$</span>chr, 
    <span class="dt">sep =</span> <span class="st">&quot;&quot;</span>)
LINE.25p.hyper.df &lt;-<span class="st"> </span>LINE.25p.hyper.df[, <span class="op">!</span>(<span class="kw">names</span>(LINE.25p.hyper.df) <span class="op">%in%</span><span class="st"> </span>
<span class="st">    </span>drops)]
LINElist &lt;-<span class="st"> </span><span class="kw">rbind</span>(LINE.25p.hyper.df, LINE.25p.hypo.df)

<span class="co"># Intersect with SINEs Prepare SINE annotation</span>

SINErmsk &lt;-<span class="st"> </span>genomation<span class="op">::</span><span class="kw">readBed</span>(<span class="st">&quot;mm10.rmsk.SINE.bed&quot;</span>)
<span class="co"># throw away random chromosomes</span>
SINErmsk &lt;-<span class="st"> </span><span class="kw">keepSeqlevels</span>(SINErmsk, chr.vec, <span class="dt">pruning.mode =</span> <span class="st">&quot;coarse&quot;</span>)

<span class="co"># rename to NCBI style</span>
SINErmskStyle &lt;-<span class="st"> </span><span class="kw">mapSeqlevels</span>(<span class="kw">seqlevels</span>(SINErmsk), <span class="st">&quot;NCBI&quot;</span>)
SINErmsk &lt;-<span class="st"> </span><span class="kw">renameSeqlevels</span>(SINErmsk, SINErmskStyle)

<span class="co"># Hypomethylated overlapping with SINEs</span>
SINE.25p.hypo &lt;-<span class="st"> </span><span class="kw">subsetByOverlaps</span>(diff25p.hypo.gR, SINErmsk)
SINE.25p.hypo.df &lt;-<span class="st"> </span><span class="kw">data.frame</span>(SINE.25p.hypo)
<span class="kw">colnames</span>(SINE.25p.hypo.df) &lt;-<span class="st"> </span>df.col.name
SINE.25p.hypo.df<span class="op">$</span>chr &lt;-<span class="st"> </span><span class="kw">paste</span>(<span class="st">&quot;chr&quot;</span>, SINE.25p.hypo.df<span class="op">$</span>chr, <span class="dt">sep =</span> <span class="st">&quot;&quot;</span>)
SINE.25p.hypo.df &lt;-<span class="st"> </span>SINE.25p.hypo.df[, <span class="op">!</span>(<span class="kw">names</span>(SINE.25p.hypo.df) <span class="op">%in%</span><span class="st"> </span>
<span class="st">    </span>drops)]

<span class="co"># Hypermethylated overlapping with SINEs</span>
SINE.25p.hyper &lt;-<span class="st"> </span><span class="kw">subsetByOverlaps</span>(diff25p.hyper.gR, SINErmsk)
SINE.25p.hyper.df &lt;-<span class="st"> </span><span class="kw">data.frame</span>(SINE.25p.hyper)
<span class="kw">colnames</span>(SINE.25p.hyper.df) &lt;-<span class="st"> </span>df.col.name
SINE.25p.hyper.df<span class="op">$</span>chr &lt;-<span class="st"> </span><span class="kw">paste</span>(<span class="st">&quot;chr&quot;</span>, SINE.25p.hyper.df<span class="op">$</span>chr, 
    <span class="dt">sep =</span> <span class="st">&quot;&quot;</span>)
SINE.25p.hyper.df &lt;-<span class="st"> </span>SINE.25p.hyper.df[, <span class="op">!</span>(<span class="kw">names</span>(SINE.25p.hyper.df) <span class="op">%in%</span><span class="st"> </span>
<span class="st">    </span>drops)]
SINElist &lt;-<span class="st"> </span><span class="kw">rbind</span>(SINE.25p.hyper.df, SINE.25p.hypo.df)

<span class="co"># All</span>
diff25p.hypo.df &lt;-<span class="st"> </span><span class="kw">getData</span>(diff25p.hypo)
diff25p.hypo.df &lt;-<span class="st"> </span>diff25p.hypo.df[, <span class="kw">c</span>(<span class="st">&quot;chr&quot;</span>, <span class="st">&quot;start&quot;</span>, <span class="st">&quot;end&quot;</span>, 
    <span class="st">&quot;meth.diff&quot;</span>)]
diff25p.hypo.df<span class="op">$</span>chr &lt;-<span class="st"> </span><span class="kw">paste</span>(<span class="st">&quot;chr&quot;</span>, diff25p.hypo.df<span class="op">$</span>chr, <span class="dt">sep =</span> <span class="st">&quot;&quot;</span>)

diff25p.hyper.df &lt;-<span class="st"> </span><span class="kw">getData</span>(diff25p.hyper)
diff25p.hyper.df &lt;-<span class="st"> </span>diff25p.hyper.df[, <span class="kw">c</span>(<span class="st">&quot;chr&quot;</span>, <span class="st">&quot;start&quot;</span>, <span class="st">&quot;end&quot;</span>, 
    <span class="st">&quot;meth.diff&quot;</span>)]
diff25p.hyper.df<span class="op">$</span>chr &lt;-<span class="st"> </span><span class="kw">paste</span>(<span class="st">&quot;chr&quot;</span>, diff25p.hyper.df<span class="op">$</span>chr, <span class="dt">sep =</span> <span class="st">&quot;&quot;</span>)

Difflist &lt;-<span class="st"> </span><span class="kw">list</span>(diff25p.hyper.df, diff25p.hypo.df)
Difflist2 &lt;-<span class="st"> </span><span class="kw">rbind</span>(diff25p.hyper.df, diff25p.hypo.df)

exon.anno &lt;-<span class="st"> </span>gene.obj<span class="op">$</span>exons
exon.25p.all &lt;-<span class="st"> </span><span class="kw">subsetByOverlaps</span>(diff25p.all.gR, exon.anno)
exon.25p.all.df &lt;-<span class="st"> </span><span class="kw">data.frame</span>(exon.25p.all)
<span class="kw">colnames</span>(exon.25p.all.df) &lt;-<span class="st"> </span>df.col.name
exon.25p.all.df<span class="op">$</span>chr &lt;-<span class="st"> </span><span class="kw">paste</span>(<span class="st">&quot;chr&quot;</span>, exon.25p.all.df<span class="op">$</span>chr, <span class="dt">sep =</span> <span class="st">&quot;&quot;</span>)
exon.25p.all.df &lt;-<span class="st"> </span>exon.25p.all.df[, <span class="kw">c</span>(<span class="st">&quot;chr&quot;</span>, <span class="st">&quot;start&quot;</span>, <span class="st">&quot;end&quot;</span>, 
    <span class="st">&quot;meth.diff&quot;</span>)]

<span class="kw">library</span>(circlize)
<span class="kw">dev.off</span>()
<span class="kw">circos.par</span>(<span class="dt">track.height =</span> <span class="fl">0.1</span>)
<span class="kw">circos.par</span>(<span class="dt">start.degree =</span> <span class="dv">120</span>)
<span class="kw">circos.par</span>(<span class="dt">gap.degree =</span> <span class="dv">2</span>)
<span class="kw">circos.initializeWithIdeogram</span>(<span class="dt">species =</span> <span class="st">&quot;mm10&quot;</span>)
<span class="kw">circos.genomicDensity</span>(Difflist, <span class="dt">col =</span> <span class="kw">c</span>(<span class="st">&quot;red&quot;</span>, <span class="st">&quot;blue&quot;</span>), <span class="dt">bg.border =</span> <span class="ot">NA</span>)
<span class="kw">circos.genomicTrackPlotRegion</span>(CpG, <span class="dt">panel.fun =</span> <span class="cf">function</span>(region, 
    value, ...) {
    <span class="kw">circos.genomicRect</span>(region, value, <span class="dt">ytop.column =</span> <span class="dv">1</span>, <span class="dt">ybottom =</span> <span class="dv">0</span>, 
        <span class="dt">border =</span> <span class="kw">ifelse</span>(value[[<span class="dv">1</span>]] <span class="op">&gt;</span><span class="st"> </span><span class="dv">0</span>, <span class="st">&quot;red&quot;</span>, <span class="st">&quot;blue&quot;</span>), <span class="dt">col =</span> <span class="kw">ifelse</span>(value[[<span class="dv">1</span>]] <span class="op">&gt;</span><span class="st"> </span>
<span class="st">            </span><span class="dv">0</span>, <span class="st">&quot;red&quot;</span>, <span class="st">&quot;blue&quot;</span>), ...)
    cell.xlim =<span class="st"> </span><span class="kw">get.cell.meta.data</span>(<span class="st">&quot;cell.xlim&quot;</span>)
    <span class="kw">circos.lines</span>(cell.xlim, <span class="kw">c</span>(<span class="dv">0</span>, <span class="dv">0</span>), <span class="dt">lty =</span> <span class="dv">2</span>, <span class="dt">col =</span> <span class="st">&quot;#00000040&quot;</span>)
}, <span class="dt">bg.border =</span> <span class="ot">NA</span>, <span class="dt">bg.col =</span> <span class="st">&quot;#FEF9E7&quot;</span>)
<span class="kw">circos.genomicTrackPlotRegion</span>(exon.25p.all.df, <span class="dt">panel.fun =</span> <span class="cf">function</span>(region, 
    value, ...) {
    <span class="kw">circos.genomicRect</span>(region, value, <span class="dt">ytop.column =</span> <span class="dv">1</span>, <span class="dt">ybottom =</span> <span class="dv">0</span>, 
        <span class="dt">border =</span> <span class="kw">ifelse</span>(value[[<span class="dv">1</span>]] <span class="op">&gt;</span><span class="st"> </span><span class="dv">0</span>, <span class="st">&quot;red&quot;</span>, <span class="st">&quot;blue&quot;</span>), <span class="dt">col =</span> <span class="kw">ifelse</span>(value[[<span class="dv">1</span>]] <span class="op">&gt;</span><span class="st"> </span>
<span class="st">            </span><span class="dv">0</span>, <span class="st">&quot;red&quot;</span>, <span class="st">&quot;blue&quot;</span>), ...)
    cell.xlim =<span class="st"> </span><span class="kw">get.cell.meta.data</span>(<span class="st">&quot;cell.xlim&quot;</span>)
    <span class="kw">circos.lines</span>(cell.xlim, <span class="kw">c</span>(<span class="dv">0</span>, <span class="dv">0</span>), <span class="dt">lty =</span> <span class="dv">2</span>, <span class="dt">col =</span> <span class="st">&quot;#00000040&quot;</span>)
}, <span class="dt">bg.border =</span> <span class="ot">NA</span>, <span class="dt">bg.col =</span> <span class="st">&quot;#D6EAF8&quot;</span>)
<span class="kw">circos.genomicTrackPlotRegion</span>(LTRlist, <span class="dt">panel.fun =</span> <span class="cf">function</span>(region, 
    value, ...) {
    <span class="kw">circos.genomicRect</span>(region, value, <span class="dt">ytop.column =</span> <span class="dv">1</span>, <span class="dt">ybottom =</span> <span class="dv">0</span>, 
        <span class="dt">border =</span> <span class="kw">ifelse</span>(value[[<span class="dv">1</span>]] <span class="op">&gt;</span><span class="st"> </span><span class="dv">0</span>, <span class="st">&quot;red&quot;</span>, <span class="st">&quot;blue&quot;</span>), <span class="dt">col =</span> <span class="kw">ifelse</span>(value[[<span class="dv">1</span>]] <span class="op">&gt;</span><span class="st"> </span>
<span class="st">            </span><span class="dv">0</span>, <span class="st">&quot;red&quot;</span>, <span class="st">&quot;blue&quot;</span>), ...)
    cell.xlim =<span class="st"> </span><span class="kw">get.cell.meta.data</span>(<span class="st">&quot;cell.xlim&quot;</span>)
    <span class="kw">circos.lines</span>(cell.xlim, <span class="kw">c</span>(<span class="dv">0</span>, <span class="dv">0</span>), <span class="dt">lty =</span> <span class="dv">2</span>, <span class="dt">col =</span> <span class="st">&quot;#00000040&quot;</span>)
}, <span class="dt">bg.border =</span> <span class="ot">NA</span>, <span class="dt">bg.col =</span> <span class="st">&quot;#EAFAF1&quot;</span>)
<span class="kw">circos.genomicTrackPlotRegion</span>(LINElist, <span class="dt">panel.fun =</span> <span class="cf">function</span>(region, 
    value, ...) {
    <span class="kw">circos.genomicRect</span>(region, value, <span class="dt">ytop.column =</span> <span class="dv">1</span>, <span class="dt">ybottom =</span> <span class="dv">0</span>, 
        <span class="dt">border =</span> <span class="kw">ifelse</span>(value[[<span class="dv">1</span>]] <span class="op">&gt;</span><span class="st"> </span><span class="dv">0</span>, <span class="st">&quot;red&quot;</span>, <span class="st">&quot;blue&quot;</span>), <span class="dt">col =</span> <span class="kw">ifelse</span>(value[[<span class="dv">1</span>]] <span class="op">&gt;</span><span class="st"> </span>
<span class="st">            </span><span class="dv">0</span>, <span class="st">&quot;red&quot;</span>, <span class="st">&quot;blue&quot;</span>), ...)
    cell.xlim =<span class="st"> </span><span class="kw">get.cell.meta.data</span>(<span class="st">&quot;cell.xlim&quot;</span>)
    <span class="kw">circos.lines</span>(cell.xlim, <span class="kw">c</span>(<span class="dv">0</span>, <span class="dv">0</span>), <span class="dt">lty =</span> <span class="dv">2</span>, <span class="dt">col =</span> <span class="st">&quot;#00000040&quot;</span>)
}, <span class="dt">bg.border =</span> <span class="ot">NA</span>, <span class="dt">bg.col =</span> <span class="st">&quot;#F4ECF7&quot;</span>)
<span class="kw">circos.genomicTrackPlotRegion</span>(SINElist, <span class="dt">panel.fun =</span> <span class="cf">function</span>(region, 
    value, ...) {
    <span class="kw">circos.genomicRect</span>(region, value, <span class="dt">ytop.column =</span> <span class="dv">1</span>, <span class="dt">ybottom =</span> <span class="dv">0</span>, 
        <span class="dt">border =</span> <span class="kw">ifelse</span>(value[[<span class="dv">1</span>]] <span class="op">&gt;</span><span class="st"> </span><span class="dv">0</span>, <span class="st">&quot;red&quot;</span>, <span class="st">&quot;blue&quot;</span>), <span class="dt">col =</span> <span class="kw">ifelse</span>(value[[<span class="dv">1</span>]] <span class="op">&gt;</span><span class="st"> </span>
<span class="st">            </span><span class="dv">0</span>, <span class="st">&quot;red&quot;</span>, <span class="st">&quot;blue&quot;</span>), ...)
    cell.xlim =<span class="st"> </span><span class="kw">get.cell.meta.data</span>(<span class="st">&quot;cell.xlim&quot;</span>)
    <span class="kw">circos.lines</span>(cell.xlim, <span class="kw">c</span>(<span class="dv">0</span>, <span class="dv">0</span>), <span class="dt">lty =</span> <span class="dv">2</span>, <span class="dt">col =</span> <span class="st">&quot;#00000040&quot;</span>)
}, <span class="dt">bg.border =</span> <span class="ot">NA</span>, <span class="dt">bg.col =</span> <span class="st">&quot;#FDEDEC&quot;</span>)
<span class="kw">circos.clear</span>()

<span class="co"># Export bed files</span>
<span class="kw">write.table</span>(LTR.25p.hypo.df, <span class="dt">file =</span> <span class="st">&quot;HP1cTKO.LTR.hypomethylated.bed&quot;</span>, 
    <span class="dt">quote =</span> F, <span class="dt">sep =</span> <span class="st">&quot;</span><span class="ch">\t</span><span class="st">&quot;</span>, <span class="dt">row.names =</span> F, <span class="dt">col.names =</span> F)
<span class="kw">write.table</span>(CpG.25p.hyper.df, <span class="dt">file =</span> <span class="st">&quot;HP1cTKO.CpG.hypermethylated.bed&quot;</span>, 
    <span class="dt">quote =</span> F, <span class="dt">sep =</span> <span class="st">&quot;</span><span class="ch">\t</span><span class="st">&quot;</span>, <span class="dt">row.names =</span> F, <span class="dt">col.names =</span> F)
<span class="kw">write.table</span>(exon.25p.all.df, <span class="dt">file =</span> <span class="st">&quot;HP1cTKO.exon.hypermethylated.bed&quot;</span>, 
    <span class="dt">quote =</span> F, <span class="dt">sep =</span> <span class="st">&quot;</span><span class="ch">\t</span><span class="st">&quot;</span>, <span class="dt">row.names =</span> F, <span class="dt">col.names =</span> F)
<span class="kw">write.table</span>(promoter.hyper.df, <span class="dt">file =</span> <span class="st">&quot;HP1cTKO.promoter.hypermethylated.bed&quot;</span>, 
    <span class="dt">quote =</span> F, <span class="dt">sep =</span> <span class="st">&quot;</span><span class="ch">\t</span><span class="st">&quot;</span>, <span class="dt">row.names =</span> F, <span class="dt">col.names =</span> F)
<span class="kw">write.table</span>(promoter.hypo.df, <span class="dt">file =</span> <span class="st">&quot;HP1cTKO.promoter.hypomethylated.bed&quot;</span>, 
    <span class="dt">quote =</span> F, <span class="dt">sep =</span> <span class="st">&quot;</span><span class="ch">\t</span><span class="st">&quot;</span>, <span class="dt">row.names =</span> F, <span class="dt">col.names =</span> F)</code></pre></div>
</div>
<div id="annotated-ideogram" class="section level3">
<h3><span class="header-section-number">6.7.2</span> Annotated Ideogram</h3>
<p>Annotated ideograms were created by storing the chromosomal locations of from genes differentially expressed (coarse set, corrected p &lt; 0.05) in .bed files. For the human genome, gene locations were converted from mouse GRCm38.p5 assembly to Human GRCh38.p12 using the UCSC’s batch liftover tool <a href="https://genome.ucsc.edu/cgi-bin/hgLiftOver" class="uri">https://genome.ucsc.edu/cgi-bin/hgLiftOver</a>. Bed files containing differentially expressed genes were then used to annotate mouse and human genomes using NCBI’s ‘genome annotation page’: <a href="https://www.ncbi.nlm.nih.gov/genome/tools/gdp/" class="uri">https://www.ncbi.nlm.nih.gov/genome/tools/gdp/</a> .</p>
</div>
</div>
<div id="about-this-thesis" class="section level2">
<h2><span class="header-section-number">6.8</span> About this Thesis</h2>
<p>This thesis was written in R markdown using a customized version of the bookdown package (<a href="https://bookdown.org/yihui/bookdown/" class="uri">https://bookdown.org/yihui/bookdown/</a>) in RStudio. Citations were managed using Zotero 5.0 and integrated into the markdown document using biblatex and citr (<a href="https://github.com/crsh/citr" class="uri">https://github.com/crsh/citr</a>). Printed versions were rendered using . This book was also rendered as an html gitbook and can be viewed online at <a href="https://github.com/qoldt/" class="uri">https://github.com/qoldt/</a> …</p>
<div id="session-info" class="section level3">
<h3><span class="header-section-number">6.8.1</span> Session Info</h3>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw">sessionInfo</span>()</code></pre></div>
<p>R version 3.5.1 (2018-07-02) Platform: x86_64-apple-darwin15.6.0 (64-bit) Running under: macOS High Sierra 10.13.6</p>
<p>Matrix products: default BLAS: /Library/Frameworks/R.framework/Versions/3.5/Resources/lib/libRblas.0.dylib LAPACK: /Library/Frameworks/R.framework/Versions/3.5/Resources/lib/libRlapack.dylib</p>
<p>locale: [1] en_CA.UTF-8/en_CA.UTF-8/en_CA.UTF-8/C/en_CA.UTF-8/en_CA.UTF-8</p>
<p>attached base packages: [1] stats graphics grDevices utils datasets methods base</p>
<p>other attached packages: [1] kableExtra_1.0.1 citr_0.3.0 beaverdown_0.2.1 knitr_1.21<br />
[5] bookdown_0.9.1 ggplot2_3.1.0 dplyr_0.7.8 usethis_1.4.0<br />
[9] devtools_2.0.1</p>
<p>loaded via a namespace (and not attached): [1] tidyselect_0.2.5 xfun_0.4 remotes_2.0.2<br />
[4] purrr_0.2.5 colorspace_1.4-0 viridisLite_0.3.0 [7] miniUI_0.1.1.1 htmltools_0.3.6 yaml_2.2.0<br />
[10] rlang_0.3.1 pkgbuild_1.0.2 pillar_1.3.1<br />
[13] later_0.7.5 glue_1.3.0 withr_2.1.2<br />
[16] sessioninfo_1.1.1 bindrcpp_0.2.2 bindr_0.1.1<br />
[19] plyr_1.8.4 stringr_1.3.1 munsell_0.5.0<br />
[22] gtable_0.2.0 rvest_0.3.2 memoise_1.1.0<br />
[25] evaluate_0.12 callr_3.1.1 httpuv_1.4.5.1<br />
[28] ps_1.3.0 highr_0.7 Rcpp_1.0.0<br />
[31] readr_1.3.1 xtable_1.8-3 formatR_1.5<br />
[34] backports_1.1.3 scales_1.0.0 promises_1.0.1<br />
[37] desc_1.2.0 pkgload_1.0.2 webshot_0.5.1<br />
[40] mime_0.6 fs_1.2.6 hms_0.4.2<br />
[43] digest_0.6.18 stringi_1.2.4 processx_3.2.1<br />
[46] shiny_1.2.0 rprojroot_1.3-2 grid_3.5.1<br />
[49] cli_1.0.1 tools_3.5.1 magrittr_1.5<br />
[52] lazyeval_0.2.1 tibble_2.0.1 crayon_1.3.4<br />
[55] pkgconfig_2.0.2 xml2_1.2.0 prettyunits_1.0.2 [58] httr_1.4.0 assertthat_0.2.0 rmarkdown_1.11<br />
[61] rstudioapi_0.9.0 R6_2.3.0 compiler_3.5.1</p>

<!--
The bib chunk below must go last in this document according to how R Markdown renders.  More info is at http://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html
-->

<!-- 
If you'd like to change the name of the bibliography to something else,
delete "References" and replace it.
-->
</div>
</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-KruegerBismarkflexiblealigner2011a">
<p>Krueger, Felix, and Simon R. Andrews. 2011. “Bismark: A Flexible Aligner and Methylation Caller for Bisulfite-Seq Applications.” <em>Bioinformatics</em> 27 (11): 1571–2. doi:<a href="https://doi.org/10.1093/bioinformatics/btr167">10.1093/bioinformatics/btr167</a>.</p>
</div>
<div id="ref-AkalinmethylKitcomprehensivepackage2012">
<p>Akalin, Altuna, Matthias Kormaksson, Sheng Li, Francine E Garrett-Bakelman, Maria E Figueroa, Ari Melnick, and Christopher E Mason. 2012. “methylKit: A Comprehensive R Package for the Analysis of Genome-Wide DNA Methylation Profiles.” <em>Genome Biology</em> 13 (10): R87. doi:<a href="https://doi.org/10.1186/gb-2012-13-10-r87">10.1186/gb-2012-13-10-r87</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="5-supplemental-data.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="7-references.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"download": [["thesis.pdf", "PDF"], ["thesis.epub", "EPUB"], ["thesis.docx", "Word"]],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
