<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Part 7 References | HP1 Deficiency Results in Accelerated Neurodegeneration</title>
  <meta name="description" content="Part 7 References | HP1 Deficiency Results in Accelerated Neurodegeneration" />
  <meta name="generator" content="bookdown  and GitBook 2.6.7" />

  <meta property="og:title" content="Part 7 References | HP1 Deficiency Results in Accelerated Neurodegeneration" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Part 7 References | HP1 Deficiency Results in Accelerated Neurodegeneration" />
  
  
  

<meta name="author" content="Andrew G. Newman" />


<meta name="date" content="2019-01-01" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="6-bioinformatics-data-processing.html">

<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />







<script src="libs/kePrint-0.0.1/kePrint.js"></script>


<style type="text/css">
div.sourceCode { overflow-x: auto; }
table.sourceCode, tr.sourceCode, td.lineNumbers, td.sourceCode {
  margin: 0; padding: 0; vertical-align: baseline; border: none; }
table.sourceCode { width: 100%; line-height: 100%; }
td.lineNumbers { text-align: right; padding-right: 4px; padding-left: 4px; color: #aaaaaa; border-right: 1px solid #aaaaaa; }
td.sourceCode { padding-left: 5px; }
code > span.kw { color: #007020; font-weight: bold; } /* Keyword */
code > span.dt { color: #902000; } /* DataType */
code > span.dv { color: #40a070; } /* DecVal */
code > span.bn { color: #40a070; } /* BaseN */
code > span.fl { color: #40a070; } /* Float */
code > span.ch { color: #4070a0; } /* Char */
code > span.st { color: #4070a0; } /* String */
code > span.co { color: #60a0b0; font-style: italic; } /* Comment */
code > span.ot { color: #007020; } /* Other */
code > span.al { color: #ff0000; font-weight: bold; } /* Alert */
code > span.fu { color: #06287e; } /* Function */
code > span.er { color: #ff0000; font-weight: bold; } /* Error */
code > span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
code > span.cn { color: #880000; } /* Constant */
code > span.sc { color: #4070a0; } /* SpecialChar */
code > span.vs { color: #4070a0; } /* VerbatimString */
code > span.ss { color: #bb6688; } /* SpecialString */
code > span.im { } /* Import */
code > span.va { color: #19177c; } /* Variable */
code > span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code > span.op { color: #666666; } /* Operator */
code > span.bu { } /* BuiltIn */
code > span.ex { } /* Extension */
code > span.pp { color: #bc7a00; } /* Preprocessor */
code > span.at { color: #7d9029; } /* Attribute */
code > span.do { color: #ba2121; font-style: italic; } /* Documentation */
code > span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code > span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code > span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
</style>

</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="contribution-of-authors.html"><a href="contribution-of-authors.html"><i class="fa fa-check"></i>Contribution of Authors</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="preface.html"><a href="preface.html"><i class="fa fa-check"></i>Preface</a></li>
<li><a href="section.html#section"></a></li>
<li class="chapter" data-level="1" data-path="1-introduction.html"><a href="1-introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="1-introduction.html"><a href="1-introduction.html#epigenetics"><i class="fa fa-check"></i><b>1.1</b> Epigenetics</a><ul>
<li><a href="1-introduction.html#dna-modifications"><em>DNA Modifications</em></a></li>
<li><a href="1-introduction.html#chromatin"><em>Chromatin</em></a></li>
<li><a href="1-introduction.html#chromatin-remodelling"><em>Chromatin Remodelling</em></a></li>
<li><a href="1-introduction.html#krab-zfps-transposable-elements-and-exaptation"><em>KRAB-ZFPs, Transposable Elements and Exaptation</em></a></li>
<li><a href="1-introduction.html#heterochromatinization-and-hp1"><em>Heterochromatinization and HP1</em></a></li>
<li><a href="1-introduction.html#non-coding-rna"><em>Non-Coding RNA</em></a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="1-introduction.html"><a href="1-introduction.html#the-heterochromatin-loss-model-of-aging"><i class="fa fa-check"></i><b>1.2</b> The Heterochromatin Loss Model of Aging</a></li>
<li class="chapter" data-level="1.3" data-path="1-introduction.html"><a href="1-introduction.html#the-aging-neuronal-epigenome"><i class="fa fa-check"></i><b>1.3</b> The Aging Neuronal Epigenome</a><ul>
<li class="chapter" data-level="1.3.1" data-path="1-introduction.html"><a href="1-introduction.html#DNAging"><i class="fa fa-check"></i><b>1.3.1</b> <em>DNA Methylation and Hydroxymethylation in Aged Neurons</em></a></li>
<li class="chapter" data-level="1.3.2" data-path="1-introduction.html"><a href="1-introduction.html#chraging"><i class="fa fa-check"></i><b>1.3.2</b> <em>Chromatin in the Aging Brain</em></a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="1-introduction.html"><a href="1-introduction.html#objective"><i class="fa fa-check"></i><b>1.4</b> Objective</a></li>
<li class="chapter" data-level="1.5" data-path="1-introduction.html"><a href="1-introduction.html#hp1-proteins---a-background"><i class="fa fa-check"></i><b>1.5</b> HP1 Proteins - A Background</a><ul>
<li><a href="1-introduction.html#the-abgs-of-hp1-proteins">The <em>ABG’s</em> of HP1 Proteins</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-results.html"><a href="2-results.html"><i class="fa fa-check"></i><b>2</b> Results</a><ul>
<li class="chapter" data-level="2.1" data-path="2-results.html"><a href="2-results.html#HP1aT"><i class="fa fa-check"></i><b>2.1</b> HP1FEC, HP1TKO and HP1<span class="math inline">\(\alpha\)</span>T</a></li>
<li class="chapter" data-level="2.2" data-path="2-results.html"><a href="2-results.html#expression-of-hp1-proteins-in-the-brain"><i class="fa fa-check"></i><b>2.2</b> Expression of HP1 Proteins in the Brain</a></li>
<li class="chapter" data-level="2.3" data-path="2-results.html"><a href="2-results.html#mosaic-deletion-of-hp1beta-and-hp1gamma-from-cortical-progenitors"><i class="fa fa-check"></i><b>2.3</b> Mosaic Deletion of HP1<span class="math inline">\(\beta\)</span> and HP1<span class="math inline">\(\gamma\)</span> from Cortical Progenitors</a></li>
<li class="chapter" data-level="2.4" data-path="2-results.html"><a href="2-results.html#deletion-of-hp1beta-hp1gamma-from-the-emx1-lineage."><i class="fa fa-check"></i><b>2.4</b> Deletion of <em>HP1<span class="math inline">\(\beta\)</span></em> &amp; <em>HP1<span class="math inline">\(\gamma\)</span></em> from the <em>Emx1</em>-lineage.</a><ul>
<li class="chapter" data-level="2.4.1" data-path="2-results.html"><a href="2-results.html#cell-fate-in-cortical-neurons"><i class="fa fa-check"></i><b>2.4.1</b> Cell Fate in Cortical Neurons</a></li>
<li class="chapter" data-level="2.4.2" data-path="2-results.html"><a href="2-results.html#imprint"><i class="fa fa-check"></i><b>2.4.2</b> Imprinted Loci</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="2-results.html"><a href="2-results.html#perturbed-regulation-of-retroelements-in-hp1-mutants"><i class="fa fa-check"></i><b>2.5</b> Perturbed Regulation of Retroelements in HP1 Mutants</a><ul>
<li class="chapter" data-level="2.5.1" data-path="2-results.html"><a href="2-results.html#line1"><i class="fa fa-check"></i><b>2.5.1</b> LINE1</a></li>
<li class="chapter" data-level="2.5.2" data-path="2-results.html"><a href="2-results.html#class-iii-ervs"><i class="fa fa-check"></i><b>2.5.2</b> Class III ERVs</a></li>
<li class="chapter" data-level="2.5.3" data-path="2-results.html"><a href="2-results.html#class-ii-ervs"><i class="fa fa-check"></i><b>2.5.3</b> Class II ERVs</a></li>
<li class="chapter" data-level="2.5.4" data-path="2-results.html"><a href="2-results.html#de-novo-retrotransposition"><i class="fa fa-check"></i><b>2.5.4</b> <em>De Novo</em> Retrotransposition</a></li>
</ul></li>
<li class="chapter" data-level="2.6" data-path="2-results.html"><a href="2-results.html#sustained-hp1betagamma-deficiency-adversely-affects-neuronal-complexity-function-and-cognition"><i class="fa fa-check"></i><b>2.6</b> Sustained HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> Deficiency Adversely Affects Neuronal Complexity, Function and Cognition</a><ul>
<li class="chapter" data-level="2.6.1" data-path="2-results.html"><a href="2-results.html#behavior"><i class="fa fa-check"></i><b>2.6.1</b> Behavior</a></li>
<li class="chapter" data-level="2.6.2" data-path="2-results.html"><a href="2-results.html#MRIresults"><i class="fa fa-check"></i><b>2.6.2</b> Structure, Volumetrics and Histology</a></li>
</ul></li>
<li class="chapter" data-level="2.7" data-path="2-results.html"><a href="2-results.html#hp1beta-and-hp1gamma-synergistically-contribute-to-repeat-silencing"><i class="fa fa-check"></i><b>2.7</b> HP1<span class="math inline">\(\beta\)</span> and HP1<span class="math inline">\(\gamma\)</span> Synergistically Contribute to Repeat Silencing</a><ul>
<li class="chapter" data-level="2.7.1" data-path="2-results.html"><a href="2-results.html#hp1gamma-is-required-for-h4k20me3-deposition-in-neurons"><i class="fa fa-check"></i><b>2.7.1</b> HP1<span class="math inline">\(\gamma\)</span> is Required for H4K20me3 Deposition in Neurons</a></li>
<li class="chapter" data-level="2.7.2" data-path="2-results.html"><a href="2-results.html#hp1gamma-binds-suv420h2-via-the-chromoshadow-domain"><i class="fa fa-check"></i><b>2.7.2</b> HP1<span class="math inline">\(\gamma\)</span> binds SUV420H2 via the Chromoshadow Domain</a></li>
<li class="chapter" data-level="2.7.3" data-path="2-results.html"><a href="2-results.html#re-addition-of-hp1gamma-can-restore-h4k20me3-in-vivo"><i class="fa fa-check"></i><b>2.7.3</b> Re-addition of HP1<span class="math inline">\(\gamma\)</span> can restore H4K20me3 In Vivo</a></li>
<li class="chapter" data-level="2.7.4" data-path="2-results.html"><a href="2-results.html#h4k20me3-loss-is-correlated-with-iap-de-repression"><i class="fa fa-check"></i><b>2.7.4</b> H4K20me3 Loss is Correlated with IAP De-repression</a></li>
<li class="chapter" data-level="2.7.5" data-path="2-results.html"><a href="2-results.html#kap1-h3k9me3-and-h4k20me3-are-lost-at-iap-elements-in-hp1ctko-es-cells"><i class="fa fa-check"></i><b>2.7.5</b> KAP1, H3K9me3 and H4K20me3 are lost at IAP Elements in HP1cTKO ES Cells</a></li>
<li class="chapter" data-level="2.7.6" data-path="2-results.html"><a href="2-results.html#dna-methylation-is-affected-following-loss-of-hp1"><i class="fa fa-check"></i><b>2.7.6</b> DNA Methylation is Affected Following Loss of HP1</a></li>
<li><a href="2-results.html#kap1-is-lost-from-icrs-in-hp1ctko-cells"><em>KAP1 is lost from ICRs in HP1cTKO Cells</em></a></li>
</ul></li>
<li class="chapter" data-level="2.8" data-path="2-results.html"><a href="2-results.html#RNAresults"><i class="fa fa-check"></i><b>2.8</b> Transcriptional Dysregulation following HP1 Deficiency Drives UPR and Inflammatory Responses</a><ul>
<li class="chapter" data-level="2.8.1" data-path="2-results.html"><a href="2-results.html#genotype-related-changes"><i class="fa fa-check"></i><b>2.8.1</b> Genotype Related Changes</a></li>
<li class="chapter" data-level="2.8.2" data-path="2-results.html"><a href="2-results.html#longitudinal-changes-to-gene-expression"><i class="fa fa-check"></i><b>2.8.2</b> Longitudinal Changes To Gene Expression</a></li>
<li class="chapter" data-level="2.8.3" data-path="2-results.html"><a href="2-results.html#pathway-analysis"><i class="fa fa-check"></i><b>2.8.3</b> Pathway Analysis</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-discussion.html"><a href="3-discussion.html"><i class="fa fa-check"></i><b>3</b> Discussion</a><ul>
<li class="chapter" data-level="3.1" data-path="3-discussion.html"><a href="3-discussion.html#phenotypes"><i class="fa fa-check"></i><b>3.1</b> Phenotypes</a><ul>
<li><a href="3-discussion.html#structural-changes-in-the-hippocampus"><em>Structural Changes in the Hippocampus</em></a></li>
<li><a href="3-discussion.html#reduction-in-cortical-volume"><em>Reduction in Cortical Volume</em></a></li>
<li><a href="3-discussion.html#behavior-and-impairments-in-learning-memory"><em>Behavior and Impairments in Learning &amp; Memory</em></a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="3-discussion.html"><a href="3-discussion.html#hp1-dependent-epigenetic-systems"><i class="fa fa-check"></i><b>3.2</b> HP1-dependent Epigenetic Systems</a><ul>
<li><a href="3-discussion.html#hp1gamma"><em>HP1<span class="math inline">\(\gamma\)</span></em></a></li>
<li><a href="3-discussion.html#hp1beta"><em>HP1<span class="math inline">\(\beta\)</span></em></a></li>
<li><a href="3-discussion.html#hp1beta-x-hp1gamma"><em>HP1<span class="math inline">\(\beta\)</span></em> x <em>HP1<span class="math inline">\(\gamma\)</span></em></a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="3-discussion.html"><a href="3-discussion.html#concordance-with-aging-and-disease"><i class="fa fa-check"></i><b>3.3</b> Concordance with Aging and Disease</a><ul>
<li><a href="3-discussion.html#complement-in-cns-aging-and-disease"><em>Complement in CNS Aging and Disease</em></a></li>
<li><a href="3-discussion.html#a-pathway-to-complement"><em>A Pathway to Complement</em></a></li>
<li><a href="3-discussion.html#unfolded-protein-response-upr"><em>Unfolded Protein Response (UPR)</em></a></li>
<li><a href="3-discussion.html#viruses-transactivation-and-neurodegeneration"><em>Viruses, Transactivation and Neurodegeneration</em></a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="3-discussion.html"><a href="3-discussion.html#concluding-remarks"><i class="fa fa-check"></i><b>3.4</b> Concluding Remarks</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-methods.html"><a href="4-methods.html"><i class="fa fa-check"></i><b>4</b> Methods</a><ul>
<li class="chapter" data-level="4.1" data-path="4-methods.html"><a href="4-methods.html#mouse-lines"><i class="fa fa-check"></i><b>4.1</b> Mouse Lines</a><ul>
<li class="chapter" data-level="4.1.1" data-path="4-methods.html"><a href="4-methods.html#HP1anull"><i class="fa fa-check"></i><b>4.1.1</b> Generation of HP1<span class="math inline">\(\alpha\)</span>KO and HP1<span class="math inline">\(\alpha\)</span>T</a></li>
<li class="chapter" data-level="4.1.2" data-path="4-methods.html"><a href="4-methods.html#HP1Bgtarget"><i class="fa fa-check"></i><b>4.1.2</b> Generation of Floxed HP1<span class="math inline">\(\beta\)</span> &amp; Floxed HP1<span class="math inline">\(\gamma\)</span></a></li>
<li class="chapter" data-level="4.1.3" data-path="4-methods.html"><a href="4-methods.html#genotyping"><i class="fa fa-check"></i><b>4.1.3</b> Genotyping</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="4-methods.html"><a href="4-methods.html#cloning"><i class="fa fa-check"></i><b>4.2</b> Cloning</a><ul>
<li class="chapter" data-level="4.2.1" data-path="4-methods.html"><a href="4-methods.html#rna-isolation-cdna-generation-rnaseq"><i class="fa fa-check"></i><b>4.2.1</b> RNA Isolation, cDNA Generation, &amp; RNAseq</a></li>
<li class="chapter" data-level="4.2.2" data-path="4-methods.html"><a href="4-methods.html#cloning-of-rna-probes-for-in-situ-hybridization"><i class="fa fa-check"></i><b>4.2.2</b> Cloning of RNA Probes for <em>In-Situ Hybridization</em></a></li>
<li class="chapter" data-level="4.2.3" data-path="4-methods.html"><a href="4-methods.html#cloning-of-expression-constructs"><i class="fa fa-check"></i><b>4.2.3</b> Cloning of Expression Constructs</a></li>
<li class="chapter" data-level="4.2.4" data-path="4-methods.html"><a href="4-methods.html#site-directed-mutagenesis"><i class="fa fa-check"></i><b>4.2.4</b> Site Directed Mutagenesis</a></li>
<li class="chapter" data-level="4.2.5" data-path="4-methods.html"><a href="4-methods.html#transformation"><i class="fa fa-check"></i><b>4.2.5</b> Transformation</a></li>
<li class="chapter" data-level="4.2.6" data-path="4-methods.html"><a href="4-methods.html#clonal-selection-plasmid-production"><i class="fa fa-check"></i><b>4.2.6</b> Clonal Selection &amp; Plasmid Production</a></li>
<li class="chapter" data-level="4.2.7" data-path="4-methods.html"><a href="4-methods.html#in-vitro-transcription-ivt"><i class="fa fa-check"></i><b>4.2.7</b> In Vitro Transcription (IVT)</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="4-methods.html"><a href="4-methods.html#cell-culture"><i class="fa fa-check"></i><b>4.3</b> Cell Culture</a><ul>
<li class="chapter" data-level="4.3.1" data-path="4-methods.html"><a href="4-methods.html#primary-neuronal-culture"><i class="fa fa-check"></i><b>4.3.1</b> Primary Neuronal Culture</a></li>
<li class="chapter" data-level="4.3.2" data-path="4-methods.html"><a href="4-methods.html#hek293t-culture-transfection"><i class="fa fa-check"></i><b>4.3.2</b> HEK293T Culture &amp; Transfection</a></li>
<li class="chapter" data-level="4.3.3" data-path="4-methods.html"><a href="4-methods.html#co-immunoprecipitation-co-ip-and-western-blot-wb"><i class="fa fa-check"></i><b>4.3.3</b> Co-Immunoprecipitation (Co-IP) and Western Blot (WB)</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="4-methods.html"><a href="4-methods.html#in-utero-electroporation-tissue-processing-and-histology"><i class="fa fa-check"></i><b>4.4</b> In Utero Electroporation, Tissue Processing and Histology</a><ul>
<li class="chapter" data-level="4.4.1" data-path="4-methods.html"><a href="4-methods.html#IUEmethod"><i class="fa fa-check"></i><b>4.4.1</b> In Utero Electroporation (IUE)</a></li>
<li class="chapter" data-level="4.4.2" data-path="4-methods.html"><a href="4-methods.html#tissue-fixation-and-perfusion"><i class="fa fa-check"></i><b>4.4.2</b> Tissue Fixation and Perfusion</a></li>
<li class="chapter" data-level="4.4.3" data-path="4-methods.html"><a href="4-methods.html#cryosectioning"><i class="fa fa-check"></i><b>4.4.3</b> Cryosectioning</a></li>
<li class="chapter" data-level="4.4.4" data-path="4-methods.html"><a href="4-methods.html#immuno-histochemistry-ihc"><i class="fa fa-check"></i><b>4.4.4</b> Immuno Histochemistry (IHC)</a></li>
<li class="chapter" data-level="4.4.5" data-path="4-methods.html"><a href="4-methods.html#quantification-of-cells-in-the-dentate-gyrus"><i class="fa fa-check"></i><b>4.4.5</b> Quantification of Cells in the Dentate Gyrus</a></li>
<li class="chapter" data-level="4.4.6" data-path="4-methods.html"><a href="4-methods.html#in-situ-hybridization-ish"><i class="fa fa-check"></i><b>4.4.6</b> In Situ Hybridization (ISH)</a></li>
<li class="chapter" data-level="4.4.7" data-path="4-methods.html"><a href="4-methods.html#paraffin-sectioning"><i class="fa fa-check"></i><b>4.4.7</b> Paraffin Sectioning</a></li>
<li class="chapter" data-level="4.4.8" data-path="4-methods.html"><a href="4-methods.html#nissl"><i class="fa fa-check"></i><b>4.4.8</b> Nissl Stain</a></li>
<li class="chapter" data-level="4.4.9" data-path="4-methods.html"><a href="4-methods.html#golgi-impregnation-and-dendritic-complexity-analysis"><i class="fa fa-check"></i><b>4.4.9</b> Golgi Impregnation and Dendritic Complexity Analysis</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="4-methods.html"><a href="4-methods.html#behavioral-experiments"><i class="fa fa-check"></i><b>4.5</b> Behavioral Experiments</a><ul>
<li class="chapter" data-level="4.5.1" data-path="4-methods.html"><a href="4-methods.html#open-field-activity-1"><i class="fa fa-check"></i><b>4.5.1</b> Open Field Activity</a></li>
<li class="chapter" data-level="4.5.2" data-path="4-methods.html"><a href="4-methods.html#paired-pulse-inhibition-1"><i class="fa fa-check"></i><b>4.5.2</b> Paired-Pulse Inhibition</a></li>
<li class="chapter" data-level="4.5.3" data-path="4-methods.html"><a href="4-methods.html#context-cued-fear-conditioning"><i class="fa fa-check"></i><b>4.5.3</b> Context-Cued Fear Conditioning</a></li>
<li class="chapter" data-level="4.5.4" data-path="4-methods.html"><a href="4-methods.html#Barnesmeth"><i class="fa fa-check"></i><b>4.5.4</b> Barnes Maze</a></li>
<li class="chapter" data-level="4.5.5" data-path="4-methods.html"><a href="4-methods.html#social-activity-monitor-sam-1"><i class="fa fa-check"></i><b>4.5.5</b> Social Activity Monitor (SAM)</a></li>
<li class="chapter" data-level="4.5.6" data-path="4-methods.html"><a href="4-methods.html#micro-behaviors-homecagescan"><i class="fa fa-check"></i><b>4.5.6</b> Micro Behaviors (HomeCageScan)</a></li>
<li class="chapter" data-level="4.5.7" data-path="4-methods.html"><a href="4-methods.html#nestcomethod"><i class="fa fa-check"></i><b>4.5.7</b> Nest Construction</a></li>
</ul></li>
<li class="chapter" data-level="4.6" data-path="4-methods.html"><a href="4-methods.html#magnetic-resonance-imaging-mri-and-volumetrics"><i class="fa fa-check"></i><b>4.6</b> Magnetic Resonance Imaging (MRI) and Volumetrics</a></li>
<li class="chapter" data-level="4.7" data-path="4-methods.html"><a href="4-methods.html#reduced-read-bisulfite-sequencing-rrbseq"><i class="fa fa-check"></i><b>4.7</b> Reduced Read Bisulfite Sequencing (RRBSeq)</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html"><i class="fa fa-check"></i><b>5</b> Supplemental Data</a><ul>
<li class="chapter" data-level="5.1" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html#Behavsupplement"><i class="fa fa-check"></i><b>5.1</b> Behavior</a><ul>
<li class="chapter" data-level="5.1.1" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html#homecagescan-micro-behaviors"><i class="fa fa-check"></i><b>5.1.1</b> HomeCageScan Micro Behaviors</a></li>
<li class="chapter" data-level="5.1.2" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html#context-cued-fear-conditioning-1"><i class="fa fa-check"></i><b>5.1.2</b> Context-Cued Fear Conditioning</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html#DE"><i class="fa fa-check"></i><b>5.2</b> Differential Gene Expression</a><ul>
<li class="chapter" data-level="5.2.1" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html#differentially-expressed-genes-in-aged-hp1betako"><i class="fa fa-check"></i><b>5.2.1</b> Differentially Expressed Genes in Aged HP1<span class="math inline">\(\beta\)</span>KO</a></li>
<li class="chapter" data-level="5.2.2" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html#differentially-expressed-genes-in-hp1betagamma-dko"><i class="fa fa-check"></i><b>5.2.2</b> Differentially Expressed Genes in HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO</a></li>
<li class="chapter" data-level="5.2.3" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html#inflam"><i class="fa fa-check"></i><b>5.2.3</b> Genes Related to Anti-viral and Interferon Response in HP1<span class="math inline">\(\beta\)</span>/<span class="math inline">\(\gamma\)</span> DKO</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="5-supplemental-data.html"><a href="5-supplemental-data.html#genomic-hotspots-sensitive-to-hp1-deficiency-in-the-hippocampus"><i class="fa fa-check"></i><b>5.3</b> Genomic Hotspots Sensitive to HP1 Deficiency in the Hippocampus</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html"><i class="fa fa-check"></i><b>6</b> Bioinformatics &amp; Data Processing</a><ul>
<li class="chapter" data-level="6.1" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#software-environment-management"><i class="fa fa-check"></i><b>6.1</b> Software Environment Management</a></li>
<li class="chapter" data-level="6.2" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#snakemake-jobs"><i class="fa fa-check"></i><b>6.2</b> Snakemake Jobs</a></li>
<li class="chapter" data-level="6.3" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#tetranscripts-pipeline"><i class="fa fa-check"></i><b>6.3</b> TETranscripts Pipeline</a></li>
<li class="chapter" data-level="6.4" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#differential-gene-expression-using-edger"><i class="fa fa-check"></i><b>6.4</b> Differential Gene Expression Using EdgeR</a></li>
<li class="chapter" data-level="6.5" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#deeptools-pipeline"><i class="fa fa-check"></i><b>6.5</b> Deeptools Pipeline</a></li>
<li class="chapter" data-level="6.6" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#lions-pipeline"><i class="fa fa-check"></i><b>6.6</b> LIONS Pipeline</a></li>
<li class="chapter" data-level="6.7" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#analysis-of-reduced-read-bisulphite-sequencing-rrbs"><i class="fa fa-check"></i><b>6.7</b> Analysis of Reduced-Read Bisulphite Sequencing (RRBS)</a><ul>
<li class="chapter" data-level="6.7.1" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#methylkit-and-circos-plot"><i class="fa fa-check"></i><b>6.7.1</b> Methylkit and Circos Plot</a></li>
<li class="chapter" data-level="6.7.2" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#annotated-ideogram"><i class="fa fa-check"></i><b>6.7.2</b> Annotated Ideogram</a></li>
</ul></li>
<li class="chapter" data-level="6.8" data-path="6-bioinformatics-data-processing.html"><a href="6-bioinformatics-data-processing.html#about-this-thesis"><i class="fa fa-check"></i><b>6.8</b> About this Thesis</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="7-references.html"><a href="7-references.html"><i class="fa fa-check"></i><b>7</b> References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">HP1 Deficiency Results in Accelerated Neurodegeneration</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="references" class="section level1">
<h1><span class="header-section-number">Part 7</span> References</h1>
<!--
To remove the indentation of the first entry.
-->

<!--
To create a hanging indent and spacing between entries.  These three lines may need to be removed for styles that don't require the hanging indent.
-->
<p>  </p>
<!--
This is just for testing with more citations for the bibliography at the end.  Add other entries into the list here if you'd like them to appear in the bibliography even if they weren't explicitly cited in the document.
-->

<div id="refs" class="references">
<div>
<p>Akalin, Altuna, Matthias Kormaksson, Sheng Li, Francine E Garrett-Bakelman, Maria E Figueroa, Ari Melnick, and Christopher E Mason. 2012. “methylKit: A Comprehensive R Package for the Analysis of Genome-Wide DNA Methylation Profiles.” <em>Genome Biology</em> 13 (10): R87. doi:<a href="https://doi.org/10.1186/gb-2012-13-10-r87">10.1186/gb-2012-13-10-r87</a>.</p>
</div>
<div>
<p>Aristotle, and Arthur Leslie Peck. 1943. <em>Generation of Animals, with an English Translation by A.L. Peck</em>. Cambridge Harvard University Press. <a href="http://archive.org/details/generationofanim00arisuoft" class="uri">http://archive.org/details/generationofanim00arisuoft</a>.</p>
</div>
<div>
<p>Ashwal-Fluss, Reut, Markus Meyer, Nagarjuna Reddy Pamudurti, Andranik Ivanov, Osnat Bartok, Mor Hanan, Naveh Evantal, Sebastian Memczak, Nikolaus Rajewsky, and Sebastian Kadener. 2014. “circRNA Biogenesis Competes with Pre-mRNA Splicing.” <em>Molecular Cell</em> 56 (1): 55–66. doi:<a href="https://doi.org/10.1016/j.molcel.2014.08.019">10.1016/j.molcel.2014.08.019</a>.</p>
</div>
<div>
<p>Aucott, R., J. Bullwinkel, Y. Yu, W. Shi, M. Billur, J. P. Brown, U. Menzel, et al. 2008. “HP1- Is Required for Development of the Cerebral Neocortex and Neuromuscular Junctions.” <em>The Journal of Cell Biology</em> 183 (4): 597–606. doi:<a href="https://doi.org/10.1083/jcb.200804041">10.1083/jcb.200804041</a>.</p>
</div>
<div>
<p>Avrahami, Dana, Changhong Li, Jia Zhang, Jonathan Schug, Ran Avrahami, Shilpa Rao, Michael B. Stadler, Lukas Burger, Dirk Schübeler, and Benjamin Glaser. 2015. “Aging-Dependent Demethylation of Regulatory Elements Correlates with Chromatin State and Improved β Cell Function.” <em>Cell Metabolism</em> 22 (4): 619–32.</p>
</div>
<div>
<p>Ayoub, Nabieh, Anand D. Jeyasekharan, and Ashok R. Venkitaraman. 2009. “Mobilization and Recruitment of HP1β: A Bimodal Response to DNA Breakage.” <em>Cell Cycle</em> 8 (18): 2946–51. doi:<a href="https://doi.org/10.4161/cc.8.18.9486">10.4161/cc.8.18.9486</a>.</p>
</div>
<div>
<p>Ayyanathan, K. 2003. “Regulated Recruitment of Hp1 to a Euchromatic Gene Induces Mitotically Heritable, Epigenetic Gene Silencing: A Mammalian Cell Culture Model of Gene Variegation.” <em>Genes &amp; Development</em> 17 (15): 1855–69. doi:<a href="https://doi.org/10.1101/gad.1102803">10.1101/gad.1102803</a>.</p>
</div>
<div>
<p>Babaian, Artem, Richard Thompson, Jake Lever, Liane Gagnier, Mohammad M. Karimi, and Dixie L. Mager. 2018. “LIONS: Analysis Suite for Detecting and Quantifying Transposable Element Initiated Transcription from RNA-Seq.” <em>bioRxiv</em>, August, 149864. doi:<a href="https://doi.org/10.1101/149864">10.1101/149864</a>.</p>
</div>
<div>
<p>Bach, Erika A., Susanne J. Szabo, Anand S. Dighe, Avi Ashkenazi, Michel Aguet, Kenneth M. Murphy, and Robert D. Schreiber. 1995. “Ligand-Induced Autoregulation of IFN-<span class="math inline">\(\gamma\)</span> Receptor <span class="math inline">\(\beta\)</span> Chain Expression in T Helper Cell Subsets.” <em>Science</em> 270 (5239): 1215–8. doi:<a href="https://doi.org/10.1126/science.270.5239.1215">10.1126/science.270.5239.1215</a>.</p>
</div>
<div>
<p>Baker, H. F., R. M. Ridley, L. W. Duchen, T. J. Crow, and C. J. Bruton. 1994. “Induction of <span class="math inline">\(\beta\)</span> (A4)-Amyloid in Primates by Injection of Alzheimer’s Disease Brain Homogenate.” <em>Molecular Neurobiology</em> 8 (1): 25–39.</p>
</div>
<div>
<p>Bannister, Andrew J., Philip Zegerman, Janet F. Partridge, Eric A. Miska, Jean O. Thomas, Robin C. Allshire, and Tony Kouzarides. 2001. “Selective Recognition of Methylated Lysine 9 on Histone H3 by the Hp1 Chromo Domain.” <em>Nature</em> 410 (6824): 120–24.</p>
</div>
<div>
<p>Barau, Joan, Aurelie Teissandier, Natasha Zamudio, Stephanie Roy, Valerie Nalesso, Yann Herault, Florian Guillou, and Deborah Bourchis. 2016. “The DNA Methyltransferase Dnmt3c Protects Male Germ Cells from Transposon Activity.” <em>Science</em> 354 (6314): 909–12.</p>
</div>
<div>
<p>Barbot, Willy, Anne Dupressoir, Vladimir Lazar, and Thierry Heidmann. 2002. “Epigenetic Regulation of an IAP Retrotransposon in the Aging Mouse: Progressive Demethylation and de-Silencing of the Element by Its Repetitive Induction.” <em>Nucleic Acids Research</em> 30 (11): 2365–73. doi:<a href="https://doi.org/10.1093/nar/30.11.2365">10.1093/nar/30.11.2365</a>.</p>
</div>
<div>
<p>Barnes, C. A. 1979. “Memory Deficits Associated with Senescence: A Neurophysiological and Behavioral Study in the Rat.” <em>Journal of Comparative and Physiological Psychology</em> 93 (1): 74–104. doi:<a href="https://doi.org/10.1037/h0077579">10.1037/h0077579</a>.</p>
</div>
<div>
<p>Barreto, Guillermo, Andrea Schäfer, Joachim Marhold, Dirk Stach, Suresh K. Swaminathan, Vikas Handa, Gabi Döderlein, Nicole Maltry, Wei Wu, and Frank Lyko. 2007. “Gadd45a Promotes Epigenetic Gene Activation by Repair-Mediated DNA Demethylation.” <em>Nature</em> 445 (7128): 671.</p>
</div>
<div>
<p>Beerman, Isabel, Christoph Bock, Brian S. Garrison, Zachary D. Smith, Hongcang Gu, Alexander Meissner, and Derrick J. Rossi. 2013. “Proliferation-Dependent Alterations of the DNA Methylation Landscape Underlie Hematopoietic Stem Cell Aging.” <em>Cell Stem Cell</em> 12 (4): 413–25.</p>
</div>
<div>
<p>Bell, Jordana T., Pei-Chien Tsai, Tsun-Po Yang, Ruth Pidsley, James Nisbet, Daniel Glass, Massimo Mangino, et al. 2012. “Epigenome-Wide Scans Identify Differentially Methylated Regions for Age and Age-Related Phenotypes in a Healthy Ageing Population.” <em>PLOS Genetics</em> 8 (4): e1002629. doi:<a href="https://doi.org/10.1371/journal.pgen.1002629">10.1371/journal.pgen.1002629</a>.</p>
</div>
<div>
<p>Ben-Asouli, Yitzhak, Yona Banai, Yehuda Pel-Or, Alexei Shir, and Raymond Kaempfer. 2002. “Human Interferon-<span class="math inline">\(\gamma\)</span> mRNA Autoregulates Its Translation Through a Pseudoknot That Activates the Interferon-Inducible Protein Kinase PKR.” <em>Cell</em> 108 (2): 221–32.</p>
</div>
<div>
<p>Bernabei, Paola, Eliana M. Coccia, Laura Rigamonti, Marita Bosticardo, Guido Forni, Sidney Pestka, Christopher D. Krause, Angela Battistini, and Francesco Novelli. 2001. “Interferon-<span class="math inline">\(\gamma\)</span> Receptor 2 Expression as the Deciding Factor in Human T, B, and Myeloid Cell Proliferation or Death.” <em>Journal of Leukocyte Biology</em> 70 (6): 950–60. doi:<a href="https://doi.org/10.1189/jlb.70.6.950">10.1189/jlb.70.6.950</a>.</p>
</div>
<div>
<p>Bestor, Timothy, Andrew Laudano, Robert Mattaliano, and Vernon Ingram. 1988. “Cloning and Sequencing of a cDNA Encoding DNA Methyltransferase of Mouse Cells: The Carboxyl-Terminal Domain of the Mammalian Enzymes Is Related to Bacterial Restriction Methyltransferases.” <em>Journal of Molecular Biology</em> 203 (4): 971–83.</p>
</div>
<div>
<p>Bierhoff, Holger, Marcel Andre Dammert, David Brocks, Silvia Dambacher, Gunnar Schotta, and Ingrid Grummt. 2014. “Quiescence-Induced LncRNAs Trigger H4k20 Trimethylation and Transcriptional Silencing.” <em>Molecular Cell</em> 54 (4): 675–82. doi:<a href="https://doi.org/10.1016/j.molcel.2014.03.032">10.1016/j.molcel.2014.03.032</a>.</p>
</div>
<div>
<p>Billur, Mustafa, Hans D. Bartunik, and Prim B. Singh. 2010. “The Essential Function of HP1β: A Case of the Tail Wagging the Dog?” <em>Trends in Biochemical Sciences</em> 35 (2): 115–23. doi:<a href="https://doi.org/10.1016/j.tibs.2009.09.003">10.1016/j.tibs.2009.09.003</a>.</p>
</div>
<div>
<p>Bilodeau, Steve, Michael H. Kagey, Garrett M. Frampton, Peter B. Rahl, and Richard A. Young. 2009. “SetDB1 Contributes to Repression of Genes Encoding Developmental Regulators and Maintenance of ES Cell State.” <em>Genes &amp; Development</em> 23 (21): 2484–9.</p>
</div>
<div>
<p>Bird, Adrian P. 1986. “CpG-Rich Islands and the Function of DNA Methylation.” <em>Nature</em> 321 (6067): 209–13.</p>
</div>
<div>
<p>Blasco, Mar’ia A. 2007. “The Epigenetic Regulation of Mammalian Telomeres.” <em>Nature Reviews Genetics</em> 8 (4): 299–309. doi:<a href="https://doi.org/10.1038/nrg2047">10.1038/nrg2047</a>.</p>
</div>
<div>
<p>Bodea, Liviu-Gabriel, Yiner Wang, Bettina Linnartz-Gerlach, Jens Kopatz, Lasse Sinkkonen, Ruth Musgrove, Tony Kaoma, et al. 2014. “Neurodegeneration by Activation of the Microglial ComplementPhagosome Pathway.” <em>Journal of Neuroscience</em> 34 (25): 8546–56. doi:<a href="https://doi.org/10.1523/JNEUROSCI.5002-13.2014">10.1523/JNEUROSCI.5002-13.2014</a>.</p>
</div>
<div>
<p>Bollati, V., D. Galimberti, L. Pergoli, E. Dalla Valle, F. Barretta, F. Cortini, E. Scarpini, P. A. Bertazzi, and A. Baccarelli. 2011. “DNA Methylation in Repetitive Elements and Alzheimer Disease.” <em>Brain, Behavior, and Immunity</em> 25 (6): 1078–83. doi:<a href="https://doi.org/10.1016/j.bbi.2011.01.017">10.1016/j.bbi.2011.01.017</a>.</p>
</div>
<div>
<p>Bollati, Valentina, Joel Schwartz, Robert Wright, Augusto Litonjua, Letizia Tarantini, Helen Suh, David Sparrow, Pantel Vokonas, and Andrea Baccarelli. 2009. “Decline in Genomic DNA Methylation Through Aging in a Cohort of Elderly Subjects.” <em>Mechanisms of Ageing and Development</em> 130 (4): 234–39. doi:<a href="https://doi.org/10.1016/j.mad.2008.12.003">10.1016/j.mad.2008.12.003</a>.</p>
</div>
<div>
<p>Bonthuis, Paul J., Wei-Chao Huang, Cornelia N. Stacher Hörndli, Elliott Ferris, Tong Cheng, and Christopher Gregg. 2015. “Noncanonical Genomic Imprinting Effects in Offspring.” <em>Cell Reports</em> 12 (6): 979–91. doi:<a href="https://doi.org/10.1016/j.celrep.2015.07.017">10.1016/j.celrep.2015.07.017</a>.</p>
</div>
<div>
<p>Booth, Lauren N., and Anne Brunet. 2016. “The Aging Epigenome.” <em>Molecular Cell</em> 62 (5): 728–44. doi:<a href="https://doi.org/10.1016/j.molcel.2016.05.013">10.1016/j.molcel.2016.05.013</a>.</p>
</div>
<div>
<p>Bormuth, I., K. Yan, T. Yonemasu, M. Gummert, M. Zhang, S. Wichert, O. Grishina, et al. 2013. “Neuronal Basic Helix-Loop-Helix Proteins Neurod2/6 Regulate Cortical Commissure Formation Before Midline Interactions.” <em>Journal of Neuroscience</em> 33 (2): 641–51. doi:<a href="https://doi.org/10.1523/JNEUROSCI.0899-12.2013">10.1523/JNEUROSCI.0899-12.2013</a>.</p>
</div>
<div>
<p>Bosch-Presegu’e, Laia, Helena Raurell-Vila, Joshua K. Thackray, Jessica Gonz’alez, Carmen Casal, Noriko Kane-Goldsmith, Miguel Vizoso, et al. 2017. “Mammalian Hp1 Isoforms Have Specific Roles in Heterochromatin Structure and Organization.” <em>Cell Reports</em> 21 (8): 2048–57. doi:<a href="https://doi.org/10.1016/j.celrep.2017.10.092">10.1016/j.celrep.2017.10.092</a>.</p>
</div>
<div>
<p>Bourc’his, D’eborah, Guo-Liang Xu, Chyuan-Sheng Lin, Brooke Bollman, and Timothy H. Bestor. 2001. “Dnmt3L and the Establishment of Maternal Genomic Imprints.” <em>Science</em> 294 (5551): 2536–9.</p>
</div>
<div>
<p>Boyle, Patrick, Kendell Clement, Hongcang Gu, Zachary D. Smith, Michael Ziller, Jennifer L. Fostel, Laurie Holmes, et al. 2012. “Gel-Free Multiplexed Reduced Representation Bisulfite Sequencing for Large-Scale DNA Methylation Profiling.” <em>Genome Biology</em> 13 (10): R92. doi:<a href="https://doi.org/10.1186/gb-2012-13-10-r92">10.1186/gb-2012-13-10-r92</a>.</p>
</div>
<div>
<p>Bradley-Whitman, M. A., and M. A. Lovell. 2013. “Epigenetic Changes in the Progression of Alzheimer’s Disease.” <em>Mechanisms of Ageing and Development</em>, Genome dynamics shaping neuroscience, 134 (10): 486–95. doi:<a href="https://doi.org/10.1016/j.mad.2013.08.005">10.1016/j.mad.2013.08.005</a>.</p>
</div>
<div>
<p>Brady, Troy, Young Nam Lee, Keshet Ronen, Nirav Malani, Charles C. Berry, Paul D. Bieniasz, and Frederic D. Bushman. 2009. “Integration Target Site Selection by a Resurrected Human Endogenous Retrovirus.” <em>Genes &amp; Development</em> 23 (5): 633–42.</p>
</div>
<div>
<p>Braff, D. L., M. A. Geyer, and N. R. Swerdlow. 2001. “Human Studies of Prepulse Inhibition of Startle: Normal Subjects, Patient Groups, and Pharmacological Studies.” <em>Psychopharmacology</em> 156 (2-3): 234–58.</p>
</div>
<div>
<p>Brasher, Sally V., Brian O. Smith, Rasmus H. Fogh, Daniel Nietlispach, Abarna Thiru, Peter R. Nielsen, R. William Broadhurst, Linda J. Ball, Natalia V. Murzina, and Ernest D. Laue. 2000. “The Structure of Mouse Hp1 Suggests a Unique Mode of Single Peptide Recognition by the Shadow Chromo Domain Dimer.” <em>The EMBO Journal</em> 19 (7): 1587–97. doi:<a href="https://doi.org/10.1093/emboj/19.7.1587">10.1093/emboj/19.7.1587</a>.</p>
</div>
<div>
<p>Brocks, David, Christopher R Schmidt, Michael Daskalakis, Hyo Sik Jang, Nakul M Shah, Daofeng Li, Jing Li, et al. 2017. “DNMT and HDAC Inhibitors Induce Cryptic Transcription Start Sites Encoded in Long Terminal Repeats.” <em>Nature Genetics</em> 49 (7): 1052–60. doi:<a href="https://doi.org/10.1038/ng.3889">10.1038/ng.3889</a>.</p>
</div>
<div>
<p>Brown, Carolyn J., Brian D. Hendrich, Jim L. Rupert, Ronald G. Lafreniere, Yigong Xing, Jeanne Lawrence, and Huntington F. Willard. 1992. “The Human XIST Gene: Analysis of a 17 Kb Inactive X-Specific RNA That Contains Conserved Repeats and Is Highly Localized Within the Nucleus.” <em>Cell</em> 71 (3): 527–42.</p>
</div>
<div>
<p>Brown, Jeremy P., Jörn Bullwinkel, Bettina Baron-Lühr, Mustafa Billur, Philipp Schneider, Heinz Winking, and Prim B. Singh. 2010. “HP1γ Function Is Required for Male Germ Cell Survival and Spermatogenesis.” <em>Epigenetics &amp; Chromatin</em> 3: 9. doi:<a href="https://doi.org/10.1186/1756-8935-3-9">10.1186/1756-8935-3-9</a>.</p>
</div>
<div>
<p>Brudek, T., T. Christensen, H. J. Hansen, T. Petersen, and A. Møller-Larsen. 2008. “Synergistic Immune Responses Induced by Endogenous Retrovirus and Herpesvirus Antigens Result in Increased Production of Inflammatory Cytokines in Multiple Sclerosis Patients.” <em>Scandinavian Journal of Immunology</em> 67 (3): 295–303. doi:<a href="https://doi.org/10.1111/j.1365-3083.2007.02067.x">10.1111/j.1365-3083.2007.02067.x</a>.</p>
</div>
<div>
<p>Caillier, Maïa, Sandrine Th’enot, Violaine Tribollet, Anne-Marie Birot, Jacques Samarut, and Anne Mey. 2010. “Role of the Epigenetic Regulator Hp1<span class="math inline">\(\gamma\)</span> in the Control of Embryonic Stem Cell Properties.” Edited by Michael J. Pazin. <em>PLoS ONE</em> 5 (11): e15507. doi:<a href="https://doi.org/10.1371/journal.pone.0015507">10.1371/journal.pone.0015507</a>.</p>
</div>
<div>
<p>Cammas, Florence, Agnes Janoshazi, Thierry Lerouge, and R’egine Losson. 2007. “Dynamic and Selective Interactions of the Transcriptional Corepressor TIF1β with the Heterochromatin Protein Hp1 Isotypes During Cell Differentiation.” <em>Differentiation</em> 75 (7): 627–37.</p>
</div>
<div>
<p>Campbell, Sydney L., and Kathryn E. Wellen. 2018. “Metabolic Signaling to the Nucleus in Cancer.” <em>Molecular Cell</em> 71 (3): 398–408.</p>
</div>
<div>
<p>Canzio, Daniele, Adam Larson, and Geeta J. Narlikar. 2014. “Mechanisms of Functional Promiscuity by Hp1 Proteins.” <em>Trends in Cell Biology</em> 24 (6): 377–86. doi:<a href="https://doi.org/10.1016/j.tcb.2014.01.002">10.1016/j.tcb.2014.01.002</a>.</p>
</div>
<div>
<p>Canzio, Daniele, Maofu Liao, Nariman Naber, Edward Pate, Adam Larson, Shenping Wu, Diana B. Marina, et al. 2013. “A Conformational Switch in Hp1 Releases Auto-Inhibition to Drive Heterochromatin Assembly.” <em>Nature</em> 496 (7445): 377–81. doi:<a href="https://doi.org/10.1038/nature12032">10.1038/nature12032</a>.</p>
</div>
<div>
<p>Cao, Ru, and Yi Zhang. 2004. “The Functions of E(Z)/Ezh2-Mediated Methylation of Lysine 27 in Histone H3.” <em>Current Opinion in Genetics &amp; Development</em> 14 (2): 155–64. doi:<a href="https://doi.org/10.1016/j.gde.2004.02.001">10.1016/j.gde.2004.02.001</a>.</p>
</div>
<div>
<p>Cervantes-Ortiz, Sandra L., Natalia Zamorano Cuervo, and Nathalie Grandvaux. 2016. “Respiratory Syncytial Virus and Cellular Stress Responses: Impact on Replication and Physiopathology.” <em>Viruses</em> 8 (5): 124. doi:<a href="https://doi.org/10.3390/v8050124">10.3390/v8050124</a>.</p>
</div>
<div>
<p>Chakrabarty, Paramita, Carolina Ceballos-Diaz, Amanda Beccard, Christopher Janus, Dennis Dickson, Todd E. Golde, and Pritam Das. 2010. “IFN-<span class="math inline">\(\gamma\)</span> Promotes Complement Expression and Attenuates Amyloid Plaque Deposition in Amyloid <span class="math inline">\(\beta\)</span> Precursor Protein Transgenic Mice.” <em>The Journal of Immunology</em> 184 (9): 5333–43. doi:<a href="https://doi.org/10.4049/jimmunol.0903382">10.4049/jimmunol.0903382</a>.</p>
</div>
<div>
<p>Chandra, Tamir, Philip Andrew Ewels, Stefan Schoenfelder, Mayra Furlan-Magaril, Steven William Wingett, Kristina Kirschner, Jean-Yves Thuret, Simon Andrews, Peter Fraser, and Wolf Reik. 2015. “Global Reorganization of the Nuclear Landscape in Senescent Cells.” <em>Cell Reports</em> 10 (4): 471–83. doi:<a href="https://doi.org/10.1016/j.celrep.2014.12.055">10.1016/j.celrep.2014.12.055</a>.</p>
</div>
<div>
<p>Cheloufi, Sihem, and Konrad Hochedlinger. 2017. “Emerging Roles of the Histone Chaperone CAF-1 in Cellular Plasticity.” <em>Current Opinion in Genetics &amp; Development</em> 46: 83–94.</p>
</div>
<div>
<p>Cheloufi, Sihem, Ulrich Elling, Barbara Hopfgartner, Youngsook L. Jung, Jernej Murn, Maria Ninova, Maria Hubmann, Aimee I. Badeaux, Cheen Euong Ang, and Danielle Tenen. 2015. “The Histone Chaperone CAF-1 Safeguards Somatic Cell Identity.” <em>Nature</em> 528 (7581): 218.</p>
</div>
<div>
<p>Chen, Hu, Svetlana Dzitoyeva, and Hari Manev. 2012. “Effect of Aging on 5-Hydroxymethylcytosine in the Mouse Hippocampus.” <em>Restorative Neurology and Neuroscience</em> 30 (3): 237–45.</p>
</div>
<div>
<p>Cherwinski, H. M., J. H. Schumacher, K. D. Brown, and T. R. Mosmann. 1987. “Two Types of Mouse Helper T Cell Clone. III. Further Differences in Lymphokine Synthesis Between Th1 and Th2 Clones Revealed by RNA Hybridization, Functionally Monospecific Bioassays, and Monoclonal Antibodies.” <em>Journal of Experimental Medicine</em> 166 (5): 1229–44. doi:<a href="https://doi.org/10.1084/jem.166.5.1229">10.1084/jem.166.5.1229</a>.</p>
</div>
<div>
<p>Chiappinelli, Katherine B., Pamela L. Strissel, Alexis Desrichard, Huili Li, Christine Henke, Benjamin Akman, Alexander Hein, et al. 2015. “Inhibiting DNA Methylation Causes an Interferon Response in Cancer via dsRNA Including Endogenous Retroviruses.” <em>Cell</em> 162 (5): 974–86. doi:<a href="https://doi.org/10.1016/j.cell.2015.07.011">10.1016/j.cell.2015.07.011</a>.</p>
</div>
<div>
<p>Christensen, Tove, Thor Petersen, Steffen Thiel, Tomasz Brudek, Svend Ellermann-Eriksen, and Ann’e Møller-Larsen. 2007. “GeneEnvironment Interactions in Multiple Sclerosis: Innate and Adaptive Immune Responses to Human Endogenous Retrovirus and Herpesvirus Antigens and the Lectin Complement Activation Pathway.” <em>Journal of Neuroimmunology</em> 183 (1): 175–88. doi:<a href="https://doi.org/10.1016/j.jneuroim.2006.09.014">10.1016/j.jneuroim.2006.09.014</a>.</p>
</div>
<div>
<p>Chuang, Linda S.-H., Hang-In Ian, Tong-Wey Koh, Huck-Hui Ng, Guoliang Xu, and Benjamin FL Li. 1997. “Human DNA-(Cytosine-5) Methyltransferase-PCNA Complex as a Target for p21WAF1.” <em>Science</em> 277 (5334): 1996–2000.</p>
</div>
<div>
<p>Chung, Chang Geon, Min Jee Kwon, Keun Hye Jeon, Do Young Hyeon, Myeong Hoon Han, Jeong Hyang Park, In Jun Cha, et al. 2017. “Golgi Outpost Synthesis Impaired by Toxic Polyglutamine Proteins Contributes to Dendritic Pathology in Neurons.” <em>Cell Reports</em> 20 (2): 356–69. doi:<a href="https://doi.org/10.1016/j.celrep.2017.06.059">10.1016/j.celrep.2017.06.059</a>.</p>
</div>
<div>
<p>Chuong, Edward B., MA Karim Rumi, Michael J. Soares, and Julie C. Baker. 2013. “Endogenous Retroviruses Function as Species-Specific Enhancer Elements in the Placenta.” <em>Nature Genetics</em> 45 (3): 325–29.</p>
</div>
<div>
<p>Clarke, Laura E., Shane A. Liddelow, Chandrani Chakraborty, Alexandra E. Münch, Myriam Heiman, and Ben A. Barres. 2018. “Normal Aging Induces A1-Like Astrocyte Reactivity.” <em>Proceedings of the National Academy of Sciences</em> 115 (8): E1896–E1905. doi:<a href="https://doi.org/10.1073/pnas.1800165115">10.1073/pnas.1800165115</a>.</p>
</div>
<div>
<p>Clavarino, Giovanna, Nuno Cl’audio, Th’er‘ese Couderc, Alexandre Dalet, Delphine Judith, Voahirana Camosseto, Enrico K. Schmidt, et al. 2012. “Induction of Gadd34 Is Necessary for dsRNA-Dependent Interferon-<span class="math inline">\(\beta\)</span> Production and Participates in the Control of Chikungunya Virus Infection.” <em>PLOS Pathogens</em> 8 (5): e1002708. doi:<a href="https://doi.org/10.1371/journal.ppat.1002708">10.1371/journal.ppat.1002708</a>.</p>
</div>
<div>
<p>Conti, Laura, Raffaele De Palma, Simona Rolla, Daniela Boselli, Gabriella Rodolico, Surinder Kaur, Olli Silvennoinen, et al. 2012. “Th17 Cells in Multiple Sclerosis Express Higher Levels of Jak2, Which Increases Their Surface Expression of IFN-<span class="math inline">\(\gamma\)</span>R2.” <em>The Journal of Immunology</em>, January, 1004013. doi:<a href="https://doi.org/10.4049/jimmunol.1004013">10.4049/jimmunol.1004013</a>.</p>
</div>
<div>
<p>Copped‘e, Fabio, and Lucia Migliore. 2015. “DNA Damage in Neurodegenerative Diseases.” <em>Mutation Research/Fundamental and Molecular Mechanisms of Mutagenesis</em>, DNA damage in chronic diseases and aging, 776 (June): 84–97. doi:<a href="https://doi.org/10.1016/j.mrfmmm.2014.11.010">10.1016/j.mrfmmm.2014.11.010</a>.</p>
</div>
<div>
<p>Cordaux, Richard, and Mark A. Batzer. 2009. “The Impact of Retrotransposons on Human Genome Evolution.” <em>Nature Reviews Genetics</em> 10 (10): 691–703.</p>
</div>
<div>
<p>Coufal, Nicole G., Jos‘e Luis Garcia-Perez, Grace E. Peng, Maria C. N. Marchetto, Alysson R. Muotri, Yangling Mu, Christian T. Carson, Angela Macia, John V. Moran, and Fred H. Gage. 2011. “Ataxia Telangiectasia Mutated (ATM) Modulates Long Interspersed Element-1 (L1) Retrotransposition in Human Neural Stem Cells.” <em>Proceedings of the National Academy of Sciences of the United States of America</em> 108 (51): 20382–7. doi:<a href="https://doi.org/10.1073/pnas.1100273108">10.1073/pnas.1100273108</a>.</p>
</div>
<div>
<p>Crotta, Stefania, Sophia Davidson, Tanel Mahlakoiv, Christophe J. Desmet, Matthew R. Buckwalter, Matthew L. Albert, Peter Staeheli, and Andreas Wack. 2013. “Type I and Type III Interferons Drive Redundant Amplification Loops to Induce a Transcriptional Signature in Influenza-Infected Airway Epithelia.” <em>PLoS Pathogens</em> 9 (11): e1003773.</p>
</div>
<div>
<p>Cuellar, Trinna L., Anna-Maria Herzner, Xiaotian Zhang, Yogesh Goyal, Colin Watanabe, Brad A. Friedman, Vasantharajan Janakiraman, et al. 2017. “<span class="math inline">\(\-\-\-\)</span>Silencing of Retrotransposons by Setdb1 Inhibits the Interferon Response in Acute Myeloid Leukemia<span class="math inline">\(\-\-\)</span>.” <em>The Journal of Cell Biology</em> 216 (11): 3535–49. doi:<a href="https://doi.org/10.1083/jcb.201612160">10.1083/jcb.201612160</a>.</p>
</div>
<div>
<p>D. Gould, Todd. 2009. <em>Mood and Anxiety Related Phenotypes in Mice: Characterization Using Behavioral Tests</em>.</p>
</div>
<div>
<p>Dai, P., C. Meyer, K. Shaw, Y. Wang, L. Anderson, S. Shuman, T. Tuschl, and L. Deng. 2018. “The Cytosolic dsRNA-Sensing Pathway Mediated by Mda5/MAVS/Irf3 Is Critical for the Induction of Type I and III IFNs After Viral Infection of Skin Keratinocytes.” <em>Journal of Investigative Dermatology</em>, International Investigative Dermatology (IID) 2018 Meeting Abstract Supplement, 138 (5, Supplement): S154. doi:<a href="https://doi.org/10.1016/j.jid.2018.03.916">10.1016/j.jid.2018.03.916</a>.</p>
</div>
<div>
<p>Dawson, Mark A., Andrew J. Bannister, Berthold Göttgens, Samuel D. Foster, Till Bartke, Anthony R. Green, and Tony Kouzarides. 2009. “JAK2 Phosphorylates Histone H3y41 and Excludes HP1α from Chromatin.” <em>Nature</em> 461 (7265): 819–22. doi:<a href="https://doi.org/10.1038/nature08448">10.1038/nature08448</a>.</p>
</div>
<div>
<p>De Cecco, Marco, Steven W. Criscione, Edward J. Peckham, Sara Hillenmeyer, Eliza A. Hamm, Jayameenakshi Manivannan, Abigail L. Peterson, Jill A. Kreiling, Nicola Neretti, and John M. Sedivy. 2013. “Genomes of Replicatively Senescent Cells Undergo Global Epigenetic Changes Leading to Gene Silencing and Activation of Transposable Elements.” <em>Aging Cell</em> 12 (2): 247–56. doi:<a href="https://doi.org/10.1111/acel.12047">10.1111/acel.12047</a>.</p>
</div>
<div>
<p>de Magalhães, João Pedro, João Curado, and George M. Church. 2009. “Meta-Analysis of Age-Related Gene Expression Profiles Identifies Common Signatures of Aging.” <em>Bioinformatics</em> 25 (7): 875–81. doi:<a href="https://doi.org/10.1093/bioinformatics/btp073">10.1093/bioinformatics/btp073</a>.</p>
</div>
<div>
<p>Deacon, Robert. 2012. “Assessing Burrowing, Nest Construction, and Hoarding in Mice.” <em>Journal of Visualized Experiments : JoVE</em>, no. 59 (January). doi:<a href="https://doi.org/10.3791/2607">10.3791/2607</a>.</p>
</div>
<div>
<p>Deacon, Robert M. J., and J. Nicholas P. Rawlins. 2002. “Learning Impairments of Hippocampal-Lesioned Mice in a Paddling Pool.” <em>Behavioral Neuroscience</em> 116 (3): 472–78. doi:<a href="https://doi.org/10.1037/0735-7044.116.3.472">10.1037/0735-7044.116.3.472</a>.</p>
</div>
<div>
<p>Desplats, Paula, Brian Spencer, Elizabeth Coffee, Pruthul Patel, Sarah Michael, Christina Patrick, Anthony Adame, Edward Rockenstein, and Eliezer Masliah. 2011. “<span class="math inline">\(\alpha\)</span>-Synuclein Sequesters Dnmt1 from the Nucleus: A NOVEL MECHANISM FOR EPIGENETIC ALTERATIONS IN LEWY BODY DISEASES.” <em>Journal of Biological Chemistry</em> 286 (11): 9031–7. doi:<a href="https://doi.org/10.1074/jbc.C110.212589">10.1074/jbc.C110.212589</a>.</p>
</div>
<div>
<p>Devanapally, Sindhuja, Snusha Ravikumar, and Antony M. Jose. 2015. “Double-Stranded RNA Made in C. Elegans Neurons Can Enter the Germline and Cause Transgenerational Gene Silencing.” <em>Proceedings of the National Academy of Sciences</em> 112 (7): 2133–8.</p>
</div>
<div>
<p>Dewannieux, Marie, Anne Dupressoir, Francis Harper, G’erard Pierron, and Thierry Heidmann. 2004. “Identification of Autonomous IAP LTR Retrotransposons Mobile in Mammalian Cells.” <em>Nature Genetics</em> 36 (5): 534–39. doi:<a href="https://doi.org/10.1038/ng1353">10.1038/ng1353</a>.</p>
</div>
<div>
<p>Dewannieux, Marie, C’ecile Esnault, and Thierry Heidmann. 2003. “LINE-Mediated Retrotransposition of Marked Alu Sequences.” <em>Nature Genetics</em> 35 (1): 41–48.</p>
</div>
<div>
<p>Di Leonardo, Aldo, Steven P. Linke, Kris Clarkin, and Geoffrey M. Wahl. 1994. “DNA Damage Triggers a Prolonged P53-Dependent G1 Arrest and Long-Term Induction of Cip1 in Normal Human Fibroblasts.” <em>Genes &amp; Development</em> 8 (21): 2540–51.</p>
</div>
<div>
<p>Dialynas, G. K. 2006. “Methylation-Independent Binding to Histone H3 and Cell Cycle-Dependent Incorporation of HP1beta into Heterochromatin.” <em>Journal of Biological Chemistry</em> 281 (20): 14350–60. doi:<a href="https://doi.org/10.1074/jbc.M600558200">10.1074/jbc.M600558200</a>.</p>
</div>
<div>
<p>Dialynas, George K., Stefan Terjung, Jeremy P. Brown, Rebecca L. Aucott, Bettina Baron-Luhr, Prim B. Singh, and Spyros D. Georgatos. 2007. “Plasticity of Hp1 Proteins in Mammalian Cells.” <em>Journal of Cell Science</em> 120 (19): 3415–24.</p>
</div>
<div>
<p>Dialynas, George K., Michael W. Vitalini, and Lori L. Wallrath. 2008. “Linking Heterochromatin Protein 1 (Hp1) to Cancer Progression.” <em>Mutation Research/Fundamental and Molecular Mechanisms of Mutagenesis</em>, Epigenetics of development and human disease, 647 (1): 13–20. doi:<a href="https://doi.org/10.1016/j.mrfmmm.2008.09.007">10.1016/j.mrfmmm.2008.09.007</a>.</p>
</div>
<div>
<p>Djeghloul, Dounia, Klaudia Kuranda, Isabelle Kuzniak, Daniela Barbieri, Irina Naguibneva, Caroline Choisy, Jean-Christophe Bories, et al. 2016. “Age-Associated Decrease of the Histone Methyltransferase Suv39h1 in HSC Perturbs Heterochromatin and B Lymphoid Differentiation.” <em>Stem Cell Reports</em> 6 (6): 970–84. doi:<a href="https://doi.org/10.1016/j.stemcr.2016.05.007">10.1016/j.stemcr.2016.05.007</a>.</p>
</div>
<div>
<p>Douville, Ren’ee, Jiankai Liu, Jeffrey Rothstein, and Avindra Nath. 2011. “Identification of Active Loci of a Human Endogenous Retrovirus in Neurons of Patients with Amyotrophic Lateral Sclerosis.” <em>Annals of Neurology</em> 69 (1): 141–51.</p>
</div>
<div>
<p>Ecco, Gabriela, Marco Cassano, Annamaria Kauzlaric, Julien Duc, Andrea Coluccio, Sandra Offner, Michaël Imbeault, Helen M. Rowe, Priscilla Turelli, and Didier Trono. 2016. “Transposable Elements and Their KRAB-ZFP Controllers Regulate Gene Expression in Adult Tissues.” <em>Developmental Cell</em> 36 (6): 611–23. doi:<a href="https://doi.org/10.1016/j.devcel.2016.02.024">10.1016/j.devcel.2016.02.024</a>.</p>
</div>
<div>
<p>Edmunds, John W., Louis C. Mahadevan, and Alison L. Clayton. 2008. “Dynamic Histone H3 Methylation During Gene Induction: HYPB/Setd2 Mediates All H3k36 Trimethylation.” <em>The EMBO Journal</em> 27 (2): 406–20.</p>
</div>
<div>
<p>Ehrlich, Melanie, Miguel A. Gama-Sosa, Lan-Hsiang Huang, Rose Marie Midgett, Kenneth C. Kuo, Roy A. McCune, and Charles Gehrke. 1982. “Amount and Distribution of 5-Methylcytosine in Human DNA from Different Types of Tissues or Cells.” <em>Nucleic Acids Research</em> 10 (8): 2709–21. doi:<a href="https://doi.org/10.1093/nar/10.8.2709">10.1093/nar/10.8.2709</a>.</p>
</div>
<div>
<p>Eimer, William A., Deepak Kumar Vijaya Kumar, Nanda Kumar Navalpur Shanmugam, Alex S. Rodriguez, Teryn Mitchell, Kevin J. Washicosky, Bence György, Xandra O. Breakefield, Rudolph E. Tanzi, and Robert D. Moir. 2018. “Alzheimer’s Disease-Associated <span class="math inline">\(\beta\)</span>-Amyloid Is Rapidly Seeded by Herpesviridae to Protect Against Brain Infection.” <em>Neuron</em> 99 (1): 56–63.e3. doi:<a href="https://doi.org/10.1016/j.neuron.2018.06.030">10.1016/j.neuron.2018.06.030</a>.</p>
</div>
<div>
<p>El Hajj, Nady, Marcus Dittrich, and Thomas Haaf. 2017. “Epigenetic Dysregulation of Protocadherins in Human Disease.” <em>Seminars in Cell &amp; Developmental Biology</em>, Spectraplakins, versatile roles in physiology and pathology, 69 (September): 172–82. doi:<a href="https://doi.org/10.1016/j.semcdb.2017.07.007">10.1016/j.semcdb.2017.07.007</a>.</p>
</div>
<div>
<p>Eskeland, Ragnhild, Martin Leeb, Graeme R. Grimes, Cl’emence Kress, Shelagh Boyle, Duncan Sproul, Nick Gilbert, et al. 2010. “Ring1B Compacts Chromatin Structure and Represses Gene Expression Independent of Histone Ubiquitination.” <em>Molecular Cell</em> 38 (3): 452–64. doi:<a href="https://doi.org/10.1016/j.molcel.2010.02.032">10.1016/j.molcel.2010.02.032</a>.</p>
</div>
<div>
<p>Evans, H. K. 2005. “Comparative Phylogenetic Analysis of Blcap/Nnat Reveals Eutherian-Specific Imprinted Gene.” <em>Molecular Biology and Evolution</em> 22 (8): 1740–8. doi:<a href="https://doi.org/10.1093/molbev/msi165">10.1093/molbev/msi165</a>.</p>
</div>
<div>
<p>Fabregat, Antonio, Steven Jupe, Lisa Matthews, Konstantinos Sidiropoulos, Marc Gillespie, Phani Garapati, Robin Haw, et al. 2018. “The Reactome Pathway Knowledgebase.” <em>Nucleic Acids Research</em> 46 (D1): D649–D655. doi:<a href="https://doi.org/10.1093/nar/gkx1132">10.1093/nar/gkx1132</a>.</p>
</div>
<div>
<p>Feinberg, Andrew P. 2018. “The Key Role of Epigenetics in Human Disease Prevention and Mitigation.” <em>New England Journal of Medicine</em> 378 (14): 1323–34.</p>
</div>
<div>
<p>Fern’andez, Agust’in F., Gustavo F. Bay’on, Roc’io G. Urdinguio, Estela G. Toraño, Mar’ia G. Garc’ia, Antonella Carella, Sandra Petrus-Reurer, et al. 2015. “H3K4me1 Marks DNA Regions Hypomethylated During Aging in Human Stem and Differentiated Cells.” <em>Genome Research</em> 25 (1): 27–40. doi:<a href="https://doi.org/10.1101/gr.169011.113">10.1101/gr.169011.113</a>.</p>
</div>
<div>
<p>Ferrari, Alessandra, Raffaella Longo, Rui Silva, Nico Mitro, Donatella Caruso, Emma De Fabiani, and Maurizio Crestani. 2018. “Epigenome Modifiers and Metabolic Rewiring: New Frontiers in Therapeutics.” <em>Pharmacology &amp; Therapeutics</em>.</p>
</div>
<div>
<p>Fiore, Barbara Di, Antonella Palena, Armando Felsan, Franco Palitti, Maurizia Caruso, and Patrizia Lavia. 1999. “Cytosine Methylation Transforms an E2f Site in the Retinoblastoma Gene Promoter into a Binding Site for the General Repressor Methylcytosine-Binding Protein 2 (MeCP2).” <em>Nucleic Acids Research</em> 27 (14): 2852–9.</p>
</div>
<div>
<p>Fiorenzano, Alessandro, Emilia Pascale, Miriam Gagliardi, Sara Terreri, Mariarosaria Papa, Gennaro Andolfi, Marco Galasso, et al. 2018. “An Ultraconserved Element Containing lncRNA Preserves Transcriptional Dynamics and Maintains ESC Self-Renewal.” <em>Stem Cell Reports</em> 10 (3): 1102–14. doi:<a href="https://doi.org/10.1016/j.stemcr.2018.01.014">10.1016/j.stemcr.2018.01.014</a>.</p>
</div>
<div>
<p>Fischer, B., H. Schmoll, D. Platt, A. Popa-Wagner, P. Riederer, and J. Bauer. 1995. “Complement C1q and C3 mRNA Expression in the Frontal Cortex of Alzheimer’s Patients.” <em>Journal of Molecular Medicine</em> 73 (9): 465–71. doi:<a href="https://doi.org/10.1007/BF00202265">10.1007/BF00202265</a>.</p>
</div>
<div>
<p>Fitzpatrick, Terry, and Sui Huang. 2012. “3’-UTR-Located Inverted Alu Repeats Facilitate mRNA Translational Repression and Stress Granule Accumulation.” <em>Nucleus</em> 3 (4): 359–69. doi:<a href="https://doi.org/10.4161/nucl.20827">10.4161/nucl.20827</a>.</p>
</div>
<div>
<p>Fnu, Sheema, Elizabeth A. Williamson, Leyma P. De Haro, Mark Brenneman, Justin Wray, Montaser Shaheen, Krishnan Radhakrishnan, Suk-Hee Lee, Jac A. Nickoloff, and Robert Hromas. 2011. “Methylation of Histone H3 Lysine 36 Enhances DNA Repair by Nonhomologous End-Joining.” <em>Proceedings of the National Academy of Sciences</em> 108 (2): 540–45. <a href="http://www.pnas.org/content/108/2/540.short" class="uri">http://www.pnas.org/content/108/2/540.short</a>.</p>
</div>
<div>
<p>Fox, Gerard B., Lei Fan, Ryan A. LeVASSEUR, and Alan I. Faden. 1998. “Effect of Traumatic Brain Injury on Mouse Spatial and Nonspatial Learning in the Barnes Circular Maze.” <em>Journal of Neurotrauma</em> 15 (12): 1037–46. doi:<a href="https://doi.org/10.1089/neu.1998.15.1037">10.1089/neu.1998.15.1037</a>.</p>
</div>
<div>
<p>Frauer, Carina, Thomas Hoffmann, Sebastian Bultmann, Valentina Casa, M. Cristina Cardoso, Iris Antes, and Heinrich Leonhardt. 2011. “Recognition of 5-Hydroxymethylcytosine by the Uhrf1 SRA Domain.” <em>PLOS ONE</em> 6 (6): e21306. doi:<a href="https://doi.org/10.1371/journal.pone.0021306">10.1371/journal.pone.0021306</a>.</p>
</div>
<div>
<p>Friedli, Marc, and Didier Trono. 2015. “The Developmental Control of Transposable Elements and the Evolution of Higher Species.” <em>Annual Review of Cell and Developmental Biology</em> 31: 429–51.</p>
</div>
<div>
<p>Frost, Bess, Martin Hemberg, Jada Lewis, and Mel B Feany. 2014. “Tau Promotes Neurodegeneration Through Global Chromatin Relaxation.” <em>Nature Neuroscience</em> 17 (3): 357–66. doi:<a href="https://doi.org/10.1038/nn.3639">10.1038/nn.3639</a>.</p>
</div>
<div>
<p>Gladwyn-Ng, Ivan, Lluís Cord’on-Barris, Christian Alfano, Catherine Creppe, Therese Couderc, Giovanni Morelli, Nicolas Thelen, Michelle America, Bettina Bessi‘eres, and F’erecht’e Encha-Razavi. 2018. “Stress-Induced Unfolded Protein Response Contributes to Zika VirusAssociated Microcephaly.” <em>Nature Neuroscience</em> 21 (1): 63.</p>
</div>
<div>
<p>Glenn, C. C., R. D. Nicholls, W. P. Robinson, S. Saitoh, N. Niikawa, A. Schinzel, B. Horsthemke, and D. J. Driscoll. 1993. “Modification of 15q11-Q13 DNA Methylation Imprints in Unique Angelman and Prader-Willi Patients.” <em>Human Molecular Genetics</em> 2 (9): 1377–82.</p>
</div>
<div>
<p>Gonzalez, Inma, Roberto Munita, Eneritz Agirre, Travis A. Dittmer, Katia Gysling, Tom Misteli, and Reini F. Luco. 2015. “A lncRNA Regulates Alternative Splicing via Establishment of a Splicing-Specific Chromatin Signature.” <em>Nature Structural &amp; Molecular Biology</em> 22 (5): 370–76. doi:<a href="https://doi.org/10.1038/nsmb.3005">10.1038/nsmb.3005</a>.</p>
</div>
<div>
<p>Goodier, John L., Lili Zhang, Melissa R. Vetter, and Haig H. Kazazian. 2007. “LINE-1 Orf1 Protein Localizes in Stress Granules with Other RNA-Binding Proteins, Including Components of RNA Interference RNA-Induced Silencing Complex.” <em>Molecular and Cellular Biology</em> 27 (18): 6469–83. doi:<a href="https://doi.org/10.1128/MCB.00332-07">10.1128/MCB.00332-07</a>.</p>
</div>
<div>
<p>Gorski, Jessica A., Tiffany Talley, Mengsheng Qiu, Luis Puelles, John L. R. Rubenstein, and Kevin R. Jones. 2002. “Cortical Excitatory Neurons and Glia, but Not GABAergic Neurons, Are Produced in the Emx1-Expressing Lineage.” <em>Journal of Neuroscience</em> 22 (15): 6309–14. doi:<a href="https://doi.org/10.1523/JNEUROSCI.22-15-06309.2002">10.1523/JNEUROSCI.22-15-06309.2002</a>.</p>
</div>
<div>
<p>Gowers, Isobel R., Kevin Walters, Endre Kiss-Toth, Robert C. Read, Gordon W. Duff, and Anthony G. Wilson. 2011. “Age-Related Loss of CpG Methylation in the Tumour Necrosis Factor Promoter.” <em>Cytokine</em> 56 (3): 792–97.</p>
</div>
<div>
<p>Groner, Anna C., Sylvain Meylan, Angela Ciuffi, Nadine Zangger, Giovanna Ambrosini, Nicolas D’enervaud, Philipp Bucher, and Didier Trono. 2010. “KRAB–Zinc Finger Proteins and Kap1 Can Mediate Long-Range Transcriptional Repression Through Heterochromatin Spreading.” <em>PLoS Genetics</em> 6 (3): e1000869.</p>
</div>
<div>
<p>Gruenbaum, Yosef, Reuven Stein, Howard Cedar, and Aharon Razin. 1981. “Methylation of CpG Sequences in Eukaryotic DNA.” <em>FEBS Letters</em> 124 (1): 67–71. doi:<a href="https://doi.org/10.1016/0014-5793(81)80055-5">10.1016/0014-5793(81)80055-5</a>.</p>
</div>
<div>
<p>Gruner, Hannah, Mariela Cort’es-L’opez, Daphne A. Cooper, Matthew Bauer, and Pedro Miura. 2016. “CircRNA Accumulation in the Aging Mouse Brain.” <em>Scientific Reports</em> 6: 38907.</p>
</div>
<div>
<p>Guo, Caiwei, Hyun-Hwan Jeong, Yi-Chen Hsieh, Hans-Ulrich Klein, David A. Bennett, Philip L. De Jager, Zhandong Liu, and Joshua M. Shulman. 2018. “Tau Activates Transposable Elements in Alzheimer’s Disease.” <em>Cell Reports</em> 23 (10): 2874–80. doi:<a href="https://doi.org/10.1016/j.celrep.2018.05.004">10.1016/j.celrep.2018.05.004</a>.</p>
</div>
<div>
<p>Guo, Junjie U., Yijing Su, Chun Zhong, Guo-li Ming, and Hongjun Song. 2011. “Hydroxylation of 5-Methylcytosine by Tet1 Promotes Active DNA Demethylation in the Adult Brain.” <em>Cell</em> 145 (3): 423–34.</p>
</div>
<div>
<p>Hahn, Maria A., Runxiang Qiu, Xiwei Wu, Arthur X. Li, Heying Zhang, Jun Wang, Jonathan Jui, et al. 2013. “Dynamics of 5-Hydroxymethylcytosine and Chromatin Marks in Mammalian Neurogenesis.” <em>Cell Reports</em> 3 (2): 291–300. doi:<a href="https://doi.org/10.1016/j.celrep.2013.01.011">10.1016/j.celrep.2013.01.011</a>.</p>
</div>
<div>
<p>Hancks, Dustin C., and Haig H. Kazazian. 2012. “Active Human Retrotransposons: Variation and Disease.” <em>Current Opinion in Genetics &amp; Development</em> 22 (3): 191–203.</p>
</div>
<div>
<p>Hannum, Gregory, Justin Guinney, Ling Zhao, Li Zhang, Guy Hughes, SriniVas Sadda, Brandy Klotzle, Marina Bibikova, Jian-Bing Fan, and Yuan Gao. 2013. “Genome-Wide Methylation Profiles Reveal Quantitative Views of Human Aging Rates.” <em>Molecular Cell</em> 49 (2): 359–67.</p>
</div>
<div>
<p>Hark, Amy T., Christopher J. Schoenherr, David J. Katz, Robert S. Ingram, John M. Levorse, and Shirley M. Tilghman. 2000. “CTCF Mediates Methylation-Sensitive Enhancer-Blocking Activity at the H19/Igf2 Locus.” <em>Nature</em> 405 (6785): 486.</p>
</div>
<div>
<p>Harrison, Fiona E, Randall S Reiserer, Andrew J Tomarken, and Michael P McDonald. 2006. “Spatial and Nonspatial Escape Strategies in the Barnes Maze.” <em>Learning &amp; Memory</em> 13 (6): 809–19. doi:<a href="https://doi.org/10.1101/lm.334306">10.1101/lm.334306</a>.</p>
</div>
<div>
<p>Harvey, William. 1653. “Anatomical Exercitations Concerning the Generation of Living Creatures to Which Are Added Particular Discourses of Births and of Conceptions.” Oxford Text Archive. <a href="http://ota.ox.ac.uk/tcp/headers/A43/A43030.html" class="uri">http://ota.ox.ac.uk/tcp/headers/A43/A43030.html</a>.</p>
</div>
<div>
<p>Hashioka, Sadayuki, Andis Klegeris, Claudia Schwab, and Patrick L. McGeer. 2009. “Interferon-<span class="math inline">\(\gamma\)</span>-Dependent Cytotoxic Activation of Human Astrocytes and Astrocytoma Cells.” <em>Neurobiology of Aging</em> 30 (12): 1924–35. doi:<a href="https://doi.org/10.1016/j.neurobiolaging.2008.02.019">10.1016/j.neurobiolaging.2008.02.019</a>.</p>
</div>
<div>
<p>Hata, Kenichiro, Masaki Okano, Hong Lei, and En Li. 2002. “Dnmt3L Cooperates with the Dnmt3 Family of de Novo DNA Methyltransferases to Establish Maternal Imprints in Mice.” <em>Development</em> 129 (8): 1983–93.</p>
</div>
<div>
<p>Heyn, Holger, Ning Li, Humberto J. Ferreira, Sebastian Moran, David G. Pisano, Antonio Gomez, Javier Diez, Jose V. Sanchez-Mut, Fernando Setien, and F. Javier Carmona. 2012. “Distinct DNA Methylomes of Newborns and Centenarians.” <em>Proceedings of the National Academy of Sciences</em> 109 (26): 10522–7.</p>
</div>
<div>
<p>Hiragami-Hamada, K., K. Shinmyozu, D. Hamada, Y. Tatsu, K. Uegaki, S. Fujiwara, and J.-i. Nakayama. 2011. “N-Terminal Phosphorylation of Hp1 Promotes Its Chromatin Binding.” <em>Molecular and Cellular Biology</em> 31 (6): 1186–1200. doi:<a href="https://doi.org/10.1128/MCB.01012-10">10.1128/MCB.01012-10</a>.</p>
</div>
<div>
<p>Hiragami-Hamada, Kyoko, Szabolcs Soeroes, Miroslav Nikolov, Bryan Wilkins, Sarah Kreuz, Carol Chen, Inti A. De La Rosa-Vel’azquez, et al. 2016. “Dynamic and Flexible H3K9me3 Bridging via HP1β Dimerization Establishes a Plastic State of Condensed Chromatin.” <em>Nature Communications</em> 7 (April): 11310. doi:<a href="https://doi.org/10.1038/ncomms11310">10.1038/ncomms11310</a>.</p>
</div>
<div>
<p>Hirota, Toru, Jesse J. Lipp, Ban-Hock Toh, and Jan-Michael Peters. 2005. “Histone H3 Serine 10 Phosphorylation by Aurora B Causes Hp1 Dissociation from Heterochromatin.” <em>Nature</em> 438 (7071): 1176–80.</p>
</div>
<div>
<p>Hong, Soyon, Victoria F. Beja-Glasser, Bianca M. Nfonoyim, Arnaud Frouin, Shaomin Li, Saranya Ramakrishnan, Katherine M. Merry, Qiaoqiao Shi, Arnon Rosenthal, and Ben A. Barres. 2016. “Complement and Microglia Mediate Early Synapse Loss in Alzheimer Mouse Models.” <em>Science</em> 352 (6286): 712–16.</p>
</div>
<div>
<p>Horie, Kyoji, Ei-suke Saito, Vincent W. Keng, Ryuji Ikeda, Hiroshi Ishihara, and Junji Takeda. 2007. “Retrotransposons Influence the Mouse Transcriptome: Implication for the Divergence of Genetic Traits.” <em>Genetics</em> 176 (2): 815–27. doi:<a href="https://doi.org/10.1534/genetics.107.071647">10.1534/genetics.107.071647</a>.</p>
</div>
<div>
<p>Horvath, Steve. 2013. “DNA Methylation Age of Human Tissues and Cell Types.” <em>Genome Biology</em> 14 (10): 3156.</p>
</div>
<div>
<p>Hu, M., X.-J. Sun, Y.-L. Zhang, Y. Kuang, C.-Q. Hu, W.-L. Wu, S.-H. Shen, et al. 2010. “Histone H3 Lysine 36 Methyltransferase Hypb/Setd2 Is Required for Embryonic Vascular Remodeling.” <em>Proceedings of the National Academy of Sciences</em> 107 (7): 2956–61. doi:<a href="https://doi.org/10.1073/pnas.0915033107">10.1073/pnas.0915033107</a>.</p>
</div>
<div>
<p>Huang, Chengyang, Trent Su, Yong Xue, Chen Cheng, Fides D. Lay, Robin A. McKee, Meiyang Li, Ajay Vashisht, James Wohlschlegel, and Bennett G. Novitch. 2017. “Cbx3 Maintains Lineage Specificity During Neural Differentiation.” <em>Genes &amp; Development</em> 31 (3): 241–46.</p>
</div>
<div>
<p>Hutnick, L. K., P. Golshani, M. Namihira, Z. Xue, A. Matynia, X. W. Yang, A. J. Silva, F. E. Schweizer, and G. Fan. 2009. “DNA Hypomethylation Restricted to the Murine Forebrain Induces Cortical Degeneration and Impairs Postnatal Neuronal Maturation.” <em>Human Molecular Genetics</em> 18 (15): 2875–88. doi:<a href="https://doi.org/10.1093/hmg/ddp222">10.1093/hmg/ddp222</a>.</p>
</div>
<div>
<p>Iacono, Giovanni, Aline Dubos, Hamid M’eziane, Marco Benevento, Ehsan Habibi, Amit Mandoli, Fabrice Riet, et al. 2018. “Increased H3k9 Methylation and Impaired Expression of Protocadherins Are Associated with the Cognitive Dysfunctions of the Kleefstra Syndrome.” <em>Nucleic Acids Research</em>, March. doi:<a href="https://doi.org/10.1093/nar/gky196">10.1093/nar/gky196</a>.</p>
</div>
<div>
<p>Ianov, Lara, Matt De Both, Monica K. Chawla, Asha Rani, Andrew J. Kennedy, Ignazio Piras, Jeremy J. Day, et al. 2017. “Hippocampal Transcriptomic Profiles: Subfield Vulnerability to Age and Cognitive Impairment.” <em>Frontiers in Aging Neuroscience</em> 9 (December). doi:<a href="https://doi.org/10.3389/fnagi.2017.00383">10.3389/fnagi.2017.00383</a>.</p>
</div>
<div>
<p>Imbeault, Michaël, Pierre-Yves Helleboid, and Didier Trono. 2017. “KRAB Zinc-Finger Proteins Contribute to the Evolution of Gene Regulatory Networks.” <em>Nature</em> 543 (7646): 550. doi:<a href="https://doi.org/10.1038/nature21683">10.1038/nature21683</a>.</p>
</div>
<div>
<p>Isbel, Luke, Lexie Prokopuk, Haoyu Wu, Lucia Daxinger, Harald Oey, Alex Spurling, Adam J Lawther, Matthew W Hale, and Emma Whitelaw. 2016. “Wiz Binds Active Promoters and CTCF-Binding Sites and Is Required for Normal Behaviour in the Mouse.” <em>eLife</em> 5. doi:<a href="https://doi.org/10.7554/eLife.15082">10.7554/eLife.15082</a>.</p>
</div>
<div>
<p>Ito, Shinsuke, Li Shen, Qing Dai, Susan C. Wu, Leonard B. Collins, James A. Swenberg, Chuan He, and Yi Zhang. 2011. “Tet Proteins Can Convert 5-Methylcytosine to 5-Formylcytosine and 5-Carboxylcytosine.” <em>Science</em> 333 (6047): 1300–1303.</p>
</div>
<div>
<p>Itzhaki, Ruth F., Richard Lathe, Brian J. Balin, Melvyn J. Ball, Elaine L. Bearer, Heiko Braak, Maria J. Bullido, et al. 2016. “Microbes and Alzheimer’s Disease.” <em>Journal of Alzheimer’s Disease</em> 51 (4): 979–84. doi:<a href="https://doi.org/10.3233/JAD-160152">10.3233/JAD-160152</a>.</p>
</div>
<div>
<p>Ivanov, Alexey V., Hongzhuang Peng, Vyacheslav Yurchenko, Kyoko L. Yap, Dmitri G. Negorev, David C. Schultz, Elyse Psulkowski, et al. 2007. “PHD Domain-Mediated E3 Ligase Activity Directs Intramolecular Sumoylation of an Adjacent Bromodomain Required for Gene Silencing.” <em>Molecular Cell</em> 28 (5): 823–37. doi:<a href="https://doi.org/10.1016/j.molcel.2007.11.012">10.1016/j.molcel.2007.11.012</a>.</p>
</div>
<div>
<p>Jain, Devanshi, Cem Meydan, Julian Lange, Corentin Claeys Bouuaert, Nathalie Lailler, Christopher E. Mason, Kathryn V. Anderson, and Scott Keeney. 2017. “Rahu Is a Mutant Allele of Dnmt3c, Encoding a DNA Methyltransferase Homolog Required for Meiosis and Transposon Repression in the Mouse Male Germline.” <em>PLoS Genetics</em> 13 (8): e1006964.</p>
</div>
<div>
<p>James, T. C., J. C. Eissenberg, C. Craig, V. Dietrich, A. Hobson, and S. C. Elgin. 1989. “Distribution Patterns of Hp1, a Heterochromatin-Associated Nonhistone Chromosomal Protein of Drosophila.” <em>European Journal of Cell Biology</em> 50 (1): 170–80.</p>
</div>
<div>
<p>James, THARAPPEL C., and S. C. Elgin. 1986. “Identification of a Nonhistone Chromosomal Protein Associated with Heterochromatin in Drosophila Melanogaster and Its Gene.” <em>Molecular and Cellular Biology</em> 6 (11): 3862–72.</p>
</div>
<div>
<p>Jamieson, Gordon A., Norman J. Maitland, Gordon K. Wilcock, John Craske, and Ruth F. Itzhaki. 1991. “Latent Herpes Simplex Virus Type 1 in Normal and Alzheimer’s Disease Brains.” <em>Journal of Medical Virology</em> 33 (4): 224–27. doi:<a href="https://doi.org/10.1002/jmv.1890330403">10.1002/jmv.1890330403</a>.</p>
</div>
<div>
<p>Jamieson, Kirsty, Elizabeth T. Wiles, Kevin J. McNaught, Simone Sidoli, Neena Leggett, Yanchun Shao, Benjamin A. Garcia, and Eric U. Selker. 2015. “Loss of Hp1 Causes Depletion of H3K27me3 from Facultative Heterochromatin and Gain of H3K27me2 at Constitutive Heterochromatin.” <em>Genome Research</em>, gr. 194555.115.</p>
</div>
<div>
<p>Jang, Chuan-Wei, Yoichiro Shibata, Joshua Starmer, Della Yee, and Terry Magnuson. 2015. “Histone H3. 3 Maintains Genome Integrity During Mammalian Development.” <em>Genes &amp; Development</em> 29 (13): 1377–92.</p>
</div>
<div>
<p>Jeon, Ho-Jun, Young-Shin Kim, Joong-Gook Kim, Kyu Heo, Jung-Hoon Pyo, Masamitsu Yamaguchi, Joung-Sun Park, and Mi-Ae Yoo. 2018. “Effect of Heterochromatin Stability on Intestinal Stem Cell Aging in Drosophila.” <em>Mechanisms of Ageing and Development</em> 173 (July): 50–60. doi:<a href="https://doi.org/10.1016/j.mad.2018.04.001">10.1016/j.mad.2018.04.001</a>.</p>
</div>
<div>
<p>Jeong, Byung-Hoon, Yun-Jung Lee, Richard I. Carp, and Yong-Sun Kim. 2010. “The Prevalence of Human Endogenous Retroviruses in Cerebrospinal Fluids from Patients with Sporadic Creutzfeldt–Jakob Disease.” <em>Journal of Clinical Virology</em> 47 (2): 136–42.</p>
</div>
<div>
<p>Jiang, Yan, Yong-Hwee Eddie Loh, Prashanth Rajarajan, Teruyoshi Hirayama, Will Liao, Bibi S Kassim, Behnam Javidfar, et al. 2017. “The Methyltransferase Setdb1 Regulates a Large Neuron-Specific Topological Chromatin Domain.” <em>Nature Genetics</em> 49 (8): 1239–50. doi:<a href="https://doi.org/10.1038/ng.3906">10.1038/ng.3906</a>.</p>
</div>
<div>
<p>Jin, Qihuang, Li-Rong Yu, Lifeng Wang, Zhijing Zhang, Lawryn H. Kasper, Ji-Eun Lee, Chaochen Wang, Paul K. Brindle, Sharon YR Dent, and Kai Ge. 2011. “Distinct Roles of Gcn5/PCAF‐mediated H3K9ac and CBP/P300‐mediated H3k18/27ac in Nuclear Receptor Transactivation.” <em>The EMBO Journal</em> 30 (2): 249–62.</p>
</div>
<div>
<p>Jintaridth, Pornrutsami, and Apiwat Mutirangura. 2010. “Distinctive Patterns of Age-Dependent Hypomethylation in Interspersed Repetitive Sequences.” <em>Physiological Genomics</em> 41 (2): 194–200.</p>
</div>
<div>
<p>Jones, Peter A. 2012. “Functions of DNA Methylation: Islands, Start Sites, Gene Bodies and Beyond.” <em>Nature Reviews Genetics</em> 13 (May): 484. <a href="http://dx.doi.org/10.1038/nrg3230" class="uri">http://dx.doi.org/10.1038/nrg3230</a>.</p>
</div>
<div>
<p>Joung, Julia, Jesse M. Engreitz, Silvana Konermann, Omar O. Abudayyeh, Vanessa K. Verdine, Francois Aguet, Jonathan S. Gootenberg, et al. 2017. “Genome-Scale Activation Screen Identifies a lncRNA Locus Regulating a Gene Neighbourhood.” <em>Nature</em> 548 (7667): 343–46. doi:<a href="https://doi.org/10.1038/nature23451">10.1038/nature23451</a>.</p>
</div>
<div>
<p>Jung, Marc, and Gerd P. Pfeifer. 2015. “Aging and DNA Methylation.” <em>BMC Biology</em> 13 (1): 7.</p>
</div>
<div>
<p>Kajikawa, Masaki, and Norihiro Okada. 2002. “LINEs Mobilize SINEs in the Eel Through a Shared 3’ Sequence.” <em>Cell</em> 111 (3): 433–44.</p>
</div>
<div>
<p>Kalousi, Alkmini, Anne-Sophie Hoffbeck, Platonas N. Selemenakis, Jordan Pinder, Kienan I. Savage, Kum Kum Khanna, Laurent Brino, Graham Dellaire, Vassilis G. Gorgoulis, and Evi Soutoglou. 2015. “The Nuclear Oncogene SET Controls DNA Repair by Kap1 and Hp1 Retention to Chromatin.” <em>Cell Reports</em> 11 (1): 149–63. doi:<a href="https://doi.org/10.1016/j.celrep.2015.03.005">10.1016/j.celrep.2015.03.005</a>.</p>
</div>
<div>
<p>Kaneda, Masahiro, Masaki Okano, Kenichiro Hata, Takashi Sado, Naomi Tsujimoto, En Li, and Hiroyuki Sasaki. 2004. “Essential Role for de Novo DNA Methyltransferase Dnmt3a in Paternal and Maternal Imprinting.” <em>Nature</em> 429 (6994): 900.</p>
</div>
<div>
<p>Kaneko, Hiroki, Sami Dridi, Valeria Tarallo, Bradley D. Gelfand, Benjamin J. Fowler, Won Gil Cho, Mark E. Kleinman, et al. 2011. “DICER1 Deficit Induces Alu RNA Toxicity in Age-Related Macular Degeneration.” <em>Nature</em> 471 (7338): 325–30. doi:<a href="https://doi.org/10.1038/nature09830">10.1038/nature09830</a>.</p>
</div>
<div>
<p>Kaneko, Ryosuke, Masahumi Kawaguchi, Tomoko Toyama, Yusuke Taguchi, and Takeshi Yagi. 2009. “Expression Levels of Protocadherin-<span class="math inline">\(\alpha\)</span> Transcripts Are Decreased by Nonsense-Mediated mRNA Decay with Frameshift Mutations and by High DNA Methylation in Their Promoter Regions.” <em>Gene</em> 430 (1): 86–94. doi:<a href="https://doi.org/10.1016/j.gene.2008.10.018">10.1016/j.gene.2008.10.018</a>.</p>
</div>
<div>
<p>Karimi, Mohammad M., Preeti Goyal, Irina A. Maksakova, Misha Bilenky, Danny Leung, Jie Xin Tang, Yoichi Shinkai, et al. 2011. “DNA Methylation and Setdb1/H3K9me3 Regulate Predominantly Distinct Sets of Genes, Retroelements, and Chimeric Transcripts in mESCs.” <em>Cell Stem Cell</em> 8 (6): 676–87. doi:<a href="https://doi.org/10.1016/j.stem.2011.04.004">10.1016/j.stem.2011.04.004</a>.</p>
</div>
<div>
<p>Karlic, Heidrun, Harald Herrmann, Franz Varga, Roman Thaler, Rene Reitermaier, Silvia Spitzer, Viviane Ghanim, Katharina Blatt, Wolfgang R. Sperr, and Peter Valent. 2014. “The Role of Epigenetics in the Regulation of Apoptosis in Myelodysplastic Syndromes and Acute Myeloid Leukemia.” <em>Critical Reviews in Oncology/Hematology</em> 90 (1): 1–16.</p>
</div>
<div>
<p>Ke, Po-Yuan, and Steve S.-L. Chen. 2011. “Activation of the Unfolded Protein Response and Autophagy After Hepatitis C Virus Infection Suppresses Innate Antiviral Immunity in Vitro.” <em>The Journal of Clinical Investigation</em> 121 (1): 37–56. doi:<a href="https://doi.org/10.1172/JCI41474">10.1172/JCI41474</a>.</p>
</div>
<div>
<p>Keller, Claudia, Ricardo Adaixo, Rieka Stunnenberg, Katrina J. Woolcock, Sebastian Hiller, and Marc Bühler. 2012. “HP1 Swi6 Mediates the Recognition and Destruction of Heterochromatic RNA Transcripts.” <em>Molecular Cell</em> 47 (2): 215–27.</p>
</div>
<div>
<p>Kennedy, Brian K., Shelley L. Berger, Anne Brunet, Judith Campisi, Ana Maria Cuervo, Elissa S. Epel, Claudio Franceschi, Gordon J. Lithgow, Richard I. Morimoto, and Jeffrey E. Pessin. 2014. “Geroscience: Linking Aging to Chronic Disease.” <em>Cell</em> 159 (4): 709–13.</p>
</div>
<div>
<p>Kesslak, J. Patrick, Orhan Nalcioglu, and Carl W. Cotman. 1991. “Quantification of Magnetic Resonance Scans for Hippocampal and Parahippocampal Atrophy in Alzheimer’s Disease.” <em>Neurology</em> 41 (1): 51–51.</p>
</div>
<div>
<p>Kim, Tae-Kyung, Martin Hemberg, Jesse M. Gray, Allen M. Costa, Daniel M. Bear, Jing Wu, David A. Harmin, Mike Laptewicz, Kellie Barbara-Haley, and Scott Kuersten. 2010. “Widespread Transcription at Neuronal Activity-Regulated Enhancers.” <em>Nature</em> 465 (7295): 182.</p>
</div>
<div>
<p>Kinnaird, Adam, Steven Zhao, Kathryn E. Wellen, and Evangelos D. Michelakis. 2016. “Metabolic Control of Epigenetics in Cancer.” <em>Nature Reviews Cancer</em> 16 (11): nrc. 2016.82.</p>
</div>
<div>
<p>Koch, Stefan, Susanne Mueller, Marco Foddis, Thomas Bienert, Dominik von Elverfeldt, Felix Knab, Tracy D. Farr, et al. 2017. “Atlas Registration for Edema-Corrected MRI Lesion Volume in Mouse Stroke Models:” <em>Journal of Cerebral Blood Flow &amp; Metabolism</em>, August. doi:<a href="https://doi.org/10.1177/0271678X17726635">10.1177/0271678X17726635</a>.</p>
</div>
<div>
<p>Kochi, Yuta, Yoichiro Kamatani, Yuya Kondo, Akari Suzuki, Eiryo Kawakami, Ryosuke Hiwa, Yukihide Momozawa, Manabu Fujimoto, Masatoshi Jinnin, and Yoshiya Tanaka. 2018. “Splicing Variant of Wdfy4 Augments Mda5 Signalling and the Risk of Clinically Amyopathic Dermatomyositis.” <em>Annals of the Rheumatic Diseases</em>, annrheumdis–2017–212149.</p>
</div>
<div>
<p>Kolasinska-Zwierz, Paulina, Thomas Down, Isabel Latorre, Tao Liu, X Shirley Liu, and Julie Ahringer. 2009. “Differential Chromatin Marking of Introns and Expressed Exons by H3K36me3.” <em>Nature Genetics</em> 41 (February): 376. <a href="http://dx.doi.org/10.1038/ng.322" class="uri">http://dx.doi.org/10.1038/ng.322</a>.</p>
</div>
<div>
<p>Kourmouli, Niki, Peter Jeppesen, Shantha Mahadevhaiah, Paul Burgoyne, Rong Wu, David M. Gilbert, Silvia Bongiorni, Giorgio Prantera, Laura Fanti, and Sergio Pimpinelli. 2004. “Heterochromatin and Tri-Methylated Lysine 20 of Histone H4 in Animals.” <em>Journal of Cell Science</em> 117 (12): 2491–2501.</p>
</div>
<div>
<p>Kourmouli, Niki, Yuh-man Sun, Sjaak van der Sar, Prim B. Singh, and Jeremy P. Brown. 2005. “Epigenetic Regulation of Mammalian Pericentric Heterochromatin in Vivo by Hp1.” <em>Biochemical and Biophysical Research Communications</em> 337 (3): 901–7. doi:<a href="https://doi.org/10.1016/j.bbrc.2005.09.132">10.1016/j.bbrc.2005.09.132</a>.</p>
</div>
<div>
<p>Kristiansen, Helle, Hans Henrik Gad, Signe Eskildsen-Larsen, Philippe Despres, and Rune Hartmann. 2011. “The Oligoadenylate Synthetase Family: An Ancient Protein Family with Multiple Antiviral Activities.” <em>Journal of Interferon &amp; Cytokine Research</em> 31 (1): 41–47.</p>
</div>
<div>
<p>Krueger, Felix, and Simon R. Andrews. 2011. “Bismark: A Flexible Aligner and Methylation Caller for Bisulfite-Seq Applications.” <em>Bioinformatics</em> 27 (11): 1571–2. doi:<a href="https://doi.org/10.1093/bioinformatics/btr167">10.1093/bioinformatics/btr167</a>.</p>
</div>
<div>
<p>Krug, Lisa, Nabanita Chatterjee, Rebeca Borges-Monroy, Stephen Hearn, Wen-Wei Liao, Kathleen Morrill, Lisa Prazak, et al. 2017. “Retrotransposon Activation Contributes to Neurodegeneration in a Drosophila TDP-43 Model of ALS.” <em>PLOS Genetics</em> 13 (3): e1006635. doi:<a href="https://doi.org/10.1371/journal.pgen.1006635">10.1371/journal.pgen.1006635</a>.</p>
</div>
<div>
<p>Kucukkilic, Ezgi, Keeley Brookes, Imelda Barber, Tamar Guetta-Baranes, ARUK Consortium, Kevin Morgan, and Edward J. Hollox. 2018. “Complement Receptor 1 Gene (Cr1) Intragenic Duplication and Risk of Alzheimer’s Disease.” <em>Human Genetics</em> 137 (4): 305–14. doi:<a href="https://doi.org/10.1007/s00439-018-1883-2">10.1007/s00439-018-1883-2</a>.</p>
</div>
<div>
<p>Lachner, Monika, D’onal O’Carroll, Stephen Rea, Karl Mechtler, and Thomas Jenuwein. 2001. “Methylation of Histone H3 Lysine 9 Creates a Binding Site for Hp1 Proteins.” <em>Nature</em> 410 (6824): 116–20.</p>
</div>
<div>
<p>Lambert, Jean-Charles, Carla A. Ibrahim-Verbaas, Denise Harold, Adam C. Naj, Rebecca Sims, C’eline Bellenguez, Gyungah Jun, et al. 2013. “Meta-Analysis of 74,046 Individuals Identifies 11 New Susceptibility Loci for Alzheimer’s Disease.” <em>Nature Genetics</em> 45 (12): 1452–8. doi:<a href="https://doi.org/10.1038/ng.2802">10.1038/ng.2802</a>.</p>
</div>
<div>
<p>Larsen, Peter A., Kelsie E. Hunnicutt, Roxanne J. Larsen, Anne D. Yoder, and Ann M. Saunders. 2018. “Warning SINEs: Alu Elements, Evolution of the Human Brain, and the Spectrum of Neurological Disease.” <em>Chromosome Research</em> 26 (1-2): 93–111. doi:<a href="https://doi.org/10.1007/s10577-018-9573-4">10.1007/s10577-018-9573-4</a>.</p>
</div>
<div>
<p>Larson, Adam G., Daniel Elnatan, Madeline M. Keenen, Michael J. Trnka, Jonathan B. Johnston, Alma L. Burlingame, David A. Agard, Sy Redding, and Geeta J. Narlikar. 2017. “Liquid Droplet Formation by HP1α Suggests a Role for Phase Separation in Heterochromatin.” <em>Nature</em> 547 (7662): 236–40. doi:<a href="https://doi.org/10.1038/nature22822">10.1038/nature22822</a>.</p>
</div>
<div>
<p>Larson, Kimberly, Shian-Jang Yan, Amy Tsurumi, Jacqueline Liu, Jun Zhou, Kriti Gaur, Dongdong Guo, Thomas H. Eickbush, and Willis X. Li. 2012. “Heterochromatin Formation Promotes Longevity and Represses Ribosomal RNA Synthesis.” <em>PLoS Genet</em> 8 (1): e1002473.</p>
</div>
<div>
<p>Lavigne, Marc, Ragnhild Eskeland, Saliha Azebi, Violaine Saint-Andr’e, Suk Min Jang, Eric Batsch’e, Hua-Ying Fan, Robert E. Kingston, Axel Imhof, and Christian Muchardt. 2009. “Interaction of Hp1 and Brg1/Brm with the Globular Domain of Histone H3 Is Required for Hp1-Mediated Repression.” Edited by Asifa Akhtar. <em>PLoS Genetics</em> 5 (12): e1000769. doi:<a href="https://doi.org/10.1371/journal.pgen.1000769">10.1371/journal.pgen.1000769</a>.</p>
</div>
<div>
<p>Lee, John D., Samantha C. Levin, Emily F. Willis, Rui Li, Trent M. Woodruff, and Peter G. Noakes. 2018. “Complement Components Are Upregulated and Correlate with Disease Progression in the TDP-43q331k Mouse Model of Amyotrophic Lateral Sclerosis.” <em>Journal of Neuroinflammation</em> 15 (1): 171. doi:<a href="https://doi.org/10.1186/s12974-018-1217-2">10.1186/s12974-018-1217-2</a>.</p>
</div>
<div>
<p>Lee, Yun-Jung, Byung-Hoon Jeong, Eun-Kyung Choi, and Yong-Sun Kim. 2013. “Involvement of Endogenous Retroviruses in Prion Diseases.” <em>Pathogens</em> 2 (3): 533–43. doi:<a href="https://doi.org/10.3390/pathogens2030533">10.3390/pathogens2030533</a>.</p>
</div>
<div>
<p>Lemaire, Peter A., Eric Anderson, Jeffrey Lary, and James L. Cole. 2008. “Mechanism of PKR Activation by dsRNA.” <em>Journal of Molecular Biology</em> 381 (2): 351–60. doi:<a href="https://doi.org/10.1016/j.jmb.2008.05.056">10.1016/j.jmb.2008.05.056</a>.</p>
</div>
<div>
<p>Leonard, Phoebe H., Adrienne Grzenda, Angela Mathison, Dean E. Morbeck, Jolene R. Fredrickson, Thiago M. de Assuncao, Trace Christensen, et al. 2015. “The Aurora A-HP1γ Pathway Regulates Gene Expression and Mitosis in Cells from the Sperm Lineage.” <em>BMC Developmental Biology</em> 15 (1): 23. doi:<a href="https://doi.org/10.1186/s12861-015-0073-x">10.1186/s12861-015-0073-x</a>.</p>
</div>
<div>
<p>LeRoy, Gary, John T. Weston, Barry M. Zee, Nicolas L. Young, Mariana D. Plazas-Mayorca, and Benjamin A. Garcia. 2009. “Heterochromatin Protein 1 Is Extensively Decorated with Histone Code-Like Post-Translational Modifications.” <em>Molecular &amp; Cellular Proteomics</em> 8 (11): 2432–42. doi:<a href="https://doi.org/10.1074/mcp.M900160-MCP200">10.1074/mcp.M900160-MCP200</a>.</p>
</div>
<div>
<p>Leung, Danny, Tingting Du, Ulrich Wagner, Wei Xie, Ah Young Lee, Preeti Goyal, Yujing Li, et al. 2014. “Regulation of DNA Methylation Turnover at LTR Retrotransposons and Imprinted Loci by the Histone Methyltransferase Setdb1.” <em>Proceedings of the National Academy of Sciences</em> 111 (18): 6690–5. doi:<a href="https://doi.org/10.1073/pnas.1322273111">10.1073/pnas.1322273111</a>.</p>
</div>
<div>
<p>Li, Andrew, Carolina Ceballos-Diaz, Nadia DiNunno, Yona Levites, Pedro E. Cruz, Jada Lewis, Todd E. Golde, and Paramita Chakrabarty. 2015. “IFN-<span class="math inline">\(\gamma\)</span> Promotes <span class="math inline">\(\tau\)</span> Phosphorylation Without Affecting Mature Tangles.” <em>The FASEB Journal</em> 29 (10): 4384–98. doi:<a href="https://doi.org/10.1096/fj.15-275834">10.1096/fj.15-275834</a>.</p>
</div>
<div>
<p>Li, Wanhe, Ying Jin, Lisa Prazak, Molly Hammell, and Josh Dubnau. 2012. “Transposable Elements in TDP-43-Mediated Neurodegenerative Disorders.” <em>PloS One</em> 7 (9): e44099. doi:<a href="https://doi.org/10.1371/journal.pone.0044099">10.1371/journal.pone.0044099</a>.</p>
</div>
<div>
<p>Li, Wanhe, Lisa Prazak, Nabanita Chatterjee, Servan Grüninger, Lisa Krug, Delphine Theodorou, and Josh Dubnau. 2013. “Activation of Transposable Elements During Aging and Neuronal Decline in Drosophila.” <em>Nature Neuroscience</em> 16 (5): 529–31. doi:<a href="https://doi.org/10.1038/nn.3368">10.1038/nn.3368</a>.</p>
</div>
<div>
<p>Li, Wenxue, Myoung-Hwa Lee, Lisa Henderson, Richa Tyagi, Muzna Bachani, Joseph Steiner, Emilie Campanac, et al. 2015. “Human Endogenous Retrovirus-K Contributes to Motor Neuron Disease.” <em>Science Translational Medicine</em> 7 (307): 307ra153–307ra153. <a href="http://stm.sciencemag.org/content/7/307/307ra153.short" class="uri">http://stm.sciencemag.org/content/7/307/307ra153.short</a>.</p>
</div>
<div>
<p>Li, Xiaofan, Eric M. Burton, and Sumita Bhaduri-McIntosh. 2017. “Chloroquine Triggers Epstein-Barr Virus Replication Through Phosphorylation of Kap1/Trim28 in Burkitt Lymphoma Cells.” <em>PLOS Pathogens</em> 13 (3): e1006249. doi:<a href="https://doi.org/10.1371/journal.ppat.1006249">10.1371/journal.ppat.1006249</a>.</p>
</div>
<div>
<p>Li, Yize, Shuvojit Banerjee, Yuyan Wang, Stephen A. Goldstein, Beihua Dong, Christina Gaughan, Robert H. Silverman, and Susan R. Weiss. 2016. “Activation of RNase L Is Dependent on Oas3 Expression During Infection with Diverse Human Viruses.” <em>Proceedings of the National Academy of Sciences</em> 113 (8): 2241–6. doi:<a href="https://doi.org/10.1073/pnas.1519657113">10.1073/pnas.1519657113</a>.</p>
</div>
<div>
<p>Lian, Hong, Li Yang, Allysa Cole, Lu Sun, Angie C. -A. Chiang, Stephanie W. Fowler, David J. Shim, et al. 2015. “NF<span class="math inline">\(\kappa\)</span>B-Activated Astroglial Release of Complement C3 Compromises Neuronal Morphology and Function Associated with Alzheimer’s Disease.” <em>Neuron</em> 85 (1): 101–15. doi:<a href="https://doi.org/10.1016/j.neuron.2014.11.018">10.1016/j.neuron.2014.11.018</a>.</p>
</div>
<div>
<p>Liddelow, Shane A., Kevin A. Guttenplan, Laura E. Clarke, Frederick C. Bennett, Christopher J. Bohlen, Lucas Schirmer, Mariko L. Bennett, et al. 2017. “Neurotoxic Reactive Astrocytes Are Induced by Activated Microglia.” <em>Nature</em> 541 (7638): 481–87. doi:<a href="https://doi.org/10.1038/nature21029">10.1038/nature21029</a>.</p>
</div>
<div>
<p>Lin, Athena W., Marta Barradas, James C. Stone, Linda van Aelst, Manuel Serrano, and Scott W. Lowe. 1998. “Premature Senescence Involving P53 and P16 Is Activated in Response to Constitutive MEK/MAPK Mitogenic Signaling.” <em>Genes &amp; Development</em> 12 (19): 3008–19.</p>
</div>
<div>
<p>Lin, Longnian, Guifen Chen, Hui Kuang, Dong Wang, and Joe Z. Tsien. 2007. “Neural Encoding of the Concept of Nest in the Mouse Brain.” <em>Proceedings of the National Academy of Sciences</em> 104 (14): 6066–71. doi:<a href="https://doi.org/10.1073/pnas.0701106104">10.1073/pnas.0701106104</a>.</p>
</div>
<div>
<p>Lister, Ryan, Mattia Pelizzola, Robert H. Dowen, R. David Hawkins, Gary Hon, Julian Tonti-Filippini, Joseph R. Nery, Leonard Lee, Zhen Ye, and Que-Minh Ngo. 2009. “Human DNA Methylomes at Base Resolution Show Widespread Epigenomic Differences.” <em>Nature</em> 462 (7271): 315.</p>
</div>
<div>
<p>Liu, Guang-Hui, Basam Z. Barkho, Sergio Ruiz, Dinh Diep, Jing Qu, Sheng-Lian Yang, Athanasia D. Panopoulos, Keiichiro Suzuki, Leo Kurian, and Christopher Walsh. 2011. “Recapitulation of Premature Ageing with iPSCs from Hutchinson–Gilford Progeria Syndrome.” <em>Nature</em> 472 (7342): 221.</p>
</div>
<div>
<p>Liu, Liang, Thomas van Groen, Inga Kadish, Yuanyuan Li, Deli Wang, Smitha R. James, Adam R. Karpf, and Trygve O. Tollefsbol. 2011. “Insufficient DNA Methylation Affects Healthy Aging and Promotes Age-Related Health Problems.” <em>Clinical Epigenetics</em> 2 (2): 349. doi:<a href="https://doi.org/10.1007/s13148-011-0042-6">10.1007/s13148-011-0042-6</a>.</p>
</div>
<div>
<p>Liu, Xing, Zhenwei Song, Yuda Huo, Jiahai Zhang, Tongge Zhu, Jianyu Wang, Xuannv Zhao, et al. 2014. “Chromatin Protein Hp1 Interacts with the Mitotic Regulator Borealin Protein and Specifies the Centromere Localization of the Chromosomal Passenger Complex.” <em>The Journal of Biological Chemistry</em> 289 (30): 20638–49. doi:<a href="https://doi.org/10.1074/jbc.M114.572842">10.1074/jbc.M114.572842</a>.</p>
</div>
<div>
<p>Lomberk, Gwen, Debora Bensi, Martín E. Fernandez-Zapico, and Raul Urrutia. 2006. “Evidence for the Existence of an Hp1-Mediated Subcode Within the Histone Code.” <em>Nature Cell Biology</em> 8 (4): 407–15. doi:<a href="https://doi.org/10.1038/ncb1383">10.1038/ncb1383</a>.</p>
</div>
<div>
<p>Loo, Yueh-Ming, Jamie Fornek, Nanette Crochet, Gagan Bajwa, Olivia Perwitasari, Luis Martinez-Sobrido, Shizuo Akira, et al. 2008. “Distinct RIG-I and Mda5 Signaling by RNA Viruses in Innate Immunity.” <em>Journal of Virology</em> 82 (1): 335–45. doi:<a href="https://doi.org/10.1128/JVI.01080-07">10.1128/JVI.01080-07</a>.</p>
</div>
<div>
<p>Lu, Xu, Matthew D. Simon, Jayanth V. Chodaparambil, Jeffrey C. Hansen, Kevan M. Shokat, and Karolin Luger. 2008. “The Effect of H3k79 Dimethylation and H4k20 Trimethylation on Nucleosome and Chromatin Structure.” <em>Nature Structural &amp; Molecular Biology</em> 15 (10): 1122–4. doi:<a href="https://doi.org/10.1038/nsmb.1489">10.1038/nsmb.1489</a>.</p>
</div>
<div>
<p>Lubit, Beverly W., Pham Tuan Duc, Orlando J. Miller, and Bernard F. Erlanger. 1976. “Localization of 5-Methylcytosine in Human Metaphase Chromosomes by Immunoelectron Microscopy.” <em>Cell</em> 9 (4): 503–9.</p>
</div>
<div>
<p>Luger, Karolin, Armin W. Mäder, Robin K. Richmond, David F. Sargent, and Timothy J. Richmond. 1997. “Crystal Structure of the Nucleosome Core Particle at 2.8 Å Resolution.” <em>Nature</em> 389 (6648): 251.</p>
</div>
<div>
<p>Luijsterburg, Martijn S., Christoffel Dinant, Hannes Lans, Jan Stap, Elzbieta Wiernasz, Saskia Lagerwerf, Daniël O. Warmerdam, et al. 2009. “Heterochromatin Protein 1 Is Recruited to Various Types of DNA Damage.” <em>The Journal of Cell Biology</em> 185 (4): 577–86. doi:<a href="https://doi.org/10.1083/jcb.200810035">10.1083/jcb.200810035</a>.</p>
</div>
<div>
<p>Macfarlan, Todd S., Wesley D. Gifford, Shawn Driscoll, Karen Lettieri, Helen M. Rowe, Dario Bonanomi, Amy Firth, Oded Singer, Didier Trono, and Samuel L. Pfaff. 2012. “Embryonic Stem Cell Potency Fluctuates with Endogenous Retrovirus Activity.” <em>Nature</em> 487 (June): 57. <a href="http://dx.doi.org/10.1038/nature11244" class="uri">http://dx.doi.org/10.1038/nature11244</a>.</p>
</div>
<div>
<p>Machado, Matias R., Pablo D. Dans, and Sergio Pantano. 2010. “Isoform-Specific Determinants in the Hp1 Binding to Histone 3: Insights from Molecular Simulations.” <em>Amino Acids</em> 38 (5): 1571–81. doi:<a href="https://doi.org/10.1007/s00726-009-0371-3">10.1007/s00726-009-0371-3</a>.</p>
</div>
<div>
<p>Madabhushi, Ram, Fan Gao, Andreas R. Pfenning, Ling Pan, Satoko Yamakawa, Jinsoo Seo, Richard Rueda, et al. 2015. “Activity-Induced DNA Breaks Govern the Expression of Neuronal Early-Response Genes.” <em>Cell</em> 161 (7): 1592–1605. doi:<a href="https://doi.org/10.1016/j.cell.2015.05.032">10.1016/j.cell.2015.05.032</a>.</p>
</div>
<div>
<p>Madabhushi, Ram, Ling Pan, and Li-Huei Tsai. 2014. “DNA Damage and Its Links to Neurodegeneration.” <em>Neuron</em> 83 (2): 266–82. doi:<a href="https://doi.org/10.1016/j.neuron.2014.06.034">10.1016/j.neuron.2014.06.034</a>.</p>
</div>
<div>
<p>Maegawa, Shinji, George Hinkal, Hyun Soo Kim, Lanlan Shen, Li Zhang, Jiexin Zhang, Nianxiang Zhang, Shoudan Liang, Lawrence A. Donehower, and Jean-Pierre J. Issa. 2010. “Widespread and Tissue Specific Age-Related DNA Methylation Changes in Mice.” <em>Genome Research</em> 20 (3): 332–40. doi:<a href="https://doi.org/10.1101/gr.096826.109">10.1101/gr.096826.109</a>.</p>
</div>
<div>
<p>Maison, Christ‘ele, Delphine Bailly, Antoine HFM Peters, Jean-Pierre Quivy, Dani‘ele Roche, Angela Taddei, Monika Lachner, Thomas Jenuwein, and Genevi‘eve Almouzni. 2002. “Higher-Order Structure in Pericentric Heterochromatin Involves a Distinct Pattern of Histone Modification and an RNA Component.” <em>Nature Genetics</em> 30 (3): 329.</p>
</div>
<div>
<p>Maison, Christ‘ele, Delphine Bailly, Dani‘ele Roche, Rocio Montes de Oca, Aline V Probst, Isabelle Vassias, Florent Dingli, et al. 2011. “SUMOylation Promotes de Novo Targeting of HP1α to Pericentric Heterochromatin.” <em>Nature Genetics</em> 43 (3): 220–27. doi:<a href="https://doi.org/10.1038/ng.765">10.1038/ng.765</a>.</p>
</div>
<div>
<p>Maiti, Atanu, and Alexander C. Drohat. 2011. “Thymine DNA Glycosylase Can Rapidly Excise 5-Formylcytosine and 5-Carboxylcytosine Potential Implications for Active Demethylation of CpG Sites.” <em>Journal of Biological Chemistry</em> 286 (41): 35334–8.</p>
</div>
<div>
<p>Maksakova, Irina A., Preeti Goyal, Jörn Bullwinkel, Jeremy P. Brown, Misha Bilenky, Dixie L. Mager, Prim B. Singh, and Matthew C. Lorincz. 2011. “H3K9me3-Binding Proteins Are Dispensable for Setdb1/H3K9me3-Dependent Retroviral Silencing.” <em>Epigenetics Chromatin</em> 4 (1): 12. <a href="http://www.biomedcentral.com/content/pdf/1756-8935-4-12.pdf" class="uri">http://www.biomedcentral.com/content/pdf/1756-8935-4-12.pdf</a>.</p>
</div>
<div>
<p>Maksakova, Irina A., Mark T. Romanish, Liane Gagnier, Catherine A. Dunn, Louie N. van de Lagemaat, and Dixie L. Mager. 2006. “Retroviral Elements and Their Hosts: Insertional Mutagenesis in the Mouse Germ Line.” <em>PLoS Genetics</em> 2 (1): e2. doi:<a href="https://doi.org/10.1371/journal.pgen.0020002">10.1371/journal.pgen.0020002</a>.</p>
</div>
<div>
<p>Maksakova, IrinaA, PeterJ Thompson, Preeti Goyal, StevenJM Jones, PrimB Singh, MohammadM Karimi, and MatthewC Lorincz. 2013. “Distinct Roles of Kap1, Hp1 and G9a/GLP in Silencing of the Two-Cell-Specific Retrotransposon MERVL in Mouse ES Cells.” <em>Epigenetics &amp; Chromatin</em> 6 (1): 1–16. doi:<a href="https://doi.org/10.1186/1756-8935-6-15">10.1186/1756-8935-6-15</a>.</p>
</div>
<div>
<p>Mameli, Giuseppe, Giordano Madeddu, Alessandra Mei, Elena Uleri, Luciana Poddighe, Lucia G. Delogu, Ivana Maida, et al. 2013. “Activation of MSRV-Type Endogenous Retroviruses During Infectious Mononucleosis and Epstein-Barr Virus Latency: The Missing Link with Multiple Sclerosis?” <em>PLOS ONE</em> 8 (11): e78474. doi:<a href="https://doi.org/10.1371/journal.pone.0078474">10.1371/journal.pone.0078474</a>.</p>
</div>
<div>
<p>Mameli, Giuseppe, Luciana Poddighe, Alessandra Mei, Elena Uleri, Stefano Sotgiu, Caterina Serra, Roberto Manetti, and Antonina Dolei. 2012. “Expression and Activation by Epstein Barr Virus of Human Endogenous Retroviruses-W in Blood Cells and Astrocytes: Inference for Multiple Sclerosis.” <em>PloS One</em> 7 (9): e44991. doi:<a href="https://doi.org/10.1371/journal.pone.0044991">10.1371/journal.pone.0044991</a>.</p>
</div>
<div>
<p>Mancini, Debora N., Shiva M. Singh, Trevor K. Archer, and David I. Rodenhiser. 1999. “Site-Specific DNA Methylation in the Neurofibromatosis (Nf1) Promoter Interferes with Binding of CREB and Sp1 Transcription Factors.” <em>Oncogene</em> 18 (28): 4108.</p>
</div>
<div>
<p>Mancini-DiNardo, Debora, Scott JS Steele, John M. Levorse, Robert S. Ingram, and Shirley M. Tilghman. 2006. “Elongation of the Kcnq1ot1 Transcript Is Required for Genomic Imprinting of Neighboring Genes.” <em>Genes &amp; Development</em> 20 (10): 1268–82.</p>
</div>
<div>
<p>Mansuroglu, Zeyni, Houda Benhelli-Mokrani, Vasco Marcato, Audrey Sultan, Marie Violet, Alban Chauderlier, Lucie Delattre, et al. 2016. “Loss of Tau Protein Affects the Structure, Transcription and Repair of Neuronal Pericentromeric Heterochromatin.” <em>Scientific Reports</em> 6 (September): 33047. doi:<a href="https://doi.org/10.1038/srep33047">10.1038/srep33047</a>.</p>
</div>
<div>
<p>Martinon, Fabio, Xi Chen, Ann-Hwee Lee, and Laurie H Glimcher. 2010. “TLR Activation of the Transcription Factor Xbp1 Regulates Innate Immune Responses in Macrophages.” <em>Nature Immunology</em> 11 (March): 411.</p>
</div>
<div>
<p>Martos, Suzanne N, Teng Li, Ramon Bossardi Ramos, Dan Lou, Hongzheng Dai, Jin-Chong Xu, Ganglong Gao, et al. 2017. “Two Approaches Reveal a New Paradigm of ‘Switchable or Genetics-Influenced Allele-Specific DNA Methylation’ with Potential in Human Disease.” <em>Cell Discovery</em> 3 (November): 17038. doi:<a href="https://doi.org/10.1038/celldisc.2017.38">10.1038/celldisc.2017.38</a>.</p>
</div>
<div>
<p>Mastroeni, Diego, Jennifer Nolz, Shobana Sekar, Elaine Delvaux, Geidy Serrano, Lori Cuyugan, Winnie S. Liang, Thomas G. Beach, Joseph Rogers, and Paul D. Coleman. 2018. “Laser-Captured Microglia in the Alzheimer’s and Parkinson’s Brain Reveal Unique Regional Expression Profiles and Suggest a Potential Role for Hepatitis B in the Alzheimer’s Brain.” <em>Neurobiology of Aging</em> 63 (March): 12–21. doi:<a href="https://doi.org/10.1016/j.neurobiolaging.2017.10.019">10.1016/j.neurobiolaging.2017.10.019</a>.</p>
</div>
<div>
<p>Matoba, Shogo, Yuting Liu, Falong Lu, Kumiko A. Iwabuchi, Li Shen, Azusa Inoue, and Yi Zhang. 2014. “Embryonic Development Following Somatic Cell Nuclear Transfer Impeded by Persisting Histone Methylation.” <em>Cell</em> 159 (4): 884–95. doi:<a href="https://doi.org/10.1016/j.cell.2014.09.055">10.1016/j.cell.2014.09.055</a>.</p>
</div>
<div>
<p>McConnell, Michael J., Michael R. Lindberg, Kristen J. Brennand, Julia C. Piper, Thierry Voet, Chris Cowing-Zitron, Svetlana Shumilina, Roger S. Lasken, Joris R. Vermeesch, and Ira M. Hall. 2013. “Mosaic Copy Number Variation in Human Neurons.” <em>Science</em> 342 (6158): 632–37.</p>
</div>
<div>
<p>McDevitt, Shane, Timur Rusanov, Tatiana Kent, Gurushankar Chandramouly, and Richard T. Pomerantz. 2018. “How RNA Transcripts Coordinate DNA Recombination and Repair.” <em>Nature Communications</em> 9 (1): 1091. doi:<a href="https://doi.org/10.1038/s41467-018-03483-7">10.1038/s41467-018-03483-7</a>.</p>
</div>
<div>
<p>Medigeshi, Guruprasad R., Alissa M. Lancaster, Alec J. Hirsch, Thomas Briese, W. Ian Lipkin, Victor DeFilippis, Klaus Früh, Peter W. Mason, Janko Nikolich-Zugich, and Jay A. Nelson. 2007. “West Nile Virus Infection Activates the Unfolded Protein Response, Leading to CHOP Induction and Apoptosis.” <em>Journal of Virology</em> 81 (20): 10849–60.</p>
</div>
<div>
<p>Meier, Ingo, Laetitia Fellini, Mira Jakovcevski, Melitta Schachner, and Fabio Morellini. 2010. “Expression of the snoRNA Host Gene Gas5 in the Hippocampus Is Upregulated by Age and Psychogenic Stress and Correlates with Reduced Novelty‐induced Behavior in C57bl/6 Mice.” <em>Hippocampus</em> 20 (9): 1027–36.</p>
</div>
<div>
<p>Mell’en, Marian, Pinar Ayata, Scott Dewell, Skirmantas Kriaucionis, and Nathaniel Heintz. 2012. “MeCP2 Binds to 5hmC Enriched Within Active Genes and Accessible Chromatin in the Nervous System.” <em>Cell</em> 151 (7): 1417–30. doi:<a href="https://doi.org/10.1016/j.cell.2012.11.022">10.1016/j.cell.2012.11.022</a>.</p>
</div>
<div>
<p>Memczak, Sebastian, Marvin Jens, Antigoni Elefsinioti, Francesca Torti, Janna Krueger, Agnieszka Rybak, Luisa Maier, et al. 2013. “Circular RNAs Are a Large Class of Animal RNAs with Regulatory Potency.” <em>Nature</em> 495 (7441): 333–38. doi:<a href="https://doi.org/10.1038/nature11928">10.1038/nature11928</a>.</p>
</div>
<div>
<p>Miller, Justine D., Yosif M. Ganat, Sarah Kishinevsky, Robert L. Bowman, Becky Liu, Edmund Y. Tu, Pankaj K. Mandal, Elsa Vera, Jae-won Shim, and Sonja Kriks. 2013. “Human iPSC-Based Modeling of Late-Onset Disease via Progerin-Induced Aging.” <em>Cell Stem Cell</em> 13 (6): 691–705.</p>
</div>
<div>
<p>Minc, Elsa, Yves Allory, Howard J. Worman, Jean-Claude Courvalin, and Brigitte Buendia. 1999. “Localization and Phosphorylation of Hp1 Proteins During the Cell Cycle in Mammalian Cells.” <em>Chromosoma</em> 108 (4): 220–34.</p>
</div>
<div>
<p>Mitchell, T. J., M. Naughton, P. Norsworthy, K. A. Davies, M. J. Walport, and B. J. Morley. 1996. “IFN-Gamma up-Regulates Expression of the Complement Components C3 and C4 by Stabilization of mRNA.” <em>Journal of Immunology (Baltimore, Md.: 1950)</em> 156 (11): 4429–34.</p>
</div>
<div>
<p>Molumby, Michael J., Rachel M. Anderson, Dillan J. Newbold, Norah K. Koblesky, Andrew M. Garrett, Dietmar Schreiner, Jason J. Radley, and Joshua A. Weiner. 2017. “Γ-Protocadherins Interact with Neuroligin-1 and Negatively Regulate Dendritic Spine Morphogenesis.” <em>Cell Reports</em> 18 (11): 2702–14. doi:<a href="https://doi.org/10.1016/j.celrep.2017.02.060">10.1016/j.celrep.2017.02.060</a>.</p>
</div>
<div>
<p>Mortusewicz, Oliver, Lothar Schermelleh, Joachim Walter, M. Cristina Cardoso, and Heinrich Leonhardt. 2005. “Recruitment of DNA Methyltransferase I to DNA Repair Sites.” <em>Proceedings of the National Academy of Sciences</em> 102 (25): 8905–9.</p>
</div>
<div>
<p>Mosch, Kerstin, Henriette Franz, Szabolcs Soeroes, Prim B. Singh, and Wolfgang Fischle. 2011. “HP1 Recruits Activity-Dependent Neuroprotective Protein to H3K9me3 Marked Pericentromeric Heterochromatin for Silencing of Major Satellite Repeats.” Edited by Michael J. Pazin. <em>PLoS ONE</em> 6 (1): e15894. doi:<a href="https://doi.org/10.1371/journal.pone.0015894">10.1371/journal.pone.0015894</a>.</p>
</div>
<div>
<p>Moskalev, Alexey, Alexander Aliper, Zeljka Smit-McBride, Anton Buzdin, and Alex Zhavoronkov. 2014. “Genetics and Epigenetics of Aging and Longevity.” <em>Cell Cycle</em> 13 (7): 1063–77.</p>
</div>
<div>
<p>Mozzetta, Chiara, Julien Pontis, Lauriane Fritsch, Philippe Robin, Manuela Portoso, Caroline Proux, Raphaël Margueron, and Slimane Ait-Si-Ali. 2014. “The Histone H3 Lysine 9 Methyltransferases G9a and GLP Regulate Polycomb Repressive Complex 2-Mediated Gene Silencing.” <em>Molecular Cell</em> 53 (2): 277–89.</p>
</div>
<div>
<p>Muotri, Alysson R., Chunmei Zhao, Maria C.N. Marchetto, and Fred H. Gage. 2009. “Environmental Influence on L1 Retrotransposons in the Adult Hippocampus.” <em>Hippocampus</em> 19 (10): 1002–7. doi:<a href="https://doi.org/10.1002/hipo.20564">10.1002/hipo.20564</a>.</p>
</div>
<div>
<p>M’etivier, Raphaël, Rozenn Gallais, Christophe Tiffoche, Christine Le P’eron, Renata Z. Jurkowska, Richard P. Carmouche, David Ibberson, Peter Barath, Florence Demay, and George Reid. 2008. “Cyclical DNA Methylation of a Transcriptionally Active Promoter.” <em>Nature</em> 452 (7183): 45.</p>
</div>
<div>
<p>Naito, Atsuhiko T., Tomokazu Sumida, Seitaro Nomura, Mei-Lan Liu, Tomoaki Higo, Akito Nakagawa, Katsuki Okada, et al. 2012. “Complement C1q Activates Canonical Wnt Signaling and Promotes Aging-Related Phenotypes.” <em>Cell</em> 149 (6): 1298–1313. doi:<a href="https://doi.org/10.1016/j.cell.2012.03.047">10.1016/j.cell.2012.03.047</a>.</p>
</div>
<div>
<p>Narita, Masashi, Sabrina Nuñez, Edith Heard, Masako Narita, Athena W. Lin, Stephen A. Hearn, David L. Spector, Gregory J. Hannon, and Scott W. Lowe. 2003. “Rb-Mediated Heterochromatin Formation and Silencing of E2f Target Genes During Cellular Senescence.” <em>Cell</em> 113 (6): 703–16.</p>
</div>
<div>
<p>Nellåker, Christoffer, Yuanrong Yao, Lorraine Jones-Brando, François Mallet, Robert H. Yolken, and H<span class="math inline">\(\backslash\)</span>a akan Karlsson. 2006. “Transactivation of Elements in the Human Endogenous Retrovirus W Family by Viral Infection.” <em>Retrovirology</em> 3 (1): 1.</p>
</div>
<div>
<p>Nelson, David M., Farah Jaber-Hijazi, John J. Cole, Neil A. Robertson, Jeffrey S. Pawlikowski, Kevin T. Norris, Steven W. Criscione, et al. 2016. “Mapping H4K20me3 onto the Chromatin Landscape of Senescent Cells Indicates a Function in Control of Cell Senescence and Tumor Suppression Through Preservation of Genetic and Epigenetic Stability.” <em>Genome Biology</em> 17 (July). doi:<a href="https://doi.org/10.1186/s13059-016-1017-x">10.1186/s13059-016-1017-x</a>.</p>
</div>
<div>
<p>Nergadze, Solomon G., Benjamin O. Farnung, Harry Wischnewski, Lela Khoriauli, Valerio Vitelli, Raghav Chawla, Elena Giulotto, and Claus M. Azzalin. 2009. “CpG-Island Promoters Drive Transcription of Human Telomeres.” <em>Rna</em>.</p>
</div>
<div>
<p>Nestor, Colm E., Raffaele Ottaviano, James Reddington, Duncan Sproul, Diana Reinhardt, Donncha Dunican, Elad Katz, J. Michael Dixon, David J. Harrison, and Richard R. Meehan. 2012. “Tissue Type Is a Major Modifier of the 5-Hydroxymethylcytosine Content of Human Genes.” <em>Genome Research</em> 22 (3): 467–77.</p>
</div>
<div>
<p>Newman, Andrew G, Paraskevi Bessa, Victor Tarabykin, and Prim B Singh. 2017. “Activity‐DEPendent Transposition.” <em>EMBO Reports</em>, February, e201643797. doi:<a href="https://doi.org/10.15252/embr.201643797">10.15252/embr.201643797</a>.</p>
</div>
<div>
<p>Newman, Andrew G., Paraskevi Bessa, V. Tarabykin, and Prim B. Singh. 2016. “Activity DEPendent Transposition (ADEPT) and the Aging Brain.” <em>Opera Medica et Physiologica</em>, no. 3-4. <a href="http://cyberleninka.ru/article/n/activity-dependent-transposition-adept-and-the-aging-brain" class="uri">http://cyberleninka.ru/article/n/activity-dependent-transposition-adept-and-the-aging-brain</a>.</p>
</div>
<div>
<p>Ni, Zhuoyu, Atsushi Ebata, Elham Alipanahiramandi, and Siu Sylvia Lee. 2012. “Two SET Domain Containing Genes Link Epigenetic Changes and Aging in Caenorhabditis Elegans.” <em>Aging Cell</em> 11 (2): 315–25.</p>
</div>
<div>
<p>Nielsen, Peter R., Daniel Nietlispach, Helen R. Mott, Juliana Callaghan, Andrew Bannister, Tony Kouzarides, Alexey G. Murzin, Natalia V. Murzina, and Ernest D. Laue. 2002. “Structure of the Hp1 Chromodomain Bound to Histone H3 Methylated at Lysine 9.” <em>Nature</em> 416 (6876): 103.</p>
</div>
<div>
<p>Nimgaonkar, V. L., K. M. Prasad, K. V. Chowdari, E. G. Severance, and R. H. Yolken. 2017. “The Complement System: A Gateway to GeneEnvironment Interactions in Schizophrenia Pathogenesis.” <em>Molecular Psychiatry</em> 22 (11): 1554–61. doi:<a href="https://doi.org/10.1038/mp.2017.151">10.1038/mp.2017.151</a>.</p>
</div>
<div>
<p>Nishibuchi, Gohei, Shinichi Machida, Akihisa Osakabe, Hiromu Murakoshi, Kyoko Hiragami-Hamada, Reiko Nakagawa, Wolfgang Fischle, et al. 2014. “N-Terminal Phosphorylation of HP1α Increases Its Nucleosome-Binding Specificity.” <em>Nucleic Acids Research</em> 42 (20): 12498–12511. doi:<a href="https://doi.org/10.1093/nar/gku995">10.1093/nar/gku995</a>.</p>
</div>
<div>
<p>Nishimoto, Yoshinori, Shinichi Nakagawa, Tetsuro Hirose, Hirotaka James Okano, Masaki Takao, Shinsuke Shibata, Satoshi Suyama, Ken-ichiro Kuwako, Takao Imai, and Shigeo Murayama. 2013. “The Long Non-Coding RNA Nuclear-Enriched Abundant Transcript 1_2 Induces Paraspeckle Formation in the Motor Neuron During the Early Phase of Amyotrophic Lateral Sclerosis.” <em>Molecular Brain</em> 6 (1): 31.</p>
</div>
<div>
<p>Noguchi, Hirofumi, Naoya Murao, Ayaka Kimura, Taito Matsuda, Masakazu Namihira, and Kinichi Nakashima. 2016. “DNA Methyltransferase 1 Is Indispensable for Development of the Hippocampal Dentate Gyrus.” <em>The Journal of Neuroscience</em> 36 (22): 6050–68. doi:<a href="https://doi.org/10.1523/JNEUROSCI.0512-16.2016">10.1523/JNEUROSCI.0512-16.2016</a>.</p>
</div>
<div>
<p>Noh, Kyung-Min, Ian Maze, Dan Zhao, Bin Xiang, Wendy Wenderski, Peter W. Lewis, Li Shen, Haitao Li, and C. David Allis. 2014. “ATRX Tolerates Activity-Dependent Histone H3 Methyl/Phos Switching to Maintain Repetitive Element Silencing in Neurons.” <em>Proceedings of the National Academy of Sciences</em>, 201411258. <a href="http://www.pnas.org/content/early/2014/12/22/1411258112.short" class="uri">http://www.pnas.org/content/early/2014/12/22/1411258112.short</a>.</p>
</div>
<div>
<p>Nonaka, Takashi, Masami Masuda-Suzukake, Tetsuaki Arai, Yoko Hasegawa, Hiroyasu Akatsu, Tomokazu Obi, Mari Yoshida, et al. 2013. “Prion-Like Properties of Pathological TDP-43 Aggregates from Diseased Brains.” <em>Cell Reports</em> 4 (1): 124–34. doi:<a href="https://doi.org/10.1016/j.celrep.2013.06.007">10.1016/j.celrep.2013.06.007</a>.</p>
</div>
<div>
<p>Norden, Diana M., and Jonathan P. Godbout. 2013. “Microglia of the Aged Brain: Primed to Be Activated and Resistant to Regulation.” <em>Neuropathology and Applied Neurobiology</em> 39 (1): 19–34.</p>
</div>
<div>
<p>Oberdoerffer, Philipp, and David A. Sinclair. 2007. “The Role of Nuclear Architecture in Genomic Instability and Ageing.” <em>Nature Reviews Molecular Cell Biology</em> 8 (9): 692.</p>
</div>
<div>
<p>Okano, Masaki, Daphne W. Bell, Daniel A. Haber, and En Li. 1999. “DNA Methyltransferases Dnmt3a and Dnmt3b Are Essential for de Novo Methylation and Mammalian Development.” <em>Cell</em> 99 (3): 247–57.</p>
</div>
<div>
<p>Ostapcuk, Veronika, Fabio Mohn, Sarah H. Carl, Anja Basters, Daniel Hess, Vytautas Iesmantavicius, Lisa Lampersberger, et al. 2018. “Activity-Dependent Neuroprotective Protein Recruits Hp1 and Chd4 to Control Lineage-Specifying Genes.” <em>Nature</em> 557 (7707): 739–43. doi:<a href="https://doi.org/10.1038/s41586-018-0153-8">10.1038/s41586-018-0153-8</a>.</p>
</div>
<div>
<p>Oyang, Elaine L., Bonnie C. Davidson, Winfong Lee, and Michael M. Poon. 2011. “Functional Characterization of the Dendritically Localized mRNA Neuronatin in Hippocampal Neurons.” <em>PLOS ONE</em> 6 (9): e24879. doi:<a href="https://doi.org/10.1371/journal.pone.0024879">10.1371/journal.pone.0024879</a>.</p>
</div>
<div>
<p>O’Carroll, D’onal, Harry Scherthan, Antoine H. F. M. Peters, Susanne Opravil, Andrew R. Haynes, Götz Laible, Stephen Rea, et al. 2000. “Isolation and Characterization ofSuv39h2, a Second Histone H3 Methyltransferase Gene That Displays Testis-Specific Expression.” <em>Molecular and Cellular Biology</em> 20 (24): 9423–33. doi:<a href="https://doi.org/10.1128/MCB.20.24.9423-9433.2000">10.1128/MCB.20.24.9423-9433.2000</a>.</p>
</div>
<div>
<p>O’Sullivan, Roderick J, Stefan Kubicek, Stuart L Schreiber, and Jan Karlseder. 2010. “Reduced Histone Biosynthesis and Chromatin Changes Arising from a Damage Signal at Telomeres.” <em>Nature Structural &amp; Molecular Biology</em> 17 (10): 1218–25. doi:<a href="https://doi.org/10.1038/nsmb.1897">10.1038/nsmb.1897</a>.</p>
</div>
<div>
<p>Pager, Cara T., Sylvia Schütz, Teresa M. Abraham, Guangxiang Luo, and Peter Sarnow. 2013. “Modulation of Hepatitis C Virus RNA Abundance and Virus Release by Dispersion of Processing Bodies and Enrichment of Stress Granules.” <em>Virology</em> 435 (2): 472–84. doi:<a href="https://doi.org/10.1016/j.virol.2012.10.027">10.1016/j.virol.2012.10.027</a>.</p>
</div>
<div>
<p>Pasquarella, Alessandra, Anja Ebert, Gustavo Pereira de Almeida, Maria Hinterberger, Maryam Kazerani, Alexander Nuber, Joachim Ellwart, Ludger Klein, Meinrad Busslinger, and Gunnar Schotta. 2016. “Retrotransposon Derepression Leads to Activation of the Unfolded Protein Response and Apoptosis in Pro-B Cells.” <em>Development</em> 143 (10): 1788–99. doi:<a href="https://doi.org/10.1242/dev.130203">10.1242/dev.130203</a>.</p>
</div>
<div>
<p>Pegoraro, Gianluca, Nard Kubben, Ute Wickert, Heike Göhler, Katrin Hoffmann, and Tom Misteli. 2009. “Ageing-Related Chromatin Defects Through Loss of the NURD Complex.” <em>Nature Cell Biology</em> 11 (10): 1261–7. doi:<a href="https://doi.org/10.1038/ncb1971">10.1038/ncb1971</a>.</p>
</div>
<div>
<p>Pereira, Sandrine, Patrice Bourgeois, Claire Navarro, Vera Esteves-Vieira, Pierre Cau, Annachiara De Sandre-Giovannoli, and Nicolas L’evy. 2008. “HGPS and Related Premature Aging Disorders: From Genomic Identification to the First Therapeutic Approaches.” <em>Mechanisms of Ageing and Development</em>, DNA damage, repair, ageing and age-related disease, 129 (7): 449–59. doi:<a href="https://doi.org/10.1016/j.mad.2008.04.003">10.1016/j.mad.2008.04.003</a>.</p>
</div>
<div>
<p>Perron, Herv’e, Françoise Lazarini, Klemens Ruprecht, Christine P’echoux-Longin, Danielle Seilhean, V’eronique Sazdovitch, Alain Cr’eange, et al. 2005. “Human Endogenous Retrovirus (HERV)-W ENV and GAG Proteins: Physiological Expression in Human Brain and Pathophysiological Modulation in Multiple Sclerosis Lesions.” <em>Journal of Neurovirology</em> 11 (1): 23–33. doi:<a href="https://doi.org/10.1080/13550280590901741">10.1080/13550280590901741</a>.</p>
</div>
<div>
<p>Peters, Antoine H. F. M., D’onal O’Carroll, Harry Scherthan, Karl Mechtler, Stephan Sauer, Christian Schöfer, Klara Weipoltshammer, et al. 2001. “Loss of the Suv39h Histone Methyltransferases Impairs Mammalian Heterochromatin and Genome Stability.” <em>Cell</em> 107 (3): 323–37. doi:<a href="https://doi.org/10.1016/S0092-8674(01)00542-6">10.1016/S0092-8674(01)00542-6</a>.</p>
</div>
<div>
<p>Prendergast, George C., and Edward B. Ziff. 1991. “Methylation-Sensitive Sequence-Specific DNA Binding by the c-Myc Basic Region.” <em>Science</em> 251 (4990): 186–89.</p>
</div>
<div>
<p>Prudencio, Mercedes, Patrick K. Gonzales, Casey N. Cook, Tania F. Gendron, Lillian M. Daughrity, Yuping Song, Mark T. W. Ebbert, et al. 2017. “Repetitive Element Transcripts Are Elevated in the Brain of C9orf72 ALS/FTLD Patients.” <em>Human Molecular Genetics</em> 26 (17): 3421–31. doi:<a href="https://doi.org/10.1093/hmg/ddx233">10.1093/hmg/ddx233</a>.</p>
</div>
<div>
<p>Pu, Mintie, Zhuoyu Ni, Minghui Wang, Xiujuan Wang, Jason G. Wood, Stephen L. Helfand, Haiyuan Yu, and Siu Sylvia Lee. 2015. “Trimethylation of Lys36 on H3 Restricts Gene Expression Change During Aging and Impacts Life Span.” <em>Genes &amp; Development</em> 29 (7): 718–31.</p>
</div>
<div>
<p>Quenneville, Simon, Gaetano Verde, Andrea Corsinotti, Adamandia Kapopoulou, Johan Jakobsson, Sandra Offner, Ilaria Baglivo, Paolo V. Pedone, Giovanna Grimaldi, and Andrea Riccio. 2011. “In Embryonic Stem Cells, Zfp57/Kap1 Recognize a Methylated Hexanucleotide to Affect Chromatin and DNA Methylation of Imprinting Control Regions.” <em>Molecular Cell</em> 44 (3): 361–72.</p>
</div>
<div>
<p>Rakyan, V. K., T. A. Down, S. Maslau, T. Andrew, T. P. Yang, H. Beyan, P. Whittaker, et al. 2010. “Human Aging-Associated DNA Hypermethylation Occurs Preferentially at Bivalent Chromatin Domains.” <em>Genome Research</em> 20 (4): 434–39. doi:<a href="https://doi.org/10.1101/gr.103101.109">10.1101/gr.103101.109</a>.</p>
</div>
<div>
<p>Ramesh, Vidya, Efil Bayam, Filippo M. Cernilogar, Ian M. Bonapace, Markus Schulze, Markus J. Riemenschneider, Gunnar Schotta, and Magdalena Götz. 2016. “Loss of Uhrf1 in Neural Stem Cells Leads to Activation of Retroviral Elements and Delayed Neurodegeneration.” <em>Genes &amp; Development</em> 30 (19): 2199–2212. doi:<a href="https://doi.org/10.1101/gad.284992.116">10.1101/gad.284992.116</a>.</p>
</div>
<div>
<p>Rath, Eva, Emanuel Berger, Anja Messlik, Tiago Nunes, Bo Liu, Sandy C. Kim, Nick Hoogenraad, Miquel Sans, R. Balfour Sartor, and Dirk Haller. 2012. “Induction of dsRNA-Activated Protein Kinase Links Mitochondrial Unfolded Protein Response to the Pathogenesis of Intestinal Inflammation.” <em>Gut</em> 61 (9): 1269–78.</p>
</div>
<div>
<p>Raurell-Vila, Helena, Laia Bosch-Presegue, Jessica Gonzalez, Noriko Kane-Goldsmith, Carmen Casal, Jeremy P. Brown, Anna Marazuela-Duque, Prim B. Singh, Lourdes Serrano, and Alejandro Vaquero. 2017. “An Hp1 Isoform-Specific Feedback Mechanism Regulates Suv39h1 Activity Under Stress Conditions.” <em>Epigenetics</em> 12 (2): 166–75. doi:<a href="https://doi.org/10.1080/15592294.2016.1278096">10.1080/15592294.2016.1278096</a>.</p>
</div>
<div>
<p>Readhead, Ben, Jean-Vianney Haure-Mirande, Cory C. Funk, Matthew A. Richards, Paul Shannon, Vahram Haroutunian, Mary Sano, et al. 2018. “Multiscale Analysis of Independent Alzheimer’s Cohorts Finds Disruption of Molecular, Genetic, and Clinical Networks by Human Herpesvirus.” <em>Neuron</em> 99 (1): 64–82.e7. doi:<a href="https://doi.org/10.1016/j.neuron.2018.05.023">10.1016/j.neuron.2018.05.023</a>.</p>
</div>
<div>
<p>Regis, Gabriella, Laura Conti, Daniela Boselli, and Francesco Novelli. 2006. “IFN<span class="math inline">\(\gamma\)</span>R2 Trafficking Tunes IFN<span class="math inline">\(\gamma\)</span>STAT1 Signaling in T Lymphocytes.” <em>Trends in Immunology</em> 27 (2): 96–101. doi:<a href="https://doi.org/10.1016/j.it.2005.12.002">10.1016/j.it.2005.12.002</a>.</p>
</div>
<div>
<p>Riclet, Raphaël, Mariam Chendeb, Jean-Luc Vonesch, Dirk Koczan, Hans-Juergen Thiesen, R’egine Losson, Florence Cammas, and Wendy Bickmore. 2008. “Disruption of the Interaction Between Transcriptional Intermediary Factor 1b and Heterochromatin Protein 1 Leads to a Switch from DNA Hyper- to Hypomethylation and H3k9 to H3k27 Trimethylation on the MEST Promoter Correlating with Gene Reactivation.” <em>Molecular Biology of the Cell</em> 20 (1): 296–305. doi:<a href="https://doi.org/10.1091/mbc.e08-05-0510">10.1091/mbc.e08-05-0510</a>.</p>
</div>
<div>
<p>Ridley, R. M., H. F. Baker, C. P. Windle, and R. M. Cummings. 2006. “Very Long Term Studies of the Seeding of <span class="math inline">\(\beta\)</span>-Amyloidosis in Primates.” <em>Journal of Neural Transmission</em> 113 (9): 1243.</p>
</div>
<div>
<p>Riggs, A. D. 1975. “X Inactivation, Differentiation, and DNA Methylation.” <em>Cytogenetic and Genome Research</em> 14 (1): 9–25. doi:<a href="https://doi.org/10.1159/000130315">10.1159/000130315</a>.</p>
</div>
<div>
<p>Riso, Vincenzo, Marco Cammisa, Harpreet Kukreja, Zahra Anvar, Gaetano Verde, Angela Sparago, Basilia Acurzio, et al. 2016. “ZFP57 Maintains the Parent-of-Origin-Specific Expression of the Imprinted Genes and Differentially Affects Non-Imprinted Targets in Mouse Embryonic Stem Cells.” <em>Nucleic Acids Research</em> 44 (17): 8165–78. doi:<a href="https://doi.org/10.1093/nar/gkw505">10.1093/nar/gkw505</a>.</p>
</div>
<div>
<p>Rodgers, A. B., C. P. Morgan, S. L. Bronson, S. Revello, and T. L. Bale. 2013. “Paternal Stress Exposure Alters Sperm MicroRNA Content and Reprograms Offspring HPA Stress Axis Regulation.” <em>Journal of Neuroscience</em> 33 (21): 9003–12. doi:<a href="https://doi.org/10.1523/JNEUROSCI.0914-13.2013">10.1523/JNEUROSCI.0914-13.2013</a>.</p>
</div>
<div>
<p>Rodgers, Ali B., Christopher P. Morgan, N. Adrian Leu, and Tracy L. Bale. 2015. “Transgenerational Epigenetic Programming via Sperm microRNA Recapitulates Effects of Paternal Stress.” <em>Proceedings of the National Academy of Sciences</em> 112 (44): 13699–13704.</p>
</div>
<div>
<p>Rojas, Margarito, Carlos F. Arias, and Susana L’opez. 2010. “Protein Kinase R Is Responsible for the Phosphorylation of eIF2<span class="math inline">\(\alpha\)</span> in Rotavirus Infection.” <em>Journal of Virology</em> 84 (20): 10457–66. doi:<a href="https://doi.org/10.1128/JVI.00625-10">10.1128/JVI.00625-10</a>.</p>
</div>
<div>
<p>Rottach, Andrea, Carina Frauer, Garwin Pichler, Ian Marc Bonapace, Fabio Spada, and Heinrich Leonhardt. 2010. “The Multi-Domain Protein Np95 Connects DNA Methylation and Histone Modification.” <em>Nucleic Acids Research</em> 38 (6): 1796–1804. <a href="http://nar.oxfordjournals.org/content/38/6/1796.short" class="uri">http://nar.oxfordjournals.org/content/38/6/1796.short</a>.</p>
</div>
<div>
<p>Roulois, David, Helen Loo Yau, Rajat Singhania, Yadong Wang, Arnavaz Danesh, Shu Yi Shen, Han Han, et al. 2015. “DNA-Demethylating Agents Target Colorectal Cancer Cells by Inducing Viral Mimicry by Endogenous Transcripts.” <em>Cell</em> 162 (5): 961–73. doi:<a href="https://doi.org/10.1016/j.cell.2015.07.056">10.1016/j.cell.2015.07.056</a>.</p>
</div>
<div>
<p>Rowe, H. M., M. Friedli, S. Offner, S. Verp, D. Mesnard, J. Marquis, T. Aktas, and D. Trono. 2013. “De Novo DNA Methylation of Endogenous Retroviruses Is Shaped by KRAB-ZFPs/Kap1 and ESET.” <em>Development</em> 140 (3): 519–29. doi:<a href="https://doi.org/10.1242/dev.087585">10.1242/dev.087585</a>.</p>
</div>
<div>
<p>Rubinstein, Rotem, Chan Aye Thu, Kerry Marie Goodman, Holly Noelle Wolcott, Fabiana Bahna, Seetha Mannepalli, Goran Ahlsen, et al. 2015. “Molecular Logic of Neuronal Self-Recognition Through Protocadherin Domain Interactions.” <em>Cell</em> 163 (3): 629–42. doi:<a href="https://doi.org/10.1016/j.cell.2015.09.026">10.1016/j.cell.2015.09.026</a>.</p>
</div>
<div>
<p>Rutherford, M. N., G. E. Hannigan, and B. R. Williams. 1988. “Interferon-Induced Binding of Nuclear Factors to Promoter Elements of the 2-5a Synthetase Gene.” <em>The EMBO Journal</em> 7 (3): 751–59.</p>
</div>
<div>
<p>Ryu, Soo Hee, Keunsoo Kang, Taekyung Yoo, Cheol O. Joe, and Jae Hoon Chung. 2011. “Transcriptional Repression of Repeat-Derived Transcripts Correlates with Histone Hypoacetylation at Repetitive DNA Elements in Aged Mice Brain.” <em>Experimental Gerontology</em> 46 (10): 811–18. doi:<a href="https://doi.org/10.1016/j.exger.2011.07.001">10.1016/j.exger.2011.07.001</a>.</p>
</div>
<div>
<p>Saito, E.-s., V. W. Keng, J. Takeda, and K. Horie. 2008. “Translation from Nonautonomous Type IAP Retrotransposon Is a Critical Determinant of Transposition Activity: Implication for Retrotransposon-Mediated Genome Evolution.” <em>Genome Research</em> 18 (6): 859–68. doi:<a href="https://doi.org/10.1101/gr.069310.107">10.1101/gr.069310.107</a>.</p>
</div>
<div>
<p>Saito, Tetsuichiro. 2006. “In Vivo Electroporation in the Embryonic Mouse Central Nervous System.” <em>Nature Protocols</em> 1 (November): 1552. <a href="http://dx.doi.org/10.1038/nprot.2006.276" class="uri">http://dx.doi.org/10.1038/nprot.2006.276</a>.</p>
</div>
<div>
<p>Salpea, Paraskevi, Valya R. Russanova, Tazuko H. Hirai, Thomae G. Sourlingas, Kalliope E. Sekeri-Pataryas, Roberto Romero, Jonathan Epstein, and Bruce H. Howard. 2012. “Postnatal Development- and Age-Related Changes in DNA-Methylation Patterns in the Human Genome.” <em>Nucleic Acids Research</em> 40 (14): 6477–94. doi:<a href="https://doi.org/10.1093/nar/gks312">10.1093/nar/gks312</a>.</p>
</div>
<div>
<p>Salvi, Jayesh S., and Karim Mekhail. 2015. “R-Loops Highlight the Nucleus in ALS.” <em>Nucleus</em> 6 (1): 23–29.</p>
</div>
<div>
<p>Sanchez, Diego, Raquel Bajo-Grañeras, Manuela Del Caño-Espinel, Rosa Garcia-Centeno, Nadia Garcia-Mateo, Raquel Pascua-Maestro, and Maria D. Ganfornina. 2015. “Aging Without Apolipoprotein D: Molecular and Cellular Modifications in the Hippocampus and Cortex.” <em>Experimental Gerontology</em> 67 (July): 19–47. doi:<a href="https://doi.org/10.1016/j.exger.2015.04.003">10.1016/j.exger.2015.04.003</a>.</p>
</div>
<div>
<p>Santoro, Raffaella, Junwei Li, and Ingrid Grummt. 2002. “The Nucleolar Remodeling Complex NoRC Mediates Heterochromatin Formation and Silencing of Ribosomal Gene Transcription.” <em>Nature Genetics</em> 32 (3): 393–96. doi:<a href="https://doi.org/10.1038/ng1010">10.1038/ng1010</a>.</p>
</div>
<div>
<p>Santoro, Raffaella, Kerstin-Maike Schmitz, Juan Sandoval, and Ingrid Grummt. 2010. “Intergenic Transcripts Originating from a Subclass of Ribosomal DNA Repeats Silence Ribosomal RNA Genes in Trans.” <em>EMBO Reports</em> 11 (1): 52–58.</p>
</div>
<div>
<p>Santos-Rosa, Helena, Robert Schneider, Andrew J. Bannister, Julia Sherriff, Bradley E. Bernstein, NC Tolga Emre, Stuart L. Schreiber, Jane Mellor, and Tony Kouzarides. 2002. “Active Genes Are Tri-Methylated at K4 of Histone H3.” <em>Nature</em> 419 (6905): 407–11.</p>
</div>
<div>
<p>Schindelin, Johannes, Ignacio Arganda-Carreras, Erwin Frise, Verena Kaynig, Mark Longair, Tobias Pietzsch, Stephan Preibisch, et al. 2012. “Fiji: An Open-Source Platform for Biological-Image Analysis.” <em>Nature Methods</em> 9 (7): 676–82. doi:<a href="https://doi.org/10.1038/nmeth.2019">10.1038/nmeth.2019</a>.</p>
</div>
<div>
<p>Schmitz, Kerstin-Maike, Christine Mayer, Anna Postepska, and Ingrid Grummt. 2010. “Interaction of Noncoding RNA with the rDNA Promoter Mediates Recruitment of DNMT3b and Silencing of rRNA Genes.” <em>Genes &amp; Development</em> 24 (20): 2264–9.</p>
</div>
<div>
<p>Schotta, G. 2004. “A Silencing Pathway to Induce H3-K9 and H4-K20 Trimethylation at Constitutive Heterochromatin.” <em>Genes &amp; Development</em> 18 (11): 1251–62. doi:<a href="https://doi.org/10.1101/gad.300704">10.1101/gad.300704</a>.</p>
</div>
<div>
<p>Schultz, David C., Kasirajan Ayyanathan, Dmitri Negorev, Gerd G. Maul, and Frank J. Rauscher. 2002. “SETDB1: A Novel KAP-1-Associated Histone H3, Lysine 9-Specific Methyltransferase That Contributes to Hp1-Mediated Silencing of Euchromatic Genes by KRAB Zinc-Finger Proteins.” <em>Genes &amp; Development</em> 16 (8): 919–32.</p>
</div>
<div>
<p>Sekar, Aswin, Allison R. Bialas, Heather de Rivera, Avery Davis, Timothy R. Hammond, Nolan Kamitaki, Katherine Tooley, et al. 2016. “Schizophrenia Risk from Complex Variation of Complement Component 4.” <em>Nature</em> 530 (7589): 177–83. doi:<a href="https://doi.org/10.1038/nature16549">10.1038/nature16549</a>.</p>
</div>
<div>
<p>Sen, Payel, Weiwei Dang, Greg Donahue, Junbiao Dai, Jean Dorsey, Xiaohua Cao, Wei Liu, Kajia Cao, Rocco Perry, and Jun Yeop Lee. 2015. “H3K36 Methylation Promotes Longevity by Enhancing Transcriptional Fidelity.” <em>Genes &amp; Development</em> 29 (13): 1362–76.</p>
</div>
<div>
<p>Shah, P. P., G. Donahue, G. L. Otte, B. C. Capell, D. M. Nelson, K. Cao, V. Aggarwala, et al. 2013. “Lamin B1 Depletion in Senescent Cells Triggers Large-Scale Changes in Gene Expression and the Chromatin Landscape.” <em>Genes &amp; Development</em> 27 (16): 1787–99. doi:<a href="https://doi.org/10.1101/gad.223834.113">10.1101/gad.223834.113</a>.</p>
</div>
<div>
<p>Sharma, Jaiprakash, Diptendu Mukherjee, Sudheendra NR Rao, Soumya Iyengar, Susarla Krishna Shankar, Parthasarathy Satishchandra, and Nihar Ranjan Jana. 2013. “Neuronatin-Mediated Aberrant Calcium Signaling and Endoplasmic Reticulum Stress Underlie Neuropathology in Lafora Disease.” <em>Journal of Biological Chemistry</em> 288 (13): 9482–90.</p>
</div>
<div>
<p>Sharma, Salil, Ines Khadimallah, Adam Williamson Corya, Yousuf Omar Ali, Xi Rao, Yunlong Liu, and Hui-Chen Lu. 2018. “Presymptomatic Change in microRNAs Modulates Tau Pathology.” <em>Scientific Reports</em> 8 (1): 9251. doi:<a href="https://doi.org/10.1038/s41598-018-27527-6">10.1038/s41598-018-27527-6</a>.</p>
</div>
<div>
<p>Sholl, D. A. 1953. “Dendritic Organization in the Neurons of the Visual and Motor Cortices of the Cat.” <em>Journal of Anatomy</em> 87 (Pt 4): 387–406.1. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1244622/" class="uri">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1244622/</a>.</p>
</div>
<div>
<p>Short, A. K., S. Yeshurun, R. Powell, V. M. Perreau, A. Fox, J. H. Kim, T. Y. Pang, and A. J. Hannan. 2017. “Exercise Alters Mouse Sperm Small Noncoding RNAs and Induces a Transgenerational Modification of Male Offspring Conditioned Fear and Anxiety.” <em>Translational Psychiatry</em> 7 (5): e1114.</p>
</div>
<div>
<p>Shumaker, Dale K., Thomas Dechat, Alexander Kohlmaier, Stephen A. Adam, Matthew R. Bozovsky, Michael R. Erdos, Maria Eriksson, Anne E. Goldman, Satya Khuon, and Francis S. Collins. 2006. “Mutant Nuclear Lamin A Leads to Progressive Alterations of Epigenetic Control in Premature Aging.” <em>Proceedings of the National Academy of Sciences</em> 103 (23): 8703–8.</p>
</div>
<div>
<p>Singh, Prim B., J. Ross Miller, Jonathan Pearce, Rashmi Kothary, Robert D. Burton, Renato Paro, Tharappel C. James, and Stephen J. Gaunt. 1991. “A Sequence Motif Found in a Drosophila Heterochromatin Protein Is Conserved in Animals and Plants.” <em>Nucleic Acids Research</em> 19 (4): 789–94. <a href="http://nar.oxfordjournals.org/content/19/4/789.short" class="uri">http://nar.oxfordjournals.org/content/19/4/789.short</a>.</p>
</div>
<div>
<p>Singhrao, S. K., J. W. Neal, B. P. Morgan, and P. Gasque. 1999. “Increased Complement Biosynthesis by Microglia and Complement Activation on Neurons in Huntington’s Disease.” <em>Experimental Neurology</em> 159 (2): 362–76. doi:<a href="https://doi.org/10.1006/exnr.1999.7170">10.1006/exnr.1999.7170</a>.</p>
</div>
<div>
<p>Sleutels, Frank, Ronald Zwart, and Denise P. Barlow. 2002. “The Non-Coding <em>Air</em> RNA Is Required for Silencing Autosomal Imprinted Genes.” <em>Nature</em> 415 (6873): 810–13. doi:<a href="https://doi.org/10.1038/415810a">10.1038/415810a</a>.</p>
</div>
<div>
<p>Slotkin, R. Keith, and Robert Martienssen. 2007. “Transposable Elements and the Epigenetic Regulation of the Genome.” <em>Nature Reviews Genetics</em> 8 (4): 272.</p>
</div>
<div>
<p>Smallwood, A., P.-O. Esteve, S. Pradhan, and M. Carey. 2007. “Functional Cooperation Between Hp1 and Dnmt1 Mediates Gene Silencing.” <em>Genes &amp; Development</em> 21 (10): 1169–78. doi:<a href="https://doi.org/10.1101/gad.1536807">10.1101/gad.1536807</a>.</p>
</div>
<div>
<p>Smallwood, Andrea, Gary C. Hon, Fulai Jin, Ryan E. Henry, Joaquin E. Espinosa, and Bing Ren. 2012. “CBX3 Regulates Efficient RNA Processing Genome-Wide.” <em>Genome Research</em>, gr. 124818.111.</p>
</div>
<div>
<p>Smeal, Tod, James Claus, Brian Kennedy, Francesca Cole, and Leonard Guarente. 1996. “Loss of Transcriptional Silencing Causes Sterility in Old Mother Cells of S. Cerevisiae.” <em>Cell</em> 84 (4): 633–42.</p>
</div>
<div>
<p>Smith, Judith A. 2014. “A New Paradigm: Innate Immune Sensing of Viruses via the Unfolded Protein Response.” <em>Frontiers in Microbiology</em> 5. doi:<a href="https://doi.org/10.3389/fmicb.2014.00222">10.3389/fmicb.2014.00222</a>.</p>
</div>
<div>
<p>Smothers, James F., and Steven Henikoff. 2000. “The Hp1 Chromo Shadow Domain Binds a Consensus Peptide Pentamer.” <em>Current Biology</em> 10 (1): 27–30.</p>
</div>
<div>
<p>———. 2001. “The Hinge and Chromo Shadow Domain Impart Distinct Targeting of Hp1-Like Proteins.” <em>Molecular and Cellular Biology</em> 21 (7): 2555–69. doi:<a href="https://doi.org/10.1128/MCB.21.7.2555-2569.2001">10.1128/MCB.21.7.2555-2569.2001</a>.</p>
</div>
<div>
<p>Soria, Gaston, and Genevi‘eve Almouzni. 2013. “Differential Contribution of Hp1 Proteins to DNA End Resection and Homology-Directed Repair.” <em>Cell Cycle</em> 12 (3): 422–29. doi:<a href="https://doi.org/10.4161/cc.23215">10.4161/cc.23215</a>.</p>
</div>
<div>
<p>Sousa-Victor, Pedro, Arshad Ayyaz, Rippei Hayashi, Yanyan Qi, David T. Madden, Victoria V. Lunyak, and Heinrich Jasper. 2017. “Piwi Is Required to Limit Exhaustion of Aging Somatic Stem Cells.” <em>Cell Reports</em> 20 (11): 2527–37.</p>
</div>
<div>
<p>Sridharan, Rupa, Michelle Gonzales-Cope, Constantinos Chronis, Giancarlo Bonora, Robin McKee, Chengyang Huang, Sanjeet Patel, et al. 2013. “Proteomic and Genomic Approaches Reveal Critical Functions of H3k9 Methylation and Heterochromatin Protein-1γ in Reprogramming to Pluripotency.” <em>Nature Cell Biology</em> 15 (7): 872–82. doi:<a href="https://doi.org/10.1038/ncb2768">10.1038/ncb2768</a>.</p>
</div>
<div>
<p>Staahl, B. T., J. Tang, W. Wu, A. Sun, A. D. Gitler, A. S. Yoo, and G. R. Crabtree. 2013. “Kinetic Analysis of npBAF to nBAF Switching Reveals Exchange of Ss18 with CREST and Integration with Neural Developmental Pathways.” <em>Journal of Neuroscience</em> 33 (25): 10348–61. doi:<a href="https://doi.org/10.1523/JNEUROSCI.1258-13.2013">10.1523/JNEUROSCI.1258-13.2013</a>.</p>
</div>
<div>
<p>Stephan, Alexander H., Daniel V. Madison, Jos’e María Mateos, Deborah A. Fraser, Emilie A. Lovelett, Laurence Coutellier, Leo Kim, et al. 2013. “A Dramatic Increase of C1q Protein in the CNS During Normal Aging.” <em>The Journal of Neuroscience</em> 33 (33): 13460–74. doi:<a href="https://doi.org/10.1523/JNEUROSCI.1333-13.2013">10.1523/JNEUROSCI.1333-13.2013</a>.</p>
</div>
<div>
<p>Stevens, Beth, Nicola J. Allen, Luis E. Vazquez, Gareth R. Howell, Karen S. Christopherson, Navid Nouri, Kristina D. Micheva, et al. 2007. “The Classical Complement Cascade Mediates CNS Synapse Elimination.” <em>Cell</em> 131 (6): 1164–78. doi:<a href="https://doi.org/10.1016/j.cell.2007.10.036">10.1016/j.cell.2007.10.036</a>.</p>
</div>
<div>
<p>Stilling, Roman M., Eva Benito, Michael Gertig, Jonas Barth, Vincenzo Capece, Susanne Burkhardt, Stefan Bonn, and Andre Fischer. 2014. “De-Regulation of Gene Expression and Alternative Splicing Affects Distinct Cellular Pathways in the Aging Hippocampus.” <em>Frontiers in Cellular Neuroscience</em> 8. doi:<a href="https://doi.org/10.3389/fncel.2014.00373">10.3389/fncel.2014.00373</a>.</p>
</div>
<div>
<p>Strom, Amy R., Alexander V. Emelyanov, Mustafa Mir, Dmitry V. Fyodorov, Xavier Darzacq, and Gary H. Karpen. 2017. “Phase Separation Drives Heterochromatin Domain Formation.” <em>Nature</em> 547 (7662): 241–45. doi:<a href="https://doi.org/10.1038/nature22989">10.1038/nature22989</a>.</p>
</div>
<div>
<p>Sun, Deqiang, Min Luo, Mira Jeong, Benjamin Rodriguez, Zheng Xia, Rebecca Hannah, Hui Wang, Thuc Le, Kym F. Faull, and Rui Chen. 2014. “Epigenomic Profiling of Young and Aged HSCs Reveals Concerted Changes During Aging That Reinforce Self-Renewal.” <em>Cell Stem Cell</em> 14 (5): 673–88.</p>
</div>
<div>
<p>Sundaram, Vasavi, Yong Cheng, Zhihai Ma, Daofeng Li, Xiaoyun Xing, Peter Edge, Michael P. Snyder, and Ting Wang. 2014. “Widespread Contribution of Transposable Elements to the Innovation of Gene Regulatory Networks.” <em>Genome Research</em> 24 (12): 1963–76.</p>
</div>
<div>
<p>Sunyer, Berta, Sudarshan Patil, Harald Höger, and Gert Lubec. 2007. “Barnes Maze, a Useful Task to Assess Spatial Reference Memory in the Mice.” <em>Nature Protocols</em>. October 4. <a href="http://dx.doi.org/10.1038/nprot.2007.390" class="uri">http://dx.doi.org/10.1038/nprot.2007.390</a>.</p>
</div>
<div>
<p>Szulwach, Keith E, Xuekun Li, Yujing Li, Chun-Xiao Song, Hao Wu, Qing Dai, Hasan Irier, et al. 2011. “5-hmC–Mediated Epigenetic Dynamics During Postnatal Neurodevelopment and Aging.” <em>Nature Neuroscience</em> 14 (12): 1607–16. doi:<a href="https://doi.org/10.1038/nn.2959">10.1038/nn.2959</a>.</p>
</div>
<div>
<p>Tahiliani, Mamta, Kian Peng Koh, Yinghua Shen, William A. Pastor, Hozefa Bandukwala, Yevgeny Brudno, Suneet Agarwal, Lakshminarayan M. Iyer, David R. Liu, and L. Aravind. 2009. “Conversion of 5-Methylcytosine to 5-Hydroxymethylcytosine in Mammalian DNA by MLL Partner Tet1.” <em>Science</em> 324 (5929): 930–35.</p>
</div>
<div>
<p>Tardif, Keith D., Kazutoshi Mori, Randal J. Kaufman, and Aleem Siddiqui. 2004. “Hepatitis C Virus Suppresses the Ire1-Xbp1 Pathway of the Unfolded Protein Response.” <em>Journal of Biological Chemistry</em>.</p>
</div>
<div>
<p>Tate, Peri H., and Adrian P. Bird. 1993. “Effects of DNA Methylation on DNA-Binding Proteins and Gene Expression.” <em>Current Opinion in Genetics &amp; Development</em> 3 (2): 226–31.</p>
</div>
<div>
<p>Theil, T., G. Alvarez-Bolado, A. Walter, and U. Ruther. 1999. “Gli3 Is Required for Emx Gene Expression During Dorsal Telencephalon Development.” <em>Development</em> 126 (16): 3561–71. <a href="http://dev.biologists.org/content/126/16/3561" class="uri">http://dev.biologists.org/content/126/16/3561</a>.</p>
</div>
<div>
<p>Thiru, Abarna, Daniel Nietlispach, Helen R. Mott, Mitsuru Okuwaki, Debbie Lyon, Peter R. Nielsen, Miriam Hirshberg, Alain Verreault, Natalia V. Murzina, and Ernest D. Laue. 2004. “Structural Basis of Hp1/PXVXL Motif Peptide Interactions and Hp1 Localisation to Heterochromatin.” <em>The EMBO Journal</em> 23 (3): 489–99. <a href="http://www.nature.com/emboj/journal/v23/n3/abs/7600088a.html" class="uri">http://www.nature.com/emboj/journal/v23/n3/abs/7600088a.html</a>.</p>
</div>
<div>
<p>Thomas, Charles A., Apuã C.M. Paquola, and Alysson R. Muotri. 2012. “LINE-1 Retrotransposition in the Nervous System.” <em>Annual Review of Cell and Developmental Biology</em> 28 (1): 555–73. doi:<a href="https://doi.org/10.1146/annurev-cellbio-101011-155822">10.1146/annurev-cellbio-101011-155822</a>.</p>
</div>
<div>
<p>Thomas, J. H., and S. Schneider. 2011. “Coevolution of Retroelements and Tandem Zinc Finger Genes.” <em>Genome Research</em> 21 (11): 1800–1812. doi:<a href="https://doi.org/10.1101/gr.121749.111">10.1101/gr.121749.111</a>.</p>
</div>
<div>
<p>Thompson, Peter J., Todd S. Macfarlan, and Matthew C. Lorincz. 2016. “Long Terminal Repeats: From Parasitic Elements to Building Blocks of the Transcriptional Regulatory Repertoire.” <em>Molecular Cell</em> 62 (5): 766–76. doi:<a href="https://doi.org/10.1016/j.molcel.2016.03.029">10.1016/j.molcel.2016.03.029</a>.</p>
</div>
<div>
<p>Toyoda, Shunsuke, Masahumi Kawaguchi, Toshihiro Kobayashi, Etsuko Tarusawa, Tomoko Toyama, Masaki Okano, Masaaki Oda, et al. 2014. “Developmental Epigenetic Modification Regulates Stochastic Expression of Clustered Protocadherin Genes, Generating Single Neuron Diversity.” <em>Neuron</em> 82 (1): 94–108. doi:<a href="https://doi.org/10.1016/j.neuron.2014.02.005">10.1016/j.neuron.2014.02.005</a>.</p>
</div>
<div>
<p>Trembecka-Lucas, Dominika O., Dominika O. Trembecka-Lucas, Aleksander T. Szczurek, Aleksander T. Szczurek, Jurek W. Dobrucki, and Jurek W. Dobrucki. 2013. “Dynamics of the Hp1<span class="math inline">\(\beta\)</span>-PCNA-Containing Complexes in DNA Replication and Repair.” <em>Nucleus</em> 4 (1): 74–82. doi:<a href="https://doi.org/10.4161/nucl.23683">10.4161/nucl.23683</a>.</p>
</div>
<div>
<p>Trouw, Leendert. A., Henrietta M. Nielsen, Lennart Minthon, Elisabet Londos, Göran Landberg, Robert Veerhuis, Sabina Janciauskiene, and Anna M. Blom. 2008. “C4b-Binding Protein in Alzheimer’s Disease: Binding to A<span class="math inline">\(\beta\)</span>142 and to Dead Cells.” <em>Molecular Immunology</em> 45 (13): 3649–60. doi:<a href="https://doi.org/10.1016/j.molimm.2008.04.025">10.1016/j.molimm.2008.04.025</a>.</p>
</div>
<div>
<p>Tzeng, Nian-Sheng, Chi-Hsiang Chung, Fu-Huang Lin, Chien-Ping Chiang, Chin-Bin Yeh, San-Yuan Huang, Ru-Band Lu, et al. 2018. “Anti-Herpetic Medications and Reduced Risk of Dementia in Patients with Herpes Simplex Virus Infectionsa Nationwide, Population-Based Cohort Study in Taiwan.” <em>Neurotherapeutics</em> 15 (2): 417–29. doi:<a href="https://doi.org/10.1007/s13311-018-0611-x">10.1007/s13311-018-0611-x</a>.</p>
</div>
<div>
<p>Udugama, Maheshi, Fiona T. M. Chang, F. Lyn Chan, Michelle C. Tang, Hilda A. Pickett, James D. R. McGhie, Lynne Mayne, Philippe Collas, Jeffrey R. Mann, and Lee H. Wong. 2015. “Histone Variant H3. 3 Provides the Heterochromatic H3 Lysine 9 Tri-Methylation Mark at Telomeres.” <em>Nucleic Acids Research</em> 43 (21): 10227–37.</p>
</div>
<div>
<p>Ueki, Akinori, Kyoko Goto, Noriko Sato, Hiroyuki Iso, and Yoshio Morita. 2006. “Prepulse Inhibition of Acoustic Startle Response in Mild Cognitive Impairment and Mild Dementia of Alzheimer Type.” <em>Psychiatry and Clinical Neurosciences</em> 60 (1): 55–62. doi:<a href="https://doi.org/10.1111/j.1440-1819.2006.01460.x">10.1111/j.1440-1819.2006.01460.x</a>.</p>
</div>
<div>
<p>Umareddy, Indira, Olivier Pluquet, Qing Yin Wang, Subhash G. Vasudevan, Eric Chevet, and Feng Gu. 2007. “Dengue Virus Serotype Infection Specifies the Activation of the Unfolded Protein Response.” <em>Virology Journal</em> 4 (1): 91.</p>
</div>
<div>
<p>Vakoc, Christopher R., Sean A. Mandat, Benjamin A. Olenchock, and Gerd A. Blobel. 2005. “Histone H3 Lysine 9 Methylation and HP1γ Are Associated with Transcription Elongation Through Mammalian Chromatin.” <em>Molecular Cell</em> 19 (3): 381–91. doi:<a href="https://doi.org/10.1016/j.molcel.2005.06.011">10.1016/j.molcel.2005.06.011</a>.</p>
</div>
<div>
<p>Vasek, Michael J., Charise Garber, Denise Dorsey, Douglas M. Durrant, Bryan Bollman, Allison Soung, Jinsheng Yu, et al. 2016. “A ComplementMicroglial Axis Drives Synapse Loss During Virus-Induced Memory Impairment.” <em>Nature</em> 534 (7608): 538–43. doi:<a href="https://doi.org/10.1038/nature18283">10.1038/nature18283</a>.</p>
</div>
<div>
<p>Vattem, Krishna M., and Ronald C. Wek. 2004. “Reinitiation Involving Upstream ORFs Regulates Atf4 mRNA Translation in Mammalian Cells.” <em>Proceedings of the National Academy of Sciences of the United States of America</em> 101 (31): 11269–74. doi:<a href="https://doi.org/10.1073/pnas.0400541101">10.1073/pnas.0400541101</a>.</p>
</div>
<div>
<p>Verschoor, Chris P., Lisa M. McEwen, Michael S. Kobor, Mark B. Loeb, and Dawn M. E. Bowdish. 2018. “DNA Methylation Patterns Are Related to Co-Morbidity Status and Circulating C-Reactive Protein Levels in the Nursing Home Elderly.” <em>Experimental Gerontology</em>, Aging and Immunity 2017, 105 (May): 47–52. doi:<a href="https://doi.org/10.1016/j.exger.2017.10.010">10.1016/j.exger.2017.10.010</a>.</p>
</div>
<div>
<p>Villeponteau, Bryant. 1997. “The Heterochromatin Loss Model of Aging.” <em>Experimental Gerontology</em> 32 (4-5): 383–94.</p>
</div>
<div>
<p>Vogel, M. J., L. Guelen, E. de Wit, D. P. Hupkes, M. Loden, W. Talhout, M. Feenstra, B. Abbas, A.-K. Classen, and B. van Steensel. 2006. “Human Heterochromatin Proteins Form Large Domains Containing KRAB-ZNF Genes.” <em>Genome Research</em> 16 (12): 1493–1504. doi:<a href="https://doi.org/10.1101/gr.5391806">10.1101/gr.5391806</a>.</p>
</div>
<div>
<p>Voon, Hsiao PJ, and Richard J. Gibbons. 2016. “Maintaining Memory of Silencing at Imprinted Differentially Methylated Regions.” <em>Cellular and Molecular Life Sciences</em> 73 (9): 1871–9.</p>
</div>
<div>
<p>Voon, Hsiao PJ, and Lee H. Wong. 2016. “New Players in Heterochromatin Silencing: Histone Variant H3. 3 and the ATRX/DAXX Chaperone.” <em>Nucleic Acids Research</em> 44 (4): 1496–1501.</p>
</div>
<div>
<p>Voon, Hsiao PJ, Jim R. Hughes, Christina Rode, A. Inti, Thomas Jenuwein, Robert Feil, Douglas R. Higgs, and Richard J. Gibbons. 2015. “ATRX Plays a Key Role in Maintaining Silencing at Interstitial Heterochromatic Loci and Imprinted Genes.” <em>Cell Reports</em> 11 (3): 405–18.</p>
</div>
<div>
<p>Waddington, C. H. 1940. “Organisers and Genes.” <em>Cambridge University Press</em>.</p>
</div>
<div>
<p>Wagner, Mirko, Jessica Steinbacher, Theo F. J. Kraus, Stylianos Michalakis, Benjamin Hackner, Toni Pfaffeneder, Arshan Perera, et al. 2015. “Age-Dependent Levels of 5-Methyl-, 5-Hydroxymethyl-, and 5-Formylcytosine in Human and Mouse Brain Tissues.” <em>Angewandte Chemie International Edition</em> 54 (42): 12511–4. doi:<a href="https://doi.org/10.1002/anie.201502722">10.1002/anie.201502722</a>.</p>
</div>
<div>
<p>Wang, Chun Mei, Sau Na Tsai, Tai Wai Yew, Yiu Wa Kwan, and Sai Ming Ngai. 2010. “Identification of Histone Methylation Multiplicities Patterns in the Brain of Senescence-Accelerated Prone Mouse 8.” <em>Biogerontology</em> 11 (1): 87–102. doi:<a href="https://doi.org/10.1007/s10522-009-9231-5">10.1007/s10522-009-9231-5</a>.</p>
</div>
<div>
<p>Wang, Weidong, Jacques Côt’e, Yutong Xue, Sharleen Zhou, Paul A. Khavari, Stephen R. Biggar, Christian Muchardt, Ganjam V. Kalpana, Stephen P. Goff, and Moshe Yaniv. 1996. “Purification and Biochemical Heterogeneity of the Mammalian SWI‐SNF Complex.” <em>The EMBO Journal</em> 15 (19): 5370–82.</p>
</div>
<div>
<p>Wei, Jinxue, Ye Liu, Liansheng Zhao, Xiao Yang, Peiyan Ni, Yingcheng Wang, Tao Li, and Xiaohong Ma. 2017. “Plasma Complement Component 4 Increases in Patients with Major Depressive Disorder.” <em>Neuropsychiatric Disease and Treatment</em> 14 (December): 37–41. doi:<a href="https://doi.org/10.2147/NDT.S151238">10.2147/NDT.S151238</a>.</p>
</div>
<div>
<p>Wei, Lai, Baoying Liu, Jingsheng Tuo, Defen Shen, Ping Chen, Zhiyu Li, Xunxian Liu, Jia Ni, Pradeep Dagur, and H. Nida Sen. 2012. “Hypomethylation of the Il17rc Promoter Associates with Age-Related Macular Degeneration.” <em>Cell Reports</em> 2 (5): 1151–8.</p>
</div>
<div>
<p>Wei, Leizhen, Arthur Samuel Levine, and Li Lan. 2016. “Transcription-Coupled Homologous Recombination After Oxidative Damage.” <em>DNA Repair</em>, May. doi:<a href="https://doi.org/10.1016/j.dnarep.2016.05.009">10.1016/j.dnarep.2016.05.009</a>.</p>
</div>
<div>
<p>Williams, Bryan RG. 1999. “PKR; a Sentinel Kinase for Cellular Stress.” <em>Oncogene</em> 18 (45): 6112–20. doi:<a href="https://doi.org/10.1038/sj.onc.1203127">10.1038/sj.onc.1203127</a>.</p>
</div>
<div>
<p>Williamson, C. M., A. Blake, S. Thomas, C. V. Beechey, J. Hancock, B. M. Cattanach, and J. Peters. 2013. “MRC Harwell, Oxfordshire.” <em>World Wide Web Site-Mouse Imprinting Data and References.[http://Www. Har. Mrc. Ac. Uk/Research/Genomic_imprinting]</em>.</p>
</div>
<div>
<p>Williamson, Christine M., Simon T. Ball, Claire Dawson, Stuti Mehta, Colin V. Beechey, Martin Fray, Lydia Teboul, T. Neil Dear, Gavin Kelsey, and Jo Peters. 2011. “Uncoupling Antisense-Mediated Silencing and DNA Methylation in the Imprinted Gnas Cluster.” <em>PLOS Genetics</em> 7 (3): e1001347. doi:<a href="https://doi.org/10.1371/journal.pgen.1001347">10.1371/journal.pgen.1001347</a>.</p>
</div>
<div>
<p>Wilson, Vincent L., and Peter A. Jones. 1983. “DNA Methylation Decreases in Aging but Not in Immortal Cells.” <em>Science</em> 220 (4601): 1055–7.</p>
</div>
<div>
<p>Wong, Lee H., Kate H. Brettingham-Moore, Lyn Chan, Julie M. Quach, Melisssa A. Anderson, Emma L. Northrop, Ross Hannan, Richard Saffery, Margaret L. Shaw, and Evan Williams. 2007. “Centromere RNA Is a Key Component for the Assembly of Nucleoproteins at the Nucleolus and Centromere.” <em>Genome Research</em> 17 (8): 000–000.</p>
</div>
<div>
<p>Wood, Jason G., Sara Hillenmeyer, Charles Lawrence, Chengyi Chang, Suzanne Hosier, Will Lightfoot, Eric Mukherjee, Nan Jiang, Christoph Schorl, and Alexander S. Brodsky. 2010. “Chromatin Remodeling in the Aging Genome of Drosophila.” <em>Aging Cell</em> 9 (6): 971–78.</p>
</div>
<div>
<p>Wood, Jason G., Brian C. Jones, Nan Jiang, Chengyi Chang, Suzanne Hosier, Priyan Wickremesinghe, Meyrolin Garcia, et al. 2016. “Chromatin-Modifying Genetic Interventions Suppress Age-Associated Transposable Element Activation and Extend Life Span in <em>Drosophila</em>.” <em>Proceedings of the National Academy of Sciences</em> 113 (40): 11277–82. doi:<a href="https://doi.org/10.1073/pnas.1604621113">10.1073/pnas.1604621113</a>.</p>
</div>
<div>
<p>Wozniak, M. A., A. P. Mee, and R. F. Itzhaki. 2009. “Herpes Simplex Virus Type 1 DNA Is Located Within Alzheimer’s Disease Amyloid Plaques.” <em>The Journal of Pathology: A Journal of the Pathological Society of Great Britain and Ireland</em> 217 (1): 131–38.</p>
</div>
<div>
<p>Wreggett, K. A., F. Hill, P. S. James, A. Hutchings, G. W. Butcher, and P. B. Singh. 1994. “A Mammalian Homologue of Drosophila Heterochromatin Protein 1 (Hp1) Is a Component of Constitutive Heterochromatin.” <em>Cytogenetic and Genome Research</em> 66 (2): 99–103.</p>
</div>
<div>
<p>Wu, Huang, Qing-Fei Yin, Zheng Luo, Run-Wen Yao, Chuan-Chuan Zheng, Jun Zhang, Jian-Feng Xiang, Li Yang, and Ling-Ling Chen. 2016. “Unusual Processing Generates SPA LncRNAs That Sequester Multiple RNA Binding Proteins.” <em>Molecular Cell</em> 64 (3): 534–48. doi:<a href="https://doi.org/10.1016/j.molcel.2016.10.007">10.1016/j.molcel.2016.10.007</a>.</p>
</div>
<div>
<p>Wutz, Anton. 2011. “Gene Silencing in X-Chromosome Inactivation: Advances in Understanding Facultative Heterochromatin Formation.” <em>Nature Reviews Genetics</em> 12 (8): 542.</p>
</div>
<div>
<p>Xie, Zheng-Hua, Yan-Nv Huang, Zhao-Xia Chen, Arthur D. Riggs, Jian-Ping Ding, Humaira Gowher, Albert Jeltsch, Hiroyuki Sasaki, Kenichiro Hata, and Guo-Liang Xu. 2006. “Mutations in DNA Methyltransferase Dnmt3b in ICF Syndrome Affect Its Regulation by Dnmt3l.” <em>Human Molecular Genetics</em> 15 (9): 1375–85.</p>
</div>
<div>
<p>Xu, Jian, and Benjamin L. Kidder. 2018. “H4K20me3 Co-Localizes with Activating Histone Modifications at Transcriptionally Dynamic Regions in Embryonic Stem Cells.” <em>BMC Genomics</em> 19 (1): 514. doi:<a href="https://doi.org/10.1186/s12864-018-4886-4">10.1186/s12864-018-4886-4</a>.</p>
</div>
<div>
<p>Yang, Nuo, and Haig H. Kazazian Jr. 2006. “L1 Retrotransposition Is Suppressed by Endogenously Encoded Small Interfering RNAs in Human Cultured Cells.” <em>Nature Structural and Molecular Biology</em> 13 (9): 763.</p>
</div>
<div>
<p>Yearim, Ahuvi, Sahar Gelfman, Ronna Shayevitch, Shai Melcer, Ohad Glaich, Jan-Philipp Mallm, Malka Nissim-Rafinia, et al. 2015. “HP1 Is Involved in Regulating the Global Impact of DNA Methylation on Alternative Splicing.” <em>Cell Reports</em> 10 (7): 1122–34. doi:<a href="https://doi.org/10.1016/j.celrep.2015.01.038">10.1016/j.celrep.2015.01.038</a>.</p>
</div>
<div>
<p>Yokoyama, Amy S., John C. Rutledge, and Valentina Medici. 2017. “DNA Methylation Alterations in Alzheimer’s Disease.” <em>Environmental Epigenetics</em> 3 (2): dvx008.</p>
</div>
<div>
<p>Zampieri, Michele, Fabio Ciccarone, Roberta Calabrese, Claudio Franceschi, Alexander Bürkle, and Paola Caiafa. 2015. “Reconfiguration of DNA Methylation in Aging.” <em>Mechanisms of Ageing and Development</em>, Biomarkers of human ageing, 151 (November): 60–70. doi:<a href="https://doi.org/10.1016/j.mad.2015.02.002">10.1016/j.mad.2015.02.002</a>.</p>
</div>
<div>
<p>Zaqout, Sami, and Angela M. Kaindl. 2016. “Golgi-Cox Staining Step by Step.” <em>Frontiers in Neuroanatomy</em> 10 (March). doi:<a href="https://doi.org/10.3389/fnana.2016.00038">10.3389/fnana.2016.00038</a>.</p>
</div>
<div>
<p>Zhang, Weiqi, Jingyi Li, Keiichiro Suzuki, Jing Qu, Ping Wang, Junzhi Zhou, Xiaomeng Liu, et al. 2015. “A Werner Syndrome Stem Cell Model Unveils Heterochromatin Alterations as a Driver of Human Aging.” <em>Science</em> 348 (6239): 1160–3. doi:<a href="https://doi.org/10.1126/science.aaa1356">10.1126/science.aaa1356</a>.</p>
</div>
<div>
<p>Zhang, Zhiyong, Chun Deng, Qianjin Lu, and Bruce Richardson. 2002. “Age-Dependent DNA Methylation Changes in the ITGAL (CD11a) Promoter.” <em>Mechanisms of Ageing and Development</em> 123 (9): 1257–68.</p>
</div>
<div>
<p>Zhao, Jinying, Yun Zhu, Jingyun Yang, Lin Li, Hao Wu, Philip L. De Jager, Peng Jin, and David A. Bennett. 2017. “A Genome-Wide Profiling of Brain DNA Hydroxymethylation in Alzheimer’s Disease.” <em>Alzheimer’s &amp; Dementia</em> 13 (6): 674–88. doi:<a href="https://doi.org/10.1016/j.jalz.2016.10.004">10.1016/j.jalz.2016.10.004</a>.</p>
</div>
<div>
<p>Zhao, Tao, Thomas Heyduk, C. David Allis, and Joel C. Eissenberg. 2000. “Heterochromatin Protein 1 Binds to Nucleosomes and DNA in Vitro.” <em>Journal of Biological Chemistry</em> 275 (36): 28332–8.</p>
</div>
<div>
<p>Zimmerly, S., H. Guo, P. S. Perlman, and A. M. Lambowitz. 1995. “Group II Intron Mobility Occurs by Target DNA-Primed Reverse Transcription.” <em>Cell</em> 82 (4): 545–54.</p>
</div>
<div>
<p>Zoetmulder, Marielle, Heidi B. Biernat, Miki Nikolic, Lise Korbo, Lars Friberg, and Poul J. Jennum. 2014. “Prepulse Inhibition Is Associated with Attention, Processing Speed, and 123i-FP-CIT SPECT in Parkinson’s Disease.” <em>Journal of Parkinson’s Disease</em> 4 (1): 77–87. doi:<a href="https://doi.org/10.3233/JPD-130307">10.3233/JPD-130307</a>.</p>
</div>
<div>
<p>Zuo, Xiaopan, Jipo Sheng, Ho-Tak Lau, Carol M. McDonald, Monica Andrade, Dana E. Cullen, Fong T. Bell, Michelina Iacovino, Michael Kyba, and Guoliang Xu. 2011. “The Zinc Finger Protein Zfp57 Requires Its Cofactor to Recruit DNA Methyltransferases and Maintains the DNA Methylation Imprint in Embryonic Stem Cells via Its Transcriptional Repression Domain.” <em>Journal of Biological Chemistry</em>, jbc. M111. 322644.</p>
</div>
</div>
</div>






            </section>

          </div>
        </div>
      </div>
<a href="6-bioinformatics-data-processing.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"download": null,
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
